export const uncertainty_visualization_corpus_topic_corpus = [{"topic_index": "0", "topic_name": "0_gpu_rendering_ray_rt", "topic_size": "98", "topic_words": ["gpu", "rendering", "ray", "rt", "performance", "tracing", "opengl", "cpu", "gpus", "culling", "vmd", "image", "hmd", "egl", "hardware", "movie", "optix", "depth", "threads", "geometry"], "topic_summary": "optix provides the fundamental machinery for gpu - accelerated ray tracing for incorporation into other packages like tachyon and vmd . optix internally implements several high performance gpuoptimized spatial acceleration data structures based on bounding volume hierarchies bvh and eliminates the need for developers to implement the routines themselves . to facilitate the development of gpu ray tracing engines , nvidia has released the optix ray tracing toolkit , which provides the fundamental components for scheduling ray intersection tests , managing the mapping of work to simd units , and coping with various hardware limits . in order to improve turnaround times for photorealistic rendering jobs in vmd tachyon , we have begun work on integrating optix , a cuda - based gpu - accelerated ray tracing toolkit . as a central component of our approach , we present an efficient parallel ray tracing framework written in a custom spmd language similar to nvidia cuda , which enables efficient auto - vectorization of code for both cpus and mics .", "best_matching_documents_indices": [258, 132, 138, 16, 27, 283, 28, 32, 296, 57]}, {"topic_index": "1", "topic_name": "1_trajectories_ligand_energy_simulation", "topic_size": "97", "topic_words": ["trajectories", "ligand", "energy", "simulation", "simulations", "md", "exploration", "analysis", "focus", "trajectory", "view", "energies", "interaction", "protein", "active", "they", "see", "it", "temporal", "our"], "topic_summary": "vmd , another popular program , is mainly devoted to the visualization of large molecular complexes and the analysis of md trajectories . other packages perform the analysis of the trajectories only by extracting information from the simulation , not by providing a unified system for and analysis . the supplementary material shows larger views of those two software packages that include all the text that usually appears in the working window . energies are the contacts or positions of the amino acids in the structure . to sum up , our contributions are : a compact set of representations that simultaneously convey structural and functional information on protein - ligand interactions .", "best_matching_documents_indices": [259, 264, 168, 139, 179, 180, 20, 22, 86, 216]}, {"topic_index": "2", "topic_name": "2_motions_motion_correlation_covariance", "topic_size": "91", "topic_words": ["motions", "motion", "correlation", "covariance", "groups", "group", "matrix", "residues", "trajectory", "techniques", "correlated", "conformational", "dynamics", "eigenvectors", "eigenvector", "rigid", "clustering", "model", "velocity", "conformations"], "topic_summary": "certainly more research will help identify and visualize essential degrees of freedoms .. using the conformational distance defined by equation , we can now characterize the trajectory , n n by a matrix dnm , of conformational distances between the configurations in it . however , small conformational changes as those that show up as a fine structure on the plane map are easily lost in this representation . for d , this produces a map of the trajectory that captures the geometrical similarity and dissimilarity of many configurations . wellknown methods include particle advection , streamlines , and line - integral convolutions . typically these representations are overlays , or animations of multiple traces of the molecular backbone , with each trace representing a different point in and gerstein .", "best_matching_documents_indices": [282, 294, 39, 300, 301, 48, 61, 194, 198, 206]}, {"topic_index": "3", "topic_name": "3_haptic_students_force_learning", "topic_size": "91", "topic_words": ["haptic", "students", "force", "learning", "molecule", "docking", "ligand", "haptics", "participants", "virtual", "feedback", "binding", "ivr", "educational", "forces", "lab", "teachers", "interaction", "experience", "our"], "topic_summary": ", . at the microscopic level , the water molecule ionizes into hydroxide anions and hydrogen ions . at the macroscopic level of the system , the user completes the. in fig . in fig .", "best_matching_documents_indices": [161, 165, 72, 9, 42, 74, 168, 77, 17, 82]}, {"topic_index": "4", "topic_name": "4_network_proteins_pathway_networks", "topic_size": "69", "topic_words": ["network", "proteins", "pathway", "networks", "interactions", "protein", "layout", "pathways", "nodes", "proteomic", "node", "graph", "experimental", "link", "edge", "signaling", "cell", "edges", "researchers", "biological"], "topic_summary": "the study of protein - protein interaction networks , gene - regulatory networks , protein pathways networks , and protein signaling networks possibly initiated a need to visualize these networks in terms of protein structures . the same interaction network visualized by the new algorithm is shown in figure , and the interactions between proteins and human proteins are more prominent in this network . the inspection of protein interaction neighborhoods is important for analysis . we developed a new algorithm for visualizing the interactions between virus proteins and human proteins as a multi - layered network . experimental results of the algorithm show that the interaction networks visualized by the new algorithm have improved the readability of the networks by reducing edge crossings while making the interactions between heterogeneous proteins prominent .. shows the same network visualized by the new method .", "best_matching_documents_indices": [290, 98, 164, 35, 166, 168, 297, 47, 177, 178]}, {"topic_index": "5", "topic_name": "5_abstraction_structure_structural_amino", "topic_size": "67", "topic_words": ["abstraction", "structure", "structural", "amino", "structures", "protein", "alignment", "acid", "representation", "molecule", "atoms", "visualization", "molecular", "information", "sequence", "figure", "molecules", "detailed", "ribbon", "visual"], "topic_summary": "based on the structural data that is available in large molecular databases , and by taking inspiration from traditional hand - made illustration styles , we describe how to visualize continuous transitions between different stages of structural abstraction as well as aspects of spatial perception and illustrativeness , fig . the field has developed a range of visual representations that provide different levels of abstraction see for a survey . these highly abstracted diagrams are valuable for providing a summary of a large molecule . for showing the internal composition of a molecule , many abstractions exist ranging from models that show every atom and bond , such as a ball and stick model , to highly abstracted representations , such as ribbon diagrams figure . it allows the user to compare several complex structures side - by - side or by superimposing them , and to position each structure freely in the", "best_matching_documents_indices": [264, 16, 146, 156, 287, 32, 291, 292, 171, 53]}, {"topic_index": "6", "topic_name": "6_cavities_ligand_cavity_voronoi", "topic_size": "57", "topic_words": ["cavities", "ligand", "cavity", "voronoi", "spheres", "sphere", "paths", "all", "molecule", "atom", "receptor", "probe", "channels", "les", "diagram", "path", "computed", "detection", "surface", "based"], "topic_summary": "furthermore , the positions of the ligand can be discretized . generally , a molecular path is a path of a small molecule or ion within a larger molecule . note that we use the same samples for the ligand positions and the distance function . as for most of the previous techniques , the protein is often approximated by the atom positions or the hard sphere model . in phase i , we uniformly sample the ligand positions and initialize the discrete distance function .", "best_matching_documents_indices": [132, 168, 137, 265, 233, 50, 120, 222]}, {"topic_index": "7", "topic_name": "7_cluster_molecules_flow_molecule", "topic_size": "56", "topic_words": ["cluster", "molecules", "flow", "molecule", "fragmentation", "clusters", "density", "molecular", "figure", "conformation", "simulations", "section", "fragment", "visualization", "number", "geometries", "vma", "these", "monomers", "temperature"], "topic_summary": "the particles are shown as spheres. the particles are shown as spheres. matlab does not support interactive display , and tec plot does not allow interactive display with multivariate visualization the display of multiple values of interest per particle . matlab does not support interactive display , and tec plot does not allow interactive display with multivariate visualization the display of multiple values of interest per particle . each simulation addresses a single molecule , and follows physical experiments by ionizing individual atoms of the molecule , with gaussian probability distribution .", "best_matching_documents_indices": [96, 263, 39, 103, 106, 206, 239, 144, 208, 210]}, {"topic_index": "8", "topic_name": "8_sequences_bioinformatics_sequential_sequence", "topic_size": "54", "topic_words": ["sequences", "bioinformatics", "sequential", "sequence", "protein", "patterns", "proteins", "mining", "consp", "database", "alignment", "family", "nucleic", "acids", "medical", "mds", "research", "annotation", "signatures", "psu"], "topic_summary": "bioinformatics ,. science , , feb . science , , mar . , . the nature of biological data , however , makes this difficult .", "best_matching_documents_indices": [163, 133, 103, 43, 142, 273, 242, 50, 245, 183]}, {"topic_index": "9", "topic_name": "9_animation_metamorphers_animations_molecular", "topic_size": "54", "topic_words": ["animation", "metamorphers", "animations", "molecular", "al", "et", "molecules", "metamorpher", "animated", "framework", "level", "rna", "create", "oxygen", "animator", "how", "environment", "complex", "description", "life"], "topic_summary": "with metamorphers , he claims , he can iterate on it in realtime in amatter of minutes . molecular systems are typically visualized with various degrees of abstraction . the use of zoom in animation , while keeping some point of reference usually a molecule in the center of the screen is often an effective way to deal with the issue of scale , but can also be disorienting if used too frequently . the work of mcgill reveals the power of visualization and animation to communicate cellular and molecular structures and dynamics . since the authors never produced animations of molecular machines before , three molecular machines were visualized and animated from scratch.", "best_matching_documents_indices": [129, 194, 74, 250, 45, 148, 122, 219]}, {"topic_index": "10", "topic_name": "10_vmd_parallel_analysis_swift", "topic_size": "54", "topic_words": ["vmd", "parallel", "analysis", "swift", "scripting", "tcl", "large", "code", "timeline", "namd", "computing", "trajectory", "many", "simulations", "mpi", "performance", "set", "python", "use", "tasks"], "topic_summary": "- scale md simulations are typically performed at remote supercomputing centers using thousands of processors , whereas the resulting data are often stored and analyzed over an extended period on smaller computing resources available locally to the user . even within a single simulation , the best parameter values are sensitive to which time step is being analyzed . the table is provided primarily to illustrate the simulation sizes that are possible as well as the computational requirements for such simulations . the table is provided primarily to illustrate the simulation sizes that are possible as well as the computational requirements for such simulations . for example , as atoms disperse , the density of the simulated system changes dramatically , and consequently , the performance behavior changes .", "best_matching_documents_indices": [258, 3, 163, 197, 229, 298, 109, 46, 240, 50]}, {"topic_index": "11", "topic_name": "11_ray_casting_sphere_rendering", "topic_size": "50", "topic_words": ["ray", "casting", "sphere", "rendering", "spheres", "surface", "shader", "gpu", "spherical", "patches", "atoms", "buffer", "grid", "pixel", "patch", "atom", "toroidal", "texture", "computed", "compute"], "topic_summary": "in case when atoms exceeds k , we specify atoms k and perform the generation of the remaining atoms , multiplies of k in the geometry shader . when approaching the h boundary we skip more and more atoms , , rendering only each. upon rendering the atoms , only the grid bounding box of each instance is drawn . the vertex shader decides on the number of atoms to be generated , , atoms parameter in the tessellation control shader . afterward , each atom is processed via the geometry shader to generate a billboard .", "best_matching_documents_indices": [129, 257, 259, 137, 25, 156, 30, 32, 167, 57]}, {"topic_index": "12", "topic_name": "12_web_client_webgl_server", "topic_size": "47", "topic_words": ["web", "client", "webgl", "server", "javascript", "browser", "rendering", "visualization", "host", "graphics", "upy", "hosts", "plug", "api", "websocket", "browsers", "java", "vrml", "online", "python"], "topic_summary": "springer vienna , . http : online accessed . , and visualization front - ends integrated with a molecular dynamics simulation program . glmol , . crokford , , .", "best_matching_documents_indices": [192, 194, 260, 110, 111, 144, 274, 146, 117, 126]}, {"topic_index": "13", "topic_name": "13_simulation_situ_flink_nodes", "topic_size": "46", "topic_words": ["simulation", "situ", "flink", "nodes", "shared", "cores", "analytics", "processes", "node", "comd", "flowvr", "analysis", "mpi", "transit", "data", "interleaved", "execution", "performance", "mb", "daemon"], "topic_summary": "the global grid represents only cells and only the non nul densities are sent to the staging nodes .. . despite the transfer of data , the performance is slightly impacted with a maximum cost of . despite the transfer of data , the performance is slightly impacted with a maximum cost of . this overhead can be explained by the amount of data to transfer atom positions versus cell densities , which is at least times larger . in the figure comd produces data every iterations .. gromacs trjconv tool modified as described in section iv - transferring the data asynchronously enables to outperform - , even though gromacs uses less cores .", "best_matching_documents_indices": [258, 163, 197, 46]}, {"topic_index": "14", "topic_name": "14_rendering_lod_al_abstraction", "topic_size": "45", "topic_words": ["rendering", "lod", "al", "abstraction", "hierarchical", "depth", "level", "shading", "method", "culling", "detail", "space", "molecular", "molecules", "levels", "render", "atoms", "performance", "models", "visualization"], "topic_summary": "level - of - detail approaches have a long history in computer graphics . on the left , only the edges of the triangle mesh are rendered . as shown in figure , the generated mesh can also be very low detailed , which allows interactive work with complex molecules on slower computers . these tools offer many analysis features for the specific application domains , however , their rendering performance does not allow for interactivity when rendering very large data sets , consisting of more than atoms , since most of them use polygon - based representations without level - of - detail techniques . with the hierarchical ray casting and no depth culling , we measured fps about faster .", "best_matching_documents_indices": [192, 129, 32, 67, 259, 291, 296, 10, 16, 212]}, {"topic_index": "15", "topic_name": "15_rule_rules_influence_din", "topic_size": "42", "topic_words": ["rule", "rules", "influence", "din", "systems", "dynamic", "panel", "network", "networks", "biological", "biology", "causal", "modeling", "based", "causality", "models", "time", "visualizing", "visualization", "links"], "topic_summary": "gostner et al . focus on rules rather than individual elements or types . , bousseau , and thollot . , . the derived data include rule clusters , averaged influence values when clustering globally or within a time window , and interpolated values during animation .", "best_matching_documents_indices": [147]}, {"topic_index": "16", "topic_name": "16_jmv_java_director_vp", "topic_size": "42", "topic_words": ["jmv", "java", "director", "vp", "fpv", "president", "research", "nih", "project", "national", "nsf", "funded", "science", "have", "under", "at", "up", "email", "tree", "institute"], "topic_summary": ".. . .. . .. . .. .", "best_matching_documents_indices": [129, 259, 135, 137, 269, 272, 288, 32, 35, 37]}, {"topic_index": "17", "topic_name": "17_napoli_interactions_ligands_ligand", "topic_size": "42", "topic_words": ["napoli", "interactions", "ligands", "ligand", "conserved", "residues", "hydrogen", "complexes", "ricin", "pdb", "atoms", "hydrophobic", "protein", "interacting", "composed", "bonds", "bond", "structures", "see", "interaction"], "topic_summary": "the system consists of five components : i secondary structure view , ii bond view , iii energy view , iv rmsf view , and v energy chart . despite the existence of several other interesting tools that allow analyzing protein - ligand interactions , there is not a single tool that couples large - scale statistical , visual , and interactive analysis . these play an important role in determining the three - dimensional structure adopted by proteins , because these bonds cause the molecule to fold into a specific shape . the residues from the hinge region , and are consistently interacting according to cali . moreover , our model was able to spot some residues that frequently interact with ligands through hydrophobic interactions : , , and .", "best_matching_documents_indices": [153, 290, 164, 133, 47, 180, 217]}, {"topic_index": "18", "topic_name": "18_surface_sphere_forces_external", "topic_size": "40", "topic_words": ["surface", "sphere", "forces", "external", "mesh", "vertex", "vertices", "vi", "deformable", "deformation", "ricci", "internal", "equation", "gradient", "contour", "force", "edge", "points", "computed", "curvature"], "topic_summary": "then we adjust the positions of the vertices to satisfy the edge length constraints . additionally , we want the deformation process to. supplementary data associated with this article can be found in the online version at .. input surfaces with subsequent re - meshing to establish correspondence . by direct computation , the gradient of the energy can be calculated as k ki j j k i , pki pkj the algorithmic details can be found in algorithm . newly created points are tracked in both directions to preserve correspondence .", "best_matching_documents_indices": [64, 128, 66, 68, 69, 169, 137, 10, 265, 181]}, {"topic_index": "19", "topic_name": "19_expression_brain_registration_atlas", "topic_size": "40", "topic_words": ["expression", "brain", "registration", "atlas", "anatomic", "gene", "image", "tissue", "genes", "expressing", "intensity", "template", "volume", "reference", "mapping", "sagittal", "mask", "annotation", "segmentation", "coronal"], "topic_summary": "there were no fiducials or other landmark methods used and the registration and quantification was done in a fully automated manner . the grid is a subdivision of the reference atlas space into isotropic cells and serves as the highest resolution for quantifying expression information for the purposes of search and comparison . our work , by contrast , is an automated volume registration scheme that allows both coronal and sagittal data to be processed in a genome - scale context of over images . in contrast to a fixed threshold approach , we found that it was necessary to employ an adaptive algorithm that was sensitive to process variation and artifacts . in contrast to a fixed threshold approach , we found that it was necessary to employ an adaptive algorithm that was sensitive to process variation and artifacts .", "best_matching_documents_indices": [64, 285, 93]}, {"topic_index": "20", "topic_name": "20_nonlinear_embedding_dimensional_methods", "topic_size": "39", "topic_words": ["nonlinear", "embedding", "dimensional", "methods", "classes", "class", "xi", "feature", "lda", "classifier", "embeddings", "data", "cancer", "dj", "measures", "expression", "pruning", "gene", "sets", "isomap"], "topic_summary": "showed similar results . showed similar results . dawson et al . quantitative and qualitative results and accompanying discussion is presented in section . quantitative and qualitative results and accompanying discussion is presented in section .", "best_matching_documents_indices": [262, 107, 75, 85, 151]}, {"topic_index": "21", "topic_name": "21_drawing_tool_geometry_event", "topic_size": "39", "topic_words": ["drawing", "tool", "geometry", "event", "user", "view", "tools", "visualization", "database", "design", "system", "geometric", "record", "monitor", "visualisations", "interactive", "new", "monitors", "interaction", "visualisation"], "topic_summary": "others require a particular type of visualisation in order to obtain a solution . examples are statistical plots such as the ramachandran plot or parallel coordinates , graphs and networks as well as visualisation techniques. the visualisation of macromolecular structures has become a standard tool in molecular biology with many tools available to support a variety of tasks using , , and representations , . similarly , many tasks can be solved using other visualisations or automatic approaches that we did not consider in this work . list of common tasks in the analysis of macromolecular structures , indicate whether each task can be solved with each visualisation , and discuss the benefits and drawbacks of the various representations with respect to the tasks .", "best_matching_documents_indices": [288, 161, 292, 69, 9, 171, 139, 205, 110, 123]}, {"topic_index": "22", "topic_name": "22_ligand_drug_binding_protein", "topic_size": "39", "topic_words": ["ligand", "drug", "binding", "protein", "molecules", "docking", "rigid", "interactions", "mutant", "molecular", "molecule", "md", "body", "dynamics", "genetic", "receptor", "simulations", "proteins", "design", "structure"], "topic_summary": "the protein data bank pdb acts ll rights reserved . they are reported as one or more models of molecular structure represented as cartesian coordinates corresponding to the averaged spatial positions of particular kinds of atoms . molecular interactions may also be observed from favourable interactions between different molecules . the formulations for rigid - body molecular dynamics are well documented in the literature rapport , . the total potential energy of two interacting molecules can be calculated using the contribution of bonded and non - bonded terms .", "best_matching_documents_indices": [3, 265, 23, 26, 155, 31, 290, 164, 37, 166]}, {"topic_index": "23", "topic_name": "23_sensor_inductor_capacitor_resonant", "topic_size": "39", "topic_words": ["sensor", "inductor", "capacitor", "resonant", "sensors", "temperature", "wireless", "coil", "inas", "monitoring", "capacitance", "pressure", "inductance", "sensing", "readout", "doping", "impedance", "\u03bcm", "humidity", "coupling"], "topic_summary": "the basic sensing principles are to detect changes of the sensor in response to the capacitance , resistance , inductance or even coupling distance through the physical parameter of interest by monitoring the resonant frequency , input impedance , or q - factor . using the following substitutions , fs lscs , q rs ls cs , where q is the quality factor of the lc sensor , and fs is the resonant frequency of the sensor , then the real part , the imaginary part , and the real part maximum of zin can be written as re zin f q f fs ffs fs f , i m zin f ffs ffs fs f , zmax re f fs fs in practical detection , the input impedance will undergo a phase rotation when one moves along the line from the terminals of the readout coil to the port of the measurement equipment due to the transmission line effect at higher frequency . the capacitor changes in response to the parameter of interest , resulting in a shift in its resonant frequency . to wirelessly interrogate the lc sensor , a readout coil is magnetically coupled with the sensor , and the resonant frequency of the sensor is detected through monitoring the impedance or input return loss of the readout coil . by using a frequency scan and observing the magnitude and phase information of the readout coil , the sensor reflected impedance can be evaluated and the point of resonance is identified as shown in fig .", "best_matching_documents_indices": [149, 150]}, {"topic_index": "24", "topic_name": "24_front_skin_morse_critical", "topic_size": "38", "topic_words": ["front", "skin", "morse", "critical", "triangulation", "triangles", "points", "saddle", "smale", "topological", "tangent", "point", "surface", "mesh", "homeomorphic", "creep", "complex", "vertex", "minimum", "delaunay"], "topic_summary": "specifically , we control the local size of the mesh to be proportional to the radius of the maximum principle curvature of the surface . specifically , we control the local size of the mesh to be proportional to the radius of the maximum principle curvature of the surface . edge length constraints . edge length constraints . however , our homeomorphic triangulation is guaranteed by locally controlling the triangles size .", "best_matching_documents_indices": [201, 289]}, {"topic_index": "25", "topic_name": "25_afm_cells_tip_microbeads", "topic_size": "35", "topic_words": ["afm", "cells", "tip", "microbeads", "cellulose", "br", "cell", "fluorescent", "cbh", "cellular", "nm", "nanofibrils", "image", "living", "measurement", "fluorescence", "crystal", "after", "light", "imaging"], "topic_summary": "cross - sectional analysis was performed to quantify the cbh i , which has a height of nm and is nm in length and width , respectively . the bond energy was estimated to be kbt , which is in good agreement with the value , kbt , estimated by scanning differential calorimetry . when this fiber is fragmented and the nanoparticle assemblies are mixed , the fragmented fiber works as a seed and the nanoaggregates are rapidly transformed into fibers . and are the amplitude images of single cbh i on the cellulose surface , the scale bar is nm . it has been known that a porphyrin derivative figure a forms metastable particulate aggregates in a solution and then gradually transforms into fibrous aggregates over time .", "best_matching_documents_indices": [24, 145, 172, 33]}, {"topic_index": "26", "topic_name": "26_namd_tcl_swift_replica", "topic_size": "35", "topic_words": ["namd", "tcl", "swift", "replica", "mpi", "replicas", "exchange", "partition", "module", "configuration", "runtime", "master", "file", "doswap", "communication", "filesystem", "partitions", "implementation", "float", "int"], "topic_summary": "it was fortunately realized that the mpi - based machine layer could be easily modified in exactly this way , allowing unmodified programs to operate independently within the local communicator of each partition , with stdout optionally redirected to a separate file per partition . it was fortunately realized that the mpi - based machine layer could be easily modified in exactly this way , allowing unmodified programs to operate independently within the local communicator of each partition , with stdout optionally redirected to a separate file per partition . significant performance variation was observed between runs for both the swift and native namd tcl implementations . the existing mpi - style replicasend recv tcl commands in namd were then re - implemented on the new partitioning interfaces , allowing namd replica - exchange scripts to run unmodified in namd but more efficiently on more platforms . therefore inter - replica communication on every timestep would never be efficient .", "best_matching_documents_indices": [258, 197, 298, 20, 119]}, {"topic_index": "27", "topic_name": "27_collaborative_coot_server_ammp", "topic_size": "33", "topic_words": ["collaborative", "coot", "server", "ammp", "collaboration", "virtual", "users", "user", "multiple", "remote", "session", "client", "shared", "tabletop", "view", "distributed", "modeling", "ainaan", "model", "wim"], "topic_summary": "most importantly : . the numerous disparate data sources in a manner suitable for collaborative work . currently , molecular modeling software allows single users to visualize , manipulate , and optimize models . some software can also perform complex computer simulations on large protein molecules . in our virtual environment , the users are able to visualize large molecular structures in real - time , create their own view , or share their view with others in the system .", "best_matching_documents_indices": [70, 295, 232, 108, 242, 254, 123, 190]}, {"topic_index": "28", "topic_name": "28_coot_freely_access_at", "topic_size": "33", "topic_words": ["coot", "freely", "access", "at", "can", "frontend", "frontier", "frontmost", "fronts", "frontto", "\ud835\udc60\ud835\udc5daw", "frp", "fru", "fruchterman", "fruitful", "frustum", "frystyk", "front", "fromwhole", "friedenberger"], "topic_summary": "empty", "best_matching_documents_indices": [130, 261, 136, 266, 267, 268, 141, 13, 276, 152]}, {"topic_index": "29", "topic_name": "29_film_thickness_nm_lubricant", "topic_size": "32", "topic_words": ["film", "thickness", "nm", "lubricant", "light", "disk", "ellipsometric", "films", "microscope", "differential", "thin", "dewetting", "optical", "fringe", "thick", "image", "surface", "disks", "obtained", "coherence"], "topic_summary": "in general , even if , dewetting occurs when the curve of has a local minimum point nm in fig . the result suggests that the difference in the dewetting pattern is determined by the sign of . was approximated as the time constant of the films with a thickness of nm was too small to obtain accurately in this experiment . the the curve had a wavy form and had local minima at and nm and maximum at nm due to the polar intermolecular interaction .", "best_matching_documents_indices": [0, 1, 36, 4, 84, 118, 218]}, {"topic_index": "30", "topic_name": "30_matrix_xi_kernel_dimensional", "topic_size": "32", "topic_words": ["matrix", "xi", "kernel", "dimensional", "vector", "space", "linear", "where", "latent", "nonlinear", "neighbors", "similarity", "each", "laplacian", "distances", "point", "eigenvectors", "embedding", "nearest", "feature"], "topic_summary": "the low dimensional representation can be used in subsequent procedures such as classification , pattern recognition , and so on . this quantifies how many points in the. , , pp . , , pp . , , pp .", "best_matching_documents_indices": [100, 133, 134, 262, 105, 75, 301, 80, 91, 127]}, {"topic_index": "31", "topic_name": "31_tree_nodes_graph_samples", "topic_size": "32", "topic_words": ["tree", "nodes", "graph", "samples", "mapper", "labels", "shannon", "prior", "graphs", "data", "spread", "\ud835\udc60\ud835\udc5da", "node", "\ud835\udc3ba", "pareto", "dataset", "distance", "xpos", "index", "generated"], "topic_summary": "see for more information . see for more information . the k. points . it can simplify the representation of a high - dimensional dataset by generating a network shows points on the plot .", "best_matching_documents_indices": [162, 127, 55, 183]}, {"topic_index": "32", "topic_name": "32_vertex_edge_paths_radius", "topic_size": "31", "topic_words": ["vertex", "edge", "paths", "radius", "voronoi", "atom", "point", "sphere", "spherical", "intersection", "start", "edges", "spheres", "regular", "vertices", "graph", "remove", "surface", "two", "empty"], "topic_summary": "the start and end vertices of an edge are elements of this set . give only a coarse description and the technique presented by medvedev et al . see for more information . the surface mesh . for each node l .", "best_matching_documents_indices": [227, 37, 169, 265, 121, 156, 222, 30, 95]}, {"topic_index": "33", "topic_name": "33_gfp_cisddp_microscopy_metallothionein", "topic_size": "31", "topic_words": ["gfp", "cisddp", "microscopy", "metallothionein", "proteins", "cell", "cells", "monomeric", "electron", "images", "nuclear", "were", "actin", "described", "cellular", "colocalization", "segmented", "distribution", "incubated", "purified"], "topic_summary": "the visualization techniques reported here improve our understanding of the - dimensional structures revealed by images sharpened by deconvolution . the examples shown below were computed with three distinct datasets . all rights reserved . it is capable of co - mapping hundreds of proteins and their distribution across a cell or tissue sample by running cycles of fluorescence tagging with monoclonal antibodies or other affinity reagents , imaging , and bleaching in situ . an important consideration in any statistical analysis procedure in high dimensional space is the fact that over fitting may occur .", "best_matching_documents_indices": [166, 235, 174, 115, 181, 56, 92, 62]}, {"topic_index": "34", "topic_name": "34_device_virtual_devices_displays", "topic_size": "29", "topic_words": ["device", "virtual", "devices", "displays", "buttons", "display", "awareness", "interaction", "movement", "hand", "environments", "mounted", "pinch", "joystick", "orientation", "gloves", "avatar", "button", "navigation", "environment"], "topic_summary": "this worked well on large and tiled displays and in the cave . intuitively , using a display should lead to a better understanding of structures . while the first , rasmol is by no means the only such application : there are over a hundred freely available molecular visualization applications available as of the time of this writing . most of these applications focus on the visualization of a single molecular structure with traditional mouse and keyboard interaction . similarly , large displays have been investigated , and have been found to have performance and preferential advantages .", "best_matching_documents_indices": [224, 97, 295, 72, 135, 171, 22, 123]}, {"topic_index": "35", "topic_name": "35_configurations_complex_ensemble_pair", "topic_size": "29", "topic_words": ["configurations", "complex", "ensemble", "pair", "pa", "protein", "contact", "view", "expert", "configuration", "selected", "complexes", "overview", "proteins", "primary", "properties", "experts", "ensembles", "docking", "individual"], "topic_summary": "our solution can be easily. for filtering . the view supports various filters , allowing users to filter the complex configurations based on the properties of individual amino acids from the primary protein using range filters as well as to explicitly enforce the presence or absence of the selected amino acids . below the status bar , the list of all applied filters is shown . the", "best_matching_documents_indices": [243, 245]}, {"topic_index": "36", "topic_name": "36_plate_well_markers_tangible", "topic_size": "28", "topic_words": ["plate", "well", "markers", "tangible", "pmv", "tilemap", "virtual", "image", "wells", "app", "ar", "artoolkit", "augmented", "captured", "images", "camera", "scanner", "cocktail", "pyartk", "interface"], "topic_summary": "also , the program provides a quick way to select all wells , or. visual - , with its numerous features for visualization and analysis , allows users to accelerate the process without risking the loss of crystals . the original version of the program considered the corner of a well as a and moved right downwards naming the sub - wells with subsequent letters the left image in fig . normally , the users have to inspect the crystal images , find their corresponding cocktails and write their scores into the screen file to prepare an analysis file . these programs require an analysis file containing cocktails and their crystal scores , in order to identify new crystalline conditions .", "best_matching_documents_indices": [68, 230, 71, 200, 51, 286]}, {"topic_index": "37", "topic_name": "37_bicluster_condition_biclusters_gene", "topic_size": "28", "topic_words": ["bicluster", "condition", "biclusters", "gene", "genes", "conditions", "biclustering", "jbc", "average", "if", "discrete", "algorithm", "data", "set", "discretization", "order", "sparseness", "size", "computed", "number"], "topic_summary": "let and be two overlapping biclusters , nsc the number of conditions shared by the bicluster , nsg the number of shared genes , and t a threshold . let and be two overlapping biclusters , nsc the number of conditions shared by the bicluster , nsg the number of shared genes , and t a threshold . let bc be the set of conditions , bg the set of genes in the bicluster. each gene and condition is tested separately , and only the five best are selected and added to the bicluster . similarly to the average condition variance , the average gene variance is computed by jbgj x g where g jbc j x jbc j x dcg : although small biclusters are more likely to occur randomly , they might contain highly relevant biological information .", "best_matching_documents_indices": [80, 85, 151]}, {"topic_index": "38", "topic_name": "38_color_participants_shadow_colors", "topic_size": "27", "topic_words": ["color", "participants", "shadow", "colors", "ambient", "surface", "shadowed", "luminance", "lighting", "lightness", "perceptions", "stereo", "stimuli", "shadows", "surfaces", "viewers", "cues", "design", "directional", "shading"], "topic_summary": "because the image color of data on the surface depends on shadows and shading , the apparent color of the data may not be the same as the unshaded key color in the legend . this lack of difference also indicates constancy effects participants mapped these patches to roughly equivalent colors despite significant changes in shadow darkening . these results collectively suggest that participants can account for the effect of shadows on surface colors . colors , shadow levels , general procedure , and stimulus distribution mirrored the condition described in the previous section , including unshadowed validation stimuli . participants matched surface colors to colors significantly lighter than the pixel value of the shadowed image color , and darker shadows did not always influence the apparent color fig .", "best_matching_documents_indices": [248]}, {"topic_index": "39", "topic_name": "39_xml_xslt_cml_vrml", "topic_size": "27", "topic_words": ["xml", "xslt", "cml", "vrml", "document", "chemical", "content", "web", "visant", "doc", "formats", "markup", "tag", "chemistry", "modl", "consortium", "xsl", "source", "perl", "xmlelement"], "topic_summary": "as a result , numerous xml applications and tools are available and many more are being developed by the month . once these technologies are ready for use , xml will be all set to rule the web . such internationally recognized standards are manifested as widely deployed web technologies such as html , sgml , and the xml family produced by the world wide web consortium and virtual reality modeling language vrml and extensible produced by the consortium . xml is a textual format for the interchange of structured data between applications , , . we believe that the full impact of the convergence between and xml is not yet fully understood , and intend this project to show the power of open standards and technologies .", "best_matching_documents_indices": [192, 35, 260, 139, 238, 82]}, {"topic_index": "40", "topic_name": "40_vessel_colon_lmap_distortion", "topic_size": "26", "topic_words": ["vessel", "colon", "lmap", "distortion", "flattening", "aortic", "branches", "w\u00fe", "preserving", "polyp", "abdominal", "configuration", "ilp", "cpr", "tree", "flattened", "visualization", "problem", "method", "vascular"], "topic_summary": "in this work , we presented lmap , a technique to conformally parameterize and deform selected regions of interest on an arbitrary surface . this often causes false intersections of vessels that. introduced in , usiv presents all branches in a single image without misleading overlap , with minimal distortion , thus preventing unnecessary user interaction and enhancing the readability of the visualization . though the size and resolution of. the expert found the current visualizations with geometric context preservation very promising , especially for training new radiologists on vc systems .", "best_matching_documents_indices": [64, 69, 62]}, {"topic_index": "41", "topic_name": "41_exon_exons_isoform_junction", "topic_size": "25", "topic_words": ["exon", "exons", "isoform", "junction", "isoforms", "samples", "expression", "abundance", "plots", "splicing", "collaborators", "alternative", "reads", "sample", "data", "abundances", "view", "gene", "genome", "genomic"], "topic_summary": "the tcga data provides expression data as an average for every exon , as is evident from the constant blocks in the expression view in figure . for example , early in the project we expected an isoform to be a well - defined combination of a set of exons , and we expected isoforms to differ only in terms of which of these exons they use . and datasets . individual genes are also occasionally depicted from left to right in reading direction , independent of the strand . when exploring this data , the domain expert noted that there is roughly equivalent expression of the exons that are not alternatively spliced between the two groups , exons .", "best_matching_documents_indices": [98, 100]}, {"topic_index": "42", "topic_name": "42_electronic_pages_chemical_journal", "topic_size": "25", "topic_words": ["electronic", "pages", "chemical", "journal", "molecular", "graphics", "review", "electron", "physics", "density", "richie", "physical", "environments", "visualizing", "dimensional", "surfaces", "atomic", "atomistic", "feature", "structures"], "topic_summary": "see for more information . new york : freeman . visualizing high - dimensional feature vectors . proteins , . berg , , tymoczko , , stryer , , stryer , , ed .", "best_matching_documents_indices": [257, 131, 26, 133, 299, 205, 206, 14, 49, 58]}, {"topic_index": "43", "topic_name": "43_lrc_mutants_mutated_plot", "topic_size": "25", "topic_words": ["lrc", "mutants", "mutated", "plot", "plots", "rigidity", "metric", "mutant", "mutations", "feature", "lrcwt", "protein", "wavemap", "parameter", "pairs", "features", "lrcmut", "wavelet", "among", "identify"], "topic_summary": "in figure , we see several pairs of amino acids , including and , that when mutated , have a non - trivial count upwards of or more of lrc scores among their mutants that had lrc scores at least standard deviations from the mean . in figure , we see several pairs of amino acids , including and , that when mutated , have a non - trivial count upwards of of lrc scores among their mutants that had lrc scores at least standard deviations from the mean . our approach identified these residues as the most resistant to mutations , because so many of the mutants involving those residues had lrc scores that were excessively high . our approach identified these residues as the most resistant to mutations , because so many of the mutants involving those residues had lrc scores that were excessively high . the heat map thus shows that when residue is mutated , in many cases the resulting mutant structure is vastly less stable that the wildtype , as might be expected .", "best_matching_documents_indices": [131, 109, 102, 23]}, {"topic_index": "44", "topic_name": "44_tunnel_tunnels_bottleneck_biochemists", "topic_size": "24", "topic_words": ["tunnel", "tunnels", "bottleneck", "biochemists", "boundary", "tetrahedra", "gates", "timesteps", "ath", "centerline", "sth", "surrounding", "dynamics", "tetrahedrization", "which", "it", "its", "chemical", "their", "important"], "topic_summary": "such a situation requires the development of methods enabling a fast , accurate , and intuitive exploration of tunnels within the whole molecular dynamics , their properties and compatibility with the given ligand . this feature would be very useful mainly in cases where a given tunnel has several narrow sites so the bottleneck position changes significantly within the molecular dynamics simulation . in this paper we proposed three novel methods for the evaluation of the biochemical relevance of tunnels in proteins in large ensembles of molecular dynamics . if exploring long molecular dynamics simulations thousands or tens of thousands of timesteps , the plotting of the large number of contours is not very comprehensible . however , the delaunay tetrahedrization provides a suitable structure for tracing the evolution and behavior of the different tunnels in subsequent time slots , and we will use it in our future research related to dynamic movement of proteins and tunnels .", "best_matching_documents_indices": [120, 233]}, {"topic_index": "45", "topic_name": "45_pathlines_solvent_molecules_abstract", "topic_size": "23", "topic_words": ["pathlines", "solvent", "molecules", "abstract", "inside", "trajectory", "pathline", "velocity", "clustering", "atoms", "clusters", "protein", "positions", "roi", "all", "b\u00f6s", "simulation", "paths", "motion", "selected"], "topic_summary": "the advantage of msms is that it is quite fast and offers a high - quality triangulation with arbitrary tessellation . to c the journal compilation c starting from early works about the definition of molecular surfaces and their efficient implementation ,. only for the trajectory of proteins , containing a total of more than atoms , the frame rate drops considerably below fps . this simplification can be computed in very little time and reduces the amount of atoms by a factor of to .", "best_matching_documents_indices": [169, 241, 22, 125, 30]}, {"topic_index": "46", "topic_name": "46_rectangles_chains_distance_zi", "topic_size": "22", "topic_words": ["rectangles", "chains", "distance", "zi", "walk", "rectangle", "solution", "two", "path", "cost", "\u00f0ak", "jzij", "moving", "covered", "dog", "along", "jzi", "paths", "polygonal", "nodes"], "topic_summary": "the pointers have been omitted here for algorithm clarity . the pointers have been omitted here for algorithm clarity . the path and the path . the path and the path . i and j are originally set to jaj and jbj , respectively , and", "best_matching_documents_indices": [272, 24, 195, 166]}, {"topic_index": "47", "topic_name": "47_bone_registration_atlas_change", "topic_size": "22", "topic_words": ["bone", "registration", "atlas", "change", "animal", "mh", "datasets", "view", "study", "skeleton", "bones", "volumes", "timepoints", "body", "resorption", "standardized", "anatomical", "approach", "volume", "dataset"], "topic_summary": "the registration error visualization helps. however , a further rigid per - segment registration can improve registration results and facilitate finer comparison , such as some of the methods described below . with side - by - side comparisons they are able to detect points of interest that should be further investigated . as shown in , the automatic atlas registration is accurate enough for many comparative purposes . the standardized layout was carefully designed , keeping in mind planarity , anatomical consistency", "best_matching_documents_indices": [62]}, {"topic_index": "48", "topic_name": "48_radiance_object_interreflections_parameters", "topic_size": "21", "topic_words": ["radiance", "object", "interreflections", "parameters", "configurations", "scene", "equation", "objects", "light", "cylinder", "parameter", "illumination", "scenes", "point", "analytic", "effects", "function", "reflectance", "tracer", "sphere"], "topic_summary": "the accuracy of the resulting function will depend on the number and variety of the configurations we provide . however , the number of samples required by this approach increases exponentially with the number of parameters . based on this analysis , we determine for each pair of objects the values of the different parameters . first , we discretize the domain of each parameter , limiting them to k different. in order to obtain a set of configurations with appropriate pairwise differences , we perform another step that reduces the number of spatial configurations even more .", "best_matching_documents_indices": [216, 16, 188]}, {"topic_index": "49", "topic_name": "49_frequent_binding_fava_conformational", "topic_size": "21", "topic_words": ["frequent", "binding", "fava", "conformational", "regions", "serine", "acids", "amino", "proteases", "bj", "csg", "proteins", "protein", "flexible", "structures", "from", "were", "specificity", "superposed", "comparison"], "topic_summary": "the summarization provided by the molecular surface map allows for a fast comparison of the different data sets . the preferred regions are derived from existing experimental structures and can be associated with frequently observed secondary structures , . protein structural alignment and a quantitative measure for protein structural distance . successful homology modeling techniques rely on the fact that structure is far more conserved than sequence . such propensities , more formally called empirical force - fields , are derived from experimentally resolved protein structures and then incorporated into sampling methods to reduce the high - dimensional conformational search - space .", "best_matching_documents_indices": [128, 133, 69, 169, 143, 50]}, {"topic_index": "50", "topic_name": "50_fragment_cylinder_texture_impostor", "topic_size": "20", "topic_words": ["fragment", "cylinder", "texture", "impostor", "if", "depth", "texels", "radius", "shader", "vertex", "point", "algorithm", "end", "else", "eye", "impostors", "primitive", "quad", "ambient", "discarded"], "topic_summary": "above , left : a rendering with constant width , anti - aliased lines . bottom for comparisons direct illumination , cast shadows , ambient occlusion . since we work in image space , contour lines can be easily added . the two corresponding shadow - maps are drawn side to side into a single texture . each grid cell points to the pixel of its first light in the texture .", "best_matching_documents_indices": [257, 265, 16, 49, 58]}, {"topic_index": "51", "topic_name": "51_sound_audio_events_sounds", "topic_size": "20", "topic_words": ["sound", "audio", "events", "sounds", "hydrogen", "users", "immersion", "time", "simulation", "proximity", "event", "scientific", "hrtf", "played", "research", "impaired", "because", "can", "hrtfs", "sighted"], "topic_summary": "yeung states that , using sound , data is distinguishable in nine dimensions . lunney et al . tests will include. the program is called sop , short for the sonification of proteins . and berenato et al .", "best_matching_documents_indices": [170, 293]}, {"topic_index": "52", "topic_name": "52_gel_intensity_peaks_electrophoresis", "topic_size": "20", "topic_words": ["gel", "intensity", "peaks", "electrophoresis", "fractions", "variable", "differential", "data", "volume", "expression", "spectrometry", "mass", "heightfield", "shown", "analysis", "visualization", "gels", "chromatography", "heightfields", "stack"], "topic_summary": "the plot exhibits several high peaks . the same applies for filtering out regions with high or low correlation . some of the intensity peaks are not separated but form streaks in the time - dimension . the number p depends on the experimental setup and can be in the range of tenths of thousands . an example of this is shown in figure .", "best_matching_documents_indices": [256, 107, 102]}, {"topic_index": "53", "topic_name": "53_chains_simplification_chain_discrete", "topic_size": "20", "topic_words": ["chains", "simplification", "chain", "discrete", "distance", "clause", "alignment", "simplified", "simplify", "pair", "xi", "problem", "\u00fen", "polygonal", "points", "polynomially", "algorithm", "complete", "solvable", "comparing"], "topic_summary": "we end up with almost identical chains whether is or . we end up with almost identical chains whether is or . the new algorithm provided better alignment results than the previous method for all empirical evaluations . we then presented algorithms to find , the. we then presented algorithms to find , the.", "best_matching_documents_indices": [272, 195]}, {"topic_index": "54", "topic_name": "54_uncertainty_secondary_diagram_structure", "topic_size": "19", "topic_words": ["uncertainty", "secondary", "diagram", "structure", "ribbon", "dssp", "amino", "experts", "acid", "sequence", "visualization", "positional", "uncertain", "diagrams", "methods", "helix", "helices", "different", "structural", "visual"], "topic_summary": "the differences regarding discrimination of secondary structures across assignment methods make direct visual comparisons impractical cf . specifically , our extension of sequence diagrams allows for a quantitative analysis of uncertainties , as absolute values are depicted as bars and as morphed secondary structures in in the visualization , the ribbon diagram is decorated with waviness . proposed a statistical framework for the quantification of structural uncertainties in molecular data . the variable position is already occupied by the tertiary structure , whereas color is occupied by secondary structure assignment or physicochemical property . knoll et al .", "best_matching_documents_indices": [65]}, {"topic_index": "55", "topic_name": "55_residue_contact_neighborhood_constraints", "topic_size": "19", "topic_words": ["residue", "contact", "neighborhood", "constraints", "contacts", "anisotropic", "residues", "statistical", "potentials", "local", "structure", "type", "similar", "cgap", "template", "contacting", "bonded", "plotted", "distance", "covalent"], "topic_summary": "in contrast to the global view of entire molecules our focus is on the visualization of statistical propensities in local molecular micro - environments - an approach that is similar in spirit to successful local alignment methods blast as opposed to global alignments . as described above , it is illustrated as a trace by connecting the residue positions contained in the neighborhood ordered by their residue number via edges . the contacting residues are further labeled with their residue type . the origins of using local statistical propensities in protein structure analysis can be traced back to the seminal work by ramachandran et al . it is noteworthy that the contacting residues are close in space but not necessarily close in sequence and hence do not represent contiguous fragments .", "best_matching_documents_indices": [69, 37, 168, 143, 25]}, {"topic_index": "56", "topic_name": "56_ppi_networks_network_centrality", "topic_size": "18", "topic_words": ["ppi", "networks", "network", "centrality", "metrics", "nodes", "algorithms", "graph", "local", "scalable", "parallel", "workflow", "community", "gephi", "sapiens", "datasets", "several", "hpc", "framework", "methods"], "topic_summary": "the degree centrality can be applied to a single node , specific nodes or all the nodes . , big - data visualization . references . gephi is open source , modular , and easily extensible through plugins . the format allows for dynamically adding multiple attributes to nodes and edges .", "best_matching_documents_indices": [203, 79, 244, 245, 87, 284]}, {"topic_index": "57", "topic_name": "57_spline_helix_backbone_ribbon", "topic_size": "18", "topic_words": ["spline", "helix", "backbone", "ribbon", "helices", "sheets", "rendering", "bonds", "structure", "cylinder", "figure", "sharp", "render", "ribbons", "sheet", "atoms", "element", "segments", "chain", "represented"], "topic_summary": "a slightly different viewpoint is used for the helices to show the correct per - pixel intersections and shading . a slightly different viewpoint is used for the helices to show the correct per - pixel intersections and shading . see figure . we achieve this change by rendering the backbone using very small generalized cylinder patches which initially have a circular cross - section . sheet rendering sheets are composed of multiple strands .", "best_matching_documents_indices": [65, 257, 291, 10, 139, 49, 58]}, {"topic_index": "58", "topic_name": "58_silicon_crystal_liquid_rij", "topic_size": "18", "topic_words": ["silicon", "crystal", "liquid", "rij", "atomic", "particle", "stochastic", "particles", "ij", "systematic", "amorphous", "simulation", "sij", "ijk", "si", "forces", "atoms", "volume", "bij", "models"], "topic_summary": "the models used are phenomenological in nature , , they do not include atomic - level detail , but do include essential molecular features such as the elongated shape , electric multipole moments and the relative strengths of attractive and repulsive intermolecular interactions . therefore , it has been subject of many experimental , , and theoretical , , studies . physicists attempt to improve this understanding with the use of numerical simulations . in general , the force fi may depend on the position of all particles , denoted by . despite continuous advances , quantum - mechanical methods are not yet able to study various dynamic processes , such as crack propagation , as this requires systems with significantly more atoms and larger time scales than ab - initio methods , , nowadays can deal with .", "best_matching_documents_indices": [101, 229, 39, 239, 209, 21, 249]}, {"topic_index": "59", "topic_name": "59_crack_dipole_orientation_glyphs", "topic_size": "18", "topic_words": ["crack", "dipole", "orientation", "glyphs", "field", "domains", "disorder", "dipoles", "material", "orientations", "oxygen", "glyph", "arrow", "tip", "atoms", "visualization", "sets", "direction", "unit", "vesicular"], "topic_summary": ", which allows for visually identifying large areas of strongly correlated orientations . also employed by grottel et al . , it is difficult to perceive patterns in the orientations of the dipole moments , even in a small neighborhood cf . a first. so , we expect the material to have gained volume .", "best_matching_documents_indices": [113, 106, 239]}, {"topic_index": "60", "topic_name": "60_genes_circadian_enrichment_go", "topic_size": "17", "topic_words": ["genes", "circadian", "enrichment", "go", "ontology", "term", "echo", "ontological", "categories", "gene", "category", "explorer", "regulation", "flies", "harmonic", "fold", "through", "terms", "enriched", "forced"], "topic_summary": "encore bridges several different databases , including go , uniprot , string , and quickgo , to provide data visualizations that enable a deeper understanding to be drawn from omics - scale circadian data . , and hurley , et al . thus , we have created an r packaged called mwshiny , which extends the shiny paradigm to include multiple windows , called multi - window shiny . performed gene ontology analysis through genomatix , using the gene ranker package , specifying only biological processes. we have above detailed a novel data analysis program , encore .", "best_matching_documents_indices": [60]}, {"topic_index": "61", "topic_name": "61_students_course_computational_student", "topic_size": "17", "topic_words": ["students", "course", "computational", "student", "biochemistry", "biology", "undergraduate", "semester", "freshman", "research", "advanced", "introduction", "biomolecular", "protein", "learning", "resources", "software", "their", "bioinformatics", "lecture"], "topic_summary": "the semester progressed students were exposed to a variety of proteins with different structures and biological functions . weekly lecture topics included : introduction to amino acids , protein secondary structure , unix linux user skills , basics of md simulation , biomolecular visualization theory and background , the definition and scope of bioinformatics , how to render quality images , literature searches and efficient reading practices , and good presentation skills . the introduction and presentation of concepts surrounding protein structure and function are critical to prepare students to comprehend and appreciate the complexity of modern molecular biology and biochemistry . current methods to introduce protein structure - function relationships focus on a two - dimensional - d presentation in textbooks and lecture presentations . therefore , this course satisfied our expectations for students to develop proficiency with biomolecular visualization software , familiarize themselves with concepts of computational biology , bioinformatics , computational thinking , and advanced research computing perform primary literature searches , explore protein.", "best_matching_documents_indices": [205, 277]}, {"topic_index": "62", "topic_name": "62_footprint_ndfs_screen_pixel", "topic_size": "16", "topic_words": ["footprint", "ndfs", "screen", "pixel", "ndf", "di", "radiance", "brdf", "projected", "lighting", "space", "bin", "probability", "corresponding", "angle", "area", "rays", "direction", "integration", "distribution"], "topic_summary": "enable scale - consistent rendering . this property also enables us to perform interactive zooming without re - computing all s - ndfs : s - ndfs for lower output resolutions can be obtained directly by linearly filtering and down - sampling the s - ndfs of higher resolutions . most such approaches are limited to projecting the geometry onto a planar image . an important property of s - ndfs is that they enable users to interactively change the lighting and material properties , as well as zooming and panning . while this approach results in significantly faster rendering times as compared to multi - sampling approaches , we will show that the visual results match those which can be achieved with modern , physicallycorrect dof renderers .", "best_matching_documents_indices": [112, 32, 199]}, {"topic_index": "63", "topic_name": "63_solvent_surface_molecular_visual", "topic_size": "16", "topic_words": ["solvent", "surface", "molecular", "visual", "kmax", "visualization", "protein", "representation", "interactive", "dynamics", "atoms", "mif", "macromolecule", "rendering", "it", "section", "vdw", "molecules", "interactivity", "atom"], "topic_summary": "computer graphics forum accepted for publication , . visual molecular dynamics . it has a relatively complex structure and showing it with all atoms results in a pretty packed visualization . the visual computer , . computer graphics forum , , .", "best_matching_documents_indices": [67, 38, 167, 81, 274, 222, 212, 156, 217, 188]}, {"topic_index": "64", "topic_name": "64_kaic_phosphorylation_din_agents", "topic_size": "15", "topic_words": ["kaic", "phosphorylation", "din", "agents", "dephosphorylation", "kaia", "phosphorylated", "rules", "rule", "kaib", "level", "influence", "firing", "inactive", "cycle", "at", "nodes", "oscillations", "when", "but"], "topic_summary": ", . , . this provides insight to the systems biologist as to which rules are likely to be indicated in the formation of biological pathways . species instead of patterns , it would require species and reactions . though this rule - based system is relatively small , it exhibits many of the complexities found in much larger models .", "best_matching_documents_indices": [147]}, {"topic_index": "65", "topic_name": "65_dive_datanodes_inheritance_datanode", "topic_size": "15", "topic_words": ["dive", "datanodes", "inheritance", "datanode", "sql", "dynameomics", "dataedges", "pipeline", "streaming", "plug", "data", "ontologies", "dashboard", "properties", "through", "api", "property", "scripting", "\u00b5scripting", "capabilities"], "topic_summary": "horrocks , ontologies and the semantic web , comm . here we discuss these features , their benefits , and how we see them integrating with existing technologies . dataedges implement multiple inheritance models . furthermore. extensive documentation and details of these capabilities are at .", "best_matching_documents_indices": [103]}, {"topic_index": "66", "topic_name": "66_quinary_structure_fruitful_frontmost", "topic_size": "15", "topic_words": ["quinary", "structure", "fruitful", "frontmost", "fronts", "frontto", "frp", "fru", "fruchterman", "frontier", "frustum", "frystyk", "fs", "fsas", "fses", "fsi", "frontend", "friendly", "frlayout", "friis"], "topic_summary": "empty", "best_matching_documents_indices": [160, 89, 2, 6, 7, 8, 11, 187, 271, 15]}, {"topic_index": "67", "topic_name": "67_method_subspaces_classification_msm", "topic_size": "15", "topic_words": ["method", "subspaces", "classification", "msm", "proteins", "subspace", "align", "proposed", "feature", "protein", "similarity", "class", "fold", "membrane", "methods", "view", "using", "based", "step", "discriminative"], "topic_summary": "therefore , a conventional k - nearest neighbor approach can be applied , where the canonical angles are used as the degree of similarity . gda is basically a generalization of linear discriminant analysis lda . finally , the similarity between protein structures. see for more information . see for more information .", "best_matching_documents_indices": [91]}, {"topic_index": "68", "topic_name": "68_spasm_double_code_swig", "topic_size": "15", "topic_words": ["spasm", "double", "code", "swig", "scripting", "file", "void", "modules", "swift", "python", "ptr", "particles", "include", "seconds", "discus", "module", "backup", "particle", "script", "run"], "topic_summary": "thus , a user can create vectors , particles , arrays , and other interesting objects all from within the scripting language interface . thus , a user can create vectors , particles , arrays , and other interesting objects all from within the scripting language interface . we could do this entirely within a swig interface file , by including a special c function for searching the particle data . we could do this entirely within a swig interface file , by including a special c function for searching the particle data . in the script , the commands directly map onto the underlying c functions given in the interface file .", "best_matching_documents_indices": [3, 298, 240, 119, 279]}, {"topic_index": "69", "topic_name": "69_sites_site_points_swapping", "topic_size": "15", "topic_words": ["sites", "site", "points", "swapping", "capacity", "lccvd", "point", "free", "cmin", "cmax", "constraint", "kd", "representatives", "voronoi", "slots", "pixels", "set", "sets", "distance", "step"], "topic_summary": "for a detailed discussion . reduced surface : an efficient way to compute molecular surfaces . however , as mentioned before , the values in the distance function can only increase . thus we call the kernel at least two times and in practice on average three to four times . it does not slow down as drastically as the original implementation for large numbers of sites .", "best_matching_documents_indices": [137, 227, 132, 278]}, {"topic_index": "70", "topic_name": "70_cell_par_cmp_pars", "topic_size": "15", "topic_words": ["cell", "par", "cmp", "pars", "cells", "lymphocytes", "graph", "bindings", "toponome", "view", "cmps", "table", "prostate", "brushing", "brush", "tissue", "probe", "been", "may", "number"], "topic_summary": "the original method requires a hierarchy being defined on the nodes . an additional step , which is carried out within the inner part of the. we introduce the term - cmp which refers. they are presented in a text inset of the graph view fig . the brushed rows are highlighted in blue and white .", "best_matching_documents_indices": [166]}, {"topic_index": "71", "topic_name": "71_microtubule_microtubules_cap_tubulins", "topic_size": "14", "topic_words": ["microtubule", "microtubules", "cap", "tubulins", "tubulin", "protofilaments", "growing", "gtp", "assembled", "generator", "shrinking", "fig", "disassembled", "random", "dynamic", "walk", "sheet", "end", "growth", "cell"], "topic_summary": "the microtubule generator creates the details of the microtubules on the molecular scale . typically described with the persistence length . the frequency in the first dimension is high enough so that the microtubule structure this work is licensed under a creative commons attribution license . all the parameters of the microtubule model as well as the visualization can be keyframed . to map a random walk to a given persistence length , the new direction of a step is constrained by the direction of the previous step .", "best_matching_documents_indices": [74]}, {"topic_index": "72", "topic_name": "72_mosumo_ontology_service_rdf", "topic_size": "14", "topic_words": ["mosumo", "ontology", "service", "rdf", "services", "owl", "semantic", "entities", "ontologies", "web", "electrostatic", "pdb", "surface", "entity", "model", "ligand", "resource", "delphi", "sio", "structure"], "topic_summary": "additional semantic data , such as more detailed biological terms and provenance information can be annexed if necessary , or , in fact , an existing formal ontology. each annotation may map to a single node in structure space , or to a set of nodes . mosumo breaks new ground in that it is a sophisticated , parameterized semantic web services for structural bioinformatics . not all of the concepts in this diagram can be thought of as physical objects : in fact the only concepts to be universally agreed upon are atoms , bonds , residues and chains and even these , at a certain level outside our scope , are disputable . the key aspects of our.", "best_matching_documents_indices": [26, 186, 242]}, {"topic_index": "73", "topic_name": "73_states_layout_transition_state", "topic_size": "14", "topic_words": ["states", "layout", "transition", "state", "graph", "transitions", "conformational", "conformations", "hierarchical", "edges", "spectral", "layouts", "clusters", "intermediate", "folding", "mdmap", "cluster", "tends", "trajectory", "folded"], "topic_summary": "we find that whenever transitions between two states occurs frequently , the spectral graph layout will place these nodes close together. while their work focuses primarily on the determination of intermediate states , our work also stresses the visualization and layout of the state transition graph arising from molecular dynamics trajectories . to achieve this , we compute a distance metric d , that measures the distance between the conformations in a high dimensional vector space rn where n is the number of atoms in the biomolecule . these are semi - stable conformations which persist for a period of time , and which may be visited repeatedly . let the set of n conformational states be given by s si i n .", "best_matching_documents_indices": [48]}, {"topic_index": "74", "topic_name": "74_mip_tiles_cache_ndf", "topic_size": "14", "topic_words": ["mip", "tiles", "cache", "ndf", "pixel", "particle", "casting", "per", "performance", "ray", "cost", "csc", "output", "rendering", "computed", "image", "cd", "ns", "zooming", "tile"], "topic_summary": "we furthermore denote the number of bins in each s - ndf by , and the number of samples that should be computed for a specific desired rendering quality by ns . we furthermore denote the number of bins in each s - ndf by , and the number of samples that should be computed for a specific desired rendering quality by ns . in stark contrast , ns is not a constant , and in order to achieve scale - consistent rendering or , simply : high quality it must be increased exponentially depending on the level of detail . in stark contrast , ns is not a constant , and in order to achieve scale - consistent rendering or , simply : high quality it must be increased exponentially depending on the level of detail . as we will show , our scale - consistent rendering not only significantly reduces noise , but also makes larger - scale structures more easily perceptible .", "best_matching_documents_indices": [112]}, {"topic_index": "75", "topic_name": "75_classifier_overview_validation_classifiers", "topic_size": "14", "topic_words": ["classifier", "overview", "validation", "classifiers", "decisions", "results", "corpus", "classification", "fp", "classifications", "surface", "mitchell", "detail", "support", "decision", "approach", "aggregate", "examining", "performance", "our"], "topic_summary": "however , they become harder to interpret when a single class dominates , and do not afford efficient visual aggregation . whole surface . our approach should apply more generally . these three observations use elements of our approach , with chosen overviews leading to details . grouping points into clusters provides a number of benefits .", "best_matching_documents_indices": [176]}, {"topic_index": "76", "topic_name": "76_parameter_search_parameters_cache", "topic_size": "13", "topic_words": ["parameter", "search", "parameters", "cache", "performance", "values", "capacity", "models", "tuning", "decomposition", "size", "best", "processors", "optimization", "model", "partitioning", "cell", "processor", "functional", "impact"], "topic_summary": "the results , particularly for step iii and for larger numbers of processors , are not always following a stable performance trend . but we found averages to be reliable measurements across so many runs of the code . now we look at the performance impact of individual parameters , and the validity of the models provided by the application programmer . we found that through the use of models , we examined a small fraction of the search space less than , and achieved performance within and often much closer to the best performance in our sampled search space . the one exception is step iii , with processors , where performance of other x , y , and z is slightly better than selected parameters in the functional model .", "best_matching_documents_indices": [29]}, {"topic_index": "77", "topic_name": "77_immersive_tests_participants_user", "topic_size": "13", "topic_words": ["immersive", "tests", "participants", "user", "test", "gfp", "comfort", "approach", "non", "tasks", "task", "difference", "users", "had", "environment", "head", "virtual", "experience", "move", "questionnaire"], "topic_summary": "in general , all tasks have been successfully completed by all participants , and of the participants , regardless the input device , considered the system to be easy to use . lastly , it was found that more participants. furthermore , believed the system can help them to understand the experiment scenarios and concepts , and thought that it was easy moving from the macro level to the micro level . the order of application of the tests was different for each user , in order to reduce the bias . in summary , comparing the usability results of the two testing setups shows that the is slightly better than the .", "best_matching_documents_indices": [72, 251]}, {"topic_index": "78", "topic_name": "78_microgrid_power_risk_tie", "topic_size": "13", "topic_words": ["microgrid", "power", "risk", "tie", "management", "energy", "profit", "wind", "scenario", "loads", "demand", "max", "batteries", "forecast", "grid", "price", "charging", "electricity", "load", "scenarios"], "topic_summary": ", pp . , pp . , pp . , pp . , pp .", "best_matching_documents_indices": [99]}, {"topic_index": "79", "topic_name": "79_kernel_ppiextractor_ppi_kernels", "topic_size": "13", "topic_words": ["kernel", "ppiextractor", "ppi", "kernels", "gene", "recall", "dip", "tree", "extraction", "bioppisvmextractor", "dictionary", "spt", "feature", "dependency", "intex", "based", "medline", "grammar", "parse", "features"], "topic_summary": "precision is harder to measure than recall , because we need an estimate of the number of false positives . ppiextractor uses only the stanford parser output to produce the parse tree and dependency structure path and graph information , yielding a comparable performance . text . the. in addition , recall errors occur in all the ppi extraction processing stages : ner , normalization , and ppi extraction .", "best_matching_documents_indices": [284]}, {"topic_index": "80", "topic_name": "80_rna_membrane_virion_modeling", "topic_size": "13", "topic_words": ["rna", "membrane", "virion", "modeling", "model", "lipid", "lipids", "polyline", "contours", "elements", "created", "models", "proteins", "structural", "viruses", "parent", "assembly", "proxy", "nucleoprotein", "see"], "topic_summary": "the models presented are based on the martini coarse - grained forcefield which does not represent all the atoms but simplifies heavy atoms by one coarse - grained cg particle . effective computational methods are available for modeling and visualizing the biomolecular components of cells and viruses . atomic structures of over a hundred thousand biomolecules are available at the protein data bank , and decades of research and development have generated a comprehensive toolbox of simulation , structure prediction , modeling , and visualization tools to utilize and extend this data , . the resulting polyline may have strong kinks . the dimensions of the system are nanometers x , y , and z respectively and represent individual trajectories of particles over almost microseconds stored in .", "best_matching_documents_indices": [68, 198]}, {"topic_index": "81", "topic_name": "81_viewpoint_views_view_visible", "topic_size": "13", "topic_words": ["viewpoint", "views", "view", "visible", "best", "measure", "objects", "algorithm", "points", "scene", "molecules", "select", "graphics", "atoms", "show", "visited", "projected", "point", "pi", "camera"], "topic_summary": "then , an information measure that computes the geometric and photometric entropy is found for each segment . as we are taking maximum values , the resulting estimator is slightly high if left as is , we would inspect too many viewpoints . this estimator is a little faster than the previous one . in order to estimate entropy , we build a function that is fast to evaluate . this.", "best_matching_documents_indices": [176, 196]}, {"topic_index": "82", "topic_name": "82_session_research_papers_chronological", "topic_size": "13", "topic_words": ["session", "research", "papers", "chronological", "scholarly", "networks", "topics", "conference", "proteomics", "proceedings", "bioinformatics", "science", "conferences", "scientific", "evolutionary", "divergent", "inbound", "outbound", "network", "ontology"], "topic_summary": ", pp . , pp . , pp . , pp . , pp .", "best_matching_documents_indices": [105]}, {"topic_index": "83", "topic_name": "83_biclusters_gene_condition_enriched", "topic_size": "12", "topic_words": ["biclusters", "gene", "condition", "enriched", "annotations", "bicluster", "seeds", "seed", "conditions", "algorithm", "genes", "go", "order", "set", "biologically", "prelic", "yeast", "ordering", "since", "were"], "topic_summary": "in section , the bicluster model and conventions used in this paper are explained and a detailed description of the algorithm can be found in section . hughes et al . the data is assumed to be a matrix of gene expression values with genes represented as columns and conditions as rows . the overlap threshold was set to percent . following the procedure described by prelic et al .", "best_matching_documents_indices": [85, 151]}, {"topic_index": "84", "topic_name": "84_gene_species_genes_entropy", "topic_size": "12", "topic_words": ["gene", "species", "genes", "entropy", "som", "tree", "mining", "phylogenomic", "set", "phylogenetic", "clustering", "cluster", "prototype", "patterns", "congruent", "pattern", "phylogenetically", "each", "character", "probability"], "topic_summary": "hierarchical clustering of genes . the gene distribution on the som plane p extracts the representative features from each gene in a uniform format by tracing the projection of each gene on the som plane p . the gene clustering is conducted through hierarchically clustering the prototypes of genes into natural grouped clusters . we find the distributions of genes on a som plane showing interesting patterns : the genes with similar gene distributions are clustered close together on the som plane . introduction to the entropy theory in phylogenomics entropy is a metric to measure the information uncertainty in information theory .", "best_matching_documents_indices": [80]}, {"topic_index": "85", "topic_name": "85_tensor_defect_defects_anisotropy", "topic_size": "12", "topic_words": ["tensor", "defect", "defects", "anisotropy", "physicists", "streamtubes", "diffusion", "field", "biaxiality", "crystal", "topological", "metrics", "core", "techniques", "eigenvalues", "\u03bb\u03bb\u03bb", "physics", "data", "nematic", "locating"], "topic_summary": "grottel et al . references . the visualization techniques presented here are completely general and can be used with any of these generalized models , as well as any intermolecular interaction model including atomistic ones , and produce answers very quickly to questions regarding core structure and defect dynamics in these systems . this displacement seemed rather subtle because we could explicitly determine it from the data . the present techniques have a number of strong advantages compared to the previous approach used by physicists .", "best_matching_documents_indices": [113, 229, 39]}, {"topic_index": "86", "topic_name": "86_pockets_pocket_druggable_druggability", "topic_size": "12", "topic_words": ["pockets", "pocket", "druggable", "druggability", "acid", "amino", "residue", "proteins", "pairs", "geodesicsom", "result", "between", "protein", "reactive", "residues", "dataset", "datasets", "extraction", "tyrosine", "tend"], "topic_summary": "each pocket in the list contains information about atoms constituting a pocket . this technique goes through the following procedures , and extracts pockets from protein surfaces . this technique goes through the following procedures , and extracts pockets from protein surfaces . section a introduces the protein surfaces , and then section b introduces the pocket extraction technique . the technique firstly extracts pockets from protein surfaces , and calculates distances between the bottom of the pockets and amino acid residues of the protein .", "best_matching_documents_indices": [41, 153, 37]}, {"topic_index": "87", "topic_name": "87_beta_voronoi_diagram_betamol", "topic_size": "12", "topic_words": ["beta", "voronoi", "diagram", "betamol", "quasi", "qtf", "vertices", "connolly", "atoms", "surface", "radius", "solvent", "spherical", "molecule", "triangles", "corresponding", "interface", "complex", "computed", "delphi"], "topic_summary": "a few years later , in , richards defined the solvent excluded surface ses , which he simply named molecular surface . only one year later , greer and bush proposed the first method to depict parts of the structure of the ses by computing the height field of the molecule with respect to a particular direction . see for more information . this is the most general case other unique cases are also dealt with using this algorithm . we would rate the parameter - based approach as better suited for more complex surfaces , while the force - directed approach is better suited for more globular molecules without deep cavities or large protrusions .", "best_matching_documents_indices": [37, 137, 169, 26, 30]}, {"topic_index": "88", "topic_name": "88_defect_iso_value_defects", "topic_size": "11", "topic_words": ["defect", "iso", "value", "defects", "density", "bins", "electron", "shows", "lattice", "values", "atoms", "bulk", "dataset", "function", "operators", "atom", "there", "volume", "field", "slicing"], "topic_summary": "the histogram is first smoothed using an gaussian kernel of appropriate width . the histogram is first smoothed using an gaussian kernel of appropriate width . finally , the values in the bins are averaged to get a single iso - value . finally , the values in the bins are averaged to get a single iso - value . please note that the change occurs in bins and respectively and that the average iso - value in these bins is around .", "best_matching_documents_indices": [299, 124, 14]}]