export const uncertainty_visualization_corpus = [{"uri": "0", "title": "Direct Visualization of a Small-Molecule Drug by Phase-Modulated Stimulated Raman Scattering Microscopy", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Terumasa Ito", "Yuki Obara", "Kazuhiko Misawa"], "summary": "Coherent Raman scattering microscopy, an emerging imaging technology using nonlinear-optical effects such as coherent anti-Stokes Raman scattering (CARS) or stimulated Raman scattering (SRS), has gained increasing attention because of its highly-sensitive, chemically-specific imaging capability with an intrinsic molecularvibrational contrast. Despite the recent technical advancements, specific detection of a small-molecule drug in living tissue or cells is still challenging. This is because the Raman signal of the targeted drug is easily buried in Raman scattering of abundant macromolecules such as lipids, proteins or water. The time-resolved CARS method is known as a promising solution to reduce such a Raman-induced background [1]. With this approach, we have developed a compact single-beam heterodyne CARS micro-spectroscopy system [2] that efficiently removes the short-lived Raman background and non-resonant background, as well as two-photon fluorescence background. However, as the conventional single-beam CARS requires a narrowband probe pulse to obtain high spectral resolution, the detection sensitivity has been limited by the weak probe power. In this paper we present single-beam phase-modulated stimulated Raman scattering (PM-SRS) microscopy that improves the sensitivity, while rejecting the above-mentioned backgrounds. Our setup is based on a pumpprobe configuration with three pulses (pump, phase-modulated (PM) probe, and local oscillator (LO) probe, illustrated in Fig. 1(a)). These pulses are obtained by dividing a single broadband spectrum into the three bands (Fig. 1(b)). Multiple Raman modes are excited by the impulsive pump, and a specific Raman mode can be selectively probed by temporal beam-coupling between the PM and LO probes via the Raman-induced refractive index modulation. To achieve both the sensitivity and spectral resolution, we allocate a large spectral bandwidth (164 cm) to the probes and use a new selective detection scheme based on the spectral focusing [3]. By giving a strong group delay dispersion to the probes (45000 fs/rad), we can obtain an improved spectral resolution down to 25 cm. High-speed phase-modulation and lock-in detection are also used to improve the signal-to-noise ratio. Using the newly-developed non-labeled microscopy, we imaged a droplet of sevoflurane, an inhaled anesthetic drug, in water (Fig. 1(c)). Despite the weak SRS signal of the drug molecules ( I/I ~ 3.0 \u00d7 10), we successfully visualized a clear vibrational contrast at the Raman peak, whereas non-resonant background is sufficiently removed. This result suggests that the PM-SRS imaging is a potentially powerful and robust tool for detecting Raman signals of small-molecule drugs in living cells and tissues.", "keywords": ["car", "scattering", "pm", "probe", "beam", "drug", "detection"], "document_vector": [84.569953, 8.991474], "paragraphs": [{"paragraph_vector": [-52.159061, 42.075428], "paragraph_keywords": ["scattering", "cars", "beam", "probe"]}, {"paragraph_vector": [-55.585472, 39.969448], "paragraph_keywords": ["drug", "signal", "detection", "droplet"]}], "content": {}, "doi": "10.1016/j.cag.2021.05.011"}, {"uri": "1", "title": "Ellipsometric microscope: Novel Method for Direct Visualization of Molecularly Thin Liquid Films", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kenji Fukuzawa", "Taichi Shimuta", "Hedong Zhang", "Yasunaga Mitsuya"], "summary": "In a magnetic disk drive, a molecularly thin film is currently used as a lubricant and controlling its kinetic behaviors is a key technology to design the head disk interface. Direct visualization of molecularly thin lubricant films is useful in investigating the kinetic properties. We demonstrated that the real time visualization is possible by using the ellipsometric microscope with a laser light source and white light source. Laser light source provides the narrow band and high intensity light. However, the SNR of the image was limited by noises due to the optical interference and the improvement of the SNR was needed for a thickness resolution of sub-nm. In this paper, we explained that the noises due to the optical interference can be reduced by using a white light source with lower coherence. In addition, combination of image processing techniques can provides a higher SNR. Subtracting an original image from the contrast-reversal image obtained by rotating the polarizer setting, weak contrasts can be enhanced and noise intensity reduced. The achieved SNR for a 3.2-nm PFPE film image is 61.4. This result indicates that the white light source-based and digitally enhanced ellipsometric microscope can provide the real time visualization of molecularly thin liquid films with a sub-nm thickness resolution.", "keywords": ["disk", "light", "snr", "laser", "noise", "nm", "film", "source", "interference", "image", "imaging", "microscope"], "document_vector": [75.065895, -3.617469], "paragraphs": [{"paragraph_vector": [-55.016281, 45.681621], "paragraph_keywords": ["film", "microscope", "imaging", "disk"]}, {"paragraph_vector": [-56.512737, 42.1725], "paragraph_keywords": ["noise", "light", "image", "m"]}, {"paragraph_vector": [-55.750175, 43.977828], "paragraph_keywords": ["film", "light", "source", "interference"]}, {"paragraph_vector": [-55.039161, 41.841278], "paragraph_keywords": ["microscope", "source", "snr", "nm"]}], "content": {}, "doi": "10.1016/j.cag.2005.10.003"}, {"uri": "2", "title": "Visualizing and Controlling Picometric Quantum Ripples in Molecules", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Hiroyuki Katsuki", "Kenji Ohmori"], "summary": "The wave nature of matter is at the heart of quantum theory. We are trying to control amplitudes and phases of matter waves in molecules, aiming at better understanding and novel applications of quantum theory. The wave nature of matter is at the heart of quantum theory. Although modem civilized societies are deeply indebted to inventions made possible by quantum mechanics, some mysteries still remain in quantum theory [1], and considerable potential for its application still exists. Amplitude and phase manipulation of matter waves is one of the keys to better understanding and novel applications of quantum theory. Such quantum manipulation is called \"coherent control\" and is expected to be useful in the development of advanced quantum technologies such as quantum information processing and bond-selective chemistry. One promising approach to the coherent control is using coherent light to modulate a matter wave with its optical fl\\rrescec phase. For example, irradiating a diatomic Tprobe molecule with a femtosecond (fs) optical probe (f or ns) pulse can create a quantum state where a \\ +1 A 2~ ~ ~ ~ (0 Q)pump control probe spatially localized atom wave, called a (phasecc'.kump (nnlockpd) , ~~~~~~~~~~~~~(phase lockl d) (lunlocked) \"wave packet (WP)\", travels back and pump (fs) & control (fs) ti forth along the bond axis. The phase of the optical field, which oscillates with a period of fs to attoseconds, is imprinted lIIdistance on the WP as its quantum phase. Hence Fig. 1. Interference of two vibrational wave-packets is induced by a manipulating the optical phase allows us pair of phase-locked femtosecond laser pulses on the B-state potential to manipulate the quantum phase of the of the 12 molecule and is observed by another phase-unlocked femtosecond or nanosecond laser pulse. (adopted from ref. 5) WP. We have developed an \"attosecond phase atom wave atom wave modulator (APM)\": a device for tuning the delay 0 temtosec between two fs laser pulses with attoseconds stability and resolution. We have utilized this APM to control 4 temtosec <quantum ripple quantum interference of two vibrational WP's in a 80 temtosec molecule and manipulate the amplitudes of the 16 picometer vibrational eigenstates within the WP's [2, 3]. 120 femtosec Recently we have succeeded in observing spatial pattems of wave functions sculptured by the 100 Iemtosec interference of two vibrational WP's in a molecule with Fig. 2. A schematic cartoon of a quantum ripple a picometric and femtosecond spatiotemporal resolution. generated in the iodine molecule under the typical This method has been applied to the iodine molecule as conditions of ref. 4. A transient picometric ripple appears only when two atom waves (vibrational wave shown in Fig. 1. And we have shown that transient packets) cross each other in about 100 femtoseconds. standing waves, which we call quantum ripples, appear and disappear synchronously with the periodic crossing of two counterpropagating WP's, as schematically shown in Fig. 2. We have also shown that the spatial pattem of the quantum ripple is highly sensitive to the phase information stored in a pair of the WP's [4]. Moreover we have demonstrated that a combination of this phase measurement and the amplitude measurement can be regarded as a robust approach to provide full quantum information of WP's [5]. This combined approach is schematically illustrated in Fig. 1. We are now developing quantum gates based on these experimental techniques [6].", "keywords": [], "document_vector": [68.282783, 16.159326], "paragraphs": [{"paragraph_vector": [-123.581565, 27.056144], "paragraph_keywords": []}], "content": {}, "doi": "10.1016/j.cag.2005.10.022"}, {"uri": "3", "title": "Flexible Molecular Docking", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["E. Fredj"], "summary": "Recent technologies, molecular dynamics, genetic algorithms, parallel and distributed computing, virtual reality, and high-speed networking, underlie a new approach to the computational study of how macromolecules interact or dock together. With the MIDAS program, a user in a virtual reality environment can interact with a genetic algorithm running on a parallel computer to help in the search for likely geometric configurations. The final configuration is obtained with the help of the molecular dynamics, which allow the system to relax to its correct position. In conclusion, biologist people are able to utilize the power of parallel computers to simulate Molecular Docking on flexible biological system in greater detail than ever before.", "keywords": ["der", "allows", "molecule", "program", "interaction", "simulation", "dynamic", "processor", "protein", "rd", "algorithm", "string", "ligand", "distributed", "system", "computer", "ga", "ieee", "position", "interact", "parallel", "atom", "energy", "receptor", "use", "docking", "load", "time", "force", "problem", "optimization"], "document_vector": [-18.131818, -34.278057], "paragraphs": [{"paragraph_vector": [-0.929098, -53.479549], "paragraph_keywords": ["interactions", "molecules", "interaction", "macromolecules"]}, {"paragraph_vector": [-1.095714, -53.648986], "paragraph_keywords": ["energy", "interactions", "interaction", "molecules"]}, {"paragraph_vector": [-2.065114, -48.526538], "paragraph_keywords": ["ligand", "optimization", "strategy", "molecules"]}, {"paragraph_vector": [-4.227989, -53.247303], "paragraph_keywords": ["overlap", "protein", "atoms", "run"]}, {"paragraph_vector": [-133.149108, 61.666343], "paragraph_keywords": ["force", "atom", "processor", "atoms"]}, {"paragraph_vector": [-137.582626, 59.133403], "paragraph_keywords": ["processors", "atoms", "rd", "pn"]}, {"paragraph_vector": [-23.555421, -45.555252], "paragraph_keywords": ["algorithm", "strings", "slave", "string"]}, {"paragraph_vector": [35.457374, -48.846649], "paragraph_keywords": ["algorithm", "ligand", "ga", "program"]}, {"paragraph_vector": [-147.702316, -70.177154], "paragraph_keywords": ["docking", "processors", "simulation", "fig"]}, {"paragraph_vector": [-108.012275, 16.948493], "paragraph_keywords": ["dynamics", "optimization", "computer", "ieee"]}, {"paragraph_vector": [157.941329, 58.344898], "paragraph_keywords": ["distributed", "computer", "graphics", "operating"]}], "content": {}, "doi": "10.1016/S0097-8493(98)00091-0"}, {"uri": "4", "title": "empty", "timestamp": "2002", "rating": "0.0", "annotation": "", "tags": [], "authors": ["K. Fukuzawa", "T. Shimuta", "T. Yoshida", "H. Zhang", "Y. Mitsuya"], "summary": "Introduction High-density hard disk drives require a very low flying height, less than 10 nm. To meet this demand, the current thickness of the lubricant is of the order of 1 nm. Therefore, distribution control of molecularly thin lubricants is indispensable to high-density recording. Dewetting of a lubricant produces a large droplet of a spontaneously cohered lubricant and it may cause a serious obstacle for distribution control. It is considered that dewetting of a liquid film with a thickness of the 1-nm order occurs even if it does not have any contamination or defects, and the stability can be determined by the sign of the second derivative of the surface energy curve \u03b3\u201d(h) (stable when \u03b3\u201d(h) > 0 and unstable when \u03b3\u201d(h) < 0) for liquid films on a flat substrate. According to this theory, most of the polar lubricant films for magnetic disks could be unstable when the thickness is more than a monolayer. Therefore, dewetting is a more serious problem in near-contact recording because contact sliding may produce thicker lubricant films. In addition, the effect of surface roughness on the dewetting should be considered because current disks have small textures with a sub-nanometer height, which may act as the onset of dewetting like a defect does. However, a systematic analysis of the dewetting has not been reported yet. In this paper, we report our investigation of the effect of the surface energy profile and the roughness on dewetting by visualizing the dewetting process directly. Materials and methods Magnetic disks with a carbon overcoat and a perfluoropolyether lubricant (Fomblin-ZDOL4000) were used as samples. The surface topography of the disk was obtained with an optical interferencebased 3-D profiler (NewView 6000, Zygo). The lubricant was applied to the disk using the dip coating. The surface energy of the lubricant films was obtained by contact angle measurement. The dewetting process of lubricants was visualized by ellipsometric microscopy that we have developed recently [1]. The ellipsometric microscope can convert the film thickness into a dark-bright contrast and directly visualize thickness distribution with a sub-nanometer thickness resolution in a static condition without disk rotation as in an optical surface analyzer. In this experiment, observations started six minutes after application. Results and discussion Figure 1 shows the measured relationship between the surface energy and film thickness. The surface energy has a wavy form due to the polar intermolecular interaction. The solid line is a fitting curve and the stability was determined by the sign of \u03b3\u201d(h). Figure 2 shows the ellipsometric microscope images at t = 14 min with different initial thicknesses. The figure also shows the disk surface topography that indicates the ridge and groove texture in the circumferential direction, whose roughness was about 0.28 nm in Rrms. The 2.6-nm-thick film (Region 1 in Fig. 1) was stable. Although the 4.3and 5.8-nm-thick films (Region 2) dewetted, the detwetting patterns were quite different from each other. The pattern of the 4.3-nm-thick film did not follow the texture pattern whereas the 5.8-nm-thick one did. The disjoining pressure \u03a0 ( = -\u03b3\u2019(h))< 0 for the 4.3-nm-thick fim and \u03a0 > 0 for the 5.8-nm-thick one, although \u03b3\u201d(h) < 0 for both films. This means that not only \u03b3\u201d(h) but also \u03b3\u2019(h) (or \u03a0) should be considered. The negative disjoining pressure is known to cause another instability besides dewetting from spreading measurements. The difference in pattern may be related to this instability. In addition, the 8.6-nm-thick film dewetted and the pattern followed the texture although it should have been stable according to the surface energy (Region 3). This means that the texture acted as the onset and another type of dewetting (nucleation and growth) occured. Summary From observations of the dewetting patterns of lubricant on disks, we found that the growth of the dewetting is determined not only by the well-known instability of films but also by the disjoining pressure and surface topography of a disk even if it has sub-nanometer roughness. [1] K. Fukuzawa et al., ASME Journal of Tribology, Vol. 126, No. 4, pp. 755-760 (2004).", "keywords": ["disk", "surface", "film", "energy", "lubricant", "thickness", "dewetting"], "document_vector": [69.543701, -5.69574], "paragraphs": [{"paragraph_vector": [-56.114791, 48.362541], "paragraph_keywords": ["lubricant", "dewetting", "surface", "disk"]}, {"paragraph_vector": [-52.770675, 46.761024], "paragraph_keywords": ["texture", "surface", "film", "thickness"]}, {"paragraph_vector": [-58.53754, 45.794937], "paragraph_keywords": [".", "known", "pressure", "instability"]}], "content": {}, "doi": "10.1016/j.cag.2020.07.010"}, {"uri": "5", "title": "Visualisation of association rules based on a molecular representation", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Zohra BEN SAID", "Fabrice GUILLET"], "summary": "In order to extract interesting knowledge from large amounts of rules produced by the data mining algorithms, visual representations of association rules are increasingly used. These representations can help users to find and to validate interesting knowledge. All techniques proposed for visualisation of rules have been developed to represent an association rule as a whole without paying attention to the relations among the items that make up the antecedent and the consequent and the contribution of each one to the rule. In this paper, we propose a new visualisation representation for association rules that allows the visualisation of the items which make up the antecedent and the consequent, the contribution of each one to the rule, and the correlations between each pair of the antecedent and each", "keywords": ["proposed", "anticipation", "item", "visualisation", "antecedent", "interestingness", "measure", "ig", "user", "number", "function", "representation", "mining", "association", "rule"], "document_vector": [162.437591, 44.496284], "paragraphs": [{"paragraph_vector": [-165.806121, -39.267463], "paragraph_keywords": ["rules", "association", "rule", "mining"]}, {"paragraph_vector": [-164.877883, -40.111827], "paragraph_keywords": ["rule", "items", "item", "ig"]}, {"paragraph_vector": [-162.554595, -38.887992], "paragraph_keywords": ["items", "rule", "rules", "chose"]}, {"paragraph_vector": [-164.359954, -38.275615], "paragraph_keywords": ["items", "spring", "rules", "coordinates"]}, {"paragraph_vector": [-164.719177, -39.179943], "paragraph_keywords": ["rules", "user", "functions", "rule"]}, {"paragraph_vector": [-165.389251, -39.792972], "paragraph_keywords": ["rules", "rule", "interestingness", "user"]}, {"paragraph_vector": [-162.95993, -37.285121], "paragraph_keywords": ["association", "rule", "user", "rules"]}], "content": {}, "doi": "10.1016/j.cag.2020.05.025"}, {"uri": "6", "title": "Objective and subjective relations in data visualization. Examples from molecular biological data collections", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Igor Rojdestvenski"], "summary": "Natural science is where the objective and the subjective meet. When we build data collections, we want them to reflect the objective relations inherent in nature, but also classify these collections, adding subjective relations between data objects. Sometimes these two approaches conflict. In this paper we present two examples of blending objective and subjective relations in a single data collection, and discuss some possible generalizations. First example deals with the visualization of metabolic networks in three dimensions. We show that the network topology has a substantial subjective component in this case and discuss possible ways to minimize human inference. In the second example we present a collection of metabolic networks as a (subjective) hierarchical structure with (objective) binary relations between objects, and show how the data visualization may be made more logical via the procedure of \u201chierarchization of relations\u201d. Introduction. Currently a substantial research effort is devoted to automated reconstruction and representation of metabolic pathways and gene networks (see, e.g., KEGG: Kanehisa 1999; CSNDB: Takai-Igarashi & Kaminuma 1999; SPAD: Tsukamoto et al 1997; BioCyc: Karp et al, 2002; Kolpakov et al 1998; GSCope: Toyoda et al 2002). Automatic visualization of metabolic pathways plays a significant role in such effort, and becomes an important problem on its own (Becker&Rojas 2001). A successful visualization is a key to understanding the intricate topology of different parts of metabolic and regulatory networks. Graphical visualization of metabolic pathways has to be, at the same time, information dense and not \u201coverloaded\u201d, recognizable and unified, close to traditional and algebraically consistent. In fact, not only the pathway discovery, as, for example, Karp (2001) suggested, becomes a case for computational symbolic theories, but pathway visualization as well. Pathway graphics and topology contains, although in a non-textual form, vast amount of information, which is a result of deep understanding of pathway functioning. Recently we designed Metabolic Network Visualizer (MNV) (Rojdestvenski&Cottam 2003), a system to create descriptions of metabolic networks using a simple XML-compliant data model, and to visualize them in three dimensions as virtual immersive worlds. The system comprises four principal components: MNV language standard and parser, MNV to VRML translator, interactive pathway constructor and custom made converters for existing pathway databases. All components are unified via Perl and HTML based GUI elements which are processed by standard Internet browsing software with the installed VRML plug-in. GUI and all the system\u2019s components are written in Perl, HTML and Javascript. The choice of languages was determined by convenience and standartization of GUI and suitability of Perl for grammatical text analysis. Currently our system has been applied to an open subset of the BioCyc database (Karp et al, 2002), namely, the AgroCyc database. It can be accessed at www.patronov.net/sciencevr/mnv/indexview.html. Species and co-species in metabolic networks. Inference vs objectivity. When working on MNV we encountered several interesting problems. One of them was the \u201cspecies vs co-species\u201d paradox. Conventionally the reactants that participate in the chemical reactions are termed substrates and cosubstrates (that are used up in the reaction), and products and coproducts (that are produced in the reaction). Normally the term \u201ccoproduct\u201d is understood as a side product of the reaction, that is not part of the \u201cpathway\u201d. \u201cCosubstrate\u201d is interpreted similarly. The problem is that it takes significant amount of inference to determine which reactants are parts of the pathway and are, concomitantly, substrates and products, and which are \u201cside species\u201d, i.e. coproducts and cosubstrates. The Fig. 1 shows part of the TCA cycle metabolic pathway from the AgroCyc database (Karp et al, 2002), with arrows illustrating the above definitions. The products and substrates (further termed \u201cspecies\u201d) represent vertices of the pathway graph (\u201ccitrate\u201d, \u201ccis-aconitate\u201d, isocitrate\u201d), while \u201ccospecies\u201d (coproducts and cosubstrates, represented by \u201cH20\u201d, \u201cacetyl-CoA\u201d and \u201cCoenzyme A\u201d) are traditionally depicted by side arrows, not vertices. In the MNV language reactants participating in a metabolic network are described by <species> tag, which has an attribute \u201cco\u201d with possible values co=\u201cyes\u201d and co=\u201dno\u201d, to distinguish species from co-species. Proceedings of the Seventh International Conference on Information Visualization (IV\u201903) 1093-9547/03 $17.00 \u00a9 2003 IEEE In our graph layout algorithm, based on the \u201cspring embedding\u201d graph layout algorithm (Kamada&Kawai, 1989), both species and co-species are represented by vertices, but with different properties. We reasoned that, similar to the traditional pathway graph layouts, the skeleton of the pathway graph has to be determined by products and substrates, i.e. by species vertices. So, when adjusting the positions of vertices we took into account only those \u201csprings\u201d that connected species and reaction vertices with other species and reaction vertices. Once the \u201cskeleton\u201d graph was laid out, we \u201cfroze\u201d the coordinates of reactions and species vertices, and positioned the co-species vertices using all springs at this time. Such visualization represented a compromise between an \u201cobjective\u201d graph topology, determined by all vertices, and \u201csubjective\u201d skeleton graph topology, which is not affected by the presence of compounds chosen as \u201cco-species\u201d. We would like to illustrate the above by comparing the TCA (Krebs) cycle pathway drawn with two different sets of rules. For the metabolic network in Fig. 2a metabolic species NAD, NADH, NADPH, NADP, WATER, ATP, PROTON, ADP, PI, CO-A are cospecies. In Fig. 2b the same network is shown with all the substances, including the above listed, being species, not co-species. Comparing Figs. 2a and 2b we see graphs of totally different topology. The Fig. 2a is quite close to a traditional TCA cycle drawing, while the elaborate \u201cweb\u201d in Fig.2b. has almost nothing to do with it.", "keywords": [], "document_vector": [158.691299, 41.733974], "paragraphs": [{"paragraph_vector": [-122.018074, 25.896022], "paragraph_keywords": []}], "content": {}, "doi": "10.1016/0097-8493(82)90024-3"}, {"uri": "7", "title": "empty", "timestamp": "1998", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "", "keywords": [], "document_vector": [17.82526, 14.276145], "paragraphs": [{"paragraph_vector": [-123.442977, 26.41703], "paragraph_keywords": []}], "content": {}, "doi": "10.1016/j.cag.2019.05.006"}, {"uri": "8", "title": "Parallel Molecular Dynamics Visualization using MPI with MPE Graphics - MPI Developer's Conference, 1996. Proceedings., Second ", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Rahul P. Kale", "Mark E. Fleharty", "Paul M. Alsing"], "summary": "This paperpresents an MPI implementation of a molecular dynamics (MD) simulation using force decomposition as a parallelization strategy. In contrast to atom decomposition and spatia! decomposition techniques, this method affords east. of load balancing and performs well for an intermediate number of atoms, even for irregular geometries. The interactions between the atoms are calculated in a separate module. Periodic boundary conditions are used to simulate an infinitely replicated confined region in space. The main thrust of our research efforts is currently directed towards in situ visualization of the MD simulations. This is accomplished using simple X Windows calls available through the MPE extension to the MPI routines. Our implementation using MPI with MPE graphics makes the algorithm portable. The code has been tested on a range of platforms including clusters of workstations as well as the the IBM SP2 at the Maui High PerJ7ormance Supercomputing Center (MHPCC) and the IBM SPI at the Albuquerque Resource Center (ARC), University of New Mexico. We have also adapted our code to interact with a high end graphics computer (SGI Onyx) using the OpenGL graphics library which allows for real-time manipulation of three-dimensional objects. The communication between the M D simulation and the graphics renderer was achieved with the use of sockets. The use of sockets allows the parallel MD simulation to run independently of the application used for graphics rendering.", "keywords": [], "document_vector": [-13.033802, -30.95527], "paragraphs": [{"paragraph_vector": [-123.473289, 27.260408], "paragraph_keywords": []}], "content": {}, "doi": "10.1016/j.cag.2011.03.039"}, {"uri": "9", "title": "StereoChem: Augmented Reality 3D Molecular Model Visualization App for Teaching and Learning Stereochemistry", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Narasimha Swamy", "Pankaj S. Chavan", "Sahana Murthy"], "summary": "Understanding stereochemistry is of critical importance for students aiming to be future chemists, molecular biologists, pharmacists, pharmacologists or physicians. Stereochemistry focuses on the study of the stereoisomers which have the same molecular formula but differ only in the spatial orientation of their atoms resulting in different physical, chemical or biological properties. Hence spatial thinking skill is essential for students to interpret stereoisomers. Augmented reality helps enhance the spatial perception of 3D molecules. StereoChem, our mobile app for visualizing augmented 3D molecular models harnesses this advantage of augmented reality to aid teaching and learning of stereochemistry. This paper gives a brief introduction to the significance and challenges of learning stereochemistry. It describes the design inputs that were considered for encouraging the ubiquitous use of our app by chemistry teachers and students, then discusses the development of the StereoChem app and the teaching-learning contexts in which it can be used. Finally, it gives an account of the user experience study done with 50 chemistry teachers. KeywordsAugmented Reality; Stereochemistry; Spatial Thinking; Jmol; Unity 3D; Vuforia", "keywords": ["teacher", "stereochemistry", "app", "structure", "molecule", "representation", "reality", "task", "chemical", "jmol", "based", "student", "learning", "stereochem", "use", "model", "image", "existing", "visualization", "stereoisomers"], "document_vector": [127.559219, -29.022947], "paragraphs": [{"paragraph_vector": [144.721923, -26.571727], "paragraph_keywords": ["representation", "students", "structures", "language"]}, {"paragraph_vector": [170.292068, -6.095546], "paragraph_keywords": ["jmol", "molecules", "helps", "students"]}, {"paragraph_vector": [168.959899, -2.954482], "paragraph_keywords": ["tasks", "ar", "stereochem", "students"]}, {"paragraph_vector": [171.77452, -3.81661], "paragraph_keywords": ["app", "image", "reality", "students"]}, {"paragraph_vector": [168.18399, -9.192684], "paragraph_keywords": ["students", "molecule", "stereochem", "learning"]}, {"paragraph_vector": [166.456008, -8.964685], "paragraph_keywords": ["teachers", "stereochem", "app", "teaching"]}, {"paragraph_vector": [172.219177, -2.276913], "paragraph_keywords": ["app", "stereochem", "teachers", "use"]}, {"paragraph_vector": [177.693237, -3.876422], "paragraph_keywords": ["learning", "engagement", "navneet", "anurag"]}], "content": {}, "doi": "10.1016/j.cag.2015.06.008"}, {"uri": "10", "title": "Visualization of Complex Molecular Ribbon Structures at Interactive Rates", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Andreas Halm", "Lars Offen", "Dieter Fellner"], "summary": "The increasing size of protein data available in internet databases demands new fast methods of visualization. This paper introduces a novel visualization technique for molecular ribbon structures. The method is based on Combined BReps \u2013 a mesh structure defining a combination of polygonal faces and Catmull/Clark surfaces. Our technique provides high frame rates when interactively moving through structures of highest complexity as well as high quality images when the viewpoint remains static.", "keywords": ["curve", "chain", "data", "structure", "molecule", "detail", "subdivision", "ribbon", "face", "mesh", "method", "sequence", "point", "generate", "figure", "quality", "resolution", "atom", "spline", "edge", "c\u03b1", "vertex", "resulting", "visualization"], "document_vector": [-168.396453, 1.088091], "paragraphs": [{"paragraph_vector": [70.029258, -2.602755], "paragraph_keywords": ["meshes", "subdivision", "surface", "ribbon"]}, {"paragraph_vector": [74.299606, 2.028649], "paragraph_keywords": ["atoms", "c\u03b1", "ribbon", "r"]}, {"paragraph_vector": [60.557315, -7.748774], "paragraph_keywords": ["structure", "protein", "vertices", "ribbon"]}, {"paragraph_vector": [36.084114, 41.632179], "paragraph_keywords": ["face", "frame", "resolution", "tesselation"]}, {"paragraph_vector": [31.391979, 25.70259], "paragraph_keywords": ["points", "figure", "ribbon", "parameter"]}, {"paragraph_vector": [4.965876, 11.56298], "paragraph_keywords": ["xi", "points", "structure", "curve"]}, {"paragraph_vector": [-83.006607, -67.121078], "paragraph_keywords": ["c\u03b1", "chain", "atoms", "ribbons"]}, {"paragraph_vector": [70.00917, 7.140573], "paragraph_keywords": ["spline", "points", "curve", "transition"]}, {"paragraph_vector": [52.466453, 18.610286], "paragraph_keywords": ["figure", "qi", "generated", "faces"]}, {"paragraph_vector": [63.941814, 31.41131], "paragraph_keywords": ["detail", "method", "allows", "mesh"]}, {"paragraph_vector": [72.304077, 51.441429], "paragraph_keywords": ["vertex", "project", "use", "ribbon"]}], "content": {}, "doi": "10.1016/j.cag.2021.05.010"}, {"uri": "11", "title": "A Fast Ultrasound Molecular Imaging Method and its 3D Visualization in vivo", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Xiaowen Hu", "Lisa Mahakian", "Charles Caskey", "Julie Beegle", "Dustin Kruse", "Joshua Rychak", "Patrick Sutcliffe", "Katherine Ferrara"], "summary": "Using targeted microbubbles (MBs), ultrasound molecular imaging can be used to selectively and specifically visualize upregulated vascular receptors. In order to acquire bound MB echoes, a delay of ~7-15 minutes is commonly required for the clearance of freely circulating MBs. Here, we test whether echoes from MBs can be distinguished from the surrounding tissue, based on the transmission of pulses at low (1.5 MHz) and reception at high (5.5 MHz) frequencies (TLRH), without the requirement for destructive pulses. Pulses with a peak negative pressure of 230 kPa were transmitted (10 fps) and a 7 order IIR pulse-to-pulse filter was applied to the TLRH radiofrequency (RF) data to distinguish the signature of bound MBs from that of flowing MBs. 3D images of the accumulation of intravenously-administrated integrin-targeted MBs in a Met-1 mouse tumor model were acquired. An in vitro study demonstrated that the T2R15 contrast imaging technique has a ~2-fold resolution improvement over 2MHz contrast pulse sequencing (CPS) imaging. By applying the 7 order IIR filter to the TLRH RF data acquired at 2 minutes, echoes from flowing MBs in the surrounding tissue region were suppressed by 26\u00b12 dB, while the signal intensity within the tumor was suppressed by 4\u00b11 dB. The targeted images correctly represented the distribution of bound MBs. After the filter, the signal intensity resulting from cyclic RGD bearing MBs was 25\u00b12 dB higher than that after the injection of non-targeted MBs. Key word: targeted microbubble; molecular imaging; slow time", "keywords": [], "document_vector": [111.684829, 7.269697], "paragraphs": [{"paragraph_vector": [-122.137084, 27.830265], "paragraph_keywords": []}], "content": {}, "doi": "10.1016/j.cag.2015.07.017"}, {"uri": "12", "title": "empty", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "Molecular computing has proved its possibility to solve weighted graph problem such as Hamiltonian Path Problem (HPP), Traveling Salesman Problem (TSP) and the Shortest Path Problem (SPP). Normally, in molecular computation, the DNA sequences used for the computation should be critically designed in order to reduce error that could occur during computation. In the previous paper, we have proposed a readout method tailored specifically to HPP in DNA Computing using real-time PCR for output visualization. Six nodes of HPP was considered. Based on the example instance, the method requires 11 oligonucleotides, where 6 oligonucleotides are for the nodes and 5 oligonucleotides are for the edges. Three TaqMan probes and five primers are required as well. In this study, a procedure for DNA sequence design is presented in order to obtain good sequences for those nodes, primers, and probes. The experiment is done based on the generated DNA sequences and the Hamiltonian Path can be determined successfully.", "keywords": ["primer", "based", "time", "sequence", "number", "dna", "path", "probe", "g", "taqman"], "document_vector": [-93.615081, 25.829168], "paragraphs": [{"paragraph_vector": [-96.847915, -35.046501], "paragraph_keywords": ["reactions", "time", "primer", "nodes"]}, {"paragraph_vector": [-93.994674, -32.555488], "paragraph_keywords": ["path", "instance", "primers", "time"]}, {"paragraph_vector": [-92.910881, -32.626216], "paragraph_keywords": ["sequences", "primers", "probes", "set"]}, {"paragraph_vector": [-93.284759, -31.053714], "paragraph_keywords": ["sequences", "c", "g", "modified"]}, {"paragraph_vector": [-93.908439, -33.868473], "paragraph_keywords": ["dna", "probes", "primers", "taqman"]}, {"paragraph_vector": [-92.819358, -32.22253], "paragraph_keywords": ["sequences", "sequence", "based", "dnasequencegenerator"]}], "content": {}, "doi": "10.1016/0097-8493(75)90018-7"}, {"uri": "13", "title": "Visualizing Spin-Dependent Molecule Symmetry at an Organic-Ferromagnetic Spinterface", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yu-Hsun Chu(a", "Chuang-Han Hsu(a", "Chun-I Lu(a", "Hung-Hsiang Yang(a", "Tsung-Han Yang(a", "Chi-Hung Luo(a", "Kai-Jheng Yang(a", "Shih-Hao Hsu(a", "Germar Hoffmann(a", "Chao-Cheng Kaun(b", "H.-H. Yang", "T.-H. Yang", "C.-H. Luo", "K.-J. Yang", "S.-H. Hsu", "G. Hoffmann", "C.-C. Kaun"], "summary": "As promising building blocks in spintronics, organic molecules have shown a variety of advantages including long spin coherence time and length as well as superparamagnetism. In addition, contacting with ferromagnetic (FM) materials, molecules induce so-called \u201cspinterfaces\u201d at the organic-FM boundaries, at which new spin-polarized states appear and change the magnetic properties in these hybrid systems. While the spinterface may influence performance in spinctronic devices, it may also extend the possibility of organic spintronics. To conduct an investigation into spinterfaces, we visualized the spin-distribution within single pentacene (PEN, C22H14) on FM Co nanoislands on Cu(111) via spin-polarized scanning tunnelling microscopy (SP-STM) [1]. Near the Fermi level, the widely-used semiconducting PEN molecules show not only spin-polarization opposite to the Co substrate, but also spin-dependent symmetries. Combined with first-principles calculations, the experimental findings agree with the pz-d hybridization model, in which spin-dependent bonding types can result in different molecule symmetries. Our observations exhibit locally-reversed spin-polarization after molecule deposition, and reveal a concise way to predict molecular spin-distribution for -conjugated molecules such as PEN in organic-FM hybrid structures.", "keywords": [], "document_vector": [64.187232, -12.829873], "paragraphs": [{"paragraph_vector": [-127.593589, 21.155019], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/VISUAL.2002.1183780"}, {"uri": "14", "title": "Detection and Visualization of Anomalous Structures in Molecular Dynamics Simulation Data", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sameep Mehta", "Raghu Machiraju", "John Wilkins"], "summary": "In this article we explore techniques to detect and visualize features in data from molecular dynamics (MD) simulations. Although the techniques proposed are general, we focus on silicon (Si) atomic systems. The first set of methods use 3D location of atoms. Defects are detected and categorized using local operators and statistical modeling. Our second set of exploratory techniques employ electron density data. This data is visualized to glean the defects. We describe techniques to automatically detect the salient iso-values for iso-surface extraction and designing transfer functions.We compare and contrast the results obtained from both sources of data. Essentially, we find that the methods of defect (feature) detection are at least as robust as those based on the exploration of electron density for Si systems. CR Categories: J.2 [Computer Applications]: Physical Sciences And Engineering \u2014Physics", "keywords": ["density", "transfer", "use", "list", "data", "value", "surface", "iso", "histogram", "bin", "defect", "lattice", "electron", "function", "atom", "visualization", "show", "figure"], "document_vector": [53.594985, -32.433654], "paragraphs": [{"paragraph_vector": [101.087417, 14.702104], "paragraph_keywords": ["defect", "transfer", "list", "atom"]}, {"paragraph_vector": [98.474403, 15.66999], "paragraph_keywords": ["density", "iso", "atoms", "use"]}, {"paragraph_vector": [97.702201, 18.521707], "paragraph_keywords": ["histogram", "value", "iso", "bins"]}, {"paragraph_vector": [98.972229, 17.592758], "paragraph_keywords": ["value", "iso", "transfer", "defect"]}, {"paragraph_vector": [99.169227, 16.023042], "paragraph_keywords": ["defects", "data", "atoms", "defect"]}, {"paragraph_vector": [48.602771, 2.373376], "paragraph_keywords": ["pages", "thank", "materials", "like"]}, {"paragraph_vector": [48.979621, 2.148157], "paragraph_keywords": ["data", "richie", "mining", "vol"]}], "content": {}, "doi": "10.1109/PacificVis.2019.00034"}, {"uri": "15", "title": "Development of a Transparency High Molecular Gel Phantom for Three-Dimensional Visualization of Radio Frequency Electromagnetic Power Absorption with Capsulate Thermo-chromic Liquid Crystal", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Madoka BABAt", "Yukihisa Sumkit", "Aki Ishiit", "Masao Takit", "Kaori Fukunagat", "Soichi Watanabet"], "summary": "Safety of radio frequency (RF) electromagnetic fields (EMF) is coimnonly discussed in terms of specific absorption rate (SAR). Phantoms are useful for the dosimetry on energy absorption of RF-EMF exposures. SAR is estimated from measurements of an increase in temperature within the pluntoin over a short period of time following the exposure. In the past study. a method for visualizing the three-dimensional (3D) distribution of temperature due to RF-EMF energy absorption was reported[l]. A difficulty of this method is that it can only visualize the temperature higher than the clouding point. We focused our interest on capsulate liquid crystal that has been used for visualization of temperature dishbution within fluids [Z]. The objective of this study is to make a phantom containing capsulate liquid crystal so as to develop a method to visualize 3-D distribution of electroinagnetic power ab-, sorption. We have investigated the suitable materials for transparency phantom and discussed for the method to adjust dielectric properties of the phantom. 1. Development of Phantom Micro-capsulated thenno-chromic liquid ctystal(MTLC) is used for,our experiments. \"lienno-chromic liquid crystal is contained in the urea resin capsule. The &\"mer of the capsnle is ahout 20 to 30 micrometer. The incident light illto MTLC is needed for visualization. The wavelength of scattered light depends on the temperahue of the capsulate liquid ctystal. The wavelength becomes shorter with the increase of teniperature. In other words, the scattered light changes in its color from red to purple with the increase of temperature. MTLC are suspended unifonnly in the gel phantom. The substrate material for the phantom should not be convective to measure SAR. Therefore. high-molecular gel constructed from \"carrageenan\". which is extracted from seaweed and has high transparency. is used as the substrate of the phantom to prevent convection. The dielectric propelties ofthe phantom are adjusted by mixing propylene glycol. or KCI (potassium chloride). 2. Complex Permittivity and RF Exposure Complex permittivity plots of the dielectric properties of phantoms that are made from carrageenan are shown in Fig. 1. Values of complex permittivity are measured by the dielectric probe (Agilent 85070C). In this graph. horizontal axis and vertical axis indicate relative permittivity (E'(w)) and loss ( ~ \" ( w ) ) , respectively. As shown in Fig. 1, each marker indicates the complex permittivity at SOO[MHz], 900[MHzl, ISIGHzJ, 1.95[GHz], and 2.45[GHz], respectively. Solid line which shows muscle properly is obtained by the parametric model of biological tissues [3]. As shown in Fig. 1, dielectric properties of the transparency p l m o m based on carrageenan can be controlled by v'arying the concentration of sucrose. propylene glycol and KC1 mixed in the phantom. We can approximately adjust a value of complex permittivity of transparency phantom to that of inuscle at 900[MHz] and 1 .S[GHz] . We perform a preliminary exposure experiment using above-mentioned phantom. A dipole antenna is used to irradiate a phantom containing MTLC with RF-EMF at the frequency of 1.439 [GHz]. The temperature distribution. which is indicated by scattering light from red to purple. on the plane lit up by the slit light is obtained clearly. The temperature distributions on other planes are also visualized by moving the slit light location. These obsenaatioms 0-7803-8404-0/04/$20.00", "keywords": [], "document_vector": [91.025871, 8.811129], "paragraphs": [{"paragraph_vector": [-123.677429, 25.91654], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/PacificVis.2018.00020"}, {"uri": "16", "title": "Ambient Occlusion and Edge Cueing to Enhance Real Time Molecular Visualization", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Marco Tarini", "Paolo Cignoni", "Claudio Montani"], "summary": "The paper presents a set of combined techniques to enhance the real-time visualization of simple or complex molecules (up to order of 10 atoms) space fill mode. The proposed approach includes an innovative technique for efficient computation and storage of ambient occlusion terms, a small set of GPU accelerated procedural impostors for space-fill and ball-and-stick rendering, and novel edge-cueing techniques. As a result, the user\u2019s understanding of the three-dimensional structure under inspection is strongly increased (even for still images), while the rendering still occurs in real time.", "keywords": ["fragment", "buffer", "p", "value", "molecule", "k", "order", "impostor", "way", "set", "space", "lighting", "shading", "point", "approach", "number", "primitive", "screen", "sec", "occlusion", "texels", "depth", "texture", "fig", "computation", "technique", "surface", "direction", "mapping", "position", "scene", "atom", "-", "term", "cylinder", "use", "vertex", "patch", "image", "rendering", "visualization", "according"], "document_vector": [-158.578201, -78.175369], "paragraphs": [{"paragraph_vector": [88.529457, -12.569615], "paragraph_keywords": ["structure", "visualization", "proteins", "problem"]}, {"paragraph_vector": [68.49887, 38.543998], "paragraph_keywords": ["techniques", "rendering", "e", "-"]}, {"paragraph_vector": [63.131145, 34.445091], "paragraph_keywords": ["shading", "rendering", "visualization", "lighting"]}, {"paragraph_vector": [63.362941, 58.004699], "paragraph_keywords": ["occlusion", "term", "approaches", "scene"]}, {"paragraph_vector": [51.109607, 71.355857], "paragraph_keywords": ["occlusion", "\u03c9", "computation", "np"]}, {"paragraph_vector": [41.168041, 62.925022], "paragraph_keywords": ["surface", "rendering", "scene", "effects"]}, {"paragraph_vector": [54.99374, 61.433883], "paragraph_keywords": ["impostor", "point", "order", "attributes"]}, {"paragraph_vector": [51.985679, 64.777236], "paragraph_keywords": ["texture", "rendering", "store", "signal"]}, {"paragraph_vector": [57.076057, 59.97573], "paragraph_keywords": ["texture", "mapping", "texels", "atoms"]}, {"paragraph_vector": [61.453197, 62.964748], "paragraph_keywords": ["use", "impostor", "t", "assigned"]}, {"paragraph_vector": [37.549465, 60.458518], "paragraph_keywords": ["inverse", "u", "defined", "y"]}, {"paragraph_vector": [34.346691, 60.268569], "paragraph_keywords": ["cylinder", "values", "vertex", "axis"]}, {"paragraph_vector": [54.94355, 59.474178], "paragraph_keywords": ["patches", "texture", "number", "texel"]}, {"paragraph_vector": [54.239322, 60.812934], "paragraph_keywords": ["texel", "patch", "texture", "p"]}, {"paragraph_vector": [45.327766, 59.946937], "paragraph_keywords": ["case", "depth", "image", "shadow"]}, {"paragraph_vector": [40.519382, 58.748672], "paragraph_keywords": ["atoms", "depth", "lines", "rendering"]}, {"paragraph_vector": [49.087234, 55.400554], "paragraph_keywords": ["depth", "buffer", "b", "effect"]}, {"paragraph_vector": [38.610347, 60.879081], "paragraph_keywords": ["atom", "depth", "halos", "atoms"]}, {"paragraph_vector": [54.065326, 59.565101], "paragraph_keywords": ["occlusion", "atoms", "rendering", "set"]}, {"paragraph_vector": [67.904769, 34.96025], "paragraph_keywords": ["visualization", "texture", "flexibility", "efficiency"]}], "content": {}, "doi": "10.1109/TVCG.2016.2598824"}, {"uri": "17", "title": "empty", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "In this paper, our initial work on the use of visualization and virtual reality for education especially for providing three-dimensional concepts of molecular structure was presented. Our system is composed of a hybrid screen, multi-projectors, PC cluster, 6DOF haptic device etc. It offers a channel to reach into the molecular space in an immersive environment, allows user to employ an intuitive 6DOF haptic device Spidar-G to control the perspective. For educators, it can serve as a good instructional aid for helping students to understand molecular structure through effective visual representation and interactive manipulation.", "keywords": ["projector", "grip", "information", "spidar", "distributed", "screen", "system", "based", ".", "-", "graphic", "display", "model", "pc", "view", "projection", "device", "image", "g", "rendering", "environment"], "document_vector": [117.748222, -24.353935], "paragraphs": [{"paragraph_vector": [163.69989, -1.663138], "paragraph_keywords": ["screen", "system", "environment", "interaction"]}, {"paragraph_vector": [164.562484, -5.74293], "paragraph_keywords": ["section", "spidar", "view", "g"]}, {"paragraph_vector": [144.118026, 28.706115], "paragraph_keywords": ["screen", "projection", "projectors", "view"]}, {"paragraph_vector": [145.666931, 26.1627], "paragraph_keywords": ["system", "image", "rendering", "view"]}, {"paragraph_vector": [168.014892, -8.105522], "paragraph_keywords": ["grip", "display", "g", "pc"]}, {"paragraph_vector": [142.560272, 29.902723], "paragraph_keywords": ["rendering", "graphics", "pc", "pcs"]}, {"paragraph_vector": [164.188613, -9.61367], "paragraph_keywords": ["design", "drug", "screen", "system"]}, {"paragraph_vector": [146.168029, 31.934556], "paragraph_keywords": ["system", ".", "graphics", "ieice"]}], "content": {}, "doi": "10.1109/ICAICT.2014.7036006"}, {"uri": "18", "title": "Molecular Dynamics Simulations of Plasma-Surface Interactions: Importance of Visualization Tools", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["David Humbird", "David B. Graves"], "summary": "Molecular visualization enhances computational plasma-surface studies, offering better ways to analyze and present the atomic-scale details of the simulation. An example is given in which a standard rendering of a simulation cell suggests it is well mixed, while a more sophisticated rendering of the same cell shows otherwise.", "keywords": [], "document_vector": [45.339607, -64.977096], "paragraphs": [{"paragraph_vector": [-122.961967, 27.098833], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/3M-NANO46308.2019.8947375"}, {"uri": "19", "title": "IMAGING ELLIPSOMETRY FOR THE VISUALIZATION OF BIO-MOLECULAR LAYERS", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Gang Jin", "Zhan-Hui Wang", "Yong-Hong Meng"], "summary": "Imaging ellipsometry is presented as a visualization technique to study bio-molecular layers. The layers have become more and more attractive in materials science, especially the layers with the thickness similar to the cellular layers and with the physiological activities are very important in molecular biology and medicine. Normally bio-molecular layers are very thin, which thickness is between sub-nanometer and several tens nanometer. They are transparent in visible range of light, so that they are recognized as a phase object in physics. Imaging ellipsometry is non-destructive and exhibits a high sensitivity to phase transitions within thin layers. It is capable of imaging local variations in the optical properties such as \u2018thickness due to the presence of different surface concentration of bio-molecule or different deposited molecules. Imaging ellipsometry is based on conventional ellipsometry with charge coupled device (CCD) technique. The images are captured with a computer with image processing technique. It has high sensitivity to thickness variation (resolution in the order of angstrom), and high sampling speed (25 pictures with more than lo5 pixels per second). Although here it had just shown some examples of visualization of bio-molecular layers, it would be possible to show a formation process of bio-molecular layers with a real-time technique. Index Terms Imaging ellipsometry, bio-molecular layers, visualization, Immunoglobulin G, human serum albumin", "keywords": [], "document_vector": [99.053016, 0.441148], "paragraphs": [{"paragraph_vector": [-124.19561, 27.440097], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/IV.2011.103"}, {"uri": "20", "title": "Visualization and Web Services for Studying Molecular Properties", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Alessandro Costantini"], "summary": "The access to large and distributed computing resources requires by the scientist a strong knowledge of a variety of interfaces and operating system environments. In the present paper we illustrate a prototypical implementation of a user interface based on visualization and Web Services implemented for the Computational Chemistry community to use molecular and matter sciences applications in a friendly and intuitive fashion. The application presented allows the final user to study, in a more intuitive and enriched way, the outcomes of the molecular simulation, fostering a better comprehension of properties and mechanisms of the investigated molecular", "keywords": ["web", "output", "data", "program", "order", "server", "quantum", "calculation", "number", "system", "figure", "state", "reaction", "service", "carried", "j", "tool", "client", "energy", "file", "grid", "related", "given", "user", "making", "visualization"], "document_vector": [-22.971342, -15.844331], "paragraphs": [{"paragraph_vector": [-168.408538, 57.039909], "paragraph_keywords": ["systems", "quantum", "study", "visualization"]}, {"paragraph_vector": [75.169342, -38.225666], "paragraph_keywords": ["quantum", "\u03c8", "diatom", "integrate"]}, {"paragraph_vector": [89.930633, -57.059093], "paragraph_keywords": ["energy", "given", "sector", "table"]}, {"paragraph_vector": [85.863868, -60.33715], "paragraph_keywords": ["figure", "reaction", "output", "probabilities"]}, {"paragraph_vector": [164.754425, 32.505233], "paragraph_keywords": ["data", "input", "calculation", "parameters"]}, {"paragraph_vector": [88.680641, -54.568172], "paragraph_keywords": ["server", "output", "file", "client"]}, {"paragraph_vector": [64.660018, -60.722087], "paragraph_keywords": ["state", "making", "figure", "output"]}, {"paragraph_vector": [169.119308, 59.924507], "paragraph_keywords": ["output", "researcher", "egee", "calculations"]}], "content": {}, "doi": "10.1109/TMAG.2006.880462"}, {"uri": "21", "title": "Molecular Dynamics Simulation and Visualization", "timestamp": "1999", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Roman \u010eurikovi\u010d", "Teruaki Motooka"], "summary": "We have developed atomic-scale material models capable of melting, crystallization and amorphization. These models feature molecular dynamics governed by Langevin equations of motion in which particle interact through attractive covalent forces and short-range repulsion forces. Also, we present an interactive virtual visualization tool for the simulation of atomic scale material behavior. An application of this research is to understand the processes that can control the quality of a single-crystal Si grown from the melt.", "keywords": ["structure", "simulation", "dynamic", "silicon", "particle", "scale", "system", "distribution", "defined", "tersoff", "atom", "temperature", "model", "time", "function", "force", "visualization", "crystal", "modeling"], "document_vector": [49.043987, -41.209358], "paragraphs": [{"paragraph_vector": [49.701801, -17.00616], "paragraph_keywords": ["crystal", "structure", "modeling", "atoms"]}, {"paragraph_vector": [49.926658, -18.896411], "paragraph_keywords": ["forces", "models", "behavior", "state"]}, {"paragraph_vector": [52.425842, -20.079742], "paragraph_keywords": ["particles", "force", "ijk", "particle"]}, {"paragraph_vector": [51.881771, -15.329806], "paragraph_keywords": ["silicon", "t", "time", "force"]}, {"paragraph_vector": [89.773468, -45.539489], "paragraph_keywords": ["time", "atoms", "distribution", "probability"]}, {"paragraph_vector": [50.00938, -19.028568], "paragraph_keywords": ["temperature", "atoms", "silicon", "simulation"]}, {"paragraph_vector": [47.486511, -16.312814], "paragraph_keywords": ["dynamics", "silicon", "scale", "modeling"]}], "content": {}, "doi": "10.1109/ISVRI.2011.5759651"}, {"uri": "22", "title": "Visualization of Large Molecular Trajectories", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["David Duran", "Pedro Hermosilla", "Timo Ropinski", "Barbora Kozl\u0131\u0301kov\u00e1", "\u00c0lvar Vinacua", "Pere-Pau V\u00e1zquez"], "summary": "The analysis of protein-ligand interactions is a time-intensive task. Researchers have to analyze multiple physico-chemical properties of the protein at once and combine them to derive conclusions about the protein-ligand interplay. Typically, several charts are inspected, and 3D animations can be played side-by-side to obtain a deeper understanding of the data. With the advances in simulation techniques, larger and larger datasets are available, with up to hundreds of thousands of steps. Unfortunately, such large trajectories are very difficult to investigate with traditional approaches. Therefore, the need for special tools that facilitate inspection of these large trajectories becomes substantial. In this paper, we present a novel system for visual exploration of very large trajectories in an interactive and user-friendly way. Several visualization motifs are automatically derived from the data to give the user the information about interactions between protein and ligand. Our system offers specialized widgets to ease and accelerate data inspection and navigation to interesting parts of the simulation. The system is suitable also for simulations where multiple ligands are involved. We have tested the usefulness of our tool on a set of datasets obtained from protein engineers, and we describe the expert feedback.", "keywords": ["snapshot", "cavity", "data", "group", "value", "exploration", "md", "interest", "distance", "chart", "interaction", "simulation", "set", "space", "cluster", "protein", "trajectory", "information", "ligand", "number", "approach", "speed", "system", "selected", "researcher", "range", "step", "clustering", "technique", "position", "tool", "atom", "configuration", "energy", "inspection", "expert", "use", "view", "feature", "region", "time", "user", "visualization", "provide"], "document_vector": [-97.384597, 4.806804], "paragraphs": [{"paragraph_vector": [105.962623, -54.356811], "paragraph_keywords": ["simulations", "group", "drug", "analysis"]}, {"paragraph_vector": [4.924899, -81.447532], "paragraph_keywords": ["simulation", "system", "exploration", "interaction"]}, {"paragraph_vector": [58.232578, -76.781082], "paragraph_keywords": ["visualization", "trajectories", "space", "exploration"]}, {"paragraph_vector": [39.196849, -40.287258], "paragraph_keywords": ["visualization", "exploration", "time", "techniques"]}, {"paragraph_vector": [-62.800815, -85.192375], "paragraph_keywords": ["data", "information", "events", "approaches"]}, {"paragraph_vector": [47.383148, -70.835762], "paragraph_keywords": ["system", "ligand", "energy", "information"]}, {"paragraph_vector": [46.544231, -53.202426], "paragraph_keywords": ["protein", "ligand", "atoms", "positions"]}, {"paragraph_vector": [45.973854, -66.85469], "paragraph_keywords": ["ligand", "atoms", "\u03bb", "protein"]}, {"paragraph_vector": [65.723793, -57.218078], "paragraph_keywords": ["ligand", "values", "protein", "interaction"]}, {"paragraph_vector": [58.421684, -60.904098], "paragraph_keywords": ["simulation", "interaction", "clusters", "zoom"]}, {"paragraph_vector": [-8.08294, -87.25962], "paragraph_keywords": ["data", "energy", "exploration", "steps"]}, {"paragraph_vector": [39.060169, -64.697082], "paragraph_keywords": ["information", "energy", "clusters", "interaction"]}, {"paragraph_vector": [143.39537, 3.315465], "paragraph_keywords": ["charts", "chart", "button", "user"]}, {"paragraph_vector": [52.645233, -52.815975], "paragraph_keywords": ["chart", "simulation", "interest", "charts"]}, {"paragraph_vector": [75.267204, 32.323879], "paragraph_keywords": ["atoms", "ligand", "list", "interacting"]}, {"paragraph_vector": [48.080234, -62.895401], "paragraph_keywords": ["plane", "clipping", "cavities", "user"]}, {"paragraph_vector": [48.15998, -52.951919], "paragraph_keywords": ["atoms", "sphere", "interaction", "selected"]}, {"paragraph_vector": [56.634185, -72.047416], "paragraph_keywords": ["user", "charts", "chart", "trajectory"]}, {"paragraph_vector": [56.125511, -75.713729], "paragraph_keywords": ["user", "charts", "data", "exploration"]}, {"paragraph_vector": [53.68085, -80.348472], "paragraph_keywords": ["user", "chart", "ligands", "cavity"]}, {"paragraph_vector": [84.795654, -56.171604], "paragraph_keywords": ["data", "charts", "system", "rendered"]}, {"paragraph_vector": [157.74559, -20.754293], "paragraph_keywords": ["features", "tool", "questionnaire", "asked"]}, {"paragraph_vector": [40.237194, -65.555191], "paragraph_keywords": ["simulations", "information", "system", "interaction"]}, {"paragraph_vector": [57.798484, -70.450027], "paragraph_keywords": ["charts", "data", "simulation", "ligands"]}, {"paragraph_vector": [73.868568, -61.397346], "paragraph_keywords": ["exploration", "analysis", "protein", "trajectories"]}, {"paragraph_vector": [49.639083, -67.808235], "paragraph_keywords": ["exploration", "interactions", "trajectories", "data"]}, {"paragraph_vector": [71.390472, -74.812705], "paragraph_keywords": ["exploration", "grant", "inspection", "labeling"]}], "content": {}, "doi": "10.1109/XSW.2013.10"}, {"uri": "23", "title": "Protein Functional Motion Query and Visualization", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Xiong Liu", "Hassan A. Karimi", "Lee-Wei Yang", "Ivet Bahar"], "summary": "Function is a dynamic property closely related to the conformational mechanisms of the protein structure in its physiological environment. Efficient methods and tools for predicting collective motions at the molecular level are becoming increasingly important for understanding and controlling the function of target proteins. The molecular motions of proteins in different mode types are determined in the present study using the Gaussian Network Model (GNM). Based on GNM, we present a system that enables efficient query and visualization of protein dynamic behaviors such as mode shapes and 3D mobility structures.", "keywords": ["dynamic", "protein", "pdb", "structure", "motion", "fluctuation", "query", "matrix", "based", "gnm", "function", "mode", "visualization", "pmdb", "site"], "document_vector": [-101.883682, 22.153963], "paragraphs": [{"paragraph_vector": [-6.994002, -70.678474], "paragraph_keywords": ["gnm", "protein", "function", "dynamics"]}, {"paragraph_vector": [-71.607742, -88.854904], "paragraph_keywords": ["gnm", "matrix", "network", "ith"]}, {"paragraph_vector": [-151.356872, -80.630455], "paragraph_keywords": ["pdb", "query", "table", "pmdb"]}, {"paragraph_vector": [-83.800415, -73.090133], "paragraph_keywords": ["mode", "modes", "sites", "shapes"]}, {"paragraph_vector": [-112.264831, -84.684524], "paragraph_keywords": ["mode", "pdb", "information", "mobility"]}], "content": {}, "doi": "10.1109/CGI.2000.852350"}, {"uri": "24", "title": "In Situ Visualization of Dynamic Interactions of Cellulase and Cellulose Molecules", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Bo Song", "Ning Xi", "Ruiguo Yang", "Zhiyong Sun", "Liangliang Chen"], "summary": "Enzymatic hydrolysis of cellulose from biomass is potentially considered as a new approach for the next generation of biofuels for sustainable energy. However, because of the rigid and complex structure of cellulose, the low efficiency of the enzymatic hydrolysis has been deemed as the bottleneck in biorefining industry. The mechanism of enzymatic hydrolysis of cellulose is not well understood due to the absence of direct observation methods. In the present study, in order to study the mechanism of enzymatic hydrolysis of cellulose, we used an atomic force microscopy (AFM) to dynamically observe the changes happening during the cellulase and cellulose interaction. To improve the imaging rate of AFM (which was too slow to capture the interactions between these two nano structures), a scan strategy named compressive scan is used, associated with prior knowledge based image reconstruction methods. With the help of compressive scan, we are able to dynamically capture the action of the cellulase\u2013CBH I (from family 7 cellobiohydrolase) on the cellulose crystal nanofibrils. The result is helpful for understanding the mechanism of enzymatic interaction and increasing the efficiency of hydrolysis process.", "keywords": ["crystal", "afm", "cellulose", "nanofibrils", "matrix", "surface", "solution", "time", "trajectory", "based", "image", "frame", "imaging", "cbh", "measurement", "sampling", "point"], "document_vector": [109.641174, 30.819213], "paragraphs": [{"paragraph_vector": [-40.394657, 60.979511], "paragraph_keywords": ["measurement", "matrix", "afm", "trajectory"]}, {"paragraph_vector": [-102.419342, 36.774242], "paragraph_keywords": ["points", "trajectory", "measurement", "matrix"]}, {"paragraph_vector": [-45.117565, 59.568897], "paragraph_keywords": ["image", "frame", "time", "information"]}, {"paragraph_vector": [-39.36914, 62.100688], "paragraph_keywords": ["frame", "image", "measurement", "use"]}, {"paragraph_vector": [-33.43349, 32.674732], "paragraph_keywords": ["solution", "cellulose", "cbh", "mica"]}, {"paragraph_vector": [-35.705387, 30.321947], "paragraph_keywords": ["cellulose", "cbh", "image", "crystal"]}, {"paragraph_vector": [-34.843364, 32.207466], "paragraph_keywords": ["cellulose", "cbh", "surface", "nanofibrils"]}, {"paragraph_vector": [-37.817104, 38.408668], "paragraph_keywords": ["cellulose", "cbh", "hydrolysis", "moving"]}], "content": {}, "doi": "10.1109/TMAG.2003.808917"}, {"uri": "25", "title": "View-dependent level-of-detail abstraction for interactive atomistic visualization of biological structures", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Dongliang Guo", "Junlan Nie", "Meng Liang", "Yu Wang", "Yanfen Wang", "Zhengping Hu"], "summary": "The visualization of biological structures is a challenging task because it requires rendering millions to billions of atoms in real time. In this paper, we propose a view-dependent approach by which a large biological scene can be visualized interactively. In this scheme, we first extract several levels of building blocks of biological structures from a molecular abstraction based on hierarchical clustering. We then define a volume-based distance metric for the clustering process to reduce \u201cinflation\u201d error and propose a quantitative error metric for the object space error evaluation. Finally, we utilize an adaptive screenspace level-of-detail selection with the error metric at run time. Empirical results demonstrate that our molecular hierarchical abstraction method achieves high quality rendering results and performs better than other existing methods. Moreover, our result also shows that the view-dependent approach provides valid results in a large biological scene with more than 10 billion elements. & 2015 Elsevier Ltd. All rights reserved.", "keywords": ["data", "volume", "molecule", "et", "distance", "representation", "selection", "space", "set", "method", "cluster", "particle", "number", "section", "clustering", "object", "sphere", "surface", "based", "level", "performance", "element", "atom", "s", "gpu", "use", "abstraction", "radius", "error", "time", "al", "vdw", "rendering", "visualization", "residue"], "document_vector": [169.185821, -70.000473], "paragraphs": [{"paragraph_vector": [73.764839, 29.689868], "paragraph_keywords": ["clustering", "rendering", "data", "based"]}, {"paragraph_vector": [92.976516, 15.41295], "paragraph_keywords": ["section", "surface", "visualization", "vdw"]}, {"paragraph_vector": [41.322235, 22.244033], "paragraph_keywords": ["ses", "surface", "spheres", "ligand"]}, {"paragraph_vector": [34.167545, 37.414997], "paragraph_keywords": ["et", "particle", "al", "rendering"]}, {"paragraph_vector": [58.029754, 37.923069], "paragraph_keywords": ["rendering", "distance", "abstraction", "lod"]}, {"paragraph_vector": [83.184883, 20.168241], "paragraph_keywords": ["rendering", "gpu", "visualization", "dna"]}, {"paragraph_vector": [72.655212, 30.545431], "paragraph_keywords": ["rendering", "error", "information", "space"]}, {"paragraph_vector": [37.020137, -5.058838], "paragraph_keywords": ["level", "levels", "molecule", "residue"]}, {"paragraph_vector": [55.440711, 21.947931], "paragraph_keywords": ["level", "based", "molecule", "ses"]}, {"paragraph_vector": [15.43553, -43.700473], "paragraph_keywords": ["distance", "clustering", "volume", "dissimilarity"]}, {"paragraph_vector": [-4.75857, -43.151645], "paragraph_keywords": ["clustering", "residue", "error", "atoms"]}, {"paragraph_vector": [-15.85276, -51.293174], "paragraph_keywords": ["volume", "vdw", "compute", "type"]}, {"paragraph_vector": [59.910701, 34.264091], "paragraph_keywords": ["error", "space", "representation", "cluster"]}, {"paragraph_vector": [61.79623, 37.734432], "paragraph_keywords": ["disthrl", "radius", "level", "l"]}, {"paragraph_vector": [55.24673, 20.2698], "paragraph_keywords": ["clustering", "error", "distance", "space"]}, {"paragraph_vector": [58.606319, 28.179193], "paragraph_keywords": ["clustering", "atoms", "performance", "rendering"]}, {"paragraph_vector": [63.225055, 38.989738], "paragraph_keywords": ["culling", "atoms", "work", "selection"]}, {"paragraph_vector": [56.940074, 35.322525], "paragraph_keywords": ["error", "space", "based", "representation"]}, {"paragraph_vector": [15.793118, 8.540961], "paragraph_keywords": ["distance", "based", "utilizing", "surface"]}], "content": {}, "doi": "10.1109/MMUL.2001.959090"}, {"uri": "26", "title": "MoSuMo: A Semantic Web service to generate electrostatic potentials across solvent excluded protein surfaces and binding pockets", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Alexander Gawronski", "Michel Dumontier"], "summary": "Investigations of molecular interactions can benefit from knowledge of the regions that correspond to binding sites and electrostatic potentials across molecular surfaces. In support of simulations aimed at a coarser level of granularity than molecular dynamics, we have developed MoSuMo, a SADI-based Semantic Web service for generating molecular surfaces from a set of 3D atomic coordinates. MoSuMo\u2019s service input requires a Protein Data Bank (PDB) identifier from which it obtains the PDB entry and for which it generates electrostatic potentials as vertex values across a geometric mesh that represents the molecular surface. Service inputs and outputs are specified as restrictions on classes from the Semanticscience Integrated Ontology (SIO), which is formalized using the Web Ontology Language (OWL). Binding sites containing their ligands are identified and described as sub-collections of surface vertices. Thus, MoSuMo provides a semantic web service for investigating molecular interactions including visualization. & 2011 Elsevier Ltd. All rights reserved.", "keywords": ["web", "output", "data", "entity", "structure", "pocket", "molecule", "rdf", "set", "mesh", "method", "protein", "input", "algorithm", "owl", "information", "ligand", "service", "pdb", "surface", "based", "binding", "chimera", "tool", "atom", "probe", "knowledge", "sadi", "solvent", "model", "mosumo", "use", "potential", "vertex", "ontology", "residue"], "document_vector": [-98.742897, -11.622806], "paragraphs": [{"paragraph_vector": [2.673545, -64.200218], "paragraph_keywords": ["function", "molecules", "shape", "coordinates"]}, {"paragraph_vector": [-176.666915, -43.42514], "paragraph_keywords": ["surface", "web", "service", "sadi"]}, {"paragraph_vector": [45.97771, 14.971903], "paragraph_keywords": ["vertices", "mesh", "e", "vertex"]}, {"paragraph_vector": [5.147047, -13.42011], "paragraph_keywords": ["potential", "ligand", "boltzmann", "poisson"]}, {"paragraph_vector": [-173.726028, -43.751613], "paragraph_keywords": ["classes", "rdf", "structure", "service"]}, {"paragraph_vector": [56.711597, -35.148193], "paragraph_keywords": ["surface", "algorithm", "probe", "ses"]}, {"paragraph_vector": [16.642683, -17.09811], "paragraph_keywords": ["charges", "residues", "based", "probe"]}, {"paragraph_vector": [4.147735, -1.928676], "paragraph_keywords": ["equation", "data", "ligand", "structures"]}, {"paragraph_vector": [-166.84375, -51.108749], "paragraph_keywords": ["ontology", "ligand", "entities", "entity"]}, {"paragraph_vector": [-175.193481, -45.584808], "paragraph_keywords": ["input", "service", "data", "class"]}, {"paragraph_vector": [-174.742736, -47.923542], "paragraph_keywords": ["mosumo", "service", "ligand", "protein"]}, {"paragraph_vector": [-175.887863, -45.320869], "paragraph_keywords": ["data", "services", "web", "order"]}, {"paragraph_vector": [-172.967102, -44.240192], "paragraph_keywords": ["ontology", "surface", "entities", "models"]}, {"paragraph_vector": [-177.694213, -50.707], "paragraph_keywords": ["rdf", "data", "vasco", "services"]}, {"paragraph_vector": [52.063549, -35.721755], "paragraph_keywords": ["pockets", "data", "mosumo", "electrostatics"]}, {"paragraph_vector": [-174.166732, -48.038867], "paragraph_keywords": ["mosumo", "surface", "model", "data"]}, {"paragraph_vector": [-174.780776, -46.9528], "paragraph_keywords": ["surface", "models", "sadi", "mosumo"]}], "content": {}, "doi": "10.1109/38.469512"}, {"uri": "27", "title": "High Performance Molecular Visualization: In-Situ and Parallel Rendering with EGL", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["John E. Stone", "Peter Messmer", "Robert Sisneros", "Klaus Schulten"], "summary": "Large scale molecular dynamics simulations produce terabytes of data that is impractical to transfer to remote facilities. It is therefore necessary to perform visualization tasks in-situ as the data are generated, or by running interactive remote visualization sessions and batch analyses co-located with direct access to high performance storage systems. A significant challenge for deploying visualization software within clouds, clusters, and supercomputers involves the operating system software required to initialize and manage graphics acceleration hardware. Recently, it has become possible for applications to use the Embedded-system Graphics Library (EGL) to eliminate the requirement for windowing system software on compute nodes, thereby eliminating a significant obstacle to broader use of high performance visualization applications. We outline the potential benefits of this approach in the context of visualization applications used in the cloud, on commodity clusters, and supercomputers. We discuss the implementation of EGL support in VMD, a widely used molecular visualization application, and we outline benefits of the approach for molecular visualization tasks on petascale computers, clouds, and remote visualization servers. We then provide a brief evaluation of the use of EGL in VMD, with tests using developmental graphics drivers on conventional workstations and on Amazon EC2 G2 GPUaccelerated cloud instance types. We expect that the techniques described here will be of broad benefit to many other visualization applications. Keywords-in-situ visualization; remote visualization; parallel rendering; molecular visualization;", "keywords": ["data", "numa", "cuda", "associated", "application", "server", "hardware", "resource", "windowing", "egl", "vmd", "running", "host", "memory", "support", "test", "compute", "amazon", "system", "mpi", "based", "performance", "library", "water", "cpu", "graphic", "gpu", "use", "required", "software", "x", "opengl", "gpus", "rendering", "visualization", "provided", "context", "node"], "document_vector": [-6.798729, -37.881916], "paragraphs": [{"paragraph_vector": [-158.884796, 51.255348], "paragraph_keywords": ["systems", "users", "access", "performance"]}, {"paragraph_vector": [-169.495239, 56.225845], "paragraph_keywords": ["software", "visualization", "system", "supercomputers"]}, {"paragraph_vector": [-161.510055, 55.695877], "paragraph_keywords": ["waters", "gpu", "nodes", "system"]}, {"paragraph_vector": [-169.043334, 63.167934], "paragraph_keywords": ["system", "jitter", "windowing", "server"]}, {"paragraph_vector": [158.563369, 61.839309], "paragraph_keywords": ["opengl", "graphics", "system", "context"]}, {"paragraph_vector": [176.95964, 60.025783], "paragraph_keywords": ["opengl", "egl", "use", "contexts"]}, {"paragraph_vector": [-168.035003, 60.462589], "paragraph_keywords": ["numa", "cpu", "compute", "memory"]}, {"paragraph_vector": [-174.531524, 60.220691], "paragraph_keywords": ["cpu", "host", "numa", "application"]}, {"paragraph_vector": [-170.2984, 59.562465], "paragraph_keywords": ["gpus", "gpu", "numa", "instance"]}, {"paragraph_vector": [-168.268981, 60.654872], "paragraph_keywords": ["gpu", "hardware", "enumeration", "node"]}, {"paragraph_vector": [139.781066, 61.097152], "paragraph_keywords": ["vmd", "display", "opengl", "egl"]}, {"paragraph_vector": [155.325454, 60.29145], "paragraph_keywords": ["egl", "managed", "context", "buffer"]}, {"paragraph_vector": [176.434234, 64.511116], "paragraph_keywords": ["vmd", "rendering", "cpu", "egl"]}, {"paragraph_vector": [170.909896, 64.398048], "paragraph_keywords": ["mpi", "amazon", "gpu", "rendering"]}, {"paragraph_vector": [179.383468, 68.083213], "paragraph_keywords": ["performance", "instances", "gpu", "results"]}, {"paragraph_vector": [166.531524, 64.731582], "paragraph_keywords": ["vmd", "rendering", "visualization", "egl"]}], "content": {}, "doi": "10.1109/VISUAL.2004.123"}, {"uri": "28", "title": "Immersive Molecular Visualization with Omnidirectional Stereoscopic Ray Tracing and Remote Rendering", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["John E. Stone", "William R. Sherman"], "summary": "Immersive molecular visualization provides the viewer with intuitive perception of complex structures and spatial relationships that are of critical interest to structural biologists. The recent availability of commodity head mounted displays (HMDs) provides a compelling opportunity for widespread adoption of immersive visualization by molecular scientists, but HMDs pose additional challenges due to the need for low-latency, high-frame-rate rendering. State-of-the-art molecular dynamics simulations produce terabytes of data that can be impractical to transfer from remote supercomputers, necessitating routine use of remote visualization. Hardware-accelerated video encoding has profoundly increased frame rates and image resolution for remote visualization, however round-trip network latencies would cause simulator sickness when using HMDs. We present a novel two-phase rendering approach that overcomes network latencies with the combination of omnidirectional stereoscopic progressive ray tracing and high performance rasterization, and its implementation within VMD, a widely used molecular visualization and analysis tool. The new rendering approach enables immersive molecular visualization with rendering techniques such as shadows, ambient occlusion lighting, depth-of-field, and high quality transparency, that are particularly helpful for the study of large biomolecular complexes. We describe ray tracing algorithms that are used to optimize interactivity and quality, and we report key performance metrics of the system. The new techniques can also benefit many other application domains. Keywords-immersive visualization; molecular visualization; remote visualization; ray tracing; head mounted displays;", "keywords": ["data", "rate", "hmds", "sample", "rt", "optix", "lighting", "update", "camera", "computing", "support", "osp", "number", "approach", "viewing", "system", "ray", "quality", "technique", "resolution", "engine", "work", "based", "surface", "scene", "performance", "geometry", "kernel", "display", "gpu", "use", "field", "eye", "view", "projection", "hmd", "vca", "pose", "image", "user", "rendering", "visualization", "video", "test", "loop"], "document_vector": [44.520053, -89.25698], "paragraphs": [{"paragraph_vector": [118.994857, 69.076202], "paragraph_keywords": ["rt", "visualization", "quality", "projections"]}, {"paragraph_vector": [132.031738, 65.264503], "paragraph_keywords": ["hmd", "image", "rendering", "rt"]}, {"paragraph_vector": [144.595886, 53.004383], "paragraph_keywords": ["projections", "hmd", "view", "projection"]}, {"paragraph_vector": [118.701469, 58.990928], "paragraph_keywords": ["projection", "axis", "eye", "direction"]}, {"paragraph_vector": [128.482589, 69.563903], "paragraph_keywords": ["rt", "data", "rendering", "level"]}, {"paragraph_vector": [103.947822, 54.169738], "paragraph_keywords": ["lighting", "quality", "features", "based"]}, {"paragraph_vector": [132.655197, 65.128898], "paragraph_keywords": ["rt", "kernels", "optix", "material"]}, {"paragraph_vector": [116.725837, 68.998664], "paragraph_keywords": ["rt", "samples", "image", "kernels"]}, {"paragraph_vector": [133.165924, 70.912185], "paragraph_keywords": ["lighting", "samples", "image", "rate"]}, {"paragraph_vector": [143.210693, 74.110366], "paragraph_keywords": ["enclosed", "gpus", "hmd", "approach"]}, {"paragraph_vector": [135.447601, 65.160232], "paragraph_keywords": ["rt", "vca", "geometry", "structures"]}, {"paragraph_vector": [145.98851, 62.646884], "paragraph_keywords": ["system", "network", "latency", "hmds"]}, {"paragraph_vector": [137.497558, 45.603797], "paragraph_keywords": ["display", "user", "hmd", "image"]}, {"paragraph_vector": [105.754219, 55.687721], "paragraph_keywords": ["hmd", "display", "image", "distortion"]}, {"paragraph_vector": [147.438522, 57.412822], "paragraph_keywords": ["hmd", "update", "loop", "display"]}, {"paragraph_vector": [164.478851, 64.501319], "paragraph_keywords": ["hmd", "tests", "performance", "rt"]}, {"paragraph_vector": [134.119598, 66.952774], "paragraph_keywords": ["rt", "capsid", "gpu", "occlusion"]}, {"paragraph_vector": [131.413314, 65.36988], "paragraph_keywords": ["visualization", "video", "approach", "rt"]}], "content": {}, "doi": "10.1109/CLEOE-IQEC.2007.4386338"}, {"uri": "29", "title": "Model-Guided Performance Tuning of Parameter Values: A Case Study with Molecular Dynamics Visualization", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yoonju Lee Nelson", "Bhupesh Bansal", "Aiichiro Nakano", "Kristina Lerman", "Marina del Rey"], "summary": "In this paper, we consider the interaction between application programmers and tools that automatically search a space of application-level parameters that are believed to impact the performance of an application significantly. We study performance tuning of a large scientific application, the visualization component of a molecular dynamics simulation. The key contribution of the approach is the use of high-level programmer-specified models of the expected performance behavior of individual parameters. We use these models to reduce the search space associated with the range of parameter values and achieve results that perform close to that of a more exhaustive search of the parameter space. With this case study, we show the importance of appropriate parameter selection, with the difference between best-case and worst-case performance with a particular input data set and processor configuration of up to a factor of 17. We show that through the use of models, we can drastically reduce search time, examining only 0.3% to 5% of the search space, and usually select an implementation that is close to the best performance, within 0.84% to 15%, even though the models are not completely accurate.", "keywords": ["data", "value", "tuning", "parameter", "application", "set", "space", "processor", "input", "point", "number", "decomposition", "system", "selected", "figure", "case", "programmer", "performance", "atom", "cache", "size", "model", "search", "use", "time", "balance", "capacity", "cell", "optimization"], "document_vector": [-25.4461, -13.73839], "paragraphs": [{"paragraph_vector": [-115.117965, 56.398197], "paragraph_keywords": ["application", "performance", "tuning", "programmer"]}, {"paragraph_vector": [-106.266151, 54.656539], "paragraph_keywords": ["models", "optimization", "search", "application"]}, {"paragraph_vector": [-104.340492, 53.603038], "paragraph_keywords": ["parameter", "search", "values", "system"]}, {"paragraph_vector": [-111.752944, 57.07376], "paragraph_keywords": ["application", "parameter", "parameters", "models"]}, {"paragraph_vector": [-113.526779, 55.776744], "paragraph_keywords": ["balance", "data", "register", "time"]}, {"paragraph_vector": [-137.194992, 52.875812], "paragraph_keywords": ["atoms", "data", "set", "analysis"]}, {"paragraph_vector": [-116.050315, 56.285282], "paragraph_keywords": ["parameter", "processor", "cache", "values"]}, {"paragraph_vector": [-110.067794, 56.442771], "paragraph_keywords": ["parameter", "values", "parameters", "section"]}, {"paragraph_vector": [-108.778305, 57.745319], "paragraph_keywords": ["parameter", "capacity", "figure", "default"]}, {"paragraph_vector": [-117.775268, 57.841022], "paragraph_keywords": ["time", "steps", "data", "values"]}, {"paragraph_vector": [-110.185874, 54.684352], "paragraph_keywords": ["performance", "points", "parameters", "tested"]}, {"paragraph_vector": [-112.499588, 56.237648], "paragraph_keywords": ["cell", "size", "processors", "decomposition"]}, {"paragraph_vector": [-109.869148, 57.245155], "paragraph_keywords": ["parameter", "size", "search", "cell"]}, {"paragraph_vector": [-110.504302, 56.649017], "paragraph_keywords": ["performance", "models", "search", "programmer"]}, {"paragraph_vector": [-100.515426, 48.28532], "paragraph_keywords": ["model", "models", "set", "search"]}], "content": {}, "doi": "10.1109/5992.790586"}, {"uri": "30", "title": "Interactive Visualization of Molecular Surface Dynamics", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Michael Krone", "Katrin Bidmon", "Thomas Ertl"], "summary": "Molecular dynamics simulations of proteins play a growing role in various fields such as pharmaceutical, biochemical and medical research. Accordingly, the need for high quality visualization of these protein systems raises. Highly interactive visualization techniques are especially needed for the analysis of time-dependent molecular simulations. Beside various other molecular representations the surface representations are of high importance for these applications. So far, users had to accept a trade-off between rendering quality and performance\u2014particularly when visualizing trajectories of time-dependent protein data. We present a new approach for visualizing the Solvent Excluded Surface of proteins using a GPU ray casting technique and thus achieving interactive frame rates even for long protein trajectories where conventional methods based on precomputation are not applicable. Furthermore, we propose a semantic simplification of the raw protein data to reduce the visual complexity of the surface and thereby accelerate the rendering without impeding perception of the protein\u2019s basic shape. We also demonstrate the application of our Solvent Excluded Surface method to visualize the spatial probability density for the protein atoms over the whole period of the trajectory in one frame, providing a qualitative analysis of the protein flexibility.", "keywords": ["called", "data", "analysis", "rate", "simulation", "face", "method", "described", "protein", "trajectory", "triangle", "number", "section", "fig", "computation", "ray", "quality", "implementation", "shader", "surface", "position", "root", "frame", "atom", "probe", "edge", "s", "solvent", "r", "gpu", "center", "radius", "cf", "patch", "time", "vdw", "user", "rendering", "visualization", "casting", "intersection", "torus"], "document_vector": [-69.613082, -54.86877], "paragraphs": [{"paragraph_vector": [70.986427, -18.995031], "paragraph_keywords": ["surface", "protein", "visualization", "trajectories"]}, {"paragraph_vector": [93.062049, 62.299869], "paragraph_keywords": ["rendering", "ray", "casting", "graphics"]}, {"paragraph_vector": [67.931381, -18.078365], "paragraph_keywords": ["surface", "section", "data", "vdw"]}, {"paragraph_vector": [20.98641, 1.754458], "paragraph_keywords": ["surface", "probe", "atom", "patches"]}, {"paragraph_vector": [25.552764, -13.712873], "paragraph_keywords": ["called", "atom", "atoms", "acid"]}, {"paragraph_vector": [73.761489, -36.560493], "paragraph_keywords": ["file", "ses", "data", "trajectories"]}, {"paragraph_vector": [22.224548, 46.500072], "paragraph_keywords": ["surface", "ses", "methods", "reduced"]}, {"paragraph_vector": [23.874011, -13.137265], "paragraph_keywords": ["rs", "probe", "ai", "atoms"]}, {"paragraph_vector": [14.771331, 11.23121], "paragraph_keywords": ["rs", "edge", "face", "probe"]}, {"paragraph_vector": [15.549099, 24.736419], "paragraph_keywords": ["rs", "vertices", "radius", "torus"]}, {"paragraph_vector": [16.336492, 46.46405], "paragraph_keywords": ["spheres", "ray", "ses", "described"]}, {"paragraph_vector": [12.261198, 49.330368], "paragraph_keywords": ["torus", "sphere", "j", "center"]}, {"paragraph_vector": [66.959938, 41.995502], "paragraph_keywords": ["rs", "moving", "data", "atoms"]}, {"paragraph_vector": [75.085075, 32.668735], "paragraph_keywords": ["protein", "data", "atoms", "rs"]}, {"paragraph_vector": [62.587181, -52.794486], "paragraph_keywords": ["atoms", "proteins", "chain", "backbone"]}, {"paragraph_vector": [70.996559, 31.508188], "paragraph_keywords": ["ses", "depth", "probability", "protein"]}, {"paragraph_vector": [42.800678, 32.674812], "paragraph_keywords": ["user", "probability", "ses", "voxel"]}, {"paragraph_vector": [20.262372, 47.872638], "paragraph_keywords": ["data", "mss", "quality", "number"]}, {"paragraph_vector": [84.20877, 20.384916], "paragraph_keywords": ["protein", "data", "rendering", "times"]}, {"paragraph_vector": [71.647796, -50.724758], "paragraph_keywords": ["frame", "ses", "msms", "atoms"]}, {"paragraph_vector": [85.152069, 22.292036], "paragraph_keywords": ["ses", "trajectories", "analysis", "core"]}, {"paragraph_vector": [148.872634, 3.477286], "paragraph_keywords": ["forschungsgemeinschaft", "dfg", "group", "discussion"]}], "content": {}, "doi": "10.1109/TPS.2005.845358"}, {"uri": "31", "title": "A New Haptic Interaction and Visualization Approach for Rigid Molecular Docking in Virtual Environments", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "Many biological activities take place through the physicochemical interaction of two molecules. This interaction occurs when one of the molecules finds a suitable location on the surface of the other for binding. This process is known as molecular docking, and it has applications to drug design. If we can determine which drug molecule binds to a particular protein, and how the protein interacts with the bonded molecule, we can possibly enhance or inhibit its activities. This information, in turn, can be used to develop new drugs that are more effective against diseases. In this paper, we propose a new approach based on a human-computer interaction paradigm for the solution of the rigid body molecular docking problem. In our approach, a rigid ligand molecule (i.e., drug) manipulated by the user is inserted into the cavities of a rigid protein molecule to search for the binding cavity, while the molecular interaction forces are conveyed to the user via a haptic device for guidance. We developed a new visualization concept, Active Haptic Workspace (AHW), for the efficient exploration of the large protein surface in high resolution using a haptic device having a small workspace. After the discovery of the true binding site and the rough alignment of the ligand molecule inside the cavity by the user, its final configuration is calculated off-line through time stepping molecular dynamics (MD) simulations. At each time step, the optimum rigid body transformations of the ligand molecule are calculated using a new approach, which minimizes the distance error between the previous rigid body coordinates of its atoms and their new coordinates calculated by the MD simulations. The simulations are continued until the ligand molecule arrives at the lowest energy configuration. Our experimental studies conducted with six human subjects testing six different molecular complexes demonstrate that given a ligand molecule and five potential binding sites on a protein surface, the subjects can successfully identify the true binding site using visual and haptic cues. Moreover, they can roughly align the ligand molecule inside the binding cavity such that the final configuration of the ligand molecule can be determined via the proposed MD simulations. *Correspondence to cbasdogan@ku.edu.tr. Subasi and Basdogan 73", "keywords": ["body", "visualization", "cavity", "md", "molecule", "feedback", "transformation", "et", "rate", "developed", "interaction", "simulation", "calculate", "coordinate", "workspace", "protein", "ligand", "approach", "number", "result", "calculated", "object", "proposed", "surface", "find", "binding", "based", "position", "atom", "configuration", "energy", "site", "docking", "search", "time", "given", "al", "device", "user", "force", "problem", "subject"], "document_vector": [142.206558, -55.490261], "paragraphs": [{"paragraph_vector": [-6.376038, -50.576538], "paragraph_keywords": ["molecules", "binding", "protein", "interaction"]}, {"paragraph_vector": [167.577758, -15.038866], "paragraph_keywords": ["feedback", "force", "use", "et"]}, {"paragraph_vector": [172.776443, -11.993283], "paragraph_keywords": ["device", "molecule", "user", "system"]}, {"paragraph_vector": [165.974487, -14.670474], "paragraph_keywords": ["ligand", "user", "protein", "device"]}, {"paragraph_vector": [165.285583, -17.960107], "paragraph_keywords": ["device", "dna", "afm", "molecule"]}, {"paragraph_vector": [165.698074, -21.95989], "paragraph_keywords": ["molecule", "ligand", "visualization", "simulations"]}, {"paragraph_vector": [168.480361, -13.709547], "paragraph_keywords": ["protein", "ligand", "docking", "section"]}, {"paragraph_vector": [1.785325, -51.611534], "paragraph_keywords": ["bonds", "docking", "molecules", "binding"]}, {"paragraph_vector": [7.855797, -56.538734], "paragraph_keywords": ["ligand", "atoms", "body", "molecule"]}, {"paragraph_vector": [20.65331, -58.299499], "paragraph_keywords": ["ligand", "molecule", "simulations", "transformation"]}, {"paragraph_vector": [29.429285, -60.39492], "paragraph_keywords": ["ligand", "matrix", "l", "body"]}, {"paragraph_vector": [19.408567, -58.102367], "paragraph_keywords": ["ligand", "molecule", "body", "configuration"]}, {"paragraph_vector": [164.689071, -25.911048], "paragraph_keywords": ["approach", "force", "atoms", "ligand"]}, {"paragraph_vector": [166.909805, -9.480119], "paragraph_keywords": ["forces", "force", "molecules", "molecule"]}, {"paragraph_vector": [166.163345, -15.570563], "paragraph_keywords": ["workspace", "device", "molecule", "surface"]}, {"paragraph_vector": [160.780059, -4.669626], "paragraph_keywords": ["protein", "rate", "cursor", "direction"]}, {"paragraph_vector": [164.689254, -16.305587], "paragraph_keywords": ["ligand", "molecule", "transformation", "coordinates"]}, {"paragraph_vector": [160.284072, -24.531555], "paragraph_keywords": ["molecule", "protein", "ligand", "coordinates"]}, {"paragraph_vector": [167.660888, -17.283378], "paragraph_keywords": ["molecule", "ligand", "subjects", "binding"]}, {"paragraph_vector": [160.163406, -21.377199], "paragraph_keywords": ["binding", "subjects", "site", "cavities"]}, {"paragraph_vector": [168.426208, -19.571985], "paragraph_keywords": ["binding", "subjects", "cavities", "site"]}, {"paragraph_vector": [167.261123, -17.157716], "paragraph_keywords": ["binding", "cavity", "subjects", "energy"]}, {"paragraph_vector": [165.374877, -18.016464], "paragraph_keywords": ["molecule", "ligand", "subjects", "approach"]}, {"paragraph_vector": [165.19374, -15.811635], "paragraph_keywords": ["molecule", "ligand", "body", "coordinates"]}, {"paragraph_vector": [166.40036, -19.009115], "paragraph_keywords": ["binding", "site", "subjects", "ligand"]}, {"paragraph_vector": [164.614807, -21.620914], "paragraph_keywords": ["ligand", "binding", "molecule", "docking"]}], "content": {}, "doi": "10.1109/TMAG.2007.902973"}, {"uri": "32", "title": "Coverage-Based Opacity Estimation for Interactive Depth of Field in Molecular Visualization", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sathish Kottravel", "Martin Falk", "Erik Sund\u00e9n", "Timo Ropinski"], "summary": "In this paper, we introduce coverage-based opacity estimation to achieve Depth of Field (DoF) effects when visualizing molecular dynamics (MD) data. The proposed algorithm is a novel object-based approach which eliminates many of the shortcomings of state-of-the-art image-based DoF algorithms. Based on observations derived from a physically-correct reference renderer, coverage-based opacity estimation exploits semi-transparency to simulate the blur inherent to DoF effects. It achieves high quality DoF effects, by augmenting each atom with a semi-transparent shell, which has a radius proportional to the distance from the focal plane of the camera. Thus, each shell represents an additional coverage area whose opacity varies radially, based on our observations derived from the results of multi-sampling DoF algorithms. By using the proposed technique, it becomes possible to generate high quality visual results, comparable to those achieved through ground-truth multi-sampling algorithms. At the same time, we obtain a significant speedup which is essential for visualizing MD data as it enables interactive rendering. In this paper, we derive the underlying theory, introduce coverage-based opacity estimation and demonstrate how it can be applied to real world MD data in order to achieve DoF effects. We further analyze the achieved results with respect to performance as well as quality and show that they are comparable to images generated with modern distributed ray tracing engines.", "keywords": ["disk", "opacity", "data", "dof", "structure", "focus", "d", "distance", "sample", "rendered", "diameter", "obtain", "algorithm", "point", "approach", "result", "object", "pixel", "figure", "depth", "ray", "based", "blur", "scene", "sampling", "atom", "coc", "effect", "region", "time", "image", "rendering", "visualization", "lens"], "document_vector": [-17.976531, -73.172683], "paragraphs": [{"paragraph_vector": [89.771827, -28.944583], "paragraph_keywords": ["algorithms", "structure", "comprehension", "data"]}, {"paragraph_vector": [92.11013, 57.975032], "paragraph_keywords": ["based", "dof", "effects", "visualization"]}, {"paragraph_vector": [4.598604, 80.296272], "paragraph_keywords": ["dof", "camera", "effects", "pixel"]}, {"paragraph_vector": [33.891361, 80.738677], "paragraph_keywords": ["scattering", "pixels", "object", "dof"]}, {"paragraph_vector": [56.966701, 66.951026], "paragraph_keywords": ["dof", "lens", "image", "effects"]}, {"paragraph_vector": [-7.561264, 73.310844], "paragraph_keywords": ["coc", "diameter", "distance", "image"]}, {"paragraph_vector": [-7.991338, 76.664695], "paragraph_keywords": ["coc", "samples", "slope", "diameter"]}, {"paragraph_vector": [3.648414, 78.298942], "paragraph_keywords": ["figure", "pixel", "samples", "ray"]}, {"paragraph_vector": [30.67427, 66.579246], "paragraph_keywords": ["disk", "object", "coc", "opacity"]}, {"paragraph_vector": [52.968017, 51.426132], "paragraph_keywords": ["atom", "dof", "atoms", "silhouette"]}, {"paragraph_vector": [32.027229, 57.943584], "paragraph_keywords": ["atom", "silhouette", "depth", "lists"]}, {"paragraph_vector": [27.557435, 56.608337], "paragraph_keywords": ["shading", "spheres", "points", "normals"]}, {"paragraph_vector": [79.363731, 46.339084], "paragraph_keywords": ["figure", "based", "data", "results"]}, {"paragraph_vector": [77.299415, 45.673713], "paragraph_keywords": ["dof", "based", "focus", "effect"]}, {"paragraph_vector": [68.898666, 42.829181], "paragraph_keywords": ["representations", "image", "investigate", "like"]}, {"paragraph_vector": [-157.329818, 10.279498], "paragraph_keywords": ["information", "e", "technology", "elliit"]}], "content": {}, "doi": "10.1109/SMA.1999.749339"}, {"uri": "33", "title": "empty", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["yUEchao Wang"], "summary": "mI lI, lIanQIng lIU, nIng XI, anD yUEchao Wang Biological Applications of a Nanomanipulator Based on AFM Cells are the struCtur al and functional unit of living organisms. Knowledge of cellular behaviors is critical for us to understand the underlying mechanisms that guide the physiological and pathological changes in living organisms. so far, virtually all knowledge of the molecular reactions in cells has come from the ensemble measurements done in test tubes with purified molecules [1], [2]. the disadvantage of this approach is that experiments with purified molecules require the lysis of cells, meaning that the obtained results may not reflect the real situations of living cells. On the cell surface, there are numerous types of biomolecules (e.g., proteins, lipids, and carbohydrates) whose activities are linked to various cellular functions and diseases, e.g., cell adhesion, tissue development, cellular communication, inflammation, neurodegenerative disorders, and cancer metastasis [3]. studies have shown that the biomolecules on the cell surface do not work alone, and they interact with each other in defined nanometer-scale regions to form molecular machineries, such as lipid rafts [3]. hence, tools that can observe and simultaneously manipulate individual molecules on living cells, without damaging their natural structures, are essential for us to understand", "keywords": ["living", "cantilever", "substrate", "curve", "molecule", "et", "manipulation", "perform", "drug", "detection", "method", "protein", "obtained", "cancer", "research", "approach", "tip", "degree", "system", "understanding", "figure", "spectroscopy", "cmc", "lymphoma", "nanomanipulator", "rituximab", "change", "process", "surface", "based", ".", "binding", "modulus", "receptor", "afm", "engineering", "al", "property", "image", "force", "cell"], "document_vector": [97.191993, 28.943996], "paragraphs": [{"paragraph_vector": [-50.5111, 28.988365], "paragraph_keywords": ["nanorobots", "construct", "nanorobot", "robot"]}, {"paragraph_vector": [-43.755317, 33.588203], "paragraph_keywords": ["afm", "based", "nanomanipulator", "feedback"]}, {"paragraph_vector": [-38.896804, 31.052806], "paragraph_keywords": ["afm", "nanomanipulator", "manipulation", "tip"]}, {"paragraph_vector": [-39.97121, 32.218486], "paragraph_keywords": ["cells", "junctions", "figure", "afm"]}, {"paragraph_vector": [-40.81306, 31.705772], "paragraph_keywords": ["cells", "lymphoma", "cell", "substrate"]}, {"paragraph_vector": [-39.033851, 27.846248], "paragraph_keywords": ["cell", "cells", "rituximab", "afm"]}, {"paragraph_vector": [-40.256484, 30.615715], "paragraph_keywords": ["curve", "afm", "hertz", "approach"]}, {"paragraph_vector": [-39.775882, 30.161643], "paragraph_keywords": ["cells", "cell", "modulus", "drug"]}, {"paragraph_vector": [-38.87144, 29.466917], "paragraph_keywords": ["cells", "cell", "cmc", "cancer"]}, {"paragraph_vector": [-35.601772, 26.641716], "paragraph_keywords": ["tip", "force", "cell", "afm"]}, {"paragraph_vector": [-38.862541, 29.834709], "paragraph_keywords": ["cell", "force", "receptor", "cells"]}, {"paragraph_vector": [-39.422405, 29.924867], "paragraph_keywords": ["cells", "cancer", "figure", "afm"]}, {"paragraph_vector": [-39.579238, 29.351377], "paragraph_keywords": ["afm", "cells", "cell", "cancer"]}, {"paragraph_vector": [-35.073451, 30.996278], "paragraph_keywords": ["afm", "cells", "speed", "cell"]}, {"paragraph_vector": [-38.373069, 29.652698], "paragraph_keywords": ["afm", "protein", "automated", "detection"]}, {"paragraph_vector": [-36.921886, 31.698806], "paragraph_keywords": ["afm", "degree", "university", "research"]}, {"paragraph_vector": [-43.314155, 33.174812], "paragraph_keywords": ["institute", "systems", "degree", "automation"]}], "content": {}, "doi": "10.1109/VARMS.2015.7151725"}, {"uri": "34", "title": "LifeBrush: Painting, simulating, and visualizing dense biomolecular environments", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Timothy Davison", "Faramarz Samavati", "Christian Jacob"], "summary": "LifeBrush is a Cyberworld for painting dynamic molecular illustrations in virtual reality (VR) that then come to life as interactive simulations. We designed our system for the biological mesoscale, a spatial scale where molecules inside cells interact to form larger structures and execute the functions of cellular life. We bring our immersive illustrations to life in VR using agent-based modelling and simulation. Our sketch-based brushes use discrete element texture synthesis to generate molecular-agents along the brush path derived from examples in a palette. In this article we add a new tool to sculpt the geometry of the environment and the molecules. We also introduce a new history based visualization that enables the user to interactively explore and distil, from the busy and chaotic mesoscale environment, the interactions between molecules that drive cellular processes. We demonstrate our system with a mitochondrion", "keywords": [], "document_vector": [-94.411758, -87.809638], "paragraphs": [{"paragraph_vector": [-130.188598, 21.078556], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/SMC.2013.100"}, {"uri": "35", "title": "Protein Protein Interaction Visualization Using VisANT", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Liangliang XU", "Fei ZHU"], "summary": "Visual tools are useful in providing an overview of relations that would be far more difficult to analyze the biological network. VisANT is a software platform for the visualization, mining, analysis and modeling of the biological networks. In our work we create and parse biological networks with VisANT and visml (a standard for exchanging Files that have visual markup).So that we can better understand the interaction between biological components. Keywords-VisANT; protein; visml; layout; KEGG", "keywords": ["color", "kegg", "method", "data", "protein", "network", "element", "edge", "visml", "interaction", "node"], "document_vector": [69.793922, 61.04026], "paragraphs": [{"paragraph_vector": [-129.834274, -36.672573], "paragraph_keywords": ["data", "network", "gene", "visml"]}, {"paragraph_vector": [-136.06575, -45.270797], "paragraph_keywords": ["nodes", "node", "element", "attribute"]}, {"paragraph_vector": [-120.120071, -32.11238], "paragraph_keywords": ["kegg", "edge", "protein", "pathway"]}, {"paragraph_vector": [126.64299, 4.331504], "paragraph_keywords": ["doc", "visml", "method", "."]}, {"paragraph_vector": [-154.118347, 15.03681], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/BIBE.2001.974406"}, {"uri": "36", "title": "Direct Visualization of Molecularly Thin Lubricant Films Using Low-Coherence Phase-Shifting Interferometry", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["H. Zhang", "T. Banno", "Y. Mitsuya", "K. Fukuzawa"], "summary": "We have developed a method utilizing low-coherence phase-shifting interferometry for directly visualizing molecularly thin lubricant films coated on diamond-like carbon surfaces. By introducing a high-precision feedback controlling phase-shifting system and an image subtraction technique, the method was improved to be capable of providing real-time three-dimensional measurements with a thickness resolution of sub-nanometer and a spatial resolution of several micrometers.", "keywords": ["lcpsi", "nagoya", "phase", "surface", "film", "shifting", "-"], "document_vector": [73.222824, -7.777187], "paragraphs": [{"paragraph_vector": [-56.782878, 45.272766], "paragraph_keywords": ["phase", "nagoya", "j", "reference"]}, {"paragraph_vector": [-54.390285, 47.699413], "paragraph_keywords": ["lcpsi", "surface", "phase", "nm"]}, {"paragraph_vector": [-57.153656, 44.565895], "paragraph_keywords": ["films", "resolution", "developed", "coated"]}], "content": {}, "doi": "10.1109/SVR.2017.30"}, {"uri": "37", "title": "BetaMol: Molecular Modeling, Analysis, and Visualization Software Based on the Beta-Complex Derived from the Voronoi Diagram", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Youngsong Cho", "Jae-Kwan Kim", "Chung-In Won", "Joonghyun Ryu", "Chong-Min Kim"], "summary": "Molecular shape is one of the most critical factors that determine molecular function. To properly understand the function of a molecule, it is necessary to explore its geometric properties more effectively and efficiently in addition to its physicochemical properties. Due to the complexity of the problems in biomolecular structure, in-silico approach is inevitable for many cases and is becoming more popular. In this paper, we introduce BetaMol, a comprehensive, powerful graphics-based software for molecular modeling, analysis, and visualization. BetaMol is based on the recent theory of the betacomplex which is derived from the Voronoi diagram of spheres. The entire set of powerful features of BetaMol is completely based on a single framework of the mathematically rigorous and computationally efficient theory of beta-complex. BetaMol is implemented in the standard C++ language with the OpenGL graphics library on Windows platform. Both BetaMol and some of its important functions ported for Linux platform are freely available at Voronoi Diagram Research Center website (http://voronoi.hanyang.ac.kr). Keywords-Voronoi diagram; quasi-triangulation; betacomplex; beta-shape; molecular modeling; molecular visualization; software; computational molecular biology; structural molecular biology", "keywords": ["data", "structure", "triangulation", "pocket", "molecule", "voronoi", "interaction", "protein", "ligand", "\u03b2", "diagram", "format", "defined", "interface", "surface", "complex", "consists", "atom", "-", "show", "file", "model", "betamol", "radius", "vertex", "beta", "boundary", "user", "shape"], "document_vector": [-107.903488, -2.04231], "paragraphs": [{"paragraph_vector": [26.870275, -22.751987], "paragraph_keywords": ["voronoi", "diagram", "atoms", "v"]}, {"paragraph_vector": [10.521587, -7.340376], "paragraph_keywords": ["q", "triangulation", "-", "structure"]}, {"paragraph_vector": [26.25385, -19.14908], "paragraph_keywords": ["beta", "radius", "atoms", "hull"]}, {"paragraph_vector": [14.486276, 0.880419], "paragraph_keywords": ["data", "beta", "structure", "format"]}, {"paragraph_vector": [39.940444, -74.468635], "paragraph_keywords": ["format", "radii", "atoms", "represents"]}, {"paragraph_vector": [24.612049, -20.225723], "paragraph_keywords": ["diagram", "voronoi", "-", "betamol"]}, {"paragraph_vector": [25.691637, -20.496055], "paragraph_keywords": ["qtf", "beta", "triangulation", "simplexes"]}, {"paragraph_vector": [36.365619, -14.893991], "paragraph_keywords": ["model", "molecules", "colors", "shape"]}, {"paragraph_vector": [-5.734202, -64.649559], "paragraph_keywords": ["beta", "molecules", "molecule", "shape"]}, {"paragraph_vector": [18.626705, -23.697614], "paragraph_keywords": ["molecule", "model", "shape", "defined"]}, {"paragraph_vector": [21.035949, -14.79757], "paragraph_keywords": ["surface", "beta", "computed", "lee"]}, {"paragraph_vector": [26.467981, -22.358486], "paragraph_keywords": ["interface", "interaction", "molecules", "atoms"]}, {"paragraph_vector": [24.630624, -18.247268], "paragraph_keywords": ["ligand", "protein", "boundary", "pocket"]}, {"paragraph_vector": [2.030107, -23.430055], "paragraph_keywords": ["pockets", "pocket", "protein", "tunnel"]}, {"paragraph_vector": [21.705842, -24.503652], "paragraph_keywords": ["ligand", "receptor", "energy", "problem"]}, {"paragraph_vector": [-0.342532, -52.155849], "paragraph_keywords": ["ligand", "library", "data", "user"]}, {"paragraph_vector": [27.869638, -23.234077], "paragraph_keywords": ["file", "library", "triangulation", "betamol"]}, {"paragraph_vector": [44.220645, -27.717731], "paragraph_keywords": ["user", "window", "molecule", "model"]}, {"paragraph_vector": [56.973983, -33.961097], "paragraph_keywords": ["molecule", "betamol", "complex", "beta"]}, {"paragraph_vector": [-158.57901, 9.82402], "paragraph_keywords": ["korea", "ministry", "education", "funded"]}], "content": {}, "doi": "10.1109/IEMBS.2009.5333932"}, {"uri": "38", "title": "Volumetric Visualization and Analysis of Molecular Interaction Field", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Li Wang", "Huagen Wan", "Qunsheng Peng", "Xin Chen", "Tao Wu", "Junhui Wang"], "summary": "Electrostatic and van der Waals interactions are of central importance for many biological processes. Yet it is difficult to visualize and analyze the two interactions with existing graphical representations of proteins such as wireframe, VDW, and molecular surface due to the three-dimensional nature of protein interaction distribution. The molecular interaction field (MIF) is defined as a three-dimensional uniform grid surrounding a protein of known structure with its every grid point storing the interaction of a probe with the protein. It would be more straightforward to volumetrically visualize and analyze the MIF for understanding of protein structure and function than existing graphical representations of proteins. A novel method for volumetrically visualizing and analyzing the MIF of a protein is proposed. The ray-casting algorithm is used to visualize the MIF to achieve convincing rendering effect and produce informative images that reveal the MIF volume features. The volume rendering results are used to analyze two example proteins, i.e., the HIV-1 protease (pdb id: 1a30) and the Dps protein (pdb id: 1dps), with biologically meaningful results being derived. KeywordsMIF; volumetric visualization; transfer function;", "keywords": ["professor", "transfer", "method", "protein", "volume", "structure", "pdb", ".", "surface", "vol", "route", "protease", "visualization", "rendering", "mif", "simulation", "dp"], "document_vector": [-99.259368, -44.116462], "paragraphs": [{"paragraph_vector": [78.321258, 3.197132], "paragraph_keywords": ["protease", "route", "simulation", "areas"]}, {"paragraph_vector": [71.592613, -0.938987], "paragraph_keywords": ["protein", "mif", "dps", "core"]}, {"paragraph_vector": [-6.614249, -21.320871], "paragraph_keywords": ["method", "professor", "surface", "."]}, {"paragraph_vector": [65.600578, 41.723976], "paragraph_keywords": [".", "vol", "volume", "transfer"]}], "content": {}, "doi": "10.1109/MHS.2006.320268"}, {"uri": "39", "title": "Atomistic Visualization: On-the-fly Data Extraction and Rendering", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Dipesh Bhattarai", "Bijaya B. Karki"], "summary": "A BST R A C T In this paper, we present an approach involving on-the-fly data extraction and rendering to achieve a real-time integration between analysis and visualization of atomistic simulation data. The integration is demonstrated along two directions: The first direction is to explore coordination environments by rendering spatial variation and temporal stability of the coordination states computed for each atom, and the second direction is to explore the aggregate atomic movement by computing ellipsoids. Ellipsoids are computed from covariance matrices of the time-varying positional data. The proposed approach is applied to visualize the data from first-principles molecular dynamics simulations.", "keywords": ["visualization", "data", "k", "analysis", "distance", "representation", "visualize", "rdf", "paper", "dynamic", "set", "particle", "information", "number", "diffusion", "system", "coordination", "color", "movement", "work", "line", "atom", "type", "r", "use", "stability", "time", "given", "environment"], "document_vector": [-179.041046, -81.154541], "paragraphs": [{"paragraph_vector": [85.907791, -35.199966], "paragraph_keywords": ["data", "visualization", "time", "coordination"]}, {"paragraph_vector": [101.605552, -48.91967], "paragraph_keywords": ["coordination", "copies", "data", "atoms"]}, {"paragraph_vector": [139.545913, -72.296279], "paragraph_keywords": ["data", "coordination", "atoms", "atom"]}, {"paragraph_vector": [78.70092, -31.809188], "paragraph_keywords": ["visualization", "data", "behavior", "section"]}, {"paragraph_vector": [78.321128, -39.58126], "paragraph_keywords": ["rdf", "distance", "data", "r"]}, {"paragraph_vector": [49.625415, -35.179653], "paragraph_keywords": ["coordination", "particle", "particles", "stability"]}, {"paragraph_vector": [63.815994, -37.755867], "paragraph_keywords": ["atom", "time", "type", "stability"]}, {"paragraph_vector": [169.640975, -75.385826], "paragraph_keywords": ["atom", "stability", "covariance", "matrix"]}, {"paragraph_vector": [112.792785, -65.829338], "paragraph_keywords": ["time", "visualization", "eigenvectors", "ellipsoid"]}, {"paragraph_vector": [67.332862, -34.89289], "paragraph_keywords": ["coordination", "atom", "color", "lines"]}, {"paragraph_vector": [53.486473, -18.964256], "paragraph_keywords": ["e", "coordination", "ellipsoids", "atoms"]}, {"paragraph_vector": [77.50634, -33.407588], "paragraph_keywords": ["visualization", "diffusion", "graphics", "tensor"]}], "content": {}, "doi": "10.1109/ISEIM.2014.6870834"}, {"uri": "40", "title": "Using Lambda Networks to Enhance Performance of Interactive Large Simulations", "timestamp": "0", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Shantenu Jha", "Mary-Ann Thyveetil", "Peter Coveney"], "summary": "The ability to use a visualisation tool to steer large simulations provides innovative and novel usage scenarios, e.g. the abiity to use new algorithms for the computation of free energy profiles along a nanopore [I]. However, we find that the performance of interactive simulations is sensitive to the quality of service of the network with variable latency and packet loss in particular having a detrimental effed. The use of dedicated networks (provisioned in this case as a circuit-switched, point-topoint optical lightpath or lambda) can lead to significant (50% or more) performance enhancement. When running on say 128 or 256 processors of a high-end supercomputer this saving has a significant value. We perform experiments to understand the impact of network characteristics on the performance of a large parallel classical molecular dynamics simulation when coupled interactively to a remote visualisation tool. This paper discusses the experiments performed and presents the results from the systematic studies.", "keywords": [], "document_vector": [-10.092327, -31.210399], "paragraphs": [{"paragraph_vector": [-129.203979, 22.263061], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/APMRC.2002.1037647"}, {"uri": "41", "title": "Druggability Analysis and Prediction based on Geometric Distances Between Amino Acid Residues and Protein Surface Pockets", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Makiko Miyoshi", "Ayaka Kaneko", "Takayuki Itoh", "Kei Yura"], "summary": "Protein is the major component of the organism. A concave (pocket) on a protein surface is known to be the best target for a drug to react. We previously presented a study on distance analysis between pockets and amino acid residue. We firstly identified pockets on the protein surface and then calculated distances between atoms of an amino acid residue and the deepest points or the outer loops of the pockets. We extracted proteins which at least one of the pockets are close to arbitrary pairs of amino acid residues, calculated the ratios of druggable proteins, and visualized the distribution of the ratios as a colored matrix. We suggested from the visualization results that particular pairs of amino acid residues may affect the druggability of the proteins in our previous study. This paper presents an extension of our study to explore the relevance between druggability of proteins and distances between a set of amino acid residues and protein surface pockets. Our technique treats the pockets as 20-dimensional vectors consisting of distances to each of amino acid residues, and applies GeodesicSOM with the set of the vectors. Spherical maps generated by GeodesicSOM are used to visualization of distribution of the pockets in the 20-dimensional vector space, and estimation of druggability of proteins with the 20dimensional vectors of the pockets. Keywords-Protein, Druggability, Visualization, SelfOrganizing Map.", "keywords": ["vector", "protein", "technique", "geodesicsom", "pocket", "druggability", "estimation", "distance", "following"], "document_vector": [-120.637611, -2.659329], "paragraphs": [{"paragraph_vector": [-8.131361, -25.214101], "paragraph_keywords": ["pocket", "technique", "surface", "protein"]}, {"paragraph_vector": [-9.297823, -25.557313], "paragraph_keywords": ["pockets", "druggability", "distance", "pk"]}, {"paragraph_vector": [-5.175912, -22.828903], "paragraph_keywords": ["pockets", "druggability", "following", "protein"]}, {"paragraph_vector": [-9.647061, -24.653289], "paragraph_keywords": ["pockets", "proteins", "vectors", "druggability"]}, {"paragraph_vector": [-11.241955, -26.712606], "paragraph_keywords": ["pockets", "technique", "estimation", "proteins"]}], "content": {}, "doi": "10.1109/CMV.2005.1"}, {"uri": "42", "title": "Using Haptic Technology for Education in Chemistry", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Shaykhah S. Aldosari", "Nourah Bint Abdulrahman", "Davide Marocco"], "summary": "In recent years the field of haptic and gesture studies is expanding and improvements are being made by developing, testing and refining devices along with software to give a sense of touch or to use gestures to interact and \u201cfeel\u201d three dimensional virtual objects. Such technologies have a significant role to play in education. The uses of modern devices based on gesture or haptic technologies and their applications in chemistry education, for example, may provide a platform to the students to better comprehend ideas about several chemical subjects. This paper covers the fundamentals of such technologies, studies a few commonly used devices, and concludes with the current use in education. A brief description of an educational gesture-based system for chemistry experiment simulations and molecular visualization is presented in this paper. It also describes some didactic scenarios and discusses current results and future works. Keywords\u2014 Haptics; haptic interface; technology enhanced learning; chemistry education; e-learning; simulation; molecular", "keywords": ["oil", "level", "molecule", "device", "user", "water", "system"], "document_vector": [116.377235, -25.71289], "paragraphs": [{"paragraph_vector": [170.812988, 5.4376], "paragraph_keywords": ["water", "oil", "user", "molecule"]}, {"paragraph_vector": [169.81314, 1.77473], "paragraph_keywords": ["water", "ions", "mix", "solution"]}, {"paragraph_vector": [171.880279, -8.754964], "paragraph_keywords": ["ion", "water", "potassium", "solution"]}, {"paragraph_vector": [174.083358, -10.882433], "paragraph_keywords": ["system", "technology", "required", "students"]}], "content": {}, "doi": "10.1109/PACIFICVIS.2016.7465258"}, {"uri": "43", "title": "The effects of different representations on malware motif identification", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ajit Narayanan", "Yi Chen", "Shaoning Pang"], "summary": "Sequence alignment is widely used in bioinformatics for revealing the genetic diversity of organisms and annotating gene functions by finding regions of similarity across biosequences. Such alignment requires sequences to be represented in the DNA or protein alphabet for tools such as Clustal to work. Previous work has demonstrated the feasibility of applying biosequence multiple alignment techniques to computer viral and worm signatures to find regions of similarity that can serve as malware \u2018motifs\u2019, or meta-signatures. However, it was not known how different ways of representing signatures in an appropriate biosequence alphabet would affect the alignment results. This paper investigates the effects of adopting three different ways of representing malware signatures on sequence alignment and motif identification. The results of the alignment were checked with perceptrons, decision tree and logistic regression. The best performing representation was used to derive rules in PRISM that give rise to \u2018motifs\u2019 that can perform the role of \u2018metasignatures\u2019. All analysis was undertaken on the publicly available data mining tool, Weka (Waikato Environment for Knowledge Analysis: http://www.cs.waikato.ac.nz/ml/weka/).", "keywords": ["accuracy", "alignment", "result", "virus", "sequence", "representation", "test", "signature", "table", "number", "worm", "prism", "aligned", "perceptrons", "classification"], "document_vector": [175.542861, 70.82991], "paragraphs": [{"paragraph_vector": [-90.706527, -46.069488], "paragraph_keywords": ["worm", "virus", "sequences", "weka"]}, {"paragraph_vector": [-92.933464, -47.557853], "paragraph_keywords": ["table", "sequences", "aligned", "alignment"]}, {"paragraph_vector": [-93.332107, -44.81385], "paragraph_keywords": ["signatures", "alignment", "virus", "accuracy"]}, {"paragraph_vector": [-87.282287, -47.443618], "paragraph_keywords": ["signatures", "sequence", ".", "malware"]}, {"paragraph_vector": [-88.868186, -45.867179], "paragraph_keywords": ["sequence", "journal", "alignment", "rules"]}], "content": {}, "doi": "10.1109/ISABEL.2010.5702889"}, {"uri": "44", "title": "ProtCV: A Tool for Extracting, Visualizing and Validating Protein Clusters Using Mass Spectra Peak-Lists", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Stavroula Ventoura", "Eugenia G. Giannopoulou", "Elias S. Manolakos"], "summary": "We present ProtCV (Protein Clustering and Visualization) a new software tool for grouping samples (mass spectra peak-lists) emanating from a high throughput proteomics analysis based on their spectral similarities, and summarizing effectively for the user the clustering results using advanced visualization methods. A unique feature of ProtCV is that it can compare clustering methods applied to the same data set based on several validity indices to assist the user identify groupings that seem to capture best the underlying structure of the data set. Moreover, ProtCV can assist in formulating hypotheses about the potential role of unidentified proteins in a cluster, identify sets of proteins which act jointly at a specific biological state etc. All these data set mining and exploration operations are very useful for interpreting the results of high throughput MS based proteomics analyses that are commonly used for biomarkers discovery.", "keywords": ["cluster", "method", "list", "protein", "protcv", "data", "bin", "k", "research", "visualization", "identification", "proteomics", "result", "peak", "clustering"], "document_vector": [167.387405, 80.449394], "paragraphs": [{"paragraph_vector": [-77.077133, -39.639091], "paragraph_keywords": ["protcv", "clustering", "distance", "lists"]}, {"paragraph_vector": [-73.392959, -39.25119], "paragraph_keywords": ["peak", "clusters", "peaks", "data"]}, {"paragraph_vector": [-78.548995, -37.765213], "paragraph_keywords": ["cluster", "bins", "protein", "clusters"]}, {"paragraph_vector": [-79.045776, -29.463979], "paragraph_keywords": ["clustering", "protein", "proteomics", "data"]}, {"paragraph_vector": [-81.805435, -40.750297], "paragraph_keywords": ["research", "proteomics", "petrolekas", "kossida"]}], "content": {}, "doi": "10.1109/VARMS.2015.7151719"}, {"uri": "45", "title": "Visualization of molecular machinery using agent-based animation", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Daniel Gehrer", "Ivan Viola"], "summary": "This paper proposes an agent-based model for animating molecular machines. Usually molecular machines are visualized using key-frame animation. Creating large molecular assemblies with key-frame animation in standard 3D software can be a tedious task, because hundreds or thousands of molecular particles have to be animated by hand, considering various biological phenomena. To avoid repetitive animation of molecular particles, a prototypic framework is implemented, that employs an agent-based approach. Instead of animating the molecular particles directly, the framework utilizes a behavior description for each type of molecular particle. The animation results from the molecular particles interacting with each other as defined by their behavior. Interaction between molecular particles is enabled by an abstract model that is implemented by the framework. The methodology for creating the framework was driven through learning by example. Three molecular machines are visualized using the framework. During this process, the framework was iteratively improved and extended. The resulted animations demonstrate that agent-based animation is a viable option for molecular machines.", "keywords": ["machine", "entity", "data", "bound", "molecule", "description", "et", "way", "oxygen", "animator", "create", "set", "proton", "agent", "sensor", "biologist", "approach", "framework", "atp", "defined", "state", "bind", "process", "work", "binding", "interact", "level", "animation", "site", "provides", "model", "software", "al", "environment", "conformation", "animated", "behavior"], "document_vector": [-82.490005, -28.504215], "paragraphs": [{"paragraph_vector": [116.80986, -38.733028], "paragraph_keywords": ["machines", "animated", "processes", "create"]}, {"paragraph_vector": [126.825576, -35.020477], "paragraph_keywords": ["behavior", "animation", "framework", "model"]}, {"paragraph_vector": [99.141319, -10.546298], "paragraph_keywords": ["et", "software", "al", "structures"]}, {"paragraph_vector": [112.950805, -17.173357], "paragraph_keywords": ["et", "animations", "biologists", "al"]}, {"paragraph_vector": [-171.647903, -35.040023], "paragraph_keywords": ["agents", "al", "et", "cells"]}, {"paragraph_vector": [122.436195, -35.068561], "paragraph_keywords": ["animation", "et", "particles", "behavior"]}, {"paragraph_vector": [126.398063, -35.63684], "paragraph_keywords": ["framework", "machine", "machines", "model"]}, {"paragraph_vector": [128.055755, -34.340618], "paragraph_keywords": ["model", "machines", "interact", "molecules"]}, {"paragraph_vector": [44.186473, -69.689117], "paragraph_keywords": ["molecules", "entity", "model", "environment"]}, {"paragraph_vector": [123.008811, -50.510253], "paragraph_keywords": ["entity", "sensor", "conformation", "cone"]}, {"paragraph_vector": [33.531032, -64.563766], "paragraph_keywords": ["binding", "sites", "direction", "entity"]}, {"paragraph_vector": [81.495071, -60.182121], "paragraph_keywords": ["entity", "position", "entities", "compounds"]}, {"paragraph_vector": [127.050575, -37.044532], "paragraph_keywords": ["behavior", "environment", "entity", "entities"]}, {"paragraph_vector": [139.782119, -40.66241], "paragraph_keywords": ["entity", "state", "behavior", "bound"]}, {"paragraph_vector": [138.822601, -40.747699], "paragraph_keywords": ["entity", "conformation", "entities", "state"]}, {"paragraph_vector": [135.316879, -49.948143], "paragraph_keywords": ["entity", "behavior", "binding", "trajectory"]}, {"paragraph_vector": [126.142692, -33.164104], "paragraph_keywords": ["binding", "framework", "proton", "site"]}, {"paragraph_vector": [123.926132, -37.080081], "paragraph_keywords": ["oxygen", "binding", "hemoglobin", "framework"]}, {"paragraph_vector": [119.539886, -41.298492], "paragraph_keywords": ["atp", "oxygen", "binding", "energy"]}, {"paragraph_vector": [107.572227, -32.549922], "paragraph_keywords": ["kinesin", "head", "atp", "bound"]}, {"paragraph_vector": [122.441139, -34.186428], "paragraph_keywords": ["animation", "head", "description", "environment"]}, {"paragraph_vector": [128.661834, -35.087333], "paragraph_keywords": ["animation", "animator", "description", "abstraction"]}, {"paragraph_vector": [122.940795, -29.624216], "paragraph_keywords": ["framework", "work", "changes", "features"]}, {"paragraph_vector": [-156.103271, 9.52347], "paragraph_keywords": ["curie", "career", "integration", "grant"]}], "content": {}, "doi": "10.1109/ISVD.2011.15"}, {"uri": "46", "title": "Scalable In situ Analysis of Molecular Dynamics Simulations", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Preeti Malakar", "Christopher Knight", "Todd Munson", "Venkatram Vishwanath", "Michael E. Papka"], "summary": "Analysis of scientific simulation data enables scientists to glean insight from simulations. In situ analysis, which can be simultaneously executed with the simulation, mitigates I/O bottlenecks and can accelerate discovery of new phenomena. However, in typical modes of operation, this requires either stalling simulation during analysis phase or transferring data for analysis. We study the scalability challenges of timeand space-shared modes of analyzing large-scale molecular dynamics simulations. We also propose topology-aware mapping for simulation and analysis. We demonstrate the benefits of our approach using LAMMPS code on two supercomputers.", "keywords": ["data", "execution", "partition", "analysis", "simulation", "situ", "space", "particle", "network", "shared", "mode", "step", "core", "interleaved", "process", "code", "lammps", "mapping", "performance", "cache", "mira", "time", "neighbor", "rank", "node"], "document_vector": [6.415036, -33.929344], "paragraphs": [{"paragraph_vector": [-167.899505, 41.008731], "paragraph_keywords": ["analysis", "simulation", "cores", "simulations"]}, {"paragraph_vector": [-168.66484, 39.837898], "paragraph_keywords": ["analysis", "simulation", "situ", "network"]}, {"paragraph_vector": [-171.246215, 39.586833], "paragraph_keywords": ["particles", "simulation", "application", "situ"]}, {"paragraph_vector": [-165.190063, 52.093879], "paragraph_keywords": ["analysis", "simulation", "partition", "step"]}, {"paragraph_vector": [-169.982513, 40.8945], "paragraph_keywords": ["simulation", "analysis", "partition", "processes"]}, {"paragraph_vector": [-169.427246, 39.18787], "paragraph_keywords": ["simulation", "analysis", "time", "neighbor"]}, {"paragraph_vector": [-165.566894, 41.436176], "paragraph_keywords": ["analysis", "simulation", "processes", "data"]}, {"paragraph_vector": [-165.59201, 49.341144], "paragraph_keywords": ["theta", "mira", "code", "time"]}, {"paragraph_vector": [-166.419601, 39.152706], "paragraph_keywords": ["ranks", "shared", "nodes", "simulation"]}, {"paragraph_vector": [-170.221771, 40.201686], "paragraph_keywords": ["shared", "simulation", "performance", "analysis"]}, {"paragraph_vector": [-170.418746, 40.006179], "paragraph_keywords": ["analysis", "shared", "mira", "mode"]}, {"paragraph_vector": [-167.623291, 39.671516], "paragraph_keywords": ["analysis", "simulation", "execution", "performance"]}], "content": {}, "doi": "10.1109/VBC.1990.109303"}, {"uri": "47", "title": "CALI: A novel visual model for frequent pattern mining in protein-ligand graphs", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Susana Medina G", "Alexandre V Fassio", "Sabrina de A Silveira", "Carlos H da Silveira", "Raquel C de Melo-Minardi"], "summary": "Protein-ligand interaction (PLI) networks show how proteins interact with small non-protein ligands through noncovalent bonding. Understanding such interactions is a crucial step towards ligand prediction, target identification and drug design. We propose CALI (Complex network-based Analysis of protein-Ligand Interactions), a graph-based, visual strategy coupled with complex network topological properties to summarize and detect frequent patterns in PLIs. Patterns obtained with CALI were compared to experimentally determined protein-ligand interactions from the CDK2 and Ricin dataset. For the CDK2, CALI found 90% of interacting residues, and all residues of the Ricin that interact with ligands. We devised a powerful visual and interactive strategy to analyze the data, providing a general view of the interaction dataset, showing the most common PLIs from a global perspective. Availability: http://bioinfo.dcc.ufmg.br/cali/ Keywords-graph; pattern; protein; ligand; visualization", "keywords": ["cali", "dataset", "fsm", "graph", "interaction", "protein", "network", "ligand", "ricin", "figure", "based", "atom", "edge", "model", "pattern", "plis", "property", "residue", "node"], "document_vector": [-23.921607, 48.304718], "paragraphs": [{"paragraph_vector": [-56.881717, -43.586738], "paragraph_keywords": ["graph", "networks", "ligand", "protein"]}, {"paragraph_vector": [-49.501728, -41.215652], "paragraph_keywords": ["based", "interactions", "protein", "diagrams"]}, {"paragraph_vector": [-105.372756, -52.486171], "paragraph_keywords": ["atoms", "graph", "nodes", "labeled"]}, {"paragraph_vector": [-44.489105, -40.565055], "paragraph_keywords": ["atoms", "figure", "graph", "protein"]}, {"paragraph_vector": [-38.580543, -35.66814], "paragraph_keywords": ["protein", "atoms", "set", "nodes"]}, {"paragraph_vector": [-123.084587, -44.662483], "paragraph_keywords": ["nodes", "model", "ligand", "network"]}, {"paragraph_vector": [-36.589839, -34.597686], "paragraph_keywords": ["ricin", "protein", "nodes", "ligand"]}, {"paragraph_vector": [-39.564865, -36.609066], "paragraph_keywords": ["degree", "residues", "cali", "nodes"]}, {"paragraph_vector": [-38.753498, -36.814037], "paragraph_keywords": ["connected", "interactions", "cali", "ligand"]}, {"paragraph_vector": [-37.662857, -38.687156], "paragraph_keywords": ["interactions", "patterns", "considering", "dataset"]}], "content": {}, "doi": "10.1109/ITMC.2013.7352656"}, {"uri": "48", "title": "MDMap : A System for Data-Driven Layout and Exploration of Molecular Dynamics Simulations", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Robert Patro", "Cheuk Yiu Ip", "Sujal Bista", "Samuel S. Cho", "Amitabh Varshney"], "summary": "Contemporary molecular dynamics simulations result in a glut of simulation data, making analysis and discovery a difficult and burdensome task. We present MDMap, a system designed to summarize long-running molecular dynamics (MD) simulations. We represent a molecular dynamics simulation as a state transition graph over a set of intermediate (stable and semi-stable) states. The transitions amongst the states together with their frequencies represent the flow of a biomolecule through the trajectory space. MDMap automatically determines potential intermediate conformations and the transitions amongst them by analyzing the conformational space explored by the MD simulation. MDMap is an automated system to visualize MD simulations as state-transition diagrams, and can replace the current tedious manual layouts of biomolecular folding landscapes with an automated tool. The layout of the representative states and the corresponding transitions among them is presented to the user as a visual synopsis of the long-running MD simulation. We compare and contrast multiple presentations of the state transition diagrams, such as conformational embedding, and spectral, hierarchical, and force-directed graph layouts. We believe this system could provide a road-map for the visualization of other stochastic time-varying simulations in a variety of different domains.", "keywords": ["visualization", "present", "data", "transition", "md", "embedding", "distance", "visualize", "graph", "simulation", "dynamic", "set", "cluster", "protein", "network", "trajectory", "biomolecules", "number", "approach", "system", "clustering", "figure", "state", "process", "based", "work", "level", "edge", "layout", "use", "time", "given", "user", "folding", "conformation"], "document_vector": [-60.4985, 10.025292], "paragraphs": [{"paragraph_vector": [126.346412, -46.01612], "paragraph_keywords": ["simulations", "time", "biomolecules", "folding"]}, {"paragraph_vector": [32.053955, -85.674766], "paragraph_keywords": ["transition", "states", "state", "visualization"]}, {"paragraph_vector": [16.662343, -78.969291], "paragraph_keywords": ["structures", "visualization", "work", "proteins"]}, {"paragraph_vector": [131.096084, -50.411769], "paragraph_keywords": ["visualization", "visualize", "folding", "graph"]}, {"paragraph_vector": [139.634231, -55.78688], "paragraph_keywords": ["dynamics", "conformations", "set", "time"]}, {"paragraph_vector": [4.277846, -88.230735], "paragraph_keywords": ["cell", "telomerase", "number", "layouts"]}, {"paragraph_vector": [24.991577, -68.72631], "paragraph_keywords": ["rmsd", "conformations", "simulation", "distance"]}, {"paragraph_vector": [32.813907, -69.240531], "paragraph_keywords": ["clusters", "cluster", "distance", "representative"]}, {"paragraph_vector": [23.634496, -74.808692], "paragraph_keywords": ["state", "clusters", "given", "set"]}, {"paragraph_vector": [19.131828, -86.68402], "paragraph_keywords": ["clusters", "transitions", "state", "cluster"]}, {"paragraph_vector": [15.38366, -82.854736], "paragraph_keywords": ["cluster", "edges", "transitions", "conformation"]}, {"paragraph_vector": [13.892832, -89.494178], "paragraph_keywords": ["graph", "state", "transition", "level"]}, {"paragraph_vector": [17.520814, -87.064498], "paragraph_keywords": ["layout", "graph", "states", "embedding"]}, {"paragraph_vector": [60.939945, -89.651779], "paragraph_keywords": ["layout", "graph", "edges", "attempts"]}, {"paragraph_vector": [58.429996, -83.750518], "paragraph_keywords": ["simulations", "transitions", "performed", "conformations"]}, {"paragraph_vector": [-28.835462, -83.719726], "paragraph_keywords": ["states", "edge", "graph", "state"]}, {"paragraph_vector": [-2.462762, -85.99105], "paragraph_keywords": ["states", "layout", "state", "transition"]}, {"paragraph_vector": [32.092273, -89.422653], "paragraph_keywords": ["user", "state", "graph", "number"]}], "content": {}, "doi": "10.1109/CEIDP.2004.1364222"}, {"uri": "49", "title": "Improved Quadric Surface Impostors for Large Bio-Molecular Visualization", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Pranav D Bagur", "Nithin Shivashankar", "Vijay Natarajan"], "summary": "The shape of biomolecules, such as proteins, may be represented using different representations like space-fill, ballstick, backbone, or secondary structures. The secondary structure of proteins, comprising of sheet-like, helix-like and loops structures, represent a higher level abstraction of its structure. With ever increasing sizes of protein structure data produced by high resolution x-ray crystallography and cryo-electron microscopy, biologists often rely on visualizations to better understand the overall structure of proteins. In this paper, we present a unified framework for accelerating the rendering of various representations of these structure using GPUs. The framework first produces \u201cimpostor primitives\u201d, which are simple linear element approximations of quadric objects, such as spheres, cylinders, and helices. Next, the rasterizations of the impostors are corrected to produce pixel-precise renderings of the quadric objects. We incorporate this framework into a bio-molecular visualization tool proteinvis to demonstrate quantitative and qualitative performance gains over earlier approaches for rendering various representations of proteins.", "keywords": ["structure", "impostor", "stage", "representation", "helix", "acid", "protein", "point", "primitive", "framework", "bond", "figure", "ray", "sphere", "quality", "backbone", "shader", ".", "axis", "direction", "geometry", "element", "atom", "spline", "graphic", "cylinder", "model", "use", "eye", "view", "gpus", "plane", "rendering", "visualization"], "document_vector": [-123.505081, -73.644577], "paragraphs": [{"paragraph_vector": [88.862182, 11.963271], "paragraph_keywords": ["models", "copies", "structure", "data"]}, {"paragraph_vector": [113.112434, 48.570945], "paragraph_keywords": ["rendering", "gpus", "accelerate", "quality"]}, {"paragraph_vector": [58.216464, 15.856284], "paragraph_keywords": ["elements", "framework", "rendering", "objects"]}, {"paragraph_vector": [82.768394, 1.543173], "paragraph_keywords": ["acids", "protein", "element", "biomolecules"]}, {"paragraph_vector": [70.162269, -9.402255], "paragraph_keywords": ["backbone", "atoms", "bonds", "sheets"]}, {"paragraph_vector": [26.075696, 57.934841], "paragraph_keywords": ["vertex", "view", "stage", "elements"]}, {"paragraph_vector": [35.448211, 61.027908], "paragraph_keywords": ["elements", "model", "impostors", "shader"]}, {"paragraph_vector": [21.313922, 64.465354], "paragraph_keywords": ["stage", "eye", "sphere", "quad"]}, {"paragraph_vector": [26.582567, 58.554149], "paragraph_keywords": ["cylinder", "point", "view", "planes"]}, {"paragraph_vector": [29.58428, 59.724899], "paragraph_keywords": ["cylinder", "helix", "axis", "point"]}, {"paragraph_vector": [73.088645, 9.403976], "paragraph_keywords": ["impostor", "cylinder", "representations", "spline"]}, {"paragraph_vector": [68.973678, 2.346376], "paragraph_keywords": ["helix", "axis", "section", "sample"]}, {"paragraph_vector": [72.877891, 9.859939], "paragraph_keywords": ["vmd", "visualization", "compare", "helix"]}, {"paragraph_vector": [76.79808, 14.996977], "paragraph_keywords": ["rendering", "helices", "backbone", "impostors"]}, {"paragraph_vector": [49.908332, 11.054855], "paragraph_keywords": [".", "protein", "models", "based"]}, {"paragraph_vector": [92.067993, 5.952685], "paragraph_keywords": ["guide", "graph", "publishing", "boston"]}], "content": {}, "doi": "10.1109/SUPERC.1990.130106"}, {"uri": "50", "title": "A Flexible Volumetric Comparison of Protein Cavities can Reveal Patterns in Ligand Binding Specificity", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ziyi Guo", "Trevor Kuhlengel", "Steven Stinson", "Seth Blumenthal", "Brian Y.Chen", "Soutir Bandyopadhyay"], "summary": "Conformational flexibility is an underlying cause of error in all comparisons of protein structure. Using flexible representations, some comparison algorithms can identify subtle functional similarities among distantly related proteins even when they exhibit different backbone conformations. The same techniques are not designed to identify subtle variations among closely related proteins that might cause differences in specificity. In such cases, molecular flexibility obscures structural details that influence the specific recognition of similar but non-identical ligands. To enhance the analysis of ligand binding specificity, this paper presents FAVA (Flexible Aggregate Volumetric Analysis), a conformationally robust tool for comparing similar binding cavities with different binding preferences. FAVA examines a large number of conformational samples to characterize local flexibility using Constructive Solid Geometry. Using molecular dynamics simulations as a source for conformational samples, we used FAVA to analyze a nonredundant sample of serine protease and enolase structures. Different snapshots from the same proteins exhibited significant variations in binding cavity shape. Nonetheless, analysis with FAVA revealed subfamilies with different binding preferences. FAVA also identified amino acids associated with differences in binding preferences, predicting established experimental results. These results illustrate a new approach to flexible comparison that uses sampled conformational data. It reveals that detailed comparisons of very similar proteins, such as those within small ligand binding cavities, are possible even in the presence of conformational flexibility. Identifying influences on specificity in this manner points to new applications of protein engineering and drug design. \u2217equal contribution. \u2020Corresponding author. Permission to make digital or hard copies of all or part of this work for personal or classroom use is granted without fee provided that copies are not made or distributed for profit or commercial advantage and that copies bear this notice and the full citation on the first page. Copyrights for components of this work owned by others than ACM must be honored. Abstracting with credit is permitted. To copy otherwise, or republish, to post on servers or to redistribute to lists, requires prior specific permission and/or a fee. Request permissions from permissions@acm.org. BCB\u201914, September 20\u201323, 2014, Newport Beach, CA, USA. Copyright 2014 ACM 978-1-4503-2894-4/14/09 ...$15.00. http://dx.doi.org/10.1145/2649387.2649428.", "keywords": ["enolase", "cavity", "alignment", "data", "structure", "volume", "shape", "bcb", "k", "analysis", "acid", "sample", "variation", "intersection", "simulation", "flexibility", "preference", "protein", "algorithm", "csg", "difference", "ligand", "point", "fava", "selected", "fig", "ai", "overlap", "specificity", "pdb", "based", "surface", "binding", "use", "region", "serine", "generated", "sampled", "comparison", "conformation", "elastase"], "document_vector": [-99.715629, 39.970691], "paragraphs": [{"paragraph_vector": [-53.269496, -50.922691], "paragraph_keywords": ["proteins", "binding", "structures", "ligand"]}, {"paragraph_vector": [-17.300273, -51.05754], "paragraph_keywords": ["regions", "proteins", "cavity", "representation"]}, {"paragraph_vector": [-25.658697, -55.068893], "paragraph_keywords": ["samples", "protein", "binding", "fava"]}, {"paragraph_vector": [-21.2054, -52.525375], "paragraph_keywords": ["samples", "use", "binding", "ligand"]}, {"paragraph_vector": [7.064465, -29.604907], "paragraph_keywords": ["fig", "ai", "cavity", "cavities"]}, {"paragraph_vector": [-11.655558, -57.204818], "paragraph_keywords": ["k", "regions", "samples", "b"]}, {"paragraph_vector": [-15.107707, -48.517547], "paragraph_keywords": ["bj", "selected", "ai", "b"]}, {"paragraph_vector": [-30.429346, -62.92007], "paragraph_keywords": ["structures", "acids", "removed", "hydrophobic"]}, {"paragraph_vector": [-24.241939, -59.156852], "paragraph_keywords": ["simulation", "solvent", "protein", "equilibration"]}, {"paragraph_vector": [-168.040283, 42.442047], "paragraph_keywords": ["samples", "regions", "protein", "binding"]}, {"paragraph_vector": [-7.086496, -49.438484], "paragraph_keywords": ["cavities", "binding", "csg", "operations"]}, {"paragraph_vector": [-28.285137, -51.743621], "paragraph_keywords": ["regions", "samples", "overlap", "approximation"]}, {"paragraph_vector": [-20.904911, -50.634998], "paragraph_keywords": ["regions", "cavities", "pdb", "elastase"]}, {"paragraph_vector": [-13.382918, -50.418758], "paragraph_keywords": ["acids", "intersection", "cavities", "cavity"]}, {"paragraph_vector": [-10.938072, -46.865734], "paragraph_keywords": ["fava", "binding", "flexibility", "protein"]}, {"paragraph_vector": [-36.715988, -51.66164], "paragraph_keywords": ["protein", "analysis", "representations", "chen"]}, {"paragraph_vector": [-59.231731, -50.765327], "paragraph_keywords": ["binding", "protein", "biol", "proteins"]}, {"paragraph_vector": [-65.869644, -49.793285], "paragraph_keywords": ["protein", "structure", "alignment", "comparison"]}, {"paragraph_vector": [-47.568355, -49.224441], "paragraph_keywords": ["protein", "structure", "alignment", "structures"]}, {"paragraph_vector": [-44.415531, -56.916152], "paragraph_keywords": ["protein", "alignment", "analysis", "modeling"]}], "content": {}, "doi": "10.1109/3DVis.2014.7160101"}, {"uri": "51", "title": "Visual-X2: Scoring and Visualization Tool for Analysis of Protein Crystallization Trial Images", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Suraj Subedi", "Marc L. Pusey"], "summary": "Protein crystallization experiments require the inspection and analysis of a large number of crystal images. So, manual inspection of these images is very inefficient and might risk missing successful results. Visual-X2 is a tool developed to aid the user for quick and efficient visualization and analysis of the results of the experiments. In this paper, we discuss various features of Visual-X2 and their applicability in the crystallization experiment. Keywords\u2014 Protein Crystallization, Visual-X2, visualization and analysis", "keywords": ["crystallization", "protein", "plate", "user", "analysis", "experiment", "cocktail"], "document_vector": [-158.046386, 25.147747], "paragraphs": [{"paragraph_vector": [-16.561918, -4.47334], "paragraph_keywords": ["cocktail", "concentration", "layout", "plate"]}, {"paragraph_vector": [-21.995204, -4.170708], "paragraph_keywords": ["user", "select", "cocktails", "crystallization"]}, {"paragraph_vector": [-34.27523, -17.067216], "paragraph_keywords": ["analysis", "file", "experiment", "conditions"]}], "content": {}, "doi": "10.1109/TVCG.2012.282"}, {"uri": "52", "title": "RLC(M) circuit models of protein structure: analysis, visualization, shape synthesis, and pattern matching", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["G. Sampath"], "summary": "Protein structure at the secondary and tertiary protein's primary sequence. These methods generate mostly levels is modeled using RLCM circuits via the mapping secondary level (and some tertiary) structure and are turn--R, helix--L, sheet or salt/disulphide bridge-4>C, and helix augmented by modifications for (more) tertiary structure as pair--M. Standard network analysis methods may be applied desired. Section VI considers the folding problem by to the resulting RLC(M) circuit, whose circuit topology is restricted because of the linear nature of the protein's viewing the tertiary level cIrcuit as a spatial proxy for primary sequence (for example, a secondary circuit can be a protein shapes. Section VII presents other possible uses for Foster I but not Foster II ladder). The impedance function the model and some suggested extensions. Z(s), a rational positive real function of the complex frequency s, can be used to study differences in structure at both levels II. PROTEIN STRUCTURE MODELING: REPRESENTATION, within and across proteins. This is illustrated with a set of ANALYSIS, AND SYNTHESIS sample proteins. In particular, the phase plot appears to have signature-like properties. Conversely, network synthesis The identification of secondary structure consisting of methods at the secondary (and sometimes tertiary) level alpha helices, beta sheets, and turns from the primary augmented by modifications at the tertiary level lead to amino acid sequence of a protein is now fairly routine [7]. artificial protein structures which may or may not exist, but In contrast, the relationship between tertiary structure and when a circuit can represent a protein it may be used to function is less well understood [5, 6]. There are broadly construct an approximate tertiary shape. The circuit may also three approaches. Analytical methods attempt to derive be used as a schematic diagram, for tracking changes during tertiary structure from the primary sequence. Many of them folding, and in searching for patterns among proteins. use some form of minimization of an energy function based", "keywords": [], "document_vector": [-115.421363, 46.239116], "paragraphs": [{"paragraph_vector": [-130.502166, 20.795013], "paragraph_keywords": []}], "content": {}, "doi": "10.1147/rd.351.0108"}, {"uri": "53", "title": "Molli: Interactive Visualization for Exploratory Protein Analysis", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sara Su", "Connor Gramazio", "Caitlin Crumm", "Lenore J. Cowen"], "summary": "Many programs have been designed to view the 3D structures of protein molecules in 2D. However, certain types of domain-specific information haven\u2019t yet been categorized systematically in a way that highlights the interface design challenges. In particular, sequence, structure, and homology are heterogeneous types of linked information that a scientist needs in working memory to manipulate and understand a protein structure. Existing protein visualization software, such as Jmol (www. jmol.org), PyMOL (www.pymol. org), and Swiss-PdbViewer, capably provide access to the three data facets individually. But they currently offer no easy way for users to coordinate operations between two or more facets or to link the views of homologous 3D models. Furthermore, these popular programs require deep use of nested menus or a scripting language to perform many functions. We developed Molli, an interactive visualization program, to give biologists and nonprogrammers multiple linked views of 3D protein models, along with point-and-click tools for manipulating them. A comparison of the user experience with Molli and Jmol in typical exploratory tasks showed Molli\u2019s utility.", "keywords": ["alignment", "data", "molli", "structure", "application", "selection", "console", "protein", "sequence", "biologist", "research", "system", "computer", "jmol", "biology", "interface", "trial", "example", ".", "study", "performance", "atom", "software", "science", "view", "pymol", "time", "participant", "user", "rendering", "visualization", "residue", "task"], "document_vector": [-168.546249, 3.98201], "paragraphs": [{"paragraph_vector": [141.312011, -15.839981], "paragraph_keywords": ["molli", "interface", "views", "protein"]}, {"paragraph_vector": [106.628746, -34.544368], "paragraph_keywords": ["protein", "structure", "backbone", "atoms"]}, {"paragraph_vector": [178.642974, -68.341468], "paragraph_keywords": ["protein", "atoms", "structure", "corresponding"]}, {"paragraph_vector": [162.263702, -12.74582], "paragraph_keywords": ["trials", "biologists", "protein", "example"]}, {"paragraph_vector": [161.247131, -28.05521], "paragraph_keywords": ["software", "participants", "jmol", "molli"]}, {"paragraph_vector": [162.613204, -20.926885], "paragraph_keywords": ["molli", "trials", "user", "participants"]}, {"paragraph_vector": [151.706588, -10.502504], "paragraph_keywords": ["molli", "ptc", "performance", "trials"]}, {"paragraph_vector": [161.294036, -17.229249], "paragraph_keywords": ["molli", "jmol", "error", "methods"]}, {"paragraph_vector": [160.717437, -37.081851], "paragraph_keywords": ["molli", "protein", "biologists", "computer"]}, {"paragraph_vector": [156.588027, -19.899343], "paragraph_keywords": [".", "protein", "visualization", "vol"]}, {"paragraph_vector": [147.242797, -22.085266], "paragraph_keywords": ["computer", "science", "university", "visualization"]}], "content": {}, "doi": "10.1109/VISUAL.1995.480793"}, {"uri": "54", "title": "Insights into Alzheimer's Disease: Molecular Dynamics (MD) Simulations of Peptide-Membrane Interactions", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Anne Brown"], "summary": "High performance computing (HPC) is essential in order to perform detailed MD simulations that allow for mechanistic insight into many diseases. Alzheimer\u2019s disease (AD) pathology is primarily caused by the A\u03b2(42) peptide. MD simulations of a toxic (human A\u03b2(42) ) and non-toxic (rat A\u03b2(42) ) were performed in physiologically-relevant membrane environment conditions (310 K, 150 mM NaCl, pH 7) to assess differences in toxicity to membranes relevant to AD. The simulation results were processed with open tools and the results rendered with a raytracer. This paper describes the background science and the visualization techniques used to produce the accompanying video and compare the mechanisms of amyloid activity. Keywords\u2014Molecular Dynamics, Simulation, Alzheimer\u2019s Disease, 3D Visualization Concepts: \u2022 Human-Centered computing ~ Visualization; Scientific Visualization; I. BIOCHEMICAL BACKGROUND Alzheimer\u2019s disease (AD) is a neurodegenerative disease frequently categorized by memory impairment and dementia [1]. The National Institute on Aging (NIA) estimates that 5.2 million individuals in the United States currently have AD, and that number is expected to exponentially increase in the next twenty years, providing a unique challenge for researchers and caregivers [1]. While current therapies, including the often-prescribed Aricept\u00ae (donepezil, Pfizer) and Namenda\u00ae (memantine, Forest), only temporarily suppress Alzheimer\u2019s disease symptoms, there is a pressing need to understand more about the mechanism behind AD. By studying the principal toxic species in AD pathology, the amyloid beta-peptide (A\u03b2), detailed insight into the origin of disease can be determined and novel therapeutics can be developed. Figure 1: (A) Human A\u03b2 (B) Rat A\u03b2 Gaining insight into the mechanism of peptide-membrane interactions of toxic and non-toxic A\u03b2 variants can begin to elucidate principal mechanistic components that are not clearly understood. Human A\u03b2, specifically the A\u03b2(42) alloform, is toxic to neuronal cells. The rat and mouse version of A\u03b2(42) has three residue changes relative to the human form (R5G, Y10F, and H13R, with the human residue being listed first in each pair -Figure 1). These amino acid changes result in reduced toxicity, plaque formation [2], and have different cognitive effects[3] for the rat and mouse forms of A\u03b2 [2,3,4]. Knowing that some peptides cause toxicity by interacting with, and disrupting, cell membranes, we are studying the interactions of human and rat A\uf06242 with membranes to elucidate differences in membrane-protein interactions and their relationship to overall membrane stability (Figure 2). It is of great interest to understand why many organisms have homologues to human A\u03b2(42) and have toxic effects, whereas only the rat and mouse variant are essentially nontoxic. In working towards understanding this interaction, greater insight into the mechanism of AD can be gained and targets for potential therapeutics can be developed. *e-mail:npolys@vt.edu Permission to make digital or hard copies of part or all of this work for personal or classroom use is granted without fee provided that copies are not made or distributed for profit or commercial advantage and that copies bear this notice and the full citation on the first page. Copyrights for thirdparty components of this work must be honored. For all other uses, contact the Owner/Author. Copyright is held by the owner/author(s). XSEDE16, July 17-21, 2016, , ACM 978-1-4503-4755-6/16/07. http://dx.doi.org/10.1145/2949550.2952773 Studying A\u03b2 in the membrane environment experimentally is challenging. Typically, organic solvents are added to the solution to aid in peptide solubilization; however, these solvents may also affect interactions between the peptide and the membrane [5-8]. Thus, it is difficult to experimentally characterize the interactions of A\u03b2(42) with the membrane, leading to questions about the mechanism of interaction between peptide and membrane. Molecular dynamics (MD) simulations have been determined to be useful in studying peptide-membrane interactions, giving atomistic insight into mechanistic details to aid in explaining experimental results [9-12]. In order to determine how and why the toxicity is different between the human and rat A\u03b2(42), MD simulations provide a unique and detailed venue to answer these questions. Figure 2: Simulated Membrane Configurations", "keywords": ["virginia", "tech"], "document_vector": [-22.898731, 5.639275], "paragraphs": [{"paragraph_vector": [-161.466903, 3.299742], "paragraph_keywords": ["virginia", "tech", "biochemistry", "department"]}], "content": {}, "doi": "10.1109/CIBCB.2005.1594939"}, {"uri": "55", "title": "A Technique for Visualizing Dihedral Signal of Large Protein Sequences", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Quang Vinh Nguyen", "Mao Lin Huang", "Yu Qian", "Kang Zhang"], "summary": "This paper presents a clustering and visualization technique for analyzing dihedral angles of large protein sequences. The clustering is used for discovering and grouping those similar dihedrals while the visualization can display and navigate sequences of dihedral angles of several proteins as well as their clustered property. In order to visualize a very large sequence of hundred thousands of dihedral signals, we plot them on a spiral coordinate system. This spiral visualization ensures the linear distribution without distortion or interruption of a very long sequence of points. A clustering algorithm is also provided to group those dihedral signals into different clusters so that it can enhance the analysis process. Our system can also zoom to display a number of selected proteins interactively.", "keywords": ["display", "cluster", "angle", "method", "protein", "view", "group", "value", "based", "sequence", "fa\u00e7ade", "line", "point", "user", "number", "visualization", "similarity", "clustering"], "document_vector": [-151.524612, 67.200355], "paragraphs": [{"paragraph_vector": [161.708663, -54.9234], "paragraph_keywords": ["proteins", "protein", "visualization", "angles"]}, {"paragraph_vector": [-110.23056, -71.90065], "paragraph_keywords": ["proteins", "sequence", "angles", "display"]}, {"paragraph_vector": [-83.602066, -16.376342], "paragraph_keywords": ["methods", "method", "similarity", "matrix"]}, {"paragraph_vector": [-89.062721, -18.500793], "paragraph_keywords": ["groups", "group", "merging", "similarity"]}, {"paragraph_vector": [-93.68412, -58.47393], "paragraph_keywords": ["fa\u00e7ade", "view", "compression", "step"]}, {"paragraph_vector": [-105.12976, -66.407035], "paragraph_keywords": ["visualization", "proteins", "view", "points"]}, {"paragraph_vector": [-106.788993, -67.608848], "paragraph_keywords": ["point", "visualization", "provide", "line"]}, {"paragraph_vector": [-105.619537, -66.706748], "paragraph_keywords": ["point", "color", "figure", "view"]}, {"paragraph_vector": [-97.923179, -61.619705], "paragraph_keywords": ["view", "angles", "protein", "dihedral"]}], "content": {}, "doi": "10.1109/5992.998642"}, {"uri": "56", "title": "Visualizing Cellular Secretion with Single-Protein Sensitivity via Interferometric Scattering Microscopy (iSCAT)", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Andr\u00e9 Gemeinhardt", "Katharina K\u00f6nig", "Francesca Nicoli", "Mahyar Dahmardeh", "Vahid Sandoghdar"], "summary": "Proteins are involved in a large number of biological processes. Apart from the field of proteomics, where ensemble studies of the entire set of proteins present in a system are carried out, detection and analysis of single proteins has become a vibrant field of research. Particularly, the study of proteins that are secreted from cells into the extracellular space is an important topic with wide implications for basic intercellular interactions and immunology. Secretory proteins are responsible for a vast amount of cellular functions involving migration, wound healing, immunological response, or intercellular communication. Challenges in the traditional methods such as immunoassays, Western blotting, mass spectrometry, or fluorescence microscopy are the need for labeling (with immunological complements, isotopes, or fluorescent markers) and the necessary processing and separation steps that hinder high temporal resolution in dynamic studies. Today, none of the existing techniques is able to detect single proteins secreted from living cells at subsecond temporal resolution without the need for labeling. Interferometric scattering microscopy (iSCAT) [1] is a powerful technique capable of sensing individual proteins [2]. In a nutshell, the weak scattered light from a protein is amplified by mixing with a reference light beam reflected from the cover glass, We have used this method to visualize single proteins in real time, as they are secreted by Laz388 cells, an Epstein-Barr virus (EBV) transformed B lymphocyte cell line [3,4] \u2013 see Fig. 1. We identify secreted subspecies by surface functionalization and study dynamics in cellular secretion as a reaction to external stimulation. iSCAT can also be readily combined with other optical techniques such as fluorescence and super-resolution microscopy. We discuss current work in progress and present an outlook for high-throughput analysis of cellular secretion with high temporal and mass resolution.", "keywords": ["protein", "secreted"], "document_vector": [90.829971, 35.396701], "paragraphs": [{"paragraph_vector": [-39.432754, 3.805088], "paragraph_keywords": ["proteins", "secreted", "cells", "microscopy"]}], "content": {}, "doi": "10.1109/TVCG.2011.259"}, {"uri": "57", "title": "Implicit Sphere Shadow Maps", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Michael Krone", "Guido Reina", "Sebastian Zahn", "Tina Tremel", "Carsten Bahnm\u00fcller", "Thomas Ertl"], "summary": "Particle data are commonly visualized by rendering a sphere for each particle. Since interactive rendering usually relies on fast local lighting, the spatial arrangement of the spheres is often very hard to perceive. That is, larger functional structures formed by the particles are not easily recognizable. Using global effects such as ambient occlusion or shadows adds important depth cues. In this work, we present Implicit Sphere Shadow Maps (ISSM), an application-tailored approach for large, dynamic particle data sets. This approach can be combined with state-of-the-art object-space ambient occlusion to further emphasize the spatial structure of molecules. We compare our technique against state-of-the-art methods for interactive rendering with respect to image quality and performance.", "keywords": ["issm", "fragment", "texel", "data", "distance", "rendered", "set", "method", "particle", "approach", "depth", "artifact", "shadow", "ray", "sphere", "resolution", "map", "size", "t", "rendering"], "document_vector": [-26.157108, -73.897216], "paragraphs": [{"paragraph_vector": [88.790969, 46.042217], "paragraph_keywords": ["data", "shadows", "particle", "rendering"]}, {"paragraph_vector": [-8.882129, 72.713241], "paragraph_keywords": ["shadow", "depth", "shadows", "source"]}, {"paragraph_vector": [-18.187948, 76.374008], "paragraph_keywords": ["shadow", "map", "shadows", "fragment"]}, {"paragraph_vector": [15.913692, 54.095699], "paragraph_keywords": ["sphere", "texture", "spheres", "camera"]}, {"paragraph_vector": [-14.87766, 73.231407], "paragraph_keywords": ["shadow", "sphere", "map", "texel"]}, {"paragraph_vector": [-34.136844, 77.234199], "paragraph_keywords": ["shadow", "shadows", "distance", "surface"]}, {"paragraph_vector": [-15.904369, 77.808944], "paragraph_keywords": ["t", "fig", "shown", "shadows"]}, {"paragraph_vector": [4.239035, 78.315887], "paragraph_keywords": ["shadow", "map", "patch", "hrts"]}, {"paragraph_vector": [-34.280536, 81.341781], "paragraph_keywords": ["data", "issm", "ao", "performance"]}, {"paragraph_vector": [-29.452415, 80.569015], "paragraph_keywords": ["shadow", "texel", "work", "approach"]}], "content": {}, "doi": "10.1109/IEMBS.1999.804360"}, {"uri": "58", "title": "TexMol: Interactive Visual Exploration of Large Flexible Multi-component Molecular Complexes", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Chandrajit Bajaj", "Peter Djeu", "Vinay Siddavanahalli"], "summary": "While molecular visualization software has advanced over the years, today, most tools still operate on individual molecular structures with limited facility to manipulate large multi-component complexes. We approach this problem by extending 3D imagebased rendering via programmable graphics units, resulting in an order of magnitude speedup over traditional triangle-based rendering. By incorporating a biochemically sensitive level-of-detail hierarchy into our molecular representation, we communicate appropriate volume occupancy and shape while dramatically reducing the visual clutter that normally inhibits higher-level spatial comprehension. Our hierarchical, image based rendering also allows dynamically computed physical properties data (e.g. electrostatics potential) to be mapped onto the molecular surface, tying molecular structure to molecular function. Finally, we present another approach to interactive molecular exploration using volumetric and structural rendering in tandem to discover molecular properties that neither rendering mode alone could reveal. These visualization techniques are realized in a high-performance, interactive molecular exploration tool we call TexMol, short for Texture Molecular viewer. CR Categories: I.3.4 [Computer Graphics]: Graphics Utilities\u2014 Application packages; I.3.5 [Computer Graphics]: Computational Geometry and Object Modeling\u2014Hierarchy and geometric transformations", "keywords": ["opacity", "structure", "end", "hierarchy", "molecule", "program", "helix", "represented", "render", "set", "algorithm", "volumetric", "primitive", "andrew", "texmol", "figure", "depth", "texture", "resolution", "based", "lod", "surface", "level", "performance", "atom", "graphic", "cylinder", "model", "gpu", "radius", "vertex", "image", "rendering", "visualization"], "document_vector": [96.468826, -81.606819], "paragraphs": [{"paragraph_vector": [35.470603, -50.426261], "paragraph_keywords": ["hierarchy", "atom", "structure", "level"]}, {"paragraph_vector": [72.597381, -5.149885], "paragraph_keywords": ["rendering", "lod", "surface", "depth"]}, {"paragraph_vector": [33.709285, 56.204574], "paragraph_keywords": ["rendering", "based", "cylinder", "molecule"]}, {"paragraph_vector": [25.023302, 59.403633], "paragraph_keywords": ["cylinder", "algorithm", "program", "bases"]}, {"paragraph_vector": [34.30725, 55.206588], "paragraph_keywords": ["program", "end", "f", "helix"]}, {"paragraph_vector": [67.060363, 10.820484], "paragraph_keywords": ["helix", "rendering", "function", "molecule"]}, {"paragraph_vector": [110.525711, 30.020681], "paragraph_keywords": ["lod", "data", "memory", "view"]}, {"paragraph_vector": [93.574356, 39.047264], "paragraph_keywords": ["lod", "vertex", "atom", "rendering"]}, {"paragraph_vector": [108.749427, 36.632583], "paragraph_keywords": ["rendering", "texmol", "figure", "rate"]}, {"paragraph_vector": [78.813896, -10.754238], "paragraph_keywords": ["baker", "based", "sept", "amitabh"]}, {"paragraph_vector": [98.995025, 27.809419], "paragraph_keywords": ["rendering", "andrew", "graphics", "conference"]}, {"paragraph_vector": [47.061576, -9.584839], "paragraph_keywords": ["surfaces", "pages", "graphics", "computer"]}], "content": {}, "doi": "10.1109/VARMS.2015.7151723"}, {"uri": "59", "title": "Using videogames to improve molecular graphics tools", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Zac Fitz-Walter", "Se\u00e1n I. O\u2019Donoghue", "Benedetta Frida Baldi"], "summary": "Well-designed videogames provide intuitive and engaging ways of understanding and interacting with highly complex systems. The aim of this study was to explore the use of videogames as a lens for the design of bioinformatics visualisation tools, with a particular focus on molecular graphics systems designed to explore 3D structures of proteins. We conducted a workshop bringing together experts in game design, molecular biology, data visualisation, and software development, to explore how videogame expertise could inform the design of the protein visualisation tool, Aquaria. Results of the workshop suggest that games could influence the design of tutorials for new users, the nature of the interaction within a 3D space, and act as a mechanism for engaging users in crowdsourced tasks. Author", "keywords": ["expert", "game", "use", "workshop", "videogames", "gamification", "visualisation", "protein", "utility", "work", "participant", "tool", "aquarium", "user", "discussion", "design", "highlighted"], "document_vector": [-168.38536, 12.131184], "paragraphs": [{"paragraph_vector": [152.978851, -6.196967], "paragraph_keywords": ["design", "videogames", "use", "visualisation"]}, {"paragraph_vector": [153.956512, -25.860614], "paragraph_keywords": ["aquaria", "copies", "workshop", "tool"]}, {"paragraph_vector": [158.664535, -3.448005], "paragraph_keywords": ["aquaria", "game", "tutorials", "highlighted"]}, {"paragraph_vector": [151.931304, -5.359435], "paragraph_keywords": ["aquaria", "users", "crowdsourcing", "gamification"]}, {"paragraph_vector": [155.157684, -2.905769], "paragraph_keywords": ["aquaria", "design", "supported", "experts"]}], "content": {}, "doi": "10.1109/BioMedCom.2012.12"}, {"uri": "60", "title": "ENCORE: A Visualization Tool for Insight into Circadian Omics", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Hannah De los Santos", "Kristin P. Bennett", "Jennifer M. Hurley"], "summary": "Circadian rhythms are 24-hour biological cycles that control daily molecular rhythms in many organisms. The cellular elements that fall under the regulation of the clock are often studied through the use of omics-scale data sets gathered over time to determine how circadian regulation impacts cellular physiology. Previously, we created the ECHO (Extended Circadian Harmonic Oscillator) tool to identify rhythms in these data sets. Using ECHO, we found that circadian oscillations widely undergo a change in amplitude over time and that these amplitude changes have a biological function in the cell. However, ECHO does not align gene ontologies with the identified oscillating genes to give functional context. Thus, we created ENCORE (ECHO Native Circadian Ontological Rhythmicity Explorer), a novel visualization tool which combines the disparate databases of Gene Ontologies, protein-protein interactions, and auxiliary information to uncover the meaning of circadianly-regulated genes. This freely-available tool performs automatic enrichment and creates publication-worthy visualizations which we used to extend previously-gathered data on circadian regulation of physiology from published omics-scale studies in three circadian model organisms: mouse, fruit fly, and Neurospora crassa.", "keywords": ["p", "data", "group", "value", "category", "analysis", "explorer", "neurospora", "gene", "expression", "interaction", "set", "including", "protein", "regulation", "echo", "information", "understanding", "figure", "chord", "campfire", "work", "clock", "circadian", "term", "ac", "encore", "ontology", "enrichment", "user", "visualization", "rhythm"], "document_vector": [2.032552, 68.680976], "paragraphs": [{"paragraph_vector": [-139.108078, -10.405241], "paragraph_keywords": ["clock", "rhythms", "echo", "regulation"]}, {"paragraph_vector": [-135.503387, -9.174441], "paragraph_keywords": ["impact", "tools", "terms", "gene"]}, {"paragraph_vector": [-137.032669, -9.353399], "paragraph_keywords": ["encore", "echo", "ac", "genes"]}, {"paragraph_vector": [-136.220535, -11.300195], "paragraph_keywords": ["terms", "r", "calculated", "gene"]}, {"paragraph_vector": [-135.399124, -9.80915], "paragraph_keywords": ["term", "users", "ontology", "hierarchy"]}, {"paragraph_vector": [-137.154327, -10.479568], "paragraph_keywords": ["users", "protein", "click", "ac"]}, {"paragraph_vector": [-131.793228, -18.576347], "paragraph_keywords": ["gene", "information", "group", "chord"]}, {"paragraph_vector": [-135.789093, -12.464997], "paragraph_keywords": ["campfire", "ontology", "floor", "pie"]}, {"paragraph_vector": [-139.354446, -11.898022], "paragraph_keywords": ["windows", "gene", "mwshiny", "information"]}, {"paragraph_vector": [-136.737289, -8.381093], "paragraph_keywords": ["data", "hours", "expression", "p"]}, {"paragraph_vector": [-136.315002, -8.552503], "paragraph_keywords": ["genes", "p", "value", "remove"]}, {"paragraph_vector": [-137.383407, -10.088607], "paragraph_keywords": ["genes", "enrichment", "encore", "mice"]}, {"paragraph_vector": [-137.595016, -9.490986], "paragraph_keywords": ["genes", "flies", "enrichment", "found"]}, {"paragraph_vector": [-136.754898, -10.442883], "paragraph_keywords": ["enrichment", "damped", "transcripts", "genes"]}, {"paragraph_vector": [-136.991165, -12.143294], "paragraph_keywords": ["encore", "data", "genes", "term"]}, {"paragraph_vector": [-137.420989, -10.3105], "paragraph_keywords": ["encore", "genes", "day", "regulation"]}, {"paragraph_vector": [-138.220489, -8.938512], "paragraph_keywords": ["encore", "data", "techniques", "expertise"]}, {"paragraph_vector": [-156.682098, 8.541172], "paragraph_keywords": ["energy", "pnnl", "rensselaer", "supported"]}], "content": {}, "doi": "10.1109/APMRC.2006.365896"}, {"uri": "61", "title": "Comparative Visualization of Protein Structure-Sequence Alignments", "timestamp": "1998", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Marc Hansen", "Doanna Meads", "Alex Pang"], "summary": "Protein fold recognition (threading) involves the prediction of a protein\u2019s three-dimensional shape based on its similarity to a protein whose structure is known. Fold predictions are low resolution; no effort is made to rotate the protein\u2019s component amino acid side chains into their correct spatial orientations. Rather, the goal is to recognize the protein family member that most closely resembles the target sequence of unknown structure and to create a sensible alignment of the target to the structure (i.e., a structure-sequence alignment). To complement this structure prediction method we have implemented a low resolution molecular graphics tool. Since amino acid side chain orientation is not relevant in fold recognition, amino acid residues are represented by abstract shapes or glyphs much like Lego (tm) blocks. We also borrow techniques from comparative streamline visualization to provide clean depictions of the entire protein structure model. By creating a low resolution representation of protein structure, we are able to approximately double the amount of information on the screen. This implementation also possesses the advantage of eliminating distracting and possibly misleading visual clutter resulting from the mapping of protein alignment information onto a high resolution display of a known structure.", "keywords": ["alignment", "data", "structure", "building", "program", "block", "acid", "target", "modeling", "protein", "indicate", "sequence", "information", "chemical", "figure", "amino", "folding", "backbone", "tool", "display", "model", "use", "dinamo", "glyph", "substitution", "problem", "visualization", "shape"], "document_vector": [-160.290863, 11.637894], "paragraphs": [{"paragraph_vector": [142.02214, -52.068416], "paragraph_keywords": ["protein", "data", "figure", "tools"]}, {"paragraph_vector": [159.15657, -56.630271], "paragraph_keywords": ["protein", "acids", "techniques", "visualization"]}, {"paragraph_vector": [-3.091947, -72.827095], "paragraph_keywords": ["protein", "sequence", "help", "acid"]}, {"paragraph_vector": [-22.955427, -62.480644], "paragraph_keywords": ["target", "protein", "sequence", "structure"]}, {"paragraph_vector": [-70.670478, -50.207244], "paragraph_keywords": ["acid", "matrix", "protein", "structure"]}, {"paragraph_vector": [148.438644, -43.38414], "paragraph_keywords": ["modeling", "homology", "alignment", "protein"]}, {"paragraph_vector": [154.859588, -50.696098], "paragraph_keywords": ["information", "protein", "acid", "format"]}, {"paragraph_vector": [110.157135, -59.815052], "paragraph_keywords": ["acid", "information", "phenylalanine", "histidine"]}, {"paragraph_vector": [160.103744, -55.664066], "paragraph_keywords": ["protein", "structure", "acid", "figure"]}, {"paragraph_vector": [159.437225, -52.609085], "paragraph_keywords": ["alignment", "goal", "acid", "indicate"]}, {"paragraph_vector": [134.623992, -37.327362], "paragraph_keywords": ["alignment", "protein", "files", "acid"]}, {"paragraph_vector": [-131.812667, -64.206214], "paragraph_keywords": ["protein", "meads", "helped", "grant"]}], "content": {}, "doi": "10.1109/TVCG.2015.2467293"}, {"uri": "62", "title": "Articulated Planar Reformation for Change Visualization in Small Animal Imaging", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Peter Kok", "Martin Baiker", "Emile A. Hendriks", "Frits H. Post", "Jouke Dijkstra", "Clemens W.G.M. L\u00f6wik", "Boudewijn P.F. Lelieveldt", "Charl P. Botha"], "summary": "The analysis of multi-timepoint whole-body small animal CT data is greatly complicated by the varying posture of the subject at different timepoints. Due to these variations, correctly relating and comparing corresponding regions of interest is challenging. In addition, occlusion may prevent effective visualization of these regions of interest. To address these problems, we have developed a method that fully automatically maps the data to a standardized layout of sub-volumes, based on an articulated atlas registration. We have dubbed this process articulated planar reformation, or APR. A sub-volume can be interactively selected for closer inspection and can be compared with the corresponding sub-volume at the other timepoints, employing a number of different comparative visualization approaches. We provide an additional tool that highlights possibly interesting areas based on the change of bone density between timepoints. Furthermore we allow visualization of the local registration error, to give an indication of the accuracy of the registration. We have evaluated our approach on a case that exhibits cancer-induced bone resorption.", "keywords": ["body", "animal", "registration", "data", "volume", "dataset", "interest", "area", "focus", "feedback", "order", "analysis", "representation", "application", "method", "difference", "research", "approach", "imaging", "section", "figure", "case", "change", "bone", "based", "ct", "atlas", "datasets", "study", ".", "tool", "example", "evaluation", "layout", "apr", "use", "view", "error", "domain", "image", "user", "visualization", "skeleton"], "document_vector": [140.222427, 0.875405], "paragraphs": [{"paragraph_vector": [-31.026954, 43.041683], "paragraph_keywords": ["imaging", "cancer", "animal", "data"]}, {"paragraph_vector": [-28.262039, 46.332813], "paragraph_keywords": ["animal", "posture", "bone", "differences"]}, {"paragraph_vector": [-24.742456, 46.096271], "paragraph_keywords": ["body", "data", "datasets", "facilitate"]}, {"paragraph_vector": [-17.235, 48.59518], "paragraph_keywords": ["cpr", "visualization", "colon", "data"]}, {"paragraph_vector": [-14.597824, 49.865062], "paragraph_keywords": ["volume", "visualization", "skeleton", "model"]}, {"paragraph_vector": [-27.923778, 46.118209], "paragraph_keywords": ["atlas", "bone", "data", "transform"]}, {"paragraph_vector": [-27.520284, 46.847305], "paragraph_keywords": ["atlas", "views", "view", "registration"]}, {"paragraph_vector": [-28.817417, 46.608711], "paragraph_keywords": ["registered", "bones", "box", "bone"]}, {"paragraph_vector": [-23.382844, 48.41434], "paragraph_keywords": ["image", "volumes", "data", "volume"]}, {"paragraph_vector": [-25.400936, 47.013713], "paragraph_keywords": ["atlas", "layout", "segment", "shown"]}, {"paragraph_vector": [-27.086936, 46.18964], "paragraph_keywords": ["timepoints", "visualization", "method", "approach"]}, {"paragraph_vector": [-27.976375, 44.551212], "paragraph_keywords": ["bone", "change", "atlas", "values"]}, {"paragraph_vector": [-26.08279, 46.058361], "paragraph_keywords": ["threshold", "regions", "error", "registration"]}, {"paragraph_vector": [-27.039525, 46.776927], "paragraph_keywords": ["visualization", "data", "cvp", "package"]}, {"paragraph_vector": [-28.129722, 46.112594], "paragraph_keywords": ["study", "domain", "case", "animal"]}, {"paragraph_vector": [-29.348604, 44.592262], "paragraph_keywords": ["view", "-", "sub", "study"]}, {"paragraph_vector": [-26.764492, 46.742664], "paragraph_keywords": ["view", "registration", "changes", "representation"]}, {"paragraph_vector": [-28.294536, 47.24575], "paragraph_keywords": ["bone", "change", "representation", "registration"]}, {"paragraph_vector": [-28.523082, 45.539009], "paragraph_keywords": ["analysis", "error", "study", "change"]}, {"paragraph_vector": [-28.441194, 46.606456], "paragraph_keywords": ["study", "time", "approach", "existing"]}, {"paragraph_vector": [-27.894046, 47.011734], "paragraph_keywords": ["atlas", "ct", "involving", "group"]}, {"paragraph_vector": [-25.649431, 47.628582], "paragraph_keywords": ["segments", "registration", "approach", "user"]}, {"paragraph_vector": [-27.652606, 47.10886], "paragraph_keywords": ["approach", "registration", "data", "organs"]}, {"paragraph_vector": [-41.649692, 3.607319], "paragraph_keywords": ["hoehn", "willingness", "expertise", "tracking"]}], "content": {}, "doi": "10.1109/ICCABS.2011.5729867"}, {"uri": "63", "title": "Collaboration Visualization on Large Dataset for Protein-Protein Interaction Network", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Hui Li", "Chunmei Liu", "Xumin Liu"], "summary": "The accumulation of huge amount of biology data and their heterogeneity has become a bottleneck in the analysis of protein-protein interaction (PPI) networks, especially in the visualization of PPI networks. Because the format of the data generated from different experimental groups is diverse, and the databases for the storage and management of the data are different, network visualization of the heterogeneous data by integrating different derived data is challenging and is a key to comprehensively understanding the mechanism of biology system. To visualize the interactions of proteins, we first utilize the robot crawl technique to dynamically integrate the information of protein-protein interactions from all the related public databases such as Protein Interaction Database (PID), Human Protein Reference Database (HPRD) and Reactome. Second, we use a graph algorithm to partition the complex network into different sub-networks to discover the \u2018Hub\u2019 proteins in the visualization protein-protein networks. Finally, we develop a protocol for the collaboration of different researchers based on the visualization of the PPI networks. Keywords-protein-protein interactiont; network viusualization, collaboration", "keywords": ["grant", "center"], "document_vector": [21.875497, 58.358627], "paragraphs": [{"paragraph_vector": [-156.743072, 16.983997], "paragraph_keywords": ["center", "grant", "research", "science"]}], "content": {}, "doi": "10.1109/MCTE.2002.1175001"}, {"uri": "64", "title": "LMap: Shape-Preserving Local Mappings for Biomedical Visualization", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Saad Nadeem", "Xianfeng Gu"], "summary": "Visualization of medical organs and biological structures is a challenging task because of their complex geometry and the resultant occlusions. Global spherical and planar mapping techniques simplify the complex geometry and resolve the occlusions to aid in visualization. However, while resolving the occlusions these techniques do not preserve the geometric context, making them less suitable for mission-critical biomedical visualization tasks. In this paper, we present a shape-preserving local mapping technique for resolving occlusions locally while preserving the overall geometric context. More specifically, we present a novel visualization algorithm, LMap, for conformally parameterizing and deforming a selected local region-of-interest (ROI) on an arbitrary surface. The resultant shape-preserving local mappings help to visualize complex surfaces while preserving the overall geometric context. The algorithm is based on the robust and efficient extrinsic Ricci flow technique, and uses the dynamic Ricci flow algorithm to guarantee the existence of a local map for a selected ROI on an arbitrary surface. We show the effectiveness and efficacy of our method in three challenging use cases: (1) multimodal brain visualization, (2) optimal coverage of virtual colonoscopy centerline flythrough, and (3) molecular surface visualization.", "keywords": ["cavity", "data", "area", "ricci", "k", "multimodal", "space", "mesh", "angle", "length", "distortion", "algorithm", "compute", "parameterization", "fold", "information", "roi", "selected", "figure", "technique", "surface", "flow", "mapping", "j", "edge", "use", "preserving", "curvature", "flattening", "brain", "colon", "given", "lmap", "visualization", "context"], "document_vector": [138.931854, 1.683424], "paragraphs": [{"paragraph_vector": [-12.531624, 39.622951], "paragraph_keywords": ["mapping", "techniques", "surface", "input"]}, {"paragraph_vector": [10.523645, 23.690128], "paragraph_keywords": ["mesh", "algorithm", "roi", "requires"]}, {"paragraph_vector": [-11.612131, 35.139842], "paragraph_keywords": ["rois", "lmap", "use", "visualization"]}, {"paragraph_vector": [-14.45233, 39.817611], "paragraph_keywords": ["lmap", "cavities", "use", "visualization"]}, {"paragraph_vector": [-12.004879, 42.629848], "paragraph_keywords": ["colon", "flattening", "approach", "surface"]}, {"paragraph_vector": [-12.719536, 40.684432], "paragraph_keywords": ["visualization", "surface", "mapping", "context"]}, {"paragraph_vector": [-6.041405, 33.25896], "paragraph_keywords": ["curvature", "angle", "mesh", "information"]}, {"paragraph_vector": [-1.317327, 24.192338], "paragraph_keywords": ["j", "edge", "curvature", "ricci"]}, {"paragraph_vector": [-1.412632, 22.305408], "paragraph_keywords": ["edge", "flow", "ricci", "compute"]}, {"paragraph_vector": [5.729017, 19.177679], "paragraph_keywords": ["k", "algorithm", "pki", "vi"]}, {"paragraph_vector": [30.170593, 24.931585], "paragraph_keywords": ["surface", "transformations", "roi", "spin"]}, {"paragraph_vector": [-9.07366, 39.586887], "paragraph_keywords": ["brain", "area", "angle", "figure"]}, {"paragraph_vector": [-26.000465, 44.023319], "paragraph_keywords": ["data", "brain", "figure", "information"]}, {"paragraph_vector": [-15.594552, 43.264606], "paragraph_keywords": ["colon", "approaches", "surface", "polyp"]}, {"paragraph_vector": [-11.597971, 41.20436], "paragraph_keywords": ["colon", "lmap", "polyp", "preservation"]}, {"paragraph_vector": [25.855007, -4.4544], "paragraph_keywords": ["surface", "information", "area", "surfaces"]}, {"paragraph_vector": [-11.891716, 38.710109], "paragraph_keywords": ["lmap", "surface", "context", "cavities"]}, {"paragraph_vector": [-79.207031, 41.559967], "paragraph_keywords": ["neuroscience", "sbuh", "university", "center"]}], "content": {}, "doi": "10.1109/CSSE.2008.503"}, {"uri": "65", "title": "Uncertainty Visualization for Secondary Structures of Proteins", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Christoph Schulz", "Karsten Schatz", "Michael Krone", "Matthias Braun", "Thomas Ertl", "Daniel Weiskopf"], "summary": "We present a technique that conveys the uncertainty in the secondary structure of proteins\u2014an abstraction model based on atomic coordinates. While protein data inherently contains uncertainty due to the acquisition method or the simulation algorithm, we argue that it is also worth investigating uncertainty induced by analysis algorithms that precede visualization. Our technique helps researchers investigate differences between multiple secondary structure assignment methods. We modify established algorithms for fuzzy classification and introduce a discrepancy-based approach to project an ensemble of sequences to a single importance-weighted sequence. In 2D, we depict the aggregated secondary structure assignments based on the per-residue deviation in a collapsible sequence diagram. In 3D, we extend the ribbon diagram using visual variables such as transparency, wave form, frequency, or amplitude to facilitate qualitative analysis of uncertainty. We evaluated the effectiveness and acceptance of our technique through expert reviews using two example applications: the combined assignment against established algorithms and time-dependent structural changes originating from simulated protein dynamics.", "keywords": ["chain", "data", "structure", "value", "analysis", "et", "helix", "acid", "hydrogen", "confidence", "ribbon", "set", "method", "presented", "protein", "shown", "algorithm", "sequence", "dssp", "approach", "result", "diagram", "color", "trust", "change", "assignment", "based", "prosign", "geometry", "element", "stride", "e", "expert", "use", "model", "uncertainty", "time", "al", "participant", "variable", "image", "visualization"], "document_vector": [-139.215377, 31.054746], "paragraphs": [{"paragraph_vector": [-157.253143, -61.641471], "paragraph_keywords": ["structure", "visualization", "analysis", "assignment"]}, {"paragraph_vector": [-147.680313, -58.762176], "paragraph_keywords": ["uncertainty", "acid", "protein", "proteins"]}, {"paragraph_vector": [-95.492279, -60.654651], "paragraph_keywords": ["structure", "chain", "\u03b2", "assignment"]}, {"paragraph_vector": [-113.243782, -62.991352], "paragraph_keywords": ["structure", "helices", "dssp", "atoms"]}, {"paragraph_vector": [-123.39627, -62.506355], "paragraph_keywords": ["structure", "assignment", "results", "dssp"]}, {"paragraph_vector": [74.084228, -7.178166], "paragraph_keywords": ["ribbon", "ribbons", "presented", "gpu"]}, {"paragraph_vector": [-153.222198, -59.707981], "paragraph_keywords": ["uncertainty", "visualization", "variables", "al"]}, {"paragraph_vector": [-166.180007, -62.470058], "paragraph_keywords": ["uncertainty", "visualization", "atoms", "uncertainties"]}, {"paragraph_vector": [-140.932861, -58.758502], "paragraph_keywords": ["uncertainty", "model", "methods", "acid"]}, {"paragraph_vector": [-88.25669, -48.123073], "paragraph_keywords": ["classification", "ai", "algorithms", "e"]}, {"paragraph_vector": [-86.732177, -42.217338], "paragraph_keywords": ["discrepancy", "structure", "methods", "probabilities"]}, {"paragraph_vector": [-152.151687, -58.604835], "paragraph_keywords": ["structure", "uncertainty", "protein", "elements"]}, {"paragraph_vector": [-139.753326, -64.762077], "paragraph_keywords": ["structure", "shown", "elements", "row"]}, {"paragraph_vector": [70.666007, 4.237323], "paragraph_keywords": ["acid", "structure", "color", "element"]}, {"paragraph_vector": [-166.363479, -62.018444], "paragraph_keywords": ["ribbon", "uncertainty", "geometry", "structure"]}, {"paragraph_vector": [67.862281, 2.267385], "paragraph_keywords": ["structure", "value", "image", "distortion"]}, {"paragraph_vector": [-144.79039, -61.224025], "paragraph_keywords": ["uncertainty", "structure", "diagram", "assignments"]}, {"paragraph_vector": [-134.481719, -62.185039], "paragraph_keywords": ["helix", "prosign", "example", "assigns"]}, {"paragraph_vector": [-146.166152, -59.159938], "paragraph_keywords": ["uncertainty", "protein", "structure", "visualization"]}, {"paragraph_vector": [-152.849365, -58.748706], "paragraph_keywords": ["questions", "uncertainty", "visualization", "participants"]}, {"paragraph_vector": [-148.065109, -59.416805], "paragraph_keywords": ["visualization", "uncertainty", "structure", "flexibility"]}, {"paragraph_vector": [-155.172592, -59.429965], "paragraph_keywords": ["diagram", "ribbon", "e", "sequence"]}, {"paragraph_vector": [-151.181793, -60.309829], "paragraph_keywords": ["e", "experts", "image", "animation"]}, {"paragraph_vector": [-151.508255, -58.283428], "paragraph_keywords": ["visualization", "information", "uncertainty", "rated"]}, {"paragraph_vector": [-147.148803, -60.824527], "paragraph_keywords": ["uncertainty", "structure", "visualization", "structures"]}], "content": {}, "doi": "10.1109/ICBBE.2009.5163751"}, {"uri": "66", "title": "Fast Blending Scheme for Molecular Surface Representation", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Julius Parulek"], "summary": "Representation of molecular surfaces is a well established way to study the interaction of molecules. The state-of-theart molecular representation is the SES model, which provides a detailed surface visualization. Nevertheless, it is computationally expensive, so the less accurate Gaussian model is traditionally preferred. We introduce a novel surface representation that resembles the SES and approaches the rendering performance of the Gaussian model. Our technique is based on the iterative blending of implicit functions and avoids any pre-computation. Additionally, we propose a GPU-based ray-casting algorithm that efficiently visualize our molecular representation. A qualitative and quantitative comparison of our model with respect to the Gaussian and SES models is presented. As showcased in the paper, our technique is a valid and appealing alternative to the Gaussian representation. This is especially relevant in all the applications where the cost of the SES is prohibitive.", "keywords": ["buffer", "called", "evaluated", "structure", "area", "md", "allows", "order", "et", "distance", "representation", "space", "set", "dynamic", "method", "gradient", "l", "equation", "derivative", "algorithm", "gaussian", "point", "approach", "object", "proposed", "ray", "f", "technique", "surface", "based", "computed", "performance", "atom", "-", "s", "solvent", "gpu", "model", "r", "radius", "time", "given", "al", "g", "function", "rendering", "visualization", "casting", "according"], "document_vector": [-14.424006, -85.955795], "paragraphs": [{"paragraph_vector": [87.994812, -25.713914], "paragraph_keywords": ["time", "datasets", "interactions", "thousands"]}, {"paragraph_vector": [48.54412, 15.848265], "paragraph_keywords": ["representation", "ses", "model", "solvent"]}, {"paragraph_vector": [38.468555, 39.075744], "paragraph_keywords": ["surface", "solvent", "radius", "representation"]}, {"paragraph_vector": [33.700977, 32.379955], "paragraph_keywords": ["surface", "method", "algorithm", "ses"]}, {"paragraph_vector": [30.567876, 22.707237], "paragraph_keywords": ["objects", "surface", "space", "function"]}, {"paragraph_vector": [30.096906, 41.855987], "paragraph_keywords": ["ray", "casting", "models", "surfaces"]}, {"paragraph_vector": [31.712697, 43.101913], "paragraph_keywords": ["function", "surface", "m", "point"]}, {"paragraph_vector": [24.848548, 15.226588], "paragraph_keywords": ["function", "l", "g", "hgi"]}, {"paragraph_vector": [23.0471, 44.960395], "paragraph_keywords": ["ray", "function", "atoms", "value"]}, {"paragraph_vector": [21.339242, 19.421741], "paragraph_keywords": ["function", "g", "radius", "value"]}, {"paragraph_vector": [17.461812, 20.669837], "paragraph_keywords": ["f", "equation", "l", "fig"]}, {"paragraph_vector": [18.087429, 19.135311], "paragraph_keywords": ["function", "figure", "equation", "atoms"]}, {"paragraph_vector": [24.83765, 13.135497], "paragraph_keywords": ["function", "iteration", "atom", "gi"]}, {"paragraph_vector": [19.804317, 20.788661], "paragraph_keywords": ["l", "r", "computed", "derivatives"]}, {"paragraph_vector": [42.711189, 15.391434], "paragraph_keywords": ["l", "derivatives", "f", "function"]}, {"paragraph_vector": [54.678462, 33.834896], "paragraph_keywords": ["approximation", "equation", "f", "derivatives"]}, {"paragraph_vector": [21.086479, 50.042526], "paragraph_keywords": ["ray", "sphere", "step", "pixel"]}, {"paragraph_vector": [77.860374, 41.248512], "paragraph_keywords": ["surface", "space", "point", "datasets"]}, {"paragraph_vector": [99.205032, -5.457332], "paragraph_keywords": ["md", "atoms", "simulation", "radius"]}, {"paragraph_vector": [67.728782, 15.00365], "paragraph_keywords": ["model", "gpu", "achieve", "surface"]}, {"paragraph_vector": [81.763595, -28.587297], "paragraph_keywords": ["model", "buffer", "performance", "note"]}, {"paragraph_vector": [40.577335, 22.76991], "paragraph_keywords": ["ses", "model", "surface", "resulting"]}, {"paragraph_vector": [61.295284, 20.957395], "paragraph_keywords": ["function", "probe", "surface", "clipping"]}, {"paragraph_vector": [52.967693, 25.759992], "paragraph_keywords": ["visualization", "surfaces", "pages", "thank"]}, {"paragraph_vector": [104.967704, 41.125854], "paragraph_keywords": ["pages", "aire", "rendering", "eurographics"]}], "content": {}, "doi": "10.1109/ICAUMS.2016.8479979"}, {"uri": "67", "title": "Seamless Visual Abstraction of Molecular Surfaces", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Julius Parulek", "Timo Ropinski", "Ivan Viola"], "summary": "Molecular visualization is often challenged with rendering of large sequences of molecular simulations in real time. We introduce a novel approach that enables us to show even large protein complexes over time in real-time. Our method is based on the level-ofdetail concept, where we exploit three different molecular surface models, solvent excluded surface (SES), Gaussian kernels and van der Waals spheres combined in one visualization. We introduce three shading levels that correspond to their geometric counterparts and a method for creating seamless transition between these representations. The SES representation with full shading and added contours stands in focus while on the other side a sphere representation with constant shading and without contours provide the context. Moreover, we introduce a methodology to render the entire molecule directly using the A-buffer technique, which further improves the performance. The rendering performance is evaluated on series of molecules of varying atom counts. CR Categories: J.3 [Computer Applications]: Life and Medical Sciences\u2014Biology and Genetics I.3.3 [COMPUTER GRAPHICS]: Picture/Image Generation\u2014Viewing algorithms", "keywords": ["data", "transition", "area", "allows", "molecule", "et", "representation", "detail", "space", "dynamic", "method", "shading", "information", "point", "approach", "occlusion", "object", "depth", "fig", "defined", "ray", "sphere", "technique", "based", "surface", "level", "performance", "atom", "s", "field", "gpu", "model", "represents", "abstraction", "time", "al", "function", "rendering", "visualization", "context"], "document_vector": [-104.877281, -69.247291], "paragraphs": [{"paragraph_vector": [74.062568, -14.540058], "paragraph_keywords": ["visualization", "solvent", "site", "dynamics"]}, {"paragraph_vector": [89.20623, -5.748654], "paragraph_keywords": ["object", "representation", "viewer", "constancy"]}, {"paragraph_vector": [67.055923, 13.950242], "paragraph_keywords": ["level", "representations", "surface", "detail"]}, {"paragraph_vector": [68.717086, 35.080352], "paragraph_keywords": ["representations", "rendering", "order", "molecule"]}, {"paragraph_vector": [73.608413, 39.128089], "paragraph_keywords": ["based", "rendering", "atoms", "et"]}, {"paragraph_vector": [70.84362, 29.813125], "paragraph_keywords": ["data", "abstraction", "representation", "rendering"]}, {"paragraph_vector": [67.067901, 25.765729], "paragraph_keywords": ["shading", "rendering", "abstraction", "levels"]}, {"paragraph_vector": [41.056434, 23.160583], "paragraph_keywords": ["transition", "t", "function", "defines"]}, {"paragraph_vector": [39.9001, 23.921188], "paragraph_keywords": ["ses", "method", "representation", "surface"]}, {"paragraph_vector": [24.106039, 48.424179], "paragraph_keywords": ["spheres", "function", "represents", "atom"]}, {"paragraph_vector": [43.371879, 28.817775], "paragraph_keywords": ["shading", "interpolation", "model", "cues"]}, {"paragraph_vector": [32.992065, 49.041996], "paragraph_keywords": ["curvature", "area", "defined", "contours"]}, {"paragraph_vector": [37.964317, 52.253391], "paragraph_keywords": ["spheres", "atom", "buffer", "increased"]}, {"paragraph_vector": [25.837049, 52.039783], "paragraph_keywords": ["ray", "pixel", "step", "surface"]}, {"paragraph_vector": [45.738231, 29.466629], "paragraph_keywords": ["fields", "lengths", "based", "performance"]}, {"paragraph_vector": [77.914688, -13.271649], "paragraph_keywords": ["atoms", "contour", "format", "model"]}, {"paragraph_vector": [68.799217, 7.045551], "paragraph_keywords": ["function", "importance", "cavity", "representation"]}, {"paragraph_vector": [83.072097, 10.51223], "paragraph_keywords": ["surface", "shading", "model", "detail"]}, {"paragraph_vector": [78.535484, -10.888123], "paragraph_keywords": ["visualization", "research", "dynamics", "ieee"]}, {"paragraph_vector": [77.985916, 33.353927], "paragraph_keywords": ["visualization", "computer", "graphics", "time"]}], "content": {}, "doi": "10.1109/ICIG.2004.36"}, {"uri": "68", "title": "Modeling in the Time of COVID-19: Statistical and Rule-based Mesoscale Models", "timestamp": "2020", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ngan Nguyen", "Ond\u0159ej Strnad", "Tobias Klein", "Deng Luo", "Ruwayda Alharbi", "Peter Wonka", "Martina Maritan", "Peter Mindek", "Ludovic Autin", "David S. Goodsell", "Ivan Viola"], "summary": "We present a new technique for the rapid modeling and construction of scientifically accurate mesoscale biological models. The resulting 3D models are based on a few 2D microscopy scans and the latest knowledge available about the biological entity, represented as a set of geometric relationships. Our new visual-programming technique is based on statistical and rule-based modeling approaches that are rapid to author, fast to construct, and easy to revise. From a few 2D microscopy scans, we determine the statistical properties of various structural aspects, such as the outer membrane shape, the spatial properties, and the distribution characteristics of the macromolecular elements on the membrane. This information is utilized in the construction of the 3D model. Once all the imaging evidence is incorporated into the model, additional information can be incorporated by interactively defining the rules that spatially characterize the rest of the biological entity, such as mutual interactions among macromolecules, and their distances and orientations relative to other structures. These rules are defined through an intuitive 3D interactive visualization as a visual-programming feedback loop. We demonstrate the applicability of our approach on a use case of the modeling procedure of the SARS-CoV-2 virion ultrastructure. This atomistic model, which we present here, can steer biological research to new promising directions in our efforts to fight the spread of the virus.", "keywords": ["data", "structure", "virus", "membrane", "shape", "distance", "rule", "virion", "parent", "create", "set", "mesh", "presented", "protein", "input", "algorithm", "applied", "sars", "information", "placed", "point", "approach", "number", "specification", "research", "mesoscale", "system", "object", "figure", "distribution", "defined", "process", "example", "based", "surface", "level", "geometry", "element", "assembly", "rna", "-", "size", "specified", "model", "use", "form", "contour", "instance", "generated", "created", "image", "user", "modeling", "skeleton"], "document_vector": [-159.219207, -39.550926], "paragraphs": [{"paragraph_vector": [148.923233, -48.912609], "paragraph_keywords": ["cells", "organisms", "biomolecules", "form"]}, {"paragraph_vector": [56.302982, -15.395034], "paragraph_keywords": ["modeling", "structure", "models", "data"]}, {"paragraph_vector": [48.177436, 14.779251], "paragraph_keywords": ["models", "modeling", "developed", "information"]}, {"paragraph_vector": [73.499702, 2.011144], "paragraph_keywords": ["modeling", "model", "presented", "user"]}, {"paragraph_vector": [97.492622, 16.165479], "paragraph_keywords": ["modeling", "system", "input", "defined"]}, {"paragraph_vector": [-169.261779, -33.115345], "paragraph_keywords": ["model", "modeling", "requires", "assembly"]}, {"paragraph_vector": [95.53688, 13.378917], "paragraph_keywords": ["rules", "modeling", "models", "rule"]}, {"paragraph_vector": [37.579147, -1.849555], "paragraph_keywords": ["membrane", "contours", "images", "information"]}, {"paragraph_vector": [46.144481, 5.317989], "paragraph_keywords": ["surface", "proteins", "contour", "band"]}, {"paragraph_vector": [36.351978, -4.850476], "paragraph_keywords": ["proteins", "contours", "membrane", "ellipse"]}, {"paragraph_vector": [10.699546, 17.604404], "paragraph_keywords": ["contours", "contour", "angle", "points"]}, {"paragraph_vector": [29.897403, 6.214828], "paragraph_keywords": ["contour", "image", "object", "distribution"]}, {"paragraph_vector": [29.659927, 6.625057], "paragraph_keywords": ["mesh", "membrane", "distribution", "contour"]}, {"paragraph_vector": [41.192394, -12.833731], "paragraph_keywords": ["elements", "element", "geometry", "position"]}, {"paragraph_vector": [97.153686, 31.798881], "paragraph_keywords": ["rules", "elements", "model", "rule"]}, {"paragraph_vector": [97.328536, 68.586944], "paragraph_keywords": ["elements", "rules", "user", "element"]}, {"paragraph_vector": [178.212326, 88.205039], "paragraph_keywords": ["parent", "rules", "element", "distance"]}, {"paragraph_vector": [125.670684, 83.822578], "paragraph_keywords": ["rule", "elements", "polygon", "element"]}, {"paragraph_vector": [-126.977996, 75.111351], "paragraph_keywords": ["element", "rule", "transformations", "rules"]}, {"paragraph_vector": [39.463195, 0.607539], "paragraph_keywords": ["viruses", "proteins", "displacement", "sars"]}, {"paragraph_vector": [53.851242, -14.895016], "paragraph_keywords": ["membrane", "model", "rna", "virus"]}, {"paragraph_vector": [32.437534, 4.446061], "paragraph_keywords": ["contour", "mesh", "instances", "protein"]}, {"paragraph_vector": [36.860439, -8.130766], "paragraph_keywords": ["protein", "rule", "skeleton", "figure"]}, {"paragraph_vector": [46.375308, -16.420547], "paragraph_keywords": ["model", "relations", "rule", "lipid"]}, {"paragraph_vector": [47.493694, -5.764172], "paragraph_keywords": ["rna", "skeleton", "rule", "line"]}, {"paragraph_vector": [50.490211, -2.829169], "paragraph_keywords": ["model", "models", "information", "modeling"]}, {"paragraph_vector": [55.722923, -17.639825], "paragraph_keywords": ["elements", "rna", "interaction", "membrane"]}, {"paragraph_vector": [58.284553, -13.829676], "paragraph_keywords": ["model", "system", "modeling", "form"]}, {"paragraph_vector": [64.730697, -10.852558], "paragraph_keywords": ["rule", "virion", "based", "specification"]}, {"paragraph_vector": [88.480735, 14.55742], "paragraph_keywords": ["kaust", "research", "ontologies", "shape"]}], "content": {}, "doi": "10.1109/INTMAG.2006.376137"}, {"uri": "69", "title": "Uncluttered Single-Image Visualization of Vascular Structures Using GPU and Integer Programming", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Joong-Ho Won", "Yongkweon Jeon", "Jarrett K. Rosenberg", "Sungroh Yoon", "Geoffrey D. Rubin", "Sandy Napel"], "summary": "Direct projection of 3D branching structures, such as networks of cables, blood vessels, or neurons onto a 2D image creates the illusion of intersecting structural parts and creates challenges for understanding and communication. We present a method for visualizing such structures, and demonstrate its utility in visualizing the abdominal aorta and its branches, whose tomographic images might be obtained by computed tomography or magnetic resonance angiography, in a single 2D stylistic image, without overlaps among branches. The visualization method, termed uncluttered single-image visualization (USIV), involves optimization of geometry. This paper proposes a novel optimization technique that utilizes an interesting connection of the optimization problem regarding USIV to the protein structure prediction problem. Adopting the integer linear programming-based formulation for the protein structure prediction problem, we tested the proposed technique using 30 visualizations produced from five patient scans with representative anatomical variants in the abdominal aortic vessel tree. The novel technique can exploit commodity-level parallelism, enabling use of general-purpose graphics processing unit (GPGPU) technology that yields a significant speedup. Comparison of the results with the other optimization technique previously reported elsewhere suggests that, in most aspects, the quality of the visualization is comparable to that of the previous one, with a significant gain in the computation time of the algorithm.", "keywords": ["university", "area", "order", "metric", "deviation", "tree", "set", "score", "method", "space", "protein", "distortion", "\u00fe", "memory", "ilp", "percent", "part", "reference", "section", "computer", "computation", "overlap", "anatomy", "backbone", "technique", "based", "library", "size", "configuration", "energy", "edge", "branch", "term", "provides", "centerline", "gpu", "use", "bounding", "scp", "precomputation", "view", "sa", "vessel", "time", "given", "gpus", "rotamers", "image", "function", "problem", "visualization", "optimization", "rotamer"], "document_vector": [144.677917, -1.971296], "paragraphs": [{"paragraph_vector": [-15.700879, 48.799236], "paragraph_keywords": ["visualization", "branches", "image", "summary"]}, {"paragraph_vector": [-5.911373, 51.464801], "paragraph_keywords": ["university", "center", "engineering", "stanford"]}, {"paragraph_vector": [-19.629316, 53.083095], "paragraph_keywords": ["scp", "section", "ilp", "method"]}, {"paragraph_vector": [-14.425836, 49.561965], "paragraph_keywords": ["vessel", "tree", "visualizing", "axis"]}, {"paragraph_vector": [-17.022771, 49.492568], "paragraph_keywords": ["eye", "flattening", "al", "surfaces"]}, {"paragraph_vector": [-15.47172, 49.29298], "paragraph_keywords": ["vessel", "segment", "bounding", "tree"]}, {"paragraph_vector": [-7.447034, 46.113071], "paragraph_keywords": ["vessel", "configuration", "overlap", "bounding"]}, {"paragraph_vector": [-14.522736, 50.546974], "paragraph_keywords": ["vessel", "\u00f0", "pairs", "sharing"]}, {"paragraph_vector": [-17.264944, 49.67073], "paragraph_keywords": ["vessel", "problem", "backbone", "protein"]}, {"paragraph_vector": [-15.254669, -67.712295], "paragraph_keywords": ["energy", "protein", "conformation", "structure"]}, {"paragraph_vector": [-10.800685, -51.226219], "paragraph_keywords": ["score", "energy", "function", "configurations"]}, {"paragraph_vector": [-13.986307, -63.256103], "paragraph_keywords": ["vertex", "rotamer", "set", "sampling"]}, {"paragraph_vector": [-90.643539, 24.626276], "paragraph_keywords": ["ilp", "xuv", "chosen", "problem"]}, {"paragraph_vector": [-154.97969, 66.798767], "paragraph_keywords": ["threads", "memory", "energy", "rotamer"]}, {"paragraph_vector": [-30.957706, 65.048202], "paragraph_keywords": ["energy", "rotamers", "rotamer", "cta"]}, {"paragraph_vector": [-21.806894, 55.567646], "paragraph_keywords": ["branch", "proportion", "bj", "bi"]}, {"paragraph_vector": [-94.88655, 23.882761], "paragraph_keywords": ["method", "based", "metrics", "branch"]}, {"paragraph_vector": [-92.491676, 49.812824], "paragraph_keywords": ["time", "library", "computation", "based"]}, {"paragraph_vector": [-93.501106, 55.636127], "paragraph_keywords": ["percent", "time", "precomputation", "parallelization"]}, {"paragraph_vector": [-18.861576, 51.678943], "paragraph_keywords": ["method", "distortion", "results", "size"]}, {"paragraph_vector": [-59.533191, 64.962211], "paragraph_keywords": ["k", "problem", "edge", "cplex"]}, {"paragraph_vector": [-39.24184, 69.495208], "paragraph_keywords": ["overlap", "gpus", "trees", "rate"]}, {"paragraph_vector": [-16.966615, 50.369689], "paragraph_keywords": ["method", "visualization", "vessel", "use"]}, {"paragraph_vector": [141.993667, -32.039947], "paragraph_keywords": ["author", "grant", "nih", "phd"]}], "content": {}, "doi": "10.1109/38.135911"}, {"uri": "70", "title": "Co-Coot : a Real-Time Collaborative Tool for Bio-Molecular Modeling and Visualization", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Jun Lee", "Lin-Woo Kang", "HyungSeok Kim", "Jee-In Kim"], "summary": "Research collaboration to model and analyze a complex 3D bio-molecular structure is often limited because of different time zones and locations of collaborations. A networkbased real-time software collaboration tool for molecular modeling and its visualization was developed to overcome the problem. Collaborative Crystallographic Object Oriented Toolkit (Co-Coot) is an add-on of software called Coot. Co-Coot enables multiple synchronous collaborative researchers at different locations to observe and manipulate their shared models of molecules simultaneously in realtime for better communication and collaboration. Co-Coot also provides additional collaborative features of sharing and transferring temporary working data of the model. It can also be used as an educational tool by providing with real-time feedback and demonstrations.", "keywords": ["model", "server", "coot", "data", "work", "time", "developed", "information", "shared", "user", "-", "co", "system", "researcher", "figure"], "document_vector": [-172.507705, -13.730871], "paragraphs": [{"paragraph_vector": [138.265258, 14.561942], "paragraph_keywords": ["environment", "time", "developed", "enzyme"]}, {"paragraph_vector": [140.353622, 12.310859], "paragraph_keywords": ["biocore", "coot", "collaboration", "-"]}, {"paragraph_vector": [152.449081, 19.536495], "paragraph_keywords": ["models", "shared", "user", "data"]}, {"paragraph_vector": [152.081375, 19.934068], "paragraph_keywords": ["model", "co", "pc", "coot"]}, {"paragraph_vector": [149.463638, 23.704042], "paragraph_keywords": ["system", "coot", "proposed", "-"]}, {"paragraph_vector": [151.222885, 16.410274], "paragraph_keywords": ["system", "simulation", "researchers", "time"]}, {"paragraph_vector": [-130.159057, 23.932067], "paragraph_keywords": ["coot", "access", "site"]}], "content": {}, "doi": "10.1109/CLEOE-EQEC.2017.8087193"}, {"uri": "71", "title": "Augmented Reality with Tangible Auto-Fabricated Models for Molecular Biology Applications", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Alexandre Gillet", "Michel Sanner", "Daniel Stoffler", "David Goodsell", "Arthur Olson"], "summary": "The evolving technology of computer auto-fabrication (\"3-D printing\") now makes it possible to produce physical models for complex biological molecules and assemblies. We report on an application that demonstrates the use of auto-fabricated tangible models and augmented reality for research and education in molecular biology, and for enhancing the scientific environment for collaboration and exploration. We have adapted an augmented reality system to allows virtual 3-D representations (generated by the Python Molecular Viewer) to be overlaid onto a tangible molecular model. Users can easily change the overlaid information, switching between different representations of the molecule, displays of molecular properties such as electrostatics, or dynamic information. The physical model provides a powerful, intuitive interface for manipulating the computer models, streamlining the interface between human intent, the physical model, and the computational activity. CR Categories: J.3 [Life and Medical Sciences]: Biology and Genetics. I.3.8 [Computer Graphics]: Applications.", "keywords": ["data", "structure", "representation", "protein", "augmented", "system", "computer", "object", "biology", "backbone", "interface", "work", "python", "level", "student", "pmv", "library", "marker", "display", "model", "ar", "use", "software", "form", "function", "environment", "modeling", "built", "video"], "document_vector": [129.541336, -40.104473], "paragraphs": [{"paragraph_vector": [127.44725, -15.125542], "paragraph_keywords": ["models", "computer", "structure", "data"]}, {"paragraph_vector": [111.43058, 15.635676], "paragraph_keywords": ["models", "python", "pmv", "representations"]}, {"paragraph_vector": [96.746353, 15.732138], "paragraph_keywords": ["modeling", "models", "model", "atoms"]}, {"paragraph_vector": [101.739761, 14.674933], "paragraph_keywords": ["models", "methods", "built", "process"]}, {"paragraph_vector": [113.029678, 21.26543], "paragraph_keywords": ["markers", "video", "display", "camera"]}, {"paragraph_vector": [112.949562, 13.17048], "paragraph_keywords": ["model", "markers", "object", "models"]}, {"paragraph_vector": [78.906234, 2.599223], "paragraph_keywords": ["model", "field", "sod", "backbone"]}, {"paragraph_vector": [131.50177, -0.227705], "paragraph_keywords": ["students", "models", "augmented", "structure"]}, {"paragraph_vector": [132.403823, -1.215487], "paragraph_keywords": ["approach", "develop", "system", "model"]}, {"paragraph_vector": [-155.042465, 10.588407], "paragraph_keywords": ["institute", "university", "science", "bisti"]}], "content": {}, "doi": "10.1109/ISSM51728.2020.9377518"}, {"uri": "72", "title": "Acceptability of Technology Enhanced Learning System in Chemistry Teachers", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Shaykhah S. Aldosari", "Davide Marocco"], "summary": "Designing an educational system for students is a challenging task. E-learning systems and simulations are gaining considerable attention from chemistry teachers and are becoming more popular. This paper proposes and studies a chemistry educational system that combines experiment simulations and molecular visualization for the secondary school level. In this study, we conducted individual structured interviews with chemistry teachers to gain more understanding of users regarding the system requirements. To analyze and evaluate the responses, a qualitative content analysis method was used. The paper also presents preliminary results of a testing of such system with students. We also examine the strengths and deficiencies in the prototype, and the implications this has for a system aimed at helping learners better understand chemistry subjects. Some future study directions are discussed as well. Index Terms \u2014 E-learning; simulation; molecular visualization; qualitative interviewing; data analysis.", "keywords": ["teacher", "liked", "controller", "motion", "molecule", "lab", "leap", "input", "chemical", "gesture", "result", "system", "help", "based", "student", "learning", "tool", "chemistry", "technology", "experiment", "use", "participant", "device", "user"], "document_vector": [124.481887, -21.780174], "paragraphs": [{"paragraph_vector": [174.923049, -9.840385], "paragraph_keywords": ["gesture", "based", "technology", "students"]}, {"paragraph_vector": [174.07376, -7.137321], "paragraph_keywords": ["technology", "students", "system", "interact"]}, {"paragraph_vector": [155.901962, 5.868018], "paragraph_keywords": ["system", "device", "motion", "leap"]}, {"paragraph_vector": [178.110549, -2.346868], "paragraph_keywords": ["students", "learning", "users", "chemistry"]}, {"paragraph_vector": [172.225723, -4.639744], "paragraph_keywords": ["visualization", "system", "interviews", "teachers"]}, {"paragraph_vector": [175.70726, -11.610136], "paragraph_keywords": ["system", "experiments", "systems", "interviewee"]}, {"paragraph_vector": [175.180435, -3.209017], "paragraph_keywords": ["students", "experiments", "liked", "molecules"]}, {"paragraph_vector": [176.132156, -4.000377], "paragraph_keywords": ["participants", "students", "experiments", "use"]}, {"paragraph_vector": [175.129425, -6.484307], "paragraph_keywords": ["system", "experiments", "teachers", "based"]}, {"paragraph_vector": [173.425888, -5.918528], "paragraph_keywords": ["system", "device", "study", "input"]}, {"paragraph_vector": [153.095046, 0.41074], "paragraph_keywords": ["system", "use", "participants", "controller"]}, {"paragraph_vector": [176.079345, -5.198284], "paragraph_keywords": ["system", "work", "chemistry", "controller"]}], "content": {}, "doi": "10.1109/IV.2003.1218038"}, {"uri": "73", "title": "Multi-Dimensional Visualization and Quantitation of Molecular. Cellular and Anatomical Structures", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["E.L. Buhle"], "summary": "The function of visualization is to help the scientist explore relationships in scientific data. Effective exploration requires rapid cycling between visualization and additional experiments, to the point of even visual \"steering\" the experiment. Integration of visualization with acquisition and computational facilities encourages further exploration. While specialized visualization tools have been developed for specific imaging tasks, many of the problems encountered in one imaging realm is common to other imaging realms. Thus, tools developed for medical imaging of anatomy can be generalized for use at the cellular and molecular level. Visualization tools common to the areas of molecular, cellular and anatomical imaging are explored. INTRODUCTION Visualization and quantitation of three-dimensional structural informat.ion presents recurring problems independent of the source of information. These problems center on the need for efficient capabilities to display, manipulate, and measure relevant information present in multidimensional (e.g. 3D and time) images. In the last decade, imaging packages have evolved to address the specific imaging problems at hand. While these problems are often shared by other members of the scientific cormunity in visualizing biological structure, the tools to interpret three dimensional images is hampered by incompatibilities and narrow scope in their visualization software and hardware. We have at.tempt.ed to explore some of the requirements necessary to assist in the process of understanding the complex 3D structural information arising from molecular models derived from e1ect.ror-i microscopy ( E M ) , cellular organelles sampled by confocal light microscopy, and delineation, evaluation and therapeutic planning of anatomical s truc t.ur-e s as visualized by CT and MRI imaging. It. i.3 hoped that the int.egration of these tools into an efficient and synergistic platform THO31 1 -1/90/0000/0066/$01 .OO", "keywords": [], "document_vector": [152.357086, -1.357084], "paragraphs": [{"paragraph_vector": [-128.638229, 23.258003], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/NANO.2014.6968134"}, {"uri": "74", "title": "Multi-Scale Procedural Animations of Microtubule Dynamics Based on Measured Data", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Tobias Klein", "Ivan Viola", "Eduard Gr\u00f6ller", "Peter Mindek"], "summary": "Biologists often use computer graphics to visualize structures, which due to physical limitations are not possible to image with a microscope. One example for such structures are microtubules, which are present in every eukaryotic cell. They are part of the cytoskeleton maintaining the shape of the cell and playing a key role in the cell division. In this paper, we propose a scientificallyaccurate multi-scale procedural model of microtubule dynamics as a novel application scenario for procedural animation, which can generate visualizations of their overall shape, molecular structure, as well as animations of the dynamic behaviour of their growth and disassembly. The model is spanning from tens of micrometers down to atomic resolution. All the aspects of the model are driven by scientific data. The advantage over a traditional, manual animation approach is that when the underlying data change, for instance due to new evidence, the model can be recreated immediately. The procedural animation concept is presented in its generic form, with several novel extensions, facilitating an easy translation to other domains with emergent multi-scale behavior.", "keywords": ["assembled", "data", "structure", "end", "bound", "molecule", "parameter", "way", "application", "modeling", "simulation", "growth", "create", "dynamic", "length", "protofilaments", "information", "biologist", "approach", "system", "cap", "fig", "biology", "process", "work", "animation", "interpolation", "expert", "tubulin", "model", "use", "generator", "environment", "instance", "time", "generated", "given", "microtubule", "t", "function", "growing", "visualization", "cell", "shape", "tubulins"], "document_vector": [-75.64859, -7.851828], "paragraphs": [{"paragraph_vector": [115.443, -30.473134], "paragraph_keywords": ["structures", "animators", "cell", "animation"]}, {"paragraph_vector": [104.582885, -20.776962], "paragraph_keywords": ["microtubule", "model", "evidence", "models"]}, {"paragraph_vector": [114.72676, -22.644487], "paragraph_keywords": ["model", "dynamics", "data", "processes"]}, {"paragraph_vector": [112.10221, -14.954418], "paragraph_keywords": ["approaches", "models", "animation", "modeling"]}, {"paragraph_vector": [89.71659, 7.985119], "paragraph_keywords": ["visualization", "data", "rendering", "structures"]}, {"paragraph_vector": [115.152305, -17.98583], "paragraph_keywords": ["animation", "mesoscale", "work", "visualization"]}, {"paragraph_vector": [114.982322, -22.178764], "paragraph_keywords": ["time", "simulation", "environment", "t"]}, {"paragraph_vector": [123.050514, -24.339065], "paragraph_keywords": ["animation", "interpolation", "molecules", "instance"]}, {"paragraph_vector": [120.390632, -26.171941], "paragraph_keywords": ["molecules", "instance", "t", "keyframes"]}, {"paragraph_vector": [117.02639, -22.645437], "paragraph_keywords": ["microtubules", "cell", "cells", "animations"]}, {"paragraph_vector": [97.910873, -20.841991], "paragraph_keywords": ["tubulin", "molecules", "protofilaments", "microtubule"]}, {"paragraph_vector": [98.683502, -20.434816], "paragraph_keywords": ["bound", "microtubule", "cell", "gtp"]}, {"paragraph_vector": [103.467979, -21.005083], "paragraph_keywords": ["microtubules", "model", "visualizations", "scale"]}, {"paragraph_vector": [101.627799, -20.661811], "paragraph_keywords": ["walk", "cell", "direction", "step"]}, {"paragraph_vector": [102.561302, -21.527927], "paragraph_keywords": ["microtubule", "generator", "noise", "points"]}, {"paragraph_vector": [102.123664, -19.733478], "paragraph_keywords": ["microtubule", "molecules", "tubulins", "states"]}, {"paragraph_vector": [101.056182, -21.421726], "paragraph_keywords": ["microtubule", "molecules", "cap", "position"]}, {"paragraph_vector": [102.233886, -21.051433], "paragraph_keywords": ["microtubule", "molecules", "molecule", "cap"]}, {"paragraph_vector": [101.152793, -6.521729], "paragraph_keywords": ["microtubule", "perception", "time", "molecules"]}, {"paragraph_vector": [103.062026, -21.03198], "paragraph_keywords": ["microtubule", "gtp", "bound", "expert"]}, {"paragraph_vector": [105.157569, -20.545015], "paragraph_keywords": ["expert", "application", "system", "communicate"]}, {"paragraph_vector": [167.132476, 3.181859], "paragraph_keywords": ["structures", "expert", "feedback", "animations"]}, {"paragraph_vector": [116.653121, -22.109895], "paragraph_keywords": ["animations", "approach", "dna", "animation"]}, {"paragraph_vector": [113.49945, -20.404067], "paragraph_keywords": ["animations", "approach", "model", "process"]}, {"paragraph_vector": [108.197471, -21.822425], "paragraph_keywords": ["model", "work", "research", "generated"]}], "content": {}, "doi": "10.1109/TVCG.2009.157"}, {"uri": "75", "title": "Investigating the Efficacy of Nonlinear Dimensionality Reduction Schemes in Classifying Gene and Protein Expression Studies", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["George Lee", "Carlos Rodriguez", "Anant Madabhushi"], "summary": "The recent explosion in procurement and availability of high-dimensional gene and protein expression profile data sets for cancer diagnostics has necessitated the development of sophisticated machine learning tools with which to analyze them. While some investigators are focused on identifying informative genes and proteins that play a role in specific diseases, other researchers have attempted instead to use patients based on their expression profiles to prognosticate disease status. A major limitation in the ability to accurately classify these high-dimensional data sets stems from the \u201ccurse of dimensionality,\u201d occurring in situations where the number of genes or peptides significantly exceeds the total number of patient samples. Previous attempts at dealing with this issue have mostly centered on the use of a dimensionality reduction (DR) scheme, Principal Component Analysis (PCA), to obtain a low-dimensional projection of the high-dimensional data. However, linear PCA and other linear DR methods, which rely on euclidean distances to estimate object similarity, do not account for the inherent underlying nonlinear structure associated with most biomedical data. While some researchers have begun to explore nonlinear DR methods for computer vision problems such as face detection and recognition, to the best of our knowledge, few such attempts have been made for classification and visualization of high-dimensional biomedical data. The motivation behind this work is to identify the appropriate DR methods for analysis of high-dimensional gene and protein expression studies. Toward this end, we empirically and rigorously compare three nonlinear (Isomap, Locally Linear Embedding, and Laplacian Eigenmaps) and three linear DR schemes (PCA, Linear Discriminant Analysis, and Multidimensional Scaling) with the intent of determining a reduced subspace representation in which the individual object classes are more easily discriminable. Owing to the inherent nonlinear structure of gene and protein expression studies, our claim is that the nonlinear DR methods provide a more truthful low-dimensional representation of the data compared to the linear DR schemes. Evaluation of the DR schemes was done by 1) assessing the discriminability of two supervised classifiers (Support Vector Machine and C4.5 Decision Trees) in the different lowdimensional data embeddings and 2) five cluster validity measures to evaluate the size, distance, and tightness of object aggregates in the low-dimensional space. For each of the seven evaluation measures considered, statistically significant improvement in the quality of the embeddings across 10 cancer data sets via the use of three nonlinear DR schemes over three linear DR techniques was observed. Similar trends were observed when linear and nonlinear DR was applied to the high-dimensional data following feature pruning to isolate the most informative features. Qualitative evaluation of the low-dimensional data embedding obtained via the six DR methods further suggests that the nonlinear schemes are better able to identify potential novel classes (e.g., cancer subtypes) within", "keywords": ["compared", "data", "md", "embedding", "d", "sample", "gene", "expression", "classifier", "space", "set", "method", "cluster", "accuracy", "protein", "\u00fe", "matrix", "obtained", "cancer", "\u00f0xa\u00fe", "number", "y", "dr", "lle", "xa", "measure", "study", "performance", "scheme", "use", "class", "feature", "xi", "lda", "g", "xb", "pca", "\u00f0xi\u00fe"], "document_vector": [111.760536, 81.253494], "paragraphs": [{"paragraph_vector": [-64.214492, 4.032301], "paragraph_keywords": ["gene", "protein", "expression", "samples"]}, {"paragraph_vector": [-65.344169, 3.715831], "paragraph_keywords": ["data", "space", "feature", "selection"]}, {"paragraph_vector": [-64.963829, 2.421591], "paragraph_keywords": ["data", "lda", "dr", "graph"]}, {"paragraph_vector": [-67.66439, 3.237669], "paragraph_keywords": ["data", "fig", "embedding", "space"]}, {"paragraph_vector": [-69.710624, 6.277919], "paragraph_keywords": ["dr", "methods", "data", "pca"]}, {"paragraph_vector": [-69.060775, 8.610437], "paragraph_keywords": ["data", "objects", "dr", "classes"]}, {"paragraph_vector": [-66.383865, 6.896229], "paragraph_keywords": ["xi", "data", "section", "y"]}, {"paragraph_vector": [-65.314537, -11.715758], "paragraph_keywords": ["data", "matrix", "eigenvectors", "d"]}, {"paragraph_vector": [-62.84267, -8.698506], "paragraph_keywords": ["xi", "data", "xa", "mds"]}, {"paragraph_vector": [-66.339546, -10.044772], "paragraph_keywords": ["matrix", "xi", "d", "weight"]}, {"paragraph_vector": [-67.436683, 11.05549], "paragraph_keywords": ["data", "dr", "samples", "feature"]}, {"paragraph_vector": [-72.642929, 19.277925], "paragraph_keywords": ["dr", "embedding", "xi", "classifier"]}, {"paragraph_vector": [-67.763809, 7.824008], "paragraph_keywords": ["u", "f", "data", "dr"]}, {"paragraph_vector": [-71.606956, -0.487812], "paragraph_keywords": ["training", "strj", "g", "svm"]}, {"paragraph_vector": [-70.308502, -0.597054], "paragraph_keywords": ["class", "rule", "xa", "training"]}, {"paragraph_vector": [-72.026947, 18.824058], "paragraph_keywords": ["g", "cluster", "\u00f0xi\u00fe", "xi"]}, {"paragraph_vector": [-72.149665, 18.863695], "paragraph_keywords": ["\u00fe", "cct", "measures", "embedding"]}, {"paragraph_vector": [-67.277122, 5.819059], "paragraph_keywords": ["methods", "dr", "data", "obtained"]}, {"paragraph_vector": [-75.736152, 15.677094], "paragraph_keywords": ["data", "sets", "classes", "lle"]}, {"paragraph_vector": [-73.140602, 17.520946], "paragraph_keywords": ["dr", "class", "methods", "schemes"]}, {"paragraph_vector": [-68.549713, 9.028363], "paragraph_keywords": ["dr", "methods", "data", "dimensions"]}, {"paragraph_vector": [-65.750526, 8.977943], "paragraph_keywords": ["data", "methods", "dr", "study"]}, {"paragraph_vector": [-69.421081, 4.219585], "paragraph_keywords": ["cancer", "dr", "foundation", "methods"]}], "content": {}, "doi": "10.1109/CEC.2007.4424694"}, {"uri": "76", "title": "New visualization of dynamical flexibility of N-Glycans: Umbrella Visualization in UnityMol", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Camille Besan\u00e7on", "Alexandre Guillot", "S\u00e9bastien Blaise", "Manuel Dauchez", "Nicolas Belloy", "Jessica Pr\u00e9voteau-Jonquet", "St\u00e9phanie Baud"], "summary": "N-glycosylations play an important role in protein functions and some alterations of glycosylation such as sialic-acid hydrolysis are related to protein dysfunction. In vitro study of Nglycans can be a challenging task because of the structural diversity and the many reactive groups of the glycan chains. Molecular dynamics is a useful tool and probably the only one in biology able to overcome this problem and give access to conformational informations through exhaustive sampling. To better decipher the impact of N-glycans, we have to visualize their influence over time on the protein structure. This is why we recently developed a new 2D graphical method called the Umbrella Visualization to assess, as a density graph, the protein surface covered by glycans during a molecular dynamics trajectory. Whereas this methodology brought relevant informations related to the glycans intrinsic flexibility and dynamics, we needed further developments in order to integrate an accurate description of the protein topology and its interactions. We propose here to transform this analysis method into a visualization mode in UnityMol. UnityMol is a molecular editor, viewer and prototyping platform, coded in C# with the Unity3D game engine. The new representation of glycan chains presented in this study takes into account both the main positions adopted by each antenna of the glycan and the intensity of their motions. The adapted Umbrella Visualization provides invaluable informations about the protein surface shadowed by the glycan antennas. Ultimately, the analysis of the collected data allows us to discuss both the flexibility of the glycan (=ability to explore distinct areas), and its stability (=ability to avoid spreading from its main conformational state) and offers the possibility of identifying the key elements in the protein/glycan interactions.", "keywords": ["unitymol", "glycans", "structure", "md", "order", "umbrella", "representation", "interaction", "simulation", "impact", "glycosylated", "method", "protein", "trajectory", "information", "texture", "shadow", "light", "surface", "engine", "tool", "frame", "component", "dedicated", "software", "plane", "user", "visualization", "glycan", "residue"], "document_vector": [-107.303382, -12.415006], "paragraphs": [{"paragraph_vector": [57.539825, -33.789154], "paragraph_keywords": ["glycan", "structures", "protein", "structure"]}, {"paragraph_vector": [58.808853, -34.357612], "paragraph_keywords": ["representation", "structures", "representations", "visualization"]}, {"paragraph_vector": [59.501346, -33.27106], "paragraph_keywords": ["glycan", "protein", "vectors", "chains"]}, {"paragraph_vector": [60.042736, -32.819232], "paragraph_keywords": ["glycan", "visualization", "sialylated", "impact"]}, {"paragraph_vector": [118.691032, -0.913927], "paragraph_keywords": ["user", "order", "md", "engine"]}, {"paragraph_vector": [115.92855, 1.17941], "paragraph_keywords": ["components", "prefab", "project", "prefabs"]}, {"paragraph_vector": [25.551019, 68.045021], "paragraph_keywords": ["glycan", "shadow", "plane", "surface"]}, {"paragraph_vector": [85.115859, 19.80163], "paragraph_keywords": ["glycan", "plane", "user", "frames"]}, {"paragraph_vector": [81.624649, 39.043861], "paragraph_keywords": ["glycan", "pixel", "b", "protein"]}, {"paragraph_vector": [59.027816, -32.833187], "paragraph_keywords": ["residues", "glycan", "protein", "order"]}, {"paragraph_vector": [88.20372, 3.079564], "paragraph_keywords": ["frames", "glycan", "number", "method"]}, {"paragraph_vector": [57.846138, -35.063125], "paragraph_keywords": ["protein", "results", "trajectory", "help"]}, {"paragraph_vector": [59.739578, -33.498069], "paragraph_keywords": ["glycan", "protein", "visualization", "surface"]}, {"paragraph_vector": [59.437805, -29.703201], "paragraph_keywords": ["protein", "method", "highlight", "glycosylated"]}], "content": {}, "doi": "10.1109/TVCG.2018.2864851"}, {"uri": "77", "title": "Designing and Evaluating a Haptic System for Biomolecular Education", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Petter Bivall Persson", "Matthew D. Cooper", "Lena A.E. Tibell", "Shaaron Ainsworth", "Anders Ynnerman", "Bengt-Harald Jonsson"], "summary": "In this paper we present an in situ evaluation of a haptic system, with a representative test population, we aim to determine what, if any, benefit haptics can have in a biomolecular education context. We have developed a haptic application for conveying concepts of molecular interactions, specifically in protein-ligand docking. Utilizing a semi-immersive environment with stereo graphics, users are able to manipulate the ligand and feel its interactions in the docking process. The evaluation used cognitive knowledge tests and interviews focused on learning gains. Compared with using time efficiency as the single quality measure this gives a better indication of a system\u2019s applicability in an educational environment. Surveys were used to gather opinions and suggestions for improvements. Students do gain from using the application in the learning process but the learning appears to be independent of the addition of haptic feedback. However the addition of force feedback did decrease time requirements and improved the students understanding of the docking process in terms of the forces involved, as is apparent from the students\u2019 descriptions of the experience. The students also indicated a number of features which could be improved in future development.", "keywords": ["performed", "response", "survey", "data", "structure", "feedback", "molecule", "analysis", "representation", "interaction", "protein", "ligand", "cff", "research", "result", "calculated", "system", "understanding", "figure", "work", "student", "learning", "atom", "-", "chemistry", "use", "docking", "model", "reasoning", "interview", "time", "function", "force", "haptics", "test", "task"], "document_vector": [117.947387, -25.37828], "paragraphs": [{"paragraph_vector": [175.16893, -12.38509], "paragraph_keywords": ["interacting", "environment", "interaction", "work"]}, {"paragraph_vector": [175.976867, -12.052289], "paragraph_keywords": ["force", "feedback", "docking", "calculations"]}, {"paragraph_vector": [175.318756, -11.861904], "paragraph_keywords": ["force", "structures", "haptics", "students"]}, {"paragraph_vector": [172.155197, -14.040511], "paragraph_keywords": ["haptics", "force", "docking", "time"]}, {"paragraph_vector": [162.058898, -11.217591], "paragraph_keywords": ["protein", "system", "ligand", "force"]}, {"paragraph_vector": [156.707427, -25.642677], "paragraph_keywords": ["atom", "force", "ligand", "forces"]}, {"paragraph_vector": [164.12976, -23.226631], "paragraph_keywords": ["ligand", "model", "energy", "process"]}, {"paragraph_vector": [174.237472, -12.455584], "paragraph_keywords": ["learning", "students", "interactions", "understanding"]}, {"paragraph_vector": [177.588897, -12.58749], "paragraph_keywords": ["students", "groups", "tests", "gender"]}, {"paragraph_vector": [178.006774, -12.136805], "paragraph_keywords": ["students", "responses", "protein", "interviews"]}, {"paragraph_vector": [171.102874, -14.158141], "paragraph_keywords": ["task", "students", "responses", "calculated"]}, {"paragraph_vector": [172.927032, -11.573403], "paragraph_keywords": ["haptics", "task", "scores", "table"]}, {"paragraph_vector": [174.529769, -14.548698], "paragraph_keywords": ["haptics", "force", "time", "p"]}, {"paragraph_vector": [167.236389, -15.191938], "paragraph_keywords": ["system", "students", "experience", "ligand"]}, {"paragraph_vector": [171.974563, -11.200031], "paragraph_keywords": ["know", "responses", "system", "use"]}, {"paragraph_vector": [176.167785, -13.40201], "paragraph_keywords": ["students", "force", "docking", "system"]}, {"paragraph_vector": [174.893981, -11.017801], "paragraph_keywords": ["students", "university", "ability", "data"]}], "content": {}, "doi": "10.1109/MPIDC.1996.534100"}, {"uri": "78", "title": "SphereGrinder - reference structure-based tool for quality assessment of protein structural models", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Piotr Lukasiak", "Maciej Antczak", "Tomasz Ratajczak", "Jacek Blazewicz"], "summary": "3D protein structure prediction is of significant interest in the biological research community. Nowadays, one can find many methodologies that can lead to model protein structures, and for that reason the plausible assessment of the quality of protein structural models has fundamental impact on the progress of structural bioinformatics. Here, we present SphereGrinder, a novel computational tool devoted to evaluation of protein 3D models according to the reference structure that gives an opportunity to proceed with structural analysis from atomic to the whole molecule level of accuracy. Proposed tool is capable of handling large predicted models set is designed and used for protein structure prediction in CASP (Critical Assessment of Techniques for Protein Structure Prediction) experiment to complement and add value to the traditional protein structure quality assessment by the global distance test and related scores. SphereGrinder is a user\u00ad friendly software that allows the comprehensive quality inspection conducted between the set of predicted protein models and the reference structure. It is implemented as an online application available for free use by all academic users at the URL http://spheregrinder.cs.put.poznan.pl. Keywordsprotein quality evaluation, protein analysis,", "keywords": [], "document_vector": [-92.389526, 39.09193], "paragraphs": [{"paragraph_vector": [-129.908172, 20.528369], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/IPDPS.2002.1016535"}, {"uri": "79", "title": "Scalable Mining and Analysis of Protein-Protein Interaction Networks", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Shaikh Arifuzzaman"], "summary": "Protein-protein interaction (PPI) networks are the networks of protein complexes formed by biochemical events and electrostatic forces. PPI networks can be used to study diseases and discover drugs. The causes of diseases are evident on a protein interaction level. For instance, an elevation of interaction edge weights of oncogenes is manifested in cancers. Further, the majority of approved drugs target a particular PPI, and thus studying PPI networks is vital to drug discovery. The availability of large datasets and need for efficient analysis necessitate the design of scalable methods leveraging modern high-performance computing (HPC) platforms. In this paper, we design a lightweight framework on a distributed-memory parallel system, which includes scalable algorithmic and analytic techniques to study PPI networks and visualize them. Our study of PPIs is based on network-centric mining and analysis approaches. Since PPI networks are signed (labeled) and weighted, many existing network mining methods working on simple unweighted networks are unsuitable to study PPIs. Further, the large volume and variety of such data limit the use of sequential tools or methods. Many existing tools also do not support a convenient workflow starting from automated data preprocessing to visualizing results and reports for efficient extraction of intelligence from large-scale PPI networks. Our framework supports automated analytics based on a large range of extensible methods for extracting signed motifs, computing centrality, and finding functional units. We design MPI (Message Passing Interface) based parallel methods and workflow, which scale to large networks. The framework is also extensible and sufficiently generic. Keywords-network mining, biological networks, protein-protein interaction, network visualization, massive networks, HPC systems.", "keywords": ["data", "workflow", "analysis", "metric", "graph", "community", "set", "score", "method", "processor", "support", "protein", "algorithm", "network", "framework", "system", "ppi", "process", "centrality", "based", "work", "datasets", "find", "tool", "kernel", "use", "unit", "visualization", "node"], "document_vector": [6.200237, 47.208156], "paragraphs": [{"paragraph_vector": [-112.614479, -21.948606], "paragraph_keywords": ["datasets", "protein", "interaction", "memory"]}, {"paragraph_vector": [-113.421234, -18.549793], "paragraph_keywords": ["based", "analysis", "network", "framework"]}, {"paragraph_vector": [-118.678054, -14.304395], "paragraph_keywords": ["analysis", "unit", "scripts", "set"]}, {"paragraph_vector": [-117.467224, -16.261991], "paragraph_keywords": ["visualization", "unit", "network", "based"]}, {"paragraph_vector": [-114.444458, -20.640249], "paragraph_keywords": ["community", "tool", "kernels", "analysis"]}, {"paragraph_vector": [-111.657814, -19.033506], "paragraph_keywords": ["metrics", "analysis", "community", "subgraphs"]}, {"paragraph_vector": [-118.191032, -21.91925], "paragraph_keywords": ["networks", "analysis", "network", "finding"]}, {"paragraph_vector": [-113.057464, -23.659126], "paragraph_keywords": ["networks", "find", "neighborhood", "protein"]}, {"paragraph_vector": [-112.656471, -23.241931], "paragraph_keywords": ["nodes", "centrality", "protein", "networks"]}, {"paragraph_vector": [-122.399887, -25.4346], "paragraph_keywords": ["methods", "network", "analysis", "networks"]}, {"paragraph_vector": [-114.289283, -17.179922], "paragraph_keywords": ["networks", "tool", "algorithms", "ppi"]}, {"paragraph_vector": [-115.984542, -22.554601], "paragraph_keywords": ["grant", "tool", "work", "variety"]}], "content": {}, "doi": "10.1109/IPDPS.2008.4536189"}, {"uri": "80", "title": "empty", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "Abstract\u2014The incongruence between gene trees and species trees is one of the most pervasive challenges in molecular phylogenetics. In this work, a machine learning approach is proposed to overcome this problem. In the machine learning approach, the gene data set is clustered by a self-organizing map (SOM). Then a phylogenetically informative core gene set is created by combining the maximum entropy gene from each cluster to conduct phylogenetic analysis. Using the same data set, this approach performs better than the previous random gene concatenation method. The SOM based information visualization is also employed to compare the species patterns in the phylogenetic tree constructions.", "keywords": ["p", "data", "analysis", "sample", "tree", "gene", "set", "cluster", "matrix", "sequence", "number", "reference", "som", "clustering", "specie", "vector", "map", "based", "learning", "pattern", "time", "plane", "entropy", "function", "prototype", "mining"], "document_vector": [156.786193, 58.265628], "paragraphs": [{"paragraph_vector": [-81.641738, -16.784603], "paragraph_keywords": ["gene", "genes", "set", "data"]}, {"paragraph_vector": [-75.688491, -16.961225], "paragraph_keywords": ["data", "matrix", "set", "learning"]}, {"paragraph_vector": [-73.590873, -20.097282], "paragraph_keywords": ["reference", "learning", "vector", "plane"]}, {"paragraph_vector": [-69.337959, -13.943406], "paragraph_keywords": ["learning", "function", "neuron", "energy"]}, {"paragraph_vector": [-82.724357, -17.264663], "paragraph_keywords": ["species", "matrix", "gene", "number"]}, {"paragraph_vector": [-82.803497, -17.532564], "paragraph_keywords": ["gene", "plane", "number", "som"]}, {"paragraph_vector": [-83.056312, -16.941118], "paragraph_keywords": ["gene", "clustering", "genes", "data"]}, {"paragraph_vector": [-81.592376, -19.373535], "paragraph_keywords": ["species", "patterns", "plane", "pattern"]}, {"paragraph_vector": [-83.665016, -15.533899], "paragraph_keywords": ["genes", "species", "gene", "analysis"]}, {"paragraph_vector": [-88.086593, -24.801038], "paragraph_keywords": ["entropy", "gene", "sequence", "block"]}, {"paragraph_vector": [-83.696556, -15.499229], "paragraph_keywords": ["gene", "entropy", "genes", "test"]}, {"paragraph_vector": [-83.73423, -12.89159], "paragraph_keywords": ["gene", "entropy", "genes", "cluster"]}, {"paragraph_vector": [-87.471992, -12.017871], "paragraph_keywords": ["gene", "genes", "tree", "set"]}, {"paragraph_vector": [-85.41999, -11.126791], "paragraph_keywords": ["gene", "mining", "tree", "genes"]}, {"paragraph_vector": [-94.439659, 4.854929], "paragraph_keywords": ["gene", "microarray", "cerevisiae", "series"]}], "content": {}, "doi": "10.1109/ICALT.2018.00065"}, {"uri": "81", "title": "empty", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["M. Essabbah", "S. Otmane", "M. Mallem"], "summary": "", "keywords": ["dynamic", "set", "model", "data", "structure", "energy", "molecule", "program", "modeling", "system", "computer", "interaction"], "document_vector": [22.319162, 10.64981], "paragraphs": [{"paragraph_vector": [75.721206, -67.359619], "paragraph_keywords": ["molecules", "modeling", "molecule", "describing"]}, {"paragraph_vector": [103.147377, -71.748588], "paragraph_keywords": ["dynamics", "energy", "modeling", "chemistry"]}, {"paragraph_vector": [149.570236, -75.113655], "paragraph_keywords": ["set", "modeling", "terms", "dynamics"]}, {"paragraph_vector": [82.25901, -20.553348], "paragraph_keywords": ["data", "modeling", "models", "interaction"]}], "content": {}, "doi": "10.1109/MHS.2005.1589986"}, {"uri": "82", "title": "Visualization of Molecular Quantum Dynamics \u2013 A Molecular Visualization Tool with Integrated Web3D and Haptics", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["R. Andrew Davies", "Nigel W. John", "John N. MacDonald", "Keith H. Hughes"], "summary": "The Department of Chemistry and the School of Informatics at the University of Wales, Bangor are working together to create tools for the visualization of molecular quantum dynamics. This paper presents the results of our initial work. A prototype Molecular Visualiser (MV) application has been developed based on Web3D standards, plus extensions for support of haptic interaction. MV provides the user with visualizations of molecular systems, potential energy surfaces, and wavepacket dynamics. These can be displayed in a web browser using VRML, or be delivered to a virtual environment in which haptic properties have been assigned based on the molecular dynamics of the system. The use of MV for both research and teaching is discussed. CR Categories: I.3 [Computing Methodologies]: Computer Graphics \u2013 Applications; J.2 [Computer Applications]: Physical Sciences and Engineering \u2013 Chemistry", "keywords": ["visualization", "data", "structure", "molecule", "inertia", "et", "space", "coordinate", "dynamic", "angle", "quantum", "method", "covalent", "obtained", "matrix", "employed", "y", "chemical", "system", "bond", "z", "mv", "based", "surface", "haptics", "mass", "chemistry", "atom", "technology", "energy", "ax", "vrml", "field", "use", "model", "allow", "software", "radius", "tensor", "t", "property", "force", "environment", "rendering", "node"], "document_vector": [120.752578, -60.413322], "paragraphs": [{"paragraph_vector": [100.523468, -19.907621], "paragraph_keywords": ["quantum", "chemical", "models", "chemist"]}, {"paragraph_vector": [107.770797, -18.139734], "paragraph_keywords": ["visualization", "models", "interaction", "environment"]}, {"paragraph_vector": [168.074295, -12.250426], "paragraph_keywords": ["visualization", "properties", "molecule", "use"]}, {"paragraph_vector": [165.519241, -12.429097], "paragraph_keywords": ["chemistry", "use", "vrml", "et"]}, {"paragraph_vector": [125.207015, 4.314299], "paragraph_keywords": ["vrml", "molecule", "mv", "reachin"]}, {"paragraph_vector": [167.777893, -13.332574], "paragraph_keywords": ["coordinates", "based", "mv", "molecule"]}, {"paragraph_vector": [56.90237, -24.580444], "paragraph_keywords": ["covalent", "radius", "elements", "atom"]}, {"paragraph_vector": [99.069915, 12.008876], "paragraph_keywords": ["nodes", "node", "symbol", "parser"]}, {"paragraph_vector": [158.745162, -41.733863], "paragraph_keywords": ["matrix", "terms", "jacobi", "eigenvectors"]}, {"paragraph_vector": [165.780929, -57.009464], "paragraph_keywords": ["symmetry", "molecule", "el", "mv"]}, {"paragraph_vector": [90.950256, -37.558586], "paragraph_keywords": ["defined", "script", "atom", "angle"]}, {"paragraph_vector": [98.462638, 20.527925], "paragraph_keywords": ["node", "space", "wavepackets", "inertia"]}, {"paragraph_vector": [143.462692, -66.080924], "paragraph_keywords": ["pes", "mv", "form", "data"]}, {"paragraph_vector": [165.450729, -15.811635], "paragraph_keywords": ["chemistry", "dynamics", "work", "figure"]}, {"paragraph_vector": [163.535232, -12.382987], "paragraph_keywords": ["chemistry", "technologies", "based", "modeling"]}, {"paragraph_vector": [169.207458, -17.854942], "paragraph_keywords": ["web", "t", "chemistry", "site"]}, {"paragraph_vector": [58.90874, -67.164581], "paragraph_keywords": ["y", "z", "axes", "iiimi"]}, {"paragraph_vector": [139.575897, -68.190788], "paragraph_keywords": ["aligns", "axis", "ia", "coordinate"]}], "content": {}, "doi": "10.1109/IV.2013.98"}, {"uri": "83", "title": "Defining, Computing, and Visualizing Molecular Interfaces", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Amitabh Varshney", "Frederick P. Brooks", "David C. RichardsonS", "William V. Wright", "Dinesh Manochat"], "summary": "A parallel, analytic approach for defining and computing the interand intra-molecular interfaces in three dimencrions icr described. The molecular interface surfaces are derived from approzimations to the power-diagrams over the participating molecular units. For a given molecular interface our approach can generate a family of interface surfaces parametrized by a! and ,B, where a is the radius of the solvent molecule (also known as the probe-radius) and p is the interface radius that defines the size of the molecular interface. Molecular interface surfaces provide biochemists with a powerful tool to study surface complementarity and to efficiently characterize the interactions during a protein-substrate docking. The complexity of our algorithm for molecular environments is O(nk log\u2019 k), where n is the number of atome in the participating molecular units and k is the average number of neighboring atoms a constant, given (Y and /3.", "keywords": [], "document_vector": [-43.757114, -16.01008], "paragraphs": [{"paragraph_vector": [-129.839385, 18.598812], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/IPDPSW.2016.121"}, {"uri": "84", "title": "Direct Visualization of Molecularly Thin Lubricant Films Using Low-Coherence Phase-Shifting Interferometry", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Hedong Zhang", "Takahiro Banno", "Yasunaga Mitsuya", "Kenji Fukuzawa"], "summary": "We have developed a method utilizing low-coherence phase-shifting interferometry (LCPSI) to directly visualize molecularly thin lubricant films coated on diamond-like carbon (DLC) surfaces. The measurement accuracy is guaranteed by the use of a low-coherence light source, a feedback-controlled phase-shifting system, and an image subtraction method. By measuring nanometer-thick perfluoropolyether films coated on DLC surfaces, we demonstrated that the LCPSI is capable of providing real-time three-dimensional measurements at a sub-nanometer thickness resolution and a spatial resolution of several micrometers.", "keywords": ["disk", "noise", "accuracy", "film", "interference", "reference", "lubricant", "system", "coherence", "mm", "fig", "light", "surface", "measurement", "thickness", "pzt", "phase", "shift", "fringe", "shifting"], "document_vector": [72.280242, -2.713445], "paragraphs": [{"paragraph_vector": [-56.975547, 44.670303], "paragraph_keywords": ["phase", "surface", "measurement", "interferograms"]}, {"paragraph_vector": [-58.580966, 44.604076], "paragraph_keywords": ["light", "lenses", "interference", "coherence"]}, {"paragraph_vector": [-54.911899, 44.076801], "paragraph_keywords": ["phase", "fringe", "interference", "shift"]}, {"paragraph_vector": [-55.981395, 44.694946], "paragraph_keywords": ["disks", "pzt", "film", "mm"]}, {"paragraph_vector": [-54.917579, 44.51493], "paragraph_keywords": ["lubricant", "disks", "line", "light"]}, {"paragraph_vector": [-56.283798, 41.664333], "paragraph_keywords": ["fringe", "phase", "fitted", "deviation"]}, {"paragraph_vector": [-54.743461, 47.287197], "paragraph_keywords": ["phase", "disk", "shifting", "surface"]}, {"paragraph_vector": [-55.516159, 45.573513], "paragraph_keywords": ["surface", "thickness", "film", "fig"]}, {"paragraph_vector": [-56.496986, 44.266021], "paragraph_keywords": ["system", "method", "surfaces", "disk"]}], "content": {}, "doi": "10.1109/IEMBS.1998.745462"}, {"uri": "85", "title": "Gene Expression Data Analysis Using a Novel Approach to Biclustering Combining Discrete and Continuous Data", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yann Christinat", "Bernd Wachmann", "Lei Zhang"], "summary": "Many different methods exist for pattern detection in gene expression data. In contrast to classical methods, biclustering has the ability to cluster a group of genes together with a group of conditions (replicates, set of patients, or drug compounds). However, since the problem is NP-complex, most algorithms use heuristic search functions and, therefore, might converge toward local maxima. By using the results of biclustering on discrete data as a starting point for a local search function on continuous data, our algorithm avoids the problem of heuristic initialization. Similar to Order-Preserving Submatrices (OPSM), our algorithm aims to detect biclusters whose rows and columns can be ordered such that row values are growing across the bicluster\u2019s columns and vice versa. Results have been generated on the yeast genome (Saccharomyces cerevisiae), a human cancer data set, and random data. Results on the yeast genome showed that 89 percent of the 100 biggest nonoverlapping biclusters were enriched with Gene Ontology annotations. A comparison with the methods OPSM and Iterative Signature Algorithm (ISA, a generalization of singular value decomposition) demonstrated a better efficiency when using gene and condition orders. We present results on random and real data sets that show the ability of our algorithm to capture statistically significant and biologically relevant biclusters.", "keywords": ["performed", "p", "data", "value", "order", "et", "gene", "expression", "set", "score", "algorithm", "\u00fe", "variance", "percent", "applied", "information", "number", "result", "distribution", "defined", "annotation", "biclusters", "computed", "based", "j", "element", "seed", "size", "bicluster", "condition", "al", "biclustering"], "document_vector": [118.750343, 79.901924], "paragraphs": [{"paragraph_vector": [-93.292816, 10.82189], "paragraph_keywords": ["algorithms", "gene", "biclustering", "data"]}, {"paragraph_vector": [-94.137252, 16.242422], "paragraph_keywords": ["algorithm", "data", "biclustering", "gene"]}, {"paragraph_vector": [-90.796867, 16.596651], "paragraph_keywords": ["data", "gene", "condition", "biclusters"]}, {"paragraph_vector": [-90.048728, 16.885232], "paragraph_keywords": ["set", "gene", "condition", "biclusters"]}, {"paragraph_vector": [-95.225578, 13.475408], "paragraph_keywords": ["condition", "\u00fe", "gene", "oc\u00f0i\u00fe"]}, {"paragraph_vector": [-94.895164, 14.432044], "paragraph_keywords": ["number", "size", "biclusters", "data"]}, {"paragraph_vector": [-95.11412, 17.600257], "paragraph_keywords": ["data", "\u00fe", "biclustering", "order"]}, {"paragraph_vector": [-93.957, 15.382787], "paragraph_keywords": ["data", "genes", "values", "algorithm"]}, {"paragraph_vector": [-93.651077, 15.305503], "paragraph_keywords": ["elements", "matrix", "distribution", "values"]}, {"paragraph_vector": [-88.984436, 8.999136], "paragraph_keywords": ["conditions", "biclusters", "seed", "seeds"]}, {"paragraph_vector": [-85.683036, 7.803832], "paragraph_keywords": ["order", "seed", "method", "conditions"]}, {"paragraph_vector": [-95.612304, 12.506285], "paragraph_keywords": ["genes", "gene", "condition", "coefficient"]}, {"paragraph_vector": [-97.543342, 12.779964], "paragraph_keywords": ["condition", "gene", "order", "addition"]}, {"paragraph_vector": [-94.618156, 12.035534], "paragraph_keywords": ["gene", "condition", "score", "computed"]}, {"paragraph_vector": [-94.15612, 13.667794], "paragraph_keywords": ["biclusters", "data", "bicluster", "conditions"]}, {"paragraph_vector": [-95.750839, 16.262086], "paragraph_keywords": ["data", "set", "size", "sparseness"]}, {"paragraph_vector": [-95.62255, 16.637727], "paragraph_keywords": ["data", "set", "found", "annotations"]}, {"paragraph_vector": [-92.661064, 16.057918], "paragraph_keywords": ["biclusters", "set", "genes", "algorithm"]}, {"paragraph_vector": [-92.144935, 16.436544], "paragraph_keywords": ["biclusters", "percent", "annotations", "condition"]}, {"paragraph_vector": [-70.564163, 3.194994], "paragraph_keywords": ["biclusters", "data", "dlbcl", "results"]}, {"paragraph_vector": [-90.075332, 15.760279], "paragraph_keywords": ["algorithm", "biclusters", "data", "annotations"]}], "content": {}, "doi": "10.1109/IV.1999.781579"}, {"uri": "86", "title": "Visual Analysis of Ligand Trajectories in Molecular Dynamics", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Adam Jur\u010d\u0131\u0301k", "Katar\u0131\u0301na Furmanov\u00e1", "Jan By\u0161ka", "Vojt\u011bch Von\u00e1sek", "Ond\u0159ej V\u00e1vra", "Pavol Ulbrich", "Helwig Hauser", "Barbora Kozl\u0131\u0301kov\u00e1"], "summary": "In many cases, protein reactions with other small molecules (ligands) occur in a deeply buried active site. When studying these types of reactions, it is crucial for biochemists to examine trajectories of ligand motion. These trajectories are predicted with in-silico methods that produce large ensembles of possible trajectories. In this paper, we propose a novel approach to the interactive visual exploration and analysis of large sets of ligand trajectories, enabling the domain experts to understand protein function based on the trajectory properties. The proposed solution is composed of multiple linked 2D and 3D views, enabling the interactive exploration and filtering of trajectories in an informed way. In the workflow, we focus on the practical aspects of the interactive visual analysis specific to ligand trajectories. We adapt the small multiples principle to resolve an overly large number of trajectories into smaller chunks that are easier to analyze. We describe how drill-down techniques can be used to create and store selections of the trajectories with desired properties, enabling the comparison of multiple datasets. In appropriately designed 2D and 3D views, biochemists can either observe individual trajectories or choose to aggregate the information into a functional boxplot or density visualization. Our solution is based on a tight collaboration with the domain experts, aiming to address their needs as much as possible. The usefulness of our novel approach is demonstrated by two case studies, conducted by the collaborating protein engineers.", "keywords": ["data", "dataset", "exploration", "order", "analysis", "distance", "chart", "binning", "way", "selection", "representation", "simulation", "set", "space", "method", "cluster", "protein", "matrix", "trajectory", "ligand", "number", "part", "system", "selected", "figure", "case", "change", "technique", "based", "datasets", "binding", "tool", "energy", "site", "use", "view", "bin", "time", "property", "biochemist", "user", "visualization"], "document_vector": [-125.549011, 22.863819], "paragraphs": [{"paragraph_vector": [8.140205, -66.650077], "paragraph_keywords": ["protein", "ligand", "trajectories", "proteins"]}, {"paragraph_vector": [17.811695, -81.700431], "paragraph_keywords": ["ligand", "trajectories", "tool", "protein"]}, {"paragraph_vector": [38.455493, -70.563545], "paragraph_keywords": ["ligand", "energy", "data", "trajectories"]}, {"paragraph_vector": [24.471014, -75.673957], "paragraph_keywords": ["ligand", "protein", "data", "site"]}, {"paragraph_vector": [91.148017, -89.069671], "paragraph_keywords": ["trajectories", "protein", "visualization", "exploration"]}, {"paragraph_vector": [64.450057, -69.785987], "paragraph_keywords": ["simulation", "exploration", "trajectories", "trajectory"]}, {"paragraph_vector": [60.268062, -68.0307], "paragraph_keywords": ["protein", "ligand", "water", "analysis"]}, {"paragraph_vector": [-178.553146, -82.451522], "paragraph_keywords": ["data", "properties", "methods", "trajectory"]}, {"paragraph_vector": [-78.101959, -74.600814], "paragraph_keywords": ["data", "analysis", "visualization", "clustering"]}, {"paragraph_vector": [47.854835, -86.234878], "paragraph_keywords": ["trajectories", "properties", "trajectory", "section"]}, {"paragraph_vector": [-49.334892, -88.931274], "paragraph_keywords": ["binning", "data", "trajectories", "properties"]}, {"paragraph_vector": [14.327381, -89.414909], "paragraph_keywords": ["charts", "data", "trajectories", "profile"]}, {"paragraph_vector": [36.033309, -89.577644], "paragraph_keywords": ["data", "selected", "bins", "trajectories"]}, {"paragraph_vector": [-9.974013, -88.740806], "paragraph_keywords": ["data", "view", "energy", "explore"]}, {"paragraph_vector": [29.59616, -82.676986], "paragraph_keywords": ["trajectories", "grid", "based", "number"]}, {"paragraph_vector": [66.542518, -82.953956], "paragraph_keywords": ["representation", "time", "datasets", "exploration"]}, {"paragraph_vector": [32.283687, -78.990875], "paragraph_keywords": ["trajectories", "datasets", "selections", "chart"]}, {"paragraph_vector": [10.906057, -78.07936], "paragraph_keywords": ["site", "trajectories", "linb", "datasets"]}, {"paragraph_vector": [-3.481571, -80.348609], "paragraph_keywords": ["trajectories", "bins", "trajectory", "tool"]}, {"paragraph_vector": [-27.01889, -82.843864], "paragraph_keywords": ["trajectories", "tool", "site", "order"]}, {"paragraph_vector": [-19.938604, -82.51593], "paragraph_keywords": ["trajectories", "site", "selection", "bins"]}, {"paragraph_vector": [19.953174, -83.88639], "paragraph_keywords": ["trajectories", "rmsd", "protein", "energy"]}, {"paragraph_vector": [74.970428, -87.655647], "paragraph_keywords": ["data", "helped", "biochemists", "energy"]}, {"paragraph_vector": [100.469963, -87.158905], "paragraph_keywords": ["data", "system", "ligand", "binding"]}, {"paragraph_vector": [59.632946, -87.757713], "paragraph_keywords": ["trajectory", "provided", "project", "data"]}], "content": {}, "doi": "10.1162/pres.17.1.73"}, {"uri": "87", "title": "Exploring the relation between the characteristics of protein interaction networks and the performances of computational complex detection methods", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Xiaoxia Liu", "Zhihao Yang", "Ziwei Zhou", "Yuanyuan Sun", "Hongfei Lin", "Jian Wang"], "summary": "In this paper, we analyze six protein interaction networks widely used for protein complex detection, and compare the performance of six classic computational methods on them in order to find the relations between network characteristics and the performances of these complex detection methods. Furthermore, we explore the difference among the two complexes detected by different methods and the real complex by using a visualization approach which can easily find the difference of two methods and locate the undetected proteins. Keywords\u2014Protein interaction network;protein complex detection; Visualization approach", "keywords": ["method", "characteristic", "find", "complex", "coach", "pin", "node"], "document_vector": [8.197798, 52.364723], "paragraphs": [{"paragraph_vector": [-109.092422, -19.844722], "paragraph_keywords": ["methods", "nodes", "coach", "collins"]}, {"paragraph_vector": [-113.365821, -46.725727], "paragraph_keywords": ["methods", "pins", "complex", "protein"]}], "content": {}, "doi": "10.1109/IPDPSW.2016.127"}, {"uri": "88", "title": "empty", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kizu-cho Souraku-gun"], "summary": "H. Yazawat, T Shioyamal, Y Esumit, F Kannarit, R. Itakura2, and K. Yamanouchi3 1. Department ofElectronics and Electrical Engineering, Keio University 3-14-1, Hiyoshi, Kohoku-ku, Yokohama, 223-8522, Japan 2. Kansai Photon Science Institute, Japan Atomic Energy Agency 8-1, Umemidai Kizu-cho Souraku-gun Kyoto, 619-0215, Japan 3. Department ofChemistry, School ofSciene, The University ofTokyo, 7-3-1 Hongo, Bunkyo-ku, Tokyo, 113-0033, Japan", "keywords": [], "document_vector": [21.607025, 9.669322], "paragraphs": [{"paragraph_vector": [-130.168731, 20.15262], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TPS.2005.844953"}, {"uri": "89", "title": "SHREC 2020: Classification in cryo-electron tomograms", "timestamp": "2020", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ilja Gubins", "Marten L. Chaillet", "Gijs van der Schot", "Remco C. Veltkamp", "Yu Hao", "Xiaohua Wan", "Xuefeng Cui", "Emmanuel Moebel", "Xiao Wang", "Daisuke Kihara", "Xiangrui Zeng", "Min Xu", "Nguyen P. Nguyen", "Tommi White", "Filiz Bunyak"], "summary": "Cryo-electron tomography (cryo-ET) is an imaging technique that allows us to three-dimensionally visualize both the structural details of macro-molecular assemblies under near-native conditions and its cellular context. Electrons strongly interact with biological samples, limiting electron dose. The latter limits the signal-to-noise ratio and hence resolution of an individual tomogram to about 50 (5 nm). Biological molecules can be obtained by averaging volumes, each depicting copies of the molecule, allowing for resolutions beyond 4 (0.4 nm). To this end, the ability to localize and classify components is crucial, but challenging due to the low signal-to-noise ratio. Computational innovation is key to mine biological information from cryo-electron tomography. To promote such innovation, we provide a novel simulated dataset to benchmark different methods of localization and classification of biological macromolecules in cryo-electron tomograms. Our publicly available dataset contains ten tomographic reconstructions of simulated cell-like volumes. Each volume contains twelve different types of complexes, varying in size, function and structure. In this paper, we have evaluated seven different methods of finding and classifying proteins. Six research groups present results obtained with learning-based methods and trained on the simulated dataset, as well as a baseline template matching, a traditional method widely used in cryo-ET research. We find that method performance correlates with particle size, especially noticeable for template matching which performance degrades rapidly as the size decreases. We learn that neural networks can achieve significantly better localization and classification performance, in particular convolutional networks with focus on high-resolution details such as those based on U-Net architecture. \u00a9 2020 Elsevier Ltd. All rights reserved.", "keywords": [], "document_vector": [129.573181, 33.345592], "paragraphs": [{"paragraph_vector": [-122.085342, 28.697782], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/VISUAL.2004.23"}, {"uri": "90", "title": "Comparative Visualization of Molecular Surfaces Using Deformable Models", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["K. Scharnowski", "M. Krone", "G. Reina", "T. Kulschewski", "J. Pleiss", "T. Ertl"], "summary": "The comparison of molecular surface attributes is of interest for computer aided drug design and the analysis of biochemical simulations. Due to the non-rigid nature of molecular surfaces, partial shape matching is feasible for mapping two surfaces onto each other. We present a novel technique to obtain a mapping relation between two surfaces using a deformable model approach. This relation is used for pair-wise comparison of local surface attributes (e.g. electrostatic potential). We combine the difference value as well as the comparability as derived from the local matching quality in a 3D molecular visualization by mapping them to color. A 2D matrix shows the global dissimilarity in an overview of different data sets in an ensemble. We apply our visualizations to simulation results provided by collaborators from the field of biochemistry to evaluate the effectiveness of our results.", "keywords": ["data", "volume", "area", "analysis", "comparison", "target", "gvf", "deformation", "mesh", "method", "input", "obtained", "triangle", "rmsd", "point", "approach", "result", "diffusion", "figure", "defined", "plot", "process", "surface", "based", "mapping", "source", "work", "computed", "field", "use", "model", "vertex", "region", "time", "g", "force", "visualization", "shape"], "document_vector": [-127.69371, -19.154026], "paragraphs": [{"paragraph_vector": [0.209652, -17.518405], "paragraph_keywords": ["function", "surfaces", "molecules", "surface"]}, {"paragraph_vector": [0.460194, -6.888654], "paragraph_keywords": ["surfaces", "surface", "input", "dissimilarity"]}, {"paragraph_vector": [-2.9724, -11.656759], "paragraph_keywords": ["surface", "based", "deformation", "input"]}, {"paragraph_vector": [18.039144, 14.558978], "paragraph_keywords": ["surfaces", "surface", "tracking", "genus"]}, {"paragraph_vector": [0.457563, -5.39739], "paragraph_keywords": ["surface", "input", "shape", "target"]}, {"paragraph_vector": [20.60333, 10.291225], "paragraph_keywords": ["surface", "surfaces", "alignment", "structures"]}, {"paragraph_vector": [20.199195, -0.15652], "paragraph_keywords": ["surface", "forces", "model", "force"]}, {"paragraph_vector": [14.569945, 17.442823], "paragraph_keywords": ["source", "gvf", "forces", "surface"]}, {"paragraph_vector": [22.725254, 9.190662], "paragraph_keywords": ["source", "field", "gvf", "surface"]}, {"paragraph_vector": [9.522065, 13.878041], "paragraph_keywords": ["surface", "potential", "deformation", "mapping"]}, {"paragraph_vector": [28.447374, 18.125801], "paragraph_keywords": ["mesh", "surface", "volume", "t"]}, {"paragraph_vector": [20.383604, 11.243477], "paragraph_keywords": ["forces", "surface", "source", "vertices"]}, {"paragraph_vector": [21.603551, 9.52638], "paragraph_keywords": ["triangles", "surface", "vertices", "dsign"]}, {"paragraph_vector": [50.843357, 19.148487], "paragraph_keywords": ["surface", "mapping", "rendering", "visualization"]}, {"paragraph_vector": [17.257011, -43.11676], "paragraph_keywords": ["data", "surface", "sets", "method"]}, {"paragraph_vector": [16.916007, -7.96038], "paragraph_keywords": ["surface", "figure", "shows", "areas"]}, {"paragraph_vector": [15.297711, -33.558765], "paragraph_keywords": ["figure", "rmsd", "resulting", "surface"]}, {"paragraph_vector": [0.944018, -9.1761], "paragraph_keywords": ["triangles", "deformation", "g", "molecules"]}, {"paragraph_vector": [2.127711, -6.351275], "paragraph_keywords": ["surface", "data", "field", "plot"]}], "content": {}, "doi": "10.1109/MNANO.2015.2441110"}, {"uri": "91", "title": "Structural Class Classification of 3D Protein Structure Based on Multi-view 2D Images", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Chendra Hadi Suryanto", "Hiroto Saigo", "Kazuhiro Fukui"], "summary": "Computing similarity or dissimilarity between protein structures is an important task in structural biology. Conventional method to compute protein structure dissimilarity requires structural alignment of the proteins. However, defining one best alignment is difficult, especially when the structures are very different. In this paper, we propose a new similarity measure for protein structure comparisons using a set of multi-view 2D images of 3D protein structures. In this approach, each protein structure is represented by a subspace from the image set. The similarity between two protein structures is then characterized by the canonical angles between the two subspaces. The primary advantage of our method is that precise alignment is not needed. We employed Grassmann Discriminant Analysis (GDA) as the subspace-based learning in the classification framework. We applied our method for the classification problem of seven SCOP structural classes of protein 3D structures. The proposed method outperformed the k-nearest neighbor method (k-NN) based on conventional alignment-based methods CE, FATCAT and TM-align. Our method was also applied to the classification of SCOP folds of membrane proteins, where the proposed method could recognize the fold HEM-binding four-helical bundle (f.21) much better than TM-Align.", "keywords": ["p", "alignment", "structure", "lbp", "membrane", "msm", "analysis", "distance", "gda", "similarity", "subspace", "classification", "set", "method", "angle", "protein", "information", "number", "extraction", "section", "result", "system", "proposed", "based", "use", "class", "feature", "\u03b1", "view", "image", "visualization"], "document_vector": [-109.580986, 60.573429], "paragraphs": [{"paragraph_vector": [-57.651615, -47.729934], "paragraph_keywords": ["alignment", "rmsd", "protein", "science"]}, {"paragraph_vector": [-54.254608, -44.515575], "paragraph_keywords": ["protein", "structure", "based", "structures"]}, {"paragraph_vector": [-70.801925, -35.473712], "paragraph_keywords": ["protein", "visualization", "structures", "images"]}, {"paragraph_vector": [-60.112342, -22.173873], "paragraph_keywords": ["msm", "method", "protein", "recognition"]}, {"paragraph_vector": [-64.653015, -26.956096], "paragraph_keywords": ["protein", "section", "based", "images"]}, {"paragraph_vector": [-72.513313, -37.281047], "paragraph_keywords": ["protein", "visualization", "structure", "image"]}, {"paragraph_vector": [-63.153369, -25.565578], "paragraph_keywords": ["lbp", "image", "number", "pixel"]}, {"paragraph_vector": [-56.795505, -19.631954], "paragraph_keywords": ["p", "subspaces", "feature", "protein"]}, {"paragraph_vector": [-59.573543, -22.404693], "paragraph_keywords": ["subspaces", "angles", "class", "grassmann"]}, {"paragraph_vector": [-66.647804, -9.176995], "paragraph_keywords": ["w", "distance", "class", "feature"]}, {"paragraph_vector": [-58.780986, -24.771751], "paragraph_keywords": ["\u03b1", "step", "protein", "matrix"]}, {"paragraph_vector": [-63.164669, -26.629899], "paragraph_keywords": ["proteins", "proposed", "method", "step"]}, {"paragraph_vector": [-67.73574, -29.342811], "paragraph_keywords": ["proteins", "feature", "class", "parameters"]}, {"paragraph_vector": [-66.607063, -31.783834], "paragraph_keywords": ["method", "ce", "gda", "protein"]}, {"paragraph_vector": [-66.017822, -31.214941], "paragraph_keywords": ["visualizations", "visualization", "use", "combination"]}, {"paragraph_vector": [-67.047897, -29.70152], "paragraph_keywords": ["methods", "test", "method", "results"]}, {"paragraph_vector": [-67.498207, -30.498617], "paragraph_keywords": ["proteins", "method", "protein", "class"]}, {"paragraph_vector": [-75.316329, -33.929401], "paragraph_keywords": ["method", "proteins", "fold", "membrane"]}, {"paragraph_vector": [-63.805347, -55.894653], "paragraph_keywords": ["protein", "system", "based", "web"]}, {"paragraph_vector": [-84.997314, -34.752956], "paragraph_keywords": ["time", "speed", "visualization", "system"]}, {"paragraph_vector": [-67.220558, -29.793624], "paragraph_keywords": ["protein", "method", "view", "proteins"]}, {"paragraph_vector": [-64.82299, -28.173114], "paragraph_keywords": ["protein", "feature", "subspaces", "extraction"]}], "content": {}, "doi": "10.1109/CLEOPR.2007.4391181"}, {"uri": "92", "title": "DECONVOLVING THE IMAGES", "timestamp": "2003", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "In researching the communication mechanisms between cells of the immune system, visualization of proteins in three dimensions can be used to determine which proteins are capable of interacting with one another at a given time by showing their spatial colocality. Volume data sets are created using digital confocal immunofluorescence microscopy. A variety of visualization approaches are then used to examine the interactions. These include volume rendering, isosurface extraction, and virtual reality. Based on our experiences, we have concluded that no single one of these approaches provides a complete solution for visualizing biological data. However, in combination, their respective strengths complement one another to provide an understanding of the data.", "keywords": ["data", "volume", "interaction", "microscope", "set", "protein", "shown", "information", "system", "understanding", "figure", "light", "animation", "technology", "deconvolution", "image", "rendering", "visualization", "cell", "sandia"], "document_vector": [-94.238258, -50.613372], "paragraphs": [{"paragraph_vector": [-41.566448, 7.601904], "paragraph_keywords": ["proteins", "cell", "microscope", "location"]}, {"paragraph_vector": [-36.799713, 8.15431], "paragraph_keywords": ["light", "image", "point", "lens"]}, {"paragraph_vector": [-2.555423, 72.072105], "paragraph_keywords": ["data", "volume", "animations", "rendering"]}, {"paragraph_vector": [115.346504, 50.553943], "paragraph_keywords": ["data", "volume", "figure", "set"]}, {"paragraph_vector": [-4.491977, 68.733612], "paragraph_keywords": ["data", "volume", "shown", "protein"]}, {"paragraph_vector": [3.499526, 69.723922], "paragraph_keywords": ["information", "visualization", "sandia", "cell"]}, {"paragraph_vector": [-154.90744, 10.272503], "paragraph_keywords": ["constantine", "kupfer", "pavlakos", "avi"]}], "content": {}, "doi": "10.1109/IV.2004.1320224"}, {"uri": "93", "title": "Neuroinformatics for Genome-Wide 3D Gene Expression Mapping in the Mouse Brain", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Lydia Ng", "Sayan D. Pathak", "Chihchau Kuan", "Christopher Lau", "Hongwei Dong", "Andrew Sodt", "Chinh Dang", "Brian Avants", "Paul Yushkevich", "James C. Gee", "David Haynor", "Ed Lein", "Allan Jones", "Mike Hawrylycz"], "summary": "Large-scale gene expression studies in the mammalian brain offer the promise of understanding the topology, networks, and, ultimately, the function of its complex anatomy, opening previously unexplored avenues in neuroscience. High-throughput methods permit genome-wide searches to discover genes that are uniquely expressed in brain circuits and regions that control behavior. Previous gene expression mapping studies in model organisms have employed in situ hybridization (ISH), a technique that uses labeled nucleic acid probes to bind to specific mRNA transcripts in tissue sections. A key requirement for this effort is the development of fast and robust algorithms for anatomically mapping and quantifying gene expression for ISH. We describe a neuroinformatics pipeline for automatically mapping expression profiles of ISH data and its use to produce the first genomic scale 3D mapping of gene expression in a mammalian brain. The pipeline is fully automated and adaptable to other organisms and tissues. Our automated study of more than 20,000 genes indicates that at least 78.8 percent are expressed at some level in the adult C56BL/6J mouse brain. In addition to providing a platform for genomic scale search, high-resolution images and visualization tools for expression analysis are available at the Allen Brain Atlas web site (http://www.brain-map.org).", "keywords": ["registration", "data", "volume", "structure", "automated", "pipeline", "ish", "gene", "expression", "method", "reconstruction", "information", "number", "approach", "mask", "scale", "reference", "section", "intensity", "template", "object", "result", "annotation", "process", "striatum", "resolution", "based", "atlas", "mapping", "level", "aba", "allen", "tissue", "mouse", "search", "brain", "region", "processing", "found", "image", "expressing", "series", "cell"], "document_vector": [71.801513, 87.416915], "paragraphs": [{"paragraph_vector": [-86.07994, 42.208316], "paragraph_keywords": ["expression", "gene", "system", "brain"]}, {"paragraph_vector": [-72.603553, 46.764247], "paragraph_keywords": ["image", "including", "-", "expression"]}, {"paragraph_vector": [-79.521568, 46.68827], "paragraph_keywords": ["gene", "expression", "brain", "sections"]}, {"paragraph_vector": [-72.461875, 49.139804], "paragraph_keywords": ["atlas", "image", "section", "brain"]}, {"paragraph_vector": [-68.176612, 53.568298], "paragraph_keywords": ["volume", "template", "annotation", "registration"]}, {"paragraph_vector": [-76.036727, 46.956451], "paragraph_keywords": ["registration", "images", "series", "intensity"]}, {"paragraph_vector": [-81.106849, 40.271518], "paragraph_keywords": ["registration", "shape", "information", "expression"]}, {"paragraph_vector": [-64.618637, 53.085006], "paragraph_keywords": ["registration", "section", "step", "found"]}, {"paragraph_vector": [-86.554931, 41.191177], "paragraph_keywords": ["objects", "tissue", "expression", "intensity"]}, {"paragraph_vector": [-85.68412, 40.510604], "paragraph_keywords": ["cell", "combined", "objects", "found"]}, {"paragraph_vector": [-77.47525, 46.890663], "paragraph_keywords": ["expression", "image", "grid", "atlas"]}, {"paragraph_vector": [-81.07798, 43.134384], "paragraph_keywords": ["genes", "expression", "striatum", "list"]}, {"paragraph_vector": [-77.670616, 42.849128], "paragraph_keywords": ["expression", "gene", "brain", "atlas"]}, {"paragraph_vector": [-85.963043, 44.163543], "paragraph_keywords": ["expression", "gene", "data", "indicates"]}, {"paragraph_vector": [-79.788276, 43.649967], "paragraph_keywords": ["brain", "projects", "expression", "atlas"]}, {"paragraph_vector": [-78.655853, 44.170578], "paragraph_keywords": ["mapping", "data", "expression", "pipeline"]}, {"paragraph_vector": [-79.248886, 41.856548], "paragraph_keywords": ["registration", "series", "gene", "section"]}, {"paragraph_vector": [-82.955902, 41.972278], "paragraph_keywords": ["data", "cell", "pipeline", "grid"]}, {"paragraph_vector": [-79.552551, 41.440277], "paragraph_keywords": ["allen", "brain", "institute", "ogy"]}], "content": {}, "doi": "10.1109/PACIFICVIS.2015.7156385"}, {"uri": "94", "title": "Visualizing the Protein Sequence Universe", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Larissa Stanberry", "Roger Higdon", "Winston Haynes", "Natali Kolker", "William Broomall", "Saliya Ekanayake"], "summary": "Modern biology is experiencing a rapid increase in data volumes that challenges our analytical skills and existing cyberinfrastructure. Exponential expansion of the Protein Sequence Universe (PSU), the protein sequence space, together with the costs and complexities of manual curation creates a major bottleneck in life sciences research. Existing resources lack scalable visualization tools that are instrumental for functional annotation. Here, we describe a multidimensional scaling (MDS) implementation to create a 3D embedding of the PSU that allows visualizing the relationships between large numbers of proteins. To demonstrate the method, we use sequence similarity scores as a measure of proximity. An example of the prokaryotic PSU shows that the low-dimensional representation preserves important grouping features such as relative proximity of functionally similar clusters and clear structural separation between clusters with specific and general functions. The advantages of the method and its implementation include the ability to scale to large numbers of sequences, integrate different similarity measures with other functional and experimental data, and facilitate protein annotation. Transdisciplinary approaches akin to the one described in this paper are urgently needed to quickly and efficiently translate the influx of new data into tangible innovations and groundbreaking discoveries. Corresponding author, 1900 9th Ave, C9S-9, Seattle, WA 98101, larissa.stanberry@seattlechildrens.org Permission to make digital or hard copies of all or part of this work for personal or classroom use is granted without fee provided that copies are not made or distributed for profit or commercial advantage and that copies bear this notice and the full citation on the first page. To copy otherwise, to republish, to post on servers or to redistribute to lists, requires prior specific permission and/or a fee. ECMLS\u201912, June 18, 2012, Delft, The Netherlands. Copyright 2012 ACM 978-1-4503-1339-1/12/06 ...$10.00.", "keywords": ["visualization", "project", "data", "alignment", "structure", "group", "md", "delsa", "jan", "et", "analysis", "distance", "similarity", "nw", "resource", "sequenced", "scaling", "computing", "cluster", "database", "including", "method", "cog", "protein", "algorithm", "psu", "sequence", "information", "approach", "scale", "figure", "annotation", "implementation", "life", "pathway", "fox", ".", "based", "editor", "tool", "performance", "science", "al", "table", "function", "genome", "page", "kolker"], "document_vector": [157.419525, 87.979751], "paragraphs": [{"paragraph_vector": [-101.08068, -43.764274], "paragraph_keywords": ["proteins", "data", "sequenced", "protein"]}, {"paragraph_vector": [-102.311408, -35.925849], "paragraph_keywords": ["data", "annotation", "protein", "table"]}, {"paragraph_vector": [-96.979362, -36.929859], "paragraph_keywords": ["data", "efforts", "life", "approach"]}, {"paragraph_vector": [-106.625251, -42.179214], "paragraph_keywords": ["data", "psu", "scale", "cog"]}, {"paragraph_vector": [-95.961723, -43.677276], "paragraph_keywords": ["protein", "clusters", "table", "proteins"]}, {"paragraph_vector": [-91.248222, -42.196281], "paragraph_keywords": ["table", "mds", "nw", "implementation"]}, {"paragraph_vector": [-101.495132, -43.804637], "paragraph_keywords": ["protein", "data", "clusters", "figure"]}, {"paragraph_vector": [-92.307838, -50.148029], "paragraph_keywords": ["clusters", "figure", "nw", "distances"]}, {"paragraph_vector": [-97.946556, -44.759513], "paragraph_keywords": ["transport", "clusters", "proteins", "protein"]}, {"paragraph_vector": [-97.127372, -40.792385], "paragraph_keywords": ["data", "mds", "scale", "proteins"]}, {"paragraph_vector": [-97.990425, -38.775161], "paragraph_keywords": ["delsa", "sciences", "informatics", "university"]}, {"paragraph_vector": [-93.150505, -36.05474], "paragraph_keywords": [".", "analysis", "et", "data"]}, {"paragraph_vector": [-105.177215, -29.138589], "paragraph_keywords": ["computing", ".", "editors", "protein"]}, {"paragraph_vector": [-94.99578, -35.108028], "paragraph_keywords": ["protein", "genes", "kolker", "."]}, {"paragraph_vector": [-92.360656, -38.286602], "paragraph_keywords": ["kolker", ".", "function", "proteins"]}, {"paragraph_vector": [-101.051651, -35.49657], "paragraph_keywords": [".", "data", "protein", "et"]}, {"paragraph_vector": [-93.042945, -44.057861], "paragraph_keywords": ["protein", "et", "database", "sequence"]}], "content": {}, "doi": "10.1109/ULTSYM.2011.0402"}, {"uri": "95", "title": "Multi-Layered Networks for Visualizing the Interactions between Heterogeneous Proteins", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yu Chen", "Guangyu Cui", "Kyungsook Han"], "summary": "Recent improvements in high-throughput proteomics technology have produced a large amount of protein interaction data. Due to the volume of data, a graphical representation of protein interactions has proven to be much easier to understand than a long list of interacting proteins. For the past years we have studied the interactions between virus proteins and host cell proteins, but most methods that are typically used for visualizing protein interactions are not appropriate for the interactions between proteins of different types. These methods are intended for the interactions between proteins of a same type, and do not distinguish the interactions between proteins of a same type from those of different types. Visualization of the interactions between different types of proteins (such as virus proteins and host cell proteins) requires a new graph layout algorithm. This paper discusses a new layout algorithm for visualizing the interactions between heterogeneous proteins, and its application to the interactions between virus proteins and human proteins.", "keywords": ["method", "protein", "algorithm", "layer", "graph", "interaction", "node"], "document_vector": [10.60317, 64.991691], "paragraphs": [{"paragraph_vector": [-123.120964, -42.345504], "paragraph_keywords": ["proteins", "graph", "layout", "interactions"]}, {"paragraph_vector": [-114.357681, -35.212009], "paragraph_keywords": ["proteins", "interactions", "layer", "method"]}, {"paragraph_vector": [-100.381324, 1.97842], "paragraph_keywords": ["nodes", "layer", "barycenter", "figure"]}, {"paragraph_vector": [-115.814361, -37.653015], "paragraph_keywords": ["layer", "proteins", "nodes", "interactions"]}, {"paragraph_vector": [-116.303276, -41.879669], "paragraph_keywords": ["proteins", "network", "interactions", "figure"]}, {"paragraph_vector": [-119.144828, -38.467193], "paragraph_keywords": ["networks", "edge", "crossings", "algorithm"]}], "content": {}, "doi": "10.1109/ISBI48211.2021.9433953"}, {"uri": "96", "title": "Visualizing Dynamic Molecular Conformations", "timestamp": "2002", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Johannes Schmidt-Ehrenberg", "Daniel Baum", "Hans-Christian Hege"], "summary": "The bioactivity of a molecule strongly depends on its metastable conformational shapes and the transitions between these. Therefore, conformation analysis and visualization is a basic prerequisite for the understanding of biochemical processes. We present techniques for visual analysis of metastable molecular conformations. Core of these are flexibly applicable methods for alignment of molecular geometries, as well as methods for depicting shape and \u2019fuzziness\u2019 of metastable conformations. All analysis tools are provided in an integrated working environment. The described techniques are demonstrated with pharmaceutically active biomolecules. CR Categories: G.3 [Mathematics of Computing]: Probability and Statistics\u2014Distribution functions; I.3.8 [Computing Methodologies]: Computer Graphics\u2014Applications; I.6.6 [Computing Methodologies]: Simulation and Modeling\u2014Simulation Output Analysis", "keywords": ["alignment", "volume", "molecule", "analysis", "drug", "space", "set", "coordinate", "conformation", "described", "trajectory", "compute", "approach", "primitive", "section", "system", "density", "probability", "technique", "computed", "geometry", "atom", "field", "cylinder", "use", "uncertainty", "time", "given", "visualization", "q", "shape"], "document_vector": [-72.975502, 16.719614], "paragraphs": [{"paragraph_vector": [-8.039492, -59.244319], "paragraph_keywords": ["drug", "molecules", "enzyme", "molecule"]}, {"paragraph_vector": [43.323448, -47.029151], "paragraph_keywords": ["conformations", "section", "probability", "density"]}, {"paragraph_vector": [85.071784, -30.159934], "paragraph_keywords": ["uncertainty", "geometries", "volume", "dynamics"]}, {"paragraph_vector": [90.371971, -60.370498], "paragraph_keywords": ["q", "p", "interactions", "quantum"]}, {"paragraph_vector": [90.373672, -58.266277], "paragraph_keywords": ["systems", "trajectory", "time", "phase"]}, {"paragraph_vector": [97.451812, -63.062145], "paragraph_keywords": ["conformation", "molecule", "visualization", "alignment"]}, {"paragraph_vector": [37.335971, -55.743038], "paragraph_keywords": ["geometries", "geometry", "atoms", "distance"]}, {"paragraph_vector": [33.84238, -50.153549], "paragraph_keywords": ["geometries", "geometry", "alignment", "reference"]}, {"paragraph_vector": [95.364509, -62.87556], "paragraph_keywords": ["geometries", "shape", "conformation", "molecule"]}, {"paragraph_vector": [36.83303, 35.544723], "paragraph_keywords": ["primitives", "grid", "density", "geometry"]}, {"paragraph_vector": [41.858562, 38.082004], "paragraph_keywords": ["color", "rasterized", "field", "slice"]}, {"paragraph_vector": [140.52568, -63.187419], "paragraph_keywords": ["visualization", "conformations", "conformation", "time"]}, {"paragraph_vector": [130.799682, -62.83535], "paragraph_keywords": ["conformations", "time", "alignment", "movie"]}, {"paragraph_vector": [-156.084167, -83.067314], "paragraph_keywords": ["group", "molecule", "atoms", "providing"]}], "content": {}, "doi": "10.1109/ICCSA.2009.41"}, {"uri": "97", "title": "Interactive 3D Protein Structure Visualization Using Virtual Reality", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Elke Moritz", "Joerg Meyer"], "summary": "Large-scale biomedical data sets of macromolecular structures such as DNA and proteins describe highly complex biomolecular entities which often consist of thousands of atoms and residues in large 3D strands of amino acids. Various types of abstract representations are used to display these data sets, ranging from traditional ball-and-stick models to feature-presenting cartoon models. Despite these abstractions, even the most simplified representations still generate a large amount of geometry data. Useful features are often difficult to recognize in the sheer amount of detail produced by the rendering software. Comparative visualization of various structures in any type of abstract representation is helpful for understanding the relation between function and structure, and prediction of properties of newly discovered proteins which might have an impact on drug design and better treatment options for diseases. We present a case study that uses high performance workstations and a Virtual Reality display system to process large amounts of geometry data for real-time 3D exploration, superimposition, and interactive navigational tasks in a virtual reality environment.", "keywords": ["ball", "stick", "model", "visualization", "protein", "data", "feature", "structure", "exploration", "environment", "molecule", "cartoon", "navigation", "representation", "figure"], "document_vector": [148.894424, -20.186927], "paragraphs": [{"paragraph_vector": [112.407699, -19.012374], "paragraph_keywords": ["structure", "data", "structures", "protein"]}, {"paragraph_vector": [117.713874, -21.495716], "paragraph_keywords": ["structures", "structure", "protein", "models"]}, {"paragraph_vector": [118.990875, -17.458395], "paragraph_keywords": ["structures", "representation", "stick", "models"]}, {"paragraph_vector": [140.35588, 7.369063], "paragraph_keywords": ["structures", "figure", "cavetm", "data"]}, {"paragraph_vector": [151.359527, 7.006618], "paragraph_keywords": ["figure", "ball", "users", "stick"]}, {"paragraph_vector": [118.50875, -20.819343], "paragraph_keywords": ["environment", "features", "structure", "barrel"]}, {"paragraph_vector": [133.357864, 31.579877], "paragraph_keywords": ["award", "laboratory", "civl", "national"]}], "content": {}, "doi": "10.1109/APRASC.2004.1422503"}, {"uri": "98", "title": "Vials: Visualizing Alternative Splicing of Genes", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Hendrik Strobelt", "Bilal Alsallakh", "Joseph Botros", "Brant Peterson", "Mark Borowsky", "Hanspeter Pfister"], "summary": "Alternative splicing is a process by which the same DNA sequence is used to assemble different proteins, called protein isoforms. Alternative splicing works by selectively omitting some of the coding regions (exons) typically associated with a gene. Detection of alternative splicing is difficult and uses a combination of advanced data acquisition methods and statistical inference. Knowledge about the abundance of isoforms is important for understanding both normal processes and diseases and to eventually improve treatment through targeted therapies. The data, however, is complex and current visualizations for isoforms are neither perceptually efficient nor scalable. To remedy this, we developed Vials, a novel visual analysis tool that enables analysts to explore the various datasets that scientists use to make judgments about isoforms: the abundance of reads associated with the coding regions of the gene, evidence for junctions, i.e., edges connecting the coding regions, and predictions of isoform frequencies. Vials is scalable as it allows for the simultaneous analysis of many samples in multiple groups. Our tool thus enables experts to (a) identify patterns of isoform abundance in groups of samples and (b) evaluate the quality of the data. We demonstrate the value of our tool in case studies using publicly available datasets.", "keywords": ["splicing", "dot", "data", "abundance", "group", "value", "analysis", "multivariate", "sample", "graph", "gene", "expression", "junction", "genome", "support", "isoform", "shown", "vial", "information", "base", "collaborator", "figure", "case", "attribute", "plot", "example", "position", "rna", "design", "edge", "type", "show", "exon", "sashimi", "use", "view", "isoforms", "visualization", "task"], "document_vector": [-68.232719, 87.567947], "paragraphs": [{"paragraph_vector": [-119.726188, 1.400695], "paragraph_keywords": ["rna", "isoforms", "mrna", "university"]}, {"paragraph_vector": [-120.722976, 3.545465], "paragraph_keywords": ["data", "analysis", "strands", "gene"]}, {"paragraph_vector": [-117.401481, 2.482543], "paragraph_keywords": ["genes", "strand", "exons", "sequence"]}, {"paragraph_vector": [-120.02143, 1.742498], "paragraph_keywords": ["exon", "exons", "expression", "reads"]}, {"paragraph_vector": [-119.864639, 2.427144], "paragraph_keywords": ["data", "isoform", "isoforms", "genome"]}, {"paragraph_vector": [-119.607048, 2.40039], "paragraph_keywords": ["data", "isoforms", "graph", "nodes"]}, {"paragraph_vector": [-119.979179, 2.520171], "paragraph_keywords": ["isoforms", "data", "isoform", "effect"]}, {"paragraph_vector": [-121.112586, 4.315843], "paragraph_keywords": ["tasks", "data", "identify", "isoform"]}, {"paragraph_vector": [-120.745689, 2.778468], "paragraph_keywords": ["plots", "sashimi", "support", "junction"]}, {"paragraph_vector": [-120.414184, 2.935747], "paragraph_keywords": ["junctions", "sashimi", "isoforms", "plots"]}, {"paragraph_vector": [-128.433166, -36.265144], "paragraph_keywords": ["attributes", "node", "data", "layout"]}, {"paragraph_vector": [-118.20198, 1.241859], "paragraph_keywords": ["views", "use", "view", "plots"]}, {"paragraph_vector": [-121.789863, 2.395322], "paragraph_keywords": ["view", "exons", "data", "length"]}, {"paragraph_vector": [-119.613449, 3.217984], "paragraph_keywords": ["exon", "plots", "dot", "isoform"]}, {"paragraph_vector": [-118.275062, 2.290608], "paragraph_keywords": ["group", "samples", "coordinates", "expression"]}, {"paragraph_vector": [-119.212585, 2.7976], "paragraph_keywords": ["junction", "data", "figure", "samples"]}, {"paragraph_vector": [-118.39949, 0.976212], "paragraph_keywords": ["edge", "exon", "design", "exons"]}, {"paragraph_vector": [-119.879623, 2.512767], "paragraph_keywords": ["exons", "isoform", "exon", "figure"]}, {"paragraph_vector": [-118.815536, 3.520296], "paragraph_keywords": ["edge", "data", "case", "studies"]}, {"paragraph_vector": [-117.484298, 3.089044], "paragraph_keywords": ["data", "expression", "samples", "exon"]}, {"paragraph_vector": [-119.796707, 3.890069], "paragraph_keywords": ["exon", "junction", "isoform", "gbm"]}, {"paragraph_vector": [-120.066444, 3.98787], "paragraph_keywords": ["isoform", "data", "junction", "abundance"]}, {"paragraph_vector": [-119.009788, 1.376161], "paragraph_keywords": ["vials", "data", "exons", "expected"]}, {"paragraph_vector": [-119.122428, 1.389548], "paragraph_keywords": ["data", "samples", "view", "exons"]}, {"paragraph_vector": [-119.48291, 2.328345], "paragraph_keywords": ["data", "vials", "samples", "isoform"]}, {"paragraph_vector": [-154.227371, 11.758106], "paragraph_keywords": ["institutes", "health"]}], "content": {}, "doi": "10.1109/TVCG.2006.115"}, {"uri": "99", "title": "A Microgrid Energy Management System and Risk Management Under an Electricity Market Environment", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["JINGSHUANG SHEN", "CHUANWEN JIANG", "YANGYANG LIU", "XU WANG"], "summary": "This paper presents a novel energy-management method for a microgrid that includes renewable energy, diesel generators, battery storage, and various loads. We assume that the microgrid takes part in a pool market and responds actively to the electricity price to maximize its profit by scheduling its controllable resources. To address various uncertainties, a risk-constrained scenario-based stochastic programming framework is proposed using the conditional value at risk method. The designed model is solved by two levels of stochastic optimization methods. One level of optimization is to submit optimal hourly bids to the day-ahead market under the forecast data. The other level of optimization is to determine the optimal scheduling using the scenario-based stochastic data of the uncertain resources. The proposed energy management system is not only beneficial for the microgrid and customers, but also applies the microgrid aggregator and virtual power plant. The results are shown to prove the validity of the proposed framework. INDEX TERMS Controllable load, smart grid, energymanagement, electricity market, microgrid, renewable energy, risk management, stochastic optimization.", "keywords": ["response", "p", "scenario", "cost", "paper", "optimization", "period", "represent", "power", "microgrid", "profit", "system", "demand", "management", "based", "energy", "wind", "s", "represents", "load", "grid", "cvar", "battery", "t", "risk", "max"], "document_vector": [-57.616096, 55.175689], "paragraphs": [{"paragraph_vector": [-140.223953, 9.090497], "paragraph_keywords": ["wind", "speed", "output", "power"]}, {"paragraph_vector": [-142.356826, 7.942656], "paragraph_keywords": ["risk", "cvar", "scenarios", "\u03b1"]}, {"paragraph_vector": [-142.445953, 10.296402], "paragraph_keywords": ["microgrid", "represents", "t", "s"]}, {"paragraph_vector": [-142.052703, 10.588038], "paragraph_keywords": ["t", "s", "power", "p"]}, {"paragraph_vector": [-141.755416, 9.767985], "paragraph_keywords": ["t", "represent", "batteries", "s"]}, {"paragraph_vector": [-141.248184, 9.375296], "paragraph_keywords": ["t", "represents", "load", "end"]}, {"paragraph_vector": [-140.32846, 9.790859], "paragraph_keywords": ["forecast", "units", "load", "shown"]}, {"paragraph_vector": [-139.890518, 9.457675], "paragraph_keywords": ["power", "market", "forecast", "microgrid"]}, {"paragraph_vector": [-140.777404, 10.452841], "paragraph_keywords": ["response", "demand", "power", "line"]}, {"paragraph_vector": [-141.483718, 9.100989], "paragraph_keywords": ["microgrid", "management", "energy", "confidence"]}, {"paragraph_vector": [-142.15599, 10.570662], "paragraph_keywords": ["microgrid", "energy", "management", "ieee"]}, {"paragraph_vector": [-141.447296, 10.874927], "paragraph_keywords": [".", "microgrid", "energy", "demand"]}, {"paragraph_vector": [-141.230941, 9.888121], "paragraph_keywords": ["shanghai", "university", "degree", "engineering"]}], "content": {}, "doi": "10.1109/CMPSAC.2004.1342683"}, {"uri": "100", "title": "Novel Visualization Methods for Protein Data", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Shahzad Mumtaz", "Ian T. Nabney", "Darren Flower"], "summary": "Visualization of high-dimensional data has always been a challenging task. Here we discuss and propose variants of non-linear data projection methods (Generative Topographic Mapping (GTM) and GTM with simultaneous feature saliency (GTM-FS)) that are adapted to be effective on very highdimensional data. The adaptations use log space values at certain steps of the Expectation Maximization (EM) algorithm and during the visualization process. We have tested the proposed algorithms by visualizing electrostatic potential data for Major Histocompatibility Complex (MHC) class-I proteins. The experiments show that the variation in the original version of GTM and GTM-FS worked successfully with data of more than 2000 dimensions and we compare the results with other linear/nonlinear projection methods: Principal Component Analysis (PCA), Neuroscale (NSC) and Gaussian Process Latent Variable Model (GPLVM).", "keywords": ["data", "parameter", "log", "gene", "space", "m", "shown", "algorithm", "point", "calculated", "probability", "f", "based", "gtm", "component", "represents", "mixture", "feature", "region", "function", "visualization"], "document_vector": [170.721099, 44.763275], "paragraphs": [{"paragraph_vector": [-67.067176, -10.846165], "paragraph_keywords": ["data", "space", "probability", "distribution"]}, {"paragraph_vector": [-68.507522, -10.539148], "paragraph_keywords": ["gtm", "mixture", "component", "m"]}, {"paragraph_vector": [-70.958007, -5.891063], "paragraph_keywords": ["log", "space", "data", "feature"]}, {"paragraph_vector": [-70.150962, -6.190574], "paragraph_keywords": ["calculated", "feature", "matrix", "space"]}, {"paragraph_vector": [-75.210952, -31.805664], "paragraph_keywords": ["data", "gene", "visualization", "matlab"]}, {"paragraph_vector": [-114.25798, -1.545263], "paragraph_keywords": ["protein", "gene", "region", "considered"]}, {"paragraph_vector": [-75.357833, -18.955846], "paragraph_keywords": ["gtm", "identify", "dataset", "fs"]}, {"paragraph_vector": [-81.315727, -1.586736], "paragraph_keywords": ["classes", "gtm", "kl", "space"]}, {"paragraph_vector": [-79.758781, -12.977322], "paragraph_keywords": ["gtm", "alleles", "gene", "visualization"]}], "content": {}, "doi": "10.1109/IEMBS.2007.4352516"}, {"uri": "101", "title": "Molecular Dynamics Simulation of Amorphous SiO2 Fracture", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Aaron Knoll", "Joe Insley", "Michael E. Papka", "Ken-ichi Nomura", "Rajiv K. Kalia", "Aiichiro Nakano", "Priya Vashishta"], "summary": "This video demonstrates output of a large scale molecular dynamics computation of 4.7 million atoms, simulating glass fracture over the course of over a nanosecond. Postprocessed data consists of 500 frames at 3 GB per frame. We generate approximate charge density volume data to highlight the overall structure. Rendering is performed using Nanovol, a volume raycasting tool for computational chemistry. Keywords-molecular dynamics; computational chemistry; charge density; volume visualization; ball and stick", "keywords": ["computing", "stick", "data", "volume", "resolution", "code", "time", "charge", "geometry", "visualization", "crack", "simulation"], "document_vector": [19.62219, -64.287353], "paragraphs": [{"paragraph_vector": [52.816234, -12.107695], "paragraph_keywords": ["data", "volume", "time", "charge"]}, {"paragraph_vector": [83.235458, 51.663372], "paragraph_keywords": ["visualization", "video", "office", "ball"]}], "content": {}, "doi": "10.1109/CBMS.2012.6266326"}, {"uri": "102", "title": "WaveMap: Interactively Discovering Features From Protein Flexibility Matrices Using Waveletbased Visual Analytics", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Scott Barlowe", "Yujie Liu", "Jing Yang", "Dennis R. Livesay", "Donald J. Jacobs", "James Mottonen", "Deeptak Verma"], "summary": "The knowledge gained from biology datasets can streamline and speed-up pharmaceutical development. However, computational models generate so much information regarding protein behavior that large-scale analysis by traditional methods is almost impossible. The volume of data produced makes the transition from data to knowledge difficult and hinders biomedical advances. In this work, we present a novel visual analytics approach named WaveMap for exploring data generated by a protein flexibility model. WaveMap integrates wavelet analysis, visualizations, and interactions to facilitate the browsing, feature identification, and comparison of protein attributes represented by two-dimensional plots. We have implemented a fully working prototype of WaveMap and illustrate its usefulness through expert evaluation and an example scenario.", "keywords": ["data", "interest", "exploration", "value", "allows", "analysis", "parameter", "flexibility", "set", "wavelet", "protein", "row", "difference", "approach", "decomposition", "result", "researcher", "behavior", "figure", "plot", "change", "scientist", "level", "knowledge", "wavemap", "model", "feature", "domain", "user", "column", "function", "visualization", "window", "residue", "task"], "document_vector": [-148.343063, 42.14783], "paragraphs": [{"paragraph_vector": [-68.087486, -68.621795], "paragraph_keywords": ["protein", "model", "models", "flexibility"]}, {"paragraph_vector": [-62.841106, -66.982505], "paragraph_keywords": ["protein", "energy", "function", "changes"]}, {"paragraph_vector": [-71.372123, -60.64315], "paragraph_keywords": ["flexibility", "protein", "residue", "change"]}, {"paragraph_vector": [-73.830284, -62.469627], "paragraph_keywords": ["protein", "flexibility", "plots", "bmpg"]}, {"paragraph_vector": [-76.402328, -66.226913], "paragraph_keywords": ["parameter", "change", "residues", "model"]}, {"paragraph_vector": [-73.858222, -61.985729], "paragraph_keywords": ["flexibility", "sets", "data", "clamp"]}, {"paragraph_vector": [-68.1688, -51.099216], "paragraph_keywords": ["data", "flexibility", "analysis", "users"]}, {"paragraph_vector": [-69.790626, -41.207103], "paragraph_keywords": ["data", "lifting", "approach", "analysis"]}, {"paragraph_vector": [-68.905006, -59.046955], "paragraph_keywords": ["data", "components", "plots", "rows"]}, {"paragraph_vector": [-71.572029, -57.896327], "paragraph_keywords": ["data", "change", "rows", "columns"]}, {"paragraph_vector": [-73.466178, -63.753402], "paragraph_keywords": ["data", "displays", "feature", "features"]}, {"paragraph_vector": [-70.497413, -47.358432], "paragraph_keywords": ["plots", "plot", "users", "layout"]}, {"paragraph_vector": [-75.627388, -60.396232], "paragraph_keywords": ["feature", "plot", "features", "window"]}, {"paragraph_vector": [-76.729034, -60.229076], "paragraph_keywords": ["analysis", "feature", "exploration", "researchers"]}, {"paragraph_vector": [-69.621253, -62.499809], "paragraph_keywords": ["set", "plot", "features", "flexibility"]}, {"paragraph_vector": [-72.773963, -61.625804], "paragraph_keywords": ["data", "area", "column", "approach"]}, {"paragraph_vector": [-72.467483, -62.465236], "paragraph_keywords": ["data", "wavelet", "approach", "feature"]}, {"paragraph_vector": [-70.236152, -34.684768], "paragraph_keywords": ["including", "custom", "techniques", "designed"]}], "content": {}, "doi": "10.1109/CLEOE-EQEC.2019.8872994"}, {"uri": "103", "title": "DIVE: A Graph-Based Visual- Analytics Framework for Big Data", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Steven J. Rysavy", "Dennis Bromley", "Valerie Daggett"], "summary": "Class", "keywords": ["university", "runtime", "streaming", "inheritance", "data", "structure", "value", "api", "daggett", "pipeline", "analysis", "scripting", "simulation", "capability", "plug", "functionality", "method", "backing", "protein", "datanode", "sql", "research", "approach", "object", "datanodes", "dive", "code", "example", ".", "datasets", "tool", "library", "type", "dataedges", "use", "software", "ontology", "instance", "time", "property", "user", "visualization", "dynameomics"], "document_vector": [19.494245, 13.346857], "paragraphs": [{"paragraph_vector": [176.220443, -32.86005], "paragraph_keywords": ["data", "dive", "stream", "exist"]}, {"paragraph_vector": [177.03337, -36.993381], "paragraph_keywords": ["inheritance", "data", "properties", "runtime"]}, {"paragraph_vector": [176.857147, -35.891433], "paragraph_keywords": ["data", "dive", "typing", "datanodes"]}, {"paragraph_vector": [177.661972, -35.944068], "paragraph_keywords": ["data", "dive", "ontology", "ontologies"]}, {"paragraph_vector": [-168.531661, -40.160263], "paragraph_keywords": ["pipeline", "data", "plug", "ontologies"]}, {"paragraph_vector": [173.098815, -31.977457], "paragraph_keywords": ["value", "pipeline", "proteins", "dive"]}, {"paragraph_vector": [106.294219, -48.896965], "paragraph_keywords": ["protein", "structures", "data", "simulations"]}, {"paragraph_vector": [123.121932, -51.894721], "paragraph_keywords": ["protein", "languages", "dive", "."]}, {"paragraph_vector": [178.756011, -37.344867], "paragraph_keywords": ["object", "data", "library", "datanodes"]}, {"paragraph_vector": [172.159866, -35.801788], "paragraph_keywords": ["dive", "object", "data", "runtime"]}, {"paragraph_vector": [169.182937, -33.509151], "paragraph_keywords": ["scripting", "\u00b5scripting", "dive", "plug"]}, {"paragraph_vector": [176.070388, -34.845069], "paragraph_keywords": ["sql", "data", "backing", "store"]}, {"paragraph_vector": [171.434646, -32.947799], "paragraph_keywords": ["data", "set", "streaming", "dynameomics"]}, {"paragraph_vector": [176.827789, -36.321079], "paragraph_keywords": ["datanode", "dn", "api", "data"]}, {"paragraph_vector": [173.020324, -32.714927], "paragraph_keywords": ["data", "dive", "structures", "protein"]}, {"paragraph_vector": [174.445404, -35.980785], "paragraph_keywords": ["data", "sql", "dive", "mscript"]}, {"paragraph_vector": [173.429962, -34.235389], "paragraph_keywords": ["data", "research", ".", "chimera"]}, {"paragraph_vector": [-132.709472, -34.900077], "paragraph_keywords": [".", "university", "research", "protein"]}, {"paragraph_vector": [-150.671234, -21.882244], "paragraph_keywords": ["university", "protein", "dynamics", "folding"]}], "content": {}, "doi": "10.1093/jmicro/dfs043"}, {"uri": "104", "title": "Visualization of protein-drug interactions for the analysis of lung cancer drug resistance", "timestamp": "2020", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Rizwan Qureshi", "Mengxu Zhu"], "summary": "Non-small cell lung cancer (NSCLC) caused by mutation of the epidermal growth factor receptor (EGFR) is a major cause of death worldwide. Tyrosine kinase inhibitors (TKIs) of EGFR have been developed and show promising results at the initial stage of therapy. However, in most cases, their efficacy becomes limited due to the emergence of secondary mutations causing drug resistance after about a year. In this work, we investigated the mechanism of drug resistance due to these mutations. We performed molecular dynamics (MD) simulations of EGFR-drug interactions to obtain Euclidean distance and binding free energy values to analyse drug resistance and visualize drug-protein interactions. A PCA-based method is proposed to find normal, rigid, flexible, and critical residues. We have established a systematic method for the visualization of protein-drug interactions, which provides an effective framework for the analysis of drug resistance in lung cancer at the atomic level.", "keywords": ["structure", "md", "distance", "drug", "egfr", "m", "protein", "resistance", "contact", "matrix", "dimer", "mutant", "system", "figure", "based", "map", "binding", "atom", "energy", "wt", "residue"], "document_vector": [-50.429851, 43.173248], "paragraphs": [{"paragraph_vector": [-16.287673, -58.587345], "paragraph_keywords": ["drug", "response", "resistance", "dynamics"]}, {"paragraph_vector": [-26.398178, -44.137462], "paragraph_keywords": ["drug", "energy", "egfr", "structure"]}, {"paragraph_vector": [161.839233, -83.462699], "paragraph_keywords": ["matrix", "atoms", "md", "binding"]}, {"paragraph_vector": [97.508049, -81.272537], "paragraph_keywords": ["residues", "distance", "pcs", "matrix"]}, {"paragraph_vector": [17.39907, -66.373008], "paragraph_keywords": ["distance", "maps", "interest", "contact"]}, {"paragraph_vector": [-13.432059, -52.609344], "paragraph_keywords": ["drug", "m", "binding", "resistance"]}, {"paragraph_vector": [-7.672031, -69.491584], "paragraph_keywords": ["drug", "m", "figure", "shown"]}, {"paragraph_vector": [-6.034132, -62.765724], "paragraph_keywords": ["data", "value", "mutants", "ieee"]}], "content": {}, "doi": "10.1109/MCG.2012.66"}, {"uri": "105", "title": "Visualizing the Evolution of Bioinformatics: Chronological Networks Created by Textual Analysis of Intelligent Systems for Molecular Biology 1998\u20132006", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Takao Furukawa", "Kumi Okuwada", "Kazuhiro Hayashi"], "summary": "This study aimed to uncover the evolutionary processes behind the developments in the field of bioinformatics by analyzing prestigious conference presentations reflecting the prominent research in the field. Bioinformatics was selected as a representative interdisciplinary research field. Textual analyses on the abstracts of proceedings papers produced chronological networks of conference sessions. In those networks, divergent session nodes corresponded to practical papers that proposed fundamental technologies promoting new research. Convergent session nodes corresponded to papers that combined various theories and methodologies proposed in the past. The convergent session nodes play a significant role in innovation, which brings together complementary knowledge derived from different backgrounds. A joint chronological network shows a differentiation between pure scientific research and applied research, contributed to the benefit of the general public. These findings were obtained by assessing the chronological networks that reflect future perspectives on session titles in scholarly communities.", "keywords": ["structure", "analysis", "paper", "similarity", "session", "abstract", "presented", "proceeding", "title", "protein", "health", "network", "information", "research", "mode", "publication", "topic", "process", "scientist", "knowledge", "bioinformatics", "term", "science", "ontology", "conference", "proteomics", "considered", "node"], "document_vector": [47.267879, 63.394287], "paragraphs": [{"paragraph_vector": [-126.892898, -15.856837], "paragraph_keywords": ["research", "conference", "knowledge", "clusters"]}, {"paragraph_vector": [-125.424797, -13.779898], "paragraph_keywords": ["papers", "proceedings", "publications", "conference"]}, {"paragraph_vector": [-126.024024, -14.562823], "paragraph_keywords": ["papers", "titles", "bioinformatics", "analyses"]}, {"paragraph_vector": [-82.790794, -7.803306], "paragraph_keywords": ["term", "paper", "tfidf", "w"]}, {"paragraph_vector": [-125.433364, -15.822748], "paragraph_keywords": ["sessions", "session", "papers", "base"]}, {"paragraph_vector": [-125.961692, -14.561784], "paragraph_keywords": ["session", "nodes", "edges", "research"]}, {"paragraph_vector": [-124.616447, -18.622049], "paragraph_keywords": ["nodes", "session", "research", "topics"]}, {"paragraph_vector": [-127.614585, -17.201484], "paragraph_keywords": ["research", "session", "nodes", "networks"]}, {"paragraph_vector": [-121.196678, -34.162899], "paragraph_keywords": ["session", "networks", "research", "presented"]}, {"paragraph_vector": [-126.642066, -16.00855], "paragraph_keywords": ["networks", "-", "sub", "nodes"]}, {"paragraph_vector": [-127.110504, -16.393533], "paragraph_keywords": ["research", "mode", "scientists", "health"]}, {"paragraph_vector": [-125.563629, -16.377059], "paragraph_keywords": ["research", "session", "nodes", "papers"]}, {"paragraph_vector": [-127.540153, -16.034532], "paragraph_keywords": ["papers", "proceedings", "science", "research"]}, {"paragraph_vector": [-128.183547, -17.646923], "paragraph_keywords": ["vol", "analysis", ".", "information"]}, {"paragraph_vector": [-129.241302, -15.890538], "paragraph_keywords": [".", "lin", "text", "mining"]}], "content": {}, "doi": "10.1109/IV.2012.96"}, {"uri": "106", "title": "A Process For Visualizing Disordered Molecular Data with a Case Study in Bulk Water", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kwan-Liu Ma", "Giulia Galli"], "summary": "Existing molecular visualizations do a good job of illuminating well-defined structures, but are ill-suited to visualizing subtle organization when the data is almost entirely disordered. We introduce a process to explore these subtle structures by visualizing an abstraction of the data. The process includes a quantitative verification step to mitigate the ambiguity induced by abstraction. We perform a case study using molecular dynamics simulation data of bulk water to demonstrate the efficacy of our process. The visualized abstractions show molecular behaviors which only happen amongst aggregates, and which would be occluded or indiscernible in a direct visualization. The effect is shown across multiple data sets with different temporal and thermal parameters. In conjunction with domain experts from the Chemistry department who performed the simulation, we confirm our observations using the quantitative verification step of our process.", "keywords": ["data", "volume", "value", "alignment", "molecule", "k", "simulation", "set", "vma", "dipole", "number", "figure", "step", "density", "average", "process", "water", "temperature", "feature", "region", "time", "function", "rendering", "visualization", "verification", "window", "behavior"], "document_vector": [-152.882614, -22.249464], "paragraphs": [{"paragraph_vector": [65.479461, 0.231215], "paragraph_keywords": ["rendering", "molecules", "glyphs", "data"]}, {"paragraph_vector": [97.330841, -32.292629], "paragraph_keywords": ["visualization", "data", "process", "molecule"]}, {"paragraph_vector": [153.079956, -54.375465], "paragraph_keywords": ["data", "visualization", "step", "verification"]}, {"paragraph_vector": [98.45684, -42.730041], "paragraph_keywords": ["data", "molecules", "simulations", "water"]}, {"paragraph_vector": [97.514747, -35.92205], "paragraph_keywords": ["density", "function", "window", "average"]}, {"paragraph_vector": [96.788383, -44.004333], "paragraph_keywords": ["window", "regions", "region", "molecules"]}, {"paragraph_vector": [-3.956878, 71.609085], "paragraph_keywords": ["data", "region", "visualization", "transfer"]}, {"paragraph_vector": [96.982879, -44.604351], "paragraph_keywords": ["simulation", "data", "generated", "temperature"]}, {"paragraph_vector": [94.86499, -47.130245], "paragraph_keywords": ["vma", "regions", "region", "alignment"]}, {"paragraph_vector": [6.299344, 74.542869], "paragraph_keywords": ["region", "values", "time", "volume"]}, {"paragraph_vector": [95.265327, -48.113597], "paragraph_keywords": ["molecule", "molecules", "data", "conditions"]}, {"paragraph_vector": [93.086006, -39.798904], "paragraph_keywords": ["data", "time", "k", "temperature"]}, {"paragraph_vector": [94.442619, -46.417163], "paragraph_keywords": ["time", "paths", "data", "molecules"]}, {"paragraph_vector": [99.811302, -46.826694], "paragraph_keywords": ["data", "visualization", "process", "regions"]}], "content": {}, "doi": "10.1109/BioVis.2011.6094045"}, {"uri": "107", "title": "Volume visualization for exploration of population trends in two-dimensional gel electrophoresis protein data", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ola Kristoffer \u00d8ye", "Dag Magne Ulvang", "Christian Michelsen", "Katarina M. J\u00f8rgensen", "Sigrun M. Hjelle", "Bj\u00f8rn Tore Gjertsen"], "summary": "We propose a visualization method for exploring and analyzing data from two-dimensional (2D) gel electrophoresis, a frequently used laboratory technique in proteomics for characterizing and studying protein content in biological samples from patients or cell cultures. We show how 2D gel electrophoresis data from a population can be treated as a volume with no preferred slice ordering, and present a visual analysis approach to investigate the relation between an external biological variable such as subject age and the protein patterns based on dynamic slice sorting. We calculate the Spearman rank correlation as a measure of the relation, and use this together with standard deviation as statistical transfer functions to let the explorer filter the data. We also show a method for embedding the selected external variable distribution and the statistical measures in the volume, providing a statistical context for the pattern exploration. Finally we show the applicability of our method in an example of analysis of data from real populations.", "keywords": ["spot", "data", "volume", "correlation", "analysis", "protein", "shown", "information", "relation", "approach", "pixel", "technique", "stack", "gel", "use", "region", "variable", "image", "user", "visualization", "electrophoresis"], "document_vector": [-71.157798, 82.886871], "paragraphs": [{"paragraph_vector": [-63.496662, -31.926988], "paragraph_keywords": ["analysis", "gel", "protein", "visualization"]}, {"paragraph_vector": [-64.36219, -32.9393], "paragraph_keywords": ["data", "gel", "protein", "box"]}, {"paragraph_vector": [-62.054138, -31.388134], "paragraph_keywords": ["data", "volume", "images", "spots"]}, {"paragraph_vector": [-62.51448, -32.549282], "paragraph_keywords": ["relations", "protein", "variable", "volume"]}, {"paragraph_vector": [-55.61993, -34.560894], "paragraph_keywords": ["sort", "use", "column", "volume"]}, {"paragraph_vector": [-55.908191, -34.659915], "paragraph_keywords": ["correlation", "regions", "user", "image"]}, {"paragraph_vector": [-63.789432, -34.270359], "paragraph_keywords": ["user", "shown", "variable", "data"]}, {"paragraph_vector": [-62.055511, -31.756658], "paragraph_keywords": ["gel", "electrophoresis", "methodology", "visualization"]}, {"paragraph_vector": [-67.412712, -6.51033], "paragraph_keywords": ["spots", "classification", "fab", "correlation"]}, {"paragraph_vector": [-64.195877, -35.250133], "paragraph_keywords": ["data", "visualization", "insight", "compared"]}], "content": {}, "doi": "10.1109/ISBI.2014.6867888"}, {"uri": "108", "title": "Immersive Environment for Molecular Visualization to Interaction between Research Groups Geographically Dispersed", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Moacyr Francischetti Corr\u00eaa", "Julio Zukerman Schpector", "Luis Carlos Trevelin", "Marcelo de Paiva Guimar\u00e3es"], "summary": "This paper presents an environment for molecular visualization with multi-projection and collaboration capabilities between geographically dispersed research groups. It was drafted a hardware and software architecture whose purpose is to reduce the negative effects of the delay in communication inherent longdistance networks. Keywordsmolecular visualization, virtual reality, multiprojection, communication protocol, RTEP, JAMP, EvPack, SmartBeat", "keywords": ["visualization", "delay", "group", "architecture", "server", "rtep", "reality", "information", "control", "system", "jitter", "multicast", "communication", "work", "message", "equipment", "latency", "use", "software", "rmi", "protocol", "time", "packet", "user", "environment"], "document_vector": [170.849273, -46.891647], "paragraphs": [{"paragraph_vector": [149.40419, 25.4724], "paragraph_keywords": ["delay", "equipment", "communication", "dispersed"]}, {"paragraph_vector": [155.92044, 33.555553], "paragraph_keywords": ["environment", "control", "user", "users"]}, {"paragraph_vector": [164.114639, 35.083496], "paragraph_keywords": ["jitter", "servers", "interval", "rtep"]}, {"paragraph_vector": [159.141754, 33.542137], "paragraph_keywords": ["messages", "environment", "server", "servers"]}, {"paragraph_vector": [156.768646, 33.046009], "paragraph_keywords": ["environment", "message", "multicast", "communication"]}, {"paragraph_vector": [167.743698, 35.259059], "paragraph_keywords": ["latency", "icmp", "rtep", "measured"]}, {"paragraph_vector": [158.46231, 35.440769], "paragraph_keywords": ["rtep", "rmi", "tables", "environment"]}, {"paragraph_vector": [136.187286, 16.066101], "paragraph_keywords": ["visualization", "environment", ".", "reality"]}], "content": {}, "doi": "10.1109/TCBB.2019.2892099"}, {"uri": "109", "title": "Elucidating Which Pairwise Mutations Affect Protein Stability: An Exhaustive Big Data Approach", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Nicholas Majeske", "Filip Jagodzinski"], "summary": "The specific sequence of amino acids in a polypeptide chain dictates the three dimensional structure, and hence function, of a protein. Mutagenesis experiments on physical proteins involving amino acid substitutions provide insights enabling pharmaceutical companies to design medicines to combat a variety of debilitating diseases. However such wet lab work is prohibitive, because even studying the effects of a single mutation may require weeks of work. Computational approaches for performing exhaustive screens of the effects of single mutations have been developed, but methods for conducting a systematic, exhaustive screen of the effects of all multiple mutations are not available due to the large number of mutant protein structures that would need to be analyzed. In this work we motivate and demonstrate a proof of concept approach for conducting in silico experiments in which we generate all possible mutant structures with 2 amino acid substitutions for three proteins with 46, 67, and 99 residues; for the largest protein we in silico generate 1,751,211 mutants. We leverage an efficient combinatorial algorithm to assess the effects of the mutations among the mutant protein structures. We also produce heat maps for several mutation metrics to facilitate identifying which pairs of amino acid in a protein have the greatest impact on protein stability based on how those amino acid substitutions affect the protein\u2019s flexibility. Keywords\u2014protein, multiple mutations, big data, visualization", "keywords": ["file", "cluster", "mutated", "lrc", "protein", "rigidity", "structure", "pdb", "mutation", "analysis", "metric", "acid", "mutant", "type", "residue", "figure"], "document_vector": [-108.167678, 40.253295], "paragraphs": [{"paragraph_vector": [-87.813728, -57.263076], "paragraph_keywords": ["analysis", "rigidity", "data", "protein"]}, {"paragraph_vector": [-90.096595, -54.39682], "paragraph_keywords": ["protein", "acids", "acid", "mutants"]}, {"paragraph_vector": [-139.170501, 63.835079], "paragraph_keywords": ["rigidity", "protein", "lrc", "analysis"]}, {"paragraph_vector": [-85.777404, -58.641052], "paragraph_keywords": ["cluster", "protein", "lrc", "mutations"]}, {"paragraph_vector": [-88.921089, -57.102504], "paragraph_keywords": ["mutants", "acids", "protein", "metric"]}, {"paragraph_vector": [-87.373443, -56.352687], "paragraph_keywords": ["residues", "pairs", "protein", "map"]}, {"paragraph_vector": [-89.326507, -55.268913], "paragraph_keywords": ["mutants", "lrc", "residues", "mutated"]}, {"paragraph_vector": [-87.014389, -57.804843], "paragraph_keywords": ["residues", "protein", "figure", "mutated"]}, {"paragraph_vector": [-85.189765, -59.223609], "paragraph_keywords": ["mutated", "protein", "scores", "lrc"]}, {"paragraph_vector": [-87.026916, -56.884311], "paragraph_keywords": ["protein", "mutants", "lrc", "mutation"]}, {"paragraph_vector": [-87.17707, -57.371234], "paragraph_keywords": ["pipeline", "mutations", "protein", "predictions"]}], "content": {}, "doi": "10.1109/CISS.2006.286395"}, {"uri": "110", "title": "A Hybrid Visualization System for Molecular Models", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Charles Marion", "Joachim Pouderoux", "Julien Jomier", "Sebastien Jourdain", "Marcus Hanwell", "Utkarsh Ayachit"], "summary": "Nowadays, 3D visualization is part of our everyday lives and remains an important tool for understanding complex structures, dynamic simulations, and research discoveries. In the meantime, the complexity of the visualization often requires specific hardware, including visualization clusters, as well as dedicated software tools. Several efforts, such as WebGL and remote visualization, have been pushing visualization to the web browser. However, technical limitations have constrained the adoption of web browsers as a means of rendering 3D datasets. To surpass these limitations, prospective action has been recently initiated with the development of a web application based on two innovative technologies: WebGL and ParaViewWeb. This application combines the benefits of local and remote rendering, therefore allowing a user to visualize small to massive 3D molecular datasets on a large number of devices. To demonstrate this approach a prototype has been developed, and in this paper we present the hybrid architecture of the proposed system, the initial implementation experiment, and a comparison with current technologies. Finally, we discuss the future work and potential improvements of the application. CR Categories: I.3.6 [Methodology and Techniques]: ;\u2014 [I.3.8]: Applications\u2014; I.3.m [Performance]: ;\u2014 [Interactive 3D graphics for mobile devices]: ;\u2014 [Remote rendering and streaming]: ;\u2014;", "keywords": ["web", "data", "molecule", "order", "et", "server", "mapper", "system", "webgl", "based", "source", "performance", "client", "technology", "component", "instance", "al", "user", "rendering", "visualization"], "document_vector": [167.882369, -63.456386], "paragraphs": [{"paragraph_vector": [81.012351, -21.235198], "paragraph_keywords": ["visualization", "tools", "et", "rendering"]}, {"paragraph_vector": [137.116134, 38.594692], "paragraph_keywords": ["web", "visualization", "server", "components"]}, {"paragraph_vector": [147.019042, 41.796169], "paragraph_keywords": ["rendering", "client", "based", "visualization"]}, {"paragraph_vector": [146.241333, 40.315235], "paragraph_keywords": ["molecule", "protocol", "performance", "molecules"]}, {"paragraph_vector": [83.911239, 6.277167], "paragraph_keywords": ["mapper", "rendering", "vtk", "gpu"]}, {"paragraph_vector": [93.076896, 43.285945], "paragraph_keywords": ["data", "protein", "datasets", "platform"]}, {"paragraph_vector": [143.814071, 37.888412], "paragraph_keywords": ["visualization", "system", "technologies", "performance"]}, {"paragraph_vector": [141.21463, -16.289413], "paragraph_keywords": ["types", "shown", "devices", "data"]}], "content": {}, "doi": "10.1109/IJCNN.2013.6707001"}, {"uri": "111", "title": "Remote Visualization of Dynamic Molecular Data using WebGL", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Finian Mwalongo", "Michael Krone", "Michael Becher", "Guido Reina", "Thomas Ertl"], "summary": "Visualization of dynamic data from molecular dynamics simulations is crucial for understanding the functioning of molecules. Many existing visualization tools have mainly focused on supporting a single user working on a desktop computer. Technical advancements in browser features increase the potential for the development of web-based collaborative visualization tools. Although web-based molecular viewers already exist, their support for dynamic molecular data at interactive rates is lacking. To address this gap, we present an efficient web application for visualization of dynamic molecular data using WebGL that exploits HTML5 technologies like WebSockets and Web Workers. We demonstrate the feasibility to visualize large dynamic molecular data in the browser at interactive frame rates. Moreover, the application can be employed for remote simulation monitoring on mobile devices. Our approach allows scientists at arbitrary locations to concurrently visually analyze the same data. The interactive parametrization of the visualization can be shared among multiple clients, providing the basis for collaborative research. CR Categories: I.3.7 [Computer Graphics]: Three-Dimensional Graphics and Realism\u2014Raytracing I.3.6 [Computer Graphics]: Methodology and Techniques\u2014Graphics data structures and data types; J.3 [Computer Applications]: Life and Medical Sciences\u2014 Biology and Genetics.", "keywords": ["web", "buffer", "data", "et", "application", "server", "extension", "simulation", "khronos", "set", "transfer", "space", "running", "memory", "shading", "triangle", "number", "object", "depth", "websocket", "color", "technique", "webgl", "shader", "based", "communication", "browser", "performance", "frame", "client", "size", "atom", "desktop", "pas", "request", "graphic", "gpu", "use", "processing", "time", "image", "rendering", "visualization", "thread", "javascript"], "document_vector": [-175.670074, -56.384731], "paragraphs": [{"paragraph_vector": [138.015945, 41.734027], "paragraph_keywords": ["data", "simulation", "browser", "offer"]}, {"paragraph_vector": [141.329986, 41.533821], "paragraph_keywords": ["based", "data", "web", "application"]}, {"paragraph_vector": [134.921127, 39.840877], "paragraph_keywords": ["rendering", "server", "data", "client"]}, {"paragraph_vector": [143.797332, 40.935962], "paragraph_keywords": ["data", "client", "server", "visualization"]}, {"paragraph_vector": [149.992065, 40.002902], "paragraph_keywords": ["thread", "data", "client", "rendering"]}, {"paragraph_vector": [148.369873, 41.720203], "paragraph_keywords": ["client", "server", "data", "connection"]}, {"paragraph_vector": [82.008872, 57.050987], "paragraph_keywords": ["data", "vertices", "index", "object"]}, {"paragraph_vector": [59.209644, 55.151687], "paragraph_keywords": ["camera", "use", "atoms", "triangle"]}, {"paragraph_vector": [33.862586, 75.621772], "paragraph_keywords": ["point", "depth", "webgl", "data"]}, {"paragraph_vector": [67.784797, 59.738204], "paragraph_keywords": ["shading", "framebuffer", "space", "precision"]}, {"paragraph_vector": [43.185443, 80.062355], "paragraph_keywords": ["depth", "texture", "pass", "shading"]}, {"paragraph_vector": [151.619934, 60.018611], "paragraph_keywords": ["data", "rendering", "client", "server"]}, {"paragraph_vector": [150.522918, 51.332859], "paragraph_keywords": ["data", "transfer", "client", "time"]}, {"paragraph_vector": [145.766891, 74.028205], "paragraph_keywords": ["data", "image", "sets", "shading"]}, {"paragraph_vector": [139.764343, 37.625419], "paragraph_keywords": ["data", "visualization", "computing", "server"]}, {"paragraph_vector": [111.840904, 28.586395], "paragraph_keywords": ["visualization", "computer", "graphics", "webgl"]}, {"paragraph_vector": [120.407379, 38.549423], "paragraph_keywords": ["webgl", "visualization", "graphics", "web"]}], "content": {}, "doi": "10.1109/MCDM.2014.7007198"}, {"uri": "112", "title": "Screen-Space Normal Distribution Function Caching for Consistent Multi-Resolution Rendering of Large Particle Data", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Mohamed Ibrahim", "Patrick Wickenh\u00e4user", "Peter Rautek", "Guido Reina", "Markus Hadwiger"], "summary": "Molecular dynamics (MD) simulations are crucial to investigating important processes in physics and thermodynamics. The simulated atoms are usually visualized as hard spheres with Phong shading, where individual particles and their local density can be perceived well in close-up views. However, for large-scale simulations with 10 million particles or more, the visualization of large fields-of-view usually suffers from strong aliasing artifacts, because the mismatch between data size and output resolution leads to severe under-sampling of the geometry. Excessive super-sampling can alleviate this problem, but is prohibitively expensive. This paper presents a novel visualization method for large-scale particle data that addresses aliasing while enabling interactive high-quality rendering. We introduce the novel concept of screen-space normal distribution functions (S-NDFs) for particle data. S-NDFs represent the distribution of surface normals that map to a given pixel in screen space, which enables high-quality re-lighting without re-rendering particles. In order to facilitate interactive zooming, we cache S-NDFs in a screen-space mipmap (S-MIP). Together, these two concepts enable interactive, scale-consistent re-lighting and shading changes, as well as zooming, without having to re-sample the particle data. We show how our method facilitates the interactive exploration of real-world large-scale MD simulation data in different scenarios.", "keywords": ["\u03c9", "output", "data", "radiance", "exploration", "ndfs", "allows", "space", "lighting", "transfer", "set", "angle", "m", "corresponding", "particle", "approach", "number", "brdf", "screen", "mip", "scale", "pixel", "distribution", "fig", "ray", "ndf", "quality", "vector", "laser", "projected", "resolution", "direction", "surface", "computed", "performance", "sampling", "cache", "footprint", "s", "tile", "feature", "given", "time", "material", "image", "user", "function", "rendering", "visualization", "casting", "page"], "document_vector": [-35.673393, -60.430397], "paragraphs": [{"paragraph_vector": [1.786971, 86.869499], "paragraph_keywords": ["distribution", "probability", "space", "screen"]}, {"paragraph_vector": [-115.72528, 89.315925], "paragraph_keywords": ["footprint", "\u03c9", "pixel", "s"]}, {"paragraph_vector": [61.596374, 89.823135], "paragraph_keywords": ["bin", "computations", "projected", "footprint"]}, {"paragraph_vector": [33.25962, 79.57846], "paragraph_keywords": ["ndfs", "surface", "vectors", "enables"]}, {"paragraph_vector": [4.848968, 82.745979], "paragraph_keywords": ["s", "space", "ndfs", "screen"]}, {"paragraph_vector": [-7.396491, 89.029113], "paragraph_keywords": ["s", "ndfs", "space", "screen"]}, {"paragraph_vector": [-161.507263, 89.407264], "paragraph_keywords": ["probability", "distribution", "defined", "projected"]}, {"paragraph_vector": [26.689386, 89.801269], "paragraph_keywords": ["footprint", "pixel", "\u03c9", "s"]}, {"paragraph_vector": [7.732877, 86.535095], "paragraph_keywords": ["bin", "projected", "angle", "computations"]}, {"paragraph_vector": [-67.663642, 89.135734], "paragraph_keywords": ["\u03c9o", "direction", "surface", "radiance"]}, {"paragraph_vector": [-126.017906, 88.310974], "paragraph_keywords": ["brdf", "corresponding", "ray", "radiance"]}, {"paragraph_vector": [90.280288, 82.410629], "paragraph_keywords": ["space", "level", "s", "texture"]}, {"paragraph_vector": [-2.858624, 89.525299], "paragraph_keywords": ["mip", "s", "tiles", "output"]}, {"paragraph_vector": [68.799522, 89.134597], "paragraph_keywords": ["s", "\u03c9o", "projected", "\u03c9"]}, {"paragraph_vector": [-36.343338, 89.396797], "paragraph_keywords": ["screen", "space", "m", "eq"]}, {"paragraph_vector": [69.111419, 89.119804], "paragraph_keywords": ["corresponding", "s", "given", "brdf"]}, {"paragraph_vector": [48.923675, 88.577774], "paragraph_keywords": ["s", "mip", "level", "space"]}, {"paragraph_vector": [164.472732, 89.955833], "paragraph_keywords": ["s", "mip", "tiles", "output"]}, {"paragraph_vector": [-29.897792, 89.196983], "paragraph_keywords": ["s", "mip", "user", "pixel"]}, {"paragraph_vector": [-155.514221, 88.838439], "paragraph_keywords": ["s", "quality", "ray", "casting"]}, {"paragraph_vector": [98.163543, 67.938812], "paragraph_keywords": ["data", "s", "laser", "changing"]}, {"paragraph_vector": [97.846908, 70.621437], "paragraph_keywords": ["features", "block", "structures", "data"]}, {"paragraph_vector": [92.263099, 58.451889], "paragraph_keywords": ["features", "data", "exploration", "transfer"]}, {"paragraph_vector": [59.006889, 74.657279], "paragraph_keywords": ["particle", "transfer", "function", "rendering"]}, {"paragraph_vector": [96.957626, 89.792594], "paragraph_keywords": ["s", "complexity", "particle", "pixel"]}, {"paragraph_vector": [-19.290174, 88.964935], "paragraph_keywords": ["s", "user", "mip", "casting"]}, {"paragraph_vector": [175.95932, 89.581359], "paragraph_keywords": ["s", "data", "scale", "changing"]}, {"paragraph_vector": [92.376892, 69.953033], "paragraph_keywords": ["laser", "data", "block", "shock"]}, {"paragraph_vector": [9.725159, 85.934547], "paragraph_keywords": ["data", "features", "s", "copper"]}, {"paragraph_vector": [86.337898, 53.497375], "paragraph_keywords": ["transfer", "features", "function", "exploration"]}, {"paragraph_vector": [55.631481, 73.766044], "paragraph_keywords": ["particle", "material", "radius", "normals"]}, {"paragraph_vector": [-178.606521, 88.396804], "paragraph_keywords": ["performance", "ray", "s", "data"]}, {"paragraph_vector": [-142.441436, 89.916152], "paragraph_keywords": ["s", "performance", "image", "mip"]}, {"paragraph_vector": [-93.996917, 88.794143], "paragraph_keywords": ["data", "s", "particle", "number"]}, {"paragraph_vector": [96.927459, 68.858375], "paragraph_keywords": ["s", "visualization", "ndfs", "computer"]}, {"paragraph_vector": [100.645698, 41.780487], "paragraph_keywords": ["visualization", "pages", "graphics", "ray"]}, {"paragraph_vector": [109.509185, 43.335353], "paragraph_keywords": ["visualization", "based", "graphics", "pages"]}, {"paragraph_vector": [101.822547, 55.038581], "paragraph_keywords": ["visualization", "rendering", "function", "data"]}, {"paragraph_vector": [-46.548091, 86.440277], "paragraph_keywords": ["ray", "s", "performance", "number"]}, {"paragraph_vector": [-121.698875, 88.670066], "paragraph_keywords": ["s", "performance", "mip", "image"]}, {"paragraph_vector": [-28.999397, 88.731361], "paragraph_keywords": ["data", "s", "particle", "number"]}, {"paragraph_vector": [95.206466, 63.699634], "paragraph_keywords": ["s", "visualization", "ndfs", "computer"]}, {"paragraph_vector": [100.433036, 40.383701], "paragraph_keywords": ["visualization", "pages", "graphics", "ray"]}, {"paragraph_vector": [128.813079, 52.992294], "paragraph_keywords": ["visualization", "based", "graphics", "pages"]}, {"paragraph_vector": [107.987411, 55.201042], "paragraph_keywords": ["visualization", "rendering", "particle", "pages"]}], "content": {}, "doi": "10.1109/CMV.2006.3"}, {"uri": "113", "title": "Visualizing Polarization Domains in Barium Titanate", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Theresa-Marie Rhyne", "Katrin Scharnowski", "Michael Krone", "Filip Sadlo", "Philipp Beck", "Johannes Roth", "Hans-Rainer Trebin"], "summary": "The 2012 IEEE Visualization Contest\u2019s topic was the detection of phase transitions in ferroelectric materials\u2014in particular, barium titanate (BaTiO3). (For more on the contest, see the sidebar.) Ferroelectric materials\u2019 electric properties make them suitable for a variety of applications. One major application is high-permittivity capacitors. Several ferroelectric materials are promising alternatives for volatileand nonvolatile-memory devices. A dielectric material is ferroelectric if it exhibits spontaneous polarization that can be reversed by applying an external electric field. Dielectric materials that show nonlinear polarization are called paraelectrics. In all ferroelectrics, a phase transition from a paraelectric to ferroelectric state occurs when the material\u2019s temperature decreases. The visualization of phase transitions, or polarization domains in general, is a novel research field. Sebastian Grottel and his colleagues recently showed how a combination of glyph-based visualizations and isosurfaces derived from the anisotropy can reveal metal oxides\u2019 electrostatic properties. We used the particle data from the contest to derive a vector field indicating the material\u2019s polarization. By combining several visualization methods, we qualitatively estimated the polarization domains\u2019 size and spatial distribution. We also investigated the development of the material\u2019s total volume over time and of the individual atoms\u2019 oscillation about their ideal lattice positions. The domains\u2019 development corresponded with that of the total volume and the atom mobility. On the basis of these observations, we estimated the phase transition\u2019s onset. Exploratory visualization is a useful tool to gain insights into new data. So, our contest submission focused on interactive visualizations that facilitated exploratory analysis of the given dataset. We integrated our visualizations in MegaMol (www. vis.uni-stuttgart.de/megamol), a flexible framework for visualizing large, dynamic particle datasets.", "keywords": ["university", "data", "ti", "volume", "transition", "analysis", "orientation", "isosurfaces", "polarization", "magnitude", "cluster", "dipole", "research", "figure", "density", "ieee", "vector", "based", ".", "displacement", "atom", "curl", "field", "phase", "time", "domain", "material", "unit", "visualization", "cell"], "document_vector": [55.955249, -27.912706], "paragraphs": [{"paragraph_vector": [73.35662, -8.522526], "paragraph_keywords": ["phase", "visualizations", "materials", "polarization"]}, {"paragraph_vector": [68.336593, -8.412327], "paragraph_keywords": ["atoms", "cell", "unit", "data"]}, {"paragraph_vector": [81.489547, -29.614831], "paragraph_keywords": ["time", "vector", "material", "volume"]}, {"paragraph_vector": [78.351226, -34.141098], "paragraph_keywords": ["ti", "dipole", "displacement", "vector"]}, {"paragraph_vector": [58.606311, -2.630478], "paragraph_keywords": ["ti", "vectors", "density", "approach"]}, {"paragraph_vector": [44.605434, 22.648803], "paragraph_keywords": ["isosurfaces", "field", "magnitude", "vector"]}, {"paragraph_vector": [60.026058, -10.689029], "paragraph_keywords": ["field", "domains", "lic", "isosurfaces"]}, {"paragraph_vector": [74.061134, -20.382783], "paragraph_keywords": ["domains", "time", "figure", "step"]}, {"paragraph_vector": [75.584915, -17.802282], "paragraph_keywords": ["data", "simulation", "domains", "visualization"]}, {"paragraph_vector": [65.50827, 6.333066], "paragraph_keywords": ["university", "visualization", "stuttgart", "."]}, {"paragraph_vector": [132.840377, 13.316613], "paragraph_keywords": ["ieee", "content", "copyright", "m"]}], "content": {}, "doi": "10.1109/TNB.2013.2263837"}, {"uri": "114", "title": "3D VISUALIZATION OF CONCENTRATIONS FROM STOCHASTIC AGENT-BASED SIGNAL TRANSDUCTION SIMULATIONS", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Martin Falk", "Michael Klann", "Matthias Reuss", "Thomas Ertl"], "summary": "Cellular signal transduction involves a transport step from the plasma membrane towards the nucleus, during which the signaling molecules are partly deactivated in control loops. This leads to a gradient in the concentration of active signaling molecules. The low number of molecules introduces spatio-temporal fluctuations and the asymmetric cellular architecture further increases the complexity. We propose a technique to represent this pattern in a continuous three-dimensional concentration map. The local concentration is computed and visualized with volume rendering techniques at interactive frame rates and is therefore well-suited for time-dependent data. Our approach allows the transition from the nano-scale of single and discrete signaling proteins to a continuous signal on the cell level. In the application context of this paper, we employ an agent-based Monte Carlo simulation to calculate the actual particle positions depending on reaction and transport parameters in the cell. The applicability of the proposed technique is demonstrated by an investigation of the effects of different transport parameters in Mitogen-activated protein kinase (MAPK) signaling.", "keywords": ["nucleus", "data", "volume", "molecule", "signaling", "simulation", "protein", "concentration", "cytoskeleton", "diffusion", "distribution", "density", "ray", "map", "based", "mapk", "model", "transport", "signal", "visualization", "cell"], "document_vector": [-79.872589, -34.176231], "paragraphs": [{"paragraph_vector": [88.002983, -11.24177], "paragraph_keywords": ["mapk", "cell", "molecules", "volume"]}, {"paragraph_vector": [84.48471, -2.454239], "paragraph_keywords": ["density", "protein", "voxels", "slice"]}, {"paragraph_vector": [30.199079, 68.766151], "paragraph_keywords": ["cell", "volume", "nucleus", "ray"]}, {"paragraph_vector": [88.638473, -11.54406], "paragraph_keywords": ["concentration", "transport", "molecules", "signaling"]}, {"paragraph_vector": [89.580902, -11.177635], "paragraph_keywords": ["data", "signal", "nucleus", "map"]}, {"paragraph_vector": [86.920204, -3.514838], "paragraph_keywords": ["model", "signaling", "visualization", "data"]}, {"paragraph_vector": [101.118766, 11.050224], "paragraph_keywords": ["vol", "volume", "based", "biology"]}], "content": {}, "doi": "10.1109/TVCG.2007.70517"}, {"uri": "115", "title": "DECONVOLVING THE IMAGES", "timestamp": "2003", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "In researching the communication mechanisms between cells of the immune system, visualization of proteins in three dimensions can be used to determine which proteins are capable of interacting with one another at a given time by showing their spatial colocality. Volume data sets are created using digital confocal immunofluorescence microscopy. A variety of visualization approaches are then used to examine the interactions. These include volume rendering, isosurface extraction, and virtual reality. Based on our experiences, we have concluded that no single one of these approaches provides a complete solution for visualizing biological data. However, in combination, their respective strengths complement one another to provide an understanding of the data.", "keywords": ["data", "volume", "interaction", "microscope", "set", "protein", "shown", "information", "system", "understanding", "figure", "light", "animation", "technology", "deconvolution", "image", "rendering", "visualization", "cell", "sandia"], "document_vector": [-99.830299, -46.644123], "paragraphs": [{"paragraph_vector": [-37.84893, 6.051702], "paragraph_keywords": ["proteins", "cell", "microscope", "location"]}, {"paragraph_vector": [-36.87606, 8.8609], "paragraph_keywords": ["light", "image", "point", "lens"]}, {"paragraph_vector": [-3.209169, 72.247131], "paragraph_keywords": ["data", "volume", "animations", "rendering"]}, {"paragraph_vector": [116.236503, 52.331268], "paragraph_keywords": ["data", "volume", "figure", "set"]}, {"paragraph_vector": [-5.65526, 70.879837], "paragraph_keywords": ["data", "volume", "shown", "protein"]}, {"paragraph_vector": [0.823512, 72.082717], "paragraph_keywords": ["information", "visualization", "sandia", "cell"]}, {"paragraph_vector": [-154.241073, 10.30369], "paragraph_keywords": ["constantine", "kupfer", "pavlakos", "avi"]}], "content": {}, "doi": "10.1109/BIBMW.2012.6470197"}, {"uri": "116", "title": "Visual Verification and Analysis of Cluster Detection for Molecular Dynamics", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["S. Grottel", "T. Ertl"], "summary": "A current research topic in molecular thermodynamics is the condensation of vapor to liquid and the investigation of this process at the molecular level. Condensation is found in many physical phenomena, e.g. the formation of atmospheric clouds or the processes inside steam turbines, where a detailed knowledge of the dynamics of condensation processes will help to optimize energy efficiency and avoid problems with droplets of macroscopic size. The key properties of these processes are the nucleation rate and the critical cluster size. For the calculation of these properties it is essential to make use of a meaningful definition of molecular clusters, which currently is a not completely resolved issue. In this paper a framework capable of interactively visualizing molecular datasets of such nucleation simulations is presented, with an emphasis on the detected molecular clusters. To check the quality of the results of the cluster detection, our framework introduces the concept of flow groups to highlight potential cluster evolution over time which is not detected by the employed algorithm. To confirm the findings of the visual analysis, we coupled the rendering view with a schematic view of the clusters\u2019 evolution. This allows to rapidly assess the quality of the molecular cluster detection algorithm and to identify locations in the simulation data in space as well as in time where the cluster detection fails. Thus, thermodynamics researchers can eliminate weaknesses in their cluster detection algorithms. Several examples for the effective and efficient usage of our tool are presented.", "keywords": ["monomer", "dataset", "data", "group", "value", "hierarchy", "allows", "molecule", "ellipsoid", "needed", "simulation", "detection", "cluster", "evolution", "memory", "algorithm", "information", "point", "approach", "number", "section", "result", "calculated", "clustering", "color", "process", "flow", "based", "datasets", "performance", "tool", "size", "configuration", "e", "file", "use", "nucleation", "glyph", "view", "time", "user", "rendering", "visualization"], "document_vector": [-74.870903, -59.280158], "paragraphs": [{"paragraph_vector": [68.268554, -42.802875], "paragraph_keywords": ["clusters", "vapor", "size", "process"]}, {"paragraph_vector": [67.382843, -35.050937], "paragraph_keywords": ["processes", "systems", "nucleation", "molecules"]}, {"paragraph_vector": [76.333061, -57.267169], "paragraph_keywords": ["clusters", "visualization", "molecule", "cluster"]}, {"paragraph_vector": [85.759513, -32.598583], "paragraph_keywords": ["cluster", "section", "molecule", "clusters"]}, {"paragraph_vector": [106.504928, 50.958099], "paragraph_keywords": ["rendering", "datasets", "performance", "based"]}, {"paragraph_vector": [134.393569, 55.690795], "paragraph_keywords": ["visualization", "mixing", "aspects", "clusters"]}, {"paragraph_vector": [75.200889, -47.211868], "paragraph_keywords": ["clusters", "datasets", "data", "rendering"]}, {"paragraph_vector": [101.726882, 37.116161], "paragraph_keywords": ["file", "data", "hierarchy", "molecules"]}, {"paragraph_vector": [96.089736, 40.715194], "paragraph_keywords": ["hierarchy", "molecules", "coordinates", "bytes"]}, {"paragraph_vector": [102.338356, 42.785457], "paragraph_keywords": ["configuration", "time", "configurations", "datasets"]}, {"paragraph_vector": [81.394393, -58.578197], "paragraph_keywords": ["cluster", "time", "t", "molecules"]}, {"paragraph_vector": [73.219177, 46.66547], "paragraph_keywords": ["glyphs", "depth", "shaders", "color"]}, {"paragraph_vector": [80.601959, -59.266429], "paragraph_keywords": ["clusters", "cluster", "position", "calculated"]}, {"paragraph_vector": [38.578369, 1.302333], "paragraph_keywords": ["color", "clusters", "ellipsoid", "axes"]}, {"paragraph_vector": [76.252296, -47.408771], "paragraph_keywords": ["monomers", "time", "cluster", "molecules"]}, {"paragraph_vector": [77.264694, -52.072853], "paragraph_keywords": ["flow", "molecules", "group", "size"]}, {"paragraph_vector": [77.783767, -54.882541], "paragraph_keywords": ["cluster", "clusters", "dataset", "tool"]}, {"paragraph_vector": [86.040031, -46.044723], "paragraph_keywords": ["cluster", "flow", "lines", "view"]}, {"paragraph_vector": [96.501625, 37.114276], "paragraph_keywords": ["data", "memory", "loading", "table"]}, {"paragraph_vector": [81.485961, -55.982444], "paragraph_keywords": ["cluster", "clusters", "time", "calculated"]}, {"paragraph_vector": [69.447822, -54.752044], "paragraph_keywords": ["cluster", "view", "molecules", "dataset"]}, {"paragraph_vector": [75.312072, -48.708583], "paragraph_keywords": ["cluster", "results", "detection", "algorithm"]}, {"paragraph_vector": [84.59719, 34.811885], "paragraph_keywords": ["preprocessing", "visualization", "simulation", "software"]}], "content": {}, "doi": "10.1109/BIBM.2011.29"}, {"uri": "117", "title": "Case Study: An Environment for Understanding Protein Simulations Using Game Graphics", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Donna Gresh", "Frank Suits"], "summary": "We describe a visualization system designed for interactive study of proteins in the field of computational biology. Our system incorporates multiple, custom, three-dimensional and two-dimensional linked views of the proteins. We take advantage of modern commodity graphics cards, which are typically designed for games rather than scientific visualization applications, to provide instantaneous linking between views and three-dimensional interactivity on standard personal computers. Furthermore, we anticipate the usefulness of game techniques such as bump maps and skinning for scientific applications. CR Categories: I.3.6 [Computer Graphics]: Methodology and Techniques\u2014Interaction techniques", "keywords": ["structure", "api", "distance", "application", "ribbon", "create", "protein", "directx", "number", "system", "figure", "color", "plot", "surface", "based", "map", "geometry", "atom", "graphic", "use", "mouse", "pair", "view", "feature", "vertex", "environment", "visualization", "residue"], "document_vector": [175.335113, -8.562333], "paragraphs": [{"paragraph_vector": [124.195899, -55.50011], "paragraph_keywords": ["proteins", "protein", "visualization", "simulations"]}, {"paragraph_vector": [123.806976, 39.354717], "paragraph_keywords": ["graphics", "intended", "api", "visualization"]}, {"paragraph_vector": [136.508636, -11.328315], "paragraph_keywords": ["protein", "views", "application", "visualization"]}, {"paragraph_vector": [20.194602, -79.829444], "paragraph_keywords": ["protein", "structure", "ribbon", "described"]}, {"paragraph_vector": [99.366676, -84.820213], "paragraph_keywords": ["points", "distance", "structure", "pairs"]}, {"paragraph_vector": [-77.110282, -88.639083], "paragraph_keywords": ["plot", "sidechain", "contacts", "view"]}, {"paragraph_vector": [128.481216, -74.783195], "paragraph_keywords": ["plot", "mouse", "line", "residues"]}, {"paragraph_vector": [123.068572, -36.512535], "paragraph_keywords": ["directx", "residues", "pair", "application"]}, {"paragraph_vector": [95.034027, 38.052909], "paragraph_keywords": ["vertex", "create", "memory", "maps"]}, {"paragraph_vector": [95.751365, 41.748302], "paragraph_keywords": ["surface", "protein", "blending", "pointer"]}, {"paragraph_vector": [136.202804, 3.71135], "paragraph_keywords": ["application", "directx", "users", "version"]}, {"paragraph_vector": [122.588928, 43.19054], "paragraph_keywords": ["directx", "features", "applications", "use"]}], "content": {}, "doi": "10.1109/CIBCB.2012.6217231"}, {"uri": "118", "title": "Visualization of Dewetting of Molecularly Thin Lubricant on Magnetic Disks by Ellipsometric Microscopy", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["K. Fukuzawa"], "summary": "H IGH-DENSITY hard disk drives (HDDs) require a very low flying height, less than 10 nm. To meet this demand, the current thickness of liquid lubricant films, which are necessary for good durability and reliability of HDDs, is molecularly thin and of the order of 1 nm. Therefore, distribution control of molecularly thin lubricant films is indispensable to high-density recording. The behavior of molecularly thin liquid films is determined by the intermolecular interaction between the liquid molecules and media surface. Recently, polar lubricants have been widely used in HDDs because the polar lubricant molecule can be tightly adsorbed to the disk surface by the polar intermolecular interaction between the polar end group and the overcoat carbon of the disk. The polar lubricants tend to form an autophobic surface, where the molecules of the first absorbed layer form the inert surface and those of the upper layer cannot adsorb to the disk surface [1]\u2013[3]. Therefore, molecules of the upper layer tend to cohere to each other and form large droplets, which is called dewetting. This means that a film with a thickness of more than a monolayer may exhibit dewetting. Thus, clarifying the mechanism of lubricant dewetting can provide important information for head-disk interface design. Intensive research has clarified that the total surface energy profile of the air/lubricant/substrate interface determines the stability of the lubricant [4]\u2013[6]. Here, is the thickness of the lubricant. The lubricant is unstable when , whereas it is stable when . Therefore, when , dewetting occurs at every part of the lubricant simultaneously. Most of the previous studies focused on thin liquids on flat substrates. However, a magnetic disk has a small roughness called texture. Thus, both the intermolecular interaction and the surface roughness should be considered to clarify the dewetting of magnetic disks. However, a systematic analysis of the dewetting has not been reported yet. In this paper, we report our investigation of the effect of the surface energy profile and", "keywords": ["fig", "texture", "disk", "shown", "obtained", "surface", "nm", "film", "energy", "lubricant", "thickness", "dewetting"], "document_vector": [69.981048, -9.807401], "paragraphs": [{"paragraph_vector": [-54.468536, 48.184192], "paragraph_keywords": ["surface", "disk", "obtained", "lubricant"]}, {"paragraph_vector": [-55.526931, 46.676029], "paragraph_keywords": ["surface", "shown", "nm", "layer"]}, {"paragraph_vector": [-56.92728, 47.533889], "paragraph_keywords": ["films", "dewetting", "fig", "thickness"]}, {"paragraph_vector": [-54.109317, 47.096935], "paragraph_keywords": ["texture", "dewetting", "grooves", "determined"]}], "content": {}, "doi": "10.1109/BIBE.2004.1317384"}, {"uri": "119", "title": "Petascale Tcl with NAMD, VMD, and Swift/T", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["James C. Phillips", "John E. Stone", "Kirby L. Vandivort", "Timothy G. Armstrong", "Justin M. Wozniak", "Michael Wilde", "Klaus Schulten"], "summary": "Tcl is the original embeddable dynamic language. Introduced in 1990, Tcl has been the foundation of the scripting interface of the popular biomolecular visualization and analysis program VMD since 1995 and was extended to the parallel molecular dynamics program NAMD in 1999. The two programs together have over 200,000 users who have enjoyed for nearly two decades the stability and flexibility provided by Tcl. VMD users can implement or extend parallel trajectory analysis and movie rendering on thousands of nodes of Blue Waters. NAMD users can implement or extend simulation protocols and multiple-copy algorithms that execute unmodified on any supercomputer without the need to recompile NAMD. We now demonstrate the integration of the Swift/T high-performance parallel scripting language to enable high-level data flow programming in NAMD and VMD. This integration is achieved without modifying or recompiling either program since the Turbine execution engine is itself based on Tcl and is dynamically loaded by the interpreter, as is the platform-specific MPI library on which it depends.", "keywords": ["machine", "script", "runtime", "called", "data", "structure", "petascale", "partition", "program", "language", "master", "analysis", "replica", "scripting", "simulation", "set", "worker", "vmd", "support", "filesystem", "number", "run", "system", "programming", "implementation", "mpi", "code", "interpreter", "based", "work", "communication", "level", "performance", "parallel", "tool", "rank", "configuration", "written", "provides", "command", "file", "use", "model", "implemented", "t", "user", "function", "visualization", "tcl", "access", "exchange", "task"], "document_vector": [-31.91006, -3.519237], "paragraphs": [{"paragraph_vector": [-143.874832, 54.599601], "paragraph_keywords": ["researchers", "languages", "tcl", "access"]}, {"paragraph_vector": [-145.571945, 55.227108], "paragraph_keywords": ["t", "throughput", "platforms", "thousands"]}, {"paragraph_vector": [-141.835098, 52.987419], "paragraph_keywords": ["languages", "use", "compiled", "modeling"]}, {"paragraph_vector": [-147.125244, 52.928035], "paragraph_keywords": ["vmd", "analysis", "python", "developed"]}, {"paragraph_vector": [-177.823501, 49.156593], "paragraph_keywords": ["vmd", "analysis", "use", "parallel"]}, {"paragraph_vector": [172.969299, 48.061233], "paragraph_keywords": ["vmd", "scripting", "rendering", "users"]}, {"paragraph_vector": [-149.241287, 34.440296], "paragraph_keywords": ["configuration", "command", "tcl", "file"]}, {"paragraph_vector": [-151.843734, 31.493762], "paragraph_keywords": ["simulation", "tcl", "configuration", "scheduler"]}, {"paragraph_vector": [-149.92279, 31.674764], "paragraph_keywords": ["atoms", "simulation", "based", "implementation"]}, {"paragraph_vector": [-149.652572, 33.318927], "paragraph_keywords": ["master", "tcl", "file", "replicas"]}, {"paragraph_vector": [-149.160507, 33.893047], "paragraph_keywords": ["mpi", "communication", "exchange", "communicator"]}, {"paragraph_vector": [-152.594757, 35.972869], "paragraph_keywords": ["mpi", "partitions", "communication", "replicas"]}, {"paragraph_vector": [-148.580856, 40.226482], "paragraph_keywords": ["partition", "simulation", "t", "differences"]}, {"paragraph_vector": [-152.234863, 35.156387], "paragraph_keywords": ["t", "data", "tasks", "work"]}, {"paragraph_vector": [-152.990951, 38.423049], "paragraph_keywords": ["tcl", "t", "data", "mpi"]}, {"paragraph_vector": [-151.386077, 40.797172], "paragraph_keywords": ["defines", "file", "functions", "based"]}, {"paragraph_vector": [-150.020721, 37.200553], "paragraph_keywords": ["function", "tcl", "body", "code"]}, {"paragraph_vector": [-154.438873, 40.624301], "paragraph_keywords": ["runtime", "replica", "state", "filesystem"]}, {"paragraph_vector": [-152.472351, 37.480129], "paragraph_keywords": ["filesystem", "machine", "performance", "partition"]}, {"paragraph_vector": [-152.916137, 38.271202], "paragraph_keywords": ["mpi", "t", "fault", "programming"]}, {"paragraph_vector": [-166.260971, 57.310028], "paragraph_keywords": ["vmd", "based", "rank", "operate"]}, {"paragraph_vector": [-119.561599, 47.353805], "paragraph_keywords": ["application", "discus", "based", "grid"]}, {"paragraph_vector": [-141.620483, 53.539733], "paragraph_keywords": ["set", "code", "t", "proc"]}, {"paragraph_vector": [-151.365646, 38.062145], "paragraph_keywords": ["set", "newtemp", "doswap", "float"]}, {"paragraph_vector": [-147.119964, 59.104225], "paragraph_keywords": ["science", "supported", "illinois", "petascale"]}], "content": {}, "doi": "10.1109/IV.2006.127"}, {"uri": "120", "title": "MoleCollar and Tunnel Heat Map Visualizations for Conveying SpatioTemporoChemical Properties Across and Along Protein Voids", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["J. By\u0161ka", "M. E. Gr\u00f6ller", "B. Kozl\u00edkov\u00e1"], "summary": "Studying the characteristics of proteins and their inner void space, including their geometry, physico-chemical properties and dynamics are instrumental for evaluating the reactivity of the protein with other small molecules. The analysis of long simulations of molecular dynamics produces a large number of voids which have to be further explored and evaluated. In this paper we propose three new methods: two of them convey important properties along the long axis of a selected void during molecular dynamics and one provides a comprehensive picture across the void. The first two proposed methods use a specific heat map to present two types of information: an overview of all detected tunnels in the dynamics and their bottleneck width and stability over time, and an overview of a specific tunnel in the dynamics showing the bottleneck position and changes of the tunnel length over time. These methods help to select a small subset of tunnels, which are explored individually and in detail. For this stage we propose the third method, which shows in one static image the temporal evolvement of the shape of the most critical tunnel part, i.e., its bottleneck. This view is enriched with abstract depictions of different physicochemical properties of the amino acids surrounding the bottleneck. The usefulness of our newly proposed methods is demonstrated on a case study and the feedback from the domain experts is included. The biochemists confirmed that our novel methods help to convey the information about the appearance and properties of tunnels in a very intuitive and comprehensible manner.", "keywords": ["cavity", "ath", "exploration", "bottleneck", "order", "analysis", "representation", "acid", "surrounding", "dynamic", "method", "timesteps", "protein", "information", "ligand", "chemical", "figure", "sth", "proposed", "case", "tunnel", "map", "study", "size", "centerline", "contour", "time", "given", "domain", "property", "biochemist", "user", "visualization", "shape", "collar"], "document_vector": [-110.210128, 26.93469], "paragraphs": [{"paragraph_vector": [-3.372819, -61.248588], "paragraph_keywords": ["protein", "ligand", "chemical", "dynamics"]}, {"paragraph_vector": [13.459348, -46.008979], "paragraph_keywords": ["methods", "tunnel", "dynamics", "tunnels"]}, {"paragraph_vector": [9.069262, -17.22591], "paragraph_keywords": ["protein", "tunnels", "grid", "techniques"]}, {"paragraph_vector": [9.753272, -19.875547], "paragraph_keywords": ["tunnels", "tunnel", "representation", "atoms"]}, {"paragraph_vector": [21.723634, -35.063129], "paragraph_keywords": ["cavities", "data", "protein", "stage"]}, {"paragraph_vector": [12.421222, -17.112323], "paragraph_keywords": ["tunnel", "proposed", "methods", "workflow"]}, {"paragraph_vector": [6.320602, -35.446537], "paragraph_keywords": ["tunnels", "tunnel", "section", "properties"]}, {"paragraph_vector": [6.754246, -36.74224], "paragraph_keywords": ["tunnels", "tunnel", "bottleneck", "map"]}, {"paragraph_vector": [8.088309, -38.712043], "paragraph_keywords": ["tunnel", "tunnels", "bottleneck", "time"]}, {"paragraph_vector": [10.762097, -37.352901], "paragraph_keywords": ["tunnel", "sth", "size", "bottleneck"]}, {"paragraph_vector": [6.134277, -36.786418], "paragraph_keywords": ["tunnel", "shape", "bottleneck", "exploration"]}, {"paragraph_vector": [7.993038, -31.104087], "paragraph_keywords": ["tunnel", "representation", "contour", "centerline"]}, {"paragraph_vector": [3.927529, -23.602941], "paragraph_keywords": ["contours", "tunnel", "contour", "bottleneck"]}, {"paragraph_vector": [39.731559, -54.552967], "paragraph_keywords": ["acids", "acid", "contour", "bar"]}, {"paragraph_vector": [19.27597, -43.188987], "paragraph_keywords": ["contours", "representation", "contour", "area"]}, {"paragraph_vector": [6.806973, -40.343013], "paragraph_keywords": ["bottleneck", "tunnel", "tunnels", "study"]}, {"paragraph_vector": [12.013637, -43.493778], "paragraph_keywords": ["tunnel", "bottleneck", "displayed", "time"]}, {"paragraph_vector": [14.436959, -43.480888], "paragraph_keywords": ["bottleneck", "biochemists", "information", "representation"]}, {"paragraph_vector": [12.684753, -44.949096], "paragraph_keywords": ["proposed", "biochemists", "bottleneck", "methods"]}, {"paragraph_vector": [8.24839, -19.892505], "paragraph_keywords": ["route", "tunnel", "methods", "tunnels"]}], "content": {}, "doi": "10.1109/MCSE.2019.2962118"}, {"uri": "121", "title": "Accelerated Visualization of Dynamic Molecular Surfaces", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Norbert Lindow", "Daniel Baum", "Steffen Prohaska", "Hans-Christian Hege"], "summary": "Molecular surfaces play an important role in studying the interactions between molecules. Visualizing the dynamic behavior of molecules is particularly interesting to gain insights into a molecular system. Only recently it has become possible to interactively visualize dynamic molecular surfaces using ray casting techniques. In this paper, we show how to further accelerate the construction and the rendering of the solvent excluded surface (SES) and the molecular skin surface (MSS). We propose several improvements to reduce the update times for displaying these molecular surfaces. First, we adopt a parallel approximate Voronoi diagram algorithm to compute the MSS. This accelerates the MSS computation by more than one order of magnitude on a single core. Second, we demonstrate that the contour-buildup algorithm is ideally suited for computing the SES due to its inherently parallel structure. For both parallel algorithms, we observe good scalability up to 8 cores and, thus, obtain interactive frame rates for molecular dynamics trajectories of up to twenty thousand atoms for the SES and up to a few thousand atoms for the MSS. Third, we reduce the rendering time for the SES using tight-fitting bounding quadrangles as rasterization primitives. These primitives also accelerate the rendering of the MSS. With these improvements, the interactive visualization of the MSS of dynamic trajectories of a few thousand atoms becomes for the first time possible. Nevertheless, the SES remains a few times faster than the MSS.", "keywords": ["data", "molecule", "ms", "voronoi", "algorithm", "compute", "point", "number", "computation", "skin", "ray", "sphere", "surface", "j", "atom", "s", "cylinder", "use", "bounding", "radius", "sa", "contour", "patch", "vdw", "rendering", "cell", "intersection", "need"], "document_vector": [30.054777, -76.263183], "paragraphs": [{"paragraph_vector": [42.068061, 11.008048], "paragraph_keywords": ["surface", "surfaces", "proteins", "visualization"]}, {"paragraph_vector": [80.396202, 43.191577], "paragraph_keywords": ["surface", "sect", "algorithm", "computation"]}, {"paragraph_vector": [35.755832, 35.377506], "paragraph_keywords": ["surface", "ses", "surfaces", "presented"]}, {"paragraph_vector": [25.763086, -0.756252], "paragraph_keywords": ["surface", "vdw", "sphere", "spheres"]}, {"paragraph_vector": [11.950301, 8.115406], "paragraph_keywords": ["surface", "vdw", "points", "voronoi"]}, {"paragraph_vector": [12.749897, 15.716805], "paragraph_keywords": ["patches", "algorithm", "voronoi", "complex"]}, {"paragraph_vector": [32.465553, 22.692667], "paragraph_keywords": ["grid", "circles", "number", "contour"]}, {"paragraph_vector": [13.656774, 7.563792], "paragraph_keywords": ["ses", "circle", "patches", "contours"]}, {"paragraph_vector": [24.854869, 20.050338], "paragraph_keywords": ["j", "compute", "voronoi", "need"]}, {"paragraph_vector": [30.773574, 25.678754], "paragraph_keywords": ["algorithm", "surfaces", "voronoi", "j"]}, {"paragraph_vector": [13.204623, 49.722541], "paragraph_keywords": ["ray", "intersection", "point", "surface"]}, {"paragraph_vector": [29.916879, 34.625942], "paragraph_keywords": ["cylinder", "radius", "patch", "patches"]}, {"paragraph_vector": [15.202371, 55.749534], "paragraph_keywords": ["cylinder", "points", "need", "use"]}, {"paragraph_vector": [26.101079, 36.307178], "paragraph_keywords": ["patch", "intersection", "axis", "bounding"]}, {"paragraph_vector": [45.100231, 34.944255], "paragraph_keywords": ["algorithm", "mss", "computation", "ses"]}, {"paragraph_vector": [81.892013, 51.06454], "paragraph_keywords": ["algorithm", "molecule", "measured", "ses"]}, {"paragraph_vector": [178.858047, 84.825813], "paragraph_keywords": ["algorithm", "ses", "reason", "avd"]}, {"paragraph_vector": [33.789279, 54.988979], "paragraph_keywords": ["mss", "tracing", "bounding", "patches"]}, {"paragraph_vector": [66.439987, 38.515888], "paragraph_keywords": ["rendering", "mss", "ses", "surface"]}], "content": {}, "doi": "10.1109/SNPD.2008.123"}, {"uri": "122", "title": "Visualizing the Origins of Life: Molecular Animation for Scientific Research and Education", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Janet Iwasa"], "summary": "How did life evolve on Earth, and what form did it take? Biochemists are attempting to address these questions using molecular clues found in our own cells. All life on the planet, from bacteria to humans, share a system of storing genetic information in the form of DNA and/or RNA molecules. Very early in Earth\u2019s history, the first living cell is likely to have evolved this system from simple chemical compounds found in the early terrestrial environment. This project seeks to use scientifically accurate molecular animations to illustrate the leading theories on how cellular life arose on the early Earth, and the experiments that are currently being done in biochemistry laboratories to test these theories. Molecular visualizations of the origins of life, created in close collaboration with researchers and science educators, will be used to communicate cutting-edge science research to the public and will also be used for presentations within the scientific community.", "keywords": ["structure", "molecule", "event", "acid", "origin", "create", "evolution", "protein", "public", "vesicle", "nucleotide", "research", "formation", "reaction", "life", "scientist", "pdb", "museum", "level", "rna", "replicase", "animation", "energy", "szostak", "use", "science", "cell"], "document_vector": [-68.760902, -4.593586], "paragraphs": [{"paragraph_vector": [121.336669, -29.660648], "paragraph_keywords": ["molecules", "life", "scientists", "presentations"]}, {"paragraph_vector": [122.415237, -34.561035], "paragraph_keywords": ["rna", "cell", "life", "dna"]}, {"paragraph_vector": [140.112472, -42.20288], "paragraph_keywords": ["rna", "replicases", "create", "life"]}, {"paragraph_vector": [117.401847, -33.8292], "paragraph_keywords": ["structure", "rna", "form", "vesicle"]}, {"paragraph_vector": [123.606819, -32.463821], "paragraph_keywords": ["molecules", "science", "animations", "rna"]}, {"paragraph_vector": [135.423461, -39.244934], "paragraph_keywords": ["nucleotides", "rnas", "rna", "vesicles"]}, {"paragraph_vector": [118.319633, -32.014129], "paragraph_keywords": ["pdb", "molecule", "references", "including"]}, {"paragraph_vector": [115.253807, -18.368656], "paragraph_keywords": ["molecules", "moving", "numbers", "acids"]}, {"paragraph_vector": [117.287208, -20.841308], "paragraph_keywords": ["scale", "animation", "events", "molecule"]}, {"paragraph_vector": [121.478134, -30.471153], "paragraph_keywords": ["public", "rna", "research", "life"]}], "content": {}, "doi": "10.1109/CGIV.2006.12"}, {"uri": "123", "title": "Adapting a Single-user, Single-display Molecular Visualization Application for Use in a Multi-user, Multi-Display Environment", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Clifton Forlines", "Ryan Lilien"], "summary": "In this paper, we discuss the adaptation of an open-source singleuser, single-display molecular visualization application for use in a multi-display, multi-user environment. Jmol, a popular, opensource Java applet for viewing PDB files, is modified in such a manner that allows synchronized coordinated views of the same molecule to be displayed in a multi-display workspace. Each display in the workspace is driven by a separate PC, and coordinated views are achieved through the passing of RasMol script commands over the network. The environment includes a tabletop display capable of sensing touch-input, two large vertical displays, and a TabletPC. The presentation of large molecules is adapted to best take advantage of the different qualities of each display, and a set of interaction techniques that allow groups working in this environment to better collaborate are also presented.", "keywords": ["machine", "group", "wall", "structure", "molecule", "application", "selection", "orientation", "team", "interaction", "workspace", "protein", "information", "ligand", "control", "point", "working", "system", "tabletop", "appearance", "command", "display", "use", "software", "view", "wim", "user", "table", "visualization", "built", "residue"], "document_vector": [178.715454, -27.566497], "paragraphs": [{"paragraph_vector": [147.332962, -38.972141], "paragraph_keywords": ["visualization", "proteins", "group", "display"]}, {"paragraph_vector": [155.863754, 21.826345], "paragraph_keywords": ["machines", "applications", "use", "copies"]}, {"paragraph_vector": [143.448257, 9.054352], "paragraph_keywords": ["orientation", "protein", "member", "group"]}, {"paragraph_vector": [143.802963, 10.731404], "paragraph_keywords": ["visualization", "displays", "built", "working"]}, {"paragraph_vector": [149.658462, 23.097391], "paragraph_keywords": ["table", "researchers", "applications", "system"]}, {"paragraph_vector": [151.834625, 6.790217], "paragraph_keywords": ["control", "table", "molecule", "ones"]}, {"paragraph_vector": [151.995162, 19.486751], "paragraph_keywords": ["wim", "wall", "state", "script"]}, {"paragraph_vector": [148.423843, 16.539733], "paragraph_keywords": ["wall", "display", "command", "group"]}, {"paragraph_vector": [-137.42012, -73.223144], "paragraph_keywords": ["selection", "controls", "application", "atoms"]}, {"paragraph_vector": [134.031494, -13.826323], "paragraph_keywords": ["ligand", "selection", "structure", "enzyme"]}, {"paragraph_vector": [131.036178, -20.928325], "paragraph_keywords": ["team", "display", "ligand", "unbound"]}, {"paragraph_vector": [143.594497, 11.177989], "paragraph_keywords": ["user", "information", "systems", "proceedings"]}, {"paragraph_vector": [141.860015, -10.729615], "paragraph_keywords": ["visualization", "display", "conference", "proceedings"]}, {"paragraph_vector": [109.905632, 27.768001], "paragraph_keywords": ["graphics", "working", "conference", "interfaces"]}], "content": {}, "doi": "10.1109/JBHI.2020.3027511"}, {"uri": "124", "title": "Molecular Surface Abstraction", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Gregory Cipriano", "Michael Gleicher"], "summary": "In this paper we introduce a visualization technique that provides an abstracted view of the shape and spatio-physico-chemical properties of complex molecules. Unlike existing molecular viewing methods, our approach suppresses small details to facilitate rapid comprehension, yet marks the location of significant features so they remain visible. Our approach uses a combination of filters and mesh restructuring to generate a simplified representation that conveys the overall shape and spatio-physico-chemical properties (e.g. electrostatic charge). Surface markings are then used in the place of important removed details, as well as to supply additional information. These simplified representations are amenable to display using stylized rendering algorithms to further enhance comprehension. Our initial experience suggests that our approach is particularly useful in browsing collections of large molecules and in readily making comparisons between them.", "keywords": ["value", "marking", "molecule", "decal", "detail", "representation", "create", "set", "mesh", "method", "lighting", "protein", "parameterization", "information", "point", "approach", "scale", "system", "figure", "texture", "process", "surface", "smoothing", "tool", "performance", "size", "display", "field", "use", "abstraction", "removed", "view", "feature", "vertex", "region", "patch", "property", "charge", "apply", "provide", "shape"], "document_vector": [-163.835632, -21.746282], "paragraphs": [{"paragraph_vector": [95.737434, -36.279129], "paragraph_keywords": ["molecules", "properties", "views", "details"]}, {"paragraph_vector": [53.766971, 15.050574], "paragraph_keywords": ["shape", "surface", "information", "fields"]}, {"paragraph_vector": [89.699371, -33.041263], "paragraph_keywords": ["surface", "surfaces", "interactions", "stylized"]}, {"paragraph_vector": [96.035285, -13.276109], "paragraph_keywords": ["molecule", "provide", "representations", "shape"]}, {"paragraph_vector": [49.327083, 2.421392], "paragraph_keywords": ["surfaces", "provide", "shape", "appearance"]}, {"paragraph_vector": [57.480113, 13.729573], "paragraph_keywords": ["shape", "surfaces", "stylized", "graphics"]}, {"paragraph_vector": [15.107488, -14.037396], "paragraph_keywords": ["features", "mesh", "scale", "surface"]}, {"paragraph_vector": [21.380044, -12.114437], "paragraph_keywords": ["features", "vertices", "curvature", "feature"]}, {"paragraph_vector": [24.528884, 2.71673], "paragraph_keywords": ["vertices", "surface", "edge", "mesh"]}, {"paragraph_vector": [48.558013, 12.705501], "paragraph_keywords": ["surface", "parameterization", "point", "vertex"]}, {"paragraph_vector": [32.757427, 20.275215], "paragraph_keywords": ["vertices", "operations", "mesh", "markings"]}, {"paragraph_vector": [14.232999, 0.917582], "paragraph_keywords": ["system", "patch", "positions", "vertices"]}, {"paragraph_vector": [8.870647, -15.687448], "paragraph_keywords": ["surface", "regions", "shape", "system"]}, {"paragraph_vector": [58.883174, 18.518646], "paragraph_keywords": ["field", "value", "kernel", "filter"]}, {"paragraph_vector": [43.618495, 15.917585], "paragraph_keywords": ["surface", "shape", "feature", "brightness"]}, {"paragraph_vector": [48.54607, 8.991234], "paragraph_keywords": ["abstraction", "surface", "surfaces", "paper"]}, {"paragraph_vector": [55.238132, 4.567549], "paragraph_keywords": ["features", "set", "occlusion", "test"]}, {"paragraph_vector": [52.352661, 11.315413], "paragraph_keywords": ["surfaces", "features", "shape", "decals"]}, {"paragraph_vector": [50.834541, 0.302167], "paragraph_keywords": ["molecules", "implies", "way", "wonder"]}], "content": {}, "doi": "10.1109/FBIT.2007.60"}, {"uri": "125", "title": "Visual Abstractions of Solvent Pathlines near Protein Cavities", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Katrin Bidmon", "Sebastian Grottel", "Fabian B\u00f6s", "J\u00fcrgen Pleiss", "Thomas Ertl"], "summary": "Water is known to play a crucial role in protein structure, flexibility and activity. The use of molecular dynamics simulations allows detailed studies of complex protein-solvent interactions. Cluster analysis and density-based approaches have been successfully used for the identification and analysis of conserved water molecules and hydration patterns of proteins. However, appropriate tools for analysing long-time molecular dynamics simulations with respect to tracking and visualising the paths of solvent molecules are lacking. Our method focuses on visualising the solvent paths entering and leaving cavities of the protein and allows to study the route and dynamics of the exchange of tightly bound internal water molecules with the bulk solvent. The proposed visualisation also represents dynamic properties such as direction and velocity in the solvent. Especially, by clustering similar pathlines with respect to designated properties the visualisation can be abstracted to represent the principal paths of solvent molecules through the cavities. Its application in the analysis of long-time scale molecular dynamics simulations not only confirmed conjectures based on previous manual observations made by chance, but also led to novel insights into the dynamical and structural role of water molecules and its interplay with protein structure.", "keywords": ["cavity", "visualisation", "chosen", "structure", "motion", "area", "molecule", "distance", "representation", "path", "cluster", "protein", "trajectory", "applied", "information", "approach", "roi", "clustering", "based", "work", "position", "water", "atom", "edge", "velocity", "solvent", "time", "pathlines", "rendering", "pathline"], "document_vector": [-110.705505, 24.302408], "paragraphs": [{"paragraph_vector": [13.421901, -57.547645], "paragraph_keywords": ["protein", "water", "solvent", "molecules"]}, {"paragraph_vector": [47.284404, -52.815273], "paragraph_keywords": ["water", "molecules", "protein", "solvent"]}, {"paragraph_vector": [75.600944, -19.135522], "paragraph_keywords": ["protein", "molecules", "solvent", "time"]}, {"paragraph_vector": [55.929985, -55.926265], "paragraph_keywords": ["water", "molecules", "cavity", "protein"]}, {"paragraph_vector": [55.750247, -50.052066], "paragraph_keywords": ["pathlines", "visualisation", "extracted", "trajectory"]}, {"paragraph_vector": [58.339912, -52.735012], "paragraph_keywords": ["molecules", "solvent", "protein", "cavity"]}, {"paragraph_vector": [54.666431, -48.123889], "paragraph_keywords": ["molecules", "protein", "solvent", "pathlines"]}, {"paragraph_vector": [58.76921, -59.35971], "paragraph_keywords": ["velocity", "visualisation", "mapped", "time"]}, {"paragraph_vector": [54.416465, -46.15306], "paragraph_keywords": ["pathlines", "velocity", "positions", "clustering"]}, {"paragraph_vector": [61.506099, -50.229331], "paragraph_keywords": ["clusters", "pathline", "edge", "velocity"]}, {"paragraph_vector": [60.234001, -41.28197], "paragraph_keywords": ["edge", "edges", "distance", "control"]}, {"paragraph_vector": [51.931266, -49.077526], "paragraph_keywords": ["water", "pathlines", "molecules", "cavity"]}, {"paragraph_vector": [61.941135, -47.321044], "paragraph_keywords": ["pathlines", "motion", "water", "performance"]}, {"paragraph_vector": [59.031902, -53.227165], "paragraph_keywords": ["solvent", "molecules", "protein", "pathlines"]}], "content": {}, "doi": "10.1109/MEMSYS.2005.1454055"}, {"uri": "126", "title": "Web Based Molecular Visualization using Procedural Shaders in X", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Feng Liu", "Scott Owen", "Ying Zhu", "Robert Harrison", "Irene Weber"], "summary": "The introduction of programmable Graphics Processing Units (GPUs) and the addition of procedural shaders to the Web3D standard X3D provide us with new techniques to develop real-time Web based visualization. In this paper we discuss the applications of these techniques to bioinformatics and chemistry visualization, specifically the visualization of large biomolecules. By using procedural shaders, we are able to produce higher quality visualizations with minimal performance penalty. We have developed methods to automatically convert from the standard bioinformatics PDB format to CML and then to X3D. The procedural shaders are automatically inserted during the CML to X3D conversion. This provides higher quality images and leads to future possibilities of more flexible and enhanced visualizations. CR Categories: I.3.1 [Computer Graphics]: Hardware Architecture Graphics processors; I.3.7 [Computer Graphics] Three-Dimensional Graphics and Realism Color, shading, shadowing, and texture I.3.7 Applications", "keywords": ["web", "structure", "molecule", "language", "xslt", "et", "developed", "set", "protein", "shading", "research", "system", "format", "quality", "pdb", "shader", "cml", "atom", "cpu", "graphic", "vrml", "display", "shaders", "use", "file", "visualization"], "document_vector": [-43.847114, -82.849601], "paragraphs": [{"paragraph_vector": [120.872131, 30.512975], "paragraph_keywords": ["shaders", "visualization", "research", "structures"]}, {"paragraph_vector": [120.781562, 36.145824], "paragraph_keywords": ["pdb", "shading", "platforms", "developed"]}, {"paragraph_vector": [119.289985, 6.411927], "paragraph_keywords": ["vrml", "web", "chemistry", "capability"]}, {"paragraph_vector": [121.96244, 11.565112], "paragraph_keywords": ["shaders", "cml", "use", "set"]}, {"paragraph_vector": [129.795455, 38.721584], "paragraph_keywords": ["shaders", "shading", "format", "platform"]}, {"paragraph_vector": [90.048248, 49.707271], "paragraph_keywords": ["shaders", "shader", "xslt", "cml"]}, {"paragraph_vector": [123.742889, 31.969364], "paragraph_keywords": ["shaders", "molecules", "shader", "structure"]}, {"paragraph_vector": [84.669609, 9.419405], "paragraph_keywords": ["molecule", "cpu", "vrml", "shaders"]}, {"paragraph_vector": [84.359504, 26.718364], "paragraph_keywords": ["display", "system", "molecules", "use"]}], "content": {}, "doi": "10.1109/VISUAL.1996.568133"}, {"uri": "127", "title": "Self-organizing Trees for visualizing protein dataset", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Nhat-Quang Doan", "Hanane Azzag", "Mustapha Lebbah", "Guillaume Santini"], "summary": "Clustering and visualizing multi-dimensional or structured data are important tasks for data analysis, especially in bioinformatics. Self-organizing models are often used to address both of these issues. In this paper we introduce a hierarchical and topological visualization technique called Self-organizing Trees (SoT) which is able to represent data in hierarchical and topological structure. The experiment is conducted on a real-world protein data set.", "keywords": ["data", "dataset", "structure", "k", "distance", "graph", "tree", "similarity", "xpos", "protein", "l", "algorithm", "information", "number", "clustering", "c", "connected", "assignment", "sot", "treec", "edge", "xi", "domain", "function", "node"], "document_vector": [165.501098, 59.530101], "paragraphs": [{"paragraph_vector": [-75.510017, -11.498845], "paragraph_keywords": ["vertex", "graph", "vertices", "matrix"]}, {"paragraph_vector": [-85.548591, -11.329948], "paragraph_keywords": ["data", "tree", "grid", "node"]}, {"paragraph_vector": [-87.372604, -11.065469], "paragraph_keywords": ["tree", "function", "neighborhood", "kt"]}, {"paragraph_vector": [-83.151603, -8.003055], "paragraph_keywords": ["tree", "nodes", "xi", "algorithm"]}, {"paragraph_vector": [-88.042442, -8.615765], "paragraph_keywords": ["tree", "nodes", "algorithm", "disconnected"]}, {"paragraph_vector": [-86.887557, -10.603119], "paragraph_keywords": ["tree", "assignment", "xpos", "algorithm"]}, {"paragraph_vector": [-71.245849, -47.218803], "paragraph_keywords": ["protein", "domains", "structure", "term"]}, {"paragraph_vector": [-69.22187, -42.339103], "paragraph_keywords": ["graph", "protein", "l", "dist"]}, {"paragraph_vector": [-88.789726, -10.96094], "paragraph_keywords": ["sot", "edges", "tree", "data"]}, {"paragraph_vector": [-80.975265, -21.052497], "paragraph_keywords": ["dataset", "protein", "rand", "information"]}, {"paragraph_vector": [-80.504386, -25.570693], "paragraph_keywords": ["protein", "proposed", "tree", "expert"]}], "content": {}, "doi": "10.1109/VISUAL.2000.885733"}, {"uri": "128", "title": "Global-To-Local Representation and Visualization of Molecular Surfaces Using Deformable Models", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Nicolay Postarnakevich", "Rahul Singh"], "summary": "Macromolecules such as proteins and enzymes are responsible for most of cellular functionality. Many molecular interactions, such as protein-protein interactions or protein-ligand binding, occur at what can be defined as the molecular surface. The topology of the molecular surface is often complex, containing various geometric features such as clefts, cavities, tunnels, and flat regions. These geometric features coupled with non-geometric physicochemical properties influence surface-based molecular interactions. Consequently analysis of molecular surfaces is crucial in elucidating structure-property relationships of molecules. In this paper we propose a method for visualizing a molecular surface in a manner that preserves and elucidates salient features. The method involves mapping of a molecular surface to a standard spherical coordinate system. The ability to map arbitrary molecular surfaces to a standard coordinate system aids in comparison of surface features across different molecules. The mapping is accomplished by enclosing the molecular surface by a sphere, and then iteratively deforming the sphere until it converges by wrapping the entire molecular surface. This allows a one-to-one relationship to be established between points on the molecular surface and points on the surface of the sphere. The presence of discontinuities such as tunnels in the molecular surface can be identified by detecting collision between patches of the deforming sphere. Subsequently, the deformable surface is restored back to the sphere, retaining the mapping. Features and properties defined at the molecular surface are then mapped and visualized in the standard spherical coordinate system. The proposed approach has several key advantages. First, it allows a global-to-local visualization of molecular surfaces. Second, it facilitates comparison of specific features as well as collection of features within and across molecules by mapping them to a common coordinate system. Third, the method allows visualization of both geometric and non-geometric surface properties. Fourth, specific molecular characteristics can be visualized individually or in combination on-demand. Finally, and crucially the advantages offered by the proposed visualization do not involve simplification of the surface characteristics thereby ensuring that no loss of potentially important information occurs.", "keywords": ["cavity", "characteristic", "structure", "pocket", "molecule", "representation", "collision", "method", "protein", "visualizing", "point", "figure", "sphere", "map", "surface", "based", ".", "mapping", "binding", "computed", "energy", "show", "check", "model", "center", "use", "requires", "curvature", "feature", "function", "force", "visualization", "rendering", "node"], "document_vector": [-130.999893, -18.838333], "paragraphs": [{"paragraph_vector": [-27.136192, -53.375511], "paragraph_keywords": ["surface", "surfaces", "protein", "features"]}, {"paragraph_vector": [23.308841, -14.070352], "paragraph_keywords": ["surface", "features", "copies", "requires"]}, {"paragraph_vector": [34.190269, -9.798249], "paragraph_keywords": ["surface", "surfaces", "section", "including"]}, {"paragraph_vector": [29.232183, -5.22292], "paragraph_keywords": ["surface", "protein", "visibility", "pockets"]}, {"paragraph_vector": [12.870124, 18.467679], "paragraph_keywords": ["surface", "sphere", "map", "features"]}, {"paragraph_vector": [14.296229, 8.656213], "paragraph_keywords": ["sphere", "forces", "surface", "energy"]}, {"paragraph_vector": [12.780104, 14.460097], "paragraph_keywords": ["node", "surface", "model", "sphere"]}, {"paragraph_vector": [13.296187, 9.085907], "paragraph_keywords": ["surface", "sphere", "node", "check"]}, {"paragraph_vector": [14.831642, 11.260853], "paragraph_keywords": ["surface", "sphere", "collision", "triangle"]}, {"paragraph_vector": [32.560661, -6.014102], "paragraph_keywords": ["surface", "sphere", "map", "shown"]}, {"paragraph_vector": [30.748964, -7.402476], "paragraph_keywords": ["surface", "center", "shows", "figure"]}, {"paragraph_vector": [39.198181, -27.361051], "paragraph_keywords": ["surface", "method", "characteristics", "surfaces"]}, {"paragraph_vector": [37.552692, -6.505112], "paragraph_keywords": ["surface", "protein", ".", "vol"]}, {"paragraph_vector": [-46.4556, -49.134597], "paragraph_keywords": ["proteins", "protein", "function", "bioinformatics"]}], "content": {}, "doi": "10.1109/GRC.2005.1547307"}, {"uri": "129", "title": "Atomistic Visualization of Mesoscopic Whole-Cell Simulations Using Ray-Casted Instancing", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Martin Falk", "Michael Krone", "Thomas Ertl"], "summary": "Molecular visualization is an important tool for analysing the results of biochemical simulations. With modern GPU ray casting approaches, it is only possible to render several million of atoms interactively unless advanced acceleration methods are employed. Whole-cell simulations consist of at least several billion atoms even for simplified cell models. However, many instances of only a few different proteins occur in the intracellular environment, which can be exploited to fit the data into the graphics memory. For each protein species, one model is stored and rendered once per instance. The proposed method exploits recent algorithmic advances for particle rendering and the repetitive nature of intracellular proteins to visualize dynamic results from mesoscopic simulations of cellular transport processes. We present two out-of-core optimizations for the interactive visualization of data sets composed of billions of atoms as well as details on the data preparation and the employed rendering techniques. Furthermore, we apply advanced shading methods to improve the image quality including methods to enhance depth and shape perception besides non-photorealistic rendering methods. We also show that the method can be used to render scenes that are composed of triangulated instances, not only implicit surfaces.", "keywords": ["data", "molecule", "d", "et", "render", "simulation", "rendered", "set", "method", "stored", "protein", "shading", "triangle", "number", "approach", "result", "object", "figure", "depth", "texture", "ray", "colour", "pdb", "based", "surface", "work", "scene", "performance", "atom", "size", "graphic", "gpu", "use", "bounding", "model", "grid", "instance", "user", "rendering", "visualization", "cell", "test"], "document_vector": [-34.830459, -65.16529], "paragraphs": [{"paragraph_vector": [89.472923, 23.08987], "paragraph_keywords": ["simulations", "visualization", "sets", "work"]}, {"paragraph_vector": [82.20034, 42.190738], "paragraph_keywords": ["data", "sets", "visualization", "method"]}, {"paragraph_vector": [76.564414, 48.215255], "paragraph_keywords": ["data", "sets", "render", "gpu"]}, {"paragraph_vector": [53.910438, 47.210399], "paragraph_keywords": ["ray", "gpu", "rendered", "rendering"]}, {"paragraph_vector": [55.171821, 68.530113], "paragraph_keywords": ["method", "ray", "voxel", "points"]}, {"paragraph_vector": [92.722908, -28.268661], "paragraph_keywords": ["cell", "proteins", "filaments", "cytoplasm"]}, {"paragraph_vector": [88.187744, -14.299211], "paragraph_keywords": ["pdb", "molecules", "protein", "figure"]}, {"paragraph_vector": [77.986946, 25.596231], "paragraph_keywords": ["atoms", "grid", "data", "molecule"]}, {"paragraph_vector": [34.545597, 46.157058], "paragraph_keywords": ["grid", "ray", "atom", "atoms"]}, {"paragraph_vector": [40.872268, 59.290866], "paragraph_keywords": ["number", "shading", "grid", "size"]}, {"paragraph_vector": [39.450298, 54.18124], "paragraph_keywords": ["depth", "fragment", "colour", "takes"]}, {"paragraph_vector": [28.961297, 45.314689], "paragraph_keywords": ["grid", "ray", "texture", "approach"]}, {"paragraph_vector": [71.236709, 28.058311], "paragraph_keywords": ["grid", "meshes", "pixel", "colour"]}, {"paragraph_vector": [27.049642, 45.028697], "paragraph_keywords": ["grid", "texture", "triangles", "stored"]}, {"paragraph_vector": [55.928089, 45.865779], "paragraph_keywords": ["file", "meshes", "renderings", "user"]}, {"paragraph_vector": [76.01078, 43.883232], "paragraph_keywords": ["depth", "shading", "rendering", "user"]}, {"paragraph_vector": [86.779144, -7.702294], "paragraph_keywords": ["data", "pdb", "activated", "proteins"]}, {"paragraph_vector": [83.481979, 5.741858], "paragraph_keywords": ["proteins", "performance", "time", "cell"]}, {"paragraph_vector": [83.327247, 43.446578], "paragraph_keywords": ["fps", "rendering", "figure", "data"]}, {"paragraph_vector": [84.840133, 31.096273], "paragraph_keywords": ["rendering", "surface", "bounding", "diffusion"]}, {"paragraph_vector": [82.507995, 26.076215], "paragraph_keywords": ["method", "methods", "rendering", "models"]}, {"paragraph_vector": [-157.458618, 12.347066], "paragraph_keywords": ["conduct", "user", "study", "dimensions"]}], "content": {}, "doi": "10.1109/ICBBE.2009.5163427"}, {"uri": "130", "title": "Quantum Molecular Modeling with Simulated Annealing", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["D. Hohl"], "summary": "A quantum-mechanical method for calculating the geometrical structure of small molecules is presented. A combination of the density functional (DF) method with finitetemperature molecular dynamics (MD) and simulated annealing techniques provides a reliable scheme for sampling energy surfaces, without being trapped in energetically unfavorable local minima. We demonstrate its value and efficiency by computing previously unknown geometries of small clusters composed of phosphorus, sulphur, selenium and oxygen atoms. The implementation combines distributed processing with real-time visualization and allows instantaneous interaction with the simulation. The computationally demanding quantum mechanics problem is solved on a Cray supercomputer, communicating with a Silicon Graphics Incorporated (SGI) workstation in a UNIX environment using SGI\u2019s Distributed Graphics Library (DGL). We show that real-time visualization and interaction enhance scientific productivity greatly. 1. Introduct ion t o t h e Underlying Physics The most stable structures of molecules can be determined, in principle, from the total energy E of the system of electrons and ions. If we determine E for all possible geometries, the ground state structure is simply that with the lowest energy. Unfortunately, this method is impracticable for all but the smallest clusters of atoms and molecules. The calculation of the total energy is difficult, and the number of local minima in the energy surface increases dramatically with increasing cluster size. A combination of density functional (DF) methods,\u2019 and molecular dynamics (MD)\u2019 with \u201csimulated annealing313 addresses both of these problems, and we outline the important features of the approach. Calculations for small molecules show that it can reproduce known structures remarkably well (sulphur and selenium clusters, S,, Se,) and give interesting predictions where experimental information is difficult to obtain (phosphorus clusters, Pa). Molecular dynamics also allows us to follow large changes in molecular structures, as we show for S70. 1.1 Calculation of t h e to t a l energy, E The total energy E of an interacting system of electrons and ions can be found from the expression E = (Ql~ l * ) / (Q l * ) , (1) where @ is the Hamiltonian operator of the system. Calculations of this type can be carried out only for systems with very few electrons, and approximate solutions for IIr are generally based on the variational principle of Rayleigh and Ritz, i.e. if I@) is an approximate wave function, then The simplest form of the many-particle wave function is that of Hartree (1928), who represented @ as a product of singleelectron functions: where the functions #i each satisfy a Schrodinger equation whose potential term is given by the mean field of the other electrons,", "keywords": [], "document_vector": [-33.227359, -7.757107], "paragraphs": [{"paragraph_vector": [-132.101409, 24.404472], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/CBMS.2008.95"}, {"uri": "131", "title": "ElectroLens: Understanding Atomistic Simulations through Spatially-Resolved Visualization of High-Dimensional Features", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Xiangyun Lei", "Fred Hohman", "Duen Horng", "(Polo) Chau", "Andrew J. Medford"], "summary": "In recent years, machine learning (ML) has gained significant popularity in the field of chemical informatics and electronic structure theory. These techniques often require researchers to engineer abstract \u201cfeatures\u201d that encode chemical concepts into a mathematical form compatible with the input to machine-learning models. However, there is no existing tool to connect these abstract features back to the actual chemical system, making it difficult to diagnose failures and to build intuition about the meaning of the features. We present ElectroLens, a new visualization tool for high-dimensional spatially-resolved features to tackle this problem. The tool visualizes high-dimensional data sets for atomistic and electron environment features by a series of linked 3D views and 2D plots. The tool is able to connect different derived features and their corresponding regions in 3D via interactive selection. It is built to be scalable, and integrate with existing infrastructure.", "keywords": ["data", "structure", "correlation", "electron", "ml", "simulation", "corresponding", "training", "electrolens", "visualizing", "point", "chemical", "system", "c", "density", "color", "ieee", "plot", ".", "based", "learning", "atom", "energy", "model", "use", "view", "feature", "environment", "region", "theory", "visualization"], "document_vector": [167.320648, 43.87886], "paragraphs": [{"paragraph_vector": [-17.831819, -35.14669], "paragraph_keywords": ["systems", "structure", "ieee", "chemists"]}, {"paragraph_vector": [-26.311483, -37.500465], "paragraph_keywords": ["features", "feature", "visualization", "plots"]}, {"paragraph_vector": [-13.735507, -30.467533], "paragraph_keywords": ["features", "atom", "systems", "ml"]}, {"paragraph_vector": [-44.82608, -57.541988], "paragraph_keywords": ["features", "chemical", "data", "visualizing"]}, {"paragraph_vector": [-10.762088, -39.026802], "paragraph_keywords": ["data", "structure", "tools", "structures"]}, {"paragraph_vector": [107.778617, -27.638896], "paragraph_keywords": ["plots", "features", "color", "view"]}, {"paragraph_vector": [-71.949462, -56.591579], "paragraph_keywords": ["plots", "features", "points", "plot"]}, {"paragraph_vector": [-108.623954, 61.459373], "paragraph_keywords": ["electrolens", "density", "ase", "energy"]}, {"paragraph_vector": [-14.7443, -29.87515], "paragraph_keywords": ["density", "electron", "features", "electrolens"]}, {"paragraph_vector": [-10.353436, -29.678789], "paragraph_keywords": ["training", "data", "forces", "environments"]}, {"paragraph_vector": [-15.396796, -41.159008], "paragraph_keywords": ["data", "electrolens", "chemical", "use"]}, {"paragraph_vector": [-7.659696, -27.509527], "paragraph_keywords": ["learning", "machine", ".", "physics"]}, {"paragraph_vector": [-12.965235, -27.957443], "paragraph_keywords": ["journal", "visualization", "data", "ieee"]}], "content": {}, "doi": "10.1109/ICBBE.2007.4"}, {"uri": "132", "title": "Facet-JFA: Faster computation of discrete Voronoi diagrams", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Talha Bin Masood", "Hari Krishna Malladi", "Vijay Natarajan"], "summary": "Jump Flooding is a method for propagating labels across a given plane from different seeds. It has been used to compute the discrete Voronoi tessellation of a given plane efficiently. We introduce a version of JFA, which optimizes the number of pixels processed by computing only the faces of the Voronoi tessellation. The pixels in the interior of the Voronoi regions are not processed resulting in a 1-skeleton representation of the Voronoi tessellation in 2D and a 2-skeleton representation in 3D. We describe an implementation of this algorithm on a GPU using CUDA and demonstrate its performance benefits on multiple data sets. As an application of the proposed algorithm, we present a GPU based method for extraction of channel centerlines in biomolecules. The fast computation of the discrete Voronoi diagram is exploited to extract channels in molecular dynamics simulation trajectories on-the-fly, thereby supporting the interactive visual analysis of static and dynamic channel structures.", "keywords": ["data", "facet", "analysis", "voronoi", "set", "space", "computing", "m", "l", "label", "algorithm", "number", "approach", "speed", "diagram", "pixel", "step", "figure", "resolution", "based", "jfa", "seed", "size", "n", "india", "channel", "gpu", "grid", "region", "time", "flooding", "thread"], "document_vector": [-117.423156, -42.710636], "paragraphs": [{"paragraph_vector": [-15.14296, 62.552764], "paragraph_keywords": ["copies", "voronoi", "acm", "pixels"]}, {"paragraph_vector": [-29.014333, 65.141082], "paragraph_keywords": ["jfa", "voronoi", "grid", "algorithm"]}, {"paragraph_vector": [42.67073, -3.875439], "paragraph_keywords": ["defined", "p", "grid", "set"]}, {"paragraph_vector": [-20.439443, 64.006095], "paragraph_keywords": ["l", "grid", "pixel", "flooding"]}, {"paragraph_vector": [-24.778169, 64.356933], "paragraph_keywords": ["m", "grid", "jfa", "pixels"]}, {"paragraph_vector": [-32.025901, 63.47364], "paragraph_keywords": ["jfa", "algorithm", "steps", "pixels"]}, {"paragraph_vector": [-35.764724, 65.858116], "paragraph_keywords": ["pixels", "number", "grid", "jfa"]}, {"paragraph_vector": [-21.25459, 62.340877], "paragraph_keywords": ["pixels", "grid", "number", "hyperplanes"]}, {"paragraph_vector": [-33.174205, 66.894866], "paragraph_keywords": ["pixels", "approach", "d", "voronoi"]}, {"paragraph_vector": [-156.225662, 67.172065], "paragraph_keywords": ["step", "jfa", "grid", "resolution"]}, {"paragraph_vector": [-164.761093, 66.840568], "paragraph_keywords": ["jfa", "number", "taken", "resolution"]}, {"paragraph_vector": [-152.260635, 65.38478], "paragraph_keywords": ["speed", "number", "observed", "jfa"]}, {"paragraph_vector": [45.713245, -29.006534], "paragraph_keywords": ["channel", "voronoi", "channels", "biomolecule"]}, {"paragraph_vector": [50.179504, 3.468236], "paragraph_keywords": ["channel", "channels", "extraction", "grid"]}, {"paragraph_vector": [39.59156, 19.657295], "paragraph_keywords": ["voronoi", "parallel", "set", "quality"]}, {"paragraph_vector": [-42.584926, 64.277786], "paragraph_keywords": ["india", "voronoi", "microsoft", "jfa"]}, {"paragraph_vector": [29.765386, -20.395338], "paragraph_keywords": ["voronoi", "visualization", "channels", "based"]}], "content": {}, "doi": "10.1109/BIBE.2008.4696741"}, {"uri": "133", "title": "Protein Structure Visualization by Dimension Reduction and Texture Mapping", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Heng Yang", "Rehman Qureshi", "Ahmet Sacan"], "summary": "Among the biological macromolecules, proteins have attracted special attention from the scientific community due to their rich functional roles. The ability to visualize and manipulate macromolecular structures on graphical display devices has facilitated the identification and analysis of these macromolecules. Structural analyses of the proteins often provide important insights into their biochemical functions. However, such analysis is often limited by the representation of protein structures and the corresponding computational resource requirements. In this study, we focus on the molecular surface of the proteins and investigate computationally and visually effective representations to serve a number of visualization and analysis purposes. Specifically, we \u201cunfold\u201d the protein surface onto a planar space, while preserving the local surface features as much as possible. In contrast to classical cartographic projections, our approach is able to preserve local shape features. Several biochemical properties associated with each surface point are mapped to generate a two dimensional map of these features. The 3D-2D mapping of the surface vertices has also been utilized to texture-map an arbitrary image back onto the protein structure to facilitate the visualization of the 3D structure.", "keywords": ["p", "data", "alignment", "structure", "value", "area", "d", "order", "analysis", "similarity", "developed", "score", "method", "evolution", "protein", "matrix", "sequence", "point", "approach", "figure", "proposed", "cartography", "surface", "based", "mapping", "map", "study", ".", "convergent", "ability", "e", "site", "s", "dimension", "feature", "given", "neighbor", "function", "problem", "comparison", "shape", "reduction"], "document_vector": [-142.364837, -9.688932], "paragraphs": [{"paragraph_vector": [-95.082344, -41.392276], "paragraph_keywords": ["protein", "sequence", "structures", "structure"]}, {"paragraph_vector": [-53.966297, -51.100379], "paragraph_keywords": ["protein", "surface", "proteins", "structures"]}, {"paragraph_vector": [10.607695, -14.290646], "paragraph_keywords": ["surface", "protein", "representation", "developed"]}, {"paragraph_vector": [2.723741, -15.056871], "paragraph_keywords": ["surface", "protein", "surfaces", "developed"]}, {"paragraph_vector": [-19.283811, -19.906623], "paragraph_keywords": ["surface", "methods", "map", "protein"]}, {"paragraph_vector": [-60.768798, -19.540866], "paragraph_keywords": ["surface", "mapping", "data", "method"]}, {"paragraph_vector": [-46.509075, -13.484806], "paragraph_keywords": ["sne", "point", "surface", "function"]}, {"paragraph_vector": [-52.478305, -18.830247], "paragraph_keywords": ["surface", "method", "neighbor", "reduction"]}, {"paragraph_vector": [-59.559997, -18.784273], "paragraph_keywords": ["score", "performance", "points", "area"]}, {"paragraph_vector": [-18.722486, -18.838983], "paragraph_keywords": ["method", "value", "shows", "neighbor"]}, {"paragraph_vector": [-15.906525, -17.667556], "paragraph_keywords": ["protein", "surface", "comparison", "methods"]}, {"paragraph_vector": [-33.403205, -45.411678], "paragraph_keywords": ["protein", "surfaces", "journal", "surface"]}, {"paragraph_vector": [-25.801794, -44.531528], "paragraph_keywords": ["surface", "protein", "proteins", "."]}], "content": {}, "doi": "10.1109/CIBCB.2012.6217228"}, {"uri": "134", "title": "Visualization of Protein Structure Relationships Using Twin Kernel Embedding", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yi Guo", "Junbin Gao"], "summary": "In this paper, a recently proposed dimensionality reduction method called Twin Kernel Embedding (TKE) is applied in 2-dimensional visualization of protein structure relationships. By matching the similarity measures of the input and the embedding spaces expressed by their respective kernels, TKE ensures that both local and global proximity information are preserved simultaneously. Experiments conducted on a subset of the Structural Classification Of Protein (SCOP) database confirmed the effectiveness of TKE in preserving the original relationships among protein structures in the lower dimensional embedding according to their similarities. This result is expected to benefit subsequent analyses of protein structures and their functions.", "keywords": ["yi", "data", "embeddings", "structure", "k", "similarity", "sammon", "space", "method", "protein", "input", "algorithm", "family", "sequence", "tke", "dr", "relationship", "based", "kernel", "function"], "document_vector": [-168.788284, 71.760879], "paragraphs": [{"paragraph_vector": [-66.055816, -16.91706], "paragraph_keywords": ["dr", "methods", "space", "protein"]}, {"paragraph_vector": [-65.881217, -20.956014], "paragraph_keywords": ["protein", "methods", "sequences", "based"]}, {"paragraph_vector": [-65.039924, -35.067382], "paragraph_keywords": ["similarity", "proximity", "input", "kernel"]}, {"paragraph_vector": [-65.886108, -12.687585], "paragraph_keywords": ["kernel", "input", "data", "space"]}, {"paragraph_vector": [-65.935012, -13.185186], "paragraph_keywords": ["kernel", "optimization", "embeddings", "input"]}, {"paragraph_vector": [-67.983032, -13.500079], "paragraph_keywords": ["tke", "kernel", "data", "space"]}, {"paragraph_vector": [-68.244613, -18.265464], "paragraph_keywords": ["families", "proteins", "figure", "tke"]}, {"paragraph_vector": [-63.480709, -19.00307], "paragraph_keywords": ["protein", "vol", "analysis", "visualization"]}, {"paragraph_vector": [-64.853019, -12.746236], "paragraph_keywords": ["vol", ".", "teodoro", "phillips"]}], "content": {}, "doi": "10.1109/ICBBE.2009.5163750"}, {"uri": "135", "title": "A 3D molecular visualization system with mobile devices", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Mariko Sasakura", "Akira Kotaki", "Junya Inada"], "summary": "It is not very intuitive to manipulate objects visualized in 3-dimension by a mouse, because a mouse can only move in 2-dimension space. We propose an interaction technique to manipulate 3D objects by mobile devices with acceleration sensors. We have developed a 3D molecular visualization system which displays results of a simulation of the molecular dynamics method. In the system, we can rotate 3D objects by leaning a mobile device and change a viewing location by moving a mobile device. We discuss how we implement the interaction technique in this paper. Keywords-Interaction technique; Mobile devices; Acceleration sensors; 3D objects;", "keywords": ["direction", "movement", "acceleration", "device", "user", "system", "figure"], "document_vector": [155.092529, -40.834346], "paragraphs": [{"paragraph_vector": [149.65625, 6.838104], "paragraph_keywords": ["device", "figure", "user", "acceleration"]}, {"paragraph_vector": [150.864761, 6.27615], "paragraph_keywords": ["device", "direction", "library", "acceleration"]}, {"paragraph_vector": [146.789642, -32.567607], "paragraph_keywords": ["system", "moving", "computer", "result"]}, {"paragraph_vector": [146.053634, -6.507898], "paragraph_keywords": ["device", "interaction", "technique", "objects"]}, {"paragraph_vector": [-154.768768, 14.515653], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/VISUAL.2005.1532828"}, {"uri": "136", "title": "New method for visualizing three-dimensional electromagnetic power absorption by capsulate liquid crystal  - Electrical Insulation and Dielectric Phenomena, 2004. CEIDP '04. 2004 Annual Report Conference on", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Y. Suzuki", "M. Baba", "M. Taki"], "summary": "A new method to measure the temperature distribution due to high frequency electromagnetic field (HFEMF) power absorption is established. It is intend to develop for S A R measurement from the temperature elevation. A micro-capsulated thermo-tropic liquid crystal (MTLC) is employed as the temperature probe because of its nature of high resolution and high sensitivity. High-molecular gel phantom with high transparency and viscosity has been made to realize the visualization technique with MTLC. Complex permittivity of the phantom is adjusted to that of muscle at 900 [MHz] and 1.45 [GHz]. The efficiency of the method is proved to visualize HF-EMF power absorption by the exposure experiment.", "keywords": [], "document_vector": [90.176795, 6.862365], "paragraphs": [{"paragraph_vector": [-132.333007, 18.700466], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/BIBM.2017.8218041"}, {"uri": "137", "title": "Ligand Excluded Surface : A New Type of Molecular Surface", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Norbert Lindow", "Daniel Baum"], "summary": "The most popular molecular surface in molecular visualization is the solvent excluded surface (SES). It provides information about the accessibility of a biomolecule for a solvent molecule that is geometrically approximated by a sphere. During a period of almost four decades, the SES has served for many purposes \u2013 including visualization, analysis of molecular interactions and the study of cavities in molecular structures. However, if one is interested in the surface that is accessible to a molecule whose shape differs significantly from a sphere, a different concept is necessary. To address this problem, we generalize the definition of the SES by replacing the probe sphere with the full geometry of the ligand defined by the arrangement of its van der Waals spheres. We call the new surface ligand excluded surface (LES) and present an efficient, grid-based algorithm for its computation. Furthermore, we show that this algorithm can also be used to compute molecular cavities that could host the ligand molecule. We provide a detailed description of its implementation on CPU and GPU. Furthermore, we present a performance and convergence analysis and compare the LES for several molecules, using as ligands either water or small organic molecules.", "keywords": ["cavity", "value", "molecule", "distance", "orientation", "sample", "le", "set", "algorithm", "compute", "ligand", "point", "number", "approach", "fig", "computation", "defined", "sphere", "surface", "computed", "binding", "position", "atom", "s", "field", "receptor", "bounding", "use", "r", "grid", "radius", "time", "spacing", "function", "conformation", "intersection"], "document_vector": [-119.462654, -40.026668], "paragraphs": [{"paragraph_vector": [19.665119, -3.245476], "paragraph_keywords": ["ses", "surface", "surfaces", "method"]}, {"paragraph_vector": [17.222333, -10.177366], "paragraph_keywords": ["ses", "ligand", "surface", "grid"]}, {"paragraph_vector": [35.161811, 37.233314], "paragraph_keywords": ["surface", "ses", "algorithm", "cavities"]}, {"paragraph_vector": [17.950332, -22.015625], "paragraph_keywords": ["cavity", "cavities", "presented", "spheres"]}, {"paragraph_vector": [13.454022, -13.543983], "paragraph_keywords": ["ligand", "r", "compute", "given"]}, {"paragraph_vector": [3.56613, -11.553439], "paragraph_keywords": ["ligand", "surface", "c", "les"]}, {"paragraph_vector": [15.847915, 6.392652], "paragraph_keywords": ["distance", "field", "ligand", "les"]}, {"paragraph_vector": [6.672497, -2.782402], "paragraph_keywords": ["ligand", "grid", "distance", "phase"]}, {"paragraph_vector": [9.05881, -3.169955], "paragraph_keywords": ["sphere", "ligand", "bounding", "spheres"]}, {"paragraph_vector": [6.533638, -5.057657], "paragraph_keywords": ["ligand", "positions", "radius", "sphere"]}, {"paragraph_vector": [8.809102, -2.453372], "paragraph_keywords": ["distance", "sphere", "ligand", "function"]}, {"paragraph_vector": [5.547967, -11.845962], "paragraph_keywords": ["ligand", "orientations", "sampled", "conformation"]}, {"paragraph_vector": [28.456769, -52.570983], "paragraph_keywords": ["orientations", "distance", "j", "k"]}, {"paragraph_vector": [8.009067, -5.98825], "paragraph_keywords": ["ligand", "points", "sample", "array"]}, {"paragraph_vector": [15.390521, 29.409393], "paragraph_keywords": ["sample", "position", "ligand", "cluster"]}, {"paragraph_vector": [4.629911, 0.603099], "paragraph_keywords": ["intersection", "receptor", "atoms", "tests"]}, {"paragraph_vector": [-7.270134, 83.63195], "paragraph_keywords": ["sphere", "ligand", "cell", "field"]}, {"paragraph_vector": [8.496949, -6.589232], "paragraph_keywords": ["distance", "points", "field", "grid"]}, {"paragraph_vector": [8.515852, -7.807097], "paragraph_keywords": ["distance", "grid", "values", "points"]}, {"paragraph_vector": [-77.416114, 78.58036], "paragraph_keywords": ["distance", "field", "ligand", "kernel"]}, {"paragraph_vector": [31.458284, 38.908309], "paragraph_keywords": ["surface", "les", "ray", "distance"]}, {"paragraph_vector": [14.995954, -17.789003], "paragraph_keywords": ["number", "orientations", "grid", "sample"]}, {"paragraph_vector": [8.485208, -10.224426], "paragraph_keywords": ["sample", "positions", "complexity", "algorithm"]}, {"paragraph_vector": [10.651855, -12.040062], "paragraph_keywords": ["ligand", "table", "memory", "distance"]}, {"paragraph_vector": [13.224613, -43.784172], "paragraph_keywords": ["les", "orientations", "binding", "cavities"]}, {"paragraph_vector": [17.301258, -16.268575], "paragraph_keywords": ["ligand", "grid", "spacing", "les"]}, {"paragraph_vector": [9.847827, -11.66811], "paragraph_keywords": ["orientations", "ligand", "time", "cavity"]}, {"paragraph_vector": [14.227244, -18.242753], "paragraph_keywords": ["computation", "ligand", "feedback", "grid"]}, {"paragraph_vector": [-153.322952, 11.72834], "paragraph_keywords": ["information"]}], "content": {}, "doi": "10.1109/VISUAL.2004.82"}, {"uri": "138", "title": "Ray Tracing and Volume Rendering Large Molecular Data on Multi-Core and Many-Core Architectures", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Aaron Knoll", "Ingo Wald", "Paul A. Navr\u00e1til", "Michael E. Papka", "Kelly P. Gaither"], "summary": "Visualizing large molecular data requires efficient means of rendering millions of data elements that combine glyphs, geometry and volumetric techniques. The geometric and volumetric loads challenge traditional rasterization-based vis methods. Ray casting presents a scalable and memoryefficient alternative, but modern techniques typically rely on GPU-based acceleration to achieve interactive rendering rates. In this paper, we present bnsView, a molecular visualization ray tracing framework that delivers fast volume rendering and ball-and-stick ray casting on both multi-core CPUs and many-core Intel R \u00a9Xeon Phi TM co-processors, implemented in a SPMD language that generates efficient SIMD vector code for multiple platforms without source modification. We show that our approach running on coprocessors is competitive with similar techniques running on GPU accelerators, and we demonstrate large-scale parallel remote visualization from TACC\u2019s Stampede supercomputer to large-format display walls using this system.", "keywords": ["xeon", "buffer", "data", "volume", "nanovol", "chosen", "spmd", "cost", "phi", "simd", "intel", "memory", "ispc", "compute", "primitive", "framework", "ball", "wald", "knoll", "ray", "ivl", "implementation", "technique", "based", "work", "hansen", "traversal", "performance", "geometry", "frame", "atom", "cpu", "-", "graphic", "gpu", "use", "stick", "r", "grid", "time", "bnsview", "paraview", "function", "rendering", "visualization", "casting", "tracing", "node", "bvh"], "document_vector": [-18.020963, -72.037422], "paragraphs": [{"paragraph_vector": [136.987213, 51.447673], "paragraph_keywords": ["visualization", "data", "copies", "stick"]}, {"paragraph_vector": [126.177444, 67.70613], "paragraph_keywords": ["visualization", "ray", "technique", "data"]}, {"paragraph_vector": [87.177574, 46.196578], "paragraph_keywords": ["rendering", "data", "visit", "paraview"]}, {"paragraph_vector": [118.919868, 68.708122], "paragraph_keywords": ["ray", "tracing", "rendering", "time"]}, {"paragraph_vector": [64.700241, 46.530506], "paragraph_keywords": ["function", "density", "charge", "basis"]}, {"paragraph_vector": [172.883255, 77.357086], "paragraph_keywords": ["vector", "simd", "data", "code"]}, {"paragraph_vector": [105.446731, 78.899742], "paragraph_keywords": ["grid", "volume", "data", "primitives"]}, {"paragraph_vector": [178.585052, 90.000083], "paragraph_keywords": ["simd", "spmd", "expressed", "partition"]}, {"paragraph_vector": [151.072204, 73.412704], "paragraph_keywords": ["data", "ray", "pass", "geometry"]}, {"paragraph_vector": [145.568511, 85.224136], "paragraph_keywords": ["ray", "traversal", "volume", "gpu"]}, {"paragraph_vector": [114.70623, 79.085021], "paragraph_keywords": ["volume", "data", "occlusion", "values"]}, {"paragraph_vector": [99.531677, 62.775222], "paragraph_keywords": ["data", "nanovol", "volume", "performance"]}, {"paragraph_vector": [47.462951, 85.031646], "paragraph_keywords": ["gpu", "performance", "bnsview", "volume"]}, {"paragraph_vector": [171.340179, 65.788825], "paragraph_keywords": ["performance", "rendering", "xeon", "scene"]}, {"paragraph_vector": [142.244003, 71.320823], "paragraph_keywords": ["ray", "rendering", "spmd", "data"]}, {"paragraph_vector": [135.616149, 68.688247], "paragraph_keywords": ["data", "science", "argonne", "framework"]}, {"paragraph_vector": [143.774856, 70.947441], "paragraph_keywords": ["visualization", "ray", "graphics", "tracing"]}, {"paragraph_vector": [101.896781, 27.272348], "paragraph_keywords": ["visualization", "knoll", "volume", "hansen"]}, {"paragraph_vector": [141.863815, 68.200241], "paragraph_keywords": ["wald", "tracing", "ray", "graphics"]}], "content": {}, "doi": "10.1109/IEEECONF51394.2020.9443364"}, {"uri": "139", "title": "Molecular Dynamics Visualization with XML and VRML", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["B. Arun", "V. Chandru", "A. D. Ganguly"], "summary": "A new Extensible Markup Language (XML) application, Molecular Dynamics Language (MoDL) has been developed. MoDL provides a simple, but powerful tool for molecular dynamics visualization and has been developed by combining, for the first time, the strengths of XML and the Virtual Reality Modeling Language. The details of MoDL, its implementation and examples of its use are presented in this paper.", "keywords": ["web", "methane", "data", "structure", "xml", "molecule", "language", "way", "simulation", "markup", "cage", "dynamic", "chemical", "bond", "head", "work", "position", "tool", "atom", "specified", "vrml", "file", "view", "specify", "user", "visualization", "provide"], "document_vector": [172.300781, -41.693058], "paragraphs": [{"paragraph_vector": [104.885887, -30.595947], "paragraph_keywords": ["molecule", "methane", "xml", "simulations"]}, {"paragraph_vector": [127.208038, 6.774358], "paragraph_keywords": ["xml", "language", "vrml", "web"]}, {"paragraph_vector": [122.206756, 2.652192], "paragraph_keywords": ["data", "vrml", "users", "file"]}, {"paragraph_vector": [120.421829, -42.75764], "paragraph_keywords": ["atoms", "bonds", "simulation", "bond"]}, {"paragraph_vector": [132.57286, -17.047176], "paragraph_keywords": ["atoms", "molecule", "translate", "object"]}, {"paragraph_vector": [114.547576, -47.843624], "paragraph_keywords": ["atoms", "simulation", "plot", "plots"]}, {"paragraph_vector": [104.527, -47.039073], "paragraph_keywords": ["vrml", "methane", "visualization", "molecule"]}, {"paragraph_vector": [79.000434, 4.481304], "paragraph_keywords": ["simulation", "user", "atoms", "provided"]}, {"paragraph_vector": [109.563934, -68.061233], "paragraph_keywords": ["molecule", "atom", "viewpoint", "user"]}, {"paragraph_vector": [126.365112, 7.498468], "paragraph_keywords": ["xml", "atoms", "vrml", "tree"]}, {"paragraph_vector": [107.96849, -33.446506], "paragraph_keywords": ["atoms", "molecule", "data", "vrml"]}, {"paragraph_vector": [123.915779, 6.044135], "paragraph_keywords": ["vrml", "visualization", "file", "process"]}], "content": {}, "doi": "10.1109/CSSS.2011.5973926"}, {"uri": "140", "title": "HPC for the Human Brain Project", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Thomas Lippert"], "summary": "The Human Brain Project, one of two European flagship projects, is a collaborative effort to reconstruct the brain, piece by piece, in multi-scale models and their supercomputer-based simulation, integrating and federating giant amounts of existing information and creating new information and knowledge about the human brain. A fundamental impact on our understanding of the human brain and its diseases as well as on novel brain-inspired computing technologies is expected. The HPC Platform will be one of the central elements of the project. Including major European supercomputing centres and several universities, its mission is to build, integrate and operate the hardware, network and software components of the supercomputing and big data infrastructures from the cell to full-scale interactive brain simulations, with data management, processing and visualization. In my contribution, I will discuss the requirements of the HBP on HPC hardware and software technology. These requirements follow the multi-scale approach of the HBP to decode the brain and recreate it virtually. On the cellular level, hardware-software architectures for quantum mechanical ab-initio molecular dynamics methods and for classical molecular dynamics methods will be included in the platform. On the level of the full-scale brain simulation, on the one hand, a development system to \u201cbuild\u201d the brain by integration of all accessible data distributed worldwide as well as for tests and evaluation of the brain software is foreseen, and, on the other hand, a system that acts as the central brain simulation facility, eventually allowing for interactive simulation and visualization of the entire human brain. Additionally, the brain needs to be equipped with the proper sensory environment, a body, provided by virtual robotics codes developed on a suitable hardware system. It is expected that the human brain project can trigger innovative solutions for future exascale architectures permitting hierarchical memory structures and interactive operation.", "keywords": ["copy", "director"], "document_vector": [-17.736612, 12.159543], "paragraphs": [{"paragraph_vector": [-132.687896, 62.627937], "paragraph_keywords": ["copies", "director", "simulation", "j\u00fclich"]}], "content": {}, "doi": "10.1109/INFVIS.1998.729566"}, {"uri": "141", "title": "empty", "timestamp": "2003", "rating": "0.0", "annotation": "", "tags": [], "authors": ["JOHN M. FRITSCH", "ROBERT A. ELLIS", "THOMAS H. JACOBI", "GARLAND R. MARSHALt"], "summary": "-A specialized graphics systems has been developed for protein research; this system achieves real-time display and manipulation for line-segment models containing up to 4095 end points. The hardware consists of three components: (l) a collection of hardware subroutines assembled from digital computer building blocks called macromodules, (2) vector-matrix transformation and display equipment and (3) a small stored-program computer. This paper provides a general description of the system. This work has been supported by the Division of Research Resources of the National Institutes of Health under Grant RR-00396. The development of macromodules has been partially supported by the Advanced Research Projects Agency of the Department of Defense under Contract SD-302. One of the authors (GRM) is partially supported by an Established Investigatorship from the American Heart Association.", "keywords": [], "document_vector": [-66.785217, -21.716693], "paragraphs": [{"paragraph_vector": [-129.622329, 21.445652], "paragraph_keywords": []}], "content": {}, "doi": "10.23919/EUSIPCO.2019.8902695"}, {"uri": "142", "title": "A Probabilistic Method to Find and Visualize Distinct Regions in Protein Sequences", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Morteza Hosseini", "Diogo Pratas", "Armando J. Pinho"], "summary": "Studies on identification of species-specific protein regions, i.e., unique or highly dissimilar regions with respect to close species, will lead us to understanding of evolutionary traits, which can be related to novel functionalities or diseases. In this paper, we propose an alignment-free method to find and visualize distinct regions between two collections of proteins. We applied the proposed method, FRUIT, on multiple synthetic and real datasets to analyze its behavior when different rates of substitutional mutation occur. Testing with different k-mer sizes showed that the higher the mutation rate, the higher the relative uniqueness. We also employed FRUIT to find and visualize distinct regions in modern human proteins relatively to the proteins of Altai, Sidron and Vindija Neanderthals. The results show that four of the most distinct proteins, named ataxin-8, 60S ribosomal protein L26, NADH-ubiquinone oxidoreductase chain 3 and cytochrome c oxidase subunit 2 are involved in SCA8, DBA11, LS and MT-C1D, and MT-C4D diseases, respectively. There is also Interferon-induced transmembrane protein 3, among others, which is part of the immune system. Besides, we report the most similar primate exomes to the found modern human one, in terms of identity, query cover and length of sequences. The reported results can give us insight to the evolution of proteomes.", "keywords": ["filter", "file", "use", "m", "protein", "fruit", "map", "neanderthal", "position", "k", "found", "number", "hash", "size", "reference", "target"], "document_vector": [-93.641746, 63.202316], "paragraphs": [{"paragraph_vector": [-88.009002, -40.044857], "paragraph_keywords": ["k", "sequence", "use", "size"]}, {"paragraph_vector": [-90.443161, -41.623798], "paragraph_keywords": ["m", "fruit", "hash", "function"]}, {"paragraph_vector": [-93.836235, -47.628032], "paragraph_keywords": ["size", "filter", "file", "k"]}, {"paragraph_vector": [-96.074409, -50.754901], "paragraph_keywords": ["acids", "mutation", "dataset", "k"]}, {"paragraph_vector": [-99.797676, -44.356716], "paragraph_keywords": ["k", "neanderthals", "table", "target"]}, {"paragraph_vector": [-99.173965, -43.943401], "paragraph_keywords": ["proteins", "protein", "xp", "find"]}, {"paragraph_vector": [-98.680152, -44.19297], "paragraph_keywords": ["protein", "found", "gorilla", "subunit"]}], "content": {}, "doi": "10.1109/TVCG.2017.2772237"}, {"uri": "143", "title": "Visualization of Anisotropic Contact Potentials within Protein Structures", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Corinna Vehlow", "Bernhard Preim", "Michael Lappe"], "summary": "The use of local covalent geometry for quality assessment and refinement of protein structure models is a well-established methodology. The question arises whether information on non-covalent geometry contained within resolved structures can be harnessed to improve structure prediction. Moreover, incorporation of different combinations of priors would pave the way towards multi-body potentials. Existing empirical force-fields do not facilitate an interactive exploration of the parameter space and an assignment of spatial propensities to contacts. Hence, we investigate the possibility of making such propensities available for synergistic modeling. We present an approach that facilitates the extraction and analysis of anisotropic contact potentials for a multitude of parameters describing an amino acid and the conditions within its microenvironment. For this purpose, two novel visualization principles will be introduced. The first visualization illustrates anisotropic residuedependent contact density potentials in the form of a map projection. A second visualization is overlaid onto this, showing similar local neighborhoods as abstract traces of residues contained within each individual neighborhood. The Contact Geometry Analysis Plugin (CGAP) (for CMView) we developed allows incorporation of geometric orientation propensities into the process of interactive protein modeling and can be used for the generation of improved energy functions. It further supports the analysis of model quality, as it directly illustrates model consistency with known spatial propensities which, in turn, enables users to detect possible structural errors.", "keywords": ["propensity", "data", "structure", "predicted", "analysis", "distance", "orientation", "score", "method", "contact", "protein", "sequence", "information", "number", "template", "trace", "figure", "defined", "jres", "prediction", "based", "map", "derived", "type", "constraint", "node", "model", "potential", "environment", "anisotropic", "given", "visualization", "residue", "neighborhood"], "document_vector": [-102.630302, 35.294574], "paragraphs": [{"paragraph_vector": [-121.168838, -73.551841], "paragraph_keywords": ["structure", "protein", "contact", "distance"]}, {"paragraph_vector": [-44.488304, -55.316219], "paragraph_keywords": ["structures", "techniques", "derived", "prediction"]}, {"paragraph_vector": [-28.827596, -62.779418], "paragraph_keywords": ["orientation", "propensities", "constraints", "contacts"]}, {"paragraph_vector": [-12.766496, -67.804748], "paragraph_keywords": ["visualization", "methods", "protein", "distance"]}, {"paragraph_vector": [-14.94585, -58.906688], "paragraph_keywords": ["contacts", "maps", "orientation", "distance"]}, {"paragraph_vector": [-25.692926, -60.064353], "paragraph_keywords": ["propensities", "contact", "protein", "potentials"]}, {"paragraph_vector": [-60.008571, -49.896636], "paragraph_keywords": ["pairwise", "structures", "data", "c\u03b1"]}, {"paragraph_vector": [-26.251659, -57.380844], "paragraph_keywords": ["jres", "contact", "defined", "space"]}, {"paragraph_vector": [-35.60765, -62.029277], "paragraph_keywords": ["sequence", "residues", "contacting", "residue"]}, {"paragraph_vector": [-25.873895, -59.656303], "paragraph_keywords": ["propensities", "traces", "potentials", "orientation"]}, {"paragraph_vector": [-26.438398, -58.677497], "paragraph_keywords": ["traces", "neighborhood", "contacts", "orientation"]}, {"paragraph_vector": [-44.040485, -76.161811], "paragraph_keywords": ["neighborhood", "structure", "residues", "type"]}, {"paragraph_vector": [-18.414962, -56.529708], "paragraph_keywords": ["cluster", "nodes", "algorithm", "density"]}, {"paragraph_vector": [-27.126434, -60.772365], "paragraph_keywords": ["contact", "neighborhood", "traces", "derived"]}, {"paragraph_vector": [-26.23134, -60.688423], "paragraph_keywords": ["models", "predicted", "traces", "orientation"]}, {"paragraph_vector": [-24.733179, -57.769126], "paragraph_keywords": ["score", "propensities", "orientation", "visualization"]}, {"paragraph_vector": [-22.873489, -62.800815], "paragraph_keywords": ["constraints", "structure", "quality", "orientation"]}, {"paragraph_vector": [19.925197, -59.41146], "paragraph_keywords": ["structure", "development", "step", "docking"]}], "content": {}, "doi": "10.1109/VISUAL.2004.47"}, {"uri": "144", "title": "uPy: A Ubiquitous CG Python API with Biological-Modeling Applications", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ludovic Autin", "Michel Sanner"], "summary": "Today\u2019s general-purpose 3D computer graphics software programs (which we call hosts) model, simulate, animate, and render everything from planets to people. Example hosts include Cinema 4D, Maya, and Blender. At the same time, people use scientific and other domaincompetent 3D software to model, simulate, visualize, and analyze data, typically from domain-specific entities. Such entities include molecules, internal organs, ecosystems, and human-engineered structures. Many recent plug-ins adapt hosts to handle such domainspecific data\u2014for example, BioBlender (http://bioblender.eu), Blender for robotics (http:// wiki.blender.org/index.php/ Robotics:", "keywords": ["cinema", "gamer", "data", "api", "analysis", "calcium", "application", "helper", "plug", "create", "mesh", "host", "research", "computer", "object", "figure", "adaptor", "generate", "sphere", "code", "example", ".", "python", "in", "ui", "geometry", "epmv", "autofill", "upy", "graphic", "supported", "file", "model", "software", "time", "domain", "user", "visualization", "modeling", "gui"], "document_vector": [106.039413, -69.101844], "paragraphs": [{"paragraph_vector": [124.152183, 29.225755], "paragraph_keywords": ["python", "host", "hosts", "data"]}, {"paragraph_vector": [124.554153, 29.752172], "paragraph_keywords": ["host", "upy", "plug", "create"]}, {"paragraph_vector": [121.780357, 29.451835], "paragraph_keywords": ["def", "host", "code", "create"]}, {"paragraph_vector": [115.512069, 34.088474], "paragraph_keywords": ["instances", "geometry", "color", "domain"]}, {"paragraph_vector": [125.267227, 28.923847], "paragraph_keywords": ["host", "data", "function", "hosts"]}, {"paragraph_vector": [108.929603, 28.499092], "paragraph_keywords": ["epmv", "host", "meshes", "gui"]}, {"paragraph_vector": [111.790313, 27.730476], "paragraph_keywords": ["gamer", "mesh", "blender", "figure"]}, {"paragraph_vector": [102.178894, 27.00127], "paragraph_keywords": ["csmol", "data", "calcium", "analysis"]}, {"paragraph_vector": [97.070541, 29.731094], "paragraph_keywords": ["autofill", "ingredient", "generate", "figure"]}, {"paragraph_vector": [93.525222, 33.672431], "paragraph_keywords": ["model", "time", "produce", "tools"]}, {"paragraph_vector": [60.504047, 51.050907], "paragraph_keywords": ["modeling", "hosts", "upy", "python"]}, {"paragraph_vector": [124.422973, 31.614385], "paragraph_keywords": ["hosts", "host", "plug", "python"]}, {"paragraph_vector": [122.484199, 29.914102], "paragraph_keywords": [".", "grant", "supported", "hosts"]}, {"paragraph_vector": [86.442848, -60.284984], "paragraph_keywords": ["research", "university", "phd", "computer"]}], "content": {}, "doi": "10.1109/BIBM.2010.5706650"}, {"uri": "145", "title": "Real-time Nanoscale Visualization of Biological Molecules at Work with High-speed Atomic Force Microscopy", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Takayuki Uchihashi"], "summary": "Atomic force microscopy (AFM) is a powerful tool to visualize biological molecules at nm resolution in liquid conditions and thus has been applied to a wide variety of biological molecules. One of the most coveted functions of AFM is \u201cfast recording\u201d because it should allow us to directly visualize dynamic processes of biological molecules at work. We have been challenging to break this limitation over 15 years and established fast imaging of dynamic molecular processes. The most advanced high-speed AFM can now capture successive images at 30\u201360 ms/frame. Further, the interaction force between the tip and the sample, which significantly influences on soft biological molecules, is greatly reduced without deterioration of the scanning performance. In this short paper, typical applications of HS-AFM on biological and artificial molecules will be demonstrated. Keywords\u2014 tomic force microscopy, high-speed AFM, protein dynamics, bacteriorhodopsin, porphyrin nanofiber", "keywords": ["photocycle", "afm", "state", "light", "change", "h", "illumination", "nm", "solution", "time", "gap", "molecule", "trimer", "crystal", "br", "figure", "s"], "document_vector": [88.965843, 21.32257], "paragraphs": [{"paragraph_vector": [-21.895023, 16.480987], "paragraph_keywords": ["crystal", "br", "trimers", "trimer"]}, {"paragraph_vector": [-22.030206, 17.502614], "paragraph_keywords": ["br", "state", "time", "kbt"]}, {"paragraph_vector": [-20.279348, 16.189426], "paragraph_keywords": ["ph", "lifetime", "irradiation", "change"]}, {"paragraph_vector": [-22.424577, 16.35279], "paragraph_keywords": ["br", "state", "illumination", "light"]}, {"paragraph_vector": [-27.03193, 22.800058], "paragraph_keywords": ["figure", "state", "illumination", "br"]}, {"paragraph_vector": [-27.643558, 25.20762], "paragraph_keywords": ["gap", "force", "probe", "solution"]}, {"paragraph_vector": [-29.155763, 24.490198], "paragraph_keywords": ["afm", "phenomena", "molecule", "hs"]}], "content": {}, "doi": "10.1109/E-SCIENCE.2006.261124"}, {"uri": "146", "title": "PocketMol a molecular visualization tool for the pocket PC - Bioinformatics and Bioengineering Conference, 2001. Proceedings of the IEEE 2nd International Symp", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Jason R. Gilder", "Michael Raymer", "Travis Doom"], "summary": "Molecular visualization programs are available on many platforms. They allow a user to visualize and manipulate molecular structures. PocketMol provides the same functionality on a Pocket PC handheld computer. Using standard protein data bank (pdb) files, the user can move, rotate, and scale a protein to explore its structure and function. The user can choose from a standard backbone view or a simplified view using only alpha carbon atoms. PocketMolGX uses the Microsoft Game API to provide fast animation that is quite smooth. PocketMol is designed as an aid for those wishing to explore or demonstrate protein structures without the availability of a fullsize computer.", "keywords": ["display", "color", "protein", "pc", "development", "us", "pocket", "pocketmol", "program", "user", "atom"], "document_vector": [-104.073036, -29.407855], "paragraphs": [{"paragraph_vector": [127.069808, 24.147123], "paragraph_keywords": ["atoms", "protein", "development", "stylus"]}, {"paragraph_vector": [120.085105, -29.710344], "paragraph_keywords": ["display", "uses", "atoms", "pocket"]}, {"paragraph_vector": [107.84201, -57.277214], "paragraph_keywords": ["protein", "pocketmol", "visualization", "color"]}], "content": {}, "doi": "10.1109/PACIFICVIS.2011.5742372"}, {"uri": "147", "title": "Dynamic Influence Networks for Rule-based Models", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Angus G. Forbes", "Andrew Burks", "Kristine Lee", "Xing Li", "Pierre Boutillier", "Jean Krivine", "Walter Fontana"], "summary": "We introduce the Dynamic Influence Network (DIN), a novel visual analytics technique for representing and analyzing rulebased models of protein-protein interaction networks. Rule-based modeling has proved instrumental in developing biological models that are concise, comprehensible, easily extensible, and that mitigate the combinatorial complexity of multi-state and multi-component biological molecules. Our technique visualizes the dynamics of these rules as they evolve over time. Using the data produced by KaSim, an open source stochastic simulator of rule-based models written in the Kappa language, DINs provide a node-link diagram that represents the influence that each rule has on the other rules. That is, rather than representing individual biological components or types, we instead represent the rules about them (as nodes) and the current influence of these rules (as links). Using our interactive DIN-Viz software tool, researchers are able to query this dynamic network to find meaningful patterns about biological processes, and to identify salient aspects of complex rule-based models. To evaluate the effectiveness of our approach, we investigate a simulation of a circadian clock model that illustrates the oscillatory behavior of the KaiC protein phosphorylation cycle.", "keywords": ["fire", "leg", "p", "din", "data", "panel", "link", "marking", "order", "analysis", "et", "mechanism", "graph", "rule", "simulation", "interaction", "represent", "dynamic", "influence", "cluster", "activity", "protein", "firing", "network", "agent", "visualizing", "information", "approach", "number", "causality", "system", "clustering", "c", "selected", "biology", "pinning", "state", "introduce", "reaction", "process", "technique", "help", "based", "viz", ".", "level", "work", "phosphorylation", "tool", "element", "animation", "edge", "site", "show", "provides", "dephosphorylation", "r", "model", "use", "kappa", "time", "b", "al", "t", "pattern", "user", "visualization", "provide", "modeling", "forbes", "behavior", "node"], "document_vector": [21.294836, 66.94664], "paragraphs": [{"paragraph_vector": [-157.115859, -26.414024], "paragraph_keywords": ["systems", "system", "level", "dynamics"]}, {"paragraph_vector": [-157.789855, -30.23476], "paragraph_keywords": ["rules", "based", "entities", "rule"]}, {"paragraph_vector": [-158.186492, -25.105278], "paragraph_keywords": ["rule", "rules", "system", "based"]}, {"paragraph_vector": [-157.152786, -25.690618], "paragraph_keywords": ["systems", "rules", "influence", "based"]}, {"paragraph_vector": [-159.533218, -27.827007], "paragraph_keywords": ["systems", "rule", "rules", "simulation"]}, {"paragraph_vector": [-156.890319, -27.264301], "paragraph_keywords": ["system", "interactions", "level", "cascades"]}, {"paragraph_vector": [-158.744049, -31.563446], "paragraph_keywords": ["rules", "protein", "proteins", "interaction"]}, {"paragraph_vector": [-159.031906, -30.159233], "paragraph_keywords": ["system", "models", "behavior", "based"]}, {"paragraph_vector": [-156.432785, -24.761396], "paragraph_keywords": ["rule", "system", "based", "analytics"]}, {"paragraph_vector": [-156.88739, -27.743795], "paragraph_keywords": ["systems", "rules", "visualization", "elements"]}, {"paragraph_vector": [-160.433486, -27.032375], "paragraph_keywords": ["models", "kappa", "modeling", "rules"]}, {"paragraph_vector": [-149.40303, -23.696222], "paragraph_keywords": ["time", "nodes", "data", "user"]}, {"paragraph_vector": [-157.834655, -27.447589], "paragraph_keywords": ["causality", "introduce", "systems", "visualization"]}, {"paragraph_vector": [-158.1492, -29.141252], "paragraph_keywords": ["rule", "based", "models", "nodes"]}, {"paragraph_vector": [-153.296646, -33.925327], "paragraph_keywords": ["rules", "system", "network", "panel"]}, {"paragraph_vector": [-154.890274, -24.078882], "paragraph_keywords": ["order", "system", "user", "din"]}, {"paragraph_vector": [-152.634262, -24.818832], "paragraph_keywords": ["rule", "activity", "network", "system"]}, {"paragraph_vector": [-159.421096, -25.165752], "paragraph_keywords": ["elements", "modeling", "information", "rule"]}, {"paragraph_vector": [-155.734176, -27.063705], "paragraph_keywords": ["time", "rules", "et", "al"]}, {"paragraph_vector": [-150.69873, -25.323919], "paragraph_keywords": ["data", "providing", "network", "nodes"]}, {"paragraph_vector": [-155.598724, -26.950719], "paragraph_keywords": ["causality", "introduce", "tasks", "visualization"]}, {"paragraph_vector": [-155.951644, -26.982912], "paragraph_keywords": ["rule", "based", "nodes", "et"]}, {"paragraph_vector": [-154.488998, -24.856555], "paragraph_keywords": ["rules", "system", "network", "panel"]}, {"paragraph_vector": [-153.304382, -23.27906], "paragraph_keywords": ["order", "influence", "rules", "rule"]}, {"paragraph_vector": [-157.318634, -56.902515], "paragraph_keywords": ["r", "rule", "activity", "system"]}, {"paragraph_vector": [-157.039901, -21.582658], "paragraph_keywords": ["din", "panel", "rules", "influence"]}, {"paragraph_vector": [-151.317565, -24.457036], "paragraph_keywords": ["rules", "influence", "data", "time"]}, {"paragraph_vector": [-140.581619, -24.783924], "paragraph_keywords": ["edges", "rules", "influence", "cluster"]}, {"paragraph_vector": [-149.250183, -24.477725], "paragraph_keywords": ["clusters", "rules", "nodes", "user"]}, {"paragraph_vector": [-148.489776, -22.864542], "paragraph_keywords": ["time", "user", "clustering", "influence"]}, {"paragraph_vector": [-151.823989, -20.831808], "paragraph_keywords": ["user", "clusters", "influence", "network"]}, {"paragraph_vector": [-149.499649, -23.801303], "paragraph_keywords": ["rules", "marking", "pinning", "influence"]}, {"paragraph_vector": [-154.239822, -16.811569], "paragraph_keywords": ["time", "r", "influence", "s"]}, {"paragraph_vector": [-155.18695, -21.837165], "paragraph_keywords": ["rules", "din", "influence", "panel"]}, {"paragraph_vector": [-152.818069, -21.782653], "paragraph_keywords": ["influence", "links", "data", "link"]}, {"paragraph_vector": [-140.497848, -27.592561], "paragraph_keywords": ["edges", "cluster", "rules", "clusters"]}, {"paragraph_vector": [-149.894943, -22.234922], "paragraph_keywords": ["rules", "clustering", "time", "clusters"]}, {"paragraph_vector": [-151.598617, -23.454952], "paragraph_keywords": ["time", "user", "influence", "value"]}, {"paragraph_vector": [-143.531265, -25.229736], "paragraph_keywords": ["rules", "clusters", "pinning", "grouping"]}, {"paragraph_vector": [-151.581832, -20.25385], "paragraph_keywords": ["rules", "time", "influence", "animation"]}, {"paragraph_vector": [-153.386764, -21.936647], "paragraph_keywords": ["time", "influence", "panel", "network"]}, {"paragraph_vector": [-152.68254, -15.57727], "paragraph_keywords": ["system", "kaia", "kaic", "phosphorylation"]}, {"paragraph_vector": [-155.531326, -12.018984], "paragraph_keywords": ["p", "dephosphorylation", "phosphorylation", "bound"]}, {"paragraph_vector": [-155.343307, -15.666357], "paragraph_keywords": ["nodes", "time", "number", "sites"]}, {"paragraph_vector": [-157.500915, -17.748064], "paragraph_keywords": ["nodes", "influence", "structure", "rules"]}, {"paragraph_vector": [-155.625335, -13.657337], "paragraph_keywords": ["c", "y", "nodes", "din"]}, {"paragraph_vector": [-156.631347, -18.214361], "paragraph_keywords": ["rules", "x", "c", "b"]}, {"paragraph_vector": [-153.801574, -13.116993], "paragraph_keywords": ["agents", "b", "leg", "level"]}, {"paragraph_vector": [-154.236083, -21.835088], "paragraph_keywords": ["time", "influence", "panel", "rules"]}, {"paragraph_vector": [-154.391067, -15.40349], "paragraph_keywords": ["kaia", "phosphorylation", "kaic", "system"]}, {"paragraph_vector": [-154.516387, -13.858138], "paragraph_keywords": ["p", "dephosphorylation", "kaic", "bound"]}, {"paragraph_vector": [-154.565093, -15.526788], "paragraph_keywords": ["nodes", "time", "number", "analysis"]}, {"paragraph_vector": [-155.724441, -21.383695], "paragraph_keywords": ["nodes", "influence", "structure", "session"]}, {"paragraph_vector": [-153.347, -13.698093], "paragraph_keywords": ["y", "nodes", "din", "c"]}, {"paragraph_vector": [-152.369201, -19.299734], "paragraph_keywords": ["rules", "c", "x", "b"]}, {"paragraph_vector": [-155.476119, -10.724513], "paragraph_keywords": ["b", "agents", "level", "leg"]}, {"paragraph_vector": [-154.134185, -13.296368], "paragraph_keywords": ["c", "b", "system", "din"]}, {"paragraph_vector": [-154.646087, -12.680045], "paragraph_keywords": ["agents", "leg", "b", "level"]}, {"paragraph_vector": [-158.011062, -26.54754], "paragraph_keywords": ["biology", "din", "user", "systems"]}, {"paragraph_vector": [-154.541931, -24.503728], "paragraph_keywords": ["rules", "range", "rule", "din"]}, {"paragraph_vector": [-151.657531, -30.848056], "paragraph_keywords": ["visualization", "support", "networks", "modeling"]}, {"paragraph_vector": [-161.376846, -28.398946], "paragraph_keywords": ["systems", "networks", "visualization", "based"]}, {"paragraph_vector": [-152.242202, -39.792789], "paragraph_keywords": ["networks", "systems", "directed", "modeling"]}, {"paragraph_vector": [-150.350189, -29.871496], "paragraph_keywords": ["journal", "visualizing", "forbes", "analysis"]}, {"paragraph_vector": [-154.634201, -13.053636], "paragraph_keywords": ["b", "c", "system", "fires"]}, {"paragraph_vector": [-154.125289, -12.597865], "paragraph_keywords": ["agents", "b", "c", "leg"]}, {"paragraph_vector": [-156.033386, -24.531042], "paragraph_keywords": ["biology", "din", "feedback", "systems"]}, {"paragraph_vector": [-154.862197, -28.910549], "paragraph_keywords": ["din", "range", "system", "use"]}, {"paragraph_vector": [-151.802917, -28.299991], "paragraph_keywords": ["visualization", "rule", "support", "rules"]}, {"paragraph_vector": [-158.265396, -28.338233], "paragraph_keywords": ["networks", "systems", "based", "rule"]}, {"paragraph_vector": [-154.392639, -31.311998], "paragraph_keywords": ["systems", "visualization", "directed", "modeling"]}, {"paragraph_vector": [-151.655059, -29.072198], "paragraph_keywords": ["journal", "biology", "analysis", "visualization"]}, {"paragraph_vector": [-154.715682, -24.814161], "paragraph_keywords": ["visualization", "modelling", "simulation", "visualizing"]}, {"paragraph_vector": [-157.379455, -27.266986], "paragraph_keywords": ["visualization", "causality", "biology", "workshop"]}], "content": {}, "doi": "10.1109/ICAICT.2012.6398539"}, {"uri": "148", "title": "Metamorphers: Storytelling TemplatesFor Illustrative Animated Transitions in Molecular Visualization", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Johannes Sorger", "Peter Mindek", "Peter Rautek", "Eduard Gr\u00f6ller", "Graham Johnson", "Ivan Viola"], "summary": "In molecular biology, illustrative animations are used to convey complex biological phenomena to broad audiences. However, such animations have to be manually authored in 3D modeling software, a time consuming task that has to be repeated from scratch for every new data set, and requires a high level of expertise in illustration, animation, and biology. We therefore propose metamorphers: a set of operations for defining animation states as well as the transitions to them in the form of re-usable storytelling templates. The re-usability is two-fold. Firstly, due to their modular nature, metamorphers can be re-used in different combinations to create a wide range of animations. Secondly, due to their abstract nature, metamorphers can be re-used to re-create an intended animation for a wide range of compatible data sets. Metamorphers thereby mask the low-level complexity of explicit animation specifications by exploiting the inherent properties of the molecular data, such as the position, size, and hierarchy level of a semantic data subset. We demonstrate the re-usability of our technique based on the authoring and application of two animation use-cases to three molecular data sets.", "keywords": ["subset", "data", "structure", "transition", "hierarchy", "molecule", "order", "et", "create", "metamorphers", "set", "camera", "input", "storytelling", "point", "control", "metamorpher", "scale", "object", "authoring", "defined", "state", "box", "technique", "compartment", "interface", "work", "level", "frame", "-", "animation", "s", "bounding", "view", "instance", "time", "al", "property", "user", "visualization", "node"], "document_vector": [-79.327598, -20.286376], "paragraphs": [{"paragraph_vector": [119.628822, -22.540702], "paragraph_keywords": ["animation", "biology", "animations", "level"]}, {"paragraph_vector": [123.505134, -22.151954], "paragraph_keywords": ["metamorphers", "level", "data", "user"]}, {"paragraph_vector": [122.94857, -21.944972], "paragraph_keywords": ["data", "visualization", "animation", "implementation"]}, {"paragraph_vector": [125.035385, -20.344768], "paragraph_keywords": ["system", "animation", "animations", "generates"]}, {"paragraph_vector": [121.175064, -26.021671], "paragraph_keywords": ["data", "animation", "state", "transition"]}, {"paragraph_vector": [120.199249, -24.949207], "paragraph_keywords": ["data", "metamorpher", "level", "properties"]}, {"paragraph_vector": [84.66764, -30.943544], "paragraph_keywords": ["molecules", "control", "animation", "data"]}, {"paragraph_vector": [95.487777, -25.353651], "paragraph_keywords": ["data", "subsets", "hierarchy", "node"]}, {"paragraph_vector": [78.259269, -31.278038], "paragraph_keywords": ["data", "bounding", "molecules", "subsets"]}, {"paragraph_vector": [110.003028, -25.683221], "paragraph_keywords": ["molecules", "transition", "positions", "models"]}, {"paragraph_vector": [120.063179, -25.662115], "paragraph_keywords": ["point", "molecules", "animation", "time"]}, {"paragraph_vector": [122.322456, -27.470033], "paragraph_keywords": ["camera", "animation", "subset", "molecule"]}, {"paragraph_vector": [112.385253, -11.904313], "paragraph_keywords": ["metamorphers", "time", "editor", "interface"]}, {"paragraph_vector": [119.65419, -22.910037], "paragraph_keywords": ["node", "time", "frames", "animation"]}, {"paragraph_vector": [88.640518, -21.876173], "paragraph_keywords": ["data", "hiv", "view", "virion"]}, {"paragraph_vector": [87.198852, -30.992086], "paragraph_keywords": ["metamorpher", "data", "compartments", "bounding"]}, {"paragraph_vector": [85.892677, -27.254302], "paragraph_keywords": ["molecule", "bar", "molecules", "metamorpher"]}, {"paragraph_vector": [92.46952, -27.420787], "paragraph_keywords": ["data", "structure", "bundling", "metamorpher"]}, {"paragraph_vector": [114.06961, -18.676301], "paragraph_keywords": ["data", "structures", "visualization", "scale"]}, {"paragraph_vector": [120.873687, -24.615695], "paragraph_keywords": ["data", "metamorphers", "point", "creating"]}, {"paragraph_vector": [121.313339, -22.992254], "paragraph_keywords": ["transitions", "work", "create", "transition"]}, {"paragraph_vector": [121.860435, -25.031476], "paragraph_keywords": ["metamorphers", "project", "list", "expertise"]}], "content": {}, "doi": "10.1109/TVCG.2020.3030415"}, {"uri": "149", "title": "LC Passive Wireless Sensors Toward a Wireless Sensing Platform: Status, Prospects, and Challenges", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Qing-An Huang", "Lei Dong", "Li-Feng Wang"], "summary": "Inductor\u2013capacitor (LC) passive wireless sensors use a transformer with loose coupling between an external readout coil and an inductor that receives power through this inductive coupling. Changes in the sensor are wirelessly and remotely detected by the readout coil, which makes them highly useful in applications that require the sensor to be powered remotely and to occupy a small volume, such as harsh and sealed environments, where physical access to the sensor is difficult. Although the sensor to accomplish this function dates from the 1960\u2019s, its rapid extension over the past decades has benefited from microelectromechanical systems. This paper provides an overview of the status and challenges in the LC passive wireless sensor toward a wireless sensing platform. The basic sensing principles are first categorized into detecting changes of the sensor in response to the capacitance, resistance, inductance, or coupling distance due to the parameter of interest through monitoring the impedance magnitude and phase spectrum. The present state of the art in sensor applications for pressure, strain, temperature, humidity, biochemical, gas, and so on is then reviewed and compared. For emerging applications from many Internet of Things scenarios, geometrical constraints, such as small and noninvasive coils, reduce the magnetic coupling between the sensor and the readout coil, resulting in a limited interrogation distance. Furthermore, an increasing number of applications also require the simultaneous measurement of multiple parameters. Recent efforts to increase the interrogation distance and to extend the measurement of single parameter to multiple parameters are finally outlined. [2016-0093]", "keywords": ["q", "substrate", "structure", "k", "lc", "distance", "parameter", "application", "developed", "inductor", "coil", "inductance", "electrode", "detection", "method", "monitor", "factor", "silicon", "capacitor", "tank", "input", "switch", "pressure", "shown", "frequency", "power", "l", "sensor", "voltage", "planar", "hydrogel", "system", "humidity", "proposed", "change", "based", "capacitance", "size", "measurement", "technology", "temperature", "term", "impedance", "circuit", "readout", "signal", "dielectric", "monitoring", "phase", "material", "sensing", "varactor", "coupling", "test"], "document_vector": [89.83448, 9.045848], "paragraphs": [{"paragraph_vector": [-56.774795, 23.917478], "paragraph_keywords": ["sensor", "frequency", "zin", "coil"]}, {"paragraph_vector": [-57.810504, 24.556007], "paragraph_keywords": ["capacitor", "q", "inductor", "sensor"]}, {"paragraph_vector": [-55.964584, 27.075227], "paragraph_keywords": ["capacitance", "fringing", "capacitor", "\u03be"]}, {"paragraph_vector": [-53.235084, 28.619331], "paragraph_keywords": ["l", "inductor", "junction", "bias"]}, {"paragraph_vector": [-56.420379, 26.49402], "paragraph_keywords": ["resistance", "t", "metal", "q"]}, {"paragraph_vector": [-57.228092, 24.183944], "paragraph_keywords": ["capacitor", "method", "sensors", "capacitance"]}, {"paragraph_vector": [-56.708824, 25.030595], "paragraph_keywords": ["inductor", "capacitor", "varactor", "inductance"]}, {"paragraph_vector": [-57.488803, 26.011985], "paragraph_keywords": ["inductance", "resistor", "inductor", "resistance"]}, {"paragraph_vector": [-57.962574, 25.952795], "paragraph_keywords": ["coil", "signal", "k", "measurement"]}, {"paragraph_vector": [-58.076301, 24.560766], "paragraph_keywords": ["detection", "readout", "impedance", "sensor"]}, {"paragraph_vector": [-58.224182, 25.91378], "paragraph_keywords": ["pressure", "sensor", "monitoring", "applications"]}, {"paragraph_vector": [-53.46014, 28.446813], "paragraph_keywords": ["sensor", "pressure", "capacitor", "needle"]}, {"paragraph_vector": [-55.03833, 26.930564], "paragraph_keywords": ["pressure", "sensor", "silicon", "process"]}, {"paragraph_vector": [-56.271003, 25.020854], "paragraph_keywords": ["tires", "strain", "sensor", "temperature"]}, {"paragraph_vector": [-56.657768, 23.321018], "paragraph_keywords": ["temperature", "sensor", "structure", "c"]}, {"paragraph_vector": [-55.709083, 26.655704], "paragraph_keywords": ["humidity", "water", "capacitance", "planar"]}, {"paragraph_vector": [-55.581104, 26.0825], "paragraph_keywords": ["sensor", "monitoring", "bacteria", "printed"]}, {"paragraph_vector": [-54.373119, 26.353221], "paragraph_keywords": ["hydrogel", "ph", "sensor", "varactor"]}, {"paragraph_vector": [-56.936115, 24.400127], "paragraph_keywords": ["sensor", "gas", "vehicles", "hydrogel"]}, {"paragraph_vector": [-56.508804, 26.42737], "paragraph_keywords": ["radiation", "devices", "position", "switch"]}, {"paragraph_vector": [-55.995796, 25.893535], "paragraph_keywords": ["sensors", "material", "development", "disease"]}, {"paragraph_vector": [-56.198257, 24.764833], "paragraph_keywords": ["sensors", "food", "cost", "devices"]}, {"paragraph_vector": [-56.300552, 24.92481], "paragraph_keywords": ["distance", "inductor", "coil", "sensor"]}, {"paragraph_vector": [-56.783294, 24.64361], "paragraph_keywords": ["distance", "factor", "sensor", "impedance"]}, {"paragraph_vector": [-57.271682, 24.71713], "paragraph_keywords": ["sensor", "coupling", "frequency", "distance"]}, {"paragraph_vector": [-58.44458, 22.448173], "paragraph_keywords": ["repeater", "sensor", "frequency", "scanning"]}, {"paragraph_vector": [-56.802124, 25.300621], "paragraph_keywords": ["frequency", "temperature", "sensor", "pressure"]}, {"paragraph_vector": [-57.054679, 24.076923], "paragraph_keywords": ["switch", "inductor", "sensor", "substrate"]}, {"paragraph_vector": [-55.918472, 24.926996], "paragraph_keywords": ["distance", "functions", "interrogating", "continue"]}], "content": {}, "doi": "10.1109/VISUAL.2004.103"}, {"uri": "150", "title": "High-Gain InAs Planar Avalanche Photodiodes", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Benjamin S. White", "Ian C. Sandall", "Xinxin Zhou", "Andrey Krysa", "Kenneth McEwan", "John P. R. David", "Chee Hing Tan"], "summary": "We report the fabrication of InAs planar avalanche photodiodes (APDs) using Be ion implantation. The planar APDs have a low background doping of 2 \u00d7 1014cm\u22123 and large depletion widths approaching 8 \u03bcm. The thick depletion width enabled a gain of 330 to be achieved at \u221226 V at 200 K without inducing a significant tunneling current. No edge breakdown was observed within the APDs. The surface leakage current was found to be low with a gain normalized dark current density of 400 \u03bcAcm\u22122 at \u221220 V at 200 K.", "keywords": ["p", "\u03bcm", "k", "background", "si", "hgcdte", "current", "apd", "achieved", "planar", "gain", "doping", "diffusion", "leakage", "c", "layer", "v", "surface", ".", "inas", "type", "temperature", "apds", "avalanche", "region", "photodiodes", "measured"], "document_vector": [63.43888, -21.703594], "paragraphs": [{"paragraph_vector": [-37.520874, 16.511463], "paragraph_keywords": ["wafer", "\u03bcm", "c", "layer"]}, {"paragraph_vector": [-37.621227, 18.61881], "paragraph_keywords": ["temperature", "current", "k", "\u03bcm"]}, {"paragraph_vector": [-38.412891, 18.691854], "paragraph_keywords": ["apd", "doping", "inas", "\u03bcm"]}, {"paragraph_vector": [-38.036277, 17.699932], "paragraph_keywords": ["doping", "apd", "si", "background"]}, {"paragraph_vector": [-38.031238, 17.332399], "paragraph_keywords": ["apd", "doping", "planar", "responsivity"]}, {"paragraph_vector": [-35.494529, 16.931692], "paragraph_keywords": ["current", "apd", "planar", "apds"]}, {"paragraph_vector": [-36.483119, 17.105335], "paragraph_keywords": ["planar", "apds", "surface", "diffusion"]}, {"paragraph_vector": [-38.831249, 17.492546], "paragraph_keywords": ["gain", "avalanche", "inas", "photodiodes"]}, {"paragraph_vector": [-35.848693, 18.325071], "paragraph_keywords": ["inas", "photodiodes", ".", "avalanche"]}, {"paragraph_vector": [-38.109031, 19.306024], "paragraph_keywords": [".", "ingaas", "cap", "p"]}], "content": {}, "doi": "10.1109/TVCG.2020.2975583"}, {"uri": "151", "title": "Gene Expression Data Analysis Using a Novel Approach to Biclustering Combining Discrete and Continuous Data", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yann Christinat", "Bernd Wachmann", "Lei Zhang"], "summary": "Many different methods exist for pattern detection in gene expression data. In contrast to classical methods, biclustering has the ability to cluster a group of genes together with a group of conditions (replicates, set of patients, or drug compounds). However, since the problem is NP-complex, most algorithms use heuristic search functions and, therefore, might converge toward local maxima. By using the results of biclustering on discrete data as a starting point for a local search function on continuous data, our algorithm avoids the problem of heuristic initialization. Similar to Order-Preserving Submatrices (OPSM), our algorithm aims to detect biclusters whose rows and columns can be ordered such that row values are growing across the bicluster\u2019s columns and vice versa. Results have been generated on the yeast genome (Saccharomyces cerevisiae), a human cancer data set, and random data. Results on the yeast genome showed that 89 percent of the 100 biggest nonoverlapping biclusters were enriched with Gene Ontology annotations. A comparison with the methods OPSM and Iterative Signature Algorithm (ISA, a generalization of singular value decomposition) demonstrated a better efficiency when using gene and condition orders. We present results on random and real data sets that show the ability of our algorithm to capture statistically significant and biologically relevant biclusters.", "keywords": ["performed", "p", "data", "value", "order", "et", "gene", "expression", "set", "score", "algorithm", "\u00fe", "variance", "percent", "applied", "information", "number", "result", "distribution", "defined", "annotation", "biclusters", "computed", "based", "j", "element", "seed", "size", "bicluster", "condition", "al", "biclustering"], "document_vector": [113.420318, 80.689094], "paragraphs": [{"paragraph_vector": [-98.838027, 12.641673], "paragraph_keywords": ["algorithms", "gene", "biclustering", "data"]}, {"paragraph_vector": [-94.679061, 14.788622], "paragraph_keywords": ["algorithm", "data", "biclustering", "gene"]}, {"paragraph_vector": [-92.337966, 17.17695], "paragraph_keywords": ["data", "gene", "condition", "biclusters"]}, {"paragraph_vector": [-92.255683, 14.863501], "paragraph_keywords": ["set", "gene", "condition", "biclusters"]}, {"paragraph_vector": [-95.476226, 14.92666], "paragraph_keywords": ["condition", "\u00fe", "gene", "oc\u00f0i\u00fe"]}, {"paragraph_vector": [-94.110237, 13.809425], "paragraph_keywords": ["number", "size", "biclusters", "data"]}, {"paragraph_vector": [-94.637672, 14.707267], "paragraph_keywords": ["data", "\u00fe", "biclustering", "order"]}, {"paragraph_vector": [-94.841453, 15.057964], "paragraph_keywords": ["data", "genes", "values", "algorithm"]}, {"paragraph_vector": [-94.236175, 15.007441], "paragraph_keywords": ["elements", "matrix", "distribution", "values"]}, {"paragraph_vector": [-87.309089, 7.840797], "paragraph_keywords": ["conditions", "biclusters", "seed", "seeds"]}, {"paragraph_vector": [-88.606834, 9.194234], "paragraph_keywords": ["order", "seed", "method", "conditions"]}, {"paragraph_vector": [-95.016494, 9.914178], "paragraph_keywords": ["genes", "gene", "condition", "coefficient"]}, {"paragraph_vector": [-95.601272, 12.228132], "paragraph_keywords": ["condition", "gene", "order", "addition"]}, {"paragraph_vector": [-94.106994, 12.064659], "paragraph_keywords": ["gene", "condition", "score", "computed"]}, {"paragraph_vector": [-92.396499, 13.597921], "paragraph_keywords": ["biclusters", "data", "bicluster", "conditions"]}, {"paragraph_vector": [-95.23561, 14.929918], "paragraph_keywords": ["data", "set", "size", "sparseness"]}, {"paragraph_vector": [-91.991363, 16.25402], "paragraph_keywords": ["data", "set", "found", "annotations"]}, {"paragraph_vector": [-92.942329, 15.689627], "paragraph_keywords": ["biclusters", "set", "genes", "algorithm"]}, {"paragraph_vector": [-92.376098, 17.220144], "paragraph_keywords": ["biclusters", "percent", "annotations", "condition"]}, {"paragraph_vector": [-70.95742, 7.119836], "paragraph_keywords": ["biclusters", "data", "dlbcl", "results"]}, {"paragraph_vector": [-91.595382, 15.770497], "paragraph_keywords": ["algorithm", "biclusters", "data", "annotations"]}], "content": {}, "doi": "10.1109/TVCG.2019.2934333"}, {"uri": "152", "title": "VRML Molecular Dynamics Trajectories", "timestamp": "1999", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Geoff Leach", "James Gilbert"], "summary": "One application of VRML is visualisation of scientific data. Using VRML results of simulations and other scientific endeavours can be published on the web with all the advantages (and disadvantages) the web offers, but with the further advantage that the results are in the form of interactive, moving 3D \u201cworlds\u201d. This is especially advantageous when the data itself is 3D. In this paper we discuss the use of VRML to publish the results of molecular dynamics simulations on the web. We also discuss a framework for application-specific compression of VRML files which allows the use of a compression technique appropriate to the particular type of data in a given application but does so in a general manner. Using a simple dz&-ential compression approach within this framework we gain an extra factor of 2 in the compression ratio to that obtained through the use of gzip for our application.", "keywords": [], "document_vector": [-161.926651, -41.047012], "paragraphs": [{"paragraph_vector": [-127.829765, 24.162628], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/CIS.2012.27"}, {"uri": "153", "title": "A Visual Analytics of Geometric Distances Between Amino Acids and Surface Pockets of Proteins", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Makiko Miyoshi", "Ayaka Kaneko", "Takayuki Itoh", "Kei Yura"], "summary": "Protein is the major component of the organism. It has a unique three-dimensional structure formed by its amino acid sequence. A concave (pocket) on the surface of a protein is known to be the best target for a drug to react. We started analyzing how \u201cdruggability\" of proteins related to the location of amino acids in a pocket. This paper presents a visualization tool for distance analysis between pockets and the amino acid residue. Provided that a protein surface is described by a triangular mesh, this tool first identifies pockets on the protein surface, specifies the deepest point and outer loops of the pocket, and calculates distances between atoms of an amino acid residue and the deepest point or the outer loops of the pocket. The tool then visualizes the statistics of the distance calculation results by polyline charts and the distribution by scatterplots. This paper proposes a biological interpretation of the visualization results. Keywords-Visualization; Protein; Amino acid; Pocket identification", "keywords": ["pair", "protein", "technique", "surface", "pocket", "druggability", "molecule", "applied", "distance", "visualization", "acid", "section", "type", "residue"], "document_vector": [-130.003067, -8.74028], "paragraphs": [{"paragraph_vector": [-13.989875, -28.730403], "paragraph_keywords": ["protein", "pockets", "acid", "presented"]}, {"paragraph_vector": [-7.771816, -27.548116], "paragraph_keywords": ["pocket", "technique", "protein", "vertices"]}, {"paragraph_vector": [-11.148872, -28.273983], "paragraph_keywords": ["pockets", "visualization", "acid", "distance"]}, {"paragraph_vector": [-10.951772, -26.611076], "paragraph_keywords": ["pockets", "datasets", "protein", "pocket"]}, {"paragraph_vector": [-11.082581, -25.974296], "paragraph_keywords": ["pockets", "residue", "acid", "pocket"]}, {"paragraph_vector": [-10.174592, -27.791107], "paragraph_keywords": ["pockets", "acid", "types", "like"]}, {"paragraph_vector": [-21.964008, -27.427352], "paragraph_keywords": ["obtained", "observations", "contribute", "acid"]}], "content": {}, "doi": "10.1109/MCG.2014.27"}, {"uri": "154", "title": "empty", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ramesh Jain Praja"], "summary": "The advent of the Internet and Web brought about the Information Age during the last decade. Recently, however, people have started exploring what the next big societal shift will be. Lately, I\u2019ve heard people talk about knowledge management. Corporations believe that they can make their operations more efficient by really \u201cknowing what they know.\u201d Many in the knowledge management community believe that techniques exist to represent explicit knowledge, but tacit knowledge has been elusive. Explicit knowledge is available in documents, and tacit knowledge usually resides in our heads. Although we can easily deal with explicit knowledge, we need a better understanding of the nature of knowledge and how knowledge is really created to tackle tacit knowledge.", "keywords": ["data", "experience", "language", "mechanism", "world", "knowledge", "multimedia"], "document_vector": [166.361938, 34.757667], "paragraphs": [{"paragraph_vector": [162.669601, 11.195584], "paragraph_keywords": ["experience", "knowledge", "language", "data"]}, {"paragraph_vector": [162.899597, 7.175803], "paragraph_keywords": ["knowledge", "multimedia", "step", "efforts"]}], "content": {}, "doi": "10.1109/BioVis.2011.6094055"}, {"uri": "155", "title": "Efficiency Enhancement of Protein Folding for Complete Molecular Simulation via Hardware Computing", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Wen-Tsai Sung"], "summary": "Accelerating a protein folding by implementing it in a reconfigurable field programmable gate array (FPGA) is described. This paper presents a methodology for the design of a reconfigurable computing system applied to a complex problem in molecular biology: the protein folding problem. This paper employed VMD tool and force field simulation theorem based on FPGA for protein folding solution. This technique consists of two components: finished protein folding process and found out active sites for drug docking. The goal of protein folding simulation is to search the global energy minimum location with stability state and the when the protein is finished the folding task, we can find out the active sites for pre-process of ligand protein docking. An efficient hardwarebased approach was devised to achieve a significant reduction of the search space of possible foldings. Several simulations were done to evaluate the performance of the system as well as the demand for FPGA\u2019s resources.. KeywordsDrug Docking; Minimum Energy; Protein Folding; Molecular Visualization; FPGA;", "keywords": ["field", "computing", "folding", "value", "matrix", "communication", "fpga", "application", "compound", "force", "path", "energy"], "document_vector": [-21.368953, -44.506237], "paragraphs": [{"paragraph_vector": [5.439429, -65.015411], "paragraph_keywords": ["application", "energy", "compound", "folding"]}, {"paragraph_vector": [-163.867828, 37.067474], "paragraph_keywords": ["path", "values", "communication", "matrix"]}, {"paragraph_vector": [-159.483963, 36.084938], "paragraph_keywords": ["communication", "network", "protein", "computing"]}], "content": {}, "doi": "10.1109/ICETAS.2017.8277866"}, {"uri": "156", "title": "Implicit Representation of Molecular Surfaces", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Julius Parulek", "Ivan Viola"], "summary": "Molecular surfaces are an established tool to analyze and to study the evolution and interaction of molecules. One of the most advanced representations of molecular surfaces is called the solvent excluded surface. We present a novel and a simple method for representing the solvent excluded surfaces (SES). Our method requires no precomputation and therefore allows us to vary SES parameters outright. We utilize the theory of implicit surfaces and their CSG operations to compose the implicit function representing the molecular surface locally. This function returns a minimal distance to the SES representation. Additionally, negative values of the implicit function determine that the point lies outside SES whereas positive ones that the point lies inside. We describe how to build this implicit function composed of three types of patches constituting the SES representation. Finally, we propose a method to visualize the iso-surface of the implicit function by means of ray-casting and the set of rendering parameters affecting the overall performance.", "keywords": ["cavity", "p", "called", "iso", "order", "parameter", "distance", "representation", "set", "method", "protein", "kmax", "point", "number", "proposed", "defined", "ray", "sphere", "surface", "based", "atom", "s", "solvent", "gpu", "r", "radius", "lie", "sp", "time", "function", "rendering", "visualization", "casting", "intersection"], "document_vector": [-168.398162, -89.907295], "paragraphs": [{"paragraph_vector": [74.899642, -38.841228], "paragraph_keywords": ["molecules", "atoms", "called", "ligands"]}, {"paragraph_vector": [65.891128, -6.224206], "paragraph_keywords": ["representation", "ses", "surface", "radius"]}, {"paragraph_vector": [72.890785, 2.064699], "paragraph_keywords": ["representation", "surface", "ses", "protein"]}, {"paragraph_vector": [41.707759, 10.313369], "paragraph_keywords": ["ses", "protein", "surface", "file"]}, {"paragraph_vector": [24.722642, 40.223423], "paragraph_keywords": ["function", "representation", "operators", "min"]}, {"paragraph_vector": [28.136978, 45.469013], "paragraph_keywords": ["ray", "casting", "surfaces", "function"]}, {"paragraph_vector": [20.625753, 45.76704], "paragraph_keywords": ["function", "p", "intersection", "atoms"]}, {"paragraph_vector": [26.754323, 9.49923], "paragraph_keywords": ["function", "set", "atoms", "defined"]}, {"paragraph_vector": [12.197424, 12.133752], "paragraph_keywords": ["p", "point", "atom", "iso"]}, {"paragraph_vector": [15.950638, 11.506167], "paragraph_keywords": ["point", "p", "intersection", "vector"]}, {"paragraph_vector": [11.747616, 9.599982], "paragraph_keywords": ["point", "intersection", "triangle", "lies"]}, {"paragraph_vector": [22.811668, 42.852897], "paragraph_keywords": ["point", "points", "utilize", "function"]}, {"paragraph_vector": [32.377632, 31.497329], "paragraph_keywords": ["atoms", "number", "radius", "sp"]}, {"paragraph_vector": [25.822977, 40.070453], "paragraph_keywords": ["surface", "ray", "p", "points"]}, {"paragraph_vector": [45.185974, 33.948413], "paragraph_keywords": ["atoms", "function", "points", "performance"]}, {"paragraph_vector": [71.192733, -10.502251], "paragraph_keywords": ["kmax", "parameters", "protein", "surface"]}, {"paragraph_vector": [27.2999, 28.373102], "paragraph_keywords": ["function", "representation", "\u03b5", "artifacts"]}, {"paragraph_vector": [29.616498, 12.897929], "paragraph_keywords": ["cavity", "distance", "protein", "surface"]}, {"paragraph_vector": [-65.458869, -80.016342], "paragraph_keywords": ["dynamics", "feedback", "simulation", "datasets"]}], "content": {}, "doi": "10.1109/SciVis.2018.8823785"}, {"uri": "157", "title": "A VR Framework for Interacting with Molecular Simulations", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Nicolas F\u00e9rey", "Olivier Delalande", "Gilles Grasseau", "Marc Baaden"], "summary": "Molecular Dynamics is nowadays routinely used to complement experimental studies and overcome some of their limitations. In particular, current experimental techniques do not allow to directly observe the full dynamics of a macromolecule at atomic detail. Molecular simulation provides time-dependent atomic positions, velocities and system energies according to biophysical models. Many molecular simulation engines can now compute a molecular dynamics trajectory of interesting biological systems in interactive time. This progress has lead to a new approach called interactive molecular dynamics. It allows to control and visualise a molecular simulation in progress. We have developed a generic library, called MDDriver, to facilitate the implementation of such interactive simulations. It allows to easily create a network between a molecular user interface and a physically-based simulation. We use this library in order to study an interesting biomolecular system, simulated by various interaction-enabled molecular engines and models. We use a classical molecular visualisation tool and a haptic device to control the dynamic behavior of the molecule. This approach provides encouraging results for interacting with a biomolecule and understanding its dynamics. Starting from this initial success, we decided to use VR functionalities more intensively, by designing a VR framework dedicated to immersive and interactive molecular simulations. This framework is based on MDDriver, on the visualisation toolkit VTK, and on the vtkVRPN library, which encapsulates the VRPN library into VTK. CR Categories: I.6.6 [Computing Methodologies]: Simulation and Modelling\u2014Simulation Output Analysis H.5.2 [Information Systems]: Information Interfaces and Presentation\u2014Graphic User Interface and Haptic I/O", "keywords": ["order", "imd", "vr", "interaction", "simulation", "vtk", "vmd", "vrpn", "mddriver", "closure", "approach", "framework", "system", "figure", "interface", "engine", "based", "position", "library", "atom", "use", "time", "user", "rendering", "behavior"], "document_vector": [136.035583, -74.933769], "paragraphs": [{"paragraph_vector": [104.289825, -48.903278], "paragraph_keywords": ["simulation", "researchers", "order", "observe"]}, {"paragraph_vector": [105.98011, -6.869297], "paragraph_keywords": ["simulation", "copies", "use", "engines"]}, {"paragraph_vector": [108.736755, -1.162007], "paragraph_keywords": ["user", "library", "simulation", "mddriver"]}, {"paragraph_vector": [100.92453, -5.436234], "paragraph_keywords": ["user", "vr", "simulation", "vtk"]}, {"paragraph_vector": [92.58229, 8.323705], "paragraph_keywords": ["vtk", "atom", "vr", "rendering"]}, {"paragraph_vector": [105.308013, 3.301729], "paragraph_keywords": ["vtk", "vrpn", "device", "feedback"]}, {"paragraph_vector": [68.838943, -41.696201], "paragraph_keywords": ["figure", "closure", "system", "mechanism"]}, {"paragraph_vector": [109.674468, -7.665439], "paragraph_keywords": ["simulation", "vr", "vtk", "closure"]}, {"paragraph_vector": [103.548042, 2.463992], "paragraph_keywords": ["atoms", "interaction", "simulation", "vtk"]}, {"paragraph_vector": [108.73571, 0.362116], "paragraph_keywords": ["simulation", "journal", "approach", "software"]}], "content": {}, "doi": "10.1109/PacificVis.2014.32"}, {"uri": "158", "title": "Spatial Index and Visualization of Protein Structure", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Hui Li", "Chunmei Liu"], "summary": "We use the protein sequences from the ASTRAL [10] library which provides up to 7640 protein sequences. The secondary structure and the tertiary structure of the protein are downloaded from the PDB library [11]. In the ASTRAL library, 7100 sequences in accordance to the protein structure classes can be divided into seven categories, namely a, b, c, d, e, f, and g. For the seven categories, a denotes a protein which only includes the alpha class, b denotes the whole beta protein, C is for the proteins including alpha/beta class, d indicates (alpha +beta) class, e is for multi-domain class, f represents membrane and cell surface proteins, and g represents small proteins [11]. The number of folds of each structural class is listed in Table 1.", "keywords": [], "document_vector": [-134.731262, 72.994712], "paragraphs": [{"paragraph_vector": [-123.550346, 26.472108], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TVCG.2017.2743979"}, {"uri": "159", "title": "Visualization and Classification of Protein Secondary Structures using Self-Organizing Maps", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Christian Gr\u00e9visse", "Ian Muller", "Juan Luis Jim\u00e9nez Laredo", "Marek Ostaszewski", "Gr\u00e9goire Danoy", "Pascal Bouvry"], "summary": "In molecular biology, it is estimated that there is a correlation between the secondary structure of a protein and its functionality. While secondary structure prediction is ultimately possible in wet lab, determining a correlation with the functionality is a hard task which can be facilitated by a computational model. In that context, this paper presents an automated algorithm for the visualization and classification of enzymatic proteins with the aim of examining whether the functionality is correlated to the secondary structure. To that end, up-to-date protein data was acquired from publicly accessible databases in order to construct their secondary structures. The resulting data were injected into a tailored version of a Kohonen Self-Organizing Map (SOM). Part of the work was to determine a proper way of reducing large secondary structures to a common length in order to be able to cope with the constant dimensionality requirement of SOMs. The final contribution consisted in the labeling of the trained nodes. Eventually, we were able to get a visual intuition and some quantified assessment on the nature of this correlation.", "keywords": ["data", "structure", "distance", "classification", "li", "set", "method", "length", "protein", "author", "algorithm", "matrix", "family", "information", "approach", "som", "result", "vector", "based", "compression", "subcollection", "element", "component", "ec", "visualization"], "document_vector": [-176.552978, 68.381134], "paragraphs": [{"paragraph_vector": [-77.874816, -24.711734], "paragraph_keywords": ["data", "based", "clusters", "som"]}, {"paragraph_vector": [-77.403022, -24.19708], "paragraph_keywords": ["structure", "based", "prediction", "function"]}, {"paragraph_vector": [-80.546035, -43.406547], "paragraph_keywords": ["protein", "authors", "data", "structures"]}, {"paragraph_vector": [-79.566886, -43.071586], "paragraph_keywords": ["length", "structure", "structures", "data"]}, {"paragraph_vector": [-80.309097, -35.577663], "paragraph_keywords": ["family", "structure", "strings", "ec"]}, {"paragraph_vector": [-77.976745, -34.572334], "paragraph_keywords": ["structure", "compression", "assume", "reduced"]}, {"paragraph_vector": [-72.782295, -19.689287], "paragraph_keywords": ["data", "length", "average", "error"]}, {"paragraph_vector": [-71.376296, -22.10049], "paragraph_keywords": ["xj", "som", "distance", "kernel"]}, {"paragraph_vector": [-121.259048, -27.27848], "paragraph_keywords": ["li", "family", "color", "value"]}, {"paragraph_vector": [-97.837799, -32.062301], "paragraph_keywords": ["som", "node", "family", "set"]}, {"paragraph_vector": [-80.303573, -26.891181], "paragraph_keywords": ["family", "classifier", "approach", "accuracy"]}, {"paragraph_vector": [-74.185462, -44.610393], "paragraph_keywords": ["sampling", "subfamilies", "regions", "finer"]}], "content": {}, "doi": "10.1109/PACIFICVIS.2017.8031605"}, {"uri": "160", "title": "MOLECULAR RENDERING APPLICATIONS OF VOLUME AND 3-D TEXTURE MAPS", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["David S. Goodsell", "Arthur J. Olson"], "summary": "Molecular properties expressed as a 3D grid of scalar values (volumetric data) are effectively imaged by the techniques of volume rendering. In this paper, we describe a ray-tracing volume renderer and some of its macromolecular applications. Volumetric data may be rendered as a collection of cloud densities and zerothickness surfaces, surrounding a solid model of the molecular structure composed of spheres and cylinders. 3D texture mapping is used to display two volumetric data sets in one image, generating cloud opacities and surface levels with one data set and coloring with the second. Several refinements are described, including atmospheric depth cuing and zclipping.", "keywords": [], "document_vector": [49.377311, -85.132194], "paragraphs": [{"paragraph_vector": [-123.065063, 26.830297], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TVCG.2006.182"}, {"uri": "161", "title": "VSEPR Theory, An Interactive and Immersive Virtual Reality", "timestamp": "2020", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Danica Fujiwara", "Krzysztof Pietroszek", "Kevin Kellar", "Christian Eckhardt", "Irene Humer"], "summary": "This work explores the role of immersive learning in regards to predicting molecular structures and orbital hybridization through an Oculus Rift HMD. Specifically, this research utilizes an interactive virtual environment to help teach the concept of the Valence Shell Electron Pair Repulsion Theory. We developed an interactive and immersive simulation in which participants constructed virtual molecules. Following the simulation, participants were asked to identify shapes of similar molecules to measure their understanding of the theory as compared to a control group. Finally, participants answered the Game Experience Questionnaire to assess the level of Immersion. The results show that our immersive virtual reality (IVR) experience improves comprehension of three dimensional structures in chemistry.", "keywords": ["group", "molecule", "simulation", "questionnaire", "experience", "reality", "research", "control", "allowed", "understanding", "understand", "immersion", "level", "learning", "geometry", "question", "atom", "vsepr", "theory", "participant", "user", "test", "ivr"], "document_vector": [119.612274, -24.068042], "paragraphs": [{"paragraph_vector": [173.90422, -0.521171], "paragraph_keywords": ["participants", "group", "bond", "based"]}, {"paragraph_vector": [174.397216, 4.418357], "paragraph_keywords": ["simulation", "world", "atoms", "molecule"]}, {"paragraph_vector": [173.681198, -0.48148], "paragraph_keywords": ["users", "atoms", "participants", "molecule"]}, {"paragraph_vector": [176.496002, -2.911288], "paragraph_keywords": ["theory", "learning", "questions", "chemistry"]}, {"paragraph_vector": [176.375503, -2.985], "paragraph_keywords": ["participants", "group", "test", "learning"]}, {"paragraph_vector": [174.109756, 2.51567], "paragraph_keywords": ["scores", "participants", "vsepr", "theory"]}, {"paragraph_vector": [175.083618, -0.782931], "paragraph_keywords": ["participants", "group", "question", "ivr"]}, {"paragraph_vector": [172.326782, -0.466454], "paragraph_keywords": ["immersion", "participants", "question", "geometry"]}, {"paragraph_vector": [174.540863, 0.385824], "paragraph_keywords": ["immersion", "theory", "vsepr", "learning"]}, {"paragraph_vector": [135.253555, -23.387277], "paragraph_keywords": ["carbon", "ivr", "collaboration", "lontschar"]}], "content": {}, "doi": "10.1109/ECONF.2015.25"}, {"uri": "162", "title": "empty", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "According as the protein-protein interaction (PPI) data more increase, we need to optimally visualize them as network, in that describing the relationship among proteins is able to easily analyze biological processes happened in a cell. In this paper, to fast layout large-scale PPI networks, we proposed a method taking hub-proteins into consideration, which have more interactions than any other proteins in a network. In other words, it enforces two core parts of Walshaw\u2019s multilevel force-directed placement algorithm (MLFDP) to be modified. The modification is achieved by coarsening and expanding all neighboring proteins of hub-protein just once, whereas only two proteins in Walshaw\u2019s method. Our experiments show that the quality of layout is better optimal and time cost is reduced up to 63% in comparison with other methods.", "keywords": ["cluster", "weight", "algorithm", "end", "network", "position", "force", "graph", "hub", "node", "layout"], "document_vector": [10.236362, 49.00753], "paragraphs": [{"paragraph_vector": [-108.267326, -17.277914], "paragraph_keywords": ["nodes", "graph", "edges", "force"]}, {"paragraph_vector": [-96.174797, -6.688461], "paragraph_keywords": ["end", "nodes", "graph", "begin"]}, {"paragraph_vector": [-107.612876, -16.521886], "paragraph_keywords": ["nodes", "node", "hub", "weight"]}, {"paragraph_vector": [-97.606246, -10.94256], "paragraph_keywords": ["graph", "node", "nodes", "hub"]}, {"paragraph_vector": [-89.324645, -5.674014], "paragraph_keywords": ["cluster", "position", "nodes", "partition"]}, {"paragraph_vector": [-109.915206, -19.648284], "paragraph_keywords": ["ppi", "mlfdp", "network", "method"]}], "content": {}, "doi": "10.1109/BIBE.2009.46"}, {"uri": "163", "title": "In-Transit Molecular Dynamics Analysis with Apache Flink", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Henrique C. Zan\u00faz", "Bruno Raffin", "Omar A. Mures", "Emilio J. Padr\u00f3n"], "summary": "In this paper, an on-line parallel analytics framework is proposed to process and store in transit all the data being generated by a Molecular Dynamics (MD) simulation run using staging nodes in the same cluster executing the simulation. The implementation and deployment of such a parallel workflow with standard HPC tools, managing problems such as data partitioning and load balancing, can be a hard task for scientists. In this paper we propose to leverage Apache Flink, a scalable stream processing engine from the Big Data domain, in this HPC context. Flink enables to program analyses within a simple window based map/reduce model, while the runtime takes care of the deployment, load balancing and fault tolerance. We build a complete in transit analytics workflow, connecting an MD simulation to Apache Flink and to a distributed database, Apache HBase, to persist all the desired data. To demonstrate the expressivity of this programming model and its suitability for HPC scientific environments, two common analytics in the MD field have been implemented. We assessed the performance of this framework, concluding that it can handle simulations of sizes used in the literature while providing an effective and versatile tool for scientists to easily incorporate on-line parallel analytics in their current workflows.", "keywords": ["storage", "data", "value", "timestep", "md", "execution", "hpc", "analysis", "simulation", "zeromq", "histogram", "number", "run", "distributed", "stream", "framework", "result", "system", "programming", "hbase", "analytics", "flink", "mpi", "based", "work", "process", "level", "performance", "atom", "message", "experiment", "version", "model", "processing", "comd", "time", "neighbor", "node"], "document_vector": [2.614968, -31.75564], "paragraphs": [{"paragraph_vector": [-147.612274, 52.400009], "paragraph_keywords": ["data", "copies", "analysis", "advantage"]}, {"paragraph_vector": [178.616592, 40.653274], "paragraph_keywords": ["simulation", "data", "analysis", "flink"]}, {"paragraph_vector": [-178.239151, 43.06451], "paragraph_keywords": ["mpi", "analysis", "md", "parallelization"]}, {"paragraph_vector": [-171.698135, 36.958221], "paragraph_keywords": ["analytics", "data", "spark", "flink"]}, {"paragraph_vector": [-174.181304, 37.19979], "paragraph_keywords": ["flink", "data", "node", "operations"]}, {"paragraph_vector": [-174.839553, 39.256431], "paragraph_keywords": ["flink", "data", "connector", "zeromq"]}, {"paragraph_vector": [-174.744995, 40.629737], "paragraph_keywords": ["data", "flink", "comd", "nodes"]}, {"paragraph_vector": [-175.934646, 39.230556], "paragraph_keywords": ["data", "z", "flink", "atoms"]}, {"paragraph_vector": [-171.196701, 45.949298], "paragraph_keywords": ["data", "atom", "comd", "number"]}, {"paragraph_vector": [-173.678329, 41.118423], "paragraph_keywords": ["flink", "hbase", "data", "comd"]}, {"paragraph_vector": [-174.061141, 41.322959], "paragraph_keywords": ["data", "simulation", "flink", "performance"]}, {"paragraph_vector": [-105.991874, -29.307117], "paragraph_keywords": ["data", "including", "hbase", "time"]}, {"paragraph_vector": [-175.890762, 39.380126], "paragraph_keywords": ["version", "experiments", "data", "hbase"]}, {"paragraph_vector": [-175.387573, 40.544589], "paragraph_keywords": ["number", "flink", "comd", "hbase"]}, {"paragraph_vector": [-175.130081, 39.817478], "paragraph_keywords": ["experiments", "nodes", "flink", "comd"]}, {"paragraph_vector": [-174.199447, 40.163219], "paragraph_keywords": ["master", "experiments", "nodes", "k"]}], "content": {}, "doi": "10.1109/HSI.2008.4581462"}, {"uri": "164", "title": "nAPOLI: a graph-based strategy to detect and visualize conserved protein-ligand interactions in large-scale", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Alexandre V. Fassio", "Lucianna H. Santos", "Sabrina A. Silveira", "Rafaela S. Ferreira", "Raquel C. de Melo-Minardi"], "summary": "Essential roles in biological systems depend on protein-ligand recognition, which is mostly driven by specific non-covalent interactions. Consequently, investigating these interactions contributes to understanding how molecular recognition occurs. Nowadays, a large-scale data set of protein-ligand complexes is available in the Protein Data Bank, what led several tools to be proposed as an effort to elucidate protein-ligand interactions. Nonetheless, there is not an all-in-one tool that couples large-scale statistical, visual, and interactive analysis of conserved protein-ligand interactions. Therefore, we propose nAPOLI (Analysis of PrOtein-Ligand Interactions), a web server that combines large-scale analysis of conserved interactions in protein-ligand complexes at the atomic-level, interactive visual representations, and comprehensive reports of the interacting residues/atoms to detect and explore conserved non-covalent interactions. We demonstrate the potential of nAPOLI in detecting important conserved interacting residues through four case studies: two involving a human cyclin-dependent kinase 2 (CDK2), one related to ricin, and other to the human nuclear receptor subfamily 3 (hNR3). nAPOLI proved to be suitable to identify conserved interactions according to literature, as well as highlight additional interactions. Finally, we illustrate, with a virtual screening ligand selection, how nAPOLI can be widely applied in structural biology and drug design. nAPOLI is freely available at bioinfo.dcc.ufmg.br/napoli/.", "keywords": ["napoli", "data", "structure", "molecule", "analysis", "hydrogen", "interaction", "set", "cluster", "protein", "ligand", "number", "result", "process", "pdb", "based", "binding", "complex", "recognition", "study", "atom", "type", "composed", "file", "receptor", "conserved", "interacting", "supporting", "found", "user", "table", "residue"], "document_vector": [-55.481182, 41.856563], "paragraphs": [{"paragraph_vector": [-9.206938, -60.216217], "paragraph_keywords": ["ligand", "protein", "university", "recognition"]}, {"paragraph_vector": [-29.694717, -36.150444], "paragraph_keywords": ["interactions", "tools", "analysis", "conserved"]}, {"paragraph_vector": [-35.689788, -37.132995], "paragraph_keywords": ["interactions", "napoli", "ligand", "algorithms"]}, {"paragraph_vector": [-28.16608, -38.528675], "paragraph_keywords": ["protein", "number", "ligand", "users"]}, {"paragraph_vector": [-27.444665, -36.649944], "paragraph_keywords": ["napoli", "atoms", "remove", "ligands"]}, {"paragraph_vector": [-27.05599, -36.156299], "paragraph_keywords": ["interactions", "atoms", "ligand", "hydrogen"]}, {"paragraph_vector": [-29.365659, -35.823402], "paragraph_keywords": ["clustering", "set", "ligand", "residues"]}, {"paragraph_vector": [-64.591171, -64.62191], "paragraph_keywords": ["napoli", "data", "set", "ligand"]}, {"paragraph_vector": [-125.949058, -49.136867], "paragraph_keywords": ["interaction", "atoms", "interactions", "atom"]}, {"paragraph_vector": [-119.878814, -80.168457], "paragraph_keywords": ["table", "residues", "data", "interactions"]}, {"paragraph_vector": [-34.825504, -29.209671], "paragraph_keywords": ["ligand", "interaction", "residues", "type"]}, {"paragraph_vector": [-27.823013, -26.49057], "paragraph_keywords": ["interactions", "data", "set", "complexes"]}, {"paragraph_vector": [-31.495901, -27.533863], "paragraph_keywords": ["napoli", "hydrogen", "ligand", "interactions"]}, {"paragraph_vector": [-30.584337, -27.920833], "paragraph_keywords": ["cluster", "set", "ligands", "interactions"]}, {"paragraph_vector": [-33.802845, -27.956418], "paragraph_keywords": ["residues", "ligands", "clusters", "compounds"]}, {"paragraph_vector": [-30.487771, -26.032817], "paragraph_keywords": ["receptor", "ligands", "interactions", "supporting"]}, {"paragraph_vector": [-31.778337, -28.868871], "paragraph_keywords": ["ligands", "ligand", "protein", "process"]}, {"paragraph_vector": [-30.841554, -34.205116], "paragraph_keywords": ["interactions", "ligand", "ligands", "based"]}, {"paragraph_vector": [-37.22079, -36.977802], "paragraph_keywords": ["interactions", "napoli", "conserved", "analysis"]}, {"paragraph_vector": [-36.97908, -57.207241], "paragraph_keywords": ["capes", "analysis", "project", "design"]}], "content": {}, "doi": "10.1109/SC.Companion.2012.327"}, {"uri": "165", "title": "Haptic Visualization with 2D Lateral Force using SPIDAR-mouse in Molecular Docking", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kikuo Asai", "Norio Takase", "Makoto Sato"], "summary": "A haptic visualization system was made with the SPIDAR-mouse for 2D lateral force feedback in molecular docking. An educational environment requires factors such as easy installation, low cost, and reasonable performance, which the SPIDAR-mouse can provide. However, it limits the representation of force to 2D, even though the molecular interaction includes 3D phenomena. Our approach to the solution is to map the surface of 3D molecules to the 2D plane. The haptic rendering is done by calculating forces with the 2D map, which largely reduces the computational cost. The molecular interaction based on the electro-static potential and conformational properties is displayed with the 2D lateral force feedback using the SPIDAR-mouse. Keywords-haptic visualization, 2D lateral force, SPIDARmouse; molecular docking", "keywords": ["mouse", "docking", "electro", "probe", "map", "surface", "molecule", "plane", "spidar", "point", "force", "visualization", "rendering", "ligand", "system", "interaction"], "document_vector": [115.979598, -27.194627], "paragraphs": [{"paragraph_vector": [166.205886, -14.626767], "paragraph_keywords": ["docking", "ligand", "feedback", "molecules"]}, {"paragraph_vector": [166.112792, -13.132238], "paragraph_keywords": ["map", "surface", "axis", "plane"]}, {"paragraph_vector": [79.188415, 10.152565], "paragraph_keywords": ["molecule", "section", "routes", "cross"]}, {"paragraph_vector": [171.056716, -7.688921], "paragraph_keywords": ["electro", "point", "areas", "table"]}, {"paragraph_vector": [164.342864, -16.714736], "paragraph_keywords": ["force", "map", "bump", "vector"]}, {"paragraph_vector": [163.787231, -8.88079], "paragraph_keywords": ["force", "mouse", "molecule", "probe"]}, {"paragraph_vector": [167.98616, -10.38428], "paragraph_keywords": ["visualization", "mouse", "forces", "mapping"]}], "content": {}, "doi": "10.1109/TVCG.2015.2500240"}, {"uri": "166", "title": "Interactive, Graph-Based Visual Analysis of High-Dimensional, Multi-Parameter Fluorescence Microscopy Data in Toponomics", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Steffen Oeltze", "Wolfgang Freiler", "Reyk Hillert", "Helmut Doleisch", "Bernhard Preim", "Walter Schubert"], "summary": "In Toponomics, the function protein pattern in cells or tissue (the toponome) is imaged and analyzed for applications in toxicology, new drug development and patient-drug-interaction. The most advanced imaging technique is robot-driven multi-parameter fluorescence microscopy. This technique is capable of co-mapping hundreds of proteins and their distribution and assembly in protein clusters across a cell or tissue sample by running cycles of fluorescence tagging with monoclonal antibodies or other affinity reagents, imaging, and bleaching in situ. The imaging results in complex multi-parameter data composed of one slice or a 3D volume per affinity reagent. Biologists are particularly interested in the localization of co-occurring proteins, the frequency of co-occurrence and the distribution of co-occurring proteins across the cell. We present an interactive visual analysis approach for the evaluation of multi-parameter fluorescence microscopy data in toponomics. Multiple, linked views facilitate the definition of features by brushing multiple dimensions. The feature specification result is linked to all views establishing a focus+context visualization in 3D. In a new attribute view, we integrate techniques from graph visualization. Each node in the graph represents an affinity reagent while each edge represents two co-occurring affinity reagent bindings. The graph visualization is enhanced by glyphs which encode specific properties of the binding. The graph view is equipped with brushing facilities. By brushing in the spatial and attribute domain, the biologist achieves a better understanding of the function protein patterns of a cell. Furthermore, an interactive table view is integrated which summarizes unique fluorescence patterns. We discuss our approach with respect to a cell probe containing lymphocytes and a prostate tissue section.", "keywords": ["brushing", "data", "dataset", "par", "value", "order", "analysis", "respect", "graph", "according", "set", "cluster", "brush", "corresponding", "protein", "frequency", "information", "biologist", "number", "imaging", "framework", "system", "selected", "fig", "color", "attribute", "cmps", "code", "map", "binding", "lymphocyte", "toponome", "-", "probe", "edge", "co", "layout", "voxel", "represents", "view", "feature", "voxels", "t", "table", "function", "cmp", "visualization", "cell", "context", "node"], "document_vector": [125.074913, 18.93589], "paragraphs": [{"paragraph_vector": [-45.825984, 2.706171], "paragraph_keywords": ["proteins", "protein", "fluorescence", "affinity"]}, {"paragraph_vector": [-48.918869, 11.621788], "paragraph_keywords": ["attribute", "imaging", "approach", "affinity"]}, {"paragraph_vector": [-29.818944, -69.3208], "paragraph_keywords": ["protein", "cell", "proteins", "clusters"]}, {"paragraph_vector": [-45.780017, 5.102941], "paragraph_keywords": ["protein", "map", "code", "proteins"]}, {"paragraph_vector": [-120.843353, -30.649019], "paragraph_keywords": ["data", "cmps", "graph", "table"]}, {"paragraph_vector": [-125.84539, -29.126165], "paragraph_keywords": ["graph", "layout", "views", "data"]}, {"paragraph_vector": [133.133697, -20.522653], "paragraph_keywords": ["data", "views", "graph", "view"]}, {"paragraph_vector": [152.31578, -41.565116], "paragraph_keywords": ["feature", "data", "attribute", "views"]}, {"paragraph_vector": [-68.827041, -60.303272], "paragraph_keywords": ["data", "attribute", "points", "feature"]}, {"paragraph_vector": [-125.27523, -37.593128], "paragraph_keywords": ["edge", "graph", "bindings", "pars"]}, {"paragraph_vector": [93.078865, 10.89164], "paragraph_keywords": ["graph", "feature", "vtk", "simvis"]}, {"paragraph_vector": [-51.396144, 11.744042], "paragraph_keywords": ["ew", "loop", "pars", "elements"]}, {"paragraph_vector": [-87.941261, 52.630813], "paragraph_keywords": ["edge", "vector", "indexes", "layout"]}, {"paragraph_vector": [-82.42086, 3.676468], "paragraph_keywords": ["edges", "edge", "nodes", "feature"]}, {"paragraph_vector": [19.191099, 61.744411], "paragraph_keywords": ["edge", "edges", "width", "order"]}, {"paragraph_vector": [-69.844551, -62.549465], "paragraph_keywords": ["feature", "color", "saturation", "binding"]}, {"paragraph_vector": [-140.333892, -32.639404], "paragraph_keywords": ["glyph", "circle", "par", "cmps"]}, {"paragraph_vector": [-45.947986, -13.508288], "paragraph_keywords": ["brush", "brushing", "nodes", "number"]}, {"paragraph_vector": [-50.445255, 5.277603], "paragraph_keywords": ["table", "view", "cmps", "brushing"]}, {"paragraph_vector": [-48.802753, 9.341924], "paragraph_keywords": ["cell", "lymphocytes", "probe", "surface"]}, {"paragraph_vector": [-47.697235, 9.15046], "paragraph_keywords": ["pars", "proteins", "view", "frequency"]}, {"paragraph_vector": [-49.52927, 12.546925], "paragraph_keywords": ["lymphocytes", "cells", "cell", "t"]}, {"paragraph_vector": [-48.475337, 12.79583], "paragraph_keywords": ["cells", "view", "par", "feature"]}, {"paragraph_vector": [-30.461687, 30.062128], "paragraph_keywords": ["context", "bindings", "focus", "cell"]}, {"paragraph_vector": [-49.78553, 13.502387], "paragraph_keywords": ["cell", "analysis", "number", "co"]}, {"paragraph_vector": [-48.237117, 7.015027], "paragraph_keywords": ["cell", "cells", "dataset", "graph"]}, {"paragraph_vector": [-49.14709, 6.825482], "paragraph_keywords": ["t", "graph", "-", "view"]}, {"paragraph_vector": [-48.61417, 11.814988], "paragraph_keywords": ["cell", "view", "context", "bindings"]}, {"paragraph_vector": [92.523124, 59.895408], "paragraph_keywords": ["view", "data", "number", "dataset"]}, {"paragraph_vector": [-51.294963, 11.88471], "paragraph_keywords": ["edges", "nodes", "pars", "hierarchy"]}, {"paragraph_vector": [-49.491298, 10.123065], "paragraph_keywords": ["cell", "graph", "affinity", "brushing"]}, {"paragraph_vector": [-151.529785, 12.999678], "paragraph_keywords": ["toponome", "atlas"]}], "content": {}, "doi": "10.1109/PacificVis.2012.6183594"}, {"uri": "167", "title": "Two-Level Approach to Efficient Visualization of Protein Dynamics", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ove Daae Lampe", "Ivan Viola"], "summary": "Proteins are highly flexible and large amplitude deformations of their structure, also called slow dynamics, are often decisive to their function. We present a two-level rendering approach that enables visualization of slow dynamics of large protein assemblies. Our approach is aligned with a hierarchical model of large scale molecules. Instead of constantly updating positions of large amounts of atoms, we update the position and rotation of residues, i.e., higher level building blocks of a protein. Residues are represented by one vertex only indicating its position and additional information defining the rotation. The atoms in the residues are generated on-the-fly on the GPU, exploiting the new graphics hardware geometry shader capabilities. Moreover, we represent the atoms by billboards instead of tessellated spheres. Our representation is then significantly faster and pixel precise. We demonstrate the usefulness of our new approach in the context of our collaborative bioinformatics project.", "keywords": ["structure", "representation", "need", "dynamic", "protein", "calculation", "applied", "information", "nma", "point", "approach", "biologist", "section", "result", "backbone", "shader", "example", "level", "position", "performance", "geometry", "atom", "-", "animation", "graphic", "use", "view", "vertex", "time", "groel", "rendering", "visualization", "residue"], "document_vector": [-96.22116, -47.606475], "paragraphs": [{"paragraph_vector": [24.33451, -74.680923], "paragraph_keywords": ["proteins", "form", "structures", "atoms"]}, {"paragraph_vector": [126.337379, -52.086544], "paragraph_keywords": ["information", "results", "e", "protein"]}, {"paragraph_vector": [80.692543, 21.783145], "paragraph_keywords": ["rendering", "proteins", "protein", "atoms"]}, {"paragraph_vector": [86.336326, 17.856376], "paragraph_keywords": ["section", "rendering", "technique", "protein"]}, {"paragraph_vector": [85.235336, 23.964546], "paragraph_keywords": ["rendering", "protein", "techniques", "atoms"]}, {"paragraph_vector": [90.771247, 40.939037], "paragraph_keywords": ["residue", "atom", "atoms", "residues"]}, {"paragraph_vector": [57.655925, 58.089752], "paragraph_keywords": ["shader", "atom", "atoms", "geometry"]}, {"paragraph_vector": [89.528991, 39.24628], "paragraph_keywords": ["residue", "atoms", "protein", "backbone"]}, {"paragraph_vector": [92.744865, 16.028261], "paragraph_keywords": ["protein", "axis", "case", "animation"]}, {"paragraph_vector": [89.90493, 48.569744], "paragraph_keywords": ["d", "r", "speed", "f"]}, {"paragraph_vector": [30.132303, 51.55231], "paragraph_keywords": ["ray", "calculations", "sphere", "origin"]}, {"paragraph_vector": [76.425743, -4.84855], "paragraph_keywords": ["protein", "proteins", "animation", "tool"]}, {"paragraph_vector": [77.467262, 3.683654], "paragraph_keywords": ["atoms", "structure", "protein", "view"]}, {"paragraph_vector": [77.260208, 3.196579], "paragraph_keywords": ["protein", "stereo", "backbone", "fisheye"]}, {"paragraph_vector": [83.614379, 25.727098], "paragraph_keywords": ["proteins", "approach", "level", "rendering"]}, {"paragraph_vector": [84.53775, -44.134239], "paragraph_keywords": ["provided", "dynamics", "christian", "funding"]}], "content": {}, "doi": "10.1109/IV.2010.43"}, {"uri": "168", "title": "Visual Analysis of Biomolecular Cavities: State of the Art", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["M. Krone", "B. Kozl\u00edkov\u00e1", "N. Lindow", "M. Baaden", "D. Baum", "J. Parulek", "H.-C. Hege", "I. Viola"], "summary": "In this report we review and structure the branch of molecular visualization that is concerned with the visual analysis of cavities in macromolecular protein structures. First the necessary background, the domain terminology, and the goals of analytical reasoning are introduced. Based on a comprehensive collection of relevant research works, we present a novel classification for cavity detection approaches and structure them into four distinct classes: grid-based, Voronoi-based, surface-based, and probe-based methods. The subclasses are then formed by their combinations. We match these approaches with corresponding visualization technologies starting with direct 3D visualization, followed with non-spatial visualization techniques that for example abstract the interactions between structures into a relational graph, straighten the cavity of interest to see its profile in one view, or aggregate the time sequence into a single contour plot. We also discuss the current state of methods for the visual analysis of cavities in dynamic data such as molecular dynamics simulations. Finally, we give an overview of the most common tools that are actively developed and used in the structural biology and biochemistry research. Our report is concluded by an outlook on future challenges in the field.", "keywords": ["survey", "cavity", "called", "data", "volume", "pocket", "molecule", "detect", "analysis", "et", "path", "distance", "voronoi", "interaction", "simulation", "detection", "space", "set", "method", "presented", "dynamic", "extract", "evolution", "protein", "author", "algorithm", "compute", "information", "ligand", "point", "approach", "research", "number", "extraction", "section", "result", "detected", "chemical", "diagram", "figure", "void", "proposed", "defined", "state", "sphere", "tunnel", "example", "based", "surface", "computed", "binding", "tool", "atom", "probe", "site", "s", "channel", "use", "grid", "time", "al", "property", "user", "function", "visualization", "provide", "shape"], "document_vector": [-143.864776, -8.522799], "paragraphs": [{"paragraph_vector": [25.444719, -35.737525], "paragraph_keywords": ["molecules", "molecule", "volumes", "cavities"]}, {"paragraph_vector": [25.320846, -32.211467], "paragraph_keywords": ["analysis", "requires", "chemical", "molecules"]}, {"paragraph_vector": [27.289344, -32.681999], "paragraph_keywords": ["analysis", "section", "cavities", "cavity"]}, {"paragraph_vector": [24.136442, -28.220952], "paragraph_keywords": ["molecule", "cavities", "paths", "path"]}, {"paragraph_vector": [21.155387, -25.671527], "paragraph_keywords": ["molecule", "state", "states", "volume"]}, {"paragraph_vector": [20.543075, -28.493497], "paragraph_keywords": ["molecule", "cavity", "probe", "states"]}, {"paragraph_vector": [20.609382, -25.435386], "paragraph_keywords": ["cavities", "cavity", "atom", "pockets"]}, {"paragraph_vector": [32.283508, -28.25884], "paragraph_keywords": ["cavities", "binding", "ligand", "report"]}, {"paragraph_vector": [159.907348, -22.509584], "paragraph_keywords": ["protein", "docking", "methods", "survey"]}, {"paragraph_vector": [-116.59362, -45.342491], "paragraph_keywords": ["protein", "interactions", "interaction", "voids"]}, {"paragraph_vector": [15.613421, -21.220932], "paragraph_keywords": ["methods", "cavities", "categories", "structures"]}, {"paragraph_vector": [18.867845, -22.932455], "paragraph_keywords": ["cavities", "based", "methods", "molecules"]}, {"paragraph_vector": [20.620265, -29.615406], "paragraph_keywords": ["methods", "cavities", "section", "protein"]}, {"paragraph_vector": [21.172754, -19.214487], "paragraph_keywords": ["grid", "point", "cavities", "threshold"]}, {"paragraph_vector": [11.9463, -16.956621], "paragraph_keywords": ["grid", "atom", "based", "path"]}, {"paragraph_vector": [28.573167, 19.892246], "paragraph_keywords": ["point", "space", "intersection", "rays"]}, {"paragraph_vector": [5.625237, -31.468545], "paragraph_keywords": ["cavities", "protein", "dynamics", "water"]}, {"paragraph_vector": [22.93707, -28.783309], "paragraph_keywords": ["cavity", "cavities", "grid", "time"]}, {"paragraph_vector": [22.202779, -46.134414], "paragraph_keywords": ["cavities", "cavity", "grid", "based"]}, {"paragraph_vector": [10.348749, -19.343952], "paragraph_keywords": ["voronoi", "edges", "algorithm", "paths"]}, {"paragraph_vector": [13.702532, -16.344732], "paragraph_keywords": ["atom", "spheres", "voronoi", "diagram"]}, {"paragraph_vector": [18.756233, -39.658851], "paragraph_keywords": ["caver", "tunnels", "et", "channels"]}, {"paragraph_vector": [16.805521, -22.03733], "paragraph_keywords": ["voronoi", "based", "diagram", "results"]}, {"paragraph_vector": [23.638551, -26.080253], "paragraph_keywords": ["cavities", "ion", "ligand", "methods"]}, {"paragraph_vector": [18.480123, -20.702465], "paragraph_keywords": ["gap", "sphere", "cavities", "spheres"]}, {"paragraph_vector": [14.840919, -23.86938], "paragraph_keywords": ["ligand", "tree", "protein", "path"]}, {"paragraph_vector": [12.69755, -20.281293], "paragraph_keywords": ["\u03b1", "spheres", "cavities", "delaunay"]}, {"paragraph_vector": [15.253152, -23.48995], "paragraph_keywords": ["cavities", "diagram", "paths", "probe"]}, {"paragraph_vector": [30.156383, -37.46796], "paragraph_keywords": ["cavities", "time", "computed", "voids"]}, {"paragraph_vector": [21.757539, -23.510639], "paragraph_keywords": ["voids", "grid", "based", "algorithm"]}, {"paragraph_vector": [16.955709, -19.341964], "paragraph_keywords": ["grid", "ligsite", "spheres", "sas"]}, {"paragraph_vector": [15.276365, -21.079286], "paragraph_keywords": ["space", "probe", "protein", "defined"]}, {"paragraph_vector": [17.854131, -18.924482], "paragraph_keywords": ["ses", "surface", "et", "al"]}, {"paragraph_vector": [28.041641, -28.331396], "paragraph_keywords": ["cavities", "computed", "cavity", "surface"]}, {"paragraph_vector": [18.548517, -21.549873], "paragraph_keywords": ["cavities", "grid", "points", "ses"]}, {"paragraph_vector": [17.349033, -18.298919], "paragraph_keywords": ["cavities", "protein", "points", "ses"]}, {"paragraph_vector": [17.175125, -15.856878], "paragraph_keywords": ["ligand", "cavities", "surface", "grid"]}, {"paragraph_vector": [24.523929, -14.440644], "paragraph_keywords": ["cavities", "spheres", "analysis", "methods"]}, {"paragraph_vector": [54.419471, -16.870418], "paragraph_keywords": ["tools", "cavities", "visualization", "methods"]}, {"paragraph_vector": [46.6324, -25.435285], "paragraph_keywords": ["visualizations", "cavity", "results", "spheres"]}, {"paragraph_vector": [49.187557, -20.355833], "paragraph_keywords": ["cavities", "surface", "paths", "proteins"]}, {"paragraph_vector": [35.137512, -38.390247], "paragraph_keywords": ["cavities", "cavity", "visualization", "user"]}, {"paragraph_vector": [30.216035, -37.407802], "paragraph_keywords": ["time", "tunnel", "properties", "channel"]}, {"paragraph_vector": [51.409599, -24.169013], "paragraph_keywords": ["occlusion", "et", "methods", "visualization"]}, {"paragraph_vector": [0.490724, -13.134963], "paragraph_keywords": ["surface", "descriptor", "shape", "cavities"]}, {"paragraph_vector": [-11.128414, -15.867588], "paragraph_keywords": ["surface", "differences", "method", "surfaces"]}, {"paragraph_vector": [62.72388, -20.916522], "paragraph_keywords": ["methods", "analysis", "cavities", "simulation"]}, {"paragraph_vector": [38.284912, -41.111763], "paragraph_keywords": ["data", "analysis", "cavities", "tools"]}, {"paragraph_vector": [62.319404, -67.075904], "paragraph_keywords": ["ligand", "provide", "visualization", "methods"]}, {"paragraph_vector": [17.525844, -36.672698], "paragraph_keywords": ["cavities", "comparison", "results", "example"]}, {"paragraph_vector": [14.323488, -46.328086], "paragraph_keywords": ["cavities", "assessment", "example", "provide"]}, {"paragraph_vector": [27.558874, -31.138114], "paragraph_keywords": ["tools", "cavities", "binding", "yasara"]}, {"paragraph_vector": [-14.703943, -56.787837], "paragraph_keywords": ["visualization", "tools", "research", "cavity"]}, {"paragraph_vector": [45.120426, -45.472667], "paragraph_keywords": ["visualization", "protein", "ligand", "interaction"]}, {"paragraph_vector": [-156.236541, 14.710935], "paragraph_keywords": ["research", "project", "grants", "providing"]}], "content": {}, "doi": "10.1109/TVCG.2019.2934612"}, {"uri": "169", "title": "Molecular Surface Maps", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Michael Krone", "Florian Frie\u00df", "Katrin Scharnowski", "Guido Reina", "Silvia Fademrecht", "Tobias Kulschewski", "J\u00fcrgen Pleiss", "Thomas Ertl"], "summary": "We present Molecular Surface Maps, a novel, view-independent, and concise representation for molecular surfaces. It transfers the well-known world map metaphor to molecular visualization. Our application maps the complex molecular surface to a simple 2D representation through a spherical intermediate, the Molecular Surface Globe. The Molecular Surface Map concisely shows arbitrary attributes of the original molecular surface, such as biochemical properties or geometrical features. This results in an intuitive overview, which allows researchers to assess all molecular surface attributes at a glance. Our representation can be used as a visual summarization of a molecule\u2019s interface with its environment. In particular, Molecular Surface Maps simplify the analysis and comparison of different data sets or points in time. Furthermore, the map representation can be used in a Space-time Cube to analyze time-dependent data from molecular simulations without the need for animation. We show the feasibility of Molecular Surface Maps for different typical analysis tasks of biomolecular data.", "keywords": ["data", "value", "area", "molecule", "lipase", "et", "analysis", "application", "morphing", "simulation", "method", "mesh", "globe", "protein", "distortion", "input", "genus", "triangle", "point", "section", "figure", "step", "color", "sphere", "example", "map", "surface", "binding", "mapping", "based", "water", "site", "show", "s", "channel", "use", "vertex", "time", "projection", "domain", "al", "user", "force", "visualization"], "document_vector": [-151.413299, -23.124937], "paragraphs": [{"paragraph_vector": [57.54673, -19.539876], "paragraph_keywords": ["surface", "visualization", "map", "properties"]}, {"paragraph_vector": [52.50291, -20.350032], "paragraph_keywords": ["surface", "maps", "surfaces", "genus"]}, {"paragraph_vector": [58.645442, -21.474393], "paragraph_keywords": ["surface", "ses", "map", "surfaces"]}, {"paragraph_vector": [-40.2075, -27.109243], "paragraph_keywords": ["surface", "data", "topic", "protein"]}, {"paragraph_vector": [17.51906, 6.198374], "paragraph_keywords": ["mapping", "surface", "sphere", "establish"]}, {"paragraph_vector": [44.401775, -17.236305], "paragraph_keywords": ["surface", "molecule", "map", "method"]}, {"paragraph_vector": [13.3442, 1.88626], "paragraph_keywords": ["channel", "surface", "genus", "globe"]}, {"paragraph_vector": [11.675054, 3.282236], "paragraph_keywords": ["surface", "sphere", "globe", "channels"]}, {"paragraph_vector": [18.647594, 9.575523], "paragraph_keywords": ["vertices", "values", "surface", "input"]}, {"paragraph_vector": [27.317825, -0.435691], "paragraph_keywords": ["surface", "points", "sphere", "map"]}, {"paragraph_vector": [52.770164, -24.850811], "paragraph_keywords": ["map", "projection", "time", "maps"]}, {"paragraph_vector": [53.80484, -20.194379], "paragraph_keywords": ["surface", "ses", "probe", "channels"]}, {"paragraph_vector": [4.294992, -2.494032], "paragraph_keywords": ["surface", "mesh", "method", "tunnel"]}, {"paragraph_vector": [44.813076, -13.310418], "paragraph_keywords": ["surface", "map", "lines", "orientation"]}, {"paragraph_vector": [64.030448, -22.775114], "paragraph_keywords": ["surface", "binding", "maps", "colorings"]}, {"paragraph_vector": [32.040821, -2.800591], "paragraph_keywords": ["surface", "point", "application", "molecule"]}, {"paragraph_vector": [13.163615, 11.483559], "paragraph_keywords": ["mesh", "surface", "holes", "channels"]}, {"paragraph_vector": [19.288213, 9.709777], "paragraph_keywords": ["forces", "mesh", "sphere", "force"]}, {"paragraph_vector": [20.843332, 8.686785], "paragraph_keywords": ["value", "vertices", "north", "pole"]}, {"paragraph_vector": [20.321239, 9.861708], "paragraph_keywords": ["map", "triangles", "vertex", "meridian"]}, {"paragraph_vector": [81.526519, 15.311398], "paragraph_keywords": ["map", "method", "mapping", "ses"]}, {"paragraph_vector": [20.388097, -11.367821], "paragraph_keywords": ["triangles", "triangle", "based", "distortion"]}, {"paragraph_vector": [55.070354, -56.480045], "paragraph_keywords": ["binding", "site", "time", "calb"]}, {"paragraph_vector": [16.481355, -48.80764], "paragraph_keywords": ["water", "protein", "binding", "surface"]}, {"paragraph_vector": [-11.296689, -55.566001], "paragraph_keywords": ["lipases", "surface", "protein", "differences"]}, {"paragraph_vector": [61.823005, -30.908496], "paragraph_keywords": ["surface", "\u03b2", "maps", "rated"]}, {"paragraph_vector": [57.985378, -25.129032], "paragraph_keywords": ["surface", "map", "example", "projections"]}, {"paragraph_vector": [-156.23616, 14.472241], "paragraph_keywords": ["providing", "software", "sfb", "forschungsgemeinschaft"]}], "content": {}, "doi": "10.1109/TVCG.2013.158"}, {"uri": "170", "title": "Enhancing Visualization of Molecular Simulations using Sonification", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Benjamin Rau", "Florian Frie\u00df", "Michael Krone", "Christoph M\u00fcller", "Thomas Ertl"], "summary": "Scientific visualization is an application area for virtual reality environments like stereoscopic displays or CAVEs. Especially interactive molecular visualizations that show the complex threedimensional structures found in structural biology are often investigated using such environments. In contrast to VR applications like simulators, molecular visualization typically lacks auditory output. Nevertheless, sonification can be used to convey information about the data. In our work, we use sound to highlight events extracted from a molecular dynamics simulation. This not only offloads information from the visual channel, but can also guide the attention of the analyst towards important phenomena even if they are occluded in the visualization. Sound also creates a higher level of immersion, which can be beneficial for educational purposes. In this paper, we detail our application that adds sonification to the visualization of molecular simulations.", "keywords": ["data", "molecule", "sound", "event", "distance", "analysis", "representation", "sample", "hydrogen", "et", "developed", "simulation", "protein", "information", "research", "section", "system", "bond", "example", "based", "work", "binding", "immersion", "level", "effect", "use", "implemented", "time", "al", "sonification", "user", "visualization", "provide"], "document_vector": [144.370117, -49.936447], "paragraphs": [{"paragraph_vector": [177.067825, 18.545885], "paragraph_keywords": ["data", "simulation", "information", "sound"]}, {"paragraph_vector": [-179.672714, 18.634328], "paragraph_keywords": ["section", "sound", "sonification", "system"]}, {"paragraph_vector": [177.813278, 16.613834], "paragraph_keywords": ["developed", "data", "sonification", "devices"]}, {"paragraph_vector": [176.021316, 16.389524], "paragraph_keywords": ["sonification", "dna", "indicators", "effect"]}, {"paragraph_vector": [177.400634, 17.143434], "paragraph_keywords": ["sonification", "data", "simulation", "sound"]}, {"paragraph_vector": [177.773437, 14.024623], "paragraph_keywords": ["simulations", "protein", "representations", "case"]}, {"paragraph_vector": [178.326736, 17.6655], "paragraph_keywords": ["events", "implemented", "sonification", "time"]}, {"paragraph_vector": [178.963592, 17.4675], "paragraph_keywords": ["time", "hydrogen", "events", "bond"]}, {"paragraph_vector": [178.417022, 17.647571], "paragraph_keywords": ["distance", "binding", "site", "user"]}, {"paragraph_vector": [176.661117, 18.133642], "paragraph_keywords": ["events", "sound", "user", "provide"]}, {"paragraph_vector": [179.711975, 17.839471], "paragraph_keywords": ["sonification", "events", "time", "data"]}, {"paragraph_vector": [177.132385, 17.603393], "paragraph_keywords": ["sound", "protein", "sonification", "spectrum"]}, {"paragraph_vector": [178.046295, 17.544538], "paragraph_keywords": ["sound", "sonification", "user", "events"]}, {"paragraph_vector": [177.530548, 18.532033], "paragraph_keywords": ["events", "localization", "sonification", "immersion"]}, {"paragraph_vector": [177.595474, 17.766998], "paragraph_keywords": ["want", "sounds", "user", "synthesis"]}], "content": {}, "doi": "10.23919/iLRN47897.2020.9155185"}, {"uri": "171", "title": "On the Use of 1D, 2D, and 3D Visualisation for Molecular Graphics", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Julian Heinrich", "Michael Burch", "Sean I. O\u2019Donoghue"], "summary": "Macromolecular structures have become easily accessible by means of large databases, for example the World Wide Protein Data Bank. Several years ago, existing data in this field of research was rare and demanded for specialists and experts which wasted a lot of time to explore the data. Today, various visualisation techniques, tools, and systems have been designed and developed to support biochemists, chemists, or molecular biologists at certain tasks such as analysing protein functions, exploring ligand-binding sites, or understanding RNA signal and message processing. The intent of this paper is to discuss how 1D, 2D, and 3D representations are typically employed to facilitate some of these tasks. We describe common tasks that involve molecular structures and illustrate how 1D, 2D, and 3D visualisations are currently being used to address these. We discuss the benefits and drawbacks of these concepts in the context of these tasks and propose methods to provide empirical evidence by means of user evaluations.", "keywords": ["list", "visualisation", "data", "structure", "molecule", "distance", "representation", "interaction", "protein", "sequence", "information", "ligand", "occlusion", "figure", "technique", "drawing", "study", "performance", "atom", "view", "feature", "time", "participant", "device", "user", "respect", "task"], "document_vector": [-178.793121, 38.213775], "paragraphs": [{"paragraph_vector": [116.426933, -55.704395], "paragraph_keywords": ["data", "structures", "dna", "protein"]}, {"paragraph_vector": [129.677719, -17.699514], "paragraph_keywords": ["visualisation", "tasks", "study", "visualisations"]}, {"paragraph_vector": [128.715286, -14.095794], "paragraph_keywords": ["occlusion", "data", "graph", "structures"]}, {"paragraph_vector": [146.228515, 5.252112], "paragraph_keywords": ["tasks", "interaction", "sequences", "sequence"]}, {"paragraph_vector": [91.84449, -41.652339], "paragraph_keywords": ["structure", "distance", "figure", "provide"]}, {"paragraph_vector": [121.928817, -38.866348], "paragraph_keywords": ["structure", "molecule", "figure", "representation"]}, {"paragraph_vector": [146.568359, 2.443461], "paragraph_keywords": ["interaction", "user", "devices", "data"]}, {"paragraph_vector": [137.486801, -17.16374], "paragraph_keywords": ["visualisations", "tasks", "representations", "molecules"]}, {"paragraph_vector": [-104.260414, -72.816764], "paragraph_keywords": ["molecule", "tasks", "sequence", "features"]}, {"paragraph_vector": [141.673492, -23.944335], "paragraph_keywords": ["task", "visualisation", "study", "solved"]}, {"paragraph_vector": [124.73484, -62.367549], "paragraph_keywords": ["tasks", "distance", "features", "information"]}, {"paragraph_vector": [130.915039, -28.070272], "paragraph_keywords": ["interaction", "data", "views", "performance"]}, {"paragraph_vector": [144.542053, -29.948266], "paragraph_keywords": ["structures", "tasks", "respect", "techniques"]}, {"paragraph_vector": [148.557479, 3.632529], "paragraph_keywords": ["time", "visualisation", "participants", "devices"]}], "content": {}, "doi": "10.1109/TVCG.2010.250"}, {"uri": "172", "title": "An odor visualization film based on multi colors fluorescent microbeads and single color fluorescent multi microbeads", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Wu Shuanghong", "Fumihiro Sassa", "Kenshi Hayashi"], "summary": "The microbeads shape sensor film was developed in this work combines fluorescent dye agents whose fluorescent intensity can be influenced in touch of a specific odor, and Molecular Imprinting Polymer. Like photographic films, the fluorescent dye microbeads film can visualize the odor gas diffusion with the CCD camera. To enhance the fluorescence intensity of Quinine Sulfate increases in the water environment, hydrogel was applied in the film. For developing multi-odor sensors, we made four kinds of fluorescent microbeads to test whether the interaction among fluorescent dyes happened. It turned out microbeads structure successfully remained characters of every fluorescent dye. Keywords\u2014fluorescent dye agent; Molecular Imprinting Polymer; microbeads; odor visualization", "keywords": ["color", "odor", "microbeads", "group", "fluorescent", "gas", "solution", "film", "fluorescence", "hydrogel", "acid", "dye"], "document_vector": [100.380073, 14.650447], "paragraphs": [{"paragraph_vector": [-34.360828, 27.253784], "paragraph_keywords": ["acid", "microbeads", "hydrogel", "quinine"]}, {"paragraph_vector": [-33.333362, 29.127975], "paragraph_keywords": ["microbeads", "fluorescent", "film", "camera"]}, {"paragraph_vector": [-34.132667, 29.027652], "paragraph_keywords": ["fluorescent", "dye", "microbeads", "odor"]}, {"paragraph_vector": [-35.654621, 28.406721], "paragraph_keywords": ["fluorescence", "sensors", "imaging", "discrimination"]}], "content": {}, "doi": "10.1109/TVCG.2021.3051632"}, {"uri": "173", "title": "High quality illustrative effects for molecular rendering", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["P. Hermosilla", "V. Guallar", "A. Vinacua", "P. Hermosillaa", "V. Guallarb", "A. Vinacuaa", "P.P. V\u00e1zqueza"], "summary": "All-atom simulations are crucial in biotechnology. In Pharmacology, for example, molecular knowledge of proteindrug interactions is essential in the understanding of certain pathologies and in the development of improved drugs. To achieve this detailed information, fast and enhanced molecular visualization is critical. Moreover, hardware and software developments quickly deliver extensive data, providing intermediate results that can be analyzed by scientists in order to interact with the simulation process and direct it to a more promising configuration. In this paper we present a GPU-friendly data structure for real-time illustrative visualization of all-atom simulations. Our system generates both ambient occlusion and halos using an occupancy pyramid that needs no precalculation and that is updated on the fly during simulation, allowing the real time rendering of simulation results at sustained high framerates.", "keywords": [], "document_vector": [-64.804794, -56.123039], "paragraphs": [{"paragraph_vector": [-128.638931, 21.784866], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/MCG.2012.93"}, {"uri": "174", "title": "empty", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "A cadmium-binding, genetically encoded protein tag, consisting of three repeats of metallothionein (3MT), can be used in electron microscopy for the visualization of multimericbut not monomeric-tagged proteins due to insufficient electron density in monomeric proteins. Here, we present a technique for detecting monomeric 3MT-tagged green fluorescent protein (GFP-3MT) using a platinum compound to intensify the electron density. Substitution of cadmium by platinum as a result of incubating purified cadmium-binding 3MT-tagged GFP (GFP-Cd-3MT) with cis-diamminedichloroplatinum(II) (cisDDP) was assessed by a UV absorption band centered at 284 nm thereby indicating platinum\u2013sulfhydryl bonds. The incubation time and the concentration of cisDDP to reach maximal absorption were 2 h and 36-fold molar equivalent of cisDDP, respectively. GFP-Pt-3MT isolated by gel filtration chromatography contained 29 platinum atoms per single GFP-3MT molecule. Electron-dense particles were observed in a GFP-Pt-3MT sample by electron microscopy without negative staining. Further image processing and image analysis demonstrated that particles with higher density relative to their surroundings were detectable in both GFP-Cd-3MT and GFP-Pt-3MT samples. These results demonstrate that replacement of cadmium with platinum, together with proper image analyses, improve detection efficiency and enable the visualization of 3MT-tagged monomeric protein by electron microscopy.", "keywords": ["\u03bcm", "bound", "molecule", "electron", "analysis", "purified", "protein", "concentration", "microscopy", "platinum", "c", "density", "metallothionein", "gfp", "cadmium", "atom", "expressed", "gel", "tag", "labeling", "image", "cisddp", "visualization", "cell", "fusion", "metal"], "document_vector": [100.059684, 35.615921], "paragraphs": [{"paragraph_vector": [-39.212554, 4.89824], "paragraph_keywords": ["components", "microscopy", "guest", "ow"]}, {"paragraph_vector": [-42.629917, 1.353614], "paragraph_keywords": ["tag", "proteins", "protein", "electron"]}, {"paragraph_vector": [-43.012428, -1.333814], "paragraph_keywords": ["metallothionein", "cisddp", "tag", "platinum"]}, {"paragraph_vector": [-50.569797, -5.697844], "paragraph_keywords": ["gfp", "c", "purified", "cisddp"]}, {"paragraph_vector": [-43.435375, -3.270529], "paragraph_keywords": ["gfp", "images", "metallothionein", "coli"]}, {"paragraph_vector": [-44.533992, -2.421167], "paragraph_keywords": ["gfp", "cisddp", "platinum", "bound"]}, {"paragraph_vector": [-44.150928, -3.863923], "paragraph_keywords": ["gfp", "images", "electron", "particles"]}, {"paragraph_vector": [-44.139556, -5.209095], "paragraph_keywords": ["protein", "electron", "gfp", "metallothionein"]}, {"paragraph_vector": [-44.648101, -2.701424], "paragraph_keywords": ["cisddp", "protein", "platinum", "proteins"]}, {"paragraph_vector": [-43.009426, -3.287154], "paragraph_keywords": ["electron", "tag", "protein", "ferritin"]}, {"paragraph_vector": [-46.490421, 12.254804], "paragraph_keywords": ["cisddp", "guest", "ovem", "ber"]}, {"paragraph_vector": [-150.808609, 16.395891], "paragraph_keywords": []}, {"paragraph_vector": [-151.883453, 15.932256], "paragraph_keywords": []}, {"paragraph_vector": [-151.640701, 16.538587], "paragraph_keywords": []}, {"paragraph_vector": [-151.076675, 14.33498], "paragraph_keywords": []}, {"paragraph_vector": [-150.756484, 15.242461], "paragraph_keywords": []}, {"paragraph_vector": [-152.338012, 16.807546], "paragraph_keywords": []}, {"paragraph_vector": [-151.397338, 15.646877], "paragraph_keywords": []}, {"paragraph_vector": [-42.8232, -1.795186], "paragraph_keywords": ["proteins", "platinum", "analysis", "bound"]}], "content": {}, "doi": "10.1109/IPDPSW.2016.26"}, {"uri": "175", "title": "Using Computational Models for Development of the Three-Dimensional Visualization in Molecular Biology", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ravil I. Muhamedyev", "Y. Daineko", "D. Bari", "Alma T. Mansharipova"], "summary": "A method for three-dimensional visualization of molecular biology processes modeled by chemical kinetic equations is presented. To implement this visualization, the software in C# and a database of three-dimensional forms that model molecular complexes are developed. The quantitative parameters in this visualization scheme are determined from kinetic equations governing the participating components, so our visualization is not only qualitative but also quantitative. As a case study, it was visualized a mathematical model for mitochondriadependent apoptosis (programmed cell death) proposed by Bagci et all [19].", "keywords": ["research", "science"], "document_vector": [-151.3031, -11.871794], "paragraphs": [{"paragraph_vector": [-156.39978, 10.799324], "paragraph_keywords": ["research", "sciences", "gf", "mes"]}], "content": {}, "doi": "10.1109/VISUAL.2004.7"}, {"uri": "176", "title": "Visualizing Validation of Protein Surface Classifiers", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["A. Sarikaya", "D. Albers", "J. Mitchell", "M. Gleicher"], "summary": "Many bioinformatics applications construct classifiers that are validated in experiments that compare their results to known ground truth over a corpus. In this paper, we introduce an approach for exploring the results of such classifier validation experiments, focusing on classifiers for regions of molecular surfaces. We provide a tool that allows for examining classification performance patterns over a test corpus. The approach combines a summary view that provides information about an entire corpus of molecules with a detail view that visualizes classifier results directly on protein surfaces. Rather than displaying miniature 3D views of each molecule, the summary provides 2D glyphs of each protein surface arranged in a reorderable, small-multiples grid. Each summary is specifically designed to support visual aggregation to allow the viewer to both get a sense of aggregate properties as well as the details that form them. The detail view provides a 3D visualization of each protein surface coupled with interaction techniques designed to support key tasks, including spatial aggregation and automated camera touring. A prototype implementation of our approach is demonstrated on protein surface classifier experiments.", "keywords": ["data", "interest", "molecule", "detail", "classifier", "classification", "decision", "support", "protein", "approach", "result", "system", "figure", "color", "example", "surface", "binding", "performance", "design", "experiment", "display", "use", "model", "overview", "viewpoint", "glyph", "view", "region", "pattern", "property", "user", "designed", "visualization", "provide", "validation", "viewer"], "document_vector": [176.571777, 45.408512], "paragraphs": [{"paragraph_vector": [-45.850677, -55.287723], "paragraph_keywords": ["results", "molecules", "surface", "molecule"]}, {"paragraph_vector": [-48.043582, -58.883235], "paragraph_keywords": ["surface", "validation", "experiments", "model"]}, {"paragraph_vector": [178.155853, -66.167472], "paragraph_keywords": ["designs", "support", "visualization", "results"]}, {"paragraph_vector": [145.220733, -39.796447], "paragraph_keywords": ["visualization", "views", "molecules", "support"]}, {"paragraph_vector": [-91.866333, -70.320373], "paragraph_keywords": ["provide", "molecules", "overview", "collection"]}, {"paragraph_vector": [-46.931785, -55.429973], "paragraph_keywords": ["molecules", "performance", "viewer", "decisions"]}, {"paragraph_vector": [177.491485, -63.989974], "paragraph_keywords": ["molecule", "glyph", "molecules", "performance"]}, {"paragraph_vector": [-44.27515, -61.289787], "paragraph_keywords": ["classifications", "elements", "support", "data"]}, {"paragraph_vector": [-46.750179, -57.509231], "paragraph_keywords": ["surface", "decisions", "color", "confidence"]}, {"paragraph_vector": [-43.077239, -61.944782], "paragraph_keywords": ["surface", "boundaries", "points", "regions"]}, {"paragraph_vector": [113.416763, 26.232395], "paragraph_keywords": ["viewpoint", "surface", "cluster", "region"]}, {"paragraph_vector": [-37.38449, -67.953147], "paragraph_keywords": ["color", "classification", "ramp", "data"]}, {"paragraph_vector": [-47.706996, -54.659152], "paragraph_keywords": ["dna", "binding", "protein", "corpus"]}, {"paragraph_vector": [-50.464179, -54.592998], "paragraph_keywords": ["shows", "performance", "errors", "molecules"]}, {"paragraph_vector": [-43.779796, -54.576622], "paragraph_keywords": ["binding", "classifier", "figure", "sites"]}, {"paragraph_vector": [-45.842063, -55.34217], "paragraph_keywords": ["approach", "molecules", "view", "overview"]}, {"paragraph_vector": [-48.377597, -54.631549], "paragraph_keywords": ["patterns", "system", "performance", "controlled"]}], "content": {}, "doi": "10.1109/TVCG.2014.2346404"}, {"uri": "177", "title": "BioLinker: Bottom-up Exploration of Protein Interaction Networks", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Tommy Dang", "Paul Murray", "Angus Forbes"], "summary": "Systems biologists and cancer researchers require interactive visualization tools that enable them to more easily navigate and discover patterns at different levels of the biological hierarchy of signaling pathways. Furthermore, biologists are often interested in understanding and exploring the causal biochemical links between processes. When exploring the literature of particular biological pathways or specific proteins within those pathways, biologists find it useful to know the contexts in which biochemical links are active and, importantly, to be aware of potential conflicts when different experiments introduce alternative interpretations of the function of a pathway or biochemical reaction. We introduce BioLinker, a interactive visualization system that helps users to perform bottom-up exploration of complex protein interaction networks. Five interconnected views provide the user with a range of ways to explore pathway data, including views that show potential conflicts within pathway databases and publications and that highlight contextual information about individual proteins. Additionally, we discuss system details to show how our system manages the large amount of protein interactions extracted from the literature of biological pathways.", "keywords": ["data", "pmc", "text", "paper", "server", "interaction", "database", "protein", "network", "cancer", "information", "index", "selected", "publication", "card", "type", "expert", "view", "biolinker", "user", "visualization", "cell", "context", "task"], "document_vector": [22.222557, 64.583221], "paragraphs": [{"paragraph_vector": [-128.241012, -36.095241], "paragraph_keywords": ["interactions", "protein", "biolinker", "networks"]}, {"paragraph_vector": [-125.070671, -32.512283], "paragraph_keywords": ["protein", "proteins", "network", "pathways"]}, {"paragraph_vector": [-125.384628, -33.259559], "paragraph_keywords": ["protein", "server", "tasks", "visualization"]}, {"paragraph_vector": [-122.765602, -25.141094], "paragraph_keywords": ["information", "paper", "pmc", "index"]}, {"paragraph_vector": [-122.485275, -30.103757], "paragraph_keywords": ["index", "cards", "interaction", "example"]}, {"paragraph_vector": [-106.82582, -8.430004], "paragraph_keywords": ["protein", "index", "cards", "cancer"]}, {"paragraph_vector": [-119.90274, -28.732166], "paragraph_keywords": ["context", "protein", "cancer", "visualization"]}, {"paragraph_vector": [-125.02568, -27.381248], "paragraph_keywords": ["index", "publication", "matrix", "time"]}, {"paragraph_vector": [-124.867576, -32.694942], "paragraph_keywords": ["visualization", "biolinker", "data", "server"]}], "content": {}, "doi": "10.1109/IHMSC.2017.24"}, {"uri": "178", "title": "Visualizing multifunctional PPI network with Gene Ontology annotation", "timestamp": "2020", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sangyeon Lee"], "summary": "Protein-protein interaction (PPI) network data contain interactions between two proteins in organisms. Some proteins are involved in more than one biological process. Those multifunctional proteins are important in the biological system because they connect various biological pathways. However, the multifunctionality of proteins increases the complexity of the PPI network and makes visualizing a challenging problem. In this research, we suggest a new method to visualize the multifunctional PPI network by integrating interaction data and Gene Ontology (GO) terms of each protein. By matching GO annotations of two proteins, we annotate edges between those two proteins with common GO terms. Also, we suggest a novel algorithm for color mapping specialized to express hierarchical relationships among GO terms based on the multidimensional scaling algorithm. Annotated edges are divided corresponding to their GO terms, and visualized by selected color, width, and opacity to represent semantic relationships among edges clearly. Those visual features are assigned to represent semantic distance among GO terms and their concreteness. This visualization method is available through the web. Keywords\u2014Multifunctional protein, network visualization, dynamic color mapping, Gene ontology", "keywords": ["color", "node", "method", "data", "protein", "process", "feature", "network", "similarity", "function", "distance", "involved", "edge", "ppi", "show", "interaction", "term"], "document_vector": [14.939537, 50.857585], "paragraphs": [{"paragraph_vector": [-107.074234, -23.710237], "paragraph_keywords": ["terms", "proteins", "data", "process"]}, {"paragraph_vector": [-121.983543, -29.873081], "paragraph_keywords": ["data", "edges", "number", "terms"]}, {"paragraph_vector": [-121.56694, -34.44178], "paragraph_keywords": ["terms", "term", "ancestor", "similarity"]}, {"paragraph_vector": [-73.348068, -37.115077], "paragraph_keywords": ["terms", "term", "depth", "lightness"]}, {"paragraph_vector": [-119.51049, -34.192317], "paragraph_keywords": ["terms", "edges", "method", "color"]}, {"paragraph_vector": [-120.594215, -33.562164], "paragraph_keywords": ["color", "terms", "shows", "distance"]}, {"paragraph_vector": [-106.471481, -26.585777], "paragraph_keywords": ["data", "network", "protein", "proteins"]}, {"paragraph_vector": [-127.55664, -34.406929], "paragraph_keywords": ["network", "edge", "method", "ppi"]}], "content": {}, "doi": "10.1109/ICSENS.2018.8589830"}, {"uri": "179", "title": "Visualization of Lattice-Based Protein Folding Simulations", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sebastian P\u00f6tzsch", "Gerik Scheuermann", "Michael T. Wolfinger", "Christoph Flamm", "Peter F. Stadler"], "summary": "Analysis of the spatial structure of proteins including folding processes is a challenge for modern bioinformatics. Due to limited experimental access to folding processes, computer simulations are a standard approach. Since realistic continuous (all-atom) simulations are far too expensive, lattice based protein folding simulations are a common coarse-graining. In this paper, we present a visualization tool for lattice based protein folding simulations. The system is based on Shneiderman\u2019s mantra \u201dOverview first, zoom and filter, details on demand\u201d and uses a collection of information visualization techniques including multiple views, focus+context and table lenses which have been tailored towards our data. We demonstrate the potential of information visualization techniques for providing insight into such simulations. Keywords\u2014 Information visualization, multiple views, overview+detail, design guidelines, focus+context.", "keywords": ["data", "structure", "acid", "simulation", "protein", "contact", "matrix", "figure", "folding", "based", "tool", "energy", "show", "model", "overview", "view", "lattice", "user", "visualization"], "document_vector": [-149.005157, 20.939643], "paragraphs": [{"paragraph_vector": [14.431958, -68.7695], "paragraph_keywords": ["protein", "acids", "structure", "lattice"]}, {"paragraph_vector": [160.413223, -51.686943], "paragraph_keywords": ["overview", "data", "visualization", "view"]}, {"paragraph_vector": [140.831893, -11.367534], "paragraph_keywords": ["structure", "acids", "lattice", "protein"]}, {"paragraph_vector": [120.700698, -54.106681], "paragraph_keywords": ["folding", "simulations", "tool", "visualization"]}, {"paragraph_vector": [53.578456, -68.418281], "paragraph_keywords": ["simulations", "energy", "view", "simulation"]}, {"paragraph_vector": [64.123306, -63.855175], "paragraph_keywords": ["energy", "shows", "simulations", "simulation"]}, {"paragraph_vector": [69.472793, -70.165817], "paragraph_keywords": ["structure", "matrix", "structures", "parts"]}, {"paragraph_vector": [61.141986, -64.983634], "paragraph_keywords": ["figure", "energy", "simulations", "structure"]}, {"paragraph_vector": [85.456977, -74.118423], "paragraph_keywords": ["based", "support", "guide", "data"]}], "content": {}, "doi": "10.1109/ISDA.2011.6121659"}, {"uri": "180", "title": "Physics-based Visual Characterization of Molecular Interaction Forces", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Pedro Hermosilla", "Jorge Estrada", "Victor Guallar", "Timo Ropinski", "Pere-Pau V\u00e1zquez"], "summary": "Molecular simulations are used in many areas of biotechnology, such as drug design and enzyme engineering. Despite the development of automatic computational protocols, analysis of molecular interactions is still a major aspect where human comprehension and intuition are key to accelerate, analyze, and propose modifications to the molecule of interest. Most visualization algorithms help the users by providing an accurate depiction of the spatial arrangement: the atoms involved in inter-molecular contacts. There are few tools that provide visual information on the forces governing molecular docking. However, these tools, commonly restricted to close interaction between atoms, do not consider whole simulation paths, long-range distances and, importantly, do not provide visual cues for a quick and intuitive comprehension of the energy functions (modeling intermolecular interactions) involved. In this paper, we propose visualizations designed to enable the characterization of interaction forces by taking into account several relevant variables such as molecule-ligand distance and the energy function, which is essential to understand binding affinities. We put emphasis on mapping molecular docking paths obtained from Molecular Dynamics or Monte Carlo simulations, and provide time-dependent visualizations for different energy components and particle resolutions: atoms, groups or residues. The presented visualizations have the potential to support domain experts in a more efficient drug or enzyme design process.", "keywords": ["data", "group", "structure", "molecule", "analysis", "path", "distance", "application", "interaction", "simulation", "drug", "protein", "compute", "information", "ligand", "solvation", "result", "range", "system", "figure", "step", "understand", "color", "computation", "process", "technique", "example", "based", "binding", "level", "atom", "design", "energy", "filtering", "expert", "van", "view", "time", "domain", "contribution", "user", "vdw", "force", "visualization", "analyze", "residue"], "document_vector": [-105.601547, -0.980078], "paragraphs": [{"paragraph_vector": [5.614051, -52.614875], "paragraph_keywords": ["analysis", "computation", "design", "group"]}, {"paragraph_vector": [48.619232, -47.332954], "paragraph_keywords": ["range", "ligand", "domain", "design"]}, {"paragraph_vector": [61.373352, -31.88607], "paragraph_keywords": ["techniques", "structures", "visualization", "energies"]}, {"paragraph_vector": [74.329177, 12.552963], "paragraph_keywords": ["interactions", "based", "frame", "techniques"]}, {"paragraph_vector": [57.593112, -38.45071], "paragraph_keywords": ["atom", "surface", "atoms", "interactions"]}, {"paragraph_vector": [16.409736, -81.35514], "paragraph_keywords": ["interaction", "tools", "energy", "binding"]}, {"paragraph_vector": [5.404053, -50.30751], "paragraph_keywords": ["energy", "ligand", "interaction", "binding"]}, {"paragraph_vector": [-20.562095, -32.659656], "paragraph_keywords": ["atoms", "experts", "energy", "domain"]}, {"paragraph_vector": [45.86816, -44.339244], "paragraph_keywords": ["energies", "interaction", "analysis", "energy"]}, {"paragraph_vector": [77.996932, 21.379898], "paragraph_keywords": ["shader", "compute", "data", "energy"]}, {"paragraph_vector": [20.443151, 15.298714], "paragraph_keywords": ["energy", "groups", "index", "cell"]}, {"paragraph_vector": [50.498374, -48.103019], "paragraph_keywords": ["filtering", "energy", "structure", "idioms"]}, {"paragraph_vector": [54.346649, -45.62281], "paragraph_keywords": ["energy", "groups", "plane", "energies"]}, {"paragraph_vector": [52.335586, -51.627582], "paragraph_keywords": ["energy", "path", "view", "user"]}, {"paragraph_vector": [67.80046, -36.56166], "paragraph_keywords": ["hues", "information", "energy", "avoid"]}, {"paragraph_vector": [62.978134, -47.786922], "paragraph_keywords": ["groups", "magnitudes", "ligand", "cones"]}, {"paragraph_vector": [-61.560459, -89.849418], "paragraph_keywords": ["analysis", "analyze", "projection", "groups"]}, {"paragraph_vector": [34.149559, -46.079856], "paragraph_keywords": ["binding", "protein", "contributions", "ligand"]}, {"paragraph_vector": [30.534721, -48.785854], "paragraph_keywords": ["energy", "vdw", "solvation", "force"]}, {"paragraph_vector": [17.662797, -50.584877], "paragraph_keywords": ["ligand", "protein", "shows", "simulation"]}, {"paragraph_vector": [40.413795, -41.165092], "paragraph_keywords": ["residues", "tool", "allows", "experts"]}, {"paragraph_vector": [53.223937, -58.601722], "paragraph_keywords": ["interaction", "protein", "use", "found"]}, {"paragraph_vector": [51.740459, -64.420066], "paragraph_keywords": ["analysis", "energies", "experts", "time"]}, {"paragraph_vector": [62.128665, -33.346473], "paragraph_keywords": ["simulation", "ligand", "inspection", "extend"]}], "content": {}, "doi": "10.1109/TVCG.2007.70614"}, {"uri": "181", "title": "LEARNING AND VISUALIZING STATISTICAL RELATIONSHIPS BETWEEN PROTEIN DISTRIBUTIONS FROM MICROSCOPY IMAGES", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Soheil Kolouri", "Saurav Basu", "Gustavo K. Rohde"], "summary": "Multichannel microscopy has emerged as a technique for imaging multiple targets (molecules, protein distributions, etc.) simultaneously. Discovering the relative changes in these targets (i.e. distribution of different proteins) is fundamental for understanding cell structure and function. We describe a new method for quantifying and visualizing relationships between multiple targets, from a set of segmented multichannel cells. The method is based on combining the canonical correlation analysis technique with a framework for analyzing images based on the concept of optimal mass transportation. We apply the method towards understanding chromatin distribution in cancer nuclei as a function of nuclear envelope shape. We also show that sub cellular distribution of mitochondria can be used to predict the sub cellular localization of actin fibers in yeast cells. Finally, we also describe the application of the method towards understanding relationships between nuclear and cellular shapes in 2D HeLa cells. We believe that the method could serve as a general tool for mining relationships between different sub cellular protein/molecule distributions as well as organelle shapes.", "keywords": ["data", "organelle", "correlation", "vol", "embedding", "analysis", "distance", "set", "method", "described", "protein", "microscopy", "information", "approach", "imaging", "distribution", "relationship", "computed", "work", ".", "\u03bd", "transport", "actin", "given", "image", "function", "cell", "shape"], "document_vector": [113.000244, 38.939357], "paragraphs": [{"paragraph_vector": [-40.825881, 3.978663], "paragraph_keywords": ["proteins", "cells", "work", "information"]}, {"paragraph_vector": [-40.451114, 0.891914], "paragraph_keywords": ["distribution", "actin", "method", "function"]}, {"paragraph_vector": [-45.646366, 2.356868], "paragraph_keywords": ["method", "images", "correlation", "interest"]}, {"paragraph_vector": [9.643724, 11.635845], "paragraph_keywords": ["mass", "embedding", "set", "\u03c9"]}, {"paragraph_vector": [-35.033809, 4.22134], "paragraph_keywords": ["\u03bd", "distance", "given", "described"]}, {"paragraph_vector": [-50.475906, -8.219279], "paragraph_keywords": ["data", "computed", "projection", "compute"]}, {"paragraph_vector": [-44.286472, 4.92914], "paragraph_keywords": ["cells", "cell", "actin", "segmented"]}, {"paragraph_vector": [-44.105682, 1.524683], "paragraph_keywords": ["described", "correlation", "method", "correlated"]}, {"paragraph_vector": [-43.565055, 2.523845], "paragraph_keywords": ["relationships", "approach", "function", "utilizing"]}, {"paragraph_vector": [-40.821468, 0.282496], "paragraph_keywords": ["cell", "vol", "rohde", "analysis"]}, {"paragraph_vector": [-39.874538, 0.495216], "paragraph_keywords": ["microscopy", "images", "image", "john"]}], "content": {}, "doi": "10.1109/PacificVis.2018.00022"}, {"uri": "182", "title": "Bio-edutainment: Learning life science through X gaming", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yiyu Cai", "Baifang Lu", "Zhaowei Fan", "Chandrasekaran Indhumathi", "Kian Teck Lim", "Ching Wern Chan", "Yuan Jiang", "Lin Li"], "summary": "Learning through gaming is one of the natural ways for knowledge and skill acquisition. This paper presents our work on bio-edutainment. Playing X games not only allows low age students to learn bio-molecular structure but also enables better understand complicated structure of bio-molecules. Immersive and interactive games may potentially motivate students to develop their interest to explore the wonder of life science. Based on our core Virtual Reality technology, the bio-edutainment system developed consists of mainly three components of visualization, modeling and interaction. Enabling technology includes also GPU technology, Networking, sensor technology and so on. The system was exhibited in Singapore Science Center. r 2005 Elsevier Ltd. All rights reserved.", "keywords": ["game", "structure", "developed", "vr", "play", "protein", "solution", "information", "edutainment", "stereo", "biology", "enhanced", "gaming", "education", "student", "learning", "technology", "design", "graphic", "use", "science", "x", "player", "visualization", "modeling"], "document_vector": [124.69062, -25.574102], "paragraphs": [{"paragraph_vector": [173.458358, 2.686589], "paragraph_keywords": ["learning", "game", "play", "playing"]}, {"paragraph_vector": [172.246475, 4.526616], "paragraph_keywords": ["structure", "vr", "biology", "information"]}, {"paragraph_vector": [-178.394546, -1.73879], "paragraph_keywords": ["education", "lab", "proteins", "information"]}, {"paragraph_vector": [174.535522, 2.714517], "paragraph_keywords": ["structure", "vr", "modeling", "visualization"]}, {"paragraph_vector": [138.500198, 29.979223], "paragraph_keywords": ["protein", "structure", "modeling", "techniques"]}, {"paragraph_vector": [142.311737, 28.880558], "paragraph_keywords": ["games", "view", "protein", "x"]}, {"paragraph_vector": [165.406021, 7.542502], "paragraph_keywords": ["structure", "protein", "proteins", "gaming"]}, {"paragraph_vector": [174.505065, 4.266672], "paragraph_keywords": ["game", "learning", "students", "vr"]}], "content": {}, "doi": "10.1109/BIBM.2018.8621256"}, {"uri": "183", "title": "Semi-supervised Topological Analysis for Elucidating Hidden Structures in High-Dimensional Transcriptome Datasets", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Tianshu Feng", "Jaime I. Davila", "Yuanhang Liu", "Sangdi Lin", "Shuai Huang"], "summary": "Topological data analysis (TDA) is a powerful method for reducing data dimensionality, mining underlying data relationships, and intuitively representing the data structure. The Mapper algorithm is one such tool that projects highdimensional data to 1-dimensional space by using a filter function that is subsequently used to reconstruct the data topology relationships. However, domain context information and prior knowledge have not been considered in current TDA modeling frameworks. Here, we report the development and evaluation of a semi-supervised topological analysis (STA) framework that incorporates discrete or continuously labeled data points and selects the most relevant filter functions accordingly. We validate the proposed STA framework with simulation data and then apply it to samples from Genotype-Tissue Expression data and ovarian cancer transcriptome datasets. The graphs generated by STA for these 2 datasets, based on gene expression profiles, are consistent with prior knowledge, thereby supporting the effectiveness of the proposed framework.", "keywords": ["university", "data", "dataset", "structure", "shannon", "analysis", "distance", "sample", "graph", "expression", "set", "method", "cluster", "label", "cancer", "applied", "point", "mapper", "framework", "result", "index", "sta", "clustering", "reference", "interval", "filter", "example", "based", ".", "datasets", "measure", "learning", "frontier", "knowledge", "model", "engineering", "\ud835\udc60\ud835\udc5da", "brain", "generated", "function", "\ud835\udc65", "node"], "document_vector": [138.780685, 67.994972], "paragraphs": [{"paragraph_vector": [-92.13372, -16.344348], "paragraph_keywords": ["data", "ordination", "models", "structure"]}, {"paragraph_vector": [-85.46009, -10.166295], "paragraph_keywords": ["data", "department", "ieee", "mapper"]}, {"paragraph_vector": [-89.447486, -3.190247], "paragraph_keywords": ["filter", "data", "functions", "mapper"]}, {"paragraph_vector": [-92.040946, -6.034518], "paragraph_keywords": ["data", "mapper", "clusters", "applied"]}, {"paragraph_vector": [-87.603797, -7.237336], "paragraph_keywords": ["data", "filter", "functions", "set"]}, {"paragraph_vector": [-83.926048, -4.20615], "paragraph_keywords": ["filter", "functions", "data", "function"]}, {"paragraph_vector": [-93.937904, -1.83265], "paragraph_keywords": ["filter", "functions", "knowledge", "function"]}, {"paragraph_vector": [-85.709632, -8.934694], "paragraph_keywords": ["data", "points", "node", "nodes"]}, {"paragraph_vector": [-86.484077, -7.633692], "paragraph_keywords": ["nodes", "graph", "\ud835\udc60\ud835\udc5da", "points"]}, {"paragraph_vector": [-90.326164, -2.450455], "paragraph_keywords": ["data", "graph", "points", "nodes"]}, {"paragraph_vector": [-87.906723, -7.645768], "paragraph_keywords": ["filter", "graphs", "functions", "function"]}, {"paragraph_vector": [-90.566253, 0.686188], "paragraph_keywords": ["frontier", "data", "\ud835\udc60\ud835\udc5da", "pareto"]}, {"paragraph_vector": [-108.505683, -17.148611], "paragraph_keywords": ["data", "points", "\ud835\udc65", "filter"]}, {"paragraph_vector": [-89.464523, -3.978233], "paragraph_keywords": ["points", "data", "intervals", "graphs"]}, {"paragraph_vector": [-91.109024, -2.932694], "paragraph_keywords": ["dataset", "datasets", "measures", "t"]}, {"paragraph_vector": [-92.83728, -1.304874], "paragraph_keywords": ["sta", "t", "isomap", "sne"]}, {"paragraph_vector": [-86.299354, 7.970748], "paragraph_keywords": ["filter", "brain", "function", "distance"]}, {"paragraph_vector": [-84.02713, 9.291367], "paragraph_keywords": ["brain", "samples", "graph", "distance"]}, {"paragraph_vector": [-88.494094, -2.246328], "paragraph_keywords": ["dif", "associated", "samples", "filter"]}, {"paragraph_vector": [-89.465972, -0.204455], "paragraph_keywords": ["samples", "data", "filter", "framework"]}, {"paragraph_vector": [-85.651863, 7.761471], "paragraph_keywords": ["subtypes", "framework", "graph", "labels"]}, {"paragraph_vector": [-86.583389, -7.10343], "paragraph_keywords": ["distance", "filter", "methods", "points"]}, {"paragraph_vector": [-87.517913, 1.095481], "paragraph_keywords": [".", "data", "filter", "vol"]}, {"paragraph_vector": [-71.74414, -8.016914], "paragraph_keywords": [".", "learning", "vol", "analysis"]}, {"paragraph_vector": [-102.099121, -8.875872], "paragraph_keywords": ["university", "engineering", "degree", "research"]}, {"paragraph_vector": [-85.672401, -15.911889], "paragraph_keywords": ["sciences", "engineering", "university", "processing"]}], "content": {}, "doi": "10.1109/TVCG.2007.70578"}, {"uri": "184", "title": "Visualizing the Energetics of the Dissociation of a Metastable Molecule", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["David Guzman", "Reuben Reyes", "Karla Vega", "Kelly Gaither", "Robert Wyatt"], "summary": "Introduction Shown in Figure 1 is the simulation of the energetics of the dissociation of a metastable molecule. The simulation was computed by Dr. Robert Wyatt, a Professor of Chemistry at The University of Texas at Austin. Dr. Wyatt\u2019s simulation uses a model molecule composed of two fragments, A and B. Starting from the molecular state AB, the simulation computes the probability that, under any given conditions, AB will dissociate into the separated components A + B. This may seem like a toy problem, but in real life many molecules demonstrate metastable behavior--a propensity to dissociate--and the work that this simulation is doing may ultimately have an impact on studies of the energetics of larger molecules. An important class of these is biomolecular. This work may be particularly applicable to energy transfer problems in biological systems.", "keywords": ["density", "display", "probability", "method", "particle", "trajectory", "wyatt", "simulated", "time", "behavior", "simulation"], "document_vector": [-50.225749, 13.774809], "paragraphs": [{"paragraph_vector": [90.093406, -57.808532], "paragraph_keywords": ["simulation", "method", "texas", "problem"]}, {"paragraph_vector": [90.765213, -51.831062], "paragraph_keywords": ["particles", "particle", "probability", "simulated"]}, {"paragraph_vector": [96.299827, -6.322766], "paragraph_keywords": ["particle", "behavior", "density", "dynamics"]}], "content": {}, "doi": "10.1109/TVCG.2010.134"}, {"uri": "185", "title": "empty", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "", "keywords": [], "document_vector": [23.248716, 9.37492], "paragraphs": [{"paragraph_vector": [-122.967292, 27.135826], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/CCGrid.2014.92"}, {"uri": "186", "title": "SimpleVisGrid: Grid Services for Visualization of Diverse Biomedical Knowledge and Molecular Systems Data", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "Biomedical data visualization is a great challenge due to the scale, complexity, and diversity of systems, system component interactions and experimental data. Standards for interoperable data are a good start to addressing these problems, but standardization of visualization technologies is an emerging topic. SimpleVisGrid builds on Cancer Biomedical Informatics Grid (caBIG) common infrastructure for cancer research, and clearly specifies and extends three standard data formats for inputs and outputs to grid services: commaseparated values (CSV), Portable Network Graphics (PNG), and Scalable Vector Graphics (SVG). Four prototype visualizations are available: 2D array data quality visualization, correlation heatmaps between high-dimensional data and associated meta-data, feature landscapes, and biochemical or semantic network graphs. The services and data model are prepared for submission for caBIG Silver-level compatibility review and for integration into automated research workflows. Making these tools available to caBIG developers and ultimately to biomedical researchers can (1) help with biomedical communication, discovery, and decisionmaking, (2) encourage more research on standardization of visualization formats, and (3) improve the efficiency of large data transfers across the grid.", "keywords": ["data", "service"], "document_vector": [137.889068, 66.16819], "paragraphs": [{"paragraph_vector": [-179.640426, -39.387825], "paragraph_keywords": ["data", "services", "providing", "package"]}], "content": {}, "doi": "10.1109/IV.2010.74"}, {"uri": "187", "title": "Direct visualization of molecularly thin lubricant films on magnetic disks with a digitally enhanced - Magnetic Recording Conference, 2002. Digest of the Asia-Pacific", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kenji Fukuzawa", "Tomohiro Noda", "Yasunaga Mitsuya"], "summary": "An ellipsometry-based direct visualization method for molecularly thin lubricant f i lm on magnetic disks is presented. Visualization of a molecularly thin film is useful for the investigation of replenishment and retention characteristics of thin lubricant film. Ellipsometric information of a sample surface is converted into a dark-bright contrast image and then is directly visualized with a CCD camera. By combining of optical and digital image processing techniques, weak contrasts are enhanced and noise intensity is reduced. Thus, our method can provide direct visualization of thin lubricant film on the order of 1 nm.", "keywords": [], "document_vector": [79.215652, -10.752017], "paragraphs": [{"paragraph_vector": [-121.571952, 28.462223], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TVCG.2016.2598825"}, {"uri": "188", "title": "A General Illumination Model for Molecular Visualization", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["P. Hermosilla", "P. V\u00e1zquez", "A. Vinacua", "T. Ropinski"], "summary": "Several visual representations have been developed over the years to visualize molecular structures, and to enable a better understanding of their underlying chemical processes. Today, the most frequently used atom-based representations are the Space-filling, the Solvent Excluded Surface, the Balls-and-Sticks, and the Licorice models. While each of these representations has its individual benefits, when applied to large-scale models spatial arrangements can be difficult to interpret when employing current visualization techniques. In the past it has been shown that global illumination techniques improve the perception of molecular visualizations; unfortunately existing approaches are tailored towards a single visual representation. We propose a general illumination model for molecular visualization that is valid for different representations. With our illumination model, it becomes possible, for the first time, to achieve consistent illumination among all atom-based molecular representations. The proposed model can be further evaluated in real-time, as it employs an analytical solution to simulate diffuse light interactions between objects. To be able to derive such a solution for the rather complicated and diverse visual representations, we propose the use of regression analysis together with adapted parameter sampling strategies as well as shape parametrization guided sampling, which are applied to the geometric building blocks of the targeted visual representations. We will discuss the proposed sampling strategies, the derived illumination model, and demonstrate its capabilities when visualizing several dynamic molecules. CCS Concepts \u2022Human-centered computing \u2192 Scientific visualization; \u2022Computing methodologies \u2192 Computer graphics; Rendering;", "keywords": ["p", "data", "radiance", "illumination", "value", "molecule", "order", "parameter", "analysis", "representation", "et", "sample", "paper", "space", "diffuse", "obtain", "equation", "algorithm", "regression", "point", "number", "section", "occlusion", "object", "proposed", "sphere", "technique", "surface", "based", "scene", "sampling", "configuration", "model", "b", "time", "al", "function", "visualization"], "document_vector": [-164.216567, -32.27296], "paragraphs": [{"paragraph_vector": [72.087913, -20.63458], "paragraph_keywords": ["perception", "illumination", "shape", "objects"]}, {"paragraph_vector": [69.651893, 38.847206], "paragraph_keywords": ["representations", "regression", "illumination", "space"]}, {"paragraph_vector": [76.220474, 49.646125], "paragraph_keywords": ["illumination", "strategies", "sampling", "model"]}, {"paragraph_vector": [93.608238, 3.091639], "paragraph_keywords": ["space", "section", "representations", "filling"]}, {"paragraph_vector": [57.778614, 44.554531], "paragraph_keywords": ["technique", "render", "al", "techniques"]}, {"paragraph_vector": [59.353748, 48.628784], "paragraph_keywords": ["al", "et", "shape", "approach"]}, {"paragraph_vector": [43.944519, 53.774761], "paragraph_keywords": ["illumination", "radiance", "occlusion", "technique"]}, {"paragraph_vector": [71.002555, 75.97171], "paragraph_keywords": ["object", "radiance", "\u03c9i", "p"]}, {"paragraph_vector": [75.078048, 71.55899], "paragraph_keywords": ["objects", "radiance", "number", "object"]}, {"paragraph_vector": [76.970466, 63.780082], "paragraph_keywords": ["parameters", "parameter", "number", "samples"]}, {"paragraph_vector": [81.438285, 53.307056], "paragraph_keywords": ["configurations", "configuration", "scenes", "parameters"]}, {"paragraph_vector": [74.21305, 63.276519], "paragraph_keywords": ["function", "parametrization", "object", "reflectance"]}, {"paragraph_vector": [83.317131, 67.816726], "paragraph_keywords": ["object", "surface", "equations", "algorithm"]}, {"paragraph_vector": [71.622604, 67.349433], "paragraph_keywords": ["cylinders", "radius", "cylinder", "sphere"]}, {"paragraph_vector": [78.110961, 44.734451], "paragraph_keywords": ["configurations", "simulations", "parameters", "case"]}, {"paragraph_vector": [72.122711, 72.657516], "paragraph_keywords": ["cylinder", "point", "b", "parameters"]}, {"paragraph_vector": [82.28337, 55.572921], "paragraph_keywords": ["algorithm", "obtained", "object", "regression"]}, {"paragraph_vector": [49.214298, 63.696018], "paragraph_keywords": ["occlusion", "object", "equation", "radiance"]}, {"paragraph_vector": [66.341972, 70.225776], "paragraph_keywords": ["model", "illumination", "models", "object"]}, {"paragraph_vector": [65.268249, 38.871971], "paragraph_keywords": ["objects", "interaction", "time", "model"]}, {"paragraph_vector": [80.37268, 56.90485], "paragraph_keywords": ["objects", "point", "diffuse", "cell"]}, {"paragraph_vector": [46.389522, 67.919876], "paragraph_keywords": ["objects", "paper", "illumination", "molecules"]}, {"paragraph_vector": [76.354133, 42.671649], "paragraph_keywords": ["models", "sizes", "cell", "model"]}, {"paragraph_vector": [68.242462, 68.56491], "paragraph_keywords": ["objects", "apply", "object", "effect"]}, {"paragraph_vector": [68.030242, 59.962699], "paragraph_keywords": ["illumination", "model", "interaction", "diffuse"]}], "content": {}, "doi": "10.1109/VISUAL.2019.8933647"}, {"uri": "189", "title": "Visualizing Analysis of the Rice Protein OsPCBP and Its Homogenous Proteins Based on the Hydrophobicity", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Zhang Qiusheng", "Xu Binglian"], "summary": "After the hydrophobicity sequences of the rice protein OsPCBP and its homogenous proteins were treated, the existing region of difference is found. The results are identical with the data obtained through the sequence alignment. In addition, the continuous wavelet transform with Mexican cap wavelet was carried out on the signal obtained by connecting the dealed hydrophobicity sequences of the homogenous proteins. The similarity of the signal displayed in the graph of the wavelet transform coefficients reflects the similarity of the homogneous proteins. Keywords-homogenous proteins;continuous wavelet transform;hydrophobicity", "keywords": ["protein", "similarity"], "document_vector": [-129.520034, 67.872802], "paragraphs": [{"paragraph_vector": [-67.38948, -39.85062], "paragraph_keywords": ["similarity", "proteins", "wavelet", "signal"]}], "content": {}, "doi": "10.1109/MCG.2013.68"}, {"uri": "190", "title": "JVRMol - A low cost Java-based distributed virtual environment for protein molecule analysis and visualization", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ildeberto A. Rodello", "Jos\u00e9 Remo F. Brega", "Antonio C. Sementille", "F\u00e1tima L. S. Nunes"], "summary": "This work uses Virtual Reality and Distributed Systems concepts to develop a low cost Distributed Virtual Environment to visualize and analyze protein molecules called JVRMol. The system has been implemented using Java programming language, including Java 3D and Java RMI APIs, in order to allow geographically disperse researchers to exchange information in a quick and efficient way, speeding up the remote research and discussion. It is also concerned with running over the Internet using common PCs.", "keywords": ["software", "protein", "pdb", "structure", "based", "time", "jvrmol", "molecule", "information", "tool", "language", "user", "message", "provide", "researcher", "interaction"], "document_vector": [168.909515, -44.829135], "paragraphs": [{"paragraph_vector": [119.3171, 6.38337], "paragraph_keywords": ["software", "proteins", "jvrmol", "work"]}, {"paragraph_vector": [119.678459, 4.093131], "paragraph_keywords": ["protein", "module", "molecule", "visualization"]}, {"paragraph_vector": [119.431915, 7.330957], "paragraph_keywords": ["protein", "information", "pdb", "view"]}, {"paragraph_vector": [119.420364, 7.511418], "paragraph_keywords": ["methods", "messages", "based", "interaction"]}, {"paragraph_vector": [159.068222, 12.111315], "paragraph_keywords": ["system", "collaboration", "tools"]}], "content": {}, "doi": "10.1109/ISM.2018.000-5"}, {"uri": "191", "title": "Quantum Molecular Modeling with Simulated Annealing", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["D. Hohl"], "summary": "A quantum-mechanical method for calculating the geometrical structure of small molecules is presented. A combination of the density functional (DF) method with finitetemperature molecular dynamics (MD) and simulated annealing techniques provides a reliable scheme for sampling energy surfaces, without being trapped in energetically unfavorable local minima. We demonstrate its value and efficiency by computing previously unknown geometries of small clusters composed of phosphorus, sulphur, selenium and oxygen atoms. The implementation combines distributed processing with real-time visualization and allows instantaneous interaction with the simulation. The computationally demanding quantum mechanics problem is solved on a Cray supercomputer, communicating with a Silicon Graphics Incorporated (SGI) workstation in a UNIX environment using SGI\u2019s Distributed Graphics Library (DGL). We show that real-time visualization and interaction enhance scientific productivity greatly. 1. Introduct ion t o t h e Underlying Physics The most stable structures of molecules can be determined, in principle, from the total energy E of the system of electrons and ions. If we determine E for all possible geometries, the ground state structure is simply that with the lowest energy. Unfortunately, this method is impracticable for all but the smallest clusters of atoms and molecules. The calculation of the total energy is difficult, and the number of local minima in the energy surface increases dramatically with increasing cluster size. A combination of density functional (DF) methods,\u2019 and molecular dynamics (MD)\u2019 with \u201csimulated annealing313 addresses both of these problems, and we outline the important features of the approach. Calculations for small molecules show that it can reproduce known structures remarkably well (sulphur and selenium clusters, S,, Se,) and give interesting predictions where experimental information is difficult to obtain (phosphorus clusters, Pa). Molecular dynamics also allows us to follow large changes in molecular structures, as we show for S70. 1.1 Calculation of t h e to t a l energy, E The total energy E of an interacting system of electrons and ions can be found from the expression E = (Ql~ l * ) / (Q l * ) , (1) where @ is the Hamiltonian operator of the system. Calculations of this type can be carried out only for systems with very few electrons, and approximate solutions for IIr are generally based on the variational principle of Rayleigh and Ritz, i.e. if I@) is an approximate wave function, then The simplest form of the many-particle wave function is that of Hartree (1928), who represented @ as a product of singleelectron functions: where the functions #i each satisfy a Schrodinger equation whose potential term is given by the mean field of the other electrons,", "keywords": [], "document_vector": [-33.868282, -10.365162], "paragraphs": [{"paragraph_vector": [-131.438568, 21.171003], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/MCG.2012.98"}, {"uri": "192", "title": "Visualization of Molecular Structures using State-of-the-Art Techniques in WebGL", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Finian Mwalongo", "Michael Krone", "Grzegorz Karch", "Michael Becher", "Guido Reina", "Thomas Ertl"], "summary": "The potential of web browsers as a platform for remote visualization has long been recognized. One of the motivations behind remote visualization has mainly been large data that cannot be easily transferred to the client. Further impeding factors for local visualization at the client side included lack of compute power or capabilities as well as lack of native support for hardware-accelerated APIs in the browser. However, moving the visualization to a remote site is challenging because of high latency and low available bandwidth affecting interactivity and visual quality. Although bandwidth is continuously improving, latency remains an issue. To obtain optimal interactivity, the rendering needs to be performed on the client machine. With the introduction of WebGL, it is now possible to exploit the power of the GPU inside the browser without thirdparty additions. We present a GPU-based ray-casting technique for the visualization of molecular structures that adheres to the restrictions of WebGL. We describe implementation details with respect to these restrictions and present the results and performance. Biomolecules with tens of thousands of atoms can be rendered at interactive rates. This shows the potential of establishing the web as an attractive platform for interactive visualization using GPUaccelerated algorithms. CR Categories: I.3.7 [Computer Graphics]: Three-Dimensional Graphics and Realism\u2014Raytracing I.3.6 [Computer Graphics]: Methodology and Techniques\u2014Graphics data structures and data types; J.3 [Computer Applications]: Life and Medical Sciences\u2014 Biology and Genetics.", "keywords": ["web", "list", "data", "volume", "structure", "et", "application", "server", "hardware", "rendered", "protein", "research", "approach", "number", "object", "ray", "sphere", "implementation", "webgl", "example", "based", ".", "work", "communication", "browser", "shader", "performance", "tool", "client", "atom", "graphic", "gpu", "use", "preprocessing", "grid", "casting", "al", "image", "conference", "rendering", "visualization", "cell", "javascript"], "document_vector": [69.637748, -87.057365], "paragraphs": [{"paragraph_vector": [144.030593, 32.04018], "paragraph_keywords": ["visualization", "web", "browser", "et"]}, {"paragraph_vector": [145.383529, 40.189884], "paragraph_keywords": ["visualization", "rendering", "server", "client"]}, {"paragraph_vector": [109.470291, 49.802154], "paragraph_keywords": ["web", "webgl", "visualization", "graphics"]}, {"paragraph_vector": [109.909576, 55.207763], "paragraph_keywords": ["visualization", "library", "data", "web"]}, {"paragraph_vector": [25.104616, 50.633747], "paragraph_keywords": ["spheres", "based", "gpu", "ray"]}, {"paragraph_vector": [30.501213, 50.582191], "paragraph_keywords": ["data", "pdb", "depth", "rendering"]}, {"paragraph_vector": [25.383571, 50.114486], "paragraph_keywords": ["ray", "grid", "rendering", "sphere"]}, {"paragraph_vector": [40.302803, 47.640617], "paragraph_keywords": ["grid", "atoms", "cell", "list"]}, {"paragraph_vector": [23.556688, 30.200725], "paragraph_keywords": ["atom", "cell", "atoms", "array"]}, {"paragraph_vector": [126.954925, 41.469837], "paragraph_keywords": ["data", "javascript", "thread", "server"]}, {"paragraph_vector": [119.424156, 56.691707], "paragraph_keywords": ["ray", "shader", "space", "grid"]}, {"paragraph_vector": [69.316825, 52.149078], "paragraph_keywords": ["server", "time", "atoms", "ray"]}, {"paragraph_vector": [87.741027, 50.419658], "paragraph_keywords": ["rendering", "webgl", "grid", "cells"]}, {"paragraph_vector": [131.150085, 46.31784], "paragraph_keywords": ["data", "grid", "client", "preprocessing"]}, {"paragraph_vector": [102.907142, 52.687831], "paragraph_keywords": ["jsmol", "ray", "webgl", "server"]}, {"paragraph_vector": [78.777473, 50.161808], "paragraph_keywords": ["rendering", "based", "volume", "web"]}, {"paragraph_vector": [149.659103, 29.697708], "paragraph_keywords": ["visualization", "data", "research", "work"]}, {"paragraph_vector": [124.958076, 47.353488], "paragraph_keywords": ["visualization", "javascript", "graphics", "ray"]}, {"paragraph_vector": [125.860168, 39.010066], "paragraph_keywords": ["visualization", "conference", "based", "rendering"]}, {"paragraph_vector": [133.16159, 48.127967], "paragraph_keywords": ["visualization", "conference", "ieee", "based"]}, {"paragraph_vector": [126.349456, 11.990345], "paragraph_keywords": ["visualization", ".", "society", "computer"]}], "content": {}, "doi": "10.1109/C-CODE.2019.8681008"}, {"uri": "193", "title": "Molecular Rendering with Medieval and Renaissance Color Theory", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Francis T. Marchese", "Suzanne M. Marchese"], "summary": "This paper describes the application of Medieval and Renaissance color theory to the computer graphic rendering of molecular models. In particular, Alberti\u2019s and Cennini\u2019s color theories were employed to render shaded geometric primitives such as cylinders and spheres that are the components of traditional ball-and-stick and space filling molecular models. These results were compared with standard rendering based on the OpenGL API or through ray tracing. It is found that by implementing Alberti\u2019s and Cennini\u2019s color theories as color maps within a simple chemical illustration program it is possible to create molecular imagery comparable to contemporary computer graphics schemes. Keywords-molecular visualization; artist color models; art.", "keywords": ["image", "model"], "document_vector": [-157.047241, -57.789474], "paragraphs": [{"paragraph_vector": [57.95631, -10.620741], "paragraph_keywords": ["images", "models", "painted", "suzanne"]}], "content": {}, "doi": "10.1109/PACIFICVIS.2017.8031603"}, {"uri": "194", "title": "Visualization of Molecular Properties at the Qantum Mechanical Level using Blender", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Thomas Haschka", "Manuel Dauchez", "Eric Henon"], "summary": "Molecular visualization is an active and ever expanding subject of research. Most of the visualization methods are however applied on large biological molecules. The representation of smaller molecules on a quantum mechanical level is to date both cumbersome and rare. We present herein a workflow that allows arbitrary volumetric datasets obtained from quantum mechanical calculations to be visualized. Using the open source Blender software and in house developed scripts we show how a researcher might create high resolution publication ready images from quantum mechanical data, such as electron density, electrostatic potential or spin density. This approach currently beats standard molecular visualization tools such as VMD and PyMOL as it allows to visualize higher resolution volumetric datasets and hence, more complex molecules and molecular properties. Further we boast superior image qualitity due to the availability of high quality volumetric absorbtion, scattering and emission shaders in Blender. Finally we show how a today\u2019s state of the art cloud platforms provide the computational power required to create high quality visual representions. All the scripts presented herein are freely available on github https://github.com/haschka/quantumblendervis", "keywords": ["script", "data", "volume", "allows", "molecule", "workflow", "order", "electron", "community", "create", "computing", "quantum", "volumetric", "gaussian", "system", "density", "resolution", "cube", "atom", "file", "use", "software", "bit", "time", "created", "image", "rendering", "blender", "node"], "document_vector": [-151.598251, -34.877979], "paragraphs": [{"paragraph_vector": [78.842773, -5.313803], "paragraph_keywords": ["volumetric", "properties", "quantum", "visualization"]}, {"paragraph_vector": [131.799163, 42.447837], "paragraph_keywords": ["use", "molecule", "tools", "volumetric"]}, {"paragraph_vector": [77.580627, 43.093498], "paragraph_keywords": ["gaussian", "workflow", "file", "data"]}, {"paragraph_vector": [56.052219, -54.571674], "paragraph_keywords": ["gaussian", "script", "cube", "files"]}, {"paragraph_vector": [75.933509, 44.681285], "paragraph_keywords": ["volumetric", "file", "e", "use"]}, {"paragraph_vector": [75.959564, 52.656604], "paragraph_keywords": ["file", "files", "gaussian", "cube"]}, {"paragraph_vector": [73.762939, 54.93827], "paragraph_keywords": ["volumetric", "rendering", "blender", "material"]}, {"paragraph_vector": [76.33329, 56.794174], "paragraph_keywords": ["texture", "volume", "file", "volumes"]}, {"paragraph_vector": [71.820281, 53.627296], "paragraph_keywords": ["script", "node", "volume", "object"]}, {"paragraph_vector": [78.589653, 54.373004], "paragraph_keywords": ["system", "nodes", "heat", "shader"]}, {"paragraph_vector": [70.9561, 30.152614], "paragraph_keywords": ["image", "density", "electron", "gaussian"]}, {"paragraph_vector": [110.527252, -18.918773], "paragraph_keywords": ["calcium", "computer", "community", "acid"]}, {"paragraph_vector": [97.896644, 53.816986], "paragraph_keywords": ["community", "authors", "presented", "guide"]}], "content": {}, "doi": "10.1109/TST.2012.6297584"}, {"uri": "195", "title": "Protein Chain Pair Simplification under the Discrete Fre\u0301chet Distance", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Tim Wylie", "Binhai Zhu"], "summary": "For protein structure alignment and comparison, a lot of work has been done using RMSD as the distance measure, which has drawbacks under certain circumstances. Thus, the discrete Fr\u00e9chet distance was recently applied to the problem of protein (backbone) structure alignment and comparison with promising results. For this problem, visualization is also important because protein chain backbones can have as many as 500-600 -carbon atoms, which constitute the vertices in the comparison. Even with an excellent alignment, the similarity of two polygonal chains can be difficult to visualize unless the chains are nearly identical. Thus, the chain pair simplification problem (CPS-3F) was proposed in 2008 to simultaneously simplify both chains with respect to each other under the discrete Fr\u00e9chet distance. The complexity of CPS-3F is unknown, so heuristic methods have been developed. Here, we define a variation of CPS-3F, called the constrained CPS-3F problem (CPS-3F\u00fe), and prove that it is polynomially solvable by presenting a dynamic programming solution, which we then prove is a factor-2 approximation for CPS-3F. We then compare the CPS-3F\u00fe solutions with previous empirical results, and further demonstrate some of the benefits of the simplified comparisons. Chain pair simplification based on the Hausdorff distance (CPS-2H) is known to be NP-complete, and here we prove that the constrained version (CPS-2H\u00fe) is also NP-complete. Finally, we discuss future work and implications along with a software library implementation, named the Fr\u00e9chet-based Protein Alignment & Comparison Toolkit (FPACT).", "keywords": ["chain", "alignment", "value", "structure", "k", "distance", "path", "cost", "similarity", "set", "method", "m", "protein", "algorithm", "approximation", "solution", "point", "number", "section", "c", "complexity", "defined", "rectangle", "simplification", "based", "simplify", "clause", "use", "pair", "vertex", "xi", "t", "time", "b", "given", "moving", "problem", "node"], "document_vector": [-94.039344, 49.029632], "paragraphs": [{"paragraph_vector": [-99.573204, 15.203926], "paragraph_keywords": ["protein", "structure", "rmsd", "chains"]}, {"paragraph_vector": [-86.633567, -25.240951], "paragraph_keywords": ["chains", "vertices", "distance", "protein"]}, {"paragraph_vector": [-102.772003, 25.216978], "paragraph_keywords": ["alignment", "protein", "problem", "simplification"]}, {"paragraph_vector": [-102.511863, 31.353279], "paragraph_keywords": ["section", "distance", "approximation", "hausdorff"]}, {"paragraph_vector": [-101.895545, 31.88643], "paragraph_keywords": ["distance", "paths", "b", "t"]}, {"paragraph_vector": [-103.598899, 32.228145], "paragraph_keywords": ["distance", "chains", "problem", "dog"]}, {"paragraph_vector": [-102.879981, 32.146907], "paragraph_keywords": ["chains", "distance", "cost", "moving"]}, {"paragraph_vector": [-101.223312, 24.582965], "paragraph_keywords": ["distance", "problem", "dtw", "pair"]}, {"paragraph_vector": [-105.855857, 38.044658], "paragraph_keywords": ["c", "b", "vertices", "sx\u00f0xi"]}, {"paragraph_vector": [-102.89418, 40.864135], "paragraph_keywords": ["covered", "rectangles", "path", "c"]}, {"paragraph_vector": [-106.530372, 39.613651], "paragraph_keywords": ["zi", "solution", "jzij", "rectangle"]}, {"paragraph_vector": [-103.637046, 21.92703], "paragraph_keywords": ["time", "complexity", "rectangles", "chains"]}, {"paragraph_vector": [-101.470222, 38.768524], "paragraph_keywords": ["solution", "space", "nodes", "rectangles"]}, {"paragraph_vector": [-87.767539, 52.083797], "paragraph_keywords": ["function", "algorithm", "array", "rows"]}, {"paragraph_vector": [-89.49897, 52.287555], "paragraph_keywords": ["path", "rectangles", "rectangle", "pointers"]}, {"paragraph_vector": [-102.10157, 25.994035], "paragraph_keywords": ["chains", "algorithm", "length", "results"]}, {"paragraph_vector": [-87.780014, -34.414035], "paragraph_keywords": ["chains", "distance", "value", "algorithm"]}, {"paragraph_vector": [-101.613227, 28.144426], "paragraph_keywords": ["chains", "moving", "cost", "simplification"]}, {"paragraph_vector": [-85.329917, -36.955493], "paragraph_keywords": ["distance", "libraries", "c", "clause"]}, {"paragraph_vector": [-101.583724, 33.301307], "paragraph_keywords": ["clause", "t", "k", "sequence"]}, {"paragraph_vector": [-102.204742, 33.707839], "paragraph_keywords": ["t", "g", "df", "points"]}, {"paragraph_vector": [-102.989242, 30.651107], "paragraph_keywords": ["points", "t", "clause", "jbj"]}, {"paragraph_vector": [-104.709655, 23.429351], "paragraph_keywords": ["distance", "chains", "alignment", "simplification"]}, {"paragraph_vector": [-153.274932, 14.588265], "paragraph_keywords": ["project"]}], "content": {}, "doi": "10.26599/TST.2019.9010056"}, {"uri": "196", "title": "Realtime automatic selection of good molecular views", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Pere-Pau V\u00e1zquez", "Miquel Feixas", "Mateu Sbert", "Antoni Llobet"], "summary": "The investigation of molecular structures often requires the use of graphics software to display different representations of the molecule of interest. Unfortunately, the commonly available visualization software is generally quite complex and requires a high degree of expertise for the user to obtain the desired images. Often, the selection of interesting views implies a considerable time and effort for nonexperienced users. Characterizing the desired properties the users may need is often impossible. In this paper we present a method to automatically determine certain views of molecules that can be used to study their chemical or physical properties. We have used Information Theory\u2019s Shannon entropy in order to characterize two kinds of views: views which show most of the structure of a molecule and views which show a low amount of information of an arrangement of molecules. The first ones can be used to study the composition of the molecule, that is to study certain chemical properties. The latter easily show how molecules are ordered in space and therefore are suitable to infer physical properties of compounds, such as resistance. Finally, we also present an adaptive, hardware accelerated algorithm that makes use of the features of graphics cards to make this calculation in realtime. Our method has proven to give good results as in most cases the views generated by our application can completely replace human involvement. For highly complex compounds, they can be either enough, or a good starting point. Often, our application also provides several views that could be missed by the users. r 2005 Elsevier Ltd. All rights reserved.", "keywords": ["data", "structure", "exploration", "molecule", "order", "seen", "set", "method", "obtain", "algorithm", "information", "point", "number", "computer", "bond", "object", "fig", "projected", "measure", "based", "scene", "element", "atom", "graphic", "use", "software", "viewpoint", "view", "image", "entropy", "rendering", "visualization"], "document_vector": [-169.901092, -5.287158], "paragraphs": [{"paragraph_vector": [80.518348, -24.253183], "paragraph_keywords": ["data", "visualization", "graphics", "images"]}, {"paragraph_vector": [114.679061, 27.154035], "paragraph_keywords": ["section", "exploration", "structure", "information"]}, {"paragraph_vector": [71.388801, -25.883974], "paragraph_keywords": ["graphics", "visualization", "system", "use"]}, {"paragraph_vector": [89.82698, 8.176077], "paragraph_keywords": ["objects", "computer", "view", "numbers"]}, {"paragraph_vector": [116.835311, 31.431608], "paragraph_keywords": ["information", "entropy", "order", "exploration"]}, {"paragraph_vector": [114.11573, 28.654819], "paragraph_keywords": ["information", "atoms", "molecule", "entropy"]}, {"paragraph_vector": [112.550834, 29.542213], "paragraph_keywords": ["information", "scene", "ai", "log"]}, {"paragraph_vector": [113.575645, 29.063142], "paragraph_keywords": ["entropy", "molecules", "formula", "molecule"]}, {"paragraph_vector": [114.091102, 29.221904], "paragraph_keywords": ["points", "entropy", "molecule", "viewpoint"]}, {"paragraph_vector": [113.845108, 28.426719], "paragraph_keywords": ["entropy", "views", "number", "sphere"]}, {"paragraph_vector": [111.39611, 33.574131], "paragraph_keywords": ["entropy", "value", "values", "estimator"]}, {"paragraph_vector": [113.409324, 30.935518], "paragraph_keywords": ["algorithm", "views", "entropy", "estimated"]}, {"paragraph_vector": [35.41323, 80.430175], "paragraph_keywords": ["color", "image", "count", "pixels"]}, {"paragraph_vector": [30.255441, 83.632293], "paragraph_keywords": ["obtain", "objects", "graphics", "rendering"]}, {"paragraph_vector": [114.373168, 30.554788], "paragraph_keywords": ["number", "view", "graphics", "views"]}, {"paragraph_vector": [108.322639, 28.333719], "paragraph_keywords": ["algorithm", "atoms", "view", "entropy"]}, {"paragraph_vector": [114.692558, 28.443071], "paragraph_keywords": ["views", "entropy", "objects", "select"]}, {"paragraph_vector": [109.865859, 28.676881], "paragraph_keywords": ["entropy", "atoms", "views", "bonds"]}, {"paragraph_vector": [80.52275, -24.323123], "paragraph_keywords": ["software", "views", "visualization", "work"]}], "content": {}, "doi": "10.1109/VR.2007.352478"}, {"uri": "197", "title": "A Flexible Framework for Asynchronous In Situ and In Transit Analytics for Scientific Simulations", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Matthieu Dreher", "Bruno Raffin"], "summary": "High performance computing systems are today composed of tens of thousands of processors and deep memory hierarchies. The next generation of machines will further increase the unbalance between I/O capabilities and processing power. To reduce the pressure on I/Os, the in situ analytics paradigm proposes to process the data as closely as possible to where and when the data are produced. Processing can be embedded in the simulation code, executed asynchronously on helper cores on the same nodes, or performed in transit on staging nodes dedicated to analytics. Today, software environnements as well as usage scenarios still need to be investigated before in situ analytics become a standard practice. In this paper we introduce a framework for designing, deploying and executing in situ scenarios. Based on a component model, the scientist designs analytics workflows by first developing processing components that are next assembled in a dataflow graph through a Python script. At runtime the graph is instantiated according to the execution context, the framework taking care of deploying the application on the target architecture and coordinating the analytics workflows with the simulation execution. Component coordination, zerocopy intra-node communications or inter-nodes data transfers rely on per-node distributed daemons. We evaluate various scenarios performing in situ and in transit analytics on large molecular dynamics systems simulated with Gromacs using up to 2048 cores. We show in particular that analytics processing can be performed on the fraction of resources the simulation does not use well, resulting in a limited impact on the simulation performance (less than 9%). Our more advanced scenario combines in situ and in transit processing to compute a molecular surface based on the Quicksurf algorithm. Keywords-In Situ Analytics and Visualization; IO; Molecular Dynamics;", "keywords": ["output", "flowvr", "data", "scenario", "application", "simulation", "daemon", "situ", "running", "memory", "input", "module", "framework", "step", "port", "distribution", "core", "analytics", "mpi", "code", "process", "python", "position", "performance", "atom", "message", "gromacs", "file", "staging", "transit", "grid", "instance", "pattern", "user", "node"], "document_vector": [6.552059, -32.414615], "paragraphs": [{"paragraph_vector": [-170.931076, 40.146938], "paragraph_keywords": ["simulation", "data", "analytics", "approach"]}, {"paragraph_vector": [-169.000686, 38.95903], "paragraph_keywords": ["data", "situ", "simulation", "process"]}, {"paragraph_vector": [168.158203, 32.179981], "paragraph_keywords": ["module", "message", "application", "input"]}, {"paragraph_vector": [-175.195617, 46.79441], "paragraph_keywords": ["modules", "instance", "application", "module"]}, {"paragraph_vector": [175.336868, 38.588562], "paragraph_keywords": ["module", "modules", "daemon", "patterns"]}, {"paragraph_vector": [166.904174, 33.750366], "paragraph_keywords": ["memory", "message", "plugin", "module"]}, {"paragraph_vector": [173.63861, 36.810634], "paragraph_keywords": ["simulation", "message", "flowvr", "stamps"]}, {"paragraph_vector": [171.800811, 39.287181], "paragraph_keywords": ["data", "port", "file", "module"]}, {"paragraph_vector": [-178.54898, 44.682666], "paragraph_keywords": ["atom", "according", "module", "positions"]}, {"paragraph_vector": [160.892166, 63.473857], "paragraph_keywords": ["distribution", "situ", "nodes", "grid"]}, {"paragraph_vector": [-169.650939, 43.789096], "paragraph_keywords": ["data", "densities", "atoms", "m"]}, {"paragraph_vector": [-169.513488, 43.221729], "paragraph_keywords": ["nodes", "simulation", "data", "situ"]}, {"paragraph_vector": [179.242996, 41.82983], "paragraph_keywords": ["node", "gromacs", "performance", "cores"]}, {"paragraph_vector": [-174.083724, 39.620506], "paragraph_keywords": ["gromacs", "data", "situ", "cores"]}, {"paragraph_vector": [178.976638, 37.469051], "paragraph_keywords": ["nodes", "data", "redistribution", "positions"]}, {"paragraph_vector": [178.070922, 38.627548], "paragraph_keywords": ["module", "data", "application", "simulation"]}, {"paragraph_vector": [173.065017, -35.372531], "paragraph_keywords": ["project", "pierre", "thank", "test"]}], "content": {}, "doi": "10.1109/ACSAT.2014.12"}, {"uri": "198", "title": "VAPLI: Novel Visual Abstraction for Protein-Lipid Interactions", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Naif Alharbi", "Michael Krone", "Matthieu Chavent", "Robert S. Laramee"], "summary": "Molecular visualization of molecular dynamics data commonly uses representative surfaces of varying resolution to depict protein molecules while a variety of geometric shapes, from lines to spheres, are often used to represent atoms and inter-atomic bonds. In general, the aim of molecular visualization is focused on efficiently rendering atoms or molecules themselves, while the interaction space between them is less explored. Furthermore, with naive approaches rendering every molecule, the particles overlap so that significant interaction details are obscured due to clutter. Contrary to common approaches, our work focuses on the interaction between lipids and proteins and the area in which this occurs, the lipid membrane. To do so, we introduce a novel abstract interaction space for Protein-Lipid Interaction (PLI). The cylindrical abstraction simplifies perception and computation of PLI. It does so by using a local projection of PLI onto a cylindrical geometry. This also addresses the challenge of visualizing complex, time-dependent interactions between these molecules. We propose a fast GPU-based implementation that maps lipid-constituents involved in the interaction onto the abstract protein interaction space. The tool provides interactive rendering of PLI for 336,260 particles over almost 2,000 time-steps. The result is a great simplification of this complex Protein-Lipid Interaction leading to both better perception and new insight for computational biologists.", "keywords": ["data", "structure", "membrane", "molecule", "et", "representation", "detail", "interaction", "represent", "space", "dynamic", "protein", "particle", "approach", "lipid", "based", "surface", "axis", "atom", "cylinder", "abstraction", "grid", "time", "t", "al", "user", "visualization"], "document_vector": [-107.384986, -52.327724], "paragraphs": [{"paragraph_vector": [68.447052, -39.246826], "paragraph_keywords": ["membrane", "molecules", "interactions", "systems"]}, {"paragraph_vector": [58.763332, -25.826299], "paragraph_keywords": ["visualization", "section", "abstraction", "biologists"]}, {"paragraph_vector": [69.649749, 25.080305], "paragraph_keywords": ["based", "level", "data", "atoms"]}, {"paragraph_vector": [36.686805, 39.504928], "paragraph_keywords": ["approach", "based", "grid", "et"]}, {"paragraph_vector": [46.509498, -7.170755], "paragraph_keywords": ["protein", "lipid", "particles", "surface"]}, {"paragraph_vector": [47.650249, -18.554485], "paragraph_keywords": ["cylinder", "space", "protein", "translation"]}, {"paragraph_vector": [103.600738, -42.76889], "paragraph_keywords": ["protein", "space", "eigenvectors", "axis"]}, {"paragraph_vector": [116.765792, -65.555931], "paragraph_keywords": ["protein", "t", "particle", "particles"]}, {"paragraph_vector": [41.609954, -6.245899], "paragraph_keywords": ["protein", "grid", "interaction", "given"]}, {"paragraph_vector": [52.359092, -24.823373], "paragraph_keywords": ["visualization", "approach", "center", "lipids"]}, {"paragraph_vector": [54.044689, -35.237392], "paragraph_keywords": ["dynamics", "molecules", "proteins", "features"]}], "content": {}, "doi": "10.1109/MACISE49704.2020.00024"}, {"uri": "199", "title": "Real-Time Ray Tracing of Complex Molecular Scenes", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Lukas Marsalek", "Anna Katharina Dehof", "Iliyan Georgiev", "Hans-Peter Lenhof", "Philipp Slusallek", "Andreas Hildebrandt"], "summary": "Molecular visualization is one of the cornerstones in structural bioinformatics and related fields. Today, rasterization is typically used for the interactive display of molecular scenes, while ray tracing aims at generating high-quality images, taking typically minutes to hours to generate and requiring the usage of an external off-line program. Recently, real-time ray tracing evolved to combine the interactivity of rasterization-based approaches with the superb image quality of ray tracing techniques. We demonstrate how real-time ray tracing integrated into a molecular modelling and visualization tool allows for better understanding of the structural arrangement of biomolecules and natural creation of publication-quality images in real-time. However, unlike most approaches, our technique naturaly integrates into the full-featured molecular modelling and visualization tool BALLView, seamlessly extending a standard workflow with interactive high-quality rendering. Keywords-structural bioinformatics; ray tracing; molecular modeling; rendering; visualization; ray casting", "keywords": ["radiance", "representation", "ballview", "camera", "functionality", "algorithm", "number", "system", "rasterization", "computer", "figure", "integration", "shadow", "ray", "quality", "surface", "based", "scene", "performance", "geometry", "effect", "time", "image", "user", "rendering", "visualization", "tracing"], "document_vector": [-91.106353, -70.13636], "paragraphs": [{"paragraph_vector": [73.256538, -29.960985], "paragraph_keywords": ["visualization", "molecules", "ballview", "force"]}, {"paragraph_vector": [108.267303, 70.521224], "paragraph_keywords": ["radiance", "camera", "point", "direction"]}, {"paragraph_vector": [-9.50273, 77.798606], "paragraph_keywords": ["image", "geometry", "equation", "effects"]}, {"paragraph_vector": [117.126602, 68.316329], "paragraph_keywords": ["ray", "tracing", "object", "time"]}, {"paragraph_vector": [113.28218, 64.493034], "paragraph_keywords": ["image", "ray", "tracing", "shadows"]}, {"paragraph_vector": [99.518928, 62.561779], "paragraph_keywords": ["ray", "representations", "tracing", "number"]}, {"paragraph_vector": [85.361457, 73.780212], "paragraph_keywords": ["ray", "rendering", "tracing", "figure"]}, {"paragraph_vector": [102.070762, 52.034999], "paragraph_keywords": ["visualization", "quality", "tracing", "ray"]}], "content": {}, "doi": "10.1109/TVCG.2011.217"}, {"uri": "200", "title": "Mobile Scanner for Protein Crystallization Plates", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ashok Shrestha", "Truong Tran", "Ramazan Aygun", "Marc L. Pusey"], "summary": "Protein crystallization well plate is a rectangular platform that contains wells usually organized as a grid structure. The crystallization conditions are studied through a screening process by setting up the trial conditions in the well plate. In the past, the expert evaluates the trial wells for the growth of crystals by manually viewing the plate under a microscope or using a high-throughput plate imaging and analysis system. While the first method is tedious and cumbersome, the second method requires financial investment. Recently, a few approaches were developed by collecting images using smartphones thus enabling low-cost automatic scoring (classification) of well images. Nevertheless, these recent methods do not detect which well on the plate is captured. If the user has a smartphone, the user may capture or scan any well by just moving the smartphone to the corresponding well. In this paper, we propose a mobile scanner that identifies the well by using a coded template under the well plate. The mobile scanner provides two modes: image and video. Image mode is used for single well analysis whereas video mode is used to scan the complete plate. In the video mode, the mobile scanner app generates a tilemap of the plate.", "keywords": ["app", "plate", "developed", "captured", "method", "label", "protein", "algorithm", "mode", "template", "system", "tilemap", "well", "code", "frame", "view", "image", "user", "scanner", "video", "lens"], "document_vector": [-154.406967, 27.302686], "paragraphs": [{"paragraph_vector": [-26.482681, 7.215691], "paragraph_keywords": ["images", "system", "lens", "fluorescence"]}, {"paragraph_vector": [-27.186769, -7.198431], "paragraph_keywords": ["cocktail", "lens", "conditions", "scores"]}, {"paragraph_vector": [-19.219726, 2.487379], "paragraph_keywords": ["plate", "user", "size", "lens"]}, {"paragraph_vector": [-18.514598, 21.119947], "paragraph_keywords": ["image", "code", "images", "captured"]}, {"paragraph_vector": [-22.740024, 17.59543], "paragraph_keywords": ["label", "template", "image", "region"]}, {"paragraph_vector": [-18.863716, 29.606212], "paragraph_keywords": ["frame", "image", "tilemap", "code"]}, {"paragraph_vector": [-20.868967, 24.168857], "paragraph_keywords": ["user", "video", "mode", "app"]}, {"paragraph_vector": [-19.936759, 26.922504], "paragraph_keywords": ["image", "tilemap", "video", "app"]}, {"paragraph_vector": [-16.089263, 28.788486], "paragraph_keywords": ["app", "plate", "image", "tilemap"]}], "content": {}, "doi": "10.1109/BIBM47256.2019.8983198"}, {"uri": "201", "title": "Guaranteed Quality Triangulation of Molecular Skin Surfaces", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ho-Lun Cheng", "Xinwei Shi"], "summary": "We present an efficient algorithm to mesh the macromolecules surface model represented by the skin surface defined by Edelsbrunner. Our algorithm overcomes several challenges residing in current surface meshing methods. First, we guarantee the mesh quality with a provable lower bound of 21\u25e6 on its minimum angle. Second, we ensure the triangulation is homeomorphic to the original surface. Third, we improve the efficiency of constructing the Restricted Delaunay Triangulation(RDT) of smooth surfaces. We achieve this by constructing the RDT using the advancing front method without computing the Delaunay tetrahedrization of the sample points on the surfaces. The difficulty of handling the front collision problem is tackled by employing the Morse theory. In particular, we construct the Morse-Smale complex to simplify the topological changes of the front. Our implementation results suggest that the algorithm decrease the time of generating high quality homeomorphic skin mesh from hours to a few minutes. CR Categories: J.3 [Life and Medical Sciences]: Biology and Genetics; I.3.8 [Computer Graphics]: Computational Geometry and Object Modeling\u2014Curve, Surface, Solid, and Object Representation", "keywords": ["p", "triangulation", "saddle", "smale", "mesh", "operation", "algorithm", "triangle", "point", "section", "figure", "skin", "ball", "defined", "sphere", "quality", "change", "morse", "protecting", "delaunay", "surface", "advancing", "complex", "height", "edge", "model", "vertex", "condition", "b", "t", "boundary", "function", "q"], "document_vector": [-116.182708, -31.25386], "paragraphs": [{"paragraph_vector": [26.112346, -3.683462], "paragraph_keywords": ["model", "surface", "ms", "surfaces"]}, {"paragraph_vector": [-3.572364, 19.493253], "paragraph_keywords": ["surface", "algorithm", "mesh", "algorithms"]}, {"paragraph_vector": [-4.2549, 22.58348], "paragraph_keywords": ["advancing", "mesh", "skin", "triangulation"]}, {"paragraph_vector": [-8.227464, 23.973321], "paragraph_keywords": ["surface", "mesh", "complex", "smale"]}, {"paragraph_vector": [-1.946008, 22.667327], "paragraph_keywords": ["skin", "section", "surface", "b"]}, {"paragraph_vector": [2.963492, 19.730358], "paragraph_keywords": ["skin", "surface", "t", "dt"]}, {"paragraph_vector": [-7.208427, 24.662242], "paragraph_keywords": ["surface", "changes", "function", "triangulation"]}, {"paragraph_vector": [-11.341727, 24.158515], "paragraph_keywords": ["p", "points", "m", "level"]}, {"paragraph_vector": [-9.807326, 22.83143], "paragraph_keywords": ["points", "decrease", "protecting", "changes"]}, {"paragraph_vector": [-9.889682, 24.607315], "paragraph_keywords": ["point", "points", "saddle", "complex"]}, {"paragraph_vector": [-9.77651, 24.978713], "paragraph_keywords": ["lines", "points", "point", "contraction"]}, {"paragraph_vector": [-10.385299, 22.385063], "paragraph_keywords": ["triangulation", "complex", "weighted", "surface"]}, {"paragraph_vector": [-12.383384, 23.994667], "paragraph_keywords": ["points", "triangles", "operation", "skin"]}, {"paragraph_vector": [-13.006959, 17.604705], "paragraph_keywords": ["pt", "triangles", "operation", "departure"]}, {"paragraph_vector": [-15.75739, 16.375543], "paragraph_keywords": ["operation", "triangles", "added", "triangle"]}, {"paragraph_vector": [-13.114077, 17.182073], "paragraph_keywords": ["triangles", "bridge", "operation", "protecting"]}, {"paragraph_vector": [-8.877351, 23.097438], "paragraph_keywords": ["edge", "length", "skin", "surface"]}, {"paragraph_vector": [-9.851285, 22.733758], "paragraph_keywords": ["operations", "edge", "triangles", "ra"]}, {"paragraph_vector": [-1.444694, 19.737548], "paragraph_keywords": ["skin", "triangulation", "paper", "surface"]}, {"paragraph_vector": [-1.815891, 26.62647], "paragraph_keywords": ["algorithm", "height", "surfaces", "function"]}], "content": {}, "doi": "10.1109/IBCAST.2019.8667214"}, {"uri": "202", "title": "Use of Gene Ontology semantic information in protein interaction data visualization", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Raimon Massanet", "Pere Caminal", "Alexandre Perera"], "summary": "The Gene Ontology project is an effort to structure knowledge on biological products and processes by adding semantic information to them. This is done in a systematic way so that this additional information can be automatically processed. In this contribution a protein-protein interaction visualization algorithm is proposed, which combines protein interaction data with Gene Ontology semantic information. The information is integrated using a semantic distance measure defined in ontologies or taxonomies. Multidimensional scaling is applied to this measure and the output complements protein interaction data in building an interaction visualization map.", "keywords": [], "document_vector": [44.406803, 56.347682], "paragraphs": [{"paragraph_vector": [-130.380081, 21.649984], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TVCG.2009.106"}, {"uri": "203", "title": "empty", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Weizhong Zhu", "Xia Lin", "Xiaohua Hu", "Bahrad A. Sokhansanj"], "summary": "A new visualization tool, called \"Visual Concept Explorer (VCE)\", was developed to visualize concept relationships in bio-medical literatura VCE integrates Pathfinder Network Scaling and Kohonen Self-organizing Feature Map Algorithm for visual mapping. As a case study, VCE was applied to visualize a chromatin protein-protein interaction (PPI) network The mapping results demonstrated that VCE could explore the semantic structure and latent domain knowledge hidden in protein-protein interaction data sets generatedfrom bio-medical literature.", "keywords": ["expert", "space", "neuron", "weight", "protein", "region", "map", "network", "domain", "pfnet", "path", "vce", "ppi", "gene", "interaction", "node"], "document_vector": [64.74234, 58.827438], "paragraphs": [{"paragraph_vector": [-99.912216, -22.357025], "paragraph_keywords": ["neuron", "t", "protein", "input"]}, {"paragraph_vector": [-99.647842, -20.838836], "paragraph_keywords": ["weight", "path", "network", "proteins"]}, {"paragraph_vector": [-116.628829, -31.728578], "paragraph_keywords": ["nodes", "network", "proteins", "q"]}, {"paragraph_vector": [-101.605781, -19.374906], "paragraph_keywords": ["proteins", "protein", "network", "biologists"]}, {"paragraph_vector": [-102.994125, -19.68424], "paragraph_keywords": ["atm", "proteins", "protein", "gene"]}, {"paragraph_vector": [-98.270545, -20.630727], "paragraph_keywords": ["regions", "map", "reflects", "issue"]}], "content": {}, "doi": "10.1109/TVCG.2017.2745280"}, {"uri": "204", "title": "empty", "timestamp": "1999", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "39 Computing is the third branch of scientific research, bridging the gap between laboratory experiment and analytical theory. Experiments search for patterns in complex natural phenomena, and theories encode the discovered patterns into mathematical equations that provide predictive laws for the behavior of nature. Computer simulations solve these equations numerically in their full complexity, whereas the equations cannot be solved analytically due to their complexity, nonlinearity, or a lack of symmetry. In computer simulations, we can control an environment with any degree of accuracy and access extreme conditions far beyond the scope of laboratory experiments. Currently, experimental work on fracture that involves designing high-fracture-toughness materials tends to focus on controlling structures at diverse length scales\u2014atoms, defects, fibers, interfaces, grains, pores, and cracks. The inherent complexity of such multiscale materials phenomena is evidenced by the rich variety of simulation paradigms ranging from continuum to atomistic. Synergism between experiments, materials simulations, and high-performance computing and communications is needed to solve challenging materials problems associated with dynamic fracture.1 Molecular-dynamics simulations have played a key role in our understanding of fracture.2\u20134 (For the MD method, see the \u201cMolecular-dynamics simulation\u201d sidebar.) In the MD approach, we obtain the system\u2019s phase-space trajectories (positions and velocities of all atoms at all times) from the numerical solution of Newton\u2019s equations (see Figure 1). This lets us study how atomistic processes determine macroscopic materials\u2019 properties. Recent advances in scalable, space-time multiresolution algorithms, coupled with access to massively parallel computing resources, have enabled us to perform some of the largest atomistic (100-million-atom MD) simulations of fracture in complex materials. However, we cannot fully realize the impact SCALABLE MOLECULAR-DYNAMICS, VISUALIZATION, AND DATAMANAGEMENT ALGORITHMS FOR MATERIALS SIMULATIONS", "keywords": [], "document_vector": [30.37549, -39.114589], "paragraphs": [{"paragraph_vector": [-127.756217, 21.637744], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/COMPSAC.2018.00078"}, {"uri": "205", "title": "Computational Methods in Chemistry and Biochemistry Education: Visualization of Proteins", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Polina Pine", "Liudmila Ivanovna Paina"], "summary": "One of the most important skills for a biochemist these days is to be able to visualize a protein, find a ligand binding site and show the three dimensional interactions with the coordinated amino acid residues. It is important to educate a modern student how to benefit from the up-to-date scientific data and software. It is known that the primary structure of protein is of primary information content, however the tertiary structure is more conserved. Biochemistry students are educated in a traditional way giving an importance to a developing fundamental skills as well as critical thinking. In this paper we demonstrate educational approached that incorporates computational methods in the course Biochemistry A CHEM 370. This approach is developed at Loyola University Chicago and based on a pedagogical method developed at Orenburg State Medical University.", "keywords": ["university", "project", "structure", "program", "vmd", "protein", "sequence", "information", "computer", "try", "pine", "student", "skill", "following", "biochemistry", "site", "file", "use", "software", "class", "given", "visualization"], "document_vector": [159.470153, -5.243957], "paragraphs": [{"paragraph_vector": [-172.912979, -4.831509], "paragraph_keywords": ["visualization", "protein", "structure", "biochemistry"]}, {"paragraph_vector": [-171.401367, -6.726786], "paragraph_keywords": ["students", "sequence", "biochemistry", "proteins"]}, {"paragraph_vector": [-175.10884, -5.845151], "paragraph_keywords": ["students", "class", "project", "given"]}, {"paragraph_vector": [138.205108, -16.276096], "paragraph_keywords": ["protein", "try", "file", "program"]}, {"paragraph_vector": [-177.887893, -56.25259], "paragraph_keywords": ["structure", "lig", "use", "protein"]}, {"paragraph_vector": [-172.290481, -6.507611], "paragraph_keywords": ["students", "skills", "visualization", "protein"]}, {"paragraph_vector": [43.506393, -11.066856], "paragraph_keywords": ["volume", "pine", "polina", "olson"]}], "content": {}, "doi": "10.1109/IV.2014.38"}, {"uri": "206", "title": "Ensemble-Space Visualization Improves Perception of 3D State of Molecular Dynamics Simulation", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Israel Huff", "Chris Weigle", "David C. Banks"], "summary": "Large-scale molecular dynamics (MD) simulations are some of the primary applications running on today\u2019s supercomputers. These simulations frequently compute the interactions of millions of atoms over millions of time steps, and petascale simulations will target billion atom simulations in the near future. Visualizing the state of such large-scale simulations poses a significant challenge for both data reduction and perception. Standard tools for 3D visualization of MD simulations \u2013 such as direct visualization of the molecules in their simulated physical arrangement \u2013 eventually grow so dense (with more atoms than screen pixels) that only the volumetric shape of the simulation domain is apparent, rather than its detailed content. In this paper, we describe a method for visualizing simulations of a compound in bulk which we refer to as the ensemble display. An ensemble display is produced by superimposing molecular neighborhoods onto fixed reference molecules. This results in a 3D visualization which preserves inter-molecular distances and angles between pairs of molecules while greatly reducing the visual complexity. We performed human-subjects experiments to test the hypothesis that subjects could deduce a bulk physical property of the simulation (temperature) from the ensemble display more accurately than from other common visualizations of MD simulation. Our results show that temperature estimates under the ensemble display had the least error of the tested visualization techniques.", "keywords": ["data", "structure", "md", "molecule", "k", "estimation", "rdf", "graph", "simulation", "space", "point", "number", "figure", "density", "work", "scene", "ice", "atom", "water", "temperature", "display", "r", "model", "radius", "view", "phase", "error", "participant", "image", "function", "visualization"], "document_vector": [-48.988231, -33.294528], "paragraphs": [{"paragraph_vector": [95.146507, -44.626594], "paragraph_keywords": ["scene", "molecule", "density", "set"]}, {"paragraph_vector": [123.081459, 36.435615], "paragraph_keywords": ["display", "pixels", "atoms", "number"]}, {"paragraph_vector": [92.320114, -30.344181], "paragraph_keywords": ["water", "atoms", "simulation", "convolution"]}, {"paragraph_vector": [114.328109, -40.665943], "paragraph_keywords": ["display", "image", "molecules", "model"]}, {"paragraph_vector": [68.066596, -26.153978], "paragraph_keywords": ["molecule", "work", "radius", "molecules"]}, {"paragraph_vector": [77.570724, -63.579555], "paragraph_keywords": ["phase", "display", "atom", "symmetries"]}, {"paragraph_vector": [113.614921, -45.562435], "paragraph_keywords": ["water", "molecules", "disorder", "display"]}, {"paragraph_vector": [97.246978, -44.645847], "paragraph_keywords": ["simulation", "k", "oxygen", "water"]}, {"paragraph_vector": [101.859931, -30.297414], "paragraph_keywords": ["display", "view", "axis", "water"]}, {"paragraph_vector": [131.920639, -42.560539], "paragraph_keywords": ["colorbar", "stimuli", "temperature", "participants"]}, {"paragraph_vector": [128.047027, -46.710525], "paragraph_keywords": ["temperature", "display", "estimation", "error"]}, {"paragraph_vector": [130.520538, -41.290569], "paragraph_keywords": ["display", "temperature", "view", "chance"]}, {"paragraph_vector": [119.006721, -42.353794], "paragraph_keywords": ["display", "temperature", "simulations", "simulation"]}, {"paragraph_vector": [110.450836, -43.873413], "paragraph_keywords": ["ornl", "display", "simulation", "dynamics"]}, {"paragraph_vector": [44.978782, -5.779232], "paragraph_keywords": ["journal", "chemical", "physics", "dynamics"]}], "content": {}, "doi": "10.1109/TVCG.2015.2467911"}, {"uri": "207", "title": "empty", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": ["G. N. Williams", "E. L. Nelson", "D. M. Bamett", "K. Parmley"], "summary": "Scientific visualization metliodologies are being utilized increasingly in attempts to understand physical phenomena via mathematical and simulation model results. Presented herein are the results of a visualization project which produced a vectorized, high-resolution, raytraced animation of the dynamics of a protein molecule. The resulting animation was recorded on 35-mm film, with a resolution of 1024 x 1024 pixels with 24 color bits. Run-time statistics were also collected which relate image generation parameter ranges and interdependencies.", "keywords": [], "document_vector": [179.355987, -18.53971], "paragraphs": [{"paragraph_vector": [-131.203475, 21.747661], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TCBB.2019.2950657"}, {"uri": "208", "title": "Visualizing Dynamic Molecular Conformations", "timestamp": "2002", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Johannes Schmidt-Ehrenberg", "Daniel Baum", "Hans-Christian Hege"], "summary": "The bioactivity of a molecule strongly depends on its metastable conformational shapes and the transitions between these. Therefore, conformation analysis and visualization is a basic prerequisite for the understanding of biochemical processes. We present techniques for visual analysis of metastable molecular conformations. Core of these are flexibly applicable methods for alignment of molecular geometries, as well as methods for depicting shape and \u2019fuzziness\u2019 of metastable conformations. All analysis tools are provided in an integrated working environment. The described techniques are demonstrated with pharmaceutically active biomolecules. CR Categories: G.3 [Mathematics of Computing]: Probability and Statistics\u2014Distribution functions; I.3.8 [Computing Methodologies]: Computer Graphics\u2014Applications; I.6.6 [Computing Methodologies]: Simulation and Modeling\u2014Simulation Output Analysis", "keywords": ["alignment", "volume", "molecule", "analysis", "drug", "space", "set", "coordinate", "conformation", "described", "trajectory", "compute", "approach", "primitive", "section", "system", "density", "probability", "technique", "computed", "geometry", "atom", "field", "cylinder", "use", "uncertainty", "time", "given", "visualization", "q", "shape"], "document_vector": [-74.394454, 16.495815], "paragraphs": [{"paragraph_vector": [-7.259002, -59.375877], "paragraph_keywords": ["drug", "molecules", "enzyme", "molecule"]}, {"paragraph_vector": [58.000617, -50.00521], "paragraph_keywords": ["conformations", "section", "probability", "density"]}, {"paragraph_vector": [83.40702, -32.156204], "paragraph_keywords": ["uncertainty", "geometries", "volume", "dynamics"]}, {"paragraph_vector": [90.674942, -62.935096], "paragraph_keywords": ["q", "p", "interactions", "quantum"]}, {"paragraph_vector": [86.978485, -58.932167], "paragraph_keywords": ["systems", "trajectory", "time", "phase"]}, {"paragraph_vector": [92.731826, -64.543609], "paragraph_keywords": ["conformation", "molecule", "visualization", "alignment"]}, {"paragraph_vector": [36.74123, -53.727397], "paragraph_keywords": ["geometries", "geometry", "atoms", "distance"]}, {"paragraph_vector": [33.884445, -53.0997], "paragraph_keywords": ["geometries", "geometry", "alignment", "reference"]}, {"paragraph_vector": [98.434127, -63.404647], "paragraph_keywords": ["geometries", "shape", "conformation", "molecule"]}, {"paragraph_vector": [35.800968, 35.667503], "paragraph_keywords": ["primitives", "grid", "density", "geometry"]}, {"paragraph_vector": [36.59029, 34.084209], "paragraph_keywords": ["color", "rasterized", "field", "slice"]}, {"paragraph_vector": [140.912796, -62.045383], "paragraph_keywords": ["visualization", "conformations", "conformation", "time"]}, {"paragraph_vector": [128.447845, -64.092132], "paragraph_keywords": ["conformations", "time", "alignment", "movie"]}, {"paragraph_vector": [174.110916, -85.785858], "paragraph_keywords": ["group", "molecule", "atoms", "providing"]}], "content": {}, "doi": "10.1109/NicoInt.2016.4"}, {"uri": "209", "title": "A toolkit for the analysis and visualization of free volume in materials", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Frank T Willmore"], "summary": "A suite of tools is presented which enable analysis of free volume in terms of accepted standard metrics. The tools are extensible through the use of standard UNIX tools to be useful for output of many standard simulation packages. The tools also include utilities for rapid development of visual output not available in other packages. The tools are also extensible and modifiable for other types of spatial data.", "keywords": ["insertion", "cavity", "data", "volume", "rattle", "particle", "point", "system", "generate", "layer", "figure", "tiff", "simmons", "tool", "atom", "center", "utility", "douglas", "image", "shape", "test"], "document_vector": [-23.662088, -9.797122], "paragraphs": [{"paragraph_vector": [56.308155, -15.497555], "paragraph_keywords": ["dynamics", "data", "volume", "tools"]}, {"paragraph_vector": [27.322057, 21.870872], "paragraph_keywords": ["particle", "point", "volume", "utility"]}, {"paragraph_vector": [79.335639, 23.139854], "paragraph_keywords": ["tiff", "layers", "scaffold", "system"]}, {"paragraph_vector": [77.119812, 21.704904], "paragraph_keywords": ["volume", "center", "generate", "image"]}, {"paragraph_vector": [45.652435, -17.611358], "paragraph_keywords": ["doi", "structure", "douglas", "truskett"]}], "content": {}, "doi": "10.1109/BIBM.2014.6999151"}, {"uri": "210", "title": "Visualization of Fullerene Fragmentation", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kai Sdeo", "Bastian Rieck", "Filip Sadlo"], "summary": "In this paper, we present a novel visualization approach for the analysis of fragmentation of molecules, with a particular focus on fullerenes. Our approach consists of different components at different levels of detail. Whereas one component is geometric but invariant to rotations, two other components are based on the topological structure of the molecules and thus additionally invariant to deformations. By combining these three components, which aim at the analysis of simulation ensembles of such molecules, and complementing them with a space-time representation that enables detailed interactive inspection of individual simulations, we obtain a versatile tool for the analysis of the fragmentation of structured, symmetrical molecules such as fullerenes. We exemplify the utility of our approach using a tightly coupled simulation approach for the dynamics of fullerenes.", "keywords": ["fragment", "molecule", "analysis", "graph", "simulation", "dynamic", "approach", "number", "cycle", "gi", "excitation", "bond", "figure", "step", "process", "technique", "based", "atom", "eccentricity", "fragmentation", "time", "visualization"], "document_vector": [-141.523345, -37.966003], "paragraphs": [{"paragraph_vector": [64.780456, -53.963218], "paragraph_keywords": ["carbon", "molecule", "dynamics", "molecules"]}, {"paragraph_vector": [60.392475, -78.42485], "paragraph_keywords": ["approach", "visualization", "time", "based"]}, {"paragraph_vector": [171.078048, -88.931449], "paragraph_keywords": ["simulation", "gi", "j", "molecule"]}, {"paragraph_vector": [45.030929, -82.976348], "paragraph_keywords": ["number", "gi", "use", "ki"]}, {"paragraph_vector": [38.352737, -89.881385], "paragraph_keywords": ["cycle", "cycles", "time", "number"]}, {"paragraph_vector": [140.286056, -88.74205], "paragraph_keywords": ["time", "indicate", "fragment", "simulation"]}, {"paragraph_vector": [117.330703, -67.51152], "paragraph_keywords": ["figure", "visualization", "fragments", "clutter"]}, {"paragraph_vector": [118.25151, -86.922805], "paragraph_keywords": ["eccentricity", "charge", "eccentricities", "fixed"]}, {"paragraph_vector": [29.958047, -89.129425], "paragraph_keywords": ["simulations", "fragmentation", "holes", "number"]}, {"paragraph_vector": [-26.479635, -89.073036], "paragraph_keywords": ["fragmentation", "figure", "simulations", "approach"]}, {"paragraph_vector": [-19.501358, -88.544189], "paragraph_keywords": ["fragmentation", "cycles", "strategies", "data"]}], "content": {}, "doi": "10.1109/TCBB.2008.36"}, {"uri": "211", "title": "Early Experiences Scaling VMD Molecular Visualization and Analysis Jobs on Blue Waters", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["John E. Stone", "Barry Isralewitz", "Klaus Schulten"], "summary": "Petascale molecular dynamics simulations provide a powerful tool for probing the dynamics of cellular processes at atomic and nanosecond resolution not achievable by experimental methods alone. Extraction of details about the dynamics of biomolecules from terabytes of simulation output requires powerful user-extensible molecular analysis and visualization tools. Petascale simulation trajectories are so large that it is now necessary to perform many analysis and visualization tasks formerly handled by off-site computational facilities in-place on the supercomputer itself. We report ongoing progress on porting, tuning, and scaling up the popular molecular visualization and analysis program VMD on the NSF Blue Waters petascale supercomputer. We describe key achievements including algorithmic and memory efficiency improvements, hand-vectorization of key CPU algorithms, new and improved GPU analysis and visualization algorithms, and parallel I/O performance results. We evaluate the performance of VMD for user-developed analysis scripts with the TIMELINE trajectory analysis tool in VMD. Finally, we describe the unique capabilities provided by the Cray XK7 GPU-accelerated compute partition of Blue Waters.", "keywords": ["visualization", "script", "runtime", "data", "structure", "job", "analysis", "scripting", "simulation", "movie", "operation", "vmd", "memory", "trajectory", "calculation", "task", "run", "system", "mpi", "code", "work", "performance", "frame", "geometry", "water", "cpu", "size", "file", "gpu", "use", "timeline", "load", "time", "sasa", "tachyon", "user", "rendering", "o", "node", "count"], "document_vector": [-9.283573, -31.691484], "paragraphs": [{"paragraph_vector": [-174.26802, 52.248596], "paragraph_keywords": ["mpi", "vmd", "analysis", "node"]}, {"paragraph_vector": [-144.970367, 56.137546], "paragraph_keywords": ["vmd", "mpi", "node", "memory"]}, {"paragraph_vector": [-163.952987, 62.68703], "paragraph_keywords": ["gpu", "work", "vmd", "cpus"]}, {"paragraph_vector": [-173.679763, 66.772544], "paragraph_keywords": ["performance", "work", "trajectory", "vmd"]}, {"paragraph_vector": [-152.159088, 49.220962], "paragraph_keywords": ["node", "file", "lustre", "waters"]}, {"paragraph_vector": [-174.272018, 47.768341], "paragraph_keywords": ["vmd", "trajectory", "performance", "nodes"]}, {"paragraph_vector": [-163.201034, 66.196784], "paragraph_keywords": ["gpu", "performance", "analysis", "nodes"]}, {"paragraph_vector": [-158.974792, 57.224582], "paragraph_keywords": ["analysis", "sasa", "timeline", "vmd"]}, {"paragraph_vector": [172.102371, 42.671337], "paragraph_keywords": ["trajectory", "timeline", "analysis", "time"]}, {"paragraph_vector": [-144.635955, 55.462966], "paragraph_keywords": ["node", "calculations", "trajectory", "nodes"]}, {"paragraph_vector": [-147.587158, 55.507171], "paragraph_keywords": ["node", "sasa", "nodes", "run"]}, {"paragraph_vector": [175.250381, 52.101173], "paragraph_keywords": ["memory", "rendering", "vmd", "tachyon"]}, {"paragraph_vector": [159.47673, 69.937637], "paragraph_keywords": ["rendering", "movie", "loading", "vmd"]}, {"paragraph_vector": [157.202316, 68.35041], "paragraph_keywords": ["movie", "runtime", "geometry", "time"]}, {"paragraph_vector": [166.108581, 67.26548], "paragraph_keywords": ["rendering", "vmd", "nodes", "movie"]}, {"paragraph_vector": [-164.002029, 63.996734], "paragraph_keywords": ["providing", "university", "center", "illinois"]}], "content": {}, "doi": "10.1109/BIBE.2017.00-29"}, {"uri": "212", "title": "Continuous Levels-of-Detail and Visual Abstraction for Seamless Molecular Visualization", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Julius Parulek", "Daniel J\u00f6nsson", "Timo Ropinski", "Stefan Bruckner", "Anders Ynnerman", "Ivan Viola"], "summary": "Molecular visualization is often challenged with rendering of large molecular structures in real time. We introduce a novel approach that enables us to show even large protein complexes. Our method is based on the level-of-detail concept, where we exploit three different abstractions combined in one visualization. Firstly, molecular surface abstraction exploits three different surfaces, solvent-excluded surface (SES), Gaussian kernels and van der Waals spheres, combined as one surface by linear interpolation. Secondly, we introduce three shading abstraction levels and a method for creating seamless transitions between these representations. The SES representation with full shading and added contours stands in focus while on the other side a sphere representation of a cluster of atoms with constant shading and without contours provide the context. Thirdly, we propose a hierarchical abstraction based on a set of clusters formed on molecular atoms. All three abstraction models are driven by one importance function classifying the scene into the near-, midand far-field. Moreover, we introduce a methodology to render the entire molecule directly using the A-buffer technique, which further improves the performance. The rendering performance is evaluated on series of molecules of varying atom counts.", "keywords": ["data", "transition", "evaluate", "molecule", "et", "representation", "detail", "space", "employ", "cluster", "method", "presented", "algorithm", "shading", "information", "approach", "clustering", "c", "figure", "depth", "ray", "sphere", "example", "surface", "based", "work", "level", "performance", "atom", "scheme", "s", "field", "gpu", "model", "abstraction", "use", "represents", "time", "function", "rendering", "visualization"], "document_vector": [-137.200195, -67.983757], "paragraphs": [{"paragraph_vector": [65.167419, -6.822053], "paragraph_keywords": ["atoms", "atom", "solvent", "example"]}, {"paragraph_vector": [86.276878, -5.237448], "paragraph_keywords": ["representation", "object", "viewer", "visualization"]}, {"paragraph_vector": [57.973697, 18.620649], "paragraph_keywords": ["detail", "abstraction", "context", "level"]}, {"paragraph_vector": [71.505439, 16.999187], "paragraph_keywords": ["representations", "rendering", "work", "molecule"]}, {"paragraph_vector": [67.582183, 36.338706], "paragraph_keywords": ["approach", "occlusion", "atoms", "based"]}, {"paragraph_vector": [75.691406, 40.926193], "paragraph_keywords": ["data", "representation", "rendering", "detail"]}, {"paragraph_vector": [63.889644, 18.30083], "paragraph_keywords": ["abstraction", "representation", "level", "visualization"]}, {"paragraph_vector": [64.287277, 31.055593], "paragraph_keywords": ["shading", "rendering", "levels", "abstraction"]}, {"paragraph_vector": [41.391384, 22.581054], "paragraph_keywords": ["function", "transition", "evaluate", "interpolation"]}, {"paragraph_vector": [47.065437, 37.055988], "paragraph_keywords": ["representation", "represents", "atom", "atoms"]}, {"paragraph_vector": [23.616569, 55.289043], "paragraph_keywords": ["depth", "spheres", "rendering", "cues"]}, {"paragraph_vector": [39.785655, 47.511909], "paragraph_keywords": ["model", "cm", "shading", "molecule"]}, {"paragraph_vector": [77.946136, 16.384988], "paragraph_keywords": ["data", "abstraction", "surface", "representations"]}, {"paragraph_vector": [26.821256, -39.964511], "paragraph_keywords": ["clustering", "cluster", "surface", "centre"]}, {"paragraph_vector": [26.787811, -44.11185], "paragraph_keywords": ["cluster", "ap", "algorithm", "clusters"]}, {"paragraph_vector": [26.720109, 47.713836], "paragraph_keywords": ["cluster", "list", "display", "increased"]}, {"paragraph_vector": [23.86009, 51.865135], "paragraph_keywords": ["ray", "performance", "sphere", "fields"]}, {"paragraph_vector": [43.346138, 26.912477], "paragraph_keywords": ["fields", "atoms", "performance", "molecules"]}, {"paragraph_vector": [54.383697, 33.292549], "paragraph_keywords": ["molecule", "compared", "example", "representation"]}, {"paragraph_vector": [63.179153, 30.043794], "paragraph_keywords": ["based", "employ", "levels", "molecules"]}, {"paragraph_vector": [62.536605, 10.475326], "paragraph_keywords": ["visualization", "surface", "shading", "rendering"]}, {"paragraph_vector": [54.694179, 21.957281], "paragraph_keywords": ["figure", "silhouette", "cues", "forum"]}], "content": {}, "doi": "10.1109/TVCG.2012.25"}, {"uri": "213", "title": "empty", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "", "keywords": [], "document_vector": [20.596986, 11.959897], "paragraphs": [{"paragraph_vector": [-129.340164, 21.69046], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TCBB.2007.70251"}, {"uri": "214", "title": "Representations that promote cognition: the case of chemistry visualization tools", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Mario Valle"], "summary": "Chemistry deals with structures too small for the naked eye; thus chemists rely heavily on graphical representations. An external observer could presume that the available chemistry visualization tools would be at the forefront regarding representation techniques and interaction methods. Our survey of 20 widely available non-commercial molecular visualization tools shows instead a high uniformity of representation methods and a general imbalance toward visualization for presentation only. To study alternatives to the current situation, we created \u201cviz laboratories\u201d: small, focused applications that highlight a specific chemistry visualization technique to collect practitioners feedback and to encourage them to think about new visualization paradigms. CR Categories: J.2 [Physical Sciences and Engineering]: Chemistry\u2014Visualization H.1.2 [User/Machine Systems]: Human information processing\u2014Perception I.3.6 [Methodology and Techniques]: Graphics data structures and data types\u2014Visual Metaphors", "keywords": ["perception", "visualization", "discovery", "chemistry", "representation", "understanding"], "document_vector": [179.520889, 0.866218], "paragraphs": [{"paragraph_vector": [137.143356, -31.516159], "paragraph_keywords": ["visualization", "chemistry", "representations", "perception"]}, {"paragraph_vector": [-161.851318, 15.498655], "paragraph_keywords": ["helps", "prototype", "integrate", "novel"]}], "content": {}, "doi": "10.1109/TCBB.2013.17"}, {"uri": "215", "title": "Visualization of Spatially Distributed Bioactive Molecules Using Enzyme-Linked Photo Assay", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["HongYao Ongl", "Kazunori Watanabe", "Sachiko Yoshidal", "Naohiro Hozumil"], "summary": "In this presentation, we propose a new simple device for visualizing bioactive molecules with a fine spatial resolution by using a membrane in which a specific enzyme is immobilized. The layer produces fluorescence after association with a specific substance. The layer, on which a biological tissue is to be mounted, is deposited on a quartz substrate that is used as a light guide to introduce UV light to the layer. Substance release is observed by a CCD camera from the opposite side of the substrate. In order to shorten the experiment time, we had automated the optical device. The paper also describes the reduction of background fluorescence by means of image processing technique. Images were acquired by employing two UV-LEDs with slightly different angle. Image processing was performed to separate background and target fluoresences by means of independent component analysis. Finally the release of GABA(y-aminobutyric acid) from specific layers in rat cerebellum was successfully observed). It is expected that, using this method, both real-time transmitter release and its response to medicine can be observed.", "keywords": [], "document_vector": [114.470024, 8.293564], "paragraphs": [{"paragraph_vector": [-130.488967, 21.893035], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/BIBM.2015.7359765"}, {"uri": "216", "title": "Real-Time Molecular Visualization Supporting Diffuse Interreflections and Ambient Occlusion", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Robin Sk\u00e5nberg", "Pere-Pau V\u00e1zquez", "Victor Guallar"], "summary": "Today molecular simulations produce complex data sets capturing the interactions of molecules in detail. Due to the complexity of this time-varying data, advanced visualization techniques are required to support its visual analysis. Current molecular visualization techniques utilize ambient occlusion as a global illumination approximation to improve spatial comprehension. Besides these shadow-like effects, interreflections are also known to improve the spatial comprehension of complex geometric structures. Unfortunately, the inherent computational complexity of interreflections would forbid interactive exploration, which is mandatory in many scenarios dealing with static and time-varying data. In this paper, we introduce a novel analytic approach for capturing interreflections of molecular structures in real-time. By exploiting the knowledge of the underlying space filling representations, we are able to reduce the required parameters and can thus apply symbolic regression to obtain an analytic expression for interreflections. We show how to obtain the data required for the symbolic regression analysis, and how to exploit our analytic solution to enhance interactive molecular visualizations.", "keywords": ["data", "radiance", "structure", "illumination", "molecule", "d", "parameter", "interreflection", "path", "distance", "ao", "paper", "expression", "interaction", "account", "simulation", "space", "described", "equation", "compute", "regression", "ligand", "approach", "exploit", "section", "result", "occlusion", "figure", "complexity", "sphere", "technique", "based", "surface", "scene", "atom", "s", "effect", "receptor", "use", "r", "time", "vdw", "rendering", "visualization", "cell", "force"], "document_vector": [-164.247482, -57.825538], "paragraphs": [{"paragraph_vector": [60.697841, -31.250635], "paragraph_keywords": ["communicate", "interaction", "structures", "ligand"]}, {"paragraph_vector": [63.743602, 21.716918], "paragraph_keywords": ["interreflections", "structures", "illumination", "interactions"]}, {"paragraph_vector": [69.488807, 12.149585], "paragraph_keywords": ["visualization", "section", "interreflections", "structures"]}, {"paragraph_vector": [72.107192, 38.167888], "paragraph_keywords": ["visualization", "depth", "molecules", "method"]}, {"paragraph_vector": [72.243034, -59.154571], "paragraph_keywords": ["atoms", "surface", "color", "molecule"]}, {"paragraph_vector": [50.297821, 67.795158], "paragraph_keywords": ["scale", "equation", "irradiance", "scene"]}, {"paragraph_vector": [47.544239, 67.039916], "paragraph_keywords": ["observation", "atoms", "light", "interreflections"]}, {"paragraph_vector": [57.296367, 51.704586], "paragraph_keywords": ["parameters", "find", "path", "scene"]}, {"paragraph_vector": [47.671554, 67.695465], "paragraph_keywords": ["radiance", "scene", "equation", "direction"]}, {"paragraph_vector": [54.600814, 33.998462], "paragraph_keywords": ["simplify", "set", "simplified", "atoms"]}, {"paragraph_vector": [44.485759, 66.415679], "paragraph_keywords": ["radiance", "parameter", "atom", "path"]}, {"paragraph_vector": [35.190361, 68.194969], "paragraph_keywords": ["radiance", "figure", "distance", "interreflections"]}, {"paragraph_vector": [59.047206, 68.113212], "paragraph_keywords": ["parameter", "range", "radii", "atoms"]}, {"paragraph_vector": [76.928581, 62.98711], "paragraph_keywords": ["regression", "data", "generate", "time"]}, {"paragraph_vector": [73.703941, 42.080375], "paragraph_keywords": ["complexity", "expressions", "derivatives", "algorithm"]}, {"paragraph_vector": [60.384506, 65.863609], "paragraph_keywords": ["time", "atom", "interreflections", "results"]}, {"paragraph_vector": [13.545463, 26.784235], "paragraph_keywords": ["atom", "r", "problem", "grid"]}, {"paragraph_vector": [22.218492, 35.060764], "paragraph_keywords": ["cell", "atoms", "size", "atom"]}, {"paragraph_vector": [33.609981, 52.016548], "paragraph_keywords": ["atom", "interreflection", "compute", "evaluate"]}, {"paragraph_vector": [48.064853, 63.383575], "paragraph_keywords": ["atoms", "j", "occlusion", "compute"]}, {"paragraph_vector": [47.310749, 66.696327], "paragraph_keywords": ["sphere", "ao", "angle", "combined"]}, {"paragraph_vector": [55.102012, 36.149936], "paragraph_keywords": ["ses", "ao", "fragment", "interreflections"]}, {"paragraph_vector": [56.624942, -14.489952], "paragraph_keywords": ["ligand", "receptor", "figure", "atoms"]}, {"paragraph_vector": [57.757419, -7.289669], "paragraph_keywords": ["ligand", "receptor", "figure", "atoms"]}, {"paragraph_vector": [48.236804, -36.501613], "paragraph_keywords": ["atoms", "forces", "interaction", "ligand"]}, {"paragraph_vector": [45.84452, -42.633861], "paragraph_keywords": ["ligand", "figure", "simulation", "forces"]}, {"paragraph_vector": [49.11296, -39.804874], "paragraph_keywords": ["forces", "ligand", "interaction", "strength"]}, {"paragraph_vector": [49.394924, -16.201694], "paragraph_keywords": ["frame", "force", "rates", "radius"]}, {"paragraph_vector": [59.943035, 45.125999], "paragraph_keywords": ["atoms", "interreflections", "section", "use"]}, {"paragraph_vector": [62.494167, 9.815475], "paragraph_keywords": ["visualization", "atoms", "like", "approach"]}, {"paragraph_vector": [10.079419, -53.091487], "paragraph_keywords": ["ieee", "pele", "computed", "monte"]}], "content": {}, "doi": "10.1109/DASC-PICom-DataCom-CyberSciTec.2017.178"}, {"uri": "217", "title": "Visual Analysis of protein-ligand interactions", "timestamp": "2018", "rating": "0.0", "annotation": "", "tags": [], "authors": ["P. V\u00e1zquez", "P. Hermosilla", "V. Guallar", "J. Estrada", "A. Vinacua"], "summary": "The analysis of protein-ligand interactions is complex because of the many factors at play. Most current methods for visual analysis provide this information in the form of simple 2D plots, which, besides being quite space hungry, often encode a low number of different properties. In this paper we present a system for compact 2D visualization of molecular simulations. It purposely omits most spatial information and presents physical information associated to single molecular components and their pairwise interactions through a set of 2D InfoVis tools with coordinated views, suitable interaction, and focus+context techniques to analyze large amounts of data. The system provides a wide range of motifs for elements such as protein secondary structures or hydrogen bond networks, and a set of tools for their interactive inspection, both for a single simulation and for comparing two different simulations. As a result, the analysis of protein-ligand interactions of Molecular Simulation trajectories is greatly facilitated. CCS Concepts \u2022Human-centered computing \u2192 Visualization techniques; Visualization systems and tools;", "keywords": ["data", "h", "structure", "molecule", "analysis", "chart", "application", "hydrogen", "interaction", "simulation", "set", "m", "protein", "information", "ligand", "system", "bond", "figure", "step", "color", "case", "backbone", "example", "binding", "tool", "atom", "energy", "design", "expert", "use", "software", "view", "feature", "user", "visualization", "provide", "analyze", "residue"], "document_vector": [-135.988784, 7.554516], "paragraphs": [{"paragraph_vector": [48.965217, -48.323574], "paragraph_keywords": ["ms", "analysis", "techniques", "information"]}, {"paragraph_vector": [43.388576, -68.066955], "paragraph_keywords": ["ligand", "protein", "information", "simulation"]}, {"paragraph_vector": [75.027565, -23.616985], "paragraph_keywords": ["visualization", "atoms", "section", "interactions"]}, {"paragraph_vector": [62.28952, -36.65715], "paragraph_keywords": ["visualize", "interactions", "molecule", "energy"]}, {"paragraph_vector": [56.718215, -39.732158], "paragraph_keywords": ["based", "interacting", "views", "simulation"]}, {"paragraph_vector": [121.229843, -39.451396], "paragraph_keywords": ["provide", "ligand", "step", "use"]}, {"paragraph_vector": [0.829124, -51.127506], "paragraph_keywords": ["energy", "binding", "protein", "interaction"]}, {"paragraph_vector": [8.635212, -44.40203], "paragraph_keywords": ["protein", "energy", "ligand", "binding"]}, {"paragraph_vector": [-22.758893, -32.191509], "paragraph_keywords": ["bonds", "aminoacids", "pattern", "h"]}, {"paragraph_vector": [113.292312, -56.982559], "paragraph_keywords": ["residues", "ligand", "backbone", "structures"]}, {"paragraph_vector": [48.929988, -59.620048], "paragraph_keywords": ["bonds", "information", "simulation", "energy"]}, {"paragraph_vector": [45.948623, -62.126739], "paragraph_keywords": ["energy", "protein", "system", "figure"]}, {"paragraph_vector": [49.381072, -64.470512], "paragraph_keywords": ["view", "ligand", "views", "application"]}, {"paragraph_vector": [59.126785, -63.262115], "paragraph_keywords": ["views", "ligand", "chart", "step"]}, {"paragraph_vector": [62.972171, -67.520683], "paragraph_keywords": ["protein", "proteins", "ligand", "binding"]}, {"paragraph_vector": [92.788223, -67.684013], "paragraph_keywords": ["energy", "simulation", "energies", "step"]}, {"paragraph_vector": [-59.212909, -75.395355], "paragraph_keywords": ["protein", "case", "analysis", "views"]}, {"paragraph_vector": [25.77499, -51.305923], "paragraph_keywords": ["binding", "hand", "bond", "wheel"]}, {"paragraph_vector": [13.678134, -53.394905], "paragraph_keywords": ["interaction", "energy", "experts", "binding"]}, {"paragraph_vector": [81.310173, -58.906185], "paragraph_keywords": ["software", "application", "simulation", "vmd"]}, {"paragraph_vector": [61.110351, -67.760398], "paragraph_keywords": ["simulation", "information", "trajectory", "residues"]}, {"paragraph_vector": [53.76115, -56.610656], "paragraph_keywords": ["ligand", "information", "grants", "inspection"]}], "content": {}, "doi": "10.1109/TCBB.2016.2603987"}, {"uri": "218", "title": "Direct Visualization of Molecularly Thin Lubricant Films on Magnetic Disks With a Digitally Enhanced Ellipsometric Microscope", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kenji Fukuzawa", "Tomohiro Noda", "Yasunaga Mitsuya"], "summary": "An ellipsometry-based direct visualization method for molecularly thin lubricant films on magnetic disks is presented. The visualization of molecularly thin film is useful in investigating the replenishment and retention characteristics of thin lubricant films. Ellipsometric information of a sample surface is converted into a dark\u2013bright contrast image that is then directly visualized with a charge coupled device camera. By combining optical and digital image processing techniques, weak contrasts can be enhanced and noise intensity reduced. The authors\u2019 method can provide direct visualization of thin lubricant films in the order of 1 nm.", "keywords": ["disk", "light", "snr", "noise", "processing", "film", "ratio", "image", "ellipsometric", "sample", "intensity", "microscope"], "document_vector": [79.223327, -8.4176], "paragraphs": [{"paragraph_vector": [-59.517589, 45.012439], "paragraph_keywords": ["image", "noise", "sample", "fig"]}, {"paragraph_vector": [-57.597553, 43.807239], "paragraph_keywords": ["light", "point", "film", "sample"]}, {"paragraph_vector": [-57.437343, 43.011329], "paragraph_keywords": ["layer", "noise", "film", "disk"]}, {"paragraph_vector": [-58.704303, 43.03305], "paragraph_keywords": ["signal", "film", "point", "ellipsometric"]}, {"paragraph_vector": [-56.003917, 44.005096], "paragraph_keywords": ["film", "intensity", "obtained", "thickness"]}, {"paragraph_vector": [-57.138435, 45.016307], "paragraph_keywords": ["film", "image", "microscope", "fig"]}, {"paragraph_vector": [-56.081356, 46.192615], "paragraph_keywords": ["film", "ellipsometric", "microscope", "thickness"]}], "content": {}, "doi": "10.1109/ISBI.2010.5490235"}, {"uri": "219", "title": "Visualization of Biomolecular Structures: State of the Art Revisited", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["B. Kozl\u0131\u0301kov\u00e1", "M. Krone", "M. Falk", "N. Lindow", "M. Baaden", "D. Baum", "I. Viola", "J. Parulek", "H.-C. Hege"], "summary": "Structural properties of molecules are of primary concern in many fields. This report provides a comprehensive overview on techniques that have been developed in the fields of molecular graphics and visualization with a focus on applications in structural biology. The field heavily relies on computerized geometric and visual representations of three-dimensional, complex, large and time-varying molecular structures. The report presents a taxonomy that demonstrates which areas of molecular visualization have already been extensively investigated and where the field is currently heading. It discusses visualizations for molecular structures, strategies for efficient display regarding image quality and frame rate, covers different aspects of level of detail and reviews visualizations illustrating the dynamic aspects of molecular simulation data. The survey concludes with an outlook on promising and important research topics to foster further success in the development of tools that help to reveal molecular", "keywords": ["son", "called", "data", "volume", "structure", "provide", "allows", "molecule", "electron", "et", "analysis", "representation", "visualize", "detail", "developed", "simulation", "dynamic", "space", "method", "presented", "described", "protein", "algorithm", "us", "particle", "approach", "dna", "chemical", "scale", "section", "result", "system", "object", "figure", "bond", "depth", "computation", "proposed", "ray", "sphere", "quality", "technique", "surface", "based", ".", "position", "level", "computed", "tool", "performance", "atom", "probe", "s", "graphic", "field", "gpu", "model", "use", "abstraction", "grid", "feature", "patch", "time", "al", "property", "image", "vdw", "user", "rendering", "visualization", "casting", "orbitals"], "document_vector": [-157.986709, -15.77803], "paragraphs": [{"paragraph_vector": [91.206253, -22.096521], "paragraph_keywords": ["electrons", "atoms", "nuclei", "bonds"]}, {"paragraph_vector": [94.04969, -31.481584], "paragraph_keywords": ["molecules", "biomolecules", "complexes", "volume"]}, {"paragraph_vector": [105.128784, -25.356197], "paragraph_keywords": ["atoms", "described", "particles", "forces"]}, {"paragraph_vector": [72.437294, -26.717414], "paragraph_keywords": ["report", "biomolecules", "structures", "visualization"]}, {"paragraph_vector": [-179.417419, -58.260047], "paragraph_keywords": ["dna", "called", "structure", "proteins"]}, {"paragraph_vector": [130.440093, -74.65731], "paragraph_keywords": ["methods", "data", "molecules", "proteins"]}, {"paragraph_vector": [109.93991, -54.112384], "paragraph_keywords": ["models", "particles", "simulation", "atoms"]}, {"paragraph_vector": [119.816612, -33.741725], "paragraph_keywords": ["visualization", "models", "simulations", "data"]}, {"paragraph_vector": [89.362396, -20.960571], "paragraph_keywords": ["section", "visualization", "data", "scale"]}, {"paragraph_vector": [43.364753, 41.436054], "paragraph_keywords": ["ray", "bonds", "models", "casting"]}, {"paragraph_vector": [54.971862, 32.431411], "paragraph_keywords": ["representation", "surface", "spheres", "vdw"]}, {"paragraph_vector": [28.027881, 1.315714], "paragraph_keywords": ["probe", "surface", "sas", "vdw"]}, {"paragraph_vector": [25.380167, -0.069599], "paragraph_keywords": ["patches", "surface", "ses", "probe"]}, {"paragraph_vector": [43.948623, 33.617351], "paragraph_keywords": ["ses", "algorithm", "al", "surface"]}, {"paragraph_vector": [24.812488, 45.07793], "paragraph_keywords": ["ses", "ray", "casting", "surface"]}, {"paragraph_vector": [42.508411, 32.362972], "paragraph_keywords": ["surface", "mss", "presented", "ses"]}, {"paragraph_vector": [28.320608, -2.493697], "paragraph_keywords": ["surface", "model", "kernel", "based"]}, {"paragraph_vector": [28.703454, 43.898422], "paragraph_keywords": ["surface", "models", "atoms", "rendering"]}, {"paragraph_vector": [70.698051, 11.68173], "paragraph_keywords": ["abstractions", "representation", "cartoon", "graphics"]}, {"paragraph_vector": [79.717124, 12.358794], "paragraph_keywords": ["structure", "protein", "helix", "plots"]}, {"paragraph_vector": [56.606662, 20.463333], "paragraph_keywords": ["surface", "based", "spheres", "abstraction"]}, {"paragraph_vector": [49.898818, 28.678043], "paragraph_keywords": ["abstraction", "surface", "sphere", "methods"]}, {"paragraph_vector": [69.647346, 21.485471], "paragraph_keywords": ["surface", "rendering", "lod", "based"]}, {"paragraph_vector": [66.801101, 42.002113], "paragraph_keywords": ["approach", "atoms", "data", "atom"]}, {"paragraph_vector": [70.684455, 9.664603], "paragraph_keywords": ["al", "et", "visualize", "grid"]}, {"paragraph_vector": [79.74102, 27.04318], "paragraph_keywords": ["shading", "models", "depth", "colouring"]}, {"paragraph_vector": [70.190826, 18.521707], "paragraph_keywords": ["method", "ao", "osao", "hatching"]}, {"paragraph_vector": [64.639984, 24.851604], "paragraph_keywords": ["depth", "method", "objects", "ssao"]}, {"paragraph_vector": [99.346534, 54.507011], "paragraph_keywords": ["rendering", "data", "time", "highlight"]}, {"paragraph_vector": [118.057357, -40.895061], "paragraph_keywords": ["representations", "visualizations", "flexibility", "atoms"]}, {"paragraph_vector": [147.201309, -73.521385], "paragraph_keywords": ["visualization", "structures", "methods", "presented"]}, {"paragraph_vector": [70.25933, -20.348142], "paragraph_keywords": ["grid", "densities", "et", "aggregation"]}, {"paragraph_vector": [79.866897, -33.179321], "paragraph_keywords": ["time", "simulation", "residues", "data"]}, {"paragraph_vector": [158.6669, -18.467788], "paragraph_keywords": ["graphics", "simulations", "systems", "aspects"]}, {"paragraph_vector": [100.29087, -6.106365], "paragraph_keywords": ["reactions", "visualization", "particle", "molecules"]}, {"paragraph_vector": [88.85897, -16.18095], "paragraph_keywords": ["systems", "visualization", "tools", "transfer"]}, {"paragraph_vector": [81.423812, -18.468759], "paragraph_keywords": ["tools", "systems", "data", "orbitals"]}, {"paragraph_vector": [113.254081, 13.644257], "paragraph_keywords": ["prototype", "game", "developers", "source"]}, {"paragraph_vector": [94.007766, -0.716357], "paragraph_keywords": ["tools", "based", "surfaces", "rendering"]}, {"paragraph_vector": [76.93843, -16.705495], "paragraph_keywords": ["based", "visualization", "web", "gpus"]}, {"paragraph_vector": [126.533653, 66.60427], "paragraph_keywords": ["ray", "simulation", "visualization", "tracing"]}, {"paragraph_vector": [108.615272, -33.37495], "paragraph_keywords": ["representations", "techniques", "methods", "level"]}, {"paragraph_vector": [-123.044517, 26.890089], "paragraph_keywords": ["structure"]}], "content": {}, "doi": "10.1109/PacificVis.2019.00032"}, {"uri": "220", "title": "Interactive Visualization of Protein Dynamics", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Henk Huitema", "Robert van Liere"], "summary": "The study of time dependent characteristics of proteins is important for gaining insight into many biological processes. However, visualizing protein dynamics by animating atom trajectories does not provide satisfactory results. When the trajectory is sampled with large times steps, the impression of smooth motion will be destroyed due to the effects of temporal aliasing. Sampling with small time steps will result in the camouflage of interesting motions. In this case study, we discuss techniques for the interactive 3D visualization of the dynamics of the Photoactive Yellow Protein. We use essential dynamics methods to filter out uninteresting atom motions from the larger concerted motions. In this way, clear and concise 3D animations of protein motions can be produced. In addition, we discuss various interactive techniques that allow exploration of the essential subspace of the protein. We discuss the merits of these techniques when applied to the analysis of the yellow protein. CR", "keywords": ["fluctuation", "motion", "exploration", "insight", "analysis", "dynamic", "space", "protein", "shown", "eigenvectors", "trajectory", "chromophore", "technique", "eigenvector", "atom", "animation", "pyp", "time", "function", "visualization"], "document_vector": [-87.10604, -1.085454], "paragraphs": [{"paragraph_vector": [129.501388, -54.542991], "paragraph_keywords": ["protein", "motions", "dynamics", "-"]}, {"paragraph_vector": [134.61827, -51.288761], "paragraph_keywords": ["protein", "motions", "chromophore", "atoms"]}, {"paragraph_vector": [94.94033, -24.574638], "paragraph_keywords": ["atom", "techniques", "showing", "visualize"]}, {"paragraph_vector": [163.280807, -69.491989], "paragraph_keywords": ["motions", "dynamics", "protein", "matrix"]}, {"paragraph_vector": [130.816314, -67.308967], "paragraph_keywords": ["eigenvector", "filtered", "trajectory", "time"]}, {"paragraph_vector": [139.012115, -60.961921], "paragraph_keywords": ["pyp", "protein", "atoms", "chromophore"]}, {"paragraph_vector": [134.923339, -62.551521], "paragraph_keywords": ["chromophore", "motions", "dynamics", "shown"]}, {"paragraph_vector": [156.126861, -70.10585], "paragraph_keywords": ["motions", "techniques", "visualization", "found"]}, {"paragraph_vector": [-76.063285, -85.160003], "paragraph_keywords": ["department", "university", "determine", "vortices"]}], "content": {}, "doi": "10.1109/tcbb.2007.1035"}, {"uri": "221", "title": "Bioinformatics and Healthcare Computing Models and Services on Grid Initiatives for Data Analysis and Management", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ali Al Mazari"], "summary": "Grid platforms serve a wide range of activities in life sciences, bioinformatics, engineering, healthcare and other industries. This paper identifies the key bioinformatics and healthcare computing models and services on grid initiatives and classifies and describes their current use in the industry. Based on what has been reported by the literature, the paper identifies eight key areas of interests and eleven models where bioinformatics and healthcare oriented services are available. The efforts of this work will assist in determining areas of interests for end-users of grid technologies and highlighting future grid studies and research toward further grid services developments.", "keywords": ["computing", "uk", "database", "service", "project", "grid", "data", "health", "discovery", "processing", "information", "research", "application", "include", "drug", "bioinformatics"], "document_vector": [-59.676788, 59.480884], "paragraphs": [{"paragraph_vector": [-114.424041, -0.350628], "paragraph_keywords": ["data", "processing", "applications", "health"]}, {"paragraph_vector": [-116.88565, 3.03428], "paragraph_keywords": ["grid", "projects", "research", "uk"]}, {"paragraph_vector": [-116.942802, 1.134317], "paragraph_keywords": ["grid", "projects", "research", "information"]}, {"paragraph_vector": [-116.363822, 2.884268], "paragraph_keywords": ["grid", "projects", "include", "bioinformatics"]}, {"paragraph_vector": [-115.410583, 1.06956], "paragraph_keywords": ["grid", "bioinformatics", "data", "healthcare"]}, {"paragraph_vector": [-114.996101, 1.032916], "paragraph_keywords": ["services", "quality", "data", "applications"]}], "content": {}, "doi": "10.1109/DEXA.2014.19"}, {"uri": "222", "title": "Visual Analysis of Dynamic Protein Cavities and Binding Sites", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Michael Krone", "Daniel Kauker", "Guido Reina", "Thomas Ertl"], "summary": "We present a visual analysis application for protein simulations that allows researchers to investigate dynamic data interactively. Special focus lies on the analysis of cavities and binding sites, which play a critical role in protein function. Cavities are extracted and classified. The surface area of all cavities and the diameter of pockets and channels is computed to provide evidence for their accessibility. These values are also plotted in 2D graphs for a quantitative time-dependent analysis. For dynamic simulation data sets, the cavities are tracked to show their stability over time. The user is provided with a range of application-related parameters to interactively adjust the analysis algorithms. A sequence diagram shows the structure of the protein and additional annotations like binding sites. Furthermore, all views support brushing and linking for consistent selection and filtering. All algorithmic steps are implemented to run interactively on a commodity workstation. As a result, the user can immediately see the effect of a parameter change. This enables the real-time analysis of a running simulation for in-situ visualization.", "keywords": ["substrate", "cavity", "data", "pocket", "analysis", "acid", "application", "graph", "simulation", "mesh", "extract", "protein", "information", "extraction", "result", "surface", "based", "binding", "site", "centerline", "channel", "use", "time", "user"], "document_vector": [-111.989295, -5.43637], "paragraphs": [{"paragraph_vector": [42.900154, -73.935493], "paragraph_keywords": ["analysis", "data", "application", "atoms"]}, {"paragraph_vector": [27.482465, -12.478111], "paragraph_keywords": ["protein", "binding", "cavities", "chemical"]}, {"paragraph_vector": [51.79076, -40.343013], "paragraph_keywords": ["analysis", "offer", "user", "binding"]}, {"paragraph_vector": [26.618152, -29.407535], "paragraph_keywords": ["cavities", "surface", "application", "extract"]}, {"paragraph_vector": [36.915882, 28.721168], "paragraph_keywords": ["surface", "molecules", "cavity", "mesh"]}, {"paragraph_vector": [17.212162, -18.024242], "paragraph_keywords": ["cavities", "centerline", "edges", "algorithm"]}, {"paragraph_vector": [0.556247, 1.113082], "paragraph_keywords": ["centerline", "cycle", "protein", "radius"]}, {"paragraph_vector": [31.808868, -39.627269], "paragraph_keywords": ["binding", "cavities", "user", "centerline"]}, {"paragraph_vector": [66.871223, -18.387411], "paragraph_keywords": ["cavities", "application", "user", "surface"]}, {"paragraph_vector": [65.685531, -13.398118], "paragraph_keywords": ["analysis", "channel", "user", "application"]}, {"paragraph_vector": [17.328397, -41.700069], "paragraph_keywords": ["pocket", "surface", "site", "binding"]}, {"paragraph_vector": [55.765789, -57.806125], "paragraph_keywords": ["simulation", "analysis", "application", "tool"]}], "content": {}, "doi": "10.1109/CW.2013.25"}, {"uri": "223", "title": "Web-based molecular graphics for large complexes", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Alexander S. Rose", "Anthony R. Bradley", "Yana Valasatava", "Jose M. Duarte", "Andreas Prli\u0107", "Peter W. Rose"], "summary": "The interactive visualization of very large macromolecular complexes on the web is becoming a challenging problem as experimental techniques advance at an unprecedented rate and deliver structures of increasing size. We have tackled this problem by introducing the binary and compressed Macromolecular Transmission Format (MMTF) to reduce network transfer and parsing time, and by developing NGL, a highly memory-efficient and scalable WebGL-based viewer. MMTF offers over 75% compression over the standard mmCIF format, is over an order of magnitude faster to parse, and contains additional information (e.g., bond information). NGL renders molecular complexes with millions of atoms interactively on desktop computers and smartphones alike, making it a tool of choice for web-based molecular visualization in research and education.", "keywords": ["file", "web", "use", "model", "ngl", "data", "memory", "pdb", "copy", "structure", "parsing", "viewer"], "document_vector": [155.929702, -26.506286], "paragraphs": [{"paragraph_vector": [125.052062, 1.521924], "paragraph_keywords": ["data", "memory", "copies", "encoding"]}, {"paragraph_vector": [124.34996, 45.3875], "paragraph_keywords": ["data", "parsing", "files", "web"]}, {"paragraph_vector": [135.906921, -10.259465], "paragraph_keywords": ["viewer", "systems", "data", "structure"]}], "content": {}, "doi": "10.1109/VISUAL.2004.36"}, {"uri": "224", "title": "Comparison of Input Devices and Displays for Protein Visualization", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Joerg Meyer"], "summary": "For the visualization of proteins, interaction with the displayed model is indispensible in order to understand the three-dimensional protein structure. Accordingly, an efficient design of the user interface including display and input devices is crucial. A large number of input devices, some in 2D, some in 3D, and a large variety of display devices exist, which are commonly used for exploring large-scale 3D data structures. Unfortunately, not all combinations of input and display devices work together in a suitable and useful manner, either because they are not capable of 3D input or because, for instance, they need a hard table surface to work properly, making their utility for 3D navigation impractical. For example, in a large-screen, stereoscopic environment, where user immersion and freedom of motion in the display space are required, such a device would bind the user too much to a fixed location.", "keywords": ["stylus", "visualization", "glove", "structure", "exploration", "pinch", "application", "vr", "interaction", "space", "protein", "input", "joystick", "navigation", "screen", "system", "computer", "object", "button", "resolution", "surface", "position", "mounted", "graphic", "display", "use", "center", "model", "device", "user", "table", "freedom", "environment", "provide", "hand", "task"], "document_vector": [160.283157, -24.885959], "paragraphs": [{"paragraph_vector": [140.67926, 3.855973], "paragraph_keywords": ["user", "input", "devices", "display"]}, {"paragraph_vector": [149.36795, 9.223379], "paragraph_keywords": ["stylus", "interaction", "device", "input"]}, {"paragraph_vector": [152.434936, 6.638123], "paragraph_keywords": ["hand", "interaction", "tracker", "use"]}, {"paragraph_vector": [148.50505, 9.399509], "paragraph_keywords": ["joystick", "wand", "object", "device"]}, {"paragraph_vector": [151.410858, 8.74211], "paragraph_keywords": ["user", "device", "desk", "navigation"]}, {"paragraph_vector": [147.806793, 14.302004], "paragraph_keywords": ["hmds", "system", "mounted", "displays"]}, {"paragraph_vector": [147.052246, 19.673919], "paragraph_keywords": ["input", "display", "tiled", "devices"]}, {"paragraph_vector": [141.735565, 3.896093], "paragraph_keywords": ["structure", "interaction", "visualization", "application"]}, {"paragraph_vector": [119.85054, -25.275642], "paragraph_keywords": ["structure", "representations", "structures", "information"]}, {"paragraph_vector": [145.624603, 11.357658], "paragraph_keywords": ["object", "interaction", "point", "model"]}, {"paragraph_vector": [151.977325, 2.560387], "paragraph_keywords": ["buttons", "stylus", "functions", "device"]}, {"paragraph_vector": [152.893478, 3.118202], "paragraph_keywords": ["users", "gamepad", "functions", "gloves"]}, {"paragraph_vector": [147.607437, 8.311008], "paragraph_keywords": ["display", "screen", "displays", "tiled"]}, {"paragraph_vector": [145.078109, 6.631702], "paragraph_keywords": ["display", "devices", "device", "table"]}, {"paragraph_vector": [145.918792, 8.121573], "paragraph_keywords": ["resolution", "display", "displays", "input"]}, {"paragraph_vector": [144.156265, 19.520519], "paragraph_keywords": ["ieee", "graphics", "displays", "computer"]}, {"paragraph_vector": [138.034317, 15.508047], "paragraph_keywords": ["visualization", "graphics", "university", "computer"]}], "content": {}, "doi": "10.1145/369028.369136"}, {"uri": "225", "title": "Visualization of Biological Sequence Similarity Search Results", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ed Huai-hsin Chit", "Phillip Barry", "Elizabeth Shoopt", "John V. Carlist", "Ernest Retzelt", "John Riedlt"], "summary": "Biological sequence similarity analysis presents visualization challenges, primarily because of the masse\u2019ve amounts of discrete, multi-dimensional data. Genomic data generated by molecular biologists is analyzed by algorithms that search for similarity to known sequences in large genomic databases. The output from these algorithms can be several thousand pages of text, and is dificult to analyze because of its length and complexity. We developed and implemented a novel graphical representation for sequence similarity search results, which visually reveals features that are dificult to find in textual reports. The method opens new possibilities in the interpretation of this discrete, multidimensional data by enabling interactive investigation of the graphical representation.", "keywords": [], "document_vector": [-164.635635, 60.612319], "paragraphs": [{"paragraph_vector": [-130.934478, 22.734312], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/HPTCDL.2014.7"}, {"uri": "226", "title": "Direct Visualization of Unwinding Activity of Duplex Rna by Dbpa, a Dead Box Helicase, at Single Molecule Resolution", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "The Eschericia coli protein, DbpA, is unique in its subclass of DEAD box RNA helicases because it possesses ATPase specific activity toward the peptidyl transferase center in 23s rRNA. Although its remarkable ATPase activity is well defined toward various substrates, its RNA helicase activity remained to be characterized. We show by using biochemical assays and Atomic Force Microscopy (AFM) that DbpA exhibits ATP-stimulated unwinding activity of RNA duplex regardless of its primary sequence. This work presents an attempt to investigate the action of DEADbox proteins by a single-molecule visualization methodology. Our AFM images enabled us to directly observe the unwinding reaction of a DEAD box helicase on long stretches of double stranded RNA (dsRNA). Specifically, we a u l d differentiate between defined steps i n the unwinding reaction. Recent studies have questioned the designation of DbpA, in particular, and DEAD box proteins in general as RNA helicases. However, accumulated evidences and the results reported here suggest that these proteins are indeed helicases that resemble in many aspects the DNA helicases. Although the biological function of DbpA and many RNA helicases from the DEAD box family remain to be elucidated, this work provides means to understand the struchral bases of the motoric and enzymatic activity of DEAD box proteins. In addition, application of AFM to the study of these motor proteins open a new way to investigate the intimate relationship between these enzymes and their co-factors needed for the biological catalysis.", "keywords": [], "document_vector": [-139.739593, 14.975264], "paragraphs": [{"paragraph_vector": [-129.005935, 21.187709], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/VISUAL.2002.1183780"}, {"uri": "227", "title": "Parallel Computation and Interactive Visualization of Time-varying Solvent Excluded Surfaces", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Michael Krone", "Carsten Dachsbacher", "Thomas Ertl"], "summary": "Molecular dynamics simulations are a principal tool in the study of molecules. The efficient investigation of the structure and dynamics of the simulated molecular systems benefits from interactive high-quality visualizations. Molecular surfaces are among the most common representations for visual analysis of the properties of these molecules, especially when studying proteins and other biomolecules. In this paper we evaluate the suitability of parallel graphics hardware (GPUs) for interactive computation and visualization of the Solvent Excluded Surface. This surface consists of spherical and toroidal geometric primitives, which can be rendered using ray casting on the GPU, thereby obtaining unsurpassed visual quality at interactive frame rates. The key, however, is the efficient calculation of the Reduced Surface that defines the occurrence and location of these primitives. The Reduced Surface can be computed quite efficient and allows for a partial update of the surface in regions where changes occur. However, the original algorithm to construct the Reduced Surface cannot be parallelized straightforward. We introduce a novel, parallel algorithm, leveraging programmable GPUs, for computing the Reduced Surface. This allows the high-quality rendering and interactive exploration of complex molecules and time-varying molecular datasets.", "keywords": ["hardware", "face", "computing", "algorithm", "triangle", "compute", "primitive", "step", "computation", "ray", "sphere", "surface", "atom", "probe", "edge", "graphic", "s", "r", "gpu", "radius", "rendering", "casting"], "document_vector": [-19.975458, -44.189605], "paragraphs": [{"paragraph_vector": [64.707511, 50.076129], "paragraph_keywords": ["ses", "surface", "rates", "algorithm"]}, {"paragraph_vector": [25.130689, 9.86203], "paragraph_keywords": ["ses", "probe", "surface", "atom"]}, {"paragraph_vector": [75.101791, 31.816122], "paragraph_keywords": ["rs", "ses", "rendering", "triangles"]}, {"paragraph_vector": [9.252791, 7.191512], "paragraph_keywords": ["rs", "probe", "face", "edge"]}, {"paragraph_vector": [43.420402, 48.944202], "paragraph_keywords": ["computation", "atoms", "computations", "neighborhood"]}, {"paragraph_vector": [12.892232, 8.173046], "paragraph_keywords": ["rs", "faces", "probe", "step"]}, {"paragraph_vector": [16.407365, 48.793254], "paragraph_keywords": ["ray", "rs", "casting", "primitives"]}, {"paragraph_vector": [19.666183, 45.328926], "paragraph_keywords": ["ses", "sphere", "graphics", "ray"]}, {"paragraph_vector": [128.270675, 72.846206], "paragraph_keywords": ["algorithm", "data", "view", "ses"]}, {"paragraph_vector": [167.534133, 74.666633], "paragraph_keywords": ["surface", ".", "visualization", "graphics"]}], "content": {}, "doi": "10.1145/1400885.1401022"}, {"uri": "228", "title": "empty", "timestamp": "2003", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yujun Wan", "Yuan-Fan9 Wan"], "summary": "We have deve10ped a pr0te1n v15ua112at10n 5y5tem 6a5ed 0n Java 3D 7M. Java 3D pr0v1de5 c0mpat16111ty am0n9 d1fferent 5y5tem5 and ena61e5 app11cat10n5 t0 6e run rem0te1y thr0u9h we6 6r0w5er5. H0wever, u51n9 Java 3D f0r v15ua112at10n ha5 50me perf0rmance 155ue5 w1th 1t. 7he pr1mary c0ncern5 a60ut m01ecu1ar v15ua112at10n t0015 6a5ed 0n Java 3D are 1n the1r 6e1n9 510w 1n term5 0f 1nteract10n 5peed and 1n the1r 1na6111ty t0 10ad 1ar9e m01ecu1e5. 7h15 6ehav10r 15 e5pec1a11y apparent when the num6er 0f at0m5 t0 6e d15p1ayed 15 hu9e, 0r when 5evera1 pr0te1n5 are t0 6e d15p1ayed 51mu1tane0u51y f0r c0mpar150n. 1n th15 paper we pre5ent techn14ue5 f0r 0r9an121n9 a Java 3D 5cene 9raph t0 tack1e the5e pr061em5. We dem0n5trate the effect1vene55 0f the5e techn14ue5 6y c0mpar1n9 the v15ua112at10n c0mp0nent 0f 0ur 5y5tem w1th tw0 0ther Java 3D 6a5ed m01ecu1ar v15ua112at10n t0015. 1n part1cu1ar, f0r van der Waa15 d15p1ay m0de, w1th the eff1c1ent 0r9an12at10n 0f the 5cene 9raph, we c0u1d ach1eve up t0 e19ht t1me5 1mpr0vement 1n render1n9 5peed and c0u1d 10ad m01ecu1e5 three t1me5 a5 1ar9e a5 the prev10u5 5y5tem5 c0u1d.", "keywords": ["m", "t"], "document_vector": [23.558336, 7.639511], "paragraphs": [{"paragraph_vector": [-169.328979, 8.178578], "paragraph_keywords": ["e", "m", "p", "t"]}, {"paragraph_vector": [-167.784317, 7.625523], "paragraph_keywords": ["m", "t", "van", "r"]}, {"paragraph_vector": [-168.255722, 7.674776], "paragraph_keywords": ["t", "m", "way", "appearance"]}, {"paragraph_vector": [-167.461044, 7.961398], "paragraph_keywords": ["t", "d", "jmv", "r"]}, {"paragraph_vector": [-167.876556, 7.709985], "paragraph_keywords": ["t", "e", "r", "d"]}, {"paragraph_vector": [-170.164123, 8.326971], "paragraph_keywords": [".", "m", "e", "d"]}], "content": {}, "doi": "10.1109/VISUAL.2004.123"}, {"uri": "229", "title": "Techniques for the Visualization of Topological Defect Behavior in Nematic Liquid Crystals", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Vadim A. Slavin", "Robert A. Pelcovits", "George Loriot", "Andrew Callan-Jones", "David H. Laidlaw"], "summary": "We present visualization tools for analyzing molecular simulations of liquid crystal (LC) behavior. The simulation data consists of terabytes of data describing the position and orientation of every molecule in the simulated system over time. Condensed matter physicists study the evolution of topological defects in these data, and our visualization tools focus on that goal. We first convert the discrete simulation data to a sampled version of a continuous second-order tensor field and then use combinations of visualization methods to simultaneously display combinations of contractions of the tensor data, providing an interactive environment for exploring these complicated data. The system, built using AVS, employs colored cutting planes, colored isosurfaces, and colored integral curves to display fields of tensor contractions including Westin's scalar cl, cp, and cs metrics and the principal eigenvector. Our approach has been in active use in the physics lab for over a year. It correctly displays structures already known; it displays the data in a spatially and temporally smoother way than earlier approaches, avoiding confusing grid effects and facilitating the study of multiple time steps; it extends the use of tools developed for visualizing diffusion tensor data, re-interpreting them in the context of molecular simulations; and it has answered long-standing questions regarding the orientation of molecules around defects and the conformational changes of the defects.", "keywords": ["data", "structure", "value", "defect", "molecule", "order", "physicist", "lc", "orientation", "anisotropy", "simulation", "dynamic", "point", "system", "figure", "core", "technique", "location", "field", "feature", "phase", "tensor", "time", "streamtubes", "visualization", "crystal"], "document_vector": [63.067325, -46.421836], "paragraphs": [{"paragraph_vector": [52.124221, -18.790433], "paragraph_keywords": ["molecules", "lc", "crystals", "models"]}, {"paragraph_vector": [-132.835571, 48.923698], "paragraph_keywords": ["phase", "molecules", "order", "time"]}, {"paragraph_vector": [78.566215, -25.344648], "paragraph_keywords": ["molecules", "visualization", "tensor", "defects"]}, {"paragraph_vector": [80.304367, -27.374376], "paragraph_keywords": ["defects", "defect", "director", "tensor"]}, {"paragraph_vector": [77.841125, -26.087377], "paragraph_keywords": ["tensor", "molecules", "anisotropy", "version"]}, {"paragraph_vector": [80.527015, -29.038021], "paragraph_keywords": ["anisotropy", "defects", "\u03bb\u03bb\u03bb", "tensor"]}, {"paragraph_vector": [78.858413, -23.125968], "paragraph_keywords": ["data", "defect", "visualization", "probe"]}, {"paragraph_vector": [78.085815, -23.970125], "paragraph_keywords": ["biaxiality", "defect", "visualization", "question"]}, {"paragraph_vector": [76.111221, -23.388999], "paragraph_keywords": ["defect", "core", "figure", "features"]}, {"paragraph_vector": [78.700866, -25.251737], "paragraph_keywords": ["tensor", "defect", "visualization", "techniques"]}, {"paragraph_vector": [81.553329, -18.90574], "paragraph_keywords": ["user", "zoom", "explore", "vicinity"]}], "content": {}, "doi": "10.1109/VISUAL.1995.480793"}, {"uri": "230", "title": "Augmented Reality with Tangible Auto-Fabricated Models for Molecular Biology Applications", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Alexandre Gillet", "Michel Sanner", "Daniel Stoffler", "David Goodsell", "Arthur Olson"], "summary": "The evolving technology of computer auto-fabrication (\"3-D printing\") now makes it possible to produce physical models for complex biological molecules and assemblies. We report on an application that demonstrates the use of auto-fabricated tangible models and augmented reality for research and education in molecular biology, and for enhancing the scientific environment for collaboration and exploration. We have adapted an augmented reality system to allows virtual 3-D representations (generated by the Python Molecular Viewer) to be overlaid onto a tangible molecular model. Users can easily change the overlaid information, switching between different representations of the molecule, displays of molecular properties such as electrostatics, or dynamic information. The physical model provides a powerful, intuitive interface for manipulating the computer models, streamlining the interface between human intent, the physical model, and the computational activity. CR Categories: J.3 [Life and Medical Sciences]: Biology and Genetics. I.3.8 [Computer Graphics]: Applications.", "keywords": ["data", "structure", "representation", "protein", "augmented", "system", "computer", "object", "biology", "backbone", "interface", "work", "python", "level", "student", "pmv", "library", "marker", "display", "model", "ar", "use", "software", "form", "function", "environment", "modeling", "built", "video"], "document_vector": [133.02005, -36.866336], "paragraphs": [{"paragraph_vector": [122.676452, -17.253763], "paragraph_keywords": ["models", "computer", "structure", "data"]}, {"paragraph_vector": [109.34468, 17.411901], "paragraph_keywords": ["models", "python", "pmv", "representations"]}, {"paragraph_vector": [95.540878, 13.320676], "paragraph_keywords": ["modeling", "models", "model", "atoms"]}, {"paragraph_vector": [99.569938, 14.107372], "paragraph_keywords": ["models", "methods", "built", "process"]}, {"paragraph_vector": [113.607452, 18.647047], "paragraph_keywords": ["markers", "video", "display", "camera"]}, {"paragraph_vector": [112.271682, 15.08678], "paragraph_keywords": ["model", "markers", "object", "models"]}, {"paragraph_vector": [77.47364, 1.031461], "paragraph_keywords": ["model", "field", "sod", "backbone"]}, {"paragraph_vector": [133.296676, -1.232317], "paragraph_keywords": ["students", "models", "augmented", "structure"]}, {"paragraph_vector": [132.110366, -1.560733], "paragraph_keywords": ["approach", "develop", "system", "model"]}, {"paragraph_vector": [-155.492279, 9.78146], "paragraph_keywords": ["institute", "university", "science", "bisti"]}], "content": {}, "doi": "10.1109/VISUAL.2004.23"}, {"uri": "231", "title": "Empirical Evaluation of Visual Graph Analytic Techniques", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "! \" # $ % & ' ( % ) * + , + . / 0 ( ( $  1 2   ! \" #  # / ( ( $ & * + % ) * + # 3 % , + $ 4 5  6 1 7 8 9 : ; < = >   ! \" # $ # + $ ? @ A ? ? & B + % ) * +  C ( D + $ , \" +  6 1 7 8 9 : ; < = >   ! \" # E + $ \" + \" & B + % ) * + 3 E + + F $ #  6 1 7 8 9 : ; < = >   ! \" # 5 $ # \" G & B + % ) * + F $ H G /  6 1 7 8 9 : ; < = >   ! \" # 5 $ G / I I I & B + % ) * + J K L M N O P M Q R : <  2 S < > = = = T ; 2 : = < 2 = =  = 2 =  9 = T U : 2 = 9 = 2 >  : = = = 9 9 = = > < > >  2 2 = T U 2 = < =  < V = 2 :  : = 2 2 T 8 =  9 1 W 1 W > 1 ; X 7 > = X Y Z T ; > 9 9 < Z 9  = 2 < 2 = > = 2 > = = [ 1 \\ U ; 6 ] ; ^ T  2 _ X Y Z 2 : ` : a 2 2  = 9 b 1 \\ U ; 6 ] ; 2 c : 1 = T  = > < = :  = : : d = 2 = 1 W 1 W =  1 ; X 7 T 8 = : =  9 b : V 9  2 =  _ < T X Y Z 2 = < :  9 2 : < : 2 S  : 9 : 9 < S = T W S 2 2 = = =  : T e f g h i N j L k l m L n O o p N O q r O s O o g M m P L t p N O q r l m L n O o m u O M m i s t q N i M f m s t p f s f L t v i M m w L t K m p x j O M O y z y { | } ~ \u007f \u0080 \u0081 | \u0082 ~ { \u0083 \u0084 \u0085 \u0086 \u0087 \u0088 \u0089 \u008a \u008b \u008a \u0087 \u008c \u008d \u0085 \u008e \u0085 \u008d \u008d \u0085 \u0089 \u008b \u008f \u0090 \u0087 \u0091 \u0085 \u008b \u0084 \u0085 \u0092 \u008a \u008c \u008b \u0086 \u008a \u008b \u0093 \u0084 \u0085 \u008c \u008a \u0094 \u0089 \u008a \u0086 \u0095 \u0094 \u0089 \u008a \u0094 \u0093 \u0085 \u008f \u008e \u0089 \u008a \u008b \u008a \u0096 \u008b \u0084 \u0087 \u008c \u0087 \u0094 \u0093 \u008d \u0085 \u008a \u008c \u0087 \u0094 \u0088 \u0088 \u008d \u008f \u0097 \u008b \u0084 \u008f \u008e \u0089 \u008a \u008b \u008a \u0089 \u0085 \u008c \u0093 \u008d \u0087 \u0086 \u0085 \u008c \u008b \u0084 \u0085 \u0090 \u008a \u008d \u0088 \u0085 \u0092 \u008f \u0090 \u0095 \u0098 \u0085 \u008f \u008e \u0089 \u008a \u008b \u008a \u0086 \u008f \u008b \u0084 \u0087 \u0094 \u008c \u008b \u008d \u0095 \u0093 \u008b \u0095 \u008d \u0085 \u0089 \u008a \u0094 \u0089 \u0095 \u0094 \u008c \u008b \u008d \u0095 \u0093 \u008b \u0095 \u008d \u0085 \u0089 z \u0099 \u0087 \u0088 \u0089 \u008a \u008b \u008a \u0087 \u008c \u008a \u0090 \u008c \u008f \u009a \u0090 \u008a \u009b \u0087 \u0094 \u0088 \u0087 \u008b \u008c \u009a \u008a \u008d \u008b \u0087 \u0094 \u008b \u0084 \u0085 \u008a \u008d \u0085 \u008a \u008c \u008f \u008e \u0094 \u0085 \u008b \u0097 \u008f \u008d \u0091 \u008c \u009c \u008c \u0085 \u0093 \u0095 \u008d \u0087 \u008b \u009b \u009c \u0098 \u008a \u0094 \u008a \u0088 \u0085 \u0098 \u0085 \u0094 \u008b \u009c \u0086 \u008a \u0094 \u0091 \u0087 \u0094 \u0088 \u009c \u008c \u008b \u008f \u0093 \u0091 \u0085 \u009d \u0093 \u0084 \u008a \u0094 \u0088 \u0085 \u009c \u0098 \u0085 \u0089 \u0087 \u0093 \u0087 \u0094 \u0085 \u009c \u008b \u008d \u008a \u0094 \u008c \u009a \u008f \u008d \u008b \u008a \u008b \u0087 \u008f \u0094 \u009c \u0084 \u0085 \u008a \u0090 \u008b \u0084 \u009c \u0086 \u0087 \u008f \u0087 \u0094 \u008e \u008f \u008d \u0098 \u008a \u008b \u0087 \u0093 \u008c \u009c \u0088 \u0085 \u0094 \u008f \u0098 \u0087 \u0093 \u008c \u008a \u0094 \u0089 \u008c \u008f \u008f \u0094 z \u0099 \u0087 \u0088 \u0089 \u008a \u008b \u008a \u008a \u0090 \u008c \u008f \u0084 \u0085 \u0090 \u009a \u008c \u008b \u008f \u008e \u0087 \u0094 \u0089 \u008b \u0084 \u0085 \u008c \u0095 \u008c \u009a \u0087 \u0093 \u0087 \u008f \u0094 \u008c \u0087 \u0094 \u0092 \u008f \u0090 \u0092 \u0085 \u0089 \u0087 \u0094 \u0086 \u0090 \u008a \u0093 \u0091 \u0098 \u008a \u008d \u0091 \u0085 \u008b \u008a \u0094 \u0089 \u008b \u008d \u008a \u0094 \u008c \u008a \u0093 \u008b \u0087 \u008f \u0094 \u008c \u0087 \u0094 \u0092 \u008f \u0090 \u0092 \u0085 \u0089 z \u009e \u008a \u008b \u008a \u009f \u0095 \u008a \u0094 \u008b \u0087 \u008b \u009b \u0089 \u008f \u0085 \u008c \u0094 \u008f \u008b \u0098 \u008a \u008b \u008b \u0085 \u008d \u0096 \u0097 \u0084 \u008a \u008b \u0098 \u008a \u008b \u008b \u0085 \u008d \u008c \u0087 \u008c \u0097 \u0084 \u008a \u008b \u0089 \u008f \u0097 \u0087 \u008b \u0084 \u008a \u0090 \u0090 \u008b \u0084 \u008a \u008b \u0089 \u008a \u008b \u008a z \u009e \u008a \u008b \u008a \u0093 \u008a \u0094 \u0086 \u0085 \u008b \u008a \u0091 \u0085 \u0094 \u008e \u008d \u008f \u0098 \u008a \u0094 \u009b \u008c \u008f \u0095 \u008d \u0093 \u0085 \u009c \u008a \u0094 \u0089 \u008e \u0095 \u008d \u008b \u0084 \u0085 \u008d \u0093 \u008a \u0094 \u0086 \u0085 \u008a \u0094 \u008a \u0090 \u009b  \u0085 \u0089 \u0089 \u0085 \u009a \u0085 \u0094 \u0089 \u0087 \u0094 \u0088 \u0095 \u009a \u008f \u0094 \u0087 \u008b \u008c \u008b \u009b \u009a \u0085 \u008a \u0094 \u0089 \u0094 \u008a \u008b \u0095 \u008d \u0085 \u009c \u0097 \u0084 \u0087 \u0093 \u0084 \u0085 \u0094 \u008a \u0086 \u0090 \u0085 \u008c \u008b \u008f \u008d \u0085 \u0089 \u0095 \u0093 \u0085 \u0093 \u008f \u008c \u008b \u008c \u009c \u008d \u0085 \u0089 \u0095 \u0093 \u0085 \u008b \u0087 \u0098 \u0085 \u009c \u0089 \u0085 \u0092 \u0085 \u0090 \u008f \u009a \u0098 \u0085 \u0094 \u008b \u008f \u008e \u0094 \u0085 \u0097 \u009a \u008d \u008f \u0089 \u0095 \u0093 \u008b \u008a \u0094 \u0089 \u0085 \u008e \u008e \u0085 \u0093 \u008b \u0087 \u0092 \u0085 \u0089 \u0085 \u0093 \u0087 \u008c \u0087 \u008f \u0094 \u00a1 \u0098 \u008a \u0091 \u0087 \u0094 \u0088 \u009a \u008f \u0097 \u0085 \u008d z \u00a2 \u0087 \u0088 \u0084 \u009a \u008f \u0097 \u0085 \u008d \u0085 \u0089 \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008b \u0085 \u0093 \u0084 \u0094 \u0087 \u009f \u0095 \u0085 \u008c \u0097 \u0084 \u0085 \u0094 \u009a \u008a \u0087 \u008d \u0085 \u0089 \u0097 \u0087 \u008b \u0084 \u0086 \u0087 \u0088 \u0089 \u008a \u008b \u008a \u008b \u0084 \u008a \u0094 \u008e \u0085 \u0097 \u0092 \u008a \u0090 \u0095 \u008a \u0086 \u0090 \u0085 \u008b \u008a \u008c \u0091 \u008c \u0093 \u008a \u0094 \u0086 \u0085 \u008a \u0093 \u0093 \u008f \u0098 \u009a \u0090 \u0087 \u008c \u0084 \u0085 \u0089 \u0090 \u0087 \u0091 \u0085 \u008b \u008f \u0089 \u0085 \u008b \u0085 \u008d \u0098 \u0087 \u0094 \u008a \u008b \u0087 \u008f \u0094 \u008f \u008e \u008d \u008f \u008f \u008b \u0093 \u008a \u0095 \u008c \u0085 \u008c \u008f \u008e \u008c \u009b \u008c \u008b \u0085 \u0098 \u0093 \u008d \u008a \u008c \u0084 \u0085 \u008c \u009c \u009a \u008d \u008f \u0086 \u0090 \u0085 \u0098 \u008c \u009c \u00a3 \u00a4 \u00a5 \u00a5 \u00a6 \u00a7 \u00a6 \u0308 \u00a9 a \u00ab \u00ac \u00a6 \u0308 \u00ad \u00ae \u00ae \u00a6 \u00a4 \u00a5 \u0304 \u00a4 \u00b0 \u0308 \u00b1 a \u00a9 \u00ad \u00ae \u00a6 \u00ac 2 a \u0086 \u0095 \u009b \u0087 \u0094 \u0088 \u008b \u008d \u0085 \u0094 \u0089 \u008c \u009c \u008d \u0085 \u0093 \u008a \u0090 \u0093 \u0095 \u0090 \u008a \u008b \u0087 \u0094 \u0088 \u0085 \u0094 \u008b \u0087 \u008d \u0085 \u008d \u0087 \u008c \u0091 \u009a \u008f \u008d \u008b \u008e \u008f \u0090 \u0087 \u008f \u008c \u0087 \u0094 \u0098 \u0087 \u0094 \u0095 \u008b \u0085 \u008c \u009c \u0089 \u0085 \u008b \u0085 \u0093 \u008b \u0087 \u0094 \u0088 \u008e \u008d \u008a \u0095 \u0089 \u0095 \u0090 \u0085 \u0094 \u008b \u0086 \u0085 \u0084 \u008a \u0092 \u0087 \u008f \u008d \u0086 \u0085 \u008e \u008f \u008d \u0085 \u0087 \u008b \u008a \u008e \u008e \u0085 \u0093 \u008b \u008c \u008b \u0084 \u0085 \u008f \u008d \u0088 \u008a \u0094 \u0087  \u008a \u008b \u0087 \u008f \u0094 z \u0083 \u0084 \u0085 \u0098 \u008f \u008c \u008b \u008f \u009a \u008b \u0087 \u0098 \u008a \u0090 \u008d \u0085 \u009a \u008d \u0085 \u008c \u0085 \u0094 \u008b \u008a \u008b \u0087 \u008f \u0094 \u008f \u008e \u008b \u0084 \u0085 \u0092 \u008a \u008c \u008b \u0086 \u0095 \u0090 \u0091 \u008c \u008f \u008e \u0089 \u008a \u008b \u008a \u0097 \u0087 \u008b \u0084 \u0087 \u0094 \u008a \u0094 \u0089 \u008f \u0095 \u008b \u008c \u0087 \u0089 \u0085 \u008b \u0084 \u0085 \u0094 \u0085 \u008b \u0097 \u008f \u008d \u0091 \u008a \u008d \u0085 \u008b \u0084 \u0085 \u0088 \u008d \u008a \u009a \u0084 \u008c z 3 \u008d \u008a \u009a \u0084 \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008c \u0087 \u008c \u0086 \u0085 \u0087 \u0094 \u0088 \u0097 \u0087 \u0089 \u0085 \u0090 \u009b \u008a \u0089 \u008f \u009a \u008b \u0085 \u0089 \u008a \u0094 \u0089 \u0095 \u008c \u0085 \u0089 \u008e \u008f \u008d \u008b \u0084 \u0085 \u008d \u0085 \u009a \u008d \u0085 \u008c \u0085 \u0094 \u008b \u008a \u008b \u0087 \u008f \u0094 \u008f \u008e \u0087 \u0094 \u008e \u008f \u008d \u0098 \u008a \u008b \u0087 \u008f \u0094 \u008c \u008b \u008d \u0085 \u008a \u0098 \u008c \u0301 \u03bc \u00b6 z \u00b7 \u0085 \u008f \u0327 1 \u0093 \u008f \u0095 \u0090 \u0089 \u0086 \u0085 \u008b \u0084 \u0085 \u0085 \u0094 \u008b \u0087 \u008d \u0085 \u0097 \u008f \u008d \u0090 \u0089 o \u008c \u008b \u008f \u009a \u0088 \u008d \u008a \u009a \u0084 z y \u008b \u0087 \u008c \u008a \u008c \u008c \u0085 \u0098 \u0086 \u0090 \u0085 \u0089 \u0087 \u0094 \u008b \u0084 \u0085 \u008e \u0090 \u008f \u008f \u008d \u0095 \u009a \u008e \u008f \u008d \u009f \u0095 \u0087 \u008b \u0085 \u008a \u0093 \u0084 \u008a \u008d \u008b \u0089 \u008a \u008b \u008a \u0086 \u008a \u008c \u0085 \u009c \u008c \u008f \u0098 \u0085 \u008a \u0094 \u0087 \u0094 \u0088 \u0087 \u008b \u008c \u008c \u008b \u008d \u0095 \u0093 \u008b \u0095 \u008d \u0085 \u0097 \u008a \u008c \u008e \u008a \u008c \u0084 \u0087 \u008f \u0094 \u0085 \u0089 \u008e \u008f \u008d \u008f \u009a \u008b \u0087 \u0098 \u0087  \u0087 \u0094 \u0088 \u008c \u009a \u0085 \u0085 \u0089 \u009b \u0089 \u0087 \u008d \u0085 \u0093 \u008b \u0087 \u008f \u0094 \u009c \u0098 \u0085 \u0098 \u008f \u008d \u009b \u009c \u008a \u0094 \u0089 \u008a \u0090 \u008c \u008f \u008b \u0084 \u0085 \u008b \u008d \u008a \u0092 \u0085 \u008d \u008c \u008a \u0090 \u008f \u008e \u0086 \u008f \u008b \u0084 \u0094 \u008f \u0089 \u0085 \u008c \u008a \u0094 \u0089 \u0093 \u008f \u0094 \u0094 \u0085 \u0093 \u008b \u0087 \u008f \u0094 \u008c z \u00a2 \u0085 \u0094 \u0093 \u0085 \u009c \u008a \u008c \u008c \u008f \u0093 \u0087 \u008a \u008b \u0087 \u008f \u0094 \u008c \u0084 \u008a \u0092 \u0085 \u0086 \u0085 \u0085 \u0094 \u0093 \u0090 \u008a \u008d \u0087 \u008e \u0087 \u0085 \u0089 \u008a \u008c \u008b \u008f \u009a \u0093 \u0090 \u008a \u008c \u008c \u0093 \u0087 \u008b \u0087  \u0085 \u0094 \u008c \u0087 \u0094 \u00b7 \u0085 \u008f \u0327 1 \u0301 \u00bb \u00b6 z \u0083 \u0084 \u0085 \u0085 \u009d \u0093 \u0087 \u008b \u0087 \u0094 \u0088 \u009a \u008d \u008f \u009a \u0085 \u008d \u008b \u0087 \u0085 \u008c \u008e \u008f \u008d \u00b7 \u0085 \u008f \u0327 1 \u0089 \u008a \u008b \u008a \u0086 \u008a \u008c \u0085 \u008a \u008d \u0085 \u0089 \u0087 \u008c \u0093 \u0095 \u008c \u008c \u0085 \u0089 \u009c \u0087 \u0094 \u0093 \u0090 \u0095 \u0089 \u0087 \u0094 \u0088 \u0087 \u0094 \u0087 \u008b \u0087 \u008a \u0090 \u008d \u0085 \u0090 \u0085 \u008a \u008c \u0085 \u008a \u0094 \u0089 \u0087 \u0098 \u009a \u0090 \u0085 \u0098 \u0085 \u0094 \u008b \u008a \u008b \u0087 \u008f \u0094 \u0090 \u008a \u0094 \u0088 \u0095 \u008a \u0088 \u0085 \u009c \u0089 \u008a \u008b \u008a \u0098 \u008f \u0089 \u0085 \u0090 \u009c \u008a \u0093 \u0093 \u0085 \u008c \u008c \u0098 \u0085 \u008b \u0084 \u008f \u0089 \u008c \u008a \u0094 \u0089 \u009f \u0095 \u0085 \u008d \u009b \u0090 \u008a \u0094 \u0088 \u0095 \u008a \u0088 \u0085 \u008c \u009c \u0095 \u0094 \u0089 \u0085 \u008d \u0090 \u009b \u0087 \u0094 \u0088 \u0089 \u008a \u008b \u008a \u0086 \u008a \u008c \u0085 \u008c \u008b \u008f \u008d \u008a \u0088 \u0085 \u009c \u0093 \u008a \u0093 \u0084 \u0087 \u0094 \u0088 \u009c \u0093 \u008f \u0094 \u0093 \u0095 \u008d \u008d \u0085 \u0094 \u0093 \u009b \u008a \u0094 \u0089 \u008c \u0093 \u008a \u0090 \u008a \u0086 \u0087 \u0090 \u0087 \u008b \u009b \u0301 1\u20444 \u00b6 z \u0083 \u0084 \u0085 \u009a \u008a \u008b \u0084 \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008c \u0095 \u008c \u0085 \u0089 \u008b \u008f \u008a \u0094 \u008a \u0090 \u009b  \u0085 \u008b \u0084 \u0085 \u008f \u009a \u008b \u0087 \u0098 \u0087  \u0085 \u0089 \u009a \u008a \u008b \u0084 \u008e \u008d \u008f \u0098 \u008f \u0094 \u0085 \u009a \u008f \u0087 \u0094 \u008b \u008b \u008f \u008a \u0094 \u008f \u008b \u0084 \u0085 \u008d z y \u0094 \u008a \u0088 \u008d \u008a \u009a \u0084 \u0087 \u0093 \u008a \u0090 \u009a \u008f \u0087 \u0094 \u008b \u008f \u008e \u0092 \u0087 \u0085 \u0097 \u009c \u008b \u0084 \u0087 \u008c \u008a \u0094 \u008a \u0090 \u009b \u008c \u0087 \u008c \u0093 \u008a \u0094 \u0086 \u0085 \u0084 \u0085 \u0090 \u009a \u008e \u0095 \u0090 \u008b \u008f \u0089 \u0085 \u008b \u0085 \u008d \u0098 \u0087 \u0094 \u0085 \u008b \u0084 \u0085 \u008c \u0084 \u008f \u008d \u008b \u0085 \u008c \u008b \u0089 \u0087 \u008c \u008b \u008a \u0094 \u0093 \u0085 \u0086 \u0085 \u008b \u0097 \u0085 \u0085 \u0094 \u008b \u0097 \u008f \u0094 \u008f \u0089 \u0085 \u008c \u008f \u008d \u0092 \u0085 \u008d \u008b \u0087 \u0093 \u0085 \u008c \u0087 \u0094 \u008b \u0084 \u0085 \u0088 \u008d \u008a \u009a \u0084 z 1\u20442 \u0094 \u0085 \u008f \u008e \u008b \u0084 \u0085 \u0097 \u0087 \u0089 \u0085 \u0090 \u009b \u0091 \u0094 \u008f \u0097 \u0094 \u008a \u008c \u009a \u0085 \u0093 \u008b \u008c \u008f \u008e \u0088 \u008d \u008a \u009a \u0084 \u008a \u0094 \u008a \u0090 \u009b \u008c \u0087 \u008c \u0087 \u008c \u008e \u0087 \u0094 \u0089 \u0087 \u0094 \u0088 \u008b \u0084 \u0085 \u008c \u0084 \u008f \u008d \u008b \u0085 \u008c \u008b \u009a \u008a \u008b \u0084 \u0087 \u0094 \u008a \u0094 \u0085 \u008b \u0097 \u008f \u008d \u0091 \u0087 \u0094 \u008f \u008d \u0089 \u0085 \u008d \u008b \u008f \u008d \u0085 \u0089 \u0095 \u0093 \u0085 \u0093 \u008f \u008c \u008b \u008c \u008a \u0094 \u0089 \u008e \u008a \u0093 \u0087 \u0090 \u0087 \u008b \u008a \u008b \u0085 \u0094 \u0085 \u0087 \u0088 \u0084 \u0086 \u008f \u008d \u0087 \u0094 \u0088 \u0094 \u008f \u0089 \u0085 \u008c \u0301 \u0327 \u00b6 z 3\u20444 \u008d \u0085 \u008c \u0093 \u008d \u0087 \u009a \u008b \u0087 \u0092 \u0085 \u0087 \u008c \u008b \u0084 \u0085 \u0098 \u008f \u008c \u008b \u0095 \u0094 \u0089 \u0085 \u008d \u0095 \u008c \u0085 \u0089 \u008a \u0094 \u0089 \u0098 \u008f \u008c \u008b \u0092 \u008a \u0090 \u0095 \u008a \u0086 \u0090 \u0085 \u0086 \u0087 \u0088 \u0089 \u008a \u008b \u008a \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008c \u008b \u0085 \u0093 \u0084 \u0094 \u0087 \u009f \u0095 \u0085 z 3\u20444 \u008d \u0085 \u008c \u0093 \u008d \u0087 \u009a \u008b \u0087 \u0092 \u0085 \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008c \u008d \u0085 \u0092 \u0085 \u008a \u0090 \u008c \u008a \u0093 \u008b \u0087 \u008f \u0094 \u008c \u008b \u0084 \u008a \u008b \u008c \u0084 \u008f \u0095 \u0090 \u0089 \u0086 \u0085 \u008b \u008a \u0091 \u0085 \u0094 \u008a \u0094 \u0089 \u009a \u008d \u008f \u0092 \u0087 \u0089 \u0085 \u008a \u008e \u008f \u0093 \u0095 \u008c \u0085 \u0089 \u008a \u0094 \u008c \u0097 \u0085 \u008d \u008b \u008f \u008a \u008c \u009a \u0085 \u0093 \u0087 \u008e \u0087 \u0093 \u009f \u0095 \u0085 \u008c \u008b \u0087 \u008f \u0094 z y \u0094 \u008f \u008b \u0084 \u0085 \u008d \u0097 \u008f \u008d \u0089 \u008c \u009c \u0087 \u0094 \u0088 \u0087 \u0092 \u0085 \u0094 \u008b \u0084 \u0085 \u0091 \u0094 \u008f \u0097 \u0094 \u009a \u008a \u008d \u008a \u0098 \u0085 \u008b \u0085 \u008d \u008c \u008a \u0094 \u0089 \u008c \u0095 \u0088 \u0088 \u0085 \u008c \u008b \u008c \u008f \u009a \u008b \u0087 \u008f \u0094 \u008c \u009c \u0087 \u008b \u0084 \u0085 \u0090 \u009a \u008c \u008b \u008f \u0089 \u0085 \u008b \u0085 \u008d \u0098 \u0087 \u0094 \u0085 \u008b \u0084 \u0085 \u0086 \u0085 \u008c \u008b \u008c \u008f \u0090 \u0095 \u008b \u0087 \u008f \u0094 \u008a \u0098 \u008f \u0094 \u0088 \u008a \u0092 \u008a \u008d \u0087 \u0085 \u008b \u009b \u008f \u008e \u0093 \u0084 \u008f \u0087 \u0093 \u0085 \u008c \u008a \u0094 \u0089 \u0098 \u0087 \u008b \u0087 \u0088 \u008a \u008b \u0085 \u008a \u008e \u0095 \u008b \u0095 \u008d \u0085 \u008d \u0087 \u008c \u0091 \u008f \u008d \u0084 \u008f \u0097 \u008b \u008f \u008b \u008a \u0091 \u0085 \u008a \u0089 \u0092 \u008a \u0094 \u008b \u008a \u0088 \u0085 \u008f \u008e \u008a \u008e \u0095 \u008b \u0095 \u008d \u0085 \u008f \u009a \u009a \u008f \u008d \u008b \u0095 \u0094 \u0087 \u008b \u009b \u0301 \u00bf \u00b6 z 3\u20444 \u008d \u0085 \u008c \u0093 \u008d \u0087 \u009a \u008b \u0087 \u0092 \u0085 \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008c \u008a \u0090 \u008c \u008f \u0087 \u0090 \u0090 \u0095 \u008c \u008b \u008d \u008a \u008b \u0085 \u008b \u0084 \u0085 \u0087 \u0098 \u009a \u0090 \u0087 \u0093 \u008a \u008b \u0087 \u008f \u0094 \u008c \u008f \u008e \u0085 \u008a \u0093 \u0084 \u0089 \u0085 \u0093 \u0087 \u008c \u0087 \u008f \u0094 \u008b \u008f \u0087 \u0098 \u009a \u008d \u008f \u0092 \u0085 \u0089 \u0085 \u0093 \u0087 \u008c \u0087 \u008f \u0094 \u0098 \u008a \u0091 \u0087 \u0094 \u0088 z \u00c0 \u008f \u0098 \u0085 \u0091 \u0085 \u009b \u009a \u008f \u0087 \u0094 \u008b \u008f \u008e \u009a \u008d \u0085 \u008c \u0093 \u008d \u0087 \u009a \u008b \u0087 \u0092 \u0085 \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008c \u0088 \u0087 \u0092 \u0085 \u0094 \u0086 \u0085 \u0090 \u008f \u0097 \u00c1 \u00b7 \u00c2 \u008f \u008d \u0097 \u008a \u008d \u0089 \u00a1 \u0090 \u008f \u008f \u0091 \u0087 \u0094 \u0088 \u00b7 1\u20442 \u009a \u008b \u0087 \u0098 \u0087  \u008a \u008b \u0087 \u008f \u0094 \u008a \u0094 \u0089 \u0089 \u0085 \u0093 \u0087 \u008c \u0087 \u008f \u0094 \u008d \u0095 \u0090 \u0085 \u008c \u008e \u008f \u008d \u008e \u0095 \u008b \u0095 \u008d \u0085 \u0085 \u0092 \u0085 \u0094 \u008b \u008c \u00b7 \u00c2 \u008f \u0093 \u0095 \u008c \u0085 \u0089 \u008f \u0094 \u008f \u009a \u008b \u0087 \u0098 \u008a \u0090 \u0089 \u0085 \u0093 \u0087 \u008c \u0087 \u008f \u0094 \u008c \u008e \u008f \u008d \u008e \u0095 \u008b \u0095 \u008d \u0085 \u008c \u0087 \u008b \u0095 \u008a \u008b \u0087 \u008f \u0094 \u008c \u00b7 \u009e \u0087 \u008c \u0093 \u008d \u0085 \u008b \u0085 \u009a \u008d \u0085 \u0089 \u0087 \u0093 \u008b \u0087 \u008f \u0094 \u008f \u008e \u0087 \u0094 \u0089 \u0087 \u0092 \u0087 \u0089 \u0095 \u008a \u0090 \u0089 \u008a \u008b \u008a \u008c \u0085 \u008b \u0098 \u0085 \u0098 \u0086 \u0085 \u008d \u008c \u0086 \u008a \u008c \u0085 \u0089 \u008f \u0094 \u008c \u0087 \u0098 \u0087 \u0090 \u008a \u008d \u0087 \u008b \u0087 \u0085 \u008c \u008a \u0094 \u0089 \u0089 \u0087 \u008e \u008e \u0085 \u008d \u0085 \u0094 \u0093 \u0085 \u008c \u0083 \u0084 \u0085 \u009a \u008a \u008b \u0084 \u008a \u0094 \u008a \u0090 \u009b \u008b \u0087 \u0093 \u008c \u0095 \u008c \u0085 \u0089 \u008b \u008f \u008a \u0094 \u008a \u0090 \u009b  \u0085 \u008b \u0084 \u0085 \u008f \u009a \u008b \u0087 \u0098 \u0087  \u0085 \u0089 \u009a \u008a \u008b \u0084 \u008e \u008d \u008f \u0098 \u008f \u0094 \u0085 \u009a \u008f \u0087 \u0094 \u008b \u008b \u008f \u008a \u0094 \u008f \u008b \u0084 \u0085 \u008d z y \u0094 \u008a \u0088 \u008d \u008a \u009a \u0084 \u0087 \u0093 \u008a \u0090 \u009a \u008f \u0087 \u0094 \u008b \u008f \u008e \u0092 \u0087 \u0085 \u0097 \u009c \u008b \u0084 \u0087 \u008c \u008a \u0094 \u008a \u0090 \u009b \u008c \u0087 \u008c \u0093 \u008a \u0094 \u0086 \u0085 \u0084 \u0085 \u0090 \u009a \u008e \u0095 \u0090 \u008b \u008f \u0089 \u0085 \u008b \u0085 \u008d \u0098 \u0087 \u0094 \u0085 \u008b \u0084 \u0085 \u008c \u0084 \u008f \u008d \u008b \u0085 \u008c \u008b \u0089 \u0087 \u008c \u008b \u008a \u0094 \u0093 \u0085 \u0086 \u0085 \u008b \u0097 \u0085 \u0085 \u0094 \u008b \u0097 \u008f \u0094 \u008f \u0089 \u0085 \u008c \u008f \u008d \u0092 \u0085 \u008d \u008b \u0087 \u0093 \u0085 \u008c \u0087 \u0094 \u008b \u0084 \u0085 \u0088 \u008d \u008a \u009a \u0084 z 1\u20442 \u0094 \u0085 \u0093 \u008a \u0094 \u0089 \u0085 \u008e \u0087 \u0094 \u0085 \u00c3 \u00c4 \u00c5 \u00c6 \u00c3 \u00c7 \u00c8 \u00c9 \u00c7 \u00ca \u00cb \u00cc \u00c7 \u00cd \u00ca \u00ce \u00cf \u00c7 \u00cd \u00d0 \u00d1 \u00cf \u00c7 \u00d2 \u00cb \u00cc \u00cb \u00c7 \u00d3 \u00cb \u00cf \u00c7 \u00d1 \u00cf \u00d4 \u00d4 \u00d5 \u00c7 \u00ce \u00d3 \u00cd \u00ca \u00ce \u00cf \u00c7 \u00d6 \u00d1 \u00cf \u00d4 \u00d7 \u00d5 \u00ca \u00ce \u00c7 \u00d8 \u00cd \u00c7 \u00c8 \u00d9 \u00ce \u00d8 \u00ce \u00ca \u00cd \u00d0 \u00da \u00db \u00da \u00ca \u00cb \u00d4 \u00da \u00dc \u00d1 \u00dd \u00d1 \u00de \u00d9 \u00df \u00e0", "keywords": [], "document_vector": [22.759767, 9.553332], "paragraphs": [{"paragraph_vector": [-128.51036, 19.890798], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/TCBB.2019.2892099"}, {"uri": "232", "title": "StrandPairsViewer: a toolkit for visualization and analysis of amino acids pairs in protein sheet structures", "timestamp": "2009", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ning ZHANG", "Shan GAO", "Guang-You DUAN", "Zhuo YANG", "Tao ZHANG", "Tao Zhang"], "summary": "Amino acids (AAs) pairs in protein sheets structures have turned out to be important in studying protein tertiary structures and protein folding pathways. Most existing software tools are sophisticated in visualizing and manipulating protein 3D structures. However, these tools are inconvenient to retrieve strands alignment information and are not so intuitive to illustrate the AAs pairs on strands. In the present study, we present a new software toolkit, called StrandPairsViewer, which can read protein structures data in PDB file and extract AAs pairs as well as peptide chain sequences, sheet structures and strand alignments. It can demonstrate AAs pairs in an intuitive color text that can be pasted into other word-processing software for further editing. The toolkit also supports aligning coding nucleic acid sequences along with the paired strands in color for finding additional folding information from nucleic acids. StrandPairsViewer also enables to make statistical analysis on AAs pairs, as well as manage user\u2019s dataset effectively. The proposed toolkit has a convenient and intuitive interface that is potentially helpful and may assist in the protein structures and protein folding study, as a useful complement to the existing visualization tools for protein 3-D structures. The proposed toolkit with a User Manual and sample data files can be accessed on the internet at http://sky.nankai.edu.cn/script/sky/english/bioinformatics/psv.zip Keywordsprotein; structure; sheet; amino acids pair; visualization", "keywords": ["toolkit", "database", "strand", "aa", "protein", "dataset", "pair", "structure", "based", "information", "user", "analysis", "visualization", "acid", "strandpairsviewer"], "document_vector": [-101.695983, 45.773147], "paragraphs": [{"paragraph_vector": [171.843307, -51.340003], "paragraph_keywords": ["database", "data", "information", "input"]}, {"paragraph_vector": [174.786422, -52.77872], "paragraph_keywords": ["pairs", "acids", "aas", "dataset"]}, {"paragraph_vector": [176.602722, -55.835548], "paragraph_keywords": ["acids", "protein", "aas", "window"]}, {"paragraph_vector": [175.361892, -52.353946], "paragraph_keywords": ["protein", "users", "toolkit", "microsoft"]}, {"paragraph_vector": [155.17926, 1.637457], "paragraph_keywords": ["user", "protein", "strandpairsviewer", "interface"]}], "content": {}, "doi": "10.1109/VISUAL.1996.568133"}, {"uri": "233", "title": "Visualization of Tunnels in Protein Molecules", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Barbora Kozl\u0131\u0301kov\u00e1"], "summary": "This paper presents two novel techniques for visualization of tunnels in complex molecules of proteins. Long-term research in the field of protein analysis proved that the reactivity of the protein molecule depends on the presence of tunnels. These structures are very important mainly in the process of finding new pharmaceuticals. Visualization of a tunnel is the next very important step after the analysis because it enables the biochemists to determine the crucial regions of the tunnel, which can have a substantial effect in the process of designing new medication. Previous methods for the visualization of tunnels define a tunnel as a set of intersecting spheres. Our approach exploits tetrahedra obtained from the process of tunnel analysis based on the Voronoi diagrams and Delaunay tetrahedrization. We proposed two novel algorithms that visualize a tunnel as a surface derived from the tetrahedra, which form the boundary constraint of the tunnel in the space of the molecule. CR Categories: I.3.7 [Computer Graphics]: Three-Dimensional Graphics and Realism;", "keywords": ["structure", "molecule", "analysis", "visualize", "detection", "set", "space", "face", "protein", "algorithm", "triangle", "approach", "chemical", "result", "figure", "tunnel", "surface", "tetrahedron", "atom", "use", "gate", "biochemist", "boundary", "visualization"], "document_vector": [-120.619308, -0.484547], "paragraphs": [{"paragraph_vector": [80.818992, -40.238788], "paragraph_keywords": ["proteins", "protein", "molecule", "tunnels"]}, {"paragraph_vector": [0.63949, -21.121252], "paragraph_keywords": ["tunnel", "molecule", "site", "tunnels"]}, {"paragraph_vector": [1.821277, -22.881576], "paragraph_keywords": ["tunnel", "tunnels", "analysis", "visualization"]}, {"paragraph_vector": [-1.807146, -20.203615], "paragraph_keywords": ["tunnel", "visualization", "tunnels", "analysis"]}, {"paragraph_vector": [4.756627, -20.394573], "paragraph_keywords": ["tunnel", "techniques", "set", "program"]}, {"paragraph_vector": [0.76275, -23.598188], "paragraph_keywords": ["tunnel", "visualization", "tunnels", "spheres"]}, {"paragraph_vector": [-2.218708, -19.677349], "paragraph_keywords": ["tunnel", "visualization", "triangle", "gates"]}, {"paragraph_vector": [-4.12799, -13.420069], "paragraph_keywords": ["tunnel", "gates", "surface", "shared"]}, {"paragraph_vector": [-0.696567, -18.377828], "paragraph_keywords": ["tunnel", "subdivision", "atoms", "surface"]}, {"paragraph_vector": [-0.680282, -21.708805], "paragraph_keywords": ["tunnel", "tunnels", "set", "visualize"]}, {"paragraph_vector": [3.043421, -17.069183], "paragraph_keywords": ["tunnel", "sampling", "box", "boundary"]}, {"paragraph_vector": [0.49928, -20.299373], "paragraph_keywords": ["tunnel", "faces", "atoms", "surface"]}, {"paragraph_vector": [0.639698, -22.169639], "paragraph_keywords": ["tunnel", "visualization", "tunnels", "approach"]}, {"paragraph_vector": [0.532022, -21.168024], "paragraph_keywords": ["tunnel", "visualization", "surface", "tunnels"]}], "content": {}, "doi": "10.1109/VISUAL.2000.885733"}, {"uri": "234", "title": "Knowledge Discovery Based on Data Analytics and Visualization Supporting Precision Medicine", "timestamp": "2020", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Veska Gancheva"], "summary": "Major challenge in the analysis of clinical data and knowledge discovery is to suggest an integrated, advanced and efficient tools, methods and technologies for access and processing of progressively increasing amounts of data in multiple formats. The problem of adapting the personal treatment to a patient is extremely complex and the medical doctor should examine and analyze large amounts of various data of different type. A comprehensive system for precision medicine, which covers all phases of data discovery, data integration, data preprocessing, building models, data storage, data analysis and visualization can be very useful to scientists in support of precision medicine. The software system aims intelligent big genomic data management, analysis and visualization and allows scientists an easy, fast and flexible approach for data processing. They can choose the services they wish to be executed, use the available data sets in databases, or enter their own data to be processed. Keywords\u2014big biomedical data analytics, data visualization, DNA sequence, knowledge discovery from data, precision medicine, protein structure", "keywords": ["display", "case", "visualization", "mouse", "list", "coordinate", "called", "data", "protein", "based", "polygon", "scene", "function", "atom", "analysis", "dna", "object"], "document_vector": [156.318023, 56.763107], "paragraphs": [{"paragraph_vector": [105.420509, 9.296837], "paragraph_keywords": ["atoms", "objects", "opengl", "function"]}, {"paragraph_vector": [122.268104, -28.44074], "paragraph_keywords": ["display", "list", "atoms", "frame"]}, {"paragraph_vector": [143.141128, -35.225498], "paragraph_keywords": ["protein", "data", "function", "atoms"]}, {"paragraph_vector": [-105.539909, -16.855829], "paragraph_keywords": ["data", "analysis", "based", "visualization"]}], "content": {}, "doi": "10.1109/VISUAL.2004.82"}, {"uri": "235", "title": "Spatiotemporal Visualization of Cell Membrane Dynamics and Protein Colocalization Reveals Correlation between Membrane Dynamics and Metastatic Invasion", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yue Hou", "Lee Cooper", "Scott Wilkinson", "Adam Marcus", "Daniel Brat"], "summary": "Invasive migration is an important cellular behavior that drives cancer metastasis, which increases the difficulty in treatment and results in poor clinical outcomes. Metastasis is comprised of a series of dynamic processes including cancer cell migration, spreading through circulation system and invasion into distant organs. However, visualization of these underlying steps of metastasis is lack. Although spatiotemporal measurements of cell motion dynamics have been well studied to identify different dynamic patterns, little work has been done to visualize how these dynamic patterns occur on migrating cells. Therefore, we develop spatiotemporal visualization tools to broaden the application of the existing dynamical cell analysis. For instance, we propose visualization techniques to classify the cell edge velocities into three states: protrusion, quiescence and retraction and then visualize when and where these three states of membrane dynamics happen on videos of migrating cells. We also create a semi-automatic tool to allow the users to select the ROIs from the correlation map and then plot the ROIs back onto the original cell migration video. These visualization tools help the biologists to better understand the abstract velocity heat maps in a user friendly way. Biologists can also take the advantage of selecting any ROI through these tools, which enables them to easily observe important migration regions. Furthermore, we investigate the correlation between cell membrane dynamics and subcellular proteins colocalization in a cancer model system and discovered a tight coupling between active membrane dynamics and periodic protein colocalizaiton. This active membrane and periodic colocalization pattern correlates with slower and less persistent migration in 3D collagen matrix. Permission to make digital or hard copies of part or all of this work for personal or classroom use is granted without fee provided that copies are not made or distributed for profit or commercial advantage and that copies bear this notice and the full citation on the first page. Copyrights for third-party components of this work must be honored. For all other uses, contact the Owner/Author. Copyright is held by the owner/author(s). BCB\u201915, September 09-12, 2015, Atlanta, GA, USA. ACM 978-1-4503-3853-0/15/09. http://dx.doi.org/10.1145/2808719.2811415.", "keywords": ["dynamic", "membrane", "colocalization", "cancer", "visualization", "cell"], "document_vector": [110.53292, 46.788429], "paragraphs": [{"paragraph_vector": [-40.73339, 1.743079], "paragraph_keywords": ["colocalization", "cell", "cancer", "membrane"]}, {"paragraph_vector": [-40.199138, -0.316893], "paragraph_keywords": ["protrusion", "pages", "recognition", "cvpr"]}], "content": {}, "doi": "10.1109/VISUAL.2004.47"}, {"uri": "236", "title": "Using of Information and Communication Technologies in Learning and Teaching of Physics in Universities: Few Parametric Model for the Dynamics of Vibrations of Diatomic Molecules", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["R. Qadmaliyev", "B.\u041c. QasimovP", "N.\u041c. QodjayevP"], "summary": "Using of Information and Communication Technologies in Education both for learning and teaching of Physics is very important component in university education today. The new approach used in Qafqaz University for learning and teaching of Physics was reported in our recent papers [1,2]. The essence of the approach is based on interactive numerical modeling in Physics. In this article we continue developing this approach considering the unified numerical model for the dynamics of diatomic molecules.", "keywords": ["potential", "molecule", "student", "atom", "mode", "calculated", "interaction"], "document_vector": [120.854972, -23.376062], "paragraphs": [{"paragraph_vector": [153.035446, -69.977943], "paragraph_keywords": ["molecule", "potential", "atoms", "calculated"]}, {"paragraph_vector": [138.095947, -72.321998], "paragraph_keywords": ["molecule", "mode", "students", "window"]}, {"paragraph_vector": [135.150115, -73.020172], "paragraph_keywords": ["molecule", "frequences", "values", "represented"]}], "content": {}, "doi": "10.1109/VISUAL.2004.103"}, {"uri": "237", "title": "Visualization techniques for molecular dynamics - IEEE Computer Graphics and Applications", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "o machine silicon surfaces economically T using diamond turning machines requires understanding materials on the atomic level. One of the simplest experimental techniques for measuring a material\u2019s atomic ductile and brittle behavior is atomic-scale indentation, in which a small tool bit (approximately 100 atoms wide) is pressed into a micron-sized ( meter) sample. We used nonequilibrium molecular dynamics to model this type of nano-indentation in pure silicon and studied the dependence of phase trans-", "keywords": [], "document_vector": [58.101451, -23.39628], "paragraphs": [{"paragraph_vector": [-129.96437, 21.023797], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/VISUAL.2004.7"}, {"uri": "238", "title": "Stylesheet Transformations for Interactive Visualization: Towards a Web3D Chemistry Curricula", "timestamp": "0", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Nicholas F. Polys"], "summary": "Recent Standards specifications offer important but underemployed techniques to maximize access-to and distribution-of information for real-time 3D visualization over the web. This paper describes and evaluates such techniques to transform structured data such as Chemical Markup Language (CML) to different forms and contexts for Web3D delivery using Extensible Stylesheet Transformations (XSLT), Extensible 3D (X3D), and VRML97. Standards design approaches offer a number of advantages: data durability, data interoperability, and an ecology of tools to be deployed for production and delivery. As we demonstrate, these techniques allow developers to port data between multiple representations and formats, to leverage the separation of the presentation (reference) from the content (referent), and the ability to define \u2018high-level\u2019 markup tags for application-specific needs. By defining a set of XSL Transformations, we are able to generate multiple views and interaction schemes with the same data set; each one \u2018personalized\u2019 for different applications and different levels of expertise.", "keywords": ["web", "consortium", "data", "group", "structure", "xml", "molecule", "transformation", "language", "xslt", "presentation", "representation", "application", "set", "encoding", "information", "number", "result", "standard", "defined", "interface", "work", "source", "scene", "tool", "geometry", "element", "cml", "atom", "type", "describing", "vrml", "content", "display", "file", "use", "software", "style", "document", "user", "visualization"], "document_vector": [176.402923, 24.30318], "paragraphs": [{"paragraph_vector": [124.117988, 5.992594], "paragraph_keywords": ["xml", "data", "web", "technologies"]}, {"paragraph_vector": [127.535469, 6.465435], "paragraph_keywords": ["data", "copies", "content", "application"]}, {"paragraph_vector": [124.931922, 7.463559], "paragraph_keywords": ["xml", "xslt", "profiles", "defined"]}, {"paragraph_vector": [126.251434, 6.026185], "paragraph_keywords": ["xml", "cml", "encoding", "tags"]}, {"paragraph_vector": [121.593429, 0.415651], "paragraph_keywords": ["data", "vrml", "describing", "web"]}, {"paragraph_vector": [126.129798, 7.395523], "paragraph_keywords": ["scene", "number", "production", "xslt"]}, {"paragraph_vector": [131.316909, 9.795667], "paragraph_keywords": ["xml", "xslt", "web", "source"]}, {"paragraph_vector": [125.27877, 6.349164], "paragraph_keywords": ["document", "nodes", "cml", "xslt"]}, {"paragraph_vector": [123.589538, 4.887899], "paragraph_keywords": ["cml", "text", "atom", "interaction"]}, {"paragraph_vector": [123.95333, 5.438406], "paragraph_keywords": ["molecule", "cml", "atom", "information"]}, {"paragraph_vector": [123.954193, 6.64496], "paragraph_keywords": ["bond", "xslt", "users", "view"]}, {"paragraph_vector": [123.215148, 5.550312], "paragraph_keywords": ["xml", "files", "user", "xslt"]}, {"paragraph_vector": [126.432678, 5.967042], "paragraph_keywords": ["cml", "xml", "user", "chemical"]}, {"paragraph_vector": [125.194061, 6.170801], "paragraph_keywords": ["xslt", "xml", "vrml", "consortium"]}], "content": {}, "doi": "10.1109/CCGrid.2014.92"}, {"uri": "239", "title": "Visualization of Electrostatic Dipoles in Molecular Dynamics of Metal Oxides", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sebastian Grottel", "Philipp Beck", "Christoph M\u00fcller", "Guido Reina", "Johannes Roth", "Hans-Rainer Trebin", "Thomas Ertl"], "summary": "Metal oxides are important for many technical applications. For example alumina (aluminum oxide) is the most commonlyused ceramic in microelectronic devices thanks to its excellent properties. Experimental studies of these materials are increasingly supplemented with computer simulations. Molecular dynamics (MD) simulations can reproduce the material behavior very well and are now reaching time scales relevant for interesting processes like crack propagation. In this work we focus on the visualization of induced electric dipole moments on oxygen atoms in crack propagation simulations. The straightforward visualization using glyphs for the individual atoms, simple shapes like spheres or arrows, is insufficient for providing information about the data set as a whole. As our contribution we show for the first time that fractional anisotropy values computed from the local neighborhood of individual atoms of MD simulation data depict important information about relevant properties of the field of induced electric dipole moments. Iso surfaces in the field of fractional anisotropy as well as adjustments of the glyph representation allow the user to identify regions of correlated orientation. We present novel and relevant findings for the application domain resulting from these visualizations, like the influence of mechanical forces on the electrostatic properties.", "keywords": ["fa", "data", "structure", "md", "propagation", "oxygen", "orientation", "application", "simulation", "set", "particle", "strain", "dipole", "research", "approach", "result", "fig", "color", "alumina", "scientist", "surface", "based", "direction", "work", "j", "atom", "crack", "field", "glyph", "cf", "region", "time", "moment", "visualization", "behavior"], "document_vector": [55.279354, -32.784313], "paragraphs": [{"paragraph_vector": [53.269947, -12.4323], "paragraph_keywords": ["visualization", "university", "alumina", "properties"]}, {"paragraph_vector": [69.837776, -3.468256], "paragraph_keywords": ["data", "oxygen", "orientations", "scientists"]}, {"paragraph_vector": [67.458671, -2.162029], "paragraph_keywords": ["visualization", "glyphs", "atoms", "simulations"]}, {"paragraph_vector": [59.324638, 3.988874], "paragraph_keywords": ["glyphs", "data", "dti", "orientation"]}, {"paragraph_vector": [68.870918, 24.047327], "paragraph_keywords": ["particle", "md", "parameter", "field"]}, {"paragraph_vector": [106.148414, -60.189933], "paragraph_keywords": ["dipole", "field", "uel", "visualization"]}, {"paragraph_vector": [70.231826, -5.079363], "paragraph_keywords": ["atoms", "orientation", "visualization", "data"]}, {"paragraph_vector": [91.19358, 6.960146], "paragraph_keywords": ["glyphs", "visualization", "based", "field"]}, {"paragraph_vector": [63.53046, 0.177134], "paragraph_keywords": ["data", "based", "field", "fa"]}, {"paragraph_vector": [23.164785, 24.56776], "paragraph_keywords": ["data", "j", "atoms", "atom"]}, {"paragraph_vector": [51.385246, 10.556094], "paragraph_keywords": ["fa", "regions", "values", "fig"]}, {"paragraph_vector": [77.270751, -1.235759], "paragraph_keywords": ["data", "sets", "visualization", "time"]}, {"paragraph_vector": [61.837123, -1.645339], "paragraph_keywords": ["dipole", "moments", "j", "simulation"]}, {"paragraph_vector": [66.713729, -2.799942], "paragraph_keywords": ["crack", "orientation", "cf", "surfaces"]}, {"paragraph_vector": [66.289649, -2.466185], "paragraph_keywords": ["crack", "moments", "dipole", "alumina"]}, {"paragraph_vector": [66.711784, -2.836887], "paragraph_keywords": ["crack", "dipoles", "surfaces", "inserted"]}, {"paragraph_vector": [65.403373, -4.321614], "paragraph_keywords": ["crack", "dipole", "simulations", "disorder"]}, {"paragraph_vector": [66.330818, -3.895246], "paragraph_keywords": ["crack", "field", "simulation", "disorder"]}, {"paragraph_vector": [68.999465, -1.790351], "paragraph_keywords": ["sfb", "research", "data", "visualization"]}], "content": {}, "doi": "10.1109/TCBB.2008.36"}, {"uri": "240", "title": "Lightweight Computational Steering of Very Large Scale Molecular Dynamics Simulations", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["David M. Beazley", "Peter S. Lomdahl"], "summary": "We present a computational steering approach for controlling, analyzing, and visualizing very large scale molecular dynamics simulations involving tens to hundreds of millions of atoms. Our approach relies on extensible scripting languages and an easy to use tool for building extensions and modules. The system is easy to modify, works with existing C code, is memory efficient, and can be used from inexpensive workstations over standard Internet connections. We demonstrate how we have been able to explore data from production MD simulations involving as many as 104 million atoms running on the CM-5 and Cray T3D. We also show how this approach can be used to integrate common scripting languages (including Python, Tcl/Tk, and Perl), simulation code, user extensions, and commercial data analysis packages.", "keywords": ["data", "spasm", "md", "language", "analysis", "scripting", "simulation", "computing", "set", "processor", "memory", "particle", "approach", "run", "scale", "system", "c", "code", "workstation", "interface", "work", "atom", "command", "file", "use", "steering", "feel", "time", "build", "image", "user", "function", "visualization"], "document_vector": [-33.605091, -2.261197], "paragraphs": [{"paragraph_vector": [-138.06044, 53.11536], "paragraph_keywords": ["simulations", "atoms", "simulation", "dynamics"]}, {"paragraph_vector": [-134.018508, 53.729667], "paragraph_keywords": ["spasm", "simulations", "simulation", "code"]}, {"paragraph_vector": [142.681549, 54.065975], "paragraph_keywords": ["data", "analysis", "system", "users"]}, {"paragraph_vector": [-134.514785, 54.740333], "paragraph_keywords": ["data", "problem", "simulation", "analysis"]}, {"paragraph_vector": [-132.891571, 52.998413], "paragraph_keywords": ["user", "system", "simulations", "steering"]}, {"paragraph_vector": [-132.040725, 52.570529], "paragraph_keywords": ["system", "approach", "simulation", "code"]}, {"paragraph_vector": [-150.665328, 47.283317], "paragraph_keywords": ["language", "functions", "scripting", "commands"]}, {"paragraph_vector": [-121.016593, 44.041431], "paragraph_keywords": ["interface", "c", "user", "functions"]}, {"paragraph_vector": [-122.633888, 51.038803], "paragraph_keywords": ["user", "particle", "interface", "code"]}, {"paragraph_vector": [-118.523994, 47.295063], "paragraph_keywords": ["code", "functions", "particles", "scripting"]}, {"paragraph_vector": [-121.555427, 44.541732], "paragraph_keywords": ["spasm", "set", "data", "code"]}, {"paragraph_vector": [134.9319, 45.273025], "paragraph_keywords": ["data", "simulation", "visualize", "spasm"]}, {"paragraph_vector": [-118.032005, 45.610794], "paragraph_keywords": ["spasm", "simulation", "workstation", "mbytes"]}, {"paragraph_vector": [-136.157928, 53.030105], "paragraph_keywords": ["analysis", "visualization", "datafiles", "data"]}, {"paragraph_vector": [-121.56842, 46.592155], "paragraph_keywords": ["approach", "works", "scripting", "spasm"]}, {"paragraph_vector": [-138.017257, 53.26411], "paragraph_keywords": ["computing", "work", "university", "data"]}, {"paragraph_vector": [-144.363265, 52.563686], "paragraph_keywords": ["condensed", "physics", "matter", "materials"]}], "content": {}, "doi": "10.1109/TCBB.2007.70251"}, {"uri": "241", "title": "Using Flow-Visualization for Studying Sub-Molecular Motions", "timestamp": "0", "rating": "0.0", "annotation": "", "tags": [], "authors": ["William C. Ray", "Abhilash Mohan", "Jeffrey Bartlett"], "summary": "Visualization of molecular structures is important for understanding basic molecular conformations and because structure is intimately tied to function. Molecular function however is also associated with molecular flexibility. The Adenylate Kinase lid hinges to trap its target in the active site. The Cyclic-AMP Response Protein hinges like a clothes-pin to clamp different DNA sequences. The AdenoAssociated Virus (AAV) capsid protein appears to unfold to expose a catalytic domain that allows the virus to escape, when presented with the low pH of the normally entrapping endosome.", "keywords": ["change", "protein", "backbone", "motion", "domain", "atom", "ph"], "document_vector": [-151.178588, -8.286987], "paragraphs": [{"paragraph_vector": [150.890731, -60.383037], "paragraph_keywords": ["change", "atoms", "protein", "motion"]}, {"paragraph_vector": [65.708915, -52.308635], "paragraph_keywords": ["ph", "backbone", "motions", "atoms"]}], "content": {}, "doi": "10.1109/TCBB.2013.17"}, {"uri": "242", "title": "An Architecture for Visualisation and Interactive Analysis of Proteins", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Philip McDermott", "James Sinnott", "Dave Thorne", "Steve Pettifer", "Terri Attwood"], "summary": "Data sets in the biological domain are often semantically complex and difficult to integrate and visualise. Converting between the file formats required by interactive analysis tools and those used by the global databases is a costly and error prone process. This paper describes a data model designed to enable efficient rendering of and interaction with biological data, and two demonstrator applications from different fields of protein analysis that provide co-ordinated views of data held in the underlying model. Keywords\u2014 Bioinformatics, Protein Analysis, Collaboration, Visualization, Representation.", "keywords": ["cinema", "web", "visualisation", "data", "alignment", "structure", "api", "associated", "concept", "analysis", "detail", "application", "represent", "set", "space", "database", "protein", "sequence", "information", "result", "service", "annotation", "technique", "example", "source", "ambrosia", "tool", "atom", "-", "bioinformatics", "model", "use", "form", "utopia", "view", "user", "rendering", "provide", "access"], "document_vector": [175.528549, 28.035772], "paragraphs": [{"paragraph_vector": [-142.135055, -50.825515], "paragraph_keywords": ["data", "information", "generate", "techniques"]}, {"paragraph_vector": [159.736709, -38.216773], "paragraph_keywords": ["data", "model", "tools", "analysis"]}, {"paragraph_vector": [-176.512542, -55.304569], "paragraph_keywords": ["data", "protein", "databases", "characteristics"]}, {"paragraph_vector": [-163.687606, -49.000053], "paragraph_keywords": ["data", "aim", "ontology", "effort"]}, {"paragraph_vector": [178.431564, -43.36322], "paragraph_keywords": ["sequence", "results", "data", "based"]}, {"paragraph_vector": [179.584106, -46.496227], "paragraph_keywords": ["structure", "data", "concepts", "ontology"]}, {"paragraph_vector": [-172.454986, -43.960422], "paragraph_keywords": ["data", "model", "represent", "information"]}, {"paragraph_vector": [161.895507, -33.218009], "paragraph_keywords": ["data", "utopia", "information", "file"]}, {"paragraph_vector": [168.604492, -27.296594], "paragraph_keywords": ["data", "access", "model", "provide"]}, {"paragraph_vector": [163.137222, -35.510803], "paragraph_keywords": ["data", "annotation", "model", "visualisation"]}, {"paragraph_vector": [154.741592, -43.6021], "paragraph_keywords": ["sequence", "cinema", "shown", "alignment"]}, {"paragraph_vector": [136.542694, -32.967411], "paragraph_keywords": ["data", "atoms", "graphics", "annotation"]}, {"paragraph_vector": [141.281982, -33.499431], "paragraph_keywords": ["ambrosia", "visualisation", "structure", "scheme"]}, {"paragraph_vector": [168.895492, -44.63269], "paragraph_keywords": ["data", "structure", "sequence", "applications"]}, {"paragraph_vector": [154.621307, 15.856346], "paragraph_keywords": ["data", "sequence", "ambrosia", "cinema"]}, {"paragraph_vector": [167.865417, -38.4258], "paragraph_keywords": ["data", "sources", "services", "utopia"]}, {"paragraph_vector": [162.741363, 4.002152], "paragraph_keywords": ["providing", "data", "tools", "model"]}], "content": {}, "doi": "10.1109/tcbb.2007.1035"}, {"uri": "243", "title": "Multiscale Visual Drilldown for the Analysis of Large Ensembles of Multi-Body Protein Complexes", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Katar\u0131\u0301na Furmanov\u00e1", "Adam Jur\u010d\u0131\u0301k", "Barbora Kozl\u0131\u0301kov\u00e1", "Helwig Hauser", "Jan By\u0161ka"], "summary": "When studying multi-body protein complexes, biochemists use computational tools that can suggest hundreds or thousands of their possible spatial configurations. However, it is not feasible to experimentally verify more than only a very small subset of them. In this paper, we propose a novel multiscale visual drilldown approach that was designed in tight collaboration with proteomic experts, enabling a systematic exploration of the configuration space. Our approach takes advantage of the hierarchical structure of the data \u2013 from the whole ensemble of protein complex configurations to the individual configurations, their contact interfaces, and the interacting amino acids. Our new solution is based on interactively linked 2D and 3D views for individual hierarchy levels. At each level, we offer a set of selection and filtering operations that enable the user to narrow down the number of configurations that need to be manually scrutinized. Furthermore, we offer a dedicated filter interface, which provides the users with an overview of the applied filtering operations and enables them to examine their impact on the explored ensemble. This way, we maintain the history of the exploration process and thus enable the user to return to an earlier point of the exploration. We demonstrate the effectiveness of our approach on two case studies conducted by collaborating proteomic experts.", "keywords": ["data", "structure", "exploration", "analysis", "acid", "selection", "ensemble", "pa", "protein", "contact", "solution", "number", "selected", "figure", "filter", "cc", "density", "process", "based", "complex", "level", "tool", "configuration", "scoring", "filtering", "expert", "use", "docking", "overview", "pair", "view", "t", "time", "domain", "property", "user"], "document_vector": [-131.405349, 34.500267], "paragraphs": [{"paragraph_vector": [-20.681726, -62.419212], "paragraph_keywords": ["protein", "complex", "function", "e"]}, {"paragraph_vector": [-41.858715, -20.894365], "paragraph_keywords": ["tools", "protein", "scoring", "data"]}, {"paragraph_vector": [-41.4202, -18.706354], "paragraph_keywords": ["protein", "experts", "figure", "process"]}, {"paragraph_vector": [-39.356685, -20.297885], "paragraph_keywords": ["protein", "properties", "based", "configurations"]}, {"paragraph_vector": [-43.505008, -20.463161], "paragraph_keywords": ["domain", "experts", "properties", "exploration"]}, {"paragraph_vector": [-38.868541, -21.209566], "paragraph_keywords": ["protein", "solutions", "complexes", "docking"]}, {"paragraph_vector": [-36.787132, -28.17732], "paragraph_keywords": ["docking", "tools", "complexes", "protein"]}, {"paragraph_vector": [-42.768791, -20.706275], "paragraph_keywords": ["ensembles", "ccs", "levels", "ensemble"]}, {"paragraph_vector": [-47.672969, -19.211187], "paragraph_keywords": ["configurations", "user", "operations", "filtering"]}, {"paragraph_vector": [-48.542976, -16.827299], "paragraph_keywords": ["filters", "items", "range", "operation"]}, {"paragraph_vector": [-44.854888, -19.023056], "paragraph_keywords": ["hierarchy", "aap", "selection", "nodes"]}, {"paragraph_vector": [-44.346477, -18.518707], "paragraph_keywords": ["protein", "bar", "contact", "interfaces"]}, {"paragraph_vector": [-43.291229, -18.791301], "paragraph_keywords": ["protein", "configurations", "properties", "users"]}, {"paragraph_vector": [-43.081554, -21.699666], "paragraph_keywords": ["protein", "acids", "view", "interactions"]}, {"paragraph_vector": [-45.823261, -19.178361], "paragraph_keywords": ["protein", "matrix", "acid", "acids"]}, {"paragraph_vector": [-41.246978, -19.874282], "paragraph_keywords": ["view", "configurations", "ppcs", "protein"]}, {"paragraph_vector": [-45.521858, -22.969331], "paragraph_keywords": ["density", "based", "proteins", "ccs"]}, {"paragraph_vector": [-46.496078, -18.625444], "paragraph_keywords": ["filtered", "filter", "view", "matrix"]}, {"paragraph_vector": [-46.828639, -18.793889], "paragraph_keywords": ["users", "ensemble", "data", "representation"]}, {"paragraph_vector": [-45.664794, -19.914642], "paragraph_keywords": ["protein", "selection", "tool", "t"]}, {"paragraph_vector": [-38.752395, -23.399312], "paragraph_keywords": ["t", "pa", "docking", "domain"]}, {"paragraph_vector": [-40.828632, -20.872175], "paragraph_keywords": ["configurations", "t", "lrp", "pa"]}, {"paragraph_vector": [-40.871997, -21.408761], "paragraph_keywords": ["lrp", "configurations", "acid", "figure"]}, {"paragraph_vector": [-42.624866, -18.912435], "paragraph_keywords": ["configurations", "t", "configuration", "selected"]}, {"paragraph_vector": [-40.101348, -21.973537], "paragraph_keywords": ["ensemble", "solution", "configurations", "configuration"]}, {"paragraph_vector": [-45.177566, -18.816394], "paragraph_keywords": ["overview", "protein", "graph", "tool"]}, {"paragraph_vector": [-45.619178, -20.904079], "paragraph_keywords": ["filters", "filter", "need", "acid"]}, {"paragraph_vector": [-44.126068, -20.211114], "paragraph_keywords": ["number", "data", "proteins", "use"]}, {"paragraph_vector": [-43.791202, -19.532356], "paragraph_keywords": ["protein", "based", "use", "scoring"]}, {"paragraph_vector": [-156.6595, 9.146088], "paragraph_keywords": ["mmiv", "university", "bergen"]}], "content": {}, "doi": "10.1145/1050491.1050512"}, {"uri": "244", "title": "Visualization and Analytics of Biological Data by Using Different Tools and Techniques", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "The importance of graph analytics cannot be undermined. It has always been a question for the researcher that how to deal with dense graphs. The visual graph analytics is one of the best sources for creating its remarkable, distinct impact in the field of data science. The graph analytics and big data has fascinated a wide range of attention from the researchers and scientist from all over the world. By using the most advanced tools for the graph, the analytics can lead most useful and productive results in various domains which include life sciences, business, computer sciences, engineering and so on. Biological data can be represented in interpretable form when exposed to graph analytic tools, which may lead to meaningful insights. This paper is aimed at the visualization of the graph with two different techniques. Various procedures were used in this research such as the collection of datasets from heterogeneous biological data sources, data integration, and formation of the new dataset (MYBIOGRID). Designing queries in Neo4j using Cypher Query Language to visualize MYBIOGRID and to determine the relationship using the property graph model. In the next step the uploading data to CIRCOS is performed and visualization of motif similarity is done. The result from this study indicates that visualization of similarity matrix of repetitive patterns thus representing the most similar and least similar patterns in the sequence. Graph databases play a vital role in graph analytics but in memory storage makes analysis very time consuming if the massive amount of data sets is to be processed. Each tool has its specific parameters, which make it a good candidate for analysis and", "keywords": ["circos", "represents", "relationship", "data", "analytics", "centrality", "based", "network", ".", "al", "tool", "analysis", "degree", "visualization", "configuration", "graph", "gene"], "document_vector": [73.584884, 67.315612], "paragraphs": [{"paragraph_vector": [-117.719535, -21.695861], "paragraph_keywords": ["represents", "data", "coordinates", "centrality"]}, {"paragraph_vector": [-119.182968, -13.836219], "paragraph_keywords": ["represents", "degree", "gene", "genes"]}, {"paragraph_vector": [-116.150337, -25.291496], "paragraph_keywords": ["data", "graph", "analysis", "cql"]}, {"paragraph_vector": [-114.770034, -20.638603], "paragraph_keywords": ["data", "analytics", "graph", "visualization"]}, {"paragraph_vector": [-141.477096, -33.339771], "paragraph_keywords": ["visualization", "network", ".", "al"]}], "content": {}, "doi": "10.1145/2535571.2535595"}, {"uri": "245", "title": "Multi-species Protein Function Prediction: Towards Web-based Visual Analytics", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Paolo Perlasca", "Giorgio Valentini", "Marco Frasca", "Marco Mesiti"], "summary": "The visualization and analysis of big bio-molecular networks is a key feature for the investigation and prediction of protein functions in a multi-species context. In this paper we present the design of a system that integrates data management, machine learning and visualization facilities to make effective the visual analysis of big networks by means of webbased interfaces.", "keywords": ["machine", "web", "data", "application", "graph", "gene", "target", "protein", "memory", "algorithm", "network", "approach", "system", "integration", "computation", "prediction", "adopted", "implementation", "based", "source", "learning", "edge", "use", "requires", "vertex", "function", "visualization", "node"], "document_vector": [49.382965, 62.172733], "paragraphs": [{"paragraph_vector": [-106.782936, -23.838315], "paragraph_keywords": ["graphs", "copies", "proteins", "automated"]}, {"paragraph_vector": [-108.070396, -24.71696], "paragraph_keywords": ["organization", "data", "visualization", "graph"]}, {"paragraph_vector": [-104.709129, -21.455247], "paragraph_keywords": ["solution", "algorithms", "system", "based"]}, {"paragraph_vector": [-106.233535, -21.974308], "paragraph_keywords": ["data", "system", "sources", "databases"]}, {"paragraph_vector": [-107.922523, -18.834877], "paragraph_keywords": ["graph", "memory", "based", "network"]}, {"paragraph_vector": [-104.934265, -20.496528], "paragraph_keywords": ["network", "algorithms", "implementation", "networks"]}, {"paragraph_vector": [-129.562591, -38.791561], "paragraph_keywords": ["network", "target", "visualization", "vertex"]}, {"paragraph_vector": [-62.827159, -28.685802], "paragraph_keywords": ["bubbles", "target", "visualization", "nodes"]}, {"paragraph_vector": [-106.496505, -21.683206], "paragraph_keywords": ["network", "based", "graph", "prediction"]}, {"paragraph_vector": [-111.015655, -20.328369], "paragraph_keywords": ["based", "graph", "protein", "learning"]}, {"paragraph_vector": [-99.394119, -31.348983], "paragraph_keywords": ["network", "gene", "based", "integration"]}], "content": {}, "doi": "10.1145/1394281.1394312"}, {"uri": "246", "title": "Visual exploration of large normal mode spaces to study protein flexibility", "timestamp": "2020", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Pierre Bedoucha", "Nathalie Reuter", "Helwig Hauser"], "summary": "When studying the function of proteins, biochemists utilize normal mode decomposition to enable the analysis of structural changes on time scales that are too long for molecular dynamics simulation. Such a decomposition yields a high-dimensional parameter space that is too large to be analyzed exhaustively. We present a novel approach to reducing and exploring this vast space through the means of interactive visualization. Our approach enables the inference of relevant protein function from single structure dynamics through protein tunnel analysis while considering normal mode combinations spanning the whole normal modes space. Our solution, based on multiple linked 2D and 3D views, enables the quick and flexible exploration of individual modes and their effect on the dynamics of tunnels with relevance for the protein function. Once an interesting motion is identified, the exploration of possible normal mode combinations is steered via a visualization-based recommendation system. This helps to quickly identify a narrow, yet relevant set of normal modes that can be investigated in detail. Our solution is the result of close cooperation between visualization and the domain. The versatility and efficiency of our approach are demonstrated in two case studies. \u00a9 2020 Published by Elsevier Ltd.", "keywords": [], "document_vector": [-114.268714, 21.192258], "paragraphs": [{"paragraph_vector": [-130.506484, 21.927385], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/2466533.2466558"}, {"uri": "247", "title": "Coupling LAMMPS and the vl3 Framework for Co-Visualization of Atomistic Simulations", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Silvio Rizzi", "Mark Hereld", "Joseph Insley", "Preeti Malakar", "Michael E. Papka", "Thomas Uram", "Venkatram Vishwanath"], "summary": "In this work we perform in-situ visualization of molecular dynamics simulations, which can help scientists to visualize simulation output on-the-fly, without incurring storage overheads. We present a case study to couple LAMMPS, the large-scale molecular dynamics simulation code with vl3, our parallel framework for large-scale visualization and analysis. Our motivation is to identify effective approaches for covisualization and exploration of large-scale atomistic simulations at interactive frame rates. We propose a system of coupled libraries and describe its architecture, with an implementation that runs on GPU-based clusters. We present the results of strong and weak scalability experiments, as well as future research avenues based on our results. Keywords-molecular dynamics simulations, large-scale visualization, co-visualization, in-situ visualization", "keywords": ["data", "simulation", "computing", "point", "number", "scale", "system", "compositing", "mpi", "icet", "lammps", "atom", "show", "use", "time", "image", "rendering", "visualization", "scalability", "rank", "node"], "document_vector": [-1.965753, -48.780101], "paragraphs": [{"paragraph_vector": [-179.213653, 51.719284], "paragraph_keywords": ["scale", "visualization", "lammps", "imd"]}, {"paragraph_vector": [155.618804, 67.736419], "paragraph_keywords": ["lammps", "atoms", "point", "mode"]}, {"paragraph_vector": [171.349411, 59.898796], "paragraph_keywords": ["lammps", "node", "mpi", "use"]}, {"paragraph_vector": [164.188018, 66.373138], "paragraph_keywords": ["lammps", "ranks", "time", "node"]}, {"paragraph_vector": [160.176116, 67.706245], "paragraph_keywords": ["frame", "time", "experiment", "ranks"]}, {"paragraph_vector": [176.806152, 61.482337], "paragraph_keywords": ["visualization", "computing", "leadership", "research"]}, {"paragraph_vector": [140.662658, 54.884532], "paragraph_keywords": ["visualization", "proceedings", "lammps", "scale"]}], "content": {}, "doi": "10.1145/3154353.3154360"}, {"uri": "248", "title": "Lightness Constancy in Surface Visualization", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Danielle Albers"], "summary": "Color is a common channel for displaying data in surface visualization, but is affected by the shadows and shading used to convey surface depth and shape. Understanding encoded data in the context of surface structure is critical for effective analysis in a variety of domains, such as in molecular biology. In the physical world, lightness constancy allows people to accurately perceive shadowed colors; however, its effectiveness in complex synthetic environments such as surface visualizations is not well understood. We report a series of crowdsourced and laboratory studies that confirm the existence of lightness constancy effects for molecular surface visualizations using ambient occlusion. We provide empirical evidence of how common visualization design decisions can impact viewers\u2019 abilities to accurately identify encoded surface colors. These findings suggest that lightness constancy aids in understanding color encodings in surface visualization and reveal a correlation between visualization techniques that improve color interpretation in shadow and those that enhance perceptions of surface depth. These results collectively suggest that understanding constancy in practice can inform effective visualization design.", "keywords": ["perception", "data", "cue", "improve", "variation", "lighting", "shading", "number", "viewing", "result", "stimulus", "occlusion", "stereo", "depth", "color", "constancy", "shadow", "surface", "measure", "work", "level", "study", "performance", "design", "experiment", "contrast", "effect", "model", "use", "condition", "luminance", "patch", "participant", "image", "ramp", "lightness", "visualization", "shape", "viewer"], "document_vector": [-166.88388, -37.063419], "paragraphs": [{"paragraph_vector": [-21.604648, -2.474197], "paragraph_keywords": ["color", "surface", "data", "visualization"]}, {"paragraph_vector": [-23.093826, -0.763841], "paragraph_keywords": ["surface", "visualization", "constancy", "results"]}, {"paragraph_vector": [-24.734025, 2.808985], "paragraph_keywords": ["constancy", "lighting", "data", "surface"]}, {"paragraph_vector": [-23.657369, 1.64691], "paragraph_keywords": ["constancy", "lightness", "lighting", "illumination"]}, {"paragraph_vector": [-1.602561, -43.930042], "paragraph_keywords": ["surface", "surfaces", "data", "visualization"]}, {"paragraph_vector": [-21.750516, -1.128566], "paragraph_keywords": ["surface", "depth", "color", "design"]}, {"paragraph_vector": [-21.909126, 0.319639], "paragraph_keywords": ["surface", "data", "viewers", "rendering"]}, {"paragraph_vector": [-22.417661, 1.152076], "paragraph_keywords": ["surface", "visualization", "color", "occlusion"]}, {"paragraph_vector": [-22.662874, 0.864525], "paragraph_keywords": ["color", "design", "surface", "participants"]}, {"paragraph_vector": [-26.102968, 3.494259], "paragraph_keywords": ["surface", "color", "surfaces", "images"]}, {"paragraph_vector": [-22.960523, 2.371102], "paragraph_keywords": ["stimuli", "participants", "colors", "selected"]}, {"paragraph_vector": [-21.583465, 0.012253], "paragraph_keywords": ["participants", "conditions", "number", "stereo"]}, {"paragraph_vector": [-23.102842, 1.68507], "paragraph_keywords": ["participants", "color", "colors", "experiment"]}, {"paragraph_vector": [-22.545, 0.34099], "paragraph_keywords": ["participants", "color", "constancy", "colors"]}, {"paragraph_vector": [-22.849533, -0.633083], "paragraph_keywords": ["contrast", "color", "colors", "patch"]}, {"paragraph_vector": [-20.854015, 2.039434], "paragraph_keywords": ["participants", "color", "shadow", "background"]}, {"paragraph_vector": [-26.282283, -0.907302], "paragraph_keywords": ["shadows", "conditions", "surface", "shadow"]}, {"paragraph_vector": [-23.661308, 0.520665], "paragraph_keywords": ["color", "performance", "luminance", "surface"]}, {"paragraph_vector": [-23.913749, 1.297067], "paragraph_keywords": ["ramps", "ramp", "luminance", "colors"]}, {"paragraph_vector": [-23.564397, 0.9047], "paragraph_keywords": ["ramps", "color", "surface", "luminance"]}, {"paragraph_vector": [-23.397041, 0.536047], "paragraph_keywords": ["lighting", "depth", "occlusion", "surface"]}, {"paragraph_vector": [-23.381023, 2.698733], "paragraph_keywords": ["lighting", "shading", "occlusion", "collection"]}, {"paragraph_vector": [-22.312437, 2.866981], "paragraph_keywords": ["stereo", "color", "depth", "lighting"]}, {"paragraph_vector": [-23.716581, 2.470857], "paragraph_keywords": ["stereo", "viewing", "participants", "performance"]}, {"paragraph_vector": [-24.412933, 0.242116], "paragraph_keywords": ["surface", "shadow", "contours", "performance"]}, {"paragraph_vector": [-23.036739, 0.447309], "paragraph_keywords": ["surface", "color", "depth", "improve"]}, {"paragraph_vector": [-21.585414, -1.010411], "paragraph_keywords": ["surfaces", "surface", "design", "visualization"]}, {"paragraph_vector": [-23.050193, -0.125941], "paragraph_keywords": ["surface", "visualization", "color", "constancy"]}, {"paragraph_vector": [-156.340316, 14.495552], "paragraph_keywords": ["awards", "nih", "award"]}], "content": {}, "doi": "10.1145/2775292.2775307"}, {"uri": "249", "title": "Modeling Material Behavior: Molecular Dynamics Simulation and Visualization", "timestamp": "1999", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Roman Durikovi", "Teruaki Motooka"], "summary": "We have developed atomic-scale material models capable of melting, crystallization and amorphization. These deformable models feature molecular dynamics governed by Langevin equations of motion in which particle interact through attractive covalent forces and short-range repulsion forces. Also, we present an interactive virtual visualization tool for the simulation of atomic scale material behavior. An application of this research is to understand the processes that can control the quality of a single-crystal Si grown from the melt.", "keywords": ["structure", "parameter", "simulation", "dynamic", "silicon", "particle", "system", "distribution", "atom", "model", "use", "time", "t", "function", "force", "visualization", "crystal", "modeling", "di"], "document_vector": [51.047275, -39.780132], "paragraphs": [{"paragraph_vector": [51.812602, -18.524412], "paragraph_keywords": ["crystal", "structure", "modeling", "atoms"]}, {"paragraph_vector": [52.514797, -17.521978], "paragraph_keywords": ["forces", "models", "behavior", "state"]}, {"paragraph_vector": [51.553836, -18.788438], "paragraph_keywords": ["particles", "force", "ijk", "particle"]}, {"paragraph_vector": [50.832538, -20.407939], "paragraph_keywords": ["silicon", "force", "terso", "method"]}, {"paragraph_vector": [95.016571, -53.038776], "paragraph_keywords": ["time", "t", "parameters", "simulation"]}, {"paragraph_vector": [86.950119, 43.496547], "paragraph_keywords": ["use", "atoms", "distribution", "view"]}, {"paragraph_vector": [49.984905, -18.166399], "paragraph_keywords": ["simulation", "visualization", "atoms", "temperature"]}, {"paragraph_vector": [49.999458, -18.68279], "paragraph_keywords": ["silicon", "scale", "atoms", "research"]}], "content": {}, "doi": "10.1145/1179622.1179655"}, {"uri": "250", "title": "Illustrative Visualization of Molecular Reactions using Omniscient Intelligence and Passive Agents", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["M. Le Muzic", "J. Parulek", "A.K. Stavrum", "I. Viola"], "summary": "In this paper we propose a new type of a particle systems, tailored for illustrative visualization purposes, in particular for visualizing molecular reactions in biological networks. Previous visualizations of biochemical processes were exploiting the results of agent-based modeling. Such modeling aims at reproducing accurately the stochastic nature of molecular interactions. However, it is impossible to expect events of interest happening at a certain time and location, which is impractical for storytelling. To obtain the means of controlling molecular interactions, we propose to govern passive agents with an omniscient intelligence, instead of giving to the agents the freedom of initiating reaction autonomously. This makes it possible to generate illustrative animated stories that communicate the functioning of the molecular machinery. The rendering performance delivers for interactive framerates of massive amounts of data, based on the dynamic tessellation capabilities of modern graphics cards. Finally, we report an informal expert feedback we obtained from the potential users.", "keywords": ["tessellation", "buffer", "mean", "data", "structure", "motion", "allows", "molecule", "d", "order", "event", "simulation", "oi", "particle", "agent", "information", "number", "approach", "system", "biology", "reaction", "produce", "process", "technique", "pathway", "shader", "based", "work", "scene", "element", "atom", "animation", "type", "gpu", "given", "time", "rendering", "visualization", "provide", "according"], "document_vector": [-84.817863, -37.132705], "paragraphs": [{"paragraph_vector": [114.73455, -11.944618], "paragraph_keywords": ["processes", "visualizations", "animation", "reactions"]}, {"paragraph_vector": [102.220916, -5.891548], "paragraph_keywords": ["process", "visualization", "technique", "metabolic"]}, {"paragraph_vector": [102.065589, -5.267931], "paragraph_keywords": ["based", "visualization", "agent", "particle"]}, {"paragraph_vector": [100.541175, -5.062759], "paragraph_keywords": ["representations", "rendering", "visualization", "vdw"]}, {"paragraph_vector": [75.596565, 36.231079], "paragraph_keywords": ["grid", "atoms", "casting", "technique"]}, {"paragraph_vector": [101.913482, -0.841015], "paragraph_keywords": ["reaction", "system", "events", "agents"]}, {"paragraph_vector": [114.698867, -13.998922], "paragraph_keywords": ["reaction", "motion", "oi", "simulation"]}, {"paragraph_vector": [99.308044, -4.447426], "paragraph_keywords": ["reaction", "reactions", "focus", "elements"]}, {"paragraph_vector": [87.876495, 12.889842], "paragraph_keywords": ["gpu", "d", "type", "system"]}, {"paragraph_vector": [78.614524, -35.090248], "paragraph_keywords": ["reaction", "type", "scene", "buffers"]}, {"paragraph_vector": [81.54055, 17.284034], "paragraph_keywords": ["reaction", "elements", "order", "scene"]}, {"paragraph_vector": [74.534774, -47.056137], "paragraph_keywords": ["reaction", "motion", "forces", "partners"]}, {"paragraph_vector": [56.181972, 63.315818], "paragraph_keywords": ["atoms", "tessellation", "shader", "molecules"]}, {"paragraph_vector": [41.68156, 57.221183], "paragraph_keywords": ["atoms", "molecules", "shader", "number"]}, {"paragraph_vector": [72.991699, 21.453601], "paragraph_keywords": ["atoms", "nad", "pathway", "shading"]}, {"paragraph_vector": [92.531478, 3.931514], "paragraph_keywords": ["cycle", "performance", "number", "nad"]}, {"paragraph_vector": [101.638732, -9.736135], "paragraph_keywords": ["scene", "atoms", "motion", "video"]}, {"paragraph_vector": [98.763969, -3.786325], "paragraph_keywords": ["system", "scene", "animation", "workflow"]}, {"paragraph_vector": [102.432014, -5.414664], "paragraph_keywords": ["visualization", "based", "system", "project"]}, {"paragraph_vector": [-155.576004, 11.489556], "paragraph_keywords": ["research", "council"]}], "content": {}, "doi": "10.1145/1811158.1811177"}, {"uri": "251", "title": "An Immersive Visualization Study on Molecules Manipulation", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Luis Antonio L. F. da Costa", "Luciana P. Nedel"], "summary": "In the context of visualization and interaction with molecules, the representation of a 3D virtual environment potentially helps the understanding of data and structure manipulation that, in other case, would demand too much effort to be interpreted in a conventional way. Additionally, the exploration of these 3D environments in an immersive way may also improve the usability regarding velocity, precision and comfort. In this work, we study the benefits of molecular manipulation in virtual environments based on Head-Mounted Displays and natural 3D interfaces. We implemented a virtual environment to visualize and manipulate molecules, and tested it with users in order to verify the advantages of the immersive approach against the desktop one. Results achieved demonstrate the benefits of using immersive visualization for this specific purpose. Keywords\u2014Immersive Visualization, Head-Mounted Display, 3D Interaction, Molecular Manipulation.", "keywords": ["visualization", "data", "rift", "structure", "molecule", "interaction", "experience", "information", "approach", "control", "number", "result", "object", "figure", "head", "comfort", "interface", "level", "atom", "experiment", "use", "mouse", "benefit", "software", "error", "time", "participant", "user", "environment", "test", "task"], "document_vector": [143.403259, -46.565277], "paragraphs": [{"paragraph_vector": [138.490509, -13.116358], "paragraph_keywords": ["visualization", "objects", "system", "user"]}, {"paragraph_vector": [154.203201, 7.077257], "paragraph_keywords": ["user", "application", "displays", "users"]}, {"paragraph_vector": [138.994628, -13.428812], "paragraph_keywords": ["data", "visualization", "molecule", "structures"]}, {"paragraph_vector": [109.716407, 4.46009], "paragraph_keywords": ["environment", "visualization", "structures", "study"]}, {"paragraph_vector": [153.149871, 6.155405], "paragraph_keywords": ["user", "developed", "approach", "interface"]}, {"paragraph_vector": [156.188858, 4.03017], "paragraph_keywords": ["user", "figure", "direction", "controls"]}, {"paragraph_vector": [158.058776, 0.277237], "paragraph_keywords": ["user", "atoms", "control", "changes"]}, {"paragraph_vector": [152.629592, 6.758744], "paragraph_keywords": ["environment", "user", "frame", "rate"]}, {"paragraph_vector": [106.984642, 8.885387], "paragraph_keywords": ["environment", "file", "atom", "information"]}, {"paragraph_vector": [157.07727, 7.08494], "paragraph_keywords": ["experience", "students", "participants", "tests"]}, {"paragraph_vector": [155.422454, 1.963633], "paragraph_keywords": ["user", "test", "agent", "pdb"]}, {"paragraph_vector": [155.075363, 3.163322], "paragraph_keywords": ["atoms", "user", "select", "task"]}, {"paragraph_vector": [157.217056, 2.135863], "paragraph_keywords": ["test", "data", "time", "hypothesis"]}, {"paragraph_vector": [154.625411, 3.53899], "paragraph_keywords": ["molecule", "value", "errors", "approach"]}, {"paragraph_vector": [153.174972, 4.36976], "paragraph_keywords": ["tests", "participants", "obtained", "approach"]}, {"paragraph_vector": [155.132888, 3.036124], "paragraph_keywords": ["approach", "users", "results", "visualization"]}, {"paragraph_vector": [153.650863, 5.839242], "paragraph_keywords": ["experiment", "approach", "field", "presents"]}], "content": {}, "doi": "10.1145/1187335.1187340"}, {"uri": "252", "title": "bcl::Cluster : A method for clustering biological molecules coupled with visualization in the Pymol Molecular Graphics System", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Nathan Alexander", "Nils Woetzel", "Jens Meiler"], "summary": "Clustering algorithms are used as data analysis tools in a wide variety of applications in Biology. Clustering has become especially important in protein structure prediction and virtual high throughput screening methods. In protein structure prediction, clustering is used to structure the conformational space of thousands of protein models. In virtual high throughput screening, databases with millions of drug-like molecules are organized by structural similarity, e.g. common scaffolds. The tree-like dendrogram structure obtained from hierarchical clustering can provide a qualitative overview of the results, which is important for focusing detailed analysis. However, in practice it is difficult to relate specific components of the dendrogram directly back to the objects of which it is comprised and to display all desired information within the two dimensions of the dendrogram. The current work presents a hierarchical agglomerative clustering method termed bcl::Cluster. bcl::Cluster utilizes the Pymol Molecular Graphics System to graphically depict dendrograms in three dimensions. This allows simultaneous display of relevant biological molecules as well as additional information about the clusters and the members comprising them. Keywords-clustering; Pymol; visualization; analysis; proteins; molecules", "keywords": ["supported", "nih"], "document_vector": [175.003173, 69.796295], "paragraphs": [{"paragraph_vector": [-156.015426, 14.197516], "paragraph_keywords": ["nih", "supported", "fellowship", "research"]}], "content": {}, "doi": "10.1145/2425333.2425366"}, {"uri": "253", "title": "Generate Galaxy-like Word Cloud Using Molecular Cloud Evolution", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Zhenfeng He", "Ying Cao", "Hui Xiong"], "summary": "The evolution of molecular cloud(MC) brings about various stars and galaxies. A simple two-stage MC based evolutionary model is proposed to reorganize a word cloud into a galaxy-like cloud. Based on the model, an evolutionary algorithm simulates the moving and colliding of stage-1 MCs, and the merging and collapsing of stage-2 MCs. In addition, it considers long range force based disturbance, which often starts the formation of stars. Our experiment suggests that it produces galaxy-like word clouds for users to choose from. Keywords-word cloud; text visualization; molecular cloud evolution; visual art; nature inspired algorithm;", "keywords": ["set", "core", "list", "step", "algorithm", "end", "mc", "cloud", "y", "speed", "word"], "document_vector": [-54.457653, -20.801332], "paragraphs": [{"paragraph_vector": [-116.022697, 58.448253], "paragraph_keywords": ["mc", "cloud", "speed", "moving"]}, {"paragraph_vector": [-115.491714, 57.010196], "paragraph_keywords": ["word", "mc", "size", "step"]}, {"paragraph_vector": [-112.125747, 60.890621], "paragraph_keywords": ["mc", "step", "set", "algorithm"]}, {"paragraph_vector": [-113.070693, 60.38467], "paragraph_keywords": ["j", "mc", "hj", "c"]}, {"paragraph_vector": [-113.954826, 56.919082], "paragraph_keywords": ["word", "speed", "clouds", "cloud"]}], "content": {}, "doi": "10.1145/1836049.1836067"}, {"uri": "254", "title": "AINAN: COLLABORATIVE APPLICATION FOR PROTEIN VISUALIZATION & ANALYSES", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Rauf Malick", "Usama Khanzada", "Athar Abbas", "Waheed Iqbal", "Kamran Azim"], "summary": "AINAAN is a collaborative application developed for molecular visualization and analysis with multiple view points. The application allows multiple users to view and manipulate single and multiple representations of molecular structures. Network sessions can be initiated by any user over the network. Members over the network can join the session, and the initiator owns the activities of the session. The application provides conventional molecular visualization facilities like rendering, rotation, with different views, along with the usage of protein network stores. Protein store is the repository of the protein structures, alignment results, and movies maintained by each user, which can be shared over the network. Sequence and structural analysis algorithms are also supported in AINAAN. Sessions can be saved by recording events and alignment results for future reference. Large scale study of protein molecular structures will need network views support for comparative analysis.", "keywords": ["ainaan", "visualization", "structure", "analysis", "application", "architecture", "server", "session", "protein", "network", "distributed", "result", "system", ".", "study", "client", "graphic", "view", "time", "user", "environment"], "document_vector": [83.689773, 56.574211], "paragraphs": [{"paragraph_vector": [-130.080398, -41.901966], "paragraph_keywords": ["protein", "structures", "networks", "study"]}, {"paragraph_vector": [132.037002, 10.027211], "paragraph_keywords": ["visualization", "protein", "user", "sessions"]}, {"paragraph_vector": [154.520111, 22.701116], "paragraph_keywords": ["session", "application", "network", "server"]}, {"paragraph_vector": [160.46376, -36.500823], "paragraph_keywords": ["application", "protein", "movie", "ainaan"]}, {"paragraph_vector": [146.333633, 21.70651], "paragraph_keywords": ["server", "client", "session", "application"]}, {"paragraph_vector": [161.302429, 22.126602], "paragraph_keywords": ["protein", "network", "views", "users"]}, {"paragraph_vector": [156.232788, 18.796356], "paragraph_keywords": ["protein", "visualization", "session", "distributed"]}, {"paragraph_vector": [46.961616, -9.810755], "paragraph_keywords": ["graphics", "visualization", ".", "environment"]}, {"paragraph_vector": [150.397125, 13.659194], "paragraph_keywords": ["jesus", "james", "sergi", "ferenc"]}], "content": {}, "doi": "10.5555/948835.948859"}, {"uri": "255", "title": "A Scatterplot-Based Visual Analytics Tool for Protein Pocket Properties", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Makiko Miyoshi", "Ayaka Kaneko", "Takayuki Itoh", "Kei Yura"], "summary": "Protein is the major component of the organism. A hole (pocket) on the surface of a protein is known to be the best target for a drug to react. We started analyzing how \"druggability\" of proteins related to the locations amino acids in a pocket. This poster presents a visualization tool for a distance distribution analysis between two types of amino acids in a pocket and proposes the biological interpretation of the visualization results. KeywordsVisualization; Protein; Amino acid; Pocket identification;", "keywords": [], "document_vector": [-124.823097, -6.268964], "paragraphs": [{"paragraph_vector": [-129.15242, 21.730669], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/2628588.2628597"}, {"uri": "256", "title": "Differential Protein Expression Analysis via Liquid-Chromatography/Mass-Spectrometry Data Visualization", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Lars Linsen", "Julia L\u00f6cherbach", "Matthias Berth", "J\u00f6rg Bernhardt", "D\u00f6rte Becher"], "summary": "Differential protein expression analysis is one of the main challenges in proteomics. It denotes the search for proteins, whose encoding genes are differentially expressed under a given experimental setup. An important task in this context is to identify the differentially expressed proteins or, more generally, all proteins present in the sample. One of the most promising and recently widely used approaches for protein identification is to cleave proteins into peptides, separate the peptides using liquid chromatography, and determine the masses of the separated peptides using mass spectrometry. The resulting data needs to be analyzed and matched against protein sequence databases. The analysis step is typically done by searching for intensity peaks in a large number of 2D graphs. We present an interactive visualization tool for the exploration of liquid-chromatography/mass-spectrometry data in a 3D space, which allows for the understanding of the data in its entirety and a detailed analysis of regions of interest. We compute differential expression over the liquid-chromatography/mass-spectrometry domain and embed it visually in our system. Our exploration tool can treat single liquid-chromatography/mass-spectrometry data sets \u2217linsen@uni-greifswald.de \u2020{loeba,berth}@decodon.com \u2021{Joerg.Bernhardt,dbecher}@uni-greifswald.de as well as data acquired using multi-dimensional protein identification technology. For efficiency purposes we perform a peakpreserving data resampling and multiresolution hierarchy generation prior to visualization. CR Categories: I.3.7 [Computer Graphics]: Three-dimensional Graphics and Realism; I.3.8 [Computer Graphics]: Applications", "keywords": ["fraction", "peptide", "data", "value", "allows", "lc", "analysis", "identification", "expression", "set", "m", "spectrometry", "protein", "system", "peak", "intensity", "figure", "multiresolution", "step", "color", "z", "ionization", "resolution", "based", "mass", "dimension", "use", "time", "chromatography", "image", "visualization", "heightfield", "test", "need"], "document_vector": [34.038593, 86.908149], "paragraphs": [{"paragraph_vector": [-76.13063, -37.993682], "paragraph_keywords": ["peptides", "protein", "ms", "proteins"]}, {"paragraph_vector": [-76.379676, -40.554565], "paragraph_keywords": ["mass", "peptides", "ionization", "particles"]}, {"paragraph_vector": [-74.467826, -41.998554], "paragraph_keywords": ["data", "time", "intensity", "figure"]}, {"paragraph_vector": [-70.904785, -38.296092], "paragraph_keywords": ["data", "intensity", "visualization", "set"]}, {"paragraph_vector": [-65.479942, -38.614936], "paragraph_keywords": ["data", "expression", "intensity", "resolution"]}, {"paragraph_vector": [97.240119, -37.300971], "paragraph_keywords": ["data", "resolution", "resampling", "visualization"]}, {"paragraph_vector": [-64.867179, -40.475898], "paragraph_keywords": ["data", "values", "vertices", "f"]}, {"paragraph_vector": [119.679023, -31.896656], "paragraph_keywords": ["intensity", "color", "data", "chosen"]}, {"paragraph_vector": [-69.280509, -31.441682], "paragraph_keywords": ["data", "expression", "test", "control"]}, {"paragraph_vector": [-67.318443, -33.559066], "paragraph_keywords": ["fractions", "intensity", "data", "peaks"]}, {"paragraph_vector": [-73.525756, -40.478065], "paragraph_keywords": ["dimension", "data", "spectrometry", "intensity"]}, {"paragraph_vector": [-67.660278, -35.899532], "paragraph_keywords": ["data", "expression", "set", "fractions"]}, {"paragraph_vector": [-67.641929, -35.268642], "paragraph_keywords": ["data", "visualization", "system", "analysis"]}, {"paragraph_vector": [-71.214584, -36.739704], "paragraph_keywords": ["data", "chromatography", "ionization", "spectrometry"]}, {"paragraph_vector": [-67.207771, -35.895832], "paragraph_keywords": ["analysis", "computer", "applications", "series"]}], "content": {}, "doi": "10.1109/SUPERC.1990.130106"}, {"uri": "257", "title": "TexMol: Interactive Visual Exploration of Large Flexible Multi-component Molecular Complexes", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Chandrajit Bajaj", "Peter Djeu", "Vinay Siddavanahalli"], "summary": "While molecular visualization software has advanced over the years, today, most tools still operate on individual molecular structures with limited facility to manipulate large multi-component complexes. We approach this problem by extending 3D imagebased rendering via programmable graphics units, resulting in an order of magnitude speedup over traditional triangle-based rendering. By incorporating a biochemically sensitive level-of-detail hierarchy into our molecular representation, we communicate appropriate volume occupancy and shape while dramatically reducing the visual clutter that normally inhibits higher-level spatial comprehension. Our hierarchical, image based rendering also allows dynamically computed physical properties data (e.g. electrostatics potential) to be mapped onto the molecular surface, tying molecular structure to molecular function. Finally, we present another approach to interactive molecular exploration using volumetric and structural rendering in tandem to discover molecular properties that neither rendering mode alone could reveal. These visualization techniques are realized in a high-performance, interactive molecular exploration tool we call TexMol, short for Texture Molecular viewer. CR Categories: I.3.4 [Computer Graphics]: Graphics Utilities\u2014 Application packages; I.3.5 [Computer Graphics]: Computational Geometry and Object Modeling\u2014Hierarchy and geometric transformations", "keywords": ["opacity", "structure", "end", "hierarchy", "molecule", "program", "helix", "represented", "render", "set", "algorithm", "volumetric", "primitive", "andrew", "texmol", "figure", "depth", "texture", "resolution", "based", "lod", "surface", "level", "performance", "atom", "graphic", "cylinder", "model", "gpu", "radius", "vertex", "image", "rendering", "visualization"], "document_vector": [115.499839, -84.521316], "paragraphs": [{"paragraph_vector": [38.813167, -52.686298], "paragraph_keywords": ["hierarchy", "atom", "structure", "level"]}, {"paragraph_vector": [75.154052, 0.991805], "paragraph_keywords": ["rendering", "lod", "surface", "depth"]}, {"paragraph_vector": [37.07463, 55.626686], "paragraph_keywords": ["rendering", "based", "cylinder", "molecule"]}, {"paragraph_vector": [32.777076, 60.541656], "paragraph_keywords": ["cylinder", "algorithm", "program", "bases"]}, {"paragraph_vector": [35.544002, 52.393013], "paragraph_keywords": ["program", "end", "f", "helix"]}, {"paragraph_vector": [68.843917, 12.717611], "paragraph_keywords": ["helix", "rendering", "function", "molecule"]}, {"paragraph_vector": [111.579925, 29.580461], "paragraph_keywords": ["lod", "data", "memory", "view"]}, {"paragraph_vector": [93.540283, 42.046566], "paragraph_keywords": ["lod", "vertex", "atom", "rendering"]}, {"paragraph_vector": [108.670364, 34.209327], "paragraph_keywords": ["rendering", "texmol", "figure", "rate"]}, {"paragraph_vector": [80.17218, -7.681585], "paragraph_keywords": ["baker", "based", "sept", "amitabh"]}, {"paragraph_vector": [100.072113, 27.962005], "paragraph_keywords": ["rendering", "andrew", "graphics", "conference"]}, {"paragraph_vector": [47.820762, -9.529542], "paragraph_keywords": ["surfaces", "pages", "graphics", "computer"]}], "content": {}, "doi": "10.1145/1529282.1529446"}, {"uri": "258", "title": "A Flexible Framework for Asynchronous In Situ and In Transit Analytics for Scientific Simulations", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Matthieu Dreher", "Bruno Raffin"], "summary": "High performance computing systems are today composed of tens of thousands of processors and deep memory hierarchies. The next generation of machines will further increase the unbalance between I/O capabilities and processing power. To reduce the pressure on I/Os, the in situ analytics paradigm proposes to process the data as closely as possible to where and when the data are produced. Processing can be embedded in the simulation code, executed asynchronously on helper cores on the same nodes, or performed in transit on staging nodes dedicated to analytics. Today, software environnements as well as usage scenarios still need to be investigated before in situ analytics become a standard practice. In this paper we introduce a framework for designing, deploying and executing in situ scenarios. Based on a component model, the scientist designs analytics workflows by first developing processing components that are next assembled in a dataflow graph through a Python script. At runtime the graph is instantiated according to the execution context, the framework taking care of deploying the application on the target architecture and coordinating the analytics workflows with the simulation execution. Component coordination, zerocopy intra-node communications or inter-nodes data transfers rely on per-node distributed daemons. We evaluate various scenarios performing in situ and in transit analytics on large molecular dynamics systems simulated with Gromacs using up to 2048 cores. We show in particular that analytics processing can be performed on the fraction of resources the simulation does not use well, resulting in a limited impact on the simulation performance (less than 9%). Our more advanced scenario combines in situ and in transit processing to compute a molecular surface based on the Quicksurf algorithm. Keywords-In Situ Analytics and Visualization; IO; Molecular Dynamics;", "keywords": ["output", "flowvr", "data", "scenario", "application", "simulation", "daemon", "situ", "running", "memory", "input", "module", "framework", "step", "port", "distribution", "core", "analytics", "mpi", "code", "process", "python", "position", "performance", "atom", "message", "gromacs", "file", "staging", "transit", "grid", "instance", "pattern", "user", "node"], "document_vector": [4.138364, -31.337638], "paragraphs": [{"paragraph_vector": [-169.853317, 39.220954], "paragraph_keywords": ["simulation", "data", "analytics", "approach"]}, {"paragraph_vector": [-171.470184, 40.73006], "paragraph_keywords": ["data", "situ", "simulation", "process"]}, {"paragraph_vector": [168.94992, 32.348793], "paragraph_keywords": ["module", "message", "application", "input"]}, {"paragraph_vector": [-178.36557, 46.75201], "paragraph_keywords": ["modules", "instance", "application", "module"]}, {"paragraph_vector": [176.293807, 37.68729], "paragraph_keywords": ["module", "modules", "daemon", "patterns"]}, {"paragraph_vector": [166.340164, 32.894252], "paragraph_keywords": ["memory", "message", "plugin", "module"]}, {"paragraph_vector": [172.778335, 38.260437], "paragraph_keywords": ["simulation", "message", "flowvr", "stamps"]}, {"paragraph_vector": [174.32199, 40.049976], "paragraph_keywords": ["data", "port", "file", "module"]}, {"paragraph_vector": [-179.507492, 44.863159], "paragraph_keywords": ["atom", "according", "module", "positions"]}, {"paragraph_vector": [164.078887, 64.544242], "paragraph_keywords": ["distribution", "situ", "nodes", "grid"]}, {"paragraph_vector": [-175.227813, 42.079761], "paragraph_keywords": ["data", "densities", "atoms", "m"]}, {"paragraph_vector": [-172.542602, 42.132461], "paragraph_keywords": ["nodes", "simulation", "data", "situ"]}, {"paragraph_vector": [178.069107, 39.760936], "paragraph_keywords": ["node", "gromacs", "performance", "cores"]}, {"paragraph_vector": [-173.595947, 39.913291], "paragraph_keywords": ["gromacs", "data", "situ", "cores"]}, {"paragraph_vector": [-178.852859, 39.857105], "paragraph_keywords": ["nodes", "data", "redistribution", "positions"]}, {"paragraph_vector": [179.331695, 39.313106], "paragraph_keywords": ["module", "data", "application", "simulation"]}, {"paragraph_vector": [168.141448, -38.190803], "paragraph_keywords": ["project", "pierre", "thank", "test"]}], "content": {}, "doi": "10.1145/1006032.1006036"}, {"uri": "259", "title": "Accelerated Visualization of Transparent Molecular Surfaces in Molecular Dynamics", "timestamp": "2016", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Adam Jur\u010d\u0131\u0301k", "Julius Parulek", "Ji\u0159\u0131\u0301 Sochor", "Barbora Kozl\u0131\u0301kov\u00e1"], "summary": "The reactivity of the biomolecular structures is highly influenced by their structural features. Thus, studying these features along with the exploration of their dynamic behavior helps to understand the processes ongoing in living cells. This can be reached by the visual representation of these processes as visualization is one of the most natural ways to convey such information. However, none of the currently available techniques provides the biochemists with an intuitive real-time representation of the dynamic movements of molecules and precise geometrical based extraction of their structural features performed instantly. In this paper we introduce such a technique enabling the user to compute and also to visualize the molecular surface along with inner voids. To obtain a better insight into the molecule, our technique enables to visualize the molecular surface transparently. The opacity can be adjusted by changing user-defined parameters in order to enhance the perception of the surfaces of inner voids. All integrated algorithms run in real-time which gives the user a big variety of exploration possibilities. The importance of our approach is even amplified with respect to the fact that currently the size of molecular dynamics simulations is increasing dramatically and offline rendering thus becomes impracticable. The usability of our technique was evaluated by the domain experts.", "keywords": ["fragment", "cavity", "buffer", "data", "structure", "molecule", "et", "method", "presented", "protein", "memory", "algorithm", "solution", "triangle", "compute", "approach", "number", "transparency", "pixel", "computation", "defined", "ray", "sphere", "technique", "surface", "computed", "based", "performance", "atom", "s", "patch", "time", "al", "rendering", "visualization", "casting", "torus"], "document_vector": [-135.474578, -28.202013], "paragraphs": [{"paragraph_vector": [64.344116, -52.316501], "paragraph_keywords": ["protein", "structure", "trajectories", "visualization"]}, {"paragraph_vector": [56.951305, 10.159744], "paragraph_keywords": ["surface", "computation", "visualization", "ses"]}, {"paragraph_vector": [35.628601, 28.743038], "paragraph_keywords": ["ses", "algorithm", "approach", "al"]}, {"paragraph_vector": [50.54993, -23.673961], "paragraph_keywords": ["techniques", "visualization", "based", "surfaces"]}, {"paragraph_vector": [52.731414, 66.942108], "paragraph_keywords": ["rendering", "fragments", "et", "al"]}, {"paragraph_vector": [52.009532, 50.995704], "paragraph_keywords": ["surface", "cavities", "computation", "opacity"]}, {"paragraph_vector": [19.556253, 40.359218], "paragraph_keywords": ["surface", "triangles", "patches", "computation"]}, {"paragraph_vector": [16.776067, 31.155559], "paragraph_keywords": ["surface", "patches", "triangles", "patch"]}, {"paragraph_vector": [29.249814, 25.40743], "paragraph_keywords": ["surface", "graph", "edges", "patches"]}, {"paragraph_vector": [11.233327, 27.917938], "paragraph_keywords": ["torus", "surface", "patches", "patch"]}, {"paragraph_vector": [12.702323, 39.219505], "paragraph_keywords": ["surface", "patch", "fragments", "patches"]}, {"paragraph_vector": [13.099144, 46.551502], "paragraph_keywords": ["obb", "computed", "patch", "plane"]}, {"paragraph_vector": [13.388068, 44.580009], "paragraph_keywords": ["patch", "ray", "planes", "clipping"]}, {"paragraph_vector": [16.427436, 34.597965], "paragraph_keywords": ["ray", "fragments", "surface", "opacity"]}, {"paragraph_vector": [32.754806, 10.527414], "paragraph_keywords": ["technique", "method", "performance", "area"]}, {"paragraph_vector": [21.428058, 46.012668], "paragraph_keywords": ["method", "performance", "kernel", "applying"]}, {"paragraph_vector": [26.964788, 25.709762], "paragraph_keywords": ["number", "maxneighbors", "protein", "maxtriangles"]}, {"paragraph_vector": [62.72311, 7.481911], "paragraph_keywords": ["solution", "computation", "biochemists", "surface"]}, {"paragraph_vector": [-154.719512, 13.34517], "paragraph_keywords": ["project", "research", "funded", "nf"]}], "content": {}, "doi": "10.1145/1385569.1385635"}, {"uri": "260", "title": "PII: S0097-8493(98)00091-0", "timestamp": "1999", "rating": "0.0", "annotation": "", "tags": [], "authors": ["WOLF-D. IHLENFELDT", "KLAUS ENGEL"], "summary": "\u00d0Chemistry faces interesting problems with regard to internet-based information visualization, especially in the context of digital library documents with multimedia components. We have developed a number of methods and applications ranging from the automatic generation of 2D structure sketches to 3D volume interactive visualization of molecular data with iso-surfaces and other 3D visualization techniques. This article gives an overview about these techniques and their potential for digital publishing and information indexing. # 1999 Elsevier Science Ltd. All rights reserved", "keywords": ["data", "volume", "structure", "value", "iso", "analysis", "application", "server", "generation", "graph", "set", "space", "mesh", "plug", "algorithm", "triangle", "information", "number", "chemist", "chemical", "bond", "format", "color", "erent", "resolution", "example", "surface", "based", "level", "scene", "element", "client", "atom", "chemistry", "vrml", "display", "allow", "user", "document", "visualization", "di", "exchange", "node"], "document_vector": [178.872207, 0.621144], "paragraphs": [{"paragraph_vector": [116.412422, -4.684168], "paragraph_keywords": ["information", "chemistry", "data", "exchange"]}, {"paragraph_vector": [114.206359, -8.105897], "paragraph_keywords": ["access", "literature", "data", "compound"]}, {"paragraph_vector": [116.694648, -6.130639], "paragraph_keywords": ["information", "chemistry", "color", "data"]}, {"paragraph_vector": [121.79515, 2.021997], "paragraph_keywords": ["chemical", "data", "structure", "formats"]}, {"paragraph_vector": [119.567787, -3.301163], "paragraph_keywords": ["structure", "rules", "structures", "plots"]}, {"paragraph_vector": [105.27581, -15.645047], "paragraph_keywords": ["atoms", "structure", "hydrogen", "ligands"]}, {"paragraph_vector": [107.767509, -12.556705], "paragraph_keywords": ["bonds", "coordinates", "display", "structures"]}, {"paragraph_vector": [103.813888, -8.610451], "paragraph_keywords": ["bond", "atoms", "chemical", "vrml"]}, {"paragraph_vector": [107.07711, -8.525156], "paragraph_keywords": ["scene", "atoms", "bonds", "atom"]}, {"paragraph_vector": [110.621414, -8.268948], "paragraph_keywords": ["information", "surface", "cases", "scene"]}, {"paragraph_vector": [137.007339, 41.619911], "paragraph_keywords": ["web", "data", "client", "iso"]}, {"paragraph_vector": [41.264678, 29.297842], "paragraph_keywords": ["mesh", "element", "approximation", "space"]}, {"paragraph_vector": [23.764759, 41.011611], "paragraph_keywords": ["surface", "elements", "level", "algorithm"]}, {"paragraph_vector": [15.848757, 35.887886], "paragraph_keywords": ["surface", "iso", "stack", "triangles"]}, {"paragraph_vector": [24.029664, 43.288238], "paragraph_keywords": ["surface", "level", "iso", "levels"]}, {"paragraph_vector": [140.906646, 40.157974], "paragraph_keywords": ["triangles", "iso", "surface", "levels"]}, {"paragraph_vector": [135.916687, 43.400718], "paragraph_keywords": ["node", "vrml", "geometry", "indexedfaceset"]}, {"paragraph_vector": [127.132812, 49.736751], "paragraph_keywords": ["server", "level", "surface", "user"]}, {"paragraph_vector": [136.093414, 47.304096], "paragraph_keywords": ["surface", "iso", "user", "level"]}, {"paragraph_vector": [115.953781, -6.706896], "paragraph_keywords": ["information", "analysis", "c", "generation"]}], "content": {}, "doi": "10.1145/1294685.1294704"}, {"uri": "261", "title": "empty", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kenji Fukuzawa", "Taichi ShimutaI", "Tomohiko Yoshida", "Hedong ZhangI", "Yasunaga MitsuyaI"], "summary": "High-density hard disk drives (HDDs) require a very low flying height, less than 10 nm. To meet this dema2nd, the current thickness of liquid lubricant films is molecularly thin and of the order of 1 nm. Therefore, distribution control of molecularly thin lubricant films is indispensable to high-density recording. The behavior of molecularly thin liquid films is determined by the intermolecular interaction between the liquid molecules and media surface. The instability of the molecularly thin lubricant on the disks is a big problem for the polar lubricant, which is widely used for HDDs. The instability is called dewetting, which is caused by the intermolecular interaction between the lubricant and the disk. In this paper, the dewetting of molecularly thin lubricant films on magnetic disks has been investigated by visualizing the dewetting process directly, using ellipsometric microscopy.", "keywords": [], "document_vector": [71.788261, -10.712191], "paragraphs": [{"paragraph_vector": [-130.292709, 21.911067], "paragraph_keywords": []}], "content": {}, "doi": "10.1109/VISUAL.2004.36"}, {"uri": "262", "title": "Investigating the Efficacy of Nonlinear Dimensionality Reduction Schemes in Classifying Gene and Protein Expression Studies", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["George Lee", "Carlos Rodriguez", "Anant Madabhushi"], "summary": "The recent explosion in procurement and availability of high-dimensional gene and protein expression profile data sets for cancer diagnostics has necessitated the development of sophisticated machine learning tools with which to analyze them. While some investigators are focused on identifying informative genes and proteins that play a role in specific diseases, other researchers have attempted instead to use patients based on their expression profiles to prognosticate disease status. A major limitation in the ability to accurately classify these high-dimensional data sets stems from the \u201ccurse of dimensionality,\u201d occurring in situations where the number of genes or peptides significantly exceeds the total number of patient samples. Previous attempts at dealing with this issue have mostly centered on the use of a dimensionality reduction (DR) scheme, Principal Component Analysis (PCA), to obtain a low-dimensional projection of the high-dimensional data. However, linear PCA and other linear DR methods, which rely on euclidean distances to estimate object similarity, do not account for the inherent underlying nonlinear structure associated with most biomedical data. While some researchers have begun to explore nonlinear DR methods for computer vision problems such as face detection and recognition, to the best of our knowledge, few such attempts have been made for classification and visualization of high-dimensional biomedical data. The motivation behind this work is to identify the appropriate DR methods for analysis of high-dimensional gene and protein expression studies. Toward this end, we empirically and rigorously compare three nonlinear (Isomap, Locally Linear Embedding, and Laplacian Eigenmaps) and three linear DR schemes (PCA, Linear Discriminant Analysis, and Multidimensional Scaling) with the intent of determining a reduced subspace representation in which the individual object classes are more easily discriminable. Owing to the inherent nonlinear structure of gene and protein expression studies, our claim is that the nonlinear DR methods provide a more truthful low-dimensional representation of the data compared to the linear DR schemes. Evaluation of the DR schemes was done by 1) assessing the discriminability of two supervised classifiers (Support Vector Machine and C4.5 Decision Trees) in the different lowdimensional data embeddings and 2) five cluster validity measures to evaluate the size, distance, and tightness of object aggregates in the low-dimensional space. For each of the seven evaluation measures considered, statistically significant improvement in the quality of the embeddings across 10 cancer data sets via the use of three nonlinear DR schemes over three linear DR techniques was observed. Similar trends were observed when linear and nonlinear DR was applied to the high-dimensional data following feature pruning to isolate the most informative features. Qualitative evaluation of the low-dimensional data embedding obtained via the six DR methods further suggests that the nonlinear schemes are better able to identify potential novel classes (e.g., cancer subtypes) within", "keywords": ["compared", "data", "md", "embedding", "d", "sample", "gene", "expression", "classifier", "space", "set", "method", "cluster", "accuracy", "protein", "\u00fe", "matrix", "obtained", "cancer", "\u00f0xa\u00fe", "number", "y", "dr", "lle", "xa", "measure", "study", "performance", "scheme", "use", "class", "feature", "xi", "lda", "g", "xb", "pca", "\u00f0xi\u00fe"], "document_vector": [-164.44963, 89.920944], "paragraphs": [{"paragraph_vector": [-65.877906, 5.122516], "paragraph_keywords": ["gene", "protein", "expression", "samples"]}, {"paragraph_vector": [-64.127624, 3.036356], "paragraph_keywords": ["data", "space", "feature", "selection"]}, {"paragraph_vector": [-65.60115, 1.714925], "paragraph_keywords": ["data", "lda", "dr", "graph"]}, {"paragraph_vector": [-68.649589, 3.808455], "paragraph_keywords": ["data", "fig", "embedding", "space"]}, {"paragraph_vector": [-69.047744, 6.008652], "paragraph_keywords": ["dr", "methods", "data", "pca"]}, {"paragraph_vector": [-69.429626, 8.351367], "paragraph_keywords": ["data", "objects", "dr", "classes"]}, {"paragraph_vector": [-67.520454, 5.617412], "paragraph_keywords": ["xi", "data", "section", "y"]}, {"paragraph_vector": [-66.134208, -10.12993], "paragraph_keywords": ["data", "matrix", "eigenvectors", "d"]}, {"paragraph_vector": [-62.777717, -8.801266], "paragraph_keywords": ["xi", "data", "xa", "mds"]}, {"paragraph_vector": [-64.287498, -9.461841], "paragraph_keywords": ["matrix", "xi", "d", "weight"]}, {"paragraph_vector": [-63.723979, 8.769574], "paragraph_keywords": ["data", "dr", "samples", "feature"]}, {"paragraph_vector": [-73.181365, 19.380413], "paragraph_keywords": ["dr", "embedding", "xi", "classifier"]}, {"paragraph_vector": [-63.443832, 8.531644], "paragraph_keywords": ["u", "f", "data", "dr"]}, {"paragraph_vector": [-71.549644, -2.126396], "paragraph_keywords": ["training", "strj", "g", "svm"]}, {"paragraph_vector": [-73.815711, -1.240622], "paragraph_keywords": ["class", "rule", "xa", "training"]}, {"paragraph_vector": [-73.280624, 19.716375], "paragraph_keywords": ["g", "cluster", "\u00f0xi\u00fe", "xi"]}, {"paragraph_vector": [-72.854492, 18.236682], "paragraph_keywords": ["\u00fe", "cct", "measures", "embedding"]}, {"paragraph_vector": [-67.290313, 5.840342], "paragraph_keywords": ["methods", "dr", "data", "obtained"]}, {"paragraph_vector": [-73.797256, 16.980657], "paragraph_keywords": ["data", "sets", "classes", "lle"]}, {"paragraph_vector": [-72.391365, 18.345937], "paragraph_keywords": ["dr", "class", "methods", "schemes"]}, {"paragraph_vector": [-67.868247, 7.145176], "paragraph_keywords": ["dr", "methods", "data", "dimensions"]}, {"paragraph_vector": [-64.364212, 8.092572], "paragraph_keywords": ["data", "methods", "dr", "study"]}, {"paragraph_vector": [-69.841529, 5.61039], "paragraph_keywords": ["cancer", "dr", "foundation", "methods"]}], "content": {}, "doi": "10.1145/3144769.3144772"}, {"uri": "263", "title": "Visualizing the Energetics of the Dissociation of a Metastable Molecule", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["David Guzman", "Reuben Reyes", "Karla Vega", "Kelly Gaither", "Robert Wyatt"], "summary": "Introduction Shown in Figure 1 is the simulation of the energetics of the dissociation of a metastable molecule. The simulation was computed by Dr. Robert Wyatt, a Professor of Chemistry at The University of Texas at Austin. Dr. Wyatt\u2019s simulation uses a model molecule composed of two fragments, A and B. Starting from the molecular state AB, the simulation computes the probability that, under any given conditions, AB will dissociate into the separated components A + B. This may seem like a toy problem, but in real life many molecules demonstrate metastable behavior--a propensity to dissociate--and the work that this simulation is doing may ultimately have an impact on studies of the energetics of larger molecules. An important class of these is biomolecular. This work may be particularly applicable to energy transfer problems in biological systems.", "keywords": ["density", "display", "probability", "method", "particle", "trajectory", "wyatt", "simulated", "time", "behavior", "simulation"], "document_vector": [-50.175781, 15.17476], "paragraphs": [{"paragraph_vector": [91.90908, -57.355251], "paragraph_keywords": ["simulation", "method", "texas", "problem"]}, {"paragraph_vector": [86.701614, -55.919151], "paragraph_keywords": ["particles", "particle", "probability", "simulated"]}, {"paragraph_vector": [96.093025, -9.092533], "paragraph_keywords": ["particle", "behavior", "density", "dynamics"]}], "content": {}, "doi": "10.1145/3281464.3281469"}, {"uri": "264", "title": "Analysis of Long Molecular Dynamics Simulations Using Interactive Focus+Context Visualization", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["J. By\u0161ka", "T. Trautner", "S. M. Marques", "J. Damborsk\u00fd", "B. Kozl\u00edkov\u00e1", "M. Waldner"], "summary": "Analyzing molecular dynamics (MD) simulations is a key aspect to understand protein dynamics and function. With increasing computational power, it is now possible to generate very long and complex simulations, which are cumbersome to explore using traditional 3D animations of protein movements. Guided by requirements derived from multiple focus groups with protein engineering experts, we designed and developed a novel interactive visual analysis approach for long and crowded MD simulations. In this approach, we link a dynamic 3D focus+context visualization with a 2D chart of time series data to guide the detection and navigation towards important spatio-temporal events. The 3D visualization renders elements of interest in more detail and increases the temporal resolution dependent on the time series data or the spatial region of interest. In case studies with different MD simulation data sets and research questions, we found that the proposed visual analysis approach facilitates exploratory analysis to generate, confirm, or reject hypotheses about causalities. Finally, we derived design guidelines for interactive visual analysis of complex MD simulation data. CCS Concepts \u2022 Human-centered computing \u2192 Scientific visualization; User centered design;", "keywords": ["importance", "data", "group", "md", "investigation", "focus", "molecule", "analysis", "event", "observe", "representation", "et", "distance", "interaction", "simulation", "set", "driven", "protein", "ligand", "speed", "behavior", "figure", "case", "tunnel", "change", "resolution", "based", "study", "tool", "frame", "element", "water", "measurement", "animation", "design", "expert", "use", "time", "domain", "al", "user", "visualization", "context", "need"], "document_vector": [-118.932716, 13.991046], "paragraphs": [{"paragraph_vector": [87.108726, -60.455081], "paragraph_keywords": ["simulations", "design", "visualization", "experts"]}, {"paragraph_vector": [87.917755, -67.472442], "paragraph_keywords": ["analysis", "simulations", "animations", "experts"]}, {"paragraph_vector": [97.757614, -67.698623], "paragraph_keywords": ["visualization", "design", "section", "based"]}, {"paragraph_vector": [86.603561, -57.765064], "paragraph_keywords": ["simulations", "md", "simulation", "movements"]}, {"paragraph_vector": [38.551219, -58.365806], "paragraph_keywords": ["protein", "exploration", "simulations", "space"]}, {"paragraph_vector": [57.727313, -69.301422], "paragraph_keywords": ["protein", "trajectories", "molecules", "proposed"]}, {"paragraph_vector": [80.317459, -67.690979], "paragraph_keywords": ["groups", "focus", "investigation", "analysis"]}, {"paragraph_vector": [78.910034, -64.83171], "paragraph_keywords": ["protein", "experts", "simulations", "time"]}, {"paragraph_vector": [62.55997, -66.999908], "paragraph_keywords": ["visualization", "analysis", "protein", "use"]}, {"paragraph_vector": [111.193275, -66.011054], "paragraph_keywords": ["visualization", "time", "simulations", "use"]}, {"paragraph_vector": [91.845733, -70.559783], "paragraph_keywords": ["focus", "protein", "visualization", "requirements"]}, {"paragraph_vector": [97.999832, -33.11333], "paragraph_keywords": ["focus", "representation", "representations", "context"]}, {"paragraph_vector": [117.658187, -55.86568], "paragraph_keywords": ["focus", "elements", "visualization", "time"]}, {"paragraph_vector": [125.506462, -62.988777], "paragraph_keywords": ["events", "resolution", "focus", "interest"]}, {"paragraph_vector": [109.25077, -70.646835], "paragraph_keywords": ["measurements", "time", "simulation", "visualized"]}, {"paragraph_vector": [120.649139, -65.393562], "paragraph_keywords": ["time", "ligand", "speed", "window"]}, {"paragraph_vector": [121.809875, -65.282394], "paragraph_keywords": ["measurements", "combination", "simulation", "ligand"]}, {"paragraph_vector": [116.80355, -67.684921], "paragraph_keywords": ["speed", "ligand", "protein", "distance"]}, {"paragraph_vector": [75.157394, -71.45037], "paragraph_keywords": ["evaluations", "protein", "experts", "frames"]}, {"paragraph_vector": [74.175163, -66.587326], "paragraph_keywords": ["protein", "ligand", "experts", "case"]}, {"paragraph_vector": [15.664717, -48.790451], "paragraph_keywords": ["ligands", "site", "tunnel", "data"]}, {"paragraph_vector": [16.442029, -46.31945], "paragraph_keywords": ["tunnel", "ligand", "water", "protein"]}, {"paragraph_vector": [21.687627, -51.309059], "paragraph_keywords": ["chloride", "ligand", "expert", "dcp"]}, {"paragraph_vector": [116.83702, -67.728904], "paragraph_keywords": ["simulation", "elements", "visualization", "measurements"]}, {"paragraph_vector": [74.432472, -69.665222], "paragraph_keywords": ["ligand", "observe", "driven", "water"]}, {"paragraph_vector": [126.834648, -63.183826], "paragraph_keywords": ["water", "speed", "studies", "helices"]}, {"paragraph_vector": [121.278335, -64.184944], "paragraph_keywords": ["focus", "events", "measurements", "elements"]}, {"paragraph_vector": [96.993949, -65.725318], "paragraph_keywords": ["field", "interaction", "protein", "visualizations"]}, {"paragraph_vector": [101.541763, -68.4011], "paragraph_keywords": ["simulations", "animations", "data", "simulation"]}, {"paragraph_vector": [134.003112, 44.261775], "paragraph_keywords": ["visualization", "john", "association", "wiley"]}], "content": {}, "doi": "10.1145/329129.329144"}, {"uri": "265", "title": "Voronoi-Based Extraction and Visualization of Molecular Paths", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Norbert Lindow", "Daniel Baum"], "summary": "Visual analysis is widely used to study the behavior of molecules. Of particular interest are the analysis of molecular interactions and the investigation of binding sites. For large molecules, however, it is difficult to detect possible binding sites and paths leading to these sites by pure visual inspection. In this paper, we present new methods for the computation and visualization of potential molecular paths. Using a novel filtering method, we extract the significant paths from the Voronoi diagram of spheres. For the interactive visualization of molecules and their paths, we present several methods using deferred shading and other state-of-theart techniques. To allow for a fast overview of reachable regions of the molecule, we illuminate the molecular surface using a large number of light sources placed on the extracted paths. We also provide a method to compute the extension surface of selected paths and visualize it using the skin surface. Furthermore, we use the extension surface to clip the molecule to allow easy visual tracking of even deeply buried paths. The methods are applied to several proteins to demonstrate their usefulness.", "keywords": ["cavity", "structure", "illumination", "end", "molecule", "et", "path", "distance", "voronoi", "graph", "presented", "protein", "algorithm", "compute", "point", "number", "cycle", "start", "diagram", "filter", "computation", "depth", "defined", "sphere", "surface", "based", "atom", "edge", "topology", "channel", "r", "use", "search", "radius", "grid", "vertex", "given", "al", "user", "rendering"], "document_vector": [-133.291046, -21.286712], "paragraphs": [{"paragraph_vector": [10.833292, -56.977333], "paragraph_keywords": ["paths", "binding", "interactions", "place"]}, {"paragraph_vector": [29.795217, 30.207036], "paragraph_keywords": ["paths", "voronoi", "diagram", "algorithm"]}, {"paragraph_vector": [14.655714, -22.443986], "paragraph_keywords": ["spheres", "path", "based", "surfaces"]}, {"paragraph_vector": [15.404726, -20.593072], "paragraph_keywords": ["protein", "et", "surfaces", "tools"]}, {"paragraph_vector": [55.617622, 36.508464], "paragraph_keywords": ["diagram", "voronoi", "molecule", "spheres"]}, {"paragraph_vector": [10.017951, 4.529232], "paragraph_keywords": ["voronoi", "spheres", "atom", "sphere"]}, {"paragraph_vector": [9.565279, 7.278576], "paragraph_keywords": ["vertex", "voronoi", "radius", "spheres"]}, {"paragraph_vector": [8.236929, 6.39869], "paragraph_keywords": ["sphere", "vertex", "tangent", "start"]}, {"paragraph_vector": [10.132874, 6.86201], "paragraph_keywords": ["edge", "plane", "curve", "ge"]}, {"paragraph_vector": [2.342706, 3.920962], "paragraph_keywords": ["grid", "vertex", "atoms", "cells"]}, {"paragraph_vector": [5.554783, 7.752496], "paragraph_keywords": ["vertex", "stack", "molecule", "algorithm"]}, {"paragraph_vector": [18.517921, 28.242372], "paragraph_keywords": ["voronoi", "molecule", "vertices", "points"]}, {"paragraph_vector": [2.578321, -5.405908], "paragraph_keywords": ["graph", "v", "topology", "voronoi"]}, {"paragraph_vector": [-1.802148, 5.156749], "paragraph_keywords": ["graph", "filter", "vertex", "vertices"]}, {"paragraph_vector": [-1.151449, 4.22078], "paragraph_keywords": ["r", "filter", "molecule", "gs"]}, {"paragraph_vector": [0.26901, 1.867259], "paragraph_keywords": ["step", "remove", "edges", "graph"]}, {"paragraph_vector": [-0.446867, 3.577874], "paragraph_keywords": ["cycle", "length", "filter", "paths"]}, {"paragraph_vector": [1.535302, 7.546689], "paragraph_keywords": ["cycle", "vertex", "intersection", "branch"]}, {"paragraph_vector": [1.827057, 2.905284], "paragraph_keywords": ["paths", "distance", "path", "vertex"]}, {"paragraph_vector": [10.741043, 9.7388], "paragraph_keywords": ["paths", "vertex", "based", "opengl"]}, {"paragraph_vector": [43.059844, 37.416866], "paragraph_keywords": ["surface", "spheres", "rendering", "visualize"]}, {"paragraph_vector": [48.94004, 38.593574], "paragraph_keywords": ["use", "paths", "surface", "lights"]}, {"paragraph_vector": [53.305881, 58.367481], "paragraph_keywords": ["radius", "light", "cell", "lights"]}, {"paragraph_vector": [65.073463, 34.383426], "paragraph_keywords": ["depth", "values", "surface", "structure"]}, {"paragraph_vector": [31.998348, 32.303169], "paragraph_keywords": ["number", "cores", "molecules", "voronoi"]}, {"paragraph_vector": [21.285472, 15.930166], "paragraph_keywords": ["filter", "handling", "seconds", "cycle"]}, {"paragraph_vector": [20.416299, 28.984842], "paragraph_keywords": ["paths", "size", "path", "performance"]}, {"paragraph_vector": [13.081001, -18.811027], "paragraph_keywords": ["paths", "use", "computation", "diagram"]}, {"paragraph_vector": [4.329772, 12.616114], "paragraph_keywords": ["paths", "cavities", "edge", "approach"]}, {"paragraph_vector": [47.341037, 34.87535], "paragraph_keywords": ["paths", "surface", "extension", "use"]}], "content": {}, "doi": "10.1145/2535571.2535594"}, {"uri": "266", "title": "empty", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "", "keywords": [], "document_vector": [21.12992, 11.433849], "paragraphs": [{"paragraph_vector": [-130.339706, 22.005044], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/1282040.1282062"}, {"uri": "267", "title": "THE DISPLAY OF MOLECULAR MODELS WITH THE AMES INTERACTIVE MODELING SYSTEM (AIMS)", "timestamp": "2003", "rating": "0.0", "annotation": "", "tags": [], "authors": ["JOHN T. EGAN"], "summary": "-The graphics capability (DISPLAYI of a large scale interactive molecular modeling system (AIMS) is described and discussed. DISPLAY allows the transparent use of a lower level graphics network GRANET. GRANET provides communication links and software packages unifying an aggregate of computers and graphics systems at Ames Research Center. The graphics devices used by DISPLAY are analyzed in terms of their display capabilities and other features. The Evans and Sutherland Picture System rE&S) with its interactive capabilities is seen to be one of the most powerful devices used by DISPLAY. The DISPLAY subsystem can present molecular models in any of three graphic formats--wireframe, ball and stick and space-filling. The software comprising DISPLAY provides five major features: (11 DIRECT--which allows models to be viewed directly from a molecular library: (2) STATIC--which provides a means of producing single plots on several types of recording media: (3~ DYNAMIC--which supports the production of multiple frame plot files: (41 BEZIER--where the representation of a model is given in terms of control points and parametric patches which define its surface: and tSl SCANLINE-whereby raster techniques are used to build an image of a model line by line. These features are discussed in terms of their software implementation, conceptual content, and application to ongoing research. It is found that despite certain limitations, the system provides a useful assortment of graphics which aid the researcher in model visualization and understanding. L INTtODUCTIOS A clearer understanding of molecular models can be promoted if visualization of the models can be obtained. The application of computer graphics for this purpose has been widely reviewed[I-10]. With the advent of sophisticated graphics devices supported by reasonably priced and powerful minicomputers, the chemist can interact with a display of a very large model, altering its structure by adding, deleting and moving selected segments. In addition to user interaction, the need often arises in a research environment for other ways of displaying information. These include the production of viewgraphs, film presentations, as well as publication quality prints of various models. To satisfy these needs, the display capability of the Ames Interactive Modeling System (AIMS) has been enhanced to provide a wide range of graphics and plotting capabilities. 2. OVERVIEW OF THE AIMS SYSTEM AIMS (Fig. 1) was designed to study molecular models within the context of chemical evolution. These models represent possible prebiotic schemes that can function without enzymatic assistance, that attempt to explain how modern genetic control systems were evolved and why they behave as they do. To assist this study it was necessary that AIMS be allowed to model both the structural aspects and the energetics of very large systems and to do so interactively. There are three major tCurrently at: Naval Research Laboratory, Washington, D.C. 20375. U.S.A. subsystems in AIMS. Each subsystem supports one or more major functions. The distinction being that a subsystem consists of software/hardware, while a function represents a major service performed by the software on the model. (i) A data preparation subsystem MOLECULE allows the building, editing, and maintenance of a data base representing model descriptions. Energy calculations can be performed on library entries and several types of displays may be obtained. This subsystem is described elsewhere [11]. (ii) A MINIMIZE subsystem permits the systematic modification of a model's structure and drives empirical energy calculations, under algorithm control, to achieve a reduction in potential energy[12]. The best possible structure is the one which minimizes this energy. An ,option exists to view either the entire process or to display selected steps in the procedure. (iii} A DISPLAY subsystem allows AIMS to utilize several types of display technologies. DISPLAY couples AIMS to a lower level Graphics Network--GRANET. Figure 2 shows an organizational view of GRANET. This comprehensive system uses communication linkages and software packages to combine a battery of computers. intelligent terminals and display generation hardware into a single system. Through this system AIMS gains access to: an interactive computer graphics device, an E&S picture system I[14]. which allows on-the-spot examination and modification of models; a high resolution COM device (DICOMED !)48), which produces high quality photographs, slides and films: less costly raster (TERAK, XEROX and VERSATEC plotter/printers) and storage tube (TEKTRONIX} devices which provide remote site viewing. 178 J.T. EGAN et al.", "keywords": [], "document_vector": [-58.072578, -23.031171], "paragraphs": [{"paragraph_vector": [-130.122573, 21.646335], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/3051519.3051521"}, {"uri": "268", "title": "Defining, Computing, and Visualizing Molecular Interfaces", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Amitabh Varshney", "Frederick P. Brooks", "David C. RichardsonS", "William V. Wright", "Dinesh Manochat"], "summary": "A parallel, analytic approach for defining and computing the interand intra-molecular interfaces in three dimencrions icr described. The molecular interface surfaces are derived from approzimations to the power-diagrams over the participating molecular units. For a given molecular interface our approach can generate a family of interface surfaces parametrized by a! and ,B, where a is the radius of the solvent molecule (also known as the probe-radius) and p is the interface radius that defines the size of the molecular interface. Molecular interface surfaces provide biochemists with a powerful tool to study surface complementarity and to efficiently characterize the interactions during a protein-substrate docking. The complexity of our algorithm for molecular environments is O(nk log\u2019 k), where n is the number of atome in the participating molecular units and k is the average number of neighboring atoms a constant, given (Y and /3.", "keywords": [], "document_vector": [-36.426319, -9.488595], "paragraphs": [{"paragraph_vector": [-130.019699, 21.762737], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/3154353.3154364"}, {"uri": "269", "title": "Biomedical Applications: From Data Capture to Modeling", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Cindy Grimm", "Elisabeta Marai"], "summary": "The incredible array of image and datacapture tools available to the scientific community is dramatically changing our understanding of biological processes. For example, imaging modalities such as computed tomography and magnetic resonance imaging let us visualize and track complex biological processes such as beating hearts. Data capture is only part of the story, however. Just as calculus helps physicists understand and model the mechanical world, computational models can help us model complex biological systems and reason and make predictions about them. Computer graphics and visualization techniques and algorithms\u2014from modeling to physically based animation\u2014make this process possible. This special issue is dedicated to the multidisciplinary effort needed to build, verify, and understand biological models. We start with \u201cMesh Processing in Medical-Image Analysis\u2014a Tutorial.\u201d Rather than completely covering state-of-the-art techniques, Joshua Levine and his colleagues focus on introductory material for new audiences: modern scanning modalities, constructing geometric models, building 2D and 3D meshes to represent these domains, and important biological applications that use this imageto-mesh pipeline. In a timely illustration of this pipeline, Nahyup Kang and his colleagues pre sent \u201cSimulating Liver Deformation during Respiration Using Sparse Local Features.\u201d Terry Yoo and his colleagues go a step further in \u201cVisualizing Cells and Humans in 3D: Biomedical Image Analysis at Nanometer and Meter Scales.\u201d Despite the difference in subject and scale, they find that the methods used for analysis and modeling are often remarkably similar. These methods derive from image processing, computer vision, and computer graphics techniques, including medical illustration, visualization, and rapid prototyping. From the applications end, in \u201cuPy: A Ubiquitous CG Python API with Biological-Modeling Applications,\u201d Ludovic Autin and his colleagues describe a publicly available Python extension module to simplify programming for science applications across graphics APIs such as Blender, Maya, Cinema4D, and DejaVu. In \u201cMolli: Interactive Visualization for Exploratory Protein Analysis,\u201d Sara Su and her colleagues describe a prototype system for interactive visualization and analysis of protein structures. Finally, in \u201cA Practical Workflow for Making Anatomical Atlases for Biological Research,\u201d Yong Wan and his colleagues use artistic tools to generate 3D anatomical atlases. In particular, they developed a mouse limb atlas to study the development of the mouse musculoskeletal system.", "keywords": ["content", "ieee", "activity", "vp", "email", "director", "president"], "document_vector": [132.825714, 5.166161], "paragraphs": [{"paragraph_vector": [-154.491607, 12.446002], "paragraph_keywords": ["director", "vp", "activities", "president"]}, {"paragraph_vector": [132.287536, 33.005287], "paragraph_keywords": ["ieee", "content", "computing", "graphics"]}], "content": {}, "doi": "10.1145/166117.166132"}, {"uri": "270", "title": "GPU-based Triangulation of the van der Waals Surface", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["S\u00e9rgio Dias"], "summary": "The problem addressed in this paper consists in triangulating the van der Waals surface without computing the geometric intersections of its atoms. Recall that the van der Waals surface is useful in computational molecular biology and biochemistry to, for example, determine the volume occupied by a molecule, as well as other important geometric properties. Assuming that every atom is represented by a ball, this amounts to compute the surface of the union of a number of balls. The novelty of our method lies in avoiding the computation of surface-surface intersections (SSI) of two or more balls. Keywords-CUDA and GPU Computing, Molecular Visualization, van der Waals Surface.", "keywords": ["van", "gpu", "der", "vertex", "algorithm", "surface", "waals", "time", "molecule", "point", "number", "atom", "array", "kernel", "voxel"], "document_vector": [-115.69815, -28.564395], "paragraphs": [{"paragraph_vector": [18.288196, 21.718374], "paragraph_keywords": ["surface", "point", "molecule", "van"]}, {"paragraph_vector": [21.258001, 35.234088], "paragraph_keywords": ["voxel", "vertex", "surface", "algorithm"]}, {"paragraph_vector": [22.130151, 64.024589], "paragraph_keywords": ["box", "voxel", "atom", "number"]}, {"paragraph_vector": [16.413082, 63.617317], "paragraph_keywords": ["surface", "voxel", "array", "vertices"]}, {"paragraph_vector": [20.345674, 49.489604], "paragraph_keywords": ["surface", "vertices", "triangle", "atom"]}, {"paragraph_vector": [-171.00888, 76.303985], "paragraph_keywords": ["time", "gpu", "program", "perform"]}, {"paragraph_vector": [58.191875, 43.5522], "paragraph_keywords": ["molecules", "molecule", "waals", "algorithm"]}], "content": {}, "doi": "10.1145/1044588.1044600"}, {"uri": "271", "title": "empty", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "", "keywords": [], "document_vector": [22.948213, 9.512829], "paragraphs": [{"paragraph_vector": [-123.515266, 26.692941], "paragraph_keywords": []}], "content": {}, "doi": "10.5555/976520.976547"}, {"uri": "272", "title": "Protein Chain Pair Simplification under the Discrete Fre\u0301chet Distance", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Tim Wylie", "Binhai Zhu"], "summary": "For protein structure alignment and comparison, a lot of work has been done using RMSD as the distance measure, which has drawbacks under certain circumstances. Thus, the discrete Fr\u00e9chet distance was recently applied to the problem of protein (backbone) structure alignment and comparison with promising results. For this problem, visualization is also important because protein chain backbones can have as many as 500-600 -carbon atoms, which constitute the vertices in the comparison. Even with an excellent alignment, the similarity of two polygonal chains can be difficult to visualize unless the chains are nearly identical. Thus, the chain pair simplification problem (CPS-3F) was proposed in 2008 to simultaneously simplify both chains with respect to each other under the discrete Fr\u00e9chet distance. The complexity of CPS-3F is unknown, so heuristic methods have been developed. Here, we define a variation of CPS-3F, called the constrained CPS-3F problem (CPS-3F\u00fe), and prove that it is polynomially solvable by presenting a dynamic programming solution, which we then prove is a factor-2 approximation for CPS-3F. We then compare the CPS-3F\u00fe solutions with previous empirical results, and further demonstrate some of the benefits of the simplified comparisons. Chain pair simplification based on the Hausdorff distance (CPS-2H) is known to be NP-complete, and here we prove that the constrained version (CPS-2H\u00fe) is also NP-complete. Finally, we discuss future work and implications along with a software library implementation, named the Fr\u00e9chet-based Protein Alignment & Comparison Toolkit (FPACT).", "keywords": ["chain", "alignment", "value", "structure", "k", "distance", "path", "cost", "similarity", "set", "method", "m", "protein", "algorithm", "approximation", "solution", "point", "number", "section", "c", "complexity", "defined", "rectangle", "simplification", "based", "simplify", "clause", "use", "pair", "vertex", "xi", "t", "time", "b", "given", "moving", "problem", "node"], "document_vector": [-102.118667, 46.127483], "paragraphs": [{"paragraph_vector": [-98.678771, 13.756443], "paragraph_keywords": ["protein", "structure", "rmsd", "chains"]}, {"paragraph_vector": [-81.696449, -27.629375], "paragraph_keywords": ["chains", "vertices", "distance", "protein"]}, {"paragraph_vector": [-104.467849, 25.170492], "paragraph_keywords": ["alignment", "protein", "problem", "simplification"]}, {"paragraph_vector": [-102.803993, 30.906312], "paragraph_keywords": ["section", "distance", "approximation", "hausdorff"]}, {"paragraph_vector": [-102.872253, 33.611095], "paragraph_keywords": ["distance", "paths", "b", "t"]}, {"paragraph_vector": [-102.958, 32.09769], "paragraph_keywords": ["distance", "chains", "problem", "dog"]}, {"paragraph_vector": [-103.077774, 32.006477], "paragraph_keywords": ["chains", "distance", "cost", "moving"]}, {"paragraph_vector": [-103.540077, 23.602005], "paragraph_keywords": ["distance", "problem", "dtw", "pair"]}, {"paragraph_vector": [-105.58319, 39.880355], "paragraph_keywords": ["c", "b", "vertices", "sx\u00f0xi"]}, {"paragraph_vector": [-103.959457, 40.29111], "paragraph_keywords": ["covered", "rectangles", "path", "c"]}, {"paragraph_vector": [-103.228332, 38.90343], "paragraph_keywords": ["zi", "solution", "jzij", "rectangle"]}, {"paragraph_vector": [-103.589447, 24.62152], "paragraph_keywords": ["time", "complexity", "rectangles", "chains"]}, {"paragraph_vector": [-104.476997, 37.840641], "paragraph_keywords": ["solution", "space", "nodes", "rectangles"]}, {"paragraph_vector": [-91.175971, 51.336219], "paragraph_keywords": ["function", "algorithm", "array", "rows"]}, {"paragraph_vector": [-90.079643, 51.751426], "paragraph_keywords": ["path", "rectangles", "rectangle", "pointers"]}, {"paragraph_vector": [-98.602066, 20.830476], "paragraph_keywords": ["chains", "algorithm", "length", "results"]}, {"paragraph_vector": [-87.253349, -34.523521], "paragraph_keywords": ["chains", "distance", "value", "algorithm"]}, {"paragraph_vector": [-102.654266, 25.452373], "paragraph_keywords": ["chains", "moving", "cost", "simplification"]}, {"paragraph_vector": [-86.235267, -34.419601], "paragraph_keywords": ["distance", "libraries", "c", "clause"]}, {"paragraph_vector": [-100.165298, 32.843833], "paragraph_keywords": ["clause", "t", "k", "sequence"]}, {"paragraph_vector": [-100.363456, 30.289381], "paragraph_keywords": ["t", "g", "df", "points"]}, {"paragraph_vector": [-102.274307, 28.778882], "paragraph_keywords": ["points", "t", "clause", "jbj"]}, {"paragraph_vector": [-104.306961, 23.814821], "paragraph_keywords": ["distance", "chains", "alignment", "simplification"]}, {"paragraph_vector": [-152.871536, 12.885156], "paragraph_keywords": ["project"]}], "content": {}, "doi": "10.1145/1144375.1144380"}, {"uri": "273", "title": "Protein Data Modelling for Concurrent Sequential Patterns", "timestamp": "2014", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Jing Lu", "Malcolm Keech", "Cuiqing Wang"], "summary": "Protein sequences from the same family typically share common patterns which imply their structural function and biological relationship. The challenge of identifying protein motifs is often addressed through mining frequent itemsets and sequential patterns, where post-processing is a useful technique. Earlier work has shown that Concurrent Sequential Patterns mining can be applied in bioinformatics, e.g. to detect frequently occurring concurrent protein sub-sequences. This paper presents a companion approach to data modelling and visualisation, applying it to real-world protein datasets from the PROSITE and NCBI databases. The results show the potential for graph-based modelling in representing the integration of higher level patterns common to all or nearly all of the protein sequences. Keywords\u2014protein sequences, data mining, concurrent sequential patterns (ConSP), bioinformatics, ConSP modelling, biological databases, knowledge representation, visualisation", "keywords": ["consp", "database", "length", "protein", "data", "family", "sequence", "pattern", "domain", "minsup", "mincon", "mining", "graph", "modelling"], "document_vector": [-131.2527, 81.796989], "paragraphs": [{"paragraph_vector": [-101.659149, -48.720573], "paragraph_keywords": ["protein", "patterns", "sequences", "mining"]}, {"paragraph_vector": [-103.155601, -48.484889], "paragraph_keywords": ["patterns", "sequence", "psdb", "database"]}, {"paragraph_vector": [-103.18518, -51.499423], "paragraph_keywords": ["patterns", "mining", "consp", "protein"]}, {"paragraph_vector": [-102.5447, -51.802127], "paragraph_keywords": ["protein", "databases", "graph", "pattern"]}, {"paragraph_vector": [-103.02182, -51.217582], "paragraph_keywords": ["protein", "family", "sequences", "patterns"]}, {"paragraph_vector": [-101.581047, -51.261463], "paragraph_keywords": ["protein", "patterns", "sequences", "length"]}, {"paragraph_vector": [-103.322113, -53.855094], "paragraph_keywords": ["mincon", "consps", "patterns", "data"]}, {"paragraph_vector": [-101.741584, -50.61412], "paragraph_keywords": ["minsup", "mining", "mincon", "patterns"]}], "content": {}, "doi": "10.1145/3093338.3093347"}, {"uri": "274", "title": "Visualization Methods for Molecular Studies on the Web Platform", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Marco Callieri", "Raluca Mihaela Andrei", "Marco Di Benedetto", "Monica Zopp\u00e8", "Roberto Scopigno"], "summary": "This work presents a technical solution for the creation of visualization schemes for biological data on the web platform. The proposed technology tries to overcome the standard approach of molecular/biochemical visualization tools, which generally provide a fixed set of visualization methods. This goal is reached by exploiting the capabilities of the WebGL API and the high level objects of the SpiderGL library, these features will give the users the possibility to implement an arbitrary visualization scheme, while keeping simple the implementation process. To better explain the philosophy and capabilities of this technology, we will describe the implementation of the web version of a specific visualization method, demonstrating how it can deal with both the requirements of scientific rigor in manipulating the data and the necessity to produce flexible and appealing rendering styles. CR Categories: I.3.2 [Graphics Systems]: Distributed/network graphics\u2014; J.3 [Life and Medical Sciences]: Biology and genetics \u2014;", "keywords": ["web", "data", "structure", "value", "platform", "molecule", "way", "movie", "method", "protein", "shading", "information", "research", "control", "u", "object", "biology", "process", "technique", "webgl", "shader", "surface", "based", "mapping", "level", "scene", "code", "line", "tool", "work", "library", "ability", "technology", "graphic", "s", "field", "r", "use", "file", "shaders", "t", "time", "function", "rendering", "visualization", "series", "spidergl", "javascript", "page"], "document_vector": [167.921707, -38.79076], "paragraphs": [{"paragraph_vector": [86.566322, -4.595744], "paragraph_keywords": ["rendering", "visualization", "data", "shading"]}, {"paragraph_vector": [133.419281, 33.827533], "paragraph_keywords": ["visualization", "web", "rendering", "browsers"]}, {"paragraph_vector": [131.678436, 36.617954], "paragraph_keywords": ["visualization", "webgl", "level", "use"]}, {"paragraph_vector": [87.521736, -16.120326], "paragraph_keywords": ["protein", "data", "movies", "tools"]}, {"paragraph_vector": [133.728622, 38.251831], "paragraph_keywords": ["web", "applets", "page", "graphics"]}, {"paragraph_vector": [132.863342, 39.135452], "paragraph_keywords": ["rendering", "level", "webgl", "use"]}, {"paragraph_vector": [129.773086, 42.253463], "paragraph_keywords": ["data", "spidergl", "objects", "management"]}, {"paragraph_vector": [132.44786, 46.020549], "paragraph_keywords": ["data", "object", "spidergl", "management"]}, {"paragraph_vector": [163.70346, 26.459226], "paragraph_keywords": ["data", "formats", "javascript", "support"]}, {"paragraph_vector": [134.667816, 40.244472], "paragraph_keywords": ["data", "code", "visualization", "javascript"]}, {"paragraph_vector": [130.815155, 37.696128], "paragraph_keywords": ["object", "spidergl", "canvas", "n"]}, {"paragraph_vector": [124.419082, 59.757465], "paragraph_keywords": ["t", "u", "s", "gl"]}, {"paragraph_vector": [125.744384, 56.971252], "paragraph_keywords": ["s", "t", "data", "r"]}, {"paragraph_vector": [134.549774, 38.085926], "paragraph_keywords": ["setup", "spidergl", "functions", "visualization"]}, {"paragraph_vector": [23.407247, -12.63974], "paragraph_keywords": ["surface", "water", "field", "understand"]}, {"paragraph_vector": [84.917488, -0.322439], "paragraph_keywords": ["mapping", "tools", "data", "particles"]}, {"paragraph_vector": [92.846092, 25.438808], "paragraph_keywords": ["rendering", "data", "use", "movie"]}, {"paragraph_vector": [63.627796, -4.520796], "paragraph_keywords": ["structure", "molecule", "surface", "file"]}, {"paragraph_vector": [85.559326, 10.708407], "paragraph_keywords": ["surface", "transparency", "effects", "representations"]}, {"paragraph_vector": [87.965454, 2.324923], "paragraph_keywords": ["lines", "surface", "method", "mapped"]}, {"paragraph_vector": [80.058021, 14.463504], "paragraph_keywords": ["lines", "particle", "field", "particles"]}, {"paragraph_vector": [72.640716, 17.533882], "paragraph_keywords": ["field", "work", "platform", "data"]}, {"paragraph_vector": [122.852478, 38.853382], "paragraph_keywords": ["visualization", "data", "techniques", "web"]}, {"paragraph_vector": [134.634368, 39.173469], "paragraph_keywords": ["visualization", "technology", "library", "use"]}], "content": {}, "doi": "10.1145/2808719.2811415"}, {"uri": "275", "title": "Visual Integration of Quantitative Proteomic Data, Pathways, and Protein Interactions", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Radu Jianu", "Kebing Yu", "Lulu Cao", "Vinh Nguyen", "Arthur R. Salomon", "David H. Laidlaw"], "summary": "We introduce several novel visualization and interaction paradigms for visual analysis of published protein-protein interaction networks, canonical signaling pathway models, and quantitative proteomic data. We evaluate them anecdotally with domain scientists to demonstrate their ability to accelerate the proteomic analysis process. Our results suggest that structuring protein interaction networks around canonical signaling pathway models, exploring pathways globally and locally at the same time, and driving the analysis primarily by the experimental data, all accelerate the understanding of protein pathways. Concrete proteomic discoveries within T-cells, mast cells, and the insulin signaling pathway validate the findings. The aim of the paper is to introduce novel protein network visualization paradigms and anecdotally assess the opportunity of incorporating them into established proteomic applications. We also make available a prototype implementation of our methods, to be used and evaluated by the proteomic community.", "keywords": ["prototype", "data", "exploration", "analysis", "signaling", "representation", "graph", "interaction", "method", "database", "protein", "algorithm", "network", "information", "research", "number", "approach", "result", "system", "researcher", "fig", "process", "technique", "pathway", "based", "drawing", "work", "computed", "phosphorylation", "design", "type", "experiment", "layout", "display", "interacting", "view", "hprd", "time", "image", "user", "plane", "visualization", "cell", "context", "node"], "document_vector": [31.221361, 67.164672], "paragraphs": [{"paragraph_vector": [-126.554489, -32.89244], "paragraph_keywords": ["cell", "protein", "researchers", "data"]}, {"paragraph_vector": [-124.984733, -34.916976], "paragraph_keywords": ["interaction", "university", "analysis", "department"]}, {"paragraph_vector": [-126.419479, -34.765544], "paragraph_keywords": ["protein", "visualization", "interaction", "drawing"]}, {"paragraph_vector": [-125.730644, -33.946128], "paragraph_keywords": ["protein", "network", "layout", "data"]}, {"paragraph_vector": [-131.602752, -38.532981], "paragraph_keywords": ["trees", "details", "graph", "viewing"]}, {"paragraph_vector": [-127.310058, -30.488147], "paragraph_keywords": ["pathway", "image", "data", "interaction"]}, {"paragraph_vector": [-40.858909, -44.332538], "paragraph_keywords": ["protein", "database", "image", "interaction"]}, {"paragraph_vector": [-125.633026, -31.418516], "paragraph_keywords": ["data", "protein", "changes", "experiment"]}, {"paragraph_vector": [-129.623092, -32.180702], "paragraph_keywords": ["proteins", "data", "network", "experiments"]}, {"paragraph_vector": [-130.788558, -38.90258], "paragraph_keywords": ["nodes", "proteins", "force", "layout"]}, {"paragraph_vector": [-129.922286, -39.90599], "paragraph_keywords": ["image", "proteins", "network", "pathway"]}, {"paragraph_vector": [-129.169509, -36.82059], "paragraph_keywords": ["view", "protein", "proteins", "exploration"]}, {"paragraph_vector": [-124.064895, -36.848575], "paragraph_keywords": ["proteins", "interaction", "data", "allow"]}, {"paragraph_vector": [-125.195526, -32.471683], "paragraph_keywords": ["phosphorylation", "protein", "data", "analysis"]}, {"paragraph_vector": [-125.634872, -33.250801], "paragraph_keywords": ["protein", "interaction", "pathway", "visualization"]}, {"paragraph_vector": [-128.913467, -35.539825], "paragraph_keywords": ["pathway", "proteins", "network", "user"]}, {"paragraph_vector": [-126.77983, -35.061508], "paragraph_keywords": ["view", "exploration", "network", "protein"]}, {"paragraph_vector": [-127.248802, -33.394126], "paragraph_keywords": ["list", "analysis", "proteins", "ones"]}, {"paragraph_vector": [-125.940093, -30.39068], "paragraph_keywords": ["protein", "researchers", "lck", "pathway"]}, {"paragraph_vector": [-128.138183, -40.647251], "paragraph_keywords": ["protein", "proteins", "image", "representation"]}, {"paragraph_vector": [-130.824142, -37.501937], "paragraph_keywords": ["exploration", "networks", "view", "views"]}, {"paragraph_vector": [-126.515556, -33.95824], "paragraph_keywords": ["view", "criteria", "proteins", "projection"]}, {"paragraph_vector": [-125.73001, -32.719272], "paragraph_keywords": ["data", "pathway", "protein", "interaction"]}], "content": {}, "doi": "10.1145/2483954.2483958"}, {"uri": "276", "title": "empty", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "We have fahcated silicon nano-needles o 140 nm diameter and 1.5 pm length for visualization of motor protein movement. Due to the geometry of the needles, they are ideal to amplify and visualize angular movement. To enable a well-defined and highly localized needle-protein connection, a specifically functionalized attachment point was included. For bioassay applications, mass production of needles in quantities of millions was implemented by a highly parallel fabrication process. Resulting needles were investigated in respect to uniformity and durability. Application of the nano-needles to motor protein motion are reported. Rotational movement of a single protein was successfully visualized by ,the needles. consuming; or strain induced tuberolling [8], which makes including a specific attachment point difficult. Finally, previously reported needles by conventional photolithography and etching do not result in the required dimensions [9, 101. Summarizing, these processes are not easily applicable to nano-needles for protein visualization, because of three issues: ( 3 ) amounts in the millions range should be quickly and easily producible, if the needles are to be used as tags. (2) the diameter (in the range of 100-200 nm) and length (1-2 pn) of the needles should be well controllable and uniform. (3) highly localized surface modification is necessary for protein attachment, to avoid random attachment. A novel approach for fulfilling all these requirements is reported. 2. FABRICATION", "keywords": [], "document_vector": [89.441818, 27.782207], "paragraphs": [{"paragraph_vector": [-129.856506, 21.951871], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/3359996.3365050"}, {"uri": "277", "title": "Introducing Protein 3-D Visualization Software to Freshman Undergraduate Students: Making Connections and Building Skills", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["A.M Brown"], "summary": "The structure and dynamics of proteins are an essential part of understanding the molecular foundations of complex biological processes and serve an important role in the field of computational biology. Biomolecular visualization software can serve as an entryway to begin exploration of, and expose students to, protein structure-function relationships and aid in their development in basic science knowledge. In addition, technical skills and effective writing and presentation of scientific material are critical for students entering the field of bioinformatics and computational biology. Training and education utilizing biomolecular visualization software and honing writing and presentation skills are often reserved for special studies or higher-level coursework. Presenting more advanced concepts and skills that can connect ideas from introductory level classes in chemistry, biology, and physics earlier in the curriculum is imperative to success in advanced classes and application in research settings for undergraduate students. Thus, student-centered activities that can satisfy the development of new skills and critical thinking, in addition to computational knowledge and practice, provide foundational principles for developing future scientists and increasing their chances of success. By designing a one-credit hour, introduction to biomolecular visualization course, freshman biochemistry undergraduate students were exposed to higher-level thinking and application, and gained skills in biomolecular visualization and scientific presentation. In addition, biomolecular visualization and molecular modeling were used to introduce the students to the field of computational biology and computational skills such as command-line usage, unix interfaces, and parallel computing. Permission to make digital or hard copies of all or part of this work for personal or classroom use is granted without fee provided that copies are not made or distributed for profit or commercial advantage and that copies bear this notice and the full citation on the first page. Copyrights for components of this work owned by others than ACM must be honored. Abstracting with credit is permitted. To copy otherwise, or republish, to post on servers or to redistribute to lists, requires prior specific permission and/or a fee. Request permissions from Permissions@acm.org. PEARC17, July 09-13, 2017, New Orleans, LA, USA, \u00a9 2017 Association for computing Machinery. ACM ISBN 978-1-4503-5272-7/17/07$15.00 http://dx.doi.org/10.1145/3093338.3093347 Overall, this study discusses the benefits of introducing bimolecular visualization software early in the undergraduate education curriculum to a small pilot group and the potential for implementation on a larger scale to prepare students by providing discipline-specific foundations relevant to the use of computational tools. We share this work as a means for starting the discussion on updating undergraduate education and curriculum to reflect computational needs and realities to best prepare and develop students for the workforce. CCS CONCEPTS \u2022 Applied computing~Molecular structural biology \u2022 Applied computing~Collaborative learning", "keywords": ["group", "structure", "program", "concept", "presentation", "resource", "computing", "outcome", "protein", "experience", "freshman", "research", "biology", "introduction", "topic", "work", "level", "student", "skill", "learning", "tool", "biochemistry", "bioinformatics", "software", "related", "semester", "image", "function", "visualization", "provided", "course"], "document_vector": [-131.887939, 20.432685], "paragraphs": [{"paragraph_vector": [-172.209991, -6.01818], "paragraph_keywords": ["structure", "protein", "function", "students"]}, {"paragraph_vector": [-171.152008, -6.285487], "paragraph_keywords": ["skills", "understanding", "students", "structure"]}, {"paragraph_vector": [-169.671493, -7.875412], "paragraph_keywords": ["course", "students", "biology", "presented"]}, {"paragraph_vector": [-170.979598, -5.60662], "paragraph_keywords": ["students", "research", "ability", "protein"]}, {"paragraph_vector": [-171.152954, -6.562704], "paragraph_keywords": ["students", "visualization", "images", "software"]}, {"paragraph_vector": [-170.670669, -7.662529], "paragraph_keywords": ["protein", "students", "semester", "structure"]}, {"paragraph_vector": [-171.083068, -5.59231], "paragraph_keywords": ["course", "students", "protein", "student"]}, {"paragraph_vector": [-170.990661, -6.164455], "paragraph_keywords": ["students", "programs", "protein", "computing"]}, {"paragraph_vector": [-170.73413, -6.148008], "paragraph_keywords": ["students", "course", "group", "biology"]}, {"paragraph_vector": [-169.863876, -5.044693], "paragraph_keywords": ["course", "students", "research", "student"]}, {"paragraph_vector": [-170.365249, -4.103806], "paragraph_keywords": ["students", "student", "peer", "course"]}, {"paragraph_vector": [-172.804809, -5.879268], "paragraph_keywords": ["students", "biology", "biochemistry", "concepts"]}, {"paragraph_vector": [-171.385925, -5.184152], "paragraph_keywords": ["students", "course", "visualization", "research"]}], "content": {}, "doi": "10.1145/952532.952554"}, {"uri": "278", "title": "Loose Capacity-Constrained Representatives for the Qualitative Visual Analysis in Molecular Dynamics", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Steffen Frey", "Thomas Schl\u00f6mer", "Sebastian Grottel", "Carsten Dachsbacher", "Oliver Deussen", "Thomas Ertl"], "summary": "Molecular dynamics is a widely used simulation technique to investigate material properties and structural changes under external forces. The availability of more powerful clusters and algorithms continues to increase the spatial and temporal extents of the simulation domain. This poses a particular challenge for the visualization of the underlying processes which might consist of millions of particles and thousands of time steps. Some application domains have developed special visual metaphors to only represent the relevant information of such data sets but these approaches typically require detailed domain knowledge that might not always be available or applicable. We propose a general technique that replaces the huge amount of simulated particles by a smaller set of representatives that are used for the visualization instead. The representatives capture the characteristics of the underlying particle density and exhibit coherency over time. We introduce loose capacity-constrained Voronoi diagrams for the generation of these representatives by means of a GPU-friendly, parallel algorithm. This way we achieve visualizations that reflect the particle distribution and geometric structure of the original data very faithfully. We evaluate our approach using real-world data sets from the application domains of material science, thermodynamics and dynamical systems theory.", "keywords": ["visualization", "mean", "representative", "data", "group", "p", "structure", "swapping", "k", "voronoi", "tree", "application", "set", "method", "cluster", "m", "lccvd", "l", "algorithm", "particle", "point", "number", "approach", "result", "clustering", "sec", "diagram", "step", "fig", "density", "quality", "based", ".", "site", "constraint", "s", "gpu", "pair", "time", "rendering", "capacity"], "document_vector": [24.433187, -47.831817], "paragraphs": [{"paragraph_vector": [91.968231, 3.669672], "paragraph_keywords": ["particles", "data", "capacity", "set"]}, {"paragraph_vector": [-87.013404, 70.448799], "paragraph_keywords": ["site", "points", "ccvds", "sec"]}, {"paragraph_vector": [-73.514358, -13.255625], "paragraph_keywords": ["clustering", "points", "cluster", "set"]}, {"paragraph_vector": [90.907043, -28.073263], "paragraph_keywords": ["visualization", "data", "sets", "particle"]}, {"paragraph_vector": [-86.67324, 24.001907], "paragraph_keywords": ["points", "capacity", "sites", "interval"]}, {"paragraph_vector": [-83.935684, 26.025114], "paragraph_keywords": ["points", "sites", "site", "point"]}, {"paragraph_vector": [-102.1147, 70.058441], "paragraph_keywords": ["sites", "site", "data", "point"]}, {"paragraph_vector": [-92.463531, 68.221633], "paragraph_keywords": ["sites", "site", "points", "swapping"]}, {"paragraph_vector": [-96.412361, 70.105552], "paragraph_keywords": ["tree", "sites", "s", "group"]}, {"paragraph_vector": [-95.183929, 69.189788], "paragraph_keywords": ["sites", "pairs", "swapping", "points"]}, {"paragraph_vector": [-93.588653, 65.444656], "paragraph_keywords": ["points", "point", "slot", "slots"]}, {"paragraph_vector": [-70.603492, 20.807903], "paragraph_keywords": ["points", "sites", "site", "covered"]}, {"paragraph_vector": [-95.903907, 71.466384], "paragraph_keywords": ["r", "sites", "point", "data"]}, {"paragraph_vector": [-121.246673, 72.265983], "paragraph_keywords": ["time", "particle", "quality", "data"]}, {"paragraph_vector": [98.531867, 72.351242], "paragraph_keywords": ["data", "fig", "set", "points"]}, {"paragraph_vector": [86.658538, -46.148582], "paragraph_keywords": ["l", "collision", "constraint", "sites"]}, {"paragraph_vector": [-96.255493, 65.314407], "paragraph_keywords": ["data", "quality", "sites", "results"]}, {"paragraph_vector": [-110.664993, 72.990371], "paragraph_keywords": ["time", "quality", "set", "data"]}, {"paragraph_vector": [86.649757, 38.87939], "paragraph_keywords": ["data", "proceedings", "pages", "based"]}, {"paragraph_vector": [107.192062, 10.925081], "paragraph_keywords": ["pages", "reduction", "visualization", "wang"]}], "content": {}, "doi": "10.1109/VISUAL.2001.964547"}, {"uri": "279", "title": "Lightweight Computational Steering of Very Large Scale Molecular Dynamics Simulations", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["David M. Beazley", "Peter S. Lomdahl"], "summary": "We present a computational steering approach for controlling, analyzing, and visualizing very large scale molecular dynamics simulations involving tens to hundreds of millions of atoms. Our approach relies on extensible scripting languages and an easy to use tool for building extensions and modules. The system is easy to modify, works with existing C code, is memory efficient, and can be used from inexpensive workstations over standard Internet connections. We demonstrate how we have been able to explore data from production MD simulations involving as many as 104 million atoms running on the CM-5 and Cray T3D. We also show how this approach can be used to integrate common scripting languages (including Python, Tcl/Tk, and Perl), simulation code, user extensions, and commercial data analysis packages.", "keywords": ["data", "spasm", "md", "language", "analysis", "scripting", "simulation", "computing", "set", "processor", "memory", "particle", "approach", "run", "scale", "system", "c", "code", "workstation", "interface", "work", "atom", "command", "file", "use", "steering", "feel", "time", "build", "image", "user", "function", "visualization"], "document_vector": [-37.64849, -4.723756], "paragraphs": [{"paragraph_vector": [-134.4682, 50.497676], "paragraph_keywords": ["simulations", "atoms", "simulation", "dynamics"]}, {"paragraph_vector": [-135.422882, 51.871635], "paragraph_keywords": ["spasm", "simulations", "simulation", "code"]}, {"paragraph_vector": [145.709411, 57.822254], "paragraph_keywords": ["data", "analysis", "system", "users"]}, {"paragraph_vector": [-136.102905, 51.168579], "paragraph_keywords": ["data", "problem", "simulation", "analysis"]}, {"paragraph_vector": [-135.539428, 52.229881], "paragraph_keywords": ["user", "system", "simulations", "steering"]}, {"paragraph_vector": [-133.419296, 53.856697], "paragraph_keywords": ["system", "approach", "simulation", "code"]}, {"paragraph_vector": [-150.498825, 49.159339], "paragraph_keywords": ["language", "functions", "scripting", "commands"]}, {"paragraph_vector": [-121.089385, 44.383789], "paragraph_keywords": ["interface", "c", "user", "functions"]}, {"paragraph_vector": [-121.840751, 51.10865], "paragraph_keywords": ["user", "particle", "interface", "code"]}, {"paragraph_vector": [-119.770912, 46.827919], "paragraph_keywords": ["code", "functions", "particles", "scripting"]}, {"paragraph_vector": [-120.900245, 46.44054], "paragraph_keywords": ["spasm", "set", "data", "code"]}, {"paragraph_vector": [132.129302, 44.648799], "paragraph_keywords": ["data", "simulation", "visualize", "spasm"]}, {"paragraph_vector": [-120.491828, 47.550487], "paragraph_keywords": ["spasm", "simulation", "workstation", "mbytes"]}, {"paragraph_vector": [-134.29013, 54.783744], "paragraph_keywords": ["analysis", "visualization", "datafiles", "data"]}, {"paragraph_vector": [-120.639602, 47.048835], "paragraph_keywords": ["approach", "works", "scripting", "spasm"]}, {"paragraph_vector": [-135.790145, 52.358371], "paragraph_keywords": ["computing", "work", "university", "data"]}, {"paragraph_vector": [-144.576797, 55.90921], "paragraph_keywords": ["condensed", "physics", "matter", "materials"]}], "content": {}, "doi": "10.1109/VISUAL.1995.480794"}, {"uri": "280", "title": "Visualization of Interactions Between Organic Polymer Surfaces and Ion Beams Obtained From Molecular Dynamics Simulations", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Hideaki Yamada", "Satoshi Hamaguchi"], "summary": "Molecular dynamics simulation combined with sophisticated visualization techniques may be one of the most powerful scientific tools for the study of atomic-level surface reactions during plasma-wall interactions. This paper shows visualization of an organic polymer model substrate before and during molecular beam injections. The classical interatomic potential model functions used in the present work are developed to describe chemical reactions, i.e., formation and breakup of chemical bonds under strongly nonthermal-equilibrium conditions. We have observed that, during beam injections, nonthermal-equilibrium chemical reactions take place in a nano-scale thin layer of the substrate top surface, which determines characteristics of the process, such as etching/deposition rates and selectivity.", "keywords": [], "document_vector": [55.716175, -43.368244], "paragraphs": [{"paragraph_vector": [-130.122741, 21.482582], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/3011141.3011222"}, {"uri": "281", "title": "DYNAMICS OF INDIVIDUAL DEOXYRIBONUCLEIC ACID MOLECULES VISUALIZED IN SITU WITH SHEAR STRALN", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Philip LeDuc", "Andrew Tsourkas", "Denis Wirtz", "Gang Bao"], "summary": "Many groups have investigated macromolecular dynamics under strain in the past due to its importance in numerous fields. The prediction of the conformation of individual molecules in shear is still heartily discussed today [1,2]. We report the fist real-time imaging of the dynamics of individual DNA molecules under a welldefined shear flow. Individual DNA molecules are placed under a shear flow while visualizing their complex dynamic conformations in real-time. DNA molecules are larger than the micrometer scale and thus can be visualized using optical microscopy via fluorochromes [3,4]. This novel experiment allows us to understand the structural deformations of individual DNA molecules. This opens up a new world of real-time visualization of individual polymers under shear and reveals a completely new and fundamental insight into the behavior of individual polymers under shear with potential applications in fields from biology, to mechanics, to medicine. MATERIALS AND METHODS A dilute solution of Coliphage T2 DNA are stained with a yellow dye (YOYO-I, Molecular Probes) and then placed on a fured coverslide. The coverslide is attached to a computer-controlled slidiag stage and centered under an upright microscope seen in Fig. 1. Two-dimensional realtime image data is captured using a system consisting of a SIT camera, an Argus-10 processor, S-VHS video, and NIH Image 1.60. RESULTS The DNA is statistically categorized in conformations involving two distinct classes of vertical and horizontal conformations with seven subclasses after systematic observations of individual DNA molecules under shear. While the extended orientation in the -ion of shear is predicted by classical theories and has been previously observed, the presence of vertical conformations is unexpected and is not considered by classical models of polymer physics [1,2]. This unexpected stretching of individual DNA molecules in the direction parallel to the flow direction can be possibly explained by the viscous drag exerted on the molecule by the shearing of the fluid. Individual polymers under shear can change both size and orientation dynamically. Further, a DNA molecule initially oriented in the direction perpendicular to the flow direction can remain in an overall vertical orientation while still undergoing multiple changes of conformational classes under a constant shear rate. DISCUSSION These DNA molecules under shear can be categorized into distinct conformational classes. An individual polymer can travel in the direction of the flow with an overall orientation perpendicular to the flow direction for time scales larger than their relaxation time. Still, ensembleaveraged and time-averaged mesnrements of conformational parameters agree with classical bulk measurements on sheared dilute polymer solutions [2]. These results will have a significant impact in polymer physics models in the future.", "keywords": [], "document_vector": [85.5372, 28.712289], "paragraphs": [{"paragraph_vector": [-129.548416, 21.229455], "paragraph_keywords": []}], "content": {}, "doi": "10.1145/3307339.3342137"}, {"uri": "282", "title": "Visualizing Motional Correlations in Molecular Dynamics using Geometric Deformations", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["M. Fioravante"], "summary": "In macromolecules, an allosteric effect is said to occur when a change at one site of a molecule affects a distant site. Understanding these allosteric effects can be important for understanding how the functions of complex molecules such as proteins are regulated. One potential application of this knowledge is the development of small molecules that alter the function of proteins involved in diseases. Studying motional correlation can help researchers to discover how a change at a source site affects the target site and thus how allosteric ligands that could serve as drugs are able to exert their therapeutic effects. By improving our ability to analyze these correlated relationships, it may be possible to develop new medications to combat deadly diseases such as Hepatitis C. We present four visual techniques which represent motional correlation on rendered three-dimensional molecular models, providing new ways to view clusters of correlated residues and paths of allosteric interactions. These techniques give us a new way of investigating the presence of motional correlations in complex molecules. We compare each of these techniques to determine which are the most useful for representing motional correlations.", "keywords": ["data", "value", "correlation", "motion", "bound", "molecule", "path", "set", "space", "method", "described", "protein", "matrix", "visualizing", "covariance", "ligand", "approach", "number", "c", "clustering", "figure", "selected", "relationship", "ray", "technique", "inhibitor", "surface", "binding", "enzyme", "study", "performance", "site", "cut", "user", "visualization", "provide", "residue"], "document_vector": [-127.165817, -42.727851], "paragraphs": [{"paragraph_vector": [73.303283, -49.473743], "paragraph_keywords": ["molecule", "site", "correlation", "ligand"]}, {"paragraph_vector": [142.518432, -78.964706], "paragraph_keywords": ["pathway", "molecule", "site", "covariance"]}, {"paragraph_vector": [-85.377609, -84.440109], "paragraph_keywords": ["matrix", "correlation", "residue", "covariance"]}, {"paragraph_vector": [99.958625, -89.070091], "paragraph_keywords": ["inhibitor", "site", "effects", "c"]}, {"paragraph_vector": [105.607292, -38.674449], "paragraph_keywords": ["representations", "tools", "visualization", "based"]}, {"paragraph_vector": [-121.053108, -82.466041], "paragraph_keywords": ["molecule", "data", "motion", "techniques"]}, {"paragraph_vector": [-81.551025, -80.852226], "paragraph_keywords": ["set", "residues", "pca", "component"]}, {"paragraph_vector": [-113.110404, -84.345916], "paragraph_keywords": ["residues", "user", "set", "residue"]}, {"paragraph_vector": [-164.232543, -86.295379], "paragraph_keywords": ["correlation", "ray", "residues", "spheres"]}, {"paragraph_vector": [-155.534484, -83.885902], "paragraph_keywords": ["correlation", "value", "sphere", "figure"]}, {"paragraph_vector": [-175.459106, -87.694625], "paragraph_keywords": ["residue", "correlation", "molecule", "selected"]}, {"paragraph_vector": [172.393692, -85.923141], "paragraph_keywords": ["path", "paths", "binding", "correlation"]}, {"paragraph_vector": [-176.783874, -83.152992], "paragraph_keywords": ["s", "search", "path", "c"]}, {"paragraph_vector": [105.181838, -1.510784], "paragraph_keywords": ["casting", "implemented", "method", "molecule"]}, {"paragraph_vector": [162.695709, -79.113983], "paragraph_keywords": ["performance", "visualization", "cut", "rendered"]}, {"paragraph_vector": [167.052902, -71.250869], "paragraph_keywords": ["enzyme", "dynamics", "found", "number"]}, {"paragraph_vector": [176.213302, -86.134086], "paragraph_keywords": ["approach", "sites", "nodes", "distance"]}, {"paragraph_vector": [164.880661, -88.039733], "paragraph_keywords": ["enzyme", "binding", "inhibitor", "gtp"]}, {"paragraph_vector": [132.469863, -82.709724], "paragraph_keywords": ["visualization", "represented", "investigation", "applied"]}], "content": {}, "doi": "10.1145/3010915.3011851"}, {"uri": "283", "title": "GPU-Accelerated Molecular Visualization on Petascale Supercomputing Platforms", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["John E. Stone", "Kirby L. Vandivort", "Klaus Schulten"], "summary": "Petascale supercomputers create new opportunities for the study of the structure and function of large biomolecular complexes such as viruses and photosynthetic organelles, permitting all-atom molecular dynamics simulations of tens to hundreds of millions of atoms. Together with simulation and analysis, visualization provides researchers with a powerful \u201ccomputational microscope\u201d. Petascale molecular dynamics simulations produce tens to hundreds of terabytes of data that can be impractical to transfer to remote facilities, making it necessary to perform visualization and analysis tasks in-place on the supercomputer where the data are generated. We describe the adaptation of key visualization features of VMD, a widely used molecular visualization and analysis tool, for GPU-accelerated petascale computers. We discuss early experiences adapting ray tracing algorithms for GPUs, and compare rendering performance for recent petascale molecular simulation test cases on Cray XE6 (CPUonly) and XK7 (GPU-accelerated) compute nodes. Finally, we highlight opportunities for further algorithmic improvements and optimizations.", "keywords": ["data", "structure", "petascale", "acceleration", "cuda", "execution", "accelerated", "representation", "optix", "hardware", "movie", "lighting", "vmd", "memory", "algorithm", "approach", "number", "primitive", "system", "density", "ray", "code", "surface", "map", "engine", "work", "complex", "performance", "water", "cpu", "atom", "type", "gpu", "use", "required", "time", "gpus", "tachyon", "unit", "rendering", "visualization", "tracing", "node"], "document_vector": [-5.734408, -43.965362], "paragraphs": [{"paragraph_vector": [144.030624, 45.908847], "paragraph_keywords": ["visualization", "gpus", "microscope", "systems"]}, {"paragraph_vector": [158.841781, 74.889472], "paragraph_keywords": ["gpu", "memory", "surface", "visualization"]}, {"paragraph_vector": [57.797916, 38.482746], "paragraph_keywords": ["ray", "structure", "details", "densities"]}, {"paragraph_vector": [165.214569, 66.028976], "paragraph_keywords": ["storage", "memory", "displaydevice", "primitives"]}, {"paragraph_vector": [-176.497741, 73.232582], "paragraph_keywords": ["gpu", "algorithms", "units", "representations"]}, {"paragraph_vector": [117.169433, 66.608795], "paragraph_keywords": ["surface", "algorithm", "particle", "counts"]}, {"paragraph_vector": [-167.046585, 70.39212], "paragraph_keywords": ["gpu", "algorithm", "memory", "map"]}, {"paragraph_vector": [20.45646, 61.851139], "paragraph_keywords": ["density", "map", "type", "produced"]}, {"paragraph_vector": [-170.160354, 74.204597], "paragraph_keywords": ["memory", "surface", "algorithm", "type"]}, {"paragraph_vector": [129.70465, 74.260482], "paragraph_keywords": ["rendering", "vmd", "performance", "tachyon"]}, {"paragraph_vector": [154.288299, 77.620185], "paragraph_keywords": ["gpu", "ray", "simd", "data"]}, {"paragraph_vector": [128.165191, 76.6846], "paragraph_keywords": ["optix", "gpu", "code", "engine"]}, {"paragraph_vector": [-176.782257, 85.349357], "paragraph_keywords": ["gpu", "ray", "memory", "required"]}, {"paragraph_vector": [131.472534, 75.024467], "paragraph_keywords": ["gpu", "ray", "samples", "tracing"]}, {"paragraph_vector": [142.454071, 73.513832], "paragraph_keywords": ["rendering", "nodes", "increases", "movie"]}, {"paragraph_vector": [149.092315, 74.339279], "paragraph_keywords": ["movie", "performance", "node", "gpu"]}], "content": {}, "doi": "10.1145/2335755.2335826"}, {"uri": "284", "title": "PPIExtractor: A Protein Interaction Extraction and Visualization System for Biomedical Literature", "timestamp": "2013", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Zhihao Yang", "Zhehuan Zhao", "Yanpeng Li", "Yuncui Hu", "Hongfei Lin"], "summary": "Protein-protein interactions (PPIs) play a key role in various aspects of the structural and functional organization of the cell. Knowledge about them unveils the molecular mechanisms of biological processes. However, the amount of biomedical literature regarding protein interactions is increasing rapidly and it is difficult for interaction database curators to detect and curate protein interaction information manually. In this paper, we present a PPI extraction system, termed PPIExtractor, which automatically extracts PPIs from biomedical text and visualizes them.Given aMedline record dataset, PPIExtractor first applies Feature Coupling Generalization (FCG) to tag protein names in text, next uses the extended semantic similarity-based method to normalize them, then combines feature-based, convolution tree and graph kernels to extract PPIs, and finally visualizes the PPI network. Experimental evaluations show that PPIExtractor can achieve state-of-the-art performance on a DIP subset with respect to comparable evaluations. PPIExtractor is freely available for academic purposes at: http://202.118.75.18:8080/PPIExtractor/.", "keywords": ["tree", "similarity", "graph", "interaction", "gene", "set", "method", "protein", "network", "information", "combination", "normalization", "module", "extraction", "ppi", "ner", "extracted", "example", "based", "recall", "performance", "dip", "-", "kernel", "use", "feature", "name", "ppis", "ppiextractor", "task"], "document_vector": [14.940867, 52.261817], "paragraphs": [{"paragraph_vector": [-104.506767, -5.975566], "paragraph_keywords": ["features", "module", "gene", "-"]}, {"paragraph_vector": [-107.679344, -7.287729], "paragraph_keywords": ["gene", "normalization", "genes", "names"]}, {"paragraph_vector": [-110.058258, -7.55397], "paragraph_keywords": ["gene", "information", "context", "vector"]}, {"paragraph_vector": [-98.224792, -8.988072], "paragraph_keywords": ["kernels", "kernel", "gene", "based"]}, {"paragraph_vector": [-101.574676, -9.753839], "paragraph_keywords": ["kernel", "spt", "tree", "based"]}, {"paragraph_vector": [-101.899658, -9.071571], "paragraph_keywords": ["kernel", "tree", "output", "parse"]}, {"paragraph_vector": [-103.814476, -10.033174], "paragraph_keywords": ["performance", "kernels", "kernel", "ppi"]}, {"paragraph_vector": [-101.270019, -9.511387], "paragraph_keywords": ["kernel", "feature", "tree", "performance"]}, {"paragraph_vector": [-101.516624, -11.271365], "paragraph_keywords": ["dip", "ppiextractor", "interactions", "interaction"]}, {"paragraph_vector": [-103.64669, -9.463167], "paragraph_keywords": ["ppiextractor", "performance", "bioppisvmextractor", "normalization"]}, {"paragraph_vector": [-104.469703, -8.844959], "paragraph_keywords": ["recall", "performance", "grammar", "method"]}, {"paragraph_vector": [-103.018035, -9.364873], "paragraph_keywords": ["ner", "recall", "errors", "protein"]}, {"paragraph_vector": [-121.589027, -20.668409], "paragraph_keywords": ["network", "ppi", "layout", "protein"]}, {"paragraph_vector": [-104.270065, -9.049407], "paragraph_keywords": ["ppi", "ppiextractor", "extraction", "based"]}], "content": {}, "doi": "10.1145/369028.369136"}, {"uri": "285", "title": "Neuroinformatics for Genome-Wide 3D Gene Expression Mapping in the Mouse Brain", "timestamp": "2007", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Lydia Ng", "Sayan D. Pathak", "Chihchau Kuan", "Christopher Lau", "Hongwei Dong", "Andrew Sodt", "Chinh Dang", "Brian Avants", "Paul Yushkevich", "James C. Gee", "David Haynor", "Ed Lein", "Allan Jones", "Mike Hawrylycz"], "summary": "Large-scale gene expression studies in the mammalian brain offer the promise of understanding the topology, networks, and, ultimately, the function of its complex anatomy, opening previously unexplored avenues in neuroscience. High-throughput methods permit genome-wide searches to discover genes that are uniquely expressed in brain circuits and regions that control behavior. Previous gene expression mapping studies in model organisms have employed in situ hybridization (ISH), a technique that uses labeled nucleic acid probes to bind to specific mRNA transcripts in tissue sections. A key requirement for this effort is the development of fast and robust algorithms for anatomically mapping and quantifying gene expression for ISH. We describe a neuroinformatics pipeline for automatically mapping expression profiles of ISH data and its use to produce the first genomic scale 3D mapping of gene expression in a mammalian brain. The pipeline is fully automated and adaptable to other organisms and tissues. Our automated study of more than 20,000 genes indicates that at least 78.8 percent are expressed at some level in the adult C56BL/6J mouse brain. In addition to providing a platform for genomic scale search, high-resolution images and visualization tools for expression analysis are available at the Allen Brain Atlas web site (http://www.brain-map.org).", "keywords": ["registration", "data", "volume", "structure", "automated", "pipeline", "ish", "gene", "expression", "method", "reconstruction", "information", "number", "approach", "mask", "scale", "reference", "section", "intensity", "template", "object", "result", "annotation", "process", "striatum", "resolution", "based", "atlas", "mapping", "level", "aba", "allen", "tissue", "mouse", "search", "brain", "region", "processing", "found", "image", "expressing", "series", "cell"], "document_vector": [44.888149, 84.050964], "paragraphs": [{"paragraph_vector": [-84.261848, 44.751281], "paragraph_keywords": ["expression", "gene", "system", "brain"]}, {"paragraph_vector": [-69.455833, 48.869628], "paragraph_keywords": ["image", "including", "-", "expression"]}, {"paragraph_vector": [-81.143684, 45.380287], "paragraph_keywords": ["gene", "expression", "brain", "sections"]}, {"paragraph_vector": [-73.080146, 48.43613], "paragraph_keywords": ["atlas", "image", "section", "brain"]}, {"paragraph_vector": [-65.773818, 55.332386], "paragraph_keywords": ["volume", "template", "annotation", "registration"]}, {"paragraph_vector": [-76.360664, 47.998725], "paragraph_keywords": ["registration", "images", "series", "intensity"]}, {"paragraph_vector": [-82.466232, 41.038085], "paragraph_keywords": ["registration", "shape", "information", "expression"]}, {"paragraph_vector": [-67.264884, 54.427318], "paragraph_keywords": ["registration", "section", "step", "found"]}, {"paragraph_vector": [-85.502723, 41.148872], "paragraph_keywords": ["objects", "tissue", "expression", "intensity"]}, {"paragraph_vector": [-87.736938, 41.290477], "paragraph_keywords": ["cell", "combined", "objects", "found"]}, {"paragraph_vector": [-76.280807, 46.526451], "paragraph_keywords": ["expression", "image", "grid", "atlas"]}, {"paragraph_vector": [-79.960739, 42.500896], "paragraph_keywords": ["genes", "expression", "striatum", "list"]}, {"paragraph_vector": [-81.505004, 43.570819], "paragraph_keywords": ["expression", "gene", "brain", "atlas"]}, {"paragraph_vector": [-81.518325, 42.778667], "paragraph_keywords": ["expression", "gene", "data", "indicates"]}, {"paragraph_vector": [-82.443649, 42.518081], "paragraph_keywords": ["brain", "projects", "expression", "atlas"]}, {"paragraph_vector": [-80.212974, 44.571086], "paragraph_keywords": ["mapping", "data", "expression", "pipeline"]}, {"paragraph_vector": [-82.086143, 41.634513], "paragraph_keywords": ["registration", "series", "gene", "section"]}, {"paragraph_vector": [-84.568954, 42.469352], "paragraph_keywords": ["data", "cell", "pipeline", "grid"]}, {"paragraph_vector": [-80.696502, 41.154392], "paragraph_keywords": ["allen", "brain", "institute", "ogy"]}], "content": {}, "doi": "10.1145/1140491.1140535"}, {"uri": "286", "title": "Research on 3D Visualization of Crystal Molecular Structure Based on Augmented Reality", "timestamp": "2008", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Yong Chang"], "summary": "Abstract\u2015Augmented Reality(AR) is a technology that allows overlaying the computer-generated virtual information into the user\u2019s view of the real world, and with changes of his position and line of sight, the virtual objects will change accordingly. AR has many potential applications in industrial operations and academic research. First, this paper discusses the development of 3D visualization of crystal molecular structure. Next, the paper addresses the design of 3D macromolecule visualization based on AR. Then, the paper discusses 3D modeling of macromolecule. 3D model of crystal molecular is built with grid modeling in OpenGL. Finally, the paper makes a verifying research on it. The data of molecular was downloaded from internet database. The AR visualization system is established in ARToolKit. The result is stable, immersive and interactive.", "keywords": ["artoolkit", "world", "coordinate", "figure", "data", "structure", "y", "visualization", "macromolecule", "system", "bond", "simulation"], "document_vector": [136.302474, -36.315338], "paragraphs": [{"paragraph_vector": [115.931388, 11.814161], "paragraph_keywords": ["visualization", "simulation", "data", "structure"]}, {"paragraph_vector": [86.648918, -15.046072], "paragraph_keywords": ["bond", "model", "system", "coordinate"]}, {"paragraph_vector": [60.586181, 33.74221], "paragraph_keywords": ["coordinate", "y", "bond", "c"]}, {"paragraph_vector": [94.381271, 39.591342], "paragraph_keywords": ["camera", "artoolkit", "system", "y"]}], "content": {}, "doi": "10.1109/HPTCDL.2014.7"}, {"uri": "287", "title": "Visualizing protein structural superpositions and alignments with StructAlignViewer", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Harald Kattnig", "Aleksandar Poleksic"], "summary": "With emerging amounts of genomic data, research in the field of computational molecular biology is calling for new tools for displaying macromolecular structures. In this short article, we describe StructAlignViewer \u2013 a software package for visualizing protein structures and spatial superpositions. StructAlignViewer can be used in a standalone mode or in conjunction with an in-house software for optimal pairwise protein structure alignment. The program can display a single protein or a superposition of two proteins. The distinguishing feature of StructAlignViewer is its ability to visualize, in real time, the pairwise structural superpositions and alignments generated by the protein structure alignment algorithm MaxPairs. Keywords-protein structure; structural alignment; structural superposition; protein visualization", "keywords": ["missing", "protein", "alignment", "algorithm", "structalignviewer", "structure", "based", "input", "pdb", "transformation", "program", "maxpairs", "found", "acid", "epsilon", "residue"], "document_vector": [-157.570831, 56.270843], "paragraphs": [{"paragraph_vector": [140.743774, -58.167076], "paragraph_keywords": ["protein", "residues", "missing", "acid"]}, {"paragraph_vector": [170.758773, -65.850051], "paragraph_keywords": ["alignment", "protein", "transformation", "b"]}, {"paragraph_vector": [-79.494911, -47.710784], "paragraph_keywords": ["maxpairs", "transformations", "alignment", "algorithm"]}, {"paragraph_vector": [-93.083976, -34.828929], "paragraph_keywords": ["alignment", "protein", "time", "structalignviewer"]}, {"paragraph_vector": [162.279037, -63.930927], "paragraph_keywords": ["protein", "alignment", "structalignviewer", "information"]}], "content": {}, "doi": "10.1145/1233341.1233420"}, {"uri": "288", "title": "Representations that promote cognition: the case of chemistry visualization tools", "timestamp": "2006", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Mario Valle"], "summary": "Chemistry deals with structures too small for the naked eye; thus chemists rely heavily on graphical representations. An external observer could presume that the available chemistry visualization tools would be at the forefront regarding representation techniques and interaction methods. Our survey of 20 widely available non-commercial molecular visualization tools shows instead a high uniformity of representation methods and a general imbalance toward visualization for presentation only. To study alternatives to the current situation, we created \u201cviz laboratories\u201d: small, focused applications that highlight a specific chemistry visualization technique to collect practitioners feedback and to encourage them to think about new visualization paradigms. CR Categories: J.2 [Physical Sciences and Engineering]: Chemistry\u2014Visualization H.1.2 [User/Machine Systems]: Human information processing\u2014Perception I.3.6 [Methodology and Techniques]: Graphics data structures and data types\u2014Visual Metaphors", "keywords": ["perception", "visualization", "discovery", "chemistry", "representation", "understanding"], "document_vector": [175.276535, 4.04605], "paragraphs": [{"paragraph_vector": [137.694137, -30.353298], "paragraph_keywords": ["visualization", "chemistry", "representations", "perception"]}, {"paragraph_vector": [-158.180725, 13.540424], "paragraph_keywords": ["helps", "prototype", "integrate", "novel"]}], "content": {}, "doi": "10.1145/2945292.2945324"}, {"uri": "289", "title": "Guaranteed Quality Triangulation of Molecular Skin Surfaces", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Ho-Lun Cheng", "Xinwei Shi"], "summary": "We present an efficient algorithm to mesh the macromolecules surface model represented by the skin surface defined by Edelsbrunner. Our algorithm overcomes several challenges residing in current surface meshing methods. First, we guarantee the mesh quality with a provable lower bound of 21\u25e6 on its minimum angle. Second, we ensure the triangulation is homeomorphic to the original surface. Third, we improve the efficiency of constructing the Restricted Delaunay Triangulation(RDT) of smooth surfaces. We achieve this by constructing the RDT using the advancing front method without computing the Delaunay tetrahedrization of the sample points on the surfaces. The difficulty of handling the front collision problem is tackled by employing the Morse theory. In particular, we construct the Morse-Smale complex to simplify the topological changes of the front. Our implementation results suggest that the algorithm decrease the time of generating high quality homeomorphic skin mesh from hours to a few minutes. CR Categories: J.3 [Life and Medical Sciences]: Biology and Genetics; I.3.8 [Computer Graphics]: Computational Geometry and Object Modeling\u2014Curve, Surface, Solid, and Object Representation", "keywords": ["p", "triangulation", "saddle", "smale", "mesh", "operation", "algorithm", "triangle", "point", "section", "figure", "skin", "ball", "defined", "sphere", "quality", "change", "morse", "protecting", "delaunay", "surface", "advancing", "complex", "height", "edge", "model", "vertex", "condition", "b", "t", "boundary", "function", "q"], "document_vector": [-119.484878, -34.376003], "paragraphs": [{"paragraph_vector": [22.576473, -1.469741], "paragraph_keywords": ["model", "surface", "ms", "surfaces"]}, {"paragraph_vector": [-1.596303, 18.424369], "paragraph_keywords": ["surface", "algorithm", "mesh", "algorithms"]}, {"paragraph_vector": [-3.557825, 20.212684], "paragraph_keywords": ["advancing", "mesh", "skin", "triangulation"]}, {"paragraph_vector": [-5.749557, 25.568841], "paragraph_keywords": ["surface", "mesh", "complex", "smale"]}, {"paragraph_vector": [-4.541517, 23.882999], "paragraph_keywords": ["skin", "section", "surface", "b"]}, {"paragraph_vector": [5.909704, 17.946577], "paragraph_keywords": ["skin", "surface", "t", "dt"]}, {"paragraph_vector": [-5.670084, 24.215629], "paragraph_keywords": ["surface", "changes", "function", "triangulation"]}, {"paragraph_vector": [-8.671086, 26.782964], "paragraph_keywords": ["p", "points", "m", "level"]}, {"paragraph_vector": [-8.646893, 23.12788], "paragraph_keywords": ["points", "decrease", "protecting", "changes"]}, {"paragraph_vector": [-10.340411, 24.300706], "paragraph_keywords": ["point", "points", "saddle", "complex"]}, {"paragraph_vector": [-9.174794, 23.255393], "paragraph_keywords": ["lines", "points", "point", "contraction"]}, {"paragraph_vector": [-9.524247, 22.930631], "paragraph_keywords": ["triangulation", "complex", "weighted", "surface"]}, {"paragraph_vector": [-11.095802, 23.612045], "paragraph_keywords": ["points", "triangles", "operation", "skin"]}, {"paragraph_vector": [-14.241675, 17.086532], "paragraph_keywords": ["pt", "triangles", "operation", "departure"]}, {"paragraph_vector": [-13.569004, 17.125505], "paragraph_keywords": ["operation", "triangles", "added", "triangle"]}, {"paragraph_vector": [-12.842618, 17.833503], "paragraph_keywords": ["triangles", "bridge", "operation", "protecting"]}, {"paragraph_vector": [-8.101636, 21.085031], "paragraph_keywords": ["edge", "length", "skin", "surface"]}, {"paragraph_vector": [-7.61427, 24.295379], "paragraph_keywords": ["operations", "edge", "triangles", "ra"]}, {"paragraph_vector": [-0.898864, 17.841856], "paragraph_keywords": ["skin", "triangulation", "paper", "surface"]}, {"paragraph_vector": [-3.163435, 25.011962], "paragraph_keywords": ["algorithm", "height", "surfaces", "function"]}], "content": {}, "doi": "10.1145/2949550.2952773"}, {"uri": "290", "title": "nAPOLI: a graph-based strategy to detect and visualize conserved protein-ligand interactions in large-scale", "timestamp": "2019", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Alexandre V. Fassio", "Lucianna H. Santos", "Sabrina A. Silveira", "Rafaela S. Ferreira", "Raquel C. de Melo-Minardi"], "summary": "Essential roles in biological systems depend on protein-ligand recognition, which is mostly driven by specific non-covalent interactions. Consequently, investigating these interactions contributes to understanding how molecular recognition occurs. Nowadays, a large-scale data set of protein-ligand complexes is available in the Protein Data Bank, what led several tools to be proposed as an effort to elucidate protein-ligand interactions. Nonetheless, there is not an all-in-one tool that couples large-scale statistical, visual, and interactive analysis of conserved protein-ligand interactions. Therefore, we propose nAPOLI (Analysis of PrOtein-Ligand Interactions), a web server that combines large-scale analysis of conserved interactions in protein-ligand complexes at the atomic-level, interactive visual representations, and comprehensive reports of the interacting residues/atoms to detect and explore conserved non-covalent interactions. We demonstrate the potential of nAPOLI in detecting important conserved interacting residues through four case studies: two involving a human cyclin-dependent kinase 2 (CDK2), one related to ricin, and other to the human nuclear receptor subfamily 3 (hNR3). nAPOLI proved to be suitable to identify conserved interactions according to literature, as well as highlight additional interactions. Finally, we illustrate, with a virtual screening ligand selection, how nAPOLI can be widely applied in structural biology and drug design. nAPOLI is freely available at bioinfo.dcc.ufmg.br/napoli/.", "keywords": ["napoli", "data", "structure", "molecule", "analysis", "hydrogen", "interaction", "set", "cluster", "protein", "ligand", "number", "result", "process", "pdb", "based", "binding", "complex", "recognition", "study", "atom", "type", "composed", "file", "receptor", "conserved", "interacting", "supporting", "found", "user", "table", "residue"], "document_vector": [-54.53408, 42.798954], "paragraphs": [{"paragraph_vector": [-10.239636, -59.707244], "paragraph_keywords": ["ligand", "protein", "university", "recognition"]}, {"paragraph_vector": [-27.296785, -37.339187], "paragraph_keywords": ["interactions", "tools", "analysis", "conserved"]}, {"paragraph_vector": [-35.821155, -36.4235], "paragraph_keywords": ["interactions", "napoli", "ligand", "algorithms"]}, {"paragraph_vector": [-29.548768, -36.513782], "paragraph_keywords": ["protein", "number", "ligand", "users"]}, {"paragraph_vector": [-28.536478, -36.418304], "paragraph_keywords": ["napoli", "atoms", "remove", "ligands"]}, {"paragraph_vector": [-26.843692, -32.878166], "paragraph_keywords": ["interactions", "atoms", "ligand", "hydrogen"]}, {"paragraph_vector": [-30.210744, -35.579822], "paragraph_keywords": ["clustering", "set", "ligand", "residues"]}, {"paragraph_vector": [-63.650901, -63.67448], "paragraph_keywords": ["napoli", "data", "set", "ligand"]}, {"paragraph_vector": [-126.183868, -44.851497], "paragraph_keywords": ["interaction", "atoms", "interactions", "atom"]}, {"paragraph_vector": [-103.376045, -81.36026], "paragraph_keywords": ["table", "residues", "data", "interactions"]}, {"paragraph_vector": [-34.394508, -28.870292], "paragraph_keywords": ["ligand", "interaction", "residues", "type"]}, {"paragraph_vector": [-30.526197, -27.661396], "paragraph_keywords": ["interactions", "data", "set", "complexes"]}, {"paragraph_vector": [-29.861808, -28.50457], "paragraph_keywords": ["napoli", "hydrogen", "ligand", "interactions"]}, {"paragraph_vector": [-31.332414, -29.720378], "paragraph_keywords": ["cluster", "set", "ligands", "interactions"]}, {"paragraph_vector": [-33.126747, -29.621749], "paragraph_keywords": ["residues", "ligands", "clusters", "compounds"]}, {"paragraph_vector": [-31.132322, -28.387704], "paragraph_keywords": ["receptor", "ligands", "interactions", "supporting"]}, {"paragraph_vector": [-31.380792, -28.302484], "paragraph_keywords": ["ligands", "ligand", "protein", "process"]}, {"paragraph_vector": [-28.9358, -34.905288], "paragraph_keywords": ["interactions", "ligand", "ligands", "based"]}, {"paragraph_vector": [-36.507602, -36.637001], "paragraph_keywords": ["interactions", "napoli", "conserved", "analysis"]}, {"paragraph_vector": [-34.217159, -58.3255], "paragraph_keywords": ["capes", "analysis", "project", "design"]}], "content": {}, "doi": "10.1145/636593.636606"}, {"uri": "291", "title": "Illustrative Molecular Visualization with Continuous Abstraction", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Matthew van der Zwan", "Wouter Lueks", "Henk Bekker", "Tobias Isenberg"], "summary": "Molecular systems may be visualized with various degrees of structural abstraction, support of spatial perception, and \u2018illustrativeness.\u2019 In this work we propose and realize methods to create seamless transformations that allow us to affect and change each of these three parameters individually. The resulting transitions give viewers a dedicated control of abstraction in illustrative molecular visualization and, consequently, allow them to seamlessly explore the resulting abstraction space for obtaining a fundamental understanding of molecular systems. We show example visualizations created with our approach and report informal feedback on our technique from domain experts.", "keywords": ["perception", "data", "structure", "transition", "molecule", "stage", "space", "ribbon", "support", "protein", "shading", "bond", "fig", "sphere", "technique", "backbone", "based", "level", "geometry", "atom", "aspect", "cylinder", "use", "abstraction", "halo", "time", "rendering", "visualization"], "document_vector": [-163.818862, -29.863334], "paragraphs": [{"paragraph_vector": [96.254425, -20.692739], "paragraph_keywords": ["visualization", "data", "molecules", "based"]}, {"paragraph_vector": [88.749481, -7.822014], "paragraph_keywords": ["techniques", "visualization", "section", "abstraction"]}, {"paragraph_vector": [68.904922, 32.737842], "paragraph_keywords": ["abstraction", "molecules", "techniques", "approaches"]}, {"paragraph_vector": [97.78379, -12.644938], "paragraph_keywords": ["visualization", "structure", "techniques", "diagram"]}, {"paragraph_vector": [86.134567, -16.648239], "paragraph_keywords": ["abstraction", "abstractions", "fig", "visualization"]}, {"paragraph_vector": [89.195686, -7.426362], "paragraph_keywords": ["abstraction", "support", "axis", "stages"]}, {"paragraph_vector": [95.19371, -12.190361], "paragraph_keywords": ["abstraction", "spheres", "transition", "protein"]}, {"paragraph_vector": [71.518592, 2.912798], "paragraph_keywords": ["rank", "transition", "backbone", "geometry"]}, {"paragraph_vector": [70.638885, 12.275206], "paragraph_keywords": ["transition", "visualization", "geometry", "atoms"]}, {"paragraph_vector": [66.438499, 31.337858], "paragraph_keywords": ["abstraction", "occlusion", "object", "applied"]}, {"paragraph_vector": [94.454307, -12.043553], "paragraph_keywords": ["colors", "shading", "fig", "create"]}, {"paragraph_vector": [89.334297, -3.977065], "paragraph_keywords": ["emphasis", "abstraction", "fig", "illustrativeness"]}, {"paragraph_vector": [95.779708, -11.880994], "paragraph_keywords": ["abstraction", "visualization", "performance", "rates"]}, {"paragraph_vector": [98.219314, -14.713866], "paragraph_keywords": ["abstraction", "levels", "commented", "systems"]}, {"paragraph_vector": [96.455001, -14.484459], "paragraph_keywords": ["abstraction", "visualization", "molecules", "need"]}], "content": {}, "doi": "10.1145/2508244.2508258"}, {"uri": "292", "title": "VIEW \u2013 An Exploratory Molecular Visualization System with User-Definable Interaction Sequences", "timestamp": "0", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Lawrence D. Bergman", "Jane S. Richardson", "David C. Richardson", "Frederick P. Brooks"], "summary": "VIEW is an exploratory visualization system for studying the structures of molecules. The system supports a high degree of complex user interaction with the image. Visualizations are constructed by selecting drawing tools from a library. Each tool uses parameters obtained from interactive selection of on-screen geometry by the user, and from a molecular database. The system is based on a tight coupling of on-screen geometry with the underlying database. Using these links, tools can create truescale drawing elements that are constrained to database values. VIEW is highly extensible by the user or a paraprogrammer associated with the user. Drawing tools are written in a C-like programming language with constructs for managing databases, constructs for creating and altering geometry, as well as standard statements such as If-Else and For loops. An event-definition mechanism allows the user to describe actions to be performed when keys are depressed or dials turned. In addition, the user is able to specify conditional events \u2013 actions that are to be taken whenever a user-defined condition becomes true. These conditions are automatically evaluated by the system as part of event processing. Such conditional events allow simple simulations to be readily programmed. Applications of conditional events have included animations of protein binding activity, and an interactive \u201cflashlight\u201d which highlights structures as a cursor is steered through a molecule. The system includes a development environment complete with a WYSIWYG editor, an interactive debugger, and a set of innovative graphical debugging features. ____________________ VIEW has been installed for over a year in a protein crystallography laboratory at Duke University. Graduate students and faculty have used the system both for exploring molecular structures and for producing presentation graphics. These users have developed their own set of tools and made extensive use of the tool library. In January 1993, a beta-version of the software was released to a small set of laboratories in the US and Europe. It is now generally available. CR", "keywords": ["data", "group", "allows", "allowing", "language", "record", "richardson", "event", "representation", "interaction", "dial", "database", "monitor", "including", "protein", "wanted", "information", "point", "number", "screen", "system", "selected", "object", "figure", "process", "code", "example", "based", "drawing", "tool", "geometry", "line", "atom", "design", "specified", "display", "allow", "view", "development", "created", "image", "user", "function", "visualization", "provide", "provided"], "document_vector": [-164.980545, -11.854486], "paragraphs": [{"paragraph_vector": [144.749908, -32.771312], "paragraph_keywords": ["data", "visualization", "deal", "permission"]}, {"paragraph_vector": [140.988494, -22.06808], "paragraph_keywords": ["design", "visualization", "process", "user"]}, {"paragraph_vector": [139.06871, -22.088939], "paragraph_keywords": ["design", "visualization", "database", "data"]}, {"paragraph_vector": [137.500045, -14.723407], "paragraph_keywords": ["user", "geometry", "drawing", "data"]}, {"paragraph_vector": [139.860351, -13.494238], "paragraph_keywords": ["tool", "protein", "view", "drawing"]}, {"paragraph_vector": [108.421134, -26.414215], "paragraph_keywords": ["tool", "drawing", "sidechain", "richardson"]}, {"paragraph_vector": [136.742477, -11.240622], "paragraph_keywords": ["tool", "group", "rotation", "positions"]}, {"paragraph_vector": [139.876907, -11.493722], "paragraph_keywords": ["language", "database", "geometry", "tools"]}, {"paragraph_vector": [144.949539, -12.019429], "paragraph_keywords": ["language", "wanted", "geometry", "database"]}, {"paragraph_vector": [134.377014, -15.042241], "paragraph_keywords": ["record", "object", "view", "format"]}, {"paragraph_vector": [136.451385, -12.878367], "paragraph_keywords": ["groups", "record", "including", "subset"]}, {"paragraph_vector": [138.751159, -11.672346], "paragraph_keywords": ["tool", "geometry", "group", "groups"]}, {"paragraph_vector": [144.114822, -12.372413], "paragraph_keywords": ["event", "dial", "tool", "radius"]}, {"paragraph_vector": [141.277862, -14.58653], "paragraph_keywords": ["function", "event", "probe", "tool"]}, {"paragraph_vector": [146.55809, -16.639209], "paragraph_keywords": ["allowing", "debuggers", "user", "execution"]}, {"paragraph_vector": [121.491744, -35.071456], "paragraph_keywords": ["tool", "tools", "probe", "axis"]}, {"paragraph_vector": [123.781303, -36.108207], "paragraph_keywords": ["figure", "dipeptide", "segments", "spheres"]}, {"paragraph_vector": [141.503402, -14.797426], "paragraph_keywords": ["user", "visualization", "system", "language"]}, {"paragraph_vector": [144.374832, -12.687667], "paragraph_keywords": ["event", "set", "monitor", "system"]}, {"paragraph_vector": [145.990051, -14.842513], "paragraph_keywords": ["view", "research", "system", "visualization"]}, {"paragraph_vector": [110.691184, -28.151147], "paragraph_keywords": ["data", "view", "interaction", "variety"]}], "content": {}, "doi": "10.1145/2683483.2683503"}, {"uri": "293", "title": "HEARING BIOCHEMICAL STRUCTURES: MOLECULAR VISUALIZATION WITH SPATIAL AUDIO", "timestamp": "2017", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Terek R. Arce", "Kyla A. McMullen"], "summary": "Accurately perceiving the structure of biochemical molecules is key to understanding their function in biological systems. Visualization software has given the scientific and medical communities a means to study these structures in great detail; however, these tools lack an intuitive means to convey this information to persons with visual impairment. Advances in spatial audio technology have allowed for sound to be perceived in 3-dimensional space when played over headphones. This work presents the development of a novel computational tool that utilizes spatial audio to convey the three dimensional structure of biochemical molecules. Introduction & Motivation Biochemical macromolecules, such as proteins, nucleic acids, polysaccharides and lipids, comprise the fundamental building blocks of all life. These compounds derive their function in large part from their spatial structure. For this reason, the 3D visualization of molecules is critical to the study of biology, chemistry, and medicine. A number of tools have been developed for visualizing biochemical structures, such as Jmol [8] and RASMOL [6]. These graphics-based programs are designed for sighted users and are not accessible to those with visual impairment or alternate learning styles [10]. Gardner's theory of multiple intelligence states that spatial perception is just one of eight areas in which human intelligence can be understood [3]. For students and researchers with visual impairments, learning aids that take advantage of other intelligences must be used in order to comprehend such spatial structures [4]. For small molecules, tactile models can be created, while larger molecular structures rely on exporting atomic coordinates to text editors and braille output. Such systems have a steep learning curve for users with visual impairments and further demonstrate a need for new methods to convey molecular structural information. It has been shown that software that uses audio tones appeals to musical intelligence and assists in the localization of points in three-dimensional space for both sighted and visually impaired users [5]. Even so, to the best of our knowledge, there has been only one study on relaying biochemical structures with sound [2]. In that study, coordinate data was mapped to audio cues, in the form of different musical tones, to convey spatial orientations of biological and chemical molecules. Such a translation of sound to x, y and z coordinates is non-intuitive for all but the most practiced users, presenting an added learning impedance to visually impaired users. SIGACCESS Newsletter Issue 117 January 2017 Page 10 Proposal Rendering 3D (or spatial) audio, realized through the use of head-related transfer functions (HRTFs), to sonify biochemical structural data, will allow for the design of intuitive interfaces for visually impaired users to study molecules. HRTFs are filters through which audio can be passed in order to give a listener the illusion of the sound coming from a point in 3D space. HRTF-based 3D audio has been successfully used to help individuals in navigating inside rooms [12] and has been shown to improve depth perception in sighted users [9]. HRTFs are often measured and tuned for each individual. While our work does not create custom HRTFs, there are many publicly-available HRTF databases, such as CIPIC [1] and LISTEN [11], as well as methods to achieve approximate fittings [7]. Structural data used to create the molecular model can be found in a variety of online repositories, such as the Research Collaboratory for Structural Bioinformatics Protein Data Bank (RCSB PDB). By producing sounds at the coordinates of the atoms in a structure, a complete representation of the biochemical molecule can be created with spatial sounds. Ongoing Work Development of an initial software prototype, which allows users to gain a sense of depth and location of different secondary structures within a protein using spatial sounds created through HRTFs, has been completed (Fig. 1). Figure 1: SOP\u2019s design. Protein models are created from RCSB PDB files. A spatial sound source traverses the protein polypeptide chain, which is output to users via headphones. SOP includes a virtual reality visualization. Future iterations will include haptic feedback. The program is called SOP, short for the Sonification of Proteins. SOP positions a protein structure centered around the user\u2019s head. The spatial awareness produced by playing sounds along the polypeptide backbone of the protein, from amino to carboxyl ends, allows users to develop a mental image of its spatial structure. While the sounds indicate the location of \u03b1carbon atoms and overall structure of the protein chain, changes in intonation and sound type indicate whether the secondary structure at that location is an \u03b1-helix, \u03b2-sheet, or loop. This variation in sound allows for multiple levels of abstraction within the same spatial model. Initial experiments testing the SOP program with sighted and visually impaired users has been approved by the Internal Review Board. Tests will include a comparison of the accuracy in SIGACCESS Newsletter Issue 117 January 2017 Page 11 determining the form of different protein structures using audio only, audio combined with vision, and vision only. The time it takes users to find secondary structures within a protein under each of these conditions will also be measured. Evaluating user accuracy in determining the relative positions of multiple secondary structures within a protein will be carried out. Finally, a qualitative evaluation to determine the efficacy of the system design and intuitiveness of the user interface will be conducted. A pilot study was carried out to determine how well users could localize a moving sound target. Spatial audio was played from a source traversing the perimeter of a basic shape (e.g. circle, triangle, square). The users\u2019 ability to follow the target, via pointing their nose at the sound source, was determined by collecting virtual reality (VR) head-mounted display (HMD) head tracking information for multiple 60-second runs. During each test, users were only shown a black screen. Half the users were able to follow the moving target within 3m, over 50% of the time. Three out of four users could follow the targets within 4m, over 50% of the time. These results give rise to the belief that the concept may be extended to proteins. While the SOP program has undergone a number of improvements (Figure 2), its design continues to be iterated on. The current version aims to provide structural information about secondary structures. Proteins have multiple levels of abstraction, ranging from primary to quaternary, each of which has different design considerations when augmenting with spatial audio. More structural information will be added to create a comprehensive audio experience in future iterations. Beyond proteins, similar methods can be applied to sonify chemical compounds and cellular structures. This line of research leads to a number of questions that must be answered. For example, how are different bond types and orientations conveyed to a user at the molecular level with 3D audio? Design choices, such as what sound types should be used, and when and where continuous sounds should be employed, are also open questions that we plan to explore. Figure 2 (A) An early prototype of the SOP program created in MATLAB. (B) A view of part of a protein and a spatial sound source traversing it. This is a more recent prototype of SOP, developed in Unity, as seen through a VR HMD. (C) A protein, as seen in the scene view. Note the red outline indicating the maximum distance a sound can be heard from, giving a depth cue to its location. Contributions Through the development of SOP and future 3D audio biochemical sonification programs, the answers to fundamental questions in conveying biological and chemical structures through sound SIGACCESS Newsletter Issue 117 January 2017 Page 12 will be determined. We envision that future releases of existing tools, such as Jmol and RASMOL, will incorporate 3D audio in their programs in order to increase their accessibility. This work will lay a foundation for the development of these programs by determining best practices for the design of 3D audio applications for biochemical structure sonification. A number of uses for the developed software are possible for persons with visual impairments. In the field of education, integrating 3D audio into biochemical visualization tools can provide a means for teaching students structural biology, allowing for greater opportunity and advancement in scientific fields such as medicine and chemistry. Such tools will also give visually impaired researchers more independence in the laboratory. Another potential area of impact is in the biomedical and pharmaceutical industries. Companies are often interested in visualization tools to assist in their understanding of drug interactions. Using the SOP software, these tools can be enhanced to help biomedical engineers and pharmacologists quickly localize drug interaction sites and navigate large protein structures. Acknowledgments I would like to thank my advisor, Dr. Kyla McMullen for her support and guidance in undertaking this work. References 1. V. R. Algazi, R. O. Duda, D. M. Thompson, and C. Avendano. The CIPIC HRTF database. IEEE Workshop on Applications of Signal Processing to Audio and Acoustics, (October):99\u2013102, 2001. 2. T. J. Cordes, C. B. Carlson, and K. T. Forest. Tonal interface to macromolecules (TIMMol): A textual and tonal tool for molecular visualization. Biochemistry and Molecular Biology Education, 36(3):203\u2013208, 2008. 3. H. Gardner. Frames of Mind: The Theory of Multiple Intelligences. 1985. 4. R. G. Golledge. Reflections on Procedures for Learning Environments Without the Use of Sight. Journal of Geography, 104(October (3)):37\u201341, 2011. 5. S. W. Mereu and R. Kazman. Audio enhanced 3D interfaces for visually impaired users. ACM SIGCAPH Computers and the Physically Handicapped, (57):10\u201315, 1997. 6. R. A. Sayle and E. J. Milner-White. RASMOL: biomolecular graphics for all. Trends in biochemical sciences, 20(9):374\u2013376, 1995. 7. B. U. Seeber, H. Fastl, and Others. Subjective selection of non-individual head-related transfer functions. Proceedings of the 2003 International Conference on Auditory Display, pages 1\u20134, 2003. 8. The Jmol Team. Jmol: an open-source Java viewer for chemical structures in 3D., 2007. 9. A. Turner, J. Berry, and N. Holliman. Can the perception of depth in stereoscopic images be influenced by 3D sound ? Displays, 7863(February):786307\u2013786307\u201310, 2011. 10. I. Vekiri. What is the value of graphical displays in learning?, 2002. 11. O. Warusfel. Listen HRTF database. online, IRCAM and AK, Available: http://recherche. ircam. fr/equipes/salles/listen/index. html, 2003. 12. Z. Zhou, A. D. Cheok, X. Yang, and Y. Qiu. An experimental study on the role of software synthesized 3D sound in augmented reality environments. Interacting with Computers, 16(5):989\u20131016, 2004. SIGACCESS Newsletter Issue 117 January 2017 Page 13 About the Authors: Terek Arce is a Computer Engineering PhD student at University of Florida. He conducts research under the guidance of by Dr. Kyla McMullen as part of the SoundPad Lab. His research interests lie at the intersection of biochemistry and 3D audio, combining them to give visually impaired and sighted users new and intuitive computational learning tools. Dr. Kyla McMullen is an assistant professor in the University of Florida\u2019s Computer & Information Sciences & Engineering Department. Dr. McMullen\u2019s research interests are in the perception, applications, and development of 3D audio technologies. Her current projects include creating technologies for persons with visual impairments, evaluating the perception of 3D sound in virtual and augmented reality, and conveying spatial targets in real-time using 3D audio.", "keywords": ["hrtfs", "protein", "structure", "information", "tool", "research", "user", "sound", "impaired", "learning", "program", "design", "head", "audio"], "document_vector": [147.864135, -38.751174], "paragraphs": [{"paragraph_vector": [169.507064, 13.139212], "paragraph_keywords": ["users", "structures", "learning", "intelligence"]}, {"paragraph_vector": [177.099227, 13.613446], "paragraph_keywords": ["protein", "users", "structure", "sounds"]}, {"paragraph_vector": [177.322891, 14.639245], "paragraph_keywords": ["structures", "users", "design", "information"]}, {"paragraph_vector": [176.573089, 17.138776], "paragraph_keywords": ["tools", "audio", "programs", "mcmullen"]}, {"paragraph_vector": [172.118789, 16.469034], "paragraph_keywords": ["perception", "technologies", "reality", "sound"]}], "content": {}, "doi": "10.1145/2597652.2616584"}, {"uri": "294", "title": "Interactive Visualization of Protein Dynamics", "timestamp": "2000", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Henk Huitema", "Robert van Liere"], "summary": "The study of time dependent characteristics of proteins is important for gaining insight into many biological processes. However, visualizing protein dynamics by animating atom trajectories does not provide satisfactory results. When the trajectory is sampled with large times steps, the impression of smooth motion will be destroyed due to the effects of temporal aliasing. Sampling with small time steps will result in the camouflage of interesting motions. In this case study, we discuss techniques for the interactive 3D visualization of the dynamics of the Photoactive Yellow Protein. We use essential dynamics methods to filter out uninteresting atom motions from the larger concerted motions. In this way, clear and concise 3D animations of protein motions can be produced. In addition, we discuss various interactive techniques that allow exploration of the essential subspace of the protein. We discuss the merits of these techniques when applied to the analysis of the yellow protein. CR", "keywords": ["fluctuation", "motion", "exploration", "insight", "analysis", "dynamic", "space", "protein", "shown", "eigenvectors", "trajectory", "chromophore", "technique", "eigenvector", "atom", "animation", "pyp", "time", "function", "visualization"], "document_vector": [-85.632164, -4.268421], "paragraphs": [{"paragraph_vector": [129.462066, -53.024421], "paragraph_keywords": ["protein", "motions", "dynamics", "-"]}, {"paragraph_vector": [131.727325, -53.39582], "paragraph_keywords": ["protein", "motions", "chromophore", "atoms"]}, {"paragraph_vector": [92.387542, -26.412862], "paragraph_keywords": ["atom", "techniques", "showing", "visualize"]}, {"paragraph_vector": [150.337326, -69.658958], "paragraph_keywords": ["motions", "dynamics", "protein", "matrix"]}, {"paragraph_vector": [137.308349, -69.9803], "paragraph_keywords": ["eigenvector", "filtered", "trajectory", "time"]}, {"paragraph_vector": [142.030349, -62.112415], "paragraph_keywords": ["pyp", "protein", "atoms", "chromophore"]}, {"paragraph_vector": [135.231887, -64.403129], "paragraph_keywords": ["chromophore", "motions", "dynamics", "shown"]}, {"paragraph_vector": [161.994354, -67.947494], "paragraph_keywords": ["motions", "techniques", "visualization", "found"]}, {"paragraph_vector": [-86.581954, -88.863136], "paragraph_keywords": ["department", "university", "determine", "vortices"]}], "content": {}, "doi": "10.1145/1854776.1854840"}, {"uri": "295", "title": "A Collaborative Multi-View Virtual Environment for Molecular Visualization and Modeling", "timestamp": "2005", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Jeffrey W. Chastine", "Ying Zhu", "Jeremy C. Brooks", "Scott Owen", "Robert W. Harrison", "Irene T. Weber"], "summary": "Molecular modeling has been a long-standing research area for biologists. However, the existing molecular modeling software lacks strong support for collaborative research. In this paper we describe our effort to develop a collaborative multi-view virtual environment for molecular visualization and modeling. In our virtual environment, the users are able to visualize large molecular structures in real-time, create their own view, or share their view with others in the system. The system allows for individual or coordinated collaborative manipulation of the virtual molecular model. Our virtual environment is integrated with a molecular dynamics simulator, and therefore our system is not merely a visualization tool, but an environment where biologists can collaboratively construct their models and test their hypotheses.", "keywords": ["visualization", "data", "area", "molecule", "awareness", "manipulation", "orientation", "server", "interaction", "update", "dynamic", "including", "information", "working", "avatar", "system", "collaboration", "state", "scientist", "interface", "client", "design", "display", "model", "use", "view", "ammp", "user", "environment", "modeling", "hand"], "document_vector": [179.1678, -42.084602], "paragraphs": [{"paragraph_vector": [139.840682, 17.804796], "paragraph_keywords": ["modeling", "environment", "collaboration", "interaction"]}, {"paragraph_vector": [144.751739, 22.721879], "paragraph_keywords": ["system", "users", "use", "interest"]}, {"paragraph_vector": [143.902618, 19.227989], "paragraph_keywords": ["systems", "interface", "user", "dynamics"]}, {"paragraph_vector": [148.273986, 9.962576], "paragraph_keywords": ["user", "awareness", "views", "shared"]}, {"paragraph_vector": [150.52777, 11.335275], "paragraph_keywords": ["avatar", "user", "indicate", "environment"]}, {"paragraph_vector": [155.821441, 23.141397], "paragraph_keywords": ["server", "system", "orientation", "ammp"]}, {"paragraph_vector": [150.942703, 22.020351], "paragraph_keywords": ["data", "environment", "state", "client"]}, {"paragraph_vector": [149.511276, 12.024462], "paragraph_keywords": ["server", "molecule", "data", "allows"]}, {"paragraph_vector": [150.723983, 23.156171], "paragraph_keywords": ["data", "hmd", "clients", "tracking"]}, {"paragraph_vector": [147.522994, 15.662388], "paragraph_keywords": ["users", "environment", "model", "scientists"]}, {"paragraph_vector": [148.087127, 17.865119], "paragraph_keywords": ["user", "defined", "occurs", "data"]}, {"paragraph_vector": [147.258239, 11.651991], "paragraph_keywords": ["hand", "users", "view", "system"]}, {"paragraph_vector": [152.484252, 23.099193], "paragraph_keywords": ["system", "scientists", "molecule", "environment"]}, {"paragraph_vector": [146.871978, 21.504117], "paragraph_keywords": ["system", "updates", "bounding", "environment"]}, {"paragraph_vector": [146.496215, 20.819131], "paragraph_keywords": ["ability", "effectiveness", "visualize", "fields"]}], "content": {}, "doi": "10.1145/299246.299269"}, {"uri": "296", "title": "Coherent Culling and Shading for Large Molecular Dynamics Visualization", "timestamp": "2010", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sebastian Grottel", "Guido Reina", "Carsten Dachsbacher", "Thomas Ertl"], "summary": "Molecular dynamics simulations are a principal tool for studying molecular systems. Such simulations are used to investigate molecular structure, dynamics, and thermodynamical properties, as well as a replacement for, or complement to, costly and dangerous experiments. With the increasing availability of computational power the resulting data sets are becoming increasingly larger, and benchmarks indicate that the interactive visualization on desktop computers poses a challenge when rendering substantially more than millions of glyphs. Trading visual quality for rendering performance is a common approach when interactivity has to be guaranteed. In this paper we address both problems and present a method for high-quality visualization of massive molecular dynamics data sets. We employ several optimization strategies on different levels of granularity, such as data quantization, data caching in video memory, and a two-level occlusion culling strategy: coarse culling via hardware occlusion queries and a vertex-level culling using maximum depth mipmaps. To ensure optimal image quality we employ GPU raycasting and deferred shading with smooth normal vector generation. We demonstrate that our method allows us to interactively render data sets containing tens of millions of high-quality glyphs.", "keywords": ["buffer", "data", "stage", "set", "space", "dynamic", "method", "culling", "memory", "shading", "number", "quantization", "primitive", "approach", "result", "occlusion", "depth", "caching", "based", "query", "level", "performance", "frame", "geometry", "gpu", "use", "glyph", "grid", "vertex", "raycasting", "image", "rendering", "visualization", "cell"], "document_vector": [-0.019888, -65.841896], "paragraphs": [{"paragraph_vector": [122.900703, 74.504951], "paragraph_keywords": ["data", "simulations", "graphics", "depth"]}, {"paragraph_vector": [85.510871, 47.422298], "paragraph_keywords": ["data", "based", "rendering", "glyphs"]}, {"paragraph_vector": [83.982841, 82.318107], "paragraph_keywords": ["data", "depth", "based", "sets"]}, {"paragraph_vector": [30.546569, 81.044258], "paragraph_keywords": ["queries", "occlusion", "visibility", "rendering"]}, {"paragraph_vector": [66.892158, 77.558906], "paragraph_keywords": ["rendering", "data", "glyphs", "culling"]}, {"paragraph_vector": [33.8381, 86.636413], "paragraph_keywords": ["data", "culling", "reduce", "structure"]}, {"paragraph_vector": [13.268781, 83.7136], "paragraph_keywords": ["culling", "cells", "depth", "occlusion"]}, {"paragraph_vector": [-18.173639, 87.144744], "paragraph_keywords": ["results", "cells", "cell", "queries"]}, {"paragraph_vector": [80.754928, 84.513549], "paragraph_keywords": ["data", "vertex", "culling", "stage"]}, {"paragraph_vector": [118.742996, 66.18148], "paragraph_keywords": ["quantization", "data", "frame", "glyphs"]}, {"paragraph_vector": [27.902385, 70.233726], "paragraph_keywords": ["glyphs", "data", "glyph", "surface"]}, {"paragraph_vector": [85.383613, 58.507949], "paragraph_keywords": ["data", "glyphs", "sets", "test"]}, {"paragraph_vector": [82.839111, 65.971549], "paragraph_keywords": ["performance", "rendering", "data", "culling"]}, {"paragraph_vector": [98.246658, 80.050575], "paragraph_keywords": ["caching", "data", "culling", "number"]}, {"paragraph_vector": [144.557601, 84.693222], "paragraph_keywords": ["data", "image", "rendering", "results"]}, {"paragraph_vector": [89.865943, 55.125419], "paragraph_keywords": ["glyphs", "performance", "normals", "data"]}, {"paragraph_vector": [128.504013, 78.599723], "paragraph_keywords": ["data", "rendering", "image", "space"]}, {"paragraph_vector": [169.248641, 68.088432], "paragraph_keywords": ["distributed", "rendering", "loading", "core"]}], "content": {}, "doi": "10.1145/1450579.1450598"}, {"uri": "297", "title": "Network Visualization in Cell Biology", "timestamp": "2012", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Kasper Dinkla", "Michel A. Westenberg"], "summary": "Many of the processes known to take place in biological cells are analyzed in the form of different types of network. The complexity of these networks increases along with our knowledge of these processes, making their analysis more difficult. Network visualization is a powerful analysis method that will have to be developed further to deal with this complexity. This survey provides a brief overview of network visualization in general, followed by an in-depth discussion of its application to three network types specific to cell biology, namely gene regulatory, protein interaction, and metabolic networks. Finally, we discuss the difficulty of visually integrating these network types and trying to compare networks of cells that belong to different organisms.", "keywords": ["called", "data", "structure", "grn", "make", "analysis", "gene", "expression", "interaction", "space", "cluster", "known", "protein", "shown", "algorithm", "network", "approach", "scale", "neighborhood", "fig", "reaction", "technique", "pathway", "example", "clutter", "edge", "metabolic", "type", "layout", "use", "problem", "visualization", "cell", "link", "node"], "document_vector": [76.080215, 68.809875], "paragraphs": [{"paragraph_vector": [-130.613952, -37.614528], "paragraph_keywords": ["link", "matrices", "node", "networks"]}, {"paragraph_vector": [-133.467941, -33.957653], "paragraph_keywords": ["network", "processes", "type", "describe"]}, {"paragraph_vector": [-134.911132, -36.582492], "paragraph_keywords": ["gene", "expression", "genes", "nodes"]}, {"paragraph_vector": [-134.910919, -35.134403], "paragraph_keywords": ["genes", "gene", "cycles", "grns"]}, {"paragraph_vector": [-131.63417, -34.471275], "paragraph_keywords": ["network", "neighborhoods", "structure", "visualizations"]}, {"paragraph_vector": [-131.343826, -41.074649], "paragraph_keywords": ["node", "pathways", "grn", "metabolic"]}, {"paragraph_vector": [-136.21643, -37.309291], "paragraph_keywords": ["proteins", "protein", "set", "interact"]}, {"paragraph_vector": [-121.253463, -38.30421], "paragraph_keywords": ["protein", "proteins", "interaction", "pins"]}, {"paragraph_vector": [-133.632171, -35.29121], "paragraph_keywords": ["node", "edges", "link", "clutter"]}, {"paragraph_vector": [-131.039688, -38.278743], "paragraph_keywords": ["metabolic", "cell", "molecules", "reactions"]}, {"paragraph_vector": [-134.674331, -38.312648], "paragraph_keywords": ["networks", "network", "pathways", "pathway"]}, {"paragraph_vector": [-134.058654, -38.224925], "paragraph_keywords": ["layout", "pathway", "pathways", "algorithm"]}, {"paragraph_vector": [-133.57286, -37.734329], "paragraph_keywords": ["pathways", "networks", "network", "pathway"]}, {"paragraph_vector": [-134.018798, -36.051509], "paragraph_keywords": ["network", "visualization", "highlighted", "view"]}, {"paragraph_vector": [-132.061096, -37.896255], "paragraph_keywords": ["visualization", "networks", "notation", "network"]}, {"paragraph_vector": [-122.049751, -40.132602], "paragraph_keywords": ["network", "networks", "alignment", "species"]}, {"paragraph_vector": [-131.550491, -37.599323], "paragraph_keywords": ["networks", "d", "network", "edges"]}, {"paragraph_vector": [-126.505073, -37.327865], "paragraph_keywords": ["data", "networks", "visualization", "comparison"]}], "content": {}, "doi": "10.1145/1179622.1179842"}, {"uri": "298", "title": "Petascale Tcl with NAMD, VMD, and Swift/T", "timestamp": "2015", "rating": "0.0", "annotation": "", "tags": [], "authors": ["James C. Phillips", "John E. Stone", "Kirby L. Vandivort", "Timothy G. Armstrong", "Justin M. Wozniak", "Michael Wilde", "Klaus Schulten"], "summary": "Tcl is the original embeddable dynamic language. Introduced in 1990, Tcl has been the foundation of the scripting interface of the popular biomolecular visualization and analysis program VMD since 1995 and was extended to the parallel molecular dynamics program NAMD in 1999. The two programs together have over 200,000 users who have enjoyed for nearly two decades the stability and flexibility provided by Tcl. VMD users can implement or extend parallel trajectory analysis and movie rendering on thousands of nodes of Blue Waters. NAMD users can implement or extend simulation protocols and multiple-copy algorithms that execute unmodified on any supercomputer without the need to recompile NAMD. We now demonstrate the integration of the Swift/T high-performance parallel scripting language to enable high-level data flow programming in NAMD and VMD. This integration is achieved without modifying or recompiling either program since the Turbine execution engine is itself based on Tcl and is dynamically loaded by the interpreter, as is the platform-specific MPI library on which it depends.", "keywords": ["machine", "script", "runtime", "called", "data", "structure", "petascale", "partition", "program", "language", "master", "analysis", "replica", "scripting", "simulation", "set", "worker", "vmd", "support", "filesystem", "number", "run", "system", "programming", "implementation", "mpi", "code", "interpreter", "based", "work", "communication", "level", "performance", "parallel", "tool", "rank", "configuration", "written", "provides", "command", "file", "use", "model", "implemented", "t", "user", "function", "visualization", "tcl", "access", "exchange", "task"], "document_vector": [-33.382545, -6.613275], "paragraphs": [{"paragraph_vector": [-139.660842, 54.818988], "paragraph_keywords": ["researchers", "languages", "tcl", "access"]}, {"paragraph_vector": [-143.766799, 53.453853], "paragraph_keywords": ["t", "throughput", "platforms", "thousands"]}, {"paragraph_vector": [-142.613723, 54.103717], "paragraph_keywords": ["languages", "use", "compiled", "modeling"]}, {"paragraph_vector": [-146.118988, 51.803813], "paragraph_keywords": ["vmd", "analysis", "python", "developed"]}, {"paragraph_vector": [-178.861373, 48.668972], "paragraph_keywords": ["vmd", "analysis", "use", "parallel"]}, {"paragraph_vector": [173.624542, 48.400436], "paragraph_keywords": ["vmd", "scripting", "rendering", "users"]}, {"paragraph_vector": [-150.884796, 32.914073], "paragraph_keywords": ["configuration", "command", "tcl", "file"]}, {"paragraph_vector": [-149.810943, 32.079082], "paragraph_keywords": ["simulation", "tcl", "configuration", "scheduler"]}, {"paragraph_vector": [-149.937011, 32.42036], "paragraph_keywords": ["atoms", "simulation", "based", "implementation"]}, {"paragraph_vector": [-151.088699, 31.992805], "paragraph_keywords": ["master", "tcl", "file", "replicas"]}, {"paragraph_vector": [-152.084533, 35.415554], "paragraph_keywords": ["mpi", "communication", "exchange", "communicator"]}, {"paragraph_vector": [-151.880371, 36.216323], "paragraph_keywords": ["mpi", "partitions", "communication", "replicas"]}, {"paragraph_vector": [-150.805908, 39.138748], "paragraph_keywords": ["partition", "simulation", "t", "differences"]}, {"paragraph_vector": [-155.833511, 39.471897], "paragraph_keywords": ["t", "data", "tasks", "work"]}, {"paragraph_vector": [-151.018859, 38.484004], "paragraph_keywords": ["tcl", "t", "data", "mpi"]}, {"paragraph_vector": [-152.107406, 40.762432], "paragraph_keywords": ["defines", "file", "functions", "based"]}, {"paragraph_vector": [-152.009567, 39.150493], "paragraph_keywords": ["function", "tcl", "body", "code"]}, {"paragraph_vector": [-152.703689, 37.499526], "paragraph_keywords": ["runtime", "replica", "state", "filesystem"]}, {"paragraph_vector": [-150.722656, 37.33847], "paragraph_keywords": ["filesystem", "machine", "performance", "partition"]}, {"paragraph_vector": [-152.498657, 38.099544], "paragraph_keywords": ["mpi", "t", "fault", "programming"]}, {"paragraph_vector": [-165.30635, 55.189785], "paragraph_keywords": ["vmd", "based", "rank", "operate"]}, {"paragraph_vector": [-120.109146, 46.68148], "paragraph_keywords": ["application", "discus", "based", "grid"]}, {"paragraph_vector": [-140.874237, 52.613941], "paragraph_keywords": ["set", "code", "t", "proc"]}, {"paragraph_vector": [-151.664184, 38.91455], "paragraph_keywords": ["set", "newtemp", "doswap", "float"]}, {"paragraph_vector": [-145.657684, 57.846748], "paragraph_keywords": ["science", "supported", "illinois", "petascale"]}], "content": {}, "doi": "10.1145/2649387.2649428"}, {"uri": "299", "title": "Detection and Visualization of Anomalous Structures in Molecular Dynamics Simulation Data", "timestamp": "2004", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Sameep Mehta", "Raghu Machiraju", "John Wilkins"], "summary": "In this article we explore techniques to detect and visualize features in data from molecular dynamics (MD) simulations. Although the techniques proposed are general, we focus on silicon (Si) atomic systems. The first set of methods use 3D location of atoms. Defects are detected and categorized using local operators and statistical modeling. Our second set of exploratory techniques employ electron density data. This data is visualized to glean the defects. We describe techniques to automatically detect the salient iso-values for iso-surface extraction and designing transfer functions.We compare and contrast the results obtained from both sources of data. Essentially, we find that the methods of defect (feature) detection are at least as robust as those based on the exploration of electron density for Si systems. CR Categories: J.2 [Computer Applications]: Physical Sciences And Engineering \u2014Physics", "keywords": ["density", "transfer", "use", "list", "data", "value", "surface", "iso", "histogram", "bin", "defect", "lattice", "electron", "function", "atom", "visualization", "show", "figure"], "document_vector": [52.073307, -35.189952], "paragraphs": [{"paragraph_vector": [101.059608, 18.061624], "paragraph_keywords": ["defect", "transfer", "list", "atom"]}, {"paragraph_vector": [99.108131, 18.16616], "paragraph_keywords": ["density", "iso", "atoms", "use"]}, {"paragraph_vector": [98.047538, 18.014591], "paragraph_keywords": ["histogram", "value", "iso", "bins"]}, {"paragraph_vector": [99.2761, 16.558668], "paragraph_keywords": ["value", "iso", "transfer", "defect"]}, {"paragraph_vector": [96.91188, 17.367677], "paragraph_keywords": ["defects", "data", "atoms", "defect"]}, {"paragraph_vector": [48.024658, 2.159448], "paragraph_keywords": ["pages", "thank", "materials", "like"]}, {"paragraph_vector": [48.240486, 1.021605], "paragraph_keywords": ["data", "richie", "mining", "vol"]}], "content": {}, "doi": "10.1145/3334480.3383026"}, {"uri": "300", "title": "Automated Illustration of Molecular Flexibility", "timestamp": "2011", "rating": "0.0", "annotation": "", "tags": [], "authors": ["Aaron Bryden", "George N. Phillips"], "summary": "In this paper, we present an approach to creating illustrations of molecular flexibility using normal mode analysis (NMA). The output of NMA is a collection of points corresponding to the locations of atoms and associated motion vectors, where a vector for each point is known. Our approach abstracts the complex object and its motion by grouping the points, models the motion of each group as an affine velocity, and depicts the motion of each group by automatically choosing glyphs such as arrows. Affine exponentials allow the extrapolation of nonlinear effects such as near rotations and spirals from the linear velocities. Our approach automatically groups points by finding sets of neighboring points whose motions fit the motion model. The geometry and motion models for each group are used to determine glyphs that depict the motion, with various aspects of the motion mapped to each glyph. We evaluated the utility of our system in real work done by structural biologists both by utilizing it in our own structural biology work and quantitatively measuring its usefulness on a set of known protein conformation changes. Additionally, in order to allow ourselves and our collaborators to effectively use our techniques we integrated our system with commonly used tools for molecular visualization.", "keywords": ["illustration", "group", "structure", "motion", "depiction", "molecule", "analysis", "path", "flexibility", "set", "protein", "affine", "algorithm", "matrix", "trajectory", "nma", "point", "approach", "number", "mode", "collaborator", "result", "system", "clustering", "object", "case", "vector", "change", "technique", "example", "position", "arrow", "energy", "velocity", "field", "model", "use", "abstraction", "glyph", "error", "given", "created", "visualization", "provide"], "document_vector": [-109.375205, 6.693618], "paragraphs": [{"paragraph_vector": [153.344085, -64.119865], "paragraph_keywords": ["molecule", "analysis", "vector", "object"]}, {"paragraph_vector": [160.955276, -64.734947], "paragraph_keywords": ["requiring", "comparisons", "animations", "display"]}, {"paragraph_vector": [162.287658, -61.066875], "paragraph_keywords": ["mode", "objects", "analysis", "vector"]}, {"paragraph_vector": [163.570495, -63.296222], "paragraph_keywords": ["motion", "points", "rotation", "analysis"]}, {"paragraph_vector": [-179.640533, -65.701507], "paragraph_keywords": ["approach", "system", "motion", "motions"]}, {"paragraph_vector": [176.575393, -69.950538], "paragraph_keywords": ["motions", "models", "scale", "energy"]}, {"paragraph_vector": [168.875823, -66.333305], "paragraph_keywords": ["vectors", "nma", "molecule", "modes"]}, {"paragraph_vector": [152.68634, -62.723812], "paragraph_keywords": ["network", "use", "block", "acids"]}, {"paragraph_vector": [153.135375, -61.014324], "paragraph_keywords": ["field", "integration", "particles", "visualization"]}, {"paragraph_vector": [144.908111, -57.542114], "paragraph_keywords": ["motion", "arrows", "field", "use"]}, {"paragraph_vector": [158.010681, -63.497047], "paragraph_keywords": ["motion", "model", "vectors", "models"]}, {"paragraph_vector": [-122.80162, -69.451782], "paragraph_keywords": ["model", "aj", "xi", "matrix"]}, {"paragraph_vector": [-97.426361, -69.901733], "paragraph_keywords": ["model", "motions", "points", "vectors"]}, {"paragraph_vector": [-179.870208, -62.70298], "paragraph_keywords": ["model", "vectors", "motion", "group"]}, {"paragraph_vector": [-175.782211, -63.88776], "paragraph_keywords": ["motion", "matrix", "group", "vectors"]}, {"paragraph_vector": [-157.231414, -67.666168], "paragraph_keywords": ["groups", "clustering", "model", "algorithms"]}, {"paragraph_vector": [-109.132942, -67.092254], "paragraph_keywords": ["groups", "group", "error", "algorithm"]}, {"paragraph_vector": [-179.466674, -66.95211], "paragraph_keywords": ["points", "trajectories", "motion", "abstraction"]}, {"paragraph_vector": [-179.75212, -65.441383], "paragraph_keywords": ["trajectory", "starting", "group", "motion"]}, {"paragraph_vector": [-177.382568, -69.740257], "paragraph_keywords": ["path", "group", "distance", "starting"]}, {"paragraph_vector": [-174.857879, -66.828964], "paragraph_keywords": ["ribbon", "arrows", "groups", "rotation"]}, {"paragraph_vector": [-174.296646, -66.031974], "paragraph_keywords": ["motion", "group", "error", "communicating"]}, {"paragraph_vector": [-174.672531, -64.136566], "paragraph_keywords": ["motion", "group", "component", "model"]}, {"paragraph_vector": [152.815704, -61.885631], "paragraph_keywords": ["system", "paper", "examples", "depictions"]}, {"paragraph_vector": [160.574569, -65.741523], "paragraph_keywords": ["pymol", "system", "illustrations", "groups"]}, {"paragraph_vector": [151.740737, -61.047008], "paragraph_keywords": ["motion", "structures", "protein", "illustrations"]}, {"paragraph_vector": [178.755187, -67.815917], "paragraph_keywords": ["clustering", "technique", "cases", "time"]}, {"paragraph_vector": [172.668014, -67.006362], "paragraph_keywords": ["motions", "models", "approach", "motion"]}, {"paragraph_vector": [175.07843, -64.867294], "paragraph_keywords": ["approach", "motions", "curvature", "motion"]}], "content": {}, "doi": "10.2312/VISSYM19991025"}, {"uri": "301", "title": "empty", "timestamp": "2001", "rating": "0.0", "annotation": "", "tags": [], "authors": [], "summary": "large computers have become one of the mainstays for investigating the structure\u2013function relationship in biomolecules. Using statistical algorithms, we get several snapshots (or a trajectory) of the molecule. The distribution of the snapshots approximates the expected distribution of molecular shapes in actual biochemical processes. By looking for conformational ensembles\u2014 groups of configurations that have a similar geometrical shape\u2014and transition paths between them, biochemists can learn about the molecular bases of biochemical processes. Such an understanding has many practical applications\u2014for example, in designing more efficient medical drugs. Identifying typical shapes in such a large set of molecular configurations is itself a difficult task, and much literature on the topic exists.1\u20133 In most simulations, researchers focus on a few characteristic quantities, such as some helical angles in the molecule, and monitor the change of these quantities in the simulation. This approach requires some advance knowledge about the parts of the molecule that are important to the dynamics. From the computational science viewpoint, however, it\u2019s more interesting to investigate methods that don\u2019t use such a priori knowledge and rely solely on the information inherent in the trajectory. Such an approach not only opens the possibility of a completely automatic analysis but also serves as an example of how to use techniques from information visualization and statistical cluster analysis in molecular simulations. Our research is part of a larger effort to identify essential degrees of freedom in molecular simulations.4 Based on a feature vector that captures the geometrical shape of each molecular configuration in the trajectory, we introduce a measure of conformational distance and use it to visualize the trajectory in a plane and identify clusters of similar configurations. In this article, we apply our method to a trajectory from an adaptive-temperature hybrid Monte Carlo simulation5 of the molecule adenylyl(3\u2019-5\u2019)cytidylyl(3\u2019-5\u2019)cytidin\u2014also called", "keywords": ["representative", "value", "d", "analysis", "distance", "similarity", "cluster", "m", "equation", "matrix", "point", "object", "figure", "vector", "map", "n", "feature", "problem", "visualization"], "document_vector": [-45.55231, 9.272877], "paragraphs": [{"paragraph_vector": [-145.430343, -78.813003], "paragraph_keywords": ["distance", "dimensions", "d", "equation"]}, {"paragraph_vector": [-66.31253, -13.948665], "paragraph_keywords": ["n", "m", "matrix", "distances"]}, {"paragraph_vector": [-91.770645, -75.345573], "paragraph_keywords": ["points", "equation", "distance", "representatives"]}, {"paragraph_vector": [-132.296508, -80.148422], "paragraph_keywords": ["objects", "cluster", "similarity", "n"]}, {"paragraph_vector": [-73.385437, -13.33017], "paragraph_keywords": ["problem", "matrix", "graph", "finding"]}, {"paragraph_vector": [-68.412094, -11.782141], "paragraph_keywords": ["cluster", "measure", "splitting", "n"]}, {"paragraph_vector": [-106.553619, -78.014213], "paragraph_keywords": ["cluster", "points", "figure", "split"]}, {"paragraph_vector": [153.545166, -64.897003], "paragraph_keywords": ["visualization", "map", "user", "changes"]}], "content": {}, "doi": "10.2312/VCBM.20141188"}]